<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="description" content="E. coli and Salmonella: tables from chapter 139 (selections giving rise to mutants, genes for which selections exist) in a parsable format, with links to EcoCyc and PDFs">
	<title>Mutant selection tables for E. coli and Salmonella</title>
	<link rel="stylesheet" href="style.css">
</head>
<body>

<div id="menu"><nav><ul><li><a href="#about-this-document">About</a></li><li><a href="#table1-selections-giving-rise-to-mutants">Selections giving rise to mutants</a></li><li><a href="#table2-genes-for-which-selections-exist">Genes for which selections exist</a></li><li><a href="#references-table">References</a></li><li><a href="#how-to-contribute">Contribute</a></li></ul></nav></div>
<div id="main">
<div id="about">
<h1><span class="anchor" id="about-this-document"></span>About this document</h1>

<p>
<a href="https://openlibrary.org/books/OL8575023M"><em>Escherichia coli and Salmonella</em></a> is the bible of microbiology. It contains data daily used by many researchers. The chapter 139 (<em>Mutant Selections Linking Physiology, Inhibitors, and Genotypes</em>), by Robert A. LaRossa, contains two famous tables listing all genes for which a positive or negative selection has been reported in the litterature. While a pdf is available from <a href="http://www.asmscience.org/files/Chapter_139_Mutant_Selections_Linking_Physiology.pdf">ASM</a>, its format makes it hard to use. Moreover it contains small mistakes that could easily be corrected (eg typos in the reference table). Eventually academics may want to collectively keep updating these tables with more recent litterature.
</p>
<p>
For these reasons, I extracted the two tables from the pdf and created this easily parsable and updatable webpage, with links between gene names and EcoCyc, and between reference number and full bibliographic information (including a pdf when available).
</p>
<p>
The raw data can also be downloaded as csv files:
</p>
<ul>
  <li>Selections giving rise to mutants: <a href="#table1-selections-giving-rise-to-mutants">view online</a> or <a href="table1.csv">download raw data</a></li>
  <li>Genes for which selections exist: <a href="#table2-genes-for-which-selections-exist">view online</a> or <a href="table2.csv">download raw data</a></li>
  <li>Bibliographic references: <a href="#references-table">view online</a> or <a href="references.csv">downlad raw data</a></li>
</ul>
<p>
And importantly, anybody can <a href="#how-to-contribute">contribute</a> to this document!<br>
<p>
</div>


<div id="table1"><table>
<caption><span class="anchor" id="table1-selections-giving-rise-to-mutants"></span><h1>Selections giving rise to mutants of <em>E. coli</em> and <em>S. typhimurium</em></h1></caption>
<col class="selection">
<col class="gene">
<col class="type">

<tr><th>Selection purpose and technique</th><th>Mutant gene
</th><th>Type</th></tr>
<tr><td>Acetate + gluconate as C source in <em>ppc</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG12630">gntR
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Acetate + gluconate as C source in <em>ppc</em></td><td>gntS
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Acetyl(<em>N</em>)histidine + ornithine satisfaction of <em>his</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10064">argB
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Acetyl(<em>N</em>)histidine + ornithine satisfaction of <em>his</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10065">argC
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Acetyl(<em>N</em>)histidine + ornithine satisfaction of <em>his</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11286">argE
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Acetyl(<em>N</em>)histidine + ornithine satisfaction of <em>his</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11223">argH
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Acetylhistidine satisfaction of <em>his</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10070">argR
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Acetyl(<em>N</em>)lactonate as sole N source </td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10525">lacI, operator
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Acetylmethionine satisfaction of <em>met</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10070">argR
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Acetylornithine + uracil satisfaction of <em>car</em> (<em>pyrA</em>)</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10063">argA
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Acetylornithine + uracil satisfaction of <em>car</em> (<em>pyrA</em>)</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10064">argB
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Acetylornithine + uracil satisfaction of <em>car</em> (<em>pyrA</em>)</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10065">argC
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Acetylornithine + uracil satisfaction of <em>car</em> (<em>pyrA</em>)</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10066">argD
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Adenosine as C source in <em>upp deoD</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11146">xapR
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Aminobutyrate as N source</td><td>gabC
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Arabinitol (D) resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10615">mtlA
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Arabinose (L) as C source in <em>cya</em> or <em>crp</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10896">rpoD
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Arabinose (L) as C source in presence of <em>lac</em> operon inducer</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10526">lacY
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Arabinose (D) as C source</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10348">fucA
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Arabitol as C source</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG12414">gatACD
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Arabitol (D) as C source in <em>fuc</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=G26">dgd
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Arbutin as a C source</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10114">bglBC
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Arbutin as C source)</td><td>osmZ (bglY)
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Arginine as N source</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10452">hisP
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Arginine-independent growth of <em>pyrH</em> in presence of uracil</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10805">pyrB
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Arginine-independent growth of <em>pyrH</em> in presence of uracil</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10806">pyrC
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Arginine-independent growth of <em>pyrH</em> in presence of uracil</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10807">pyrD
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Arginine-resistant, proline-independent growth of ∆<em>pro</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10070">argR
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>β-Glycerol-phosphate as C source in presence of high phosphate</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10727">phoA
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>β-Glycerol-phosphate as C source in presence of high phosphate</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10733">phoR
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>β-Glycerol-phosphate as C source in presence of high phosphate</td><td>phoS
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>β-Glycerol-phosphate as C source in presence of high phosphate</td><td>phoT
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Biotin sulfoxide utilization in <em>chl bio</em></td><td>chlE
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Branched-chain amino acid satisfaction of ∆<em>ilvDC hisT</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10493">ilvA
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Butyrate as C source in <em>fadR</em>(Con)</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11668">atoC
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Butyrate or valerate as better C source</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10491">iclR
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Cellobiose as C source</td><td>celABCDF
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Chemostat growth improved</td><td>mut
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Citrate as C source</td><td>cit
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Citrate as C source in presence of low cAMP</td><td>cpd
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>D-Amino acid satisfaction of amino acid auxotrophy</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11407">dadA
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>DAP-independent growth of ∆(<em>mal-asd</em>) in the presence of serine, methionine,and glycine D-Histidine + glycylglutamine satisfaction of <em>his</em> auxotrophs</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10383">glnA
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Decanoate as C source</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10281">fadR
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Deoxyadenosine (low) satisfaction of purine requirement</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10030">add
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Deoxyadenosine + hypoxanthine satisfaction of purine requirement</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10030">add
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Diamino (2,6)purine satisfaction of purine auxotrophy</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10223">deoR
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Erythromycin growth dependence</td><td>mac
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Ethanol as C source</td><td>adh
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Ethyleneglycol as C source in propanediol utilizer</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10348">fucA
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Fructose as C source in <em>ptsF</em> or <em>ptsM</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10971">srlD
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Fructose 1-phosphate as C source</td><td>uhpR
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Fucose (L) as C source in propanediol utilizer</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10348">fucA
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Fucose + arabinose-supported growth in <em>galP</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10592">mglA
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Fucose + arabinose-supported growth in <em>galP</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10593">mglB
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Fucose + arabinose-supported growth in <em>galP</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10594">mglC
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Galactose as C source for <em>gal</em> (leader)::IS</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10845">rho
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Galacturonide as C source</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11055">uidA
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Galacturonide as C source</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=G6867">uidR
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Glucanate as C source in <em>eda</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10257">edd
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Glucosamine as C source</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10787">ptsG
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Glucosamine 6-phosphate as C source</td><td>uhpR
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Glucosamine as anaerobic C source in <em>ptsG</em></td><td>dgsA
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Glucose 1-phosphate as C source</td><td>uhpR
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Glucose as C source in ∆<em>ptsHI</em></td><td>galC
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Glucose as C source in <em>ppc</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10491">iclR
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Glucose as C source in ∆<em>ptsHI</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10364">galR
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Glucosides (β) as C source</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10423">gyrA (hisU)
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Glucosides (β) as C source</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10424">gyrB (hisW)
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Glutamate as C source</td><td>gadS
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Glutamate as C source</td><td>gltH
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Glutamate as C source</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10406">gltS
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Glutamate as C source at 42°C</td><td>gltR
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Glutamine as C source</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10388">glnP
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Glycerol as C source in <em>ppc</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10491">iclR
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Glycerol + gluconate as C source in <em>eda</em></td><td>gntM
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Glycerol + methylglucuronide as C source in <em>eda</em></td><td>gurBCD
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Glycerol 3-phosphate as C source in <em>glpT</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11046">ugpAB
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Hemin satisfaction of <em>hemA</em> auxotroph</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10428">hemB
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Hemin satisfaction of <em>hemA</em> auxotroph</td><td>rfa
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Hexuronate as C source in noninducible strain</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG12739">exuR
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Histidinal satisfaction of <em>his</em> auxotroph requirement at 30°C</td><td>his structural genes
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Histidinal satisfaction of <em>his</em> auxotroph requirement at 30°C</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10449">hisG
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Histidine (D) satisfaction of <em>his</em> auxotroph requirement</td><td>his structural genes
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Histidine (D) satisfaction of <em>his</em> auxotroph requirement</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10449">hisG
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Histidine (D) satisfaction of <em>his</em> auxotroph requirement</td><td>dhuA
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Histidine as C source</td><td>hut
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Histidine as N source</td><td>hut
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Improved growth in <em>icd</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10402">gltA
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Inosine as C source in <em>upp deoD</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11146">xapR
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Inosine as improved C source</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10223">deoR
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Keto(2)-3-deoxygalactonate as C source</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=G7003">kdgR
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Keto(2)-3-deoxygalactonate as C source</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=G7789">dgoR
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Lactate + fumarate as anaerobic energy source</td><td>chlABDE
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Lactate + nitrate as anaerobic energy source (<em>chl</em> to <em>chl</em>+)</td><td>chlE
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Lactitol as C source</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10525">lacI, operator
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Lactobionic acid as C source</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10525">lacI, operator
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Lactobionic acid as C source</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10527">lacZ
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Lactose utilization after conjugation of F′<em>lac</em> into <em>Salmonella</em> sp.</td><td>hsd
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Lactose utilization in ∆<em>lacZ</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10252">ebgA
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Leucine (D) satisfaction of <em>leu</em> auxotrophs</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10547">lrp (livR, lstR, lss, mbl, oppl)
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Lyxose (D) as C source</td><td>manC
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Malate (L) as improved C source</td><td>dct
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Malate (D) as C source</td><td>dml
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Maltose as C source in <em>malT</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10560">malP
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Maltose as C source in <em>malT</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10561">malQ
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Maltose utilization in <em>malT</em></td><td>bymA
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Mannitol (limiting) as C source</td><td>mtlC
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Mannose as C source</td><td>ptdG
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Melibiose as C source</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10525">lacI, operator
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Melibiose as C source and acetyl(<em>N</em>)lactonate as N source</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10525">lacI, operator
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Methyl-β-galactoside-supported growth</td><td>mglD
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Methylgalacturonide as C source</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11055">uidA
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Methylgalacturonide as C source</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=G6867">uidR
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Methylgalacturonide as C source</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG20249">uxuR
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Methyl(β)glucoside as C source</td><td>bglT
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Methylglucuronide + glycerol-supported growth</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11055">uidA
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Neolactose as C source</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10525">lacI, operator
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Nicotinamide as sole N source</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11135">pncA
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Phenylgalactoside as C source</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10525">lacI, operator
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Proline as sole N source with glucose as C source</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10801">putA
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Proline-independent growth of <em>proAB</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10066">argD
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Propanediol as C source</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10348">fucA
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Propionate as C source</td><td>prp
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Raffinose as C source in <em>lacI</em></td><td>mel (generic)
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Raffinose as C source</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10525">lacI, operator
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Ribitol utilization</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG12414">gatACD
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Salicin as C source</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10114">bglB,C
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Salicin as C source</td><td>osmZ
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Serine (L) as C source</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10163">cpxA
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Serine (L) as C source</td><td>ecfB
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Sorbose (L) as C source in crosses with wild strains</td><td>sorAT
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Streptomycin independence of <em>rpsL</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10903">rpsD
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Streptomycin independence of <em>rpsL</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10904">rpsE
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Succinate as improved C source</td><td>cpd
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Succinate-independent growth in <em>lpd</em></td><td>sdh
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Sucrose as C source</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10249">dsdA
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Sucrose as C source</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG13161">dsdC
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Threonine as N source</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10493">ilvA
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Tyramine as N source</td><td>tyn
</td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Uridine as C source, improved utilization</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10200">cytR
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Valerate as C source in <em>fadR</em>(Con)</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11668">atoC
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Xylitol as C source in propanediol utilizer</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10348">fucA
</a></td><td>Expansion of metabolic capacity</td></tr>
<tr><td>Acid resistance</td><td>atr
</td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Acid resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10729">phoE
</a></td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Arsenate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10401">glpT
</a></td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Arsenate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10733">phoR
</a></td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Arsenate resistance</td><td>phoS
</td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Arsenate resistance</td><td>phoT
</td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Arsenate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10782">pstABCS
</a></td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Arsenate resistance</td><td>pit
</td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Azide resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10098">atpA–atpE
</a></td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Azide resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10184">cysB
</a></td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Azide resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10192">cysK
</a></td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Azide resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10936">secA
</a></td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Chlorate resistance</td><td>chlABCDEG
</td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Chlorate resistance</td><td>narCG
</td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Chlorate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10639">narH
</a></td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Chromate + selenate resistance</td><td>cysL
</td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Chromate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10183">cysA
</a></td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Chromate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10184">cysB
</a></td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Chromate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10185">cysC
</a></td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Chromate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10186">cysD
</a></td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Chromate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10189">cysH
</a></td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Chromate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10190">cysI
</a></td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Chromate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10191">cysJ
</a></td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Copper resistance in <em>ompC</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10671">ompF
</a></td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Copper resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10672">ompR
</a></td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Diazoborine resistance</td><td>envM
</td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Lithium resistance (multicopy)</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10652">nhaA
</a></td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Lithium-resistant utilization of melibiose</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10578">melB
</a></td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Manganese resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11463">corABCD
</a></td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Manganese resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10359">fur
</a></td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Manganese resistance</td><td>mng
</td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Nickel resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11463">corABCD
</a></td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Osmotolerance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10164">crp
</a></td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Osmotolerance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10170">cyaA
</a></td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Osmotolerance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10679">osmB
</a></td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Osmotolerance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10767">proA
</a></td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Osmotolerance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10768">proB
</a></td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Peroxide resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10681">oxyR
</a></td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Selenate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10183">cysA
</a></td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Selenate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10184">cysB
</a></td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Selenate resistance in <em>cysM</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10184">cysB
</a></td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Selenate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10192">cysK
</a></td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Selenate resistance</td><td>cysL
</td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Selenite resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10192">cysK
</a></td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Selenite resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10418">gshA
</a></td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Selenite resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10419">gshB
</a></td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Tellurite resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10728">phoB
</a></td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Tributyl tin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10098">atpA–atpE
</a></td><td>Resistance to inorganic chemicals</td></tr>
<tr><td>Acetylnorvaline resistance in <em>argR</em>(Con)</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11286">argE
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Acetylnorvaline resistance in <em>argR</em>(Con)</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10070">argR
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Acridine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10238">dnaE
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Actinomycin D resistance in the presence of EDTA</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10738">pldA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Adenine resistance in <em>hpt gpt</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10051">apt
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Adenine resistance in <em>hpt gpt</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10800">purR
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Alafosfalin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10694">pepA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Alafosfalin resistance</td><td>tpp
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Alanyl-2-aminopropionate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10674">oppA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Albicidin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11035">tsx
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Albomycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10271">exbB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Albomycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10302">fhuA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Albomycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10303">fhuB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Albomycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10696">pepN
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Albomycin resistance</td><td>sidCF
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Albomycin resistance</td><td>sidK
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Albomycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11012">tonB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Aldohexuronate resistance in <em>eda</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG12738">exuT
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Aldohexuronate resistance in <em>eda</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG12739">exuR
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Allyl alcohol resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10031">adhE
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Amidinopenicillin tolerance</td><td>sloB
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Amikacin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10163">cpxA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Amino(2)purine resistance in <em>dam</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10624">mutH
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Amino(2)purine resistance in <em>dam</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11281">mutL
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Amino(2)purine resistance in <em>dam</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10625">mutS
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Amino(4)phenylalanine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10078">aroF
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Amino(4)phenylalanine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11039">tyrA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Amino(4)phenylalanine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11042">tyrR
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Amino(6)nicotinamide resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10630">nadA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Amino(6)nicotinamide resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=G6350">nadD
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Amino(6)nicotinamide resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11135">pncA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Amino(6)nicotinamide resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10742">pncB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Amino(6)nicotinamide resistance</td><td>pncX
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Amino(6)nicotinate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10742">pncB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Aminobutyrate resistance</td><td>ilv structural genes
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Aminobutyrate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10494">ilvB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Aminobutyrate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10499">ilvH
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Aminobutyrate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10500">ilvI
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Aminoethylcysteine (thialysine) resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10550">lysC
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Aminoethylcysteine (thialysine) resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11337">lysP
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Aminoethylcysteine (thialysine) resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10552">lysS
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Aminoglycoside resistance</td><td>nek
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Amino(2)hydroxy(3)pentoate resistance</td><td>thr structural genes
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Aminopterin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11002">thyA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Aminotriazole resistance</td><td>amtA, amtB
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Aminotriazole resistance</td><td>his structural genes
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Aminotriazole resistance in <em>relA</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10424">gyrB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Ampicillin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10040">ampC
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Ampicillin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10041">ampD
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Ampicillin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10042">ampE
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Ampicillin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10269">envZ
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Ampicillin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11319">galU
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Ampicillin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10443">hipA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Ampicillin resistance</td><td>hipQ
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Ampicillin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10670">ompC
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Ampicillin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10671">ompF
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Ampicillin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10672">ompR
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Ampicillin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10789">ptsI
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Ampicillin resistance</td><td>rfa (general)
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Ampicillin resistance</td><td>tolD
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Ampicillin resistance</td><td>tolE
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Arabinitol (D) resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10615">mtlA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Arbitol (D) resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG12414">gatACD
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Arbutin resistance in <em>dgk</em></td><td>mdoB
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Arginine-resistant, proline-independent growth of ∆<em>pro</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10070">argR
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Auroventin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10098">atpA–atpE
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Aza(5)cytidine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10211">dcm
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Azadeoxycytidine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10137">cdd
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Aza(6)uracil resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10414">gpt
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Aza(8)guanine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10650">ndk
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Aza(8)guanine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11332">upp
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Azaleucine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10084">aroP
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Azaleucine resistance</td><td>azl
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Azaleucine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11008">tolB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Azaleucine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10532">leuS
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Azaleucine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10537">livG
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Azaleucine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10538">livH
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Azaserine + tryptophan resistance</td><td>dhuA
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Azaserine + tryptophan resistance in <em>dhuA</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG12124">hisJ
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Azaserine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10084">aroP
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Azaserine resistance</td><td>azaAB
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Azaserine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10183">cysA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Azaserine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10184">cysB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Azaserine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10185">cysC
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Azaserine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10186">cysD
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Azaserine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10187">cysE
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Azaserine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10188">cysG
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Azaserine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10189">cysH
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Azaserine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10190">cysI
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Azaserine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10191">cysJ
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Azaserine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10192">cysK
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Azaserine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10193">cysM
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Azaserine resistance</td><td>mut (generic)
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Azetidine carboxylate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10768">proB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Azetidine carboxylate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10801">putA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Azetidine carboxylate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10802">putP
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Azidothymidine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10994">tdk
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Bacilysin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10248">dppA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Bacitracin resistance</td><td>rfa
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Bacteriocin resistance</td><td>tol
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Bacteriocin 4-59 resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10669">ompA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Bacteriocin 4-59 resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11012">tonB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Bacteriocin JF246 resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10669">ompA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Baikiain resistance in constitutive background</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10801">putA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>β-Glycerol-phosphate resistance in <em>glpD</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10727">phoA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Beta-lactam conjugate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10302">fhuA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Beta-lactam resistance (see mecillinam resistance)</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10164">crp
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Beta-lactam resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10170">cyaA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Beta-lactam resistance</td><td>lyt
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Beta-lactam resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10034">alaS
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Beta-lactam resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10071">argS
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Bialaphos resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10248">dppA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Borrelidin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11001">thrS
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Bromodeoxyuridine + UV light resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11058">ung
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Cadaverine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10132">cadB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Caffeine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG30029">glnV
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Calmodulin inhibitor resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG30052">leuW
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>cAMP + glucose 6-phosphate + D-xylose + L-arabinose resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10164">crp
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>cAMP + glucose 6-phosphate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10164">crp
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Camphor resistance</td><td>mbrABCD
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Canavanine + azauracil resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10070">argR
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Canavanine + thiouracil resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11332">upp
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Canavanine + thiouracil resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10490">argP
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Canavanine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10071">argS
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Canavanine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10490">argP
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Canavanine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10070">argR
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>CCCP resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10098">atpA–atpE
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>CCCP resistance due to gene dosage</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11439">emrB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Cephalosporin E-0702 resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11012">tonB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Cephalothin resistance</td><td>rfa
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Chelator resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10669">ompA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Chloramphenicol resistance</td><td>cmlA
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Chloramphenicol resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11434">marA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Chloramphenicol resistance</td><td>rrn
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Chloramphenicol resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10671">ompF
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Chloramphenicol resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10672">ompR
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Chloro(β)-D-alanine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11407">dadA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Chloro(β)-D-alanine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10583">metC (ecfA)
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Chloro-3-hydroxyacetone resistance in <em>uhp</em>(Con)</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11054">uhpT
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Chloroacetaldehyde resistance</td><td>adhCE
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Chloroacetate(β) resistance</td><td>cbt
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Chlorobiocin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10424">gyrB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Chloroethanol resistance</td><td>adhCE
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Chlorohydroxyacetone phosphate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10401">glpT
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Chlorpromazine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10542">lon
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Chuangxinmycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11029">trpR
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Cloacin DF13 resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10671">ompF
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Coenzyme A feedback resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10922">coaA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Colicin “ K-type” tolerance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10428">hemB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Colicin (multiple) resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10269">envZ
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Colicin (other) resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10271">exbB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Colicin (other) resistance</td><td>exbC
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Colicin A tolerance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10163">cpxA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Colicin A tolerance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10671">ompF
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Colicin A group tolerance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10155">cirA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Colicin A group tolerance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11010">tolQ
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Colicin B resistance</td><td>exbC
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Colicin B resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10293">fepABCDG
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Colicin B resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11012">tonB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Colicin B tolerance</td><td>cbt
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Colicin D resistance</td><td>cbt
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Colicin D tolerance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11008">tolB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Colicin E resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11009">tolC
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Colicin E1 tolerance</td><td>cet
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Colicin E1 tolerance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11010">tolQAB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Colicin E2 tolerance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10671">ompF
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Colicin E3 tolerance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10126">btuB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Colicin E3 tolerance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11011">tolR
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Colicin E2 and E3 tolerance</td><td>tolE
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Colicin E2 and E3 tolerance</td><td>tolZ
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Colicin E2, E3, D, Ia, and Ib tolerance</td><td>tolI
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Colicin I resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10271">exbB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Colicin I resistance</td><td>exbC
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Colicin I resistance</td><td>ecfB
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Colicin Ia and Ib tolerance</td><td>tolJ
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Colicin K resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10583">metC (ecfA)
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Colicin K resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11035">tsx
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Colicin K resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10669">ompA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Colicin K tolerance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10669">ompA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Colicin K tolerance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10671">ompF
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Colicin L tolerance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10269">envZ
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Colicin L tolerance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10671">ompF
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Colicin L tolerance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10302">fhuA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Colicin L, A, and S4 tolerance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10293">fepABCDG
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Colicin M resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11012">tonB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Colicin M resistance</td><td>tolM
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Colicin M tolerance</td><td>tolD
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Coumermycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10424">gyrB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Coumermycin resistance</td><td>hisW
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Cyclopentane glycine resistance</td><td>ilv structural genes
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Cyclopentane glycine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10493">ilvA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Cycloserine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10443">hipA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Cycloserine (D) resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG12504">cycA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Dapsone resistance</td><td>thdA
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Dehydrobiotin resistance</td><td>bio operon
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Dehydrobiotin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11471">bioP
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Dehydrobiotin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10123">birA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Dehydroproline + azetidine carboxylate resistance at high osmolarity in <em>putA</em><em>putP</em></td><td>proU
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Dehydroproline resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10768">proB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Dehydroproline resistance in <em>putA putP</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11612">proP
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Deoxy(2)adenosine resistance in <em>deoC</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10222">deoD
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Deoxyadenosine + fluorouracil resistance in <em>upp</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10219">deoA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Deoxyadenosine + fluorouracil resistance in <em>upp</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10222">deoD
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Deoxydihydroxyphosphonyl methyl fructose resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11054">uhpT
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Deoxy(3)-3-fluoroglucose-independent utilization of lactate</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10788">ptsHIG
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Deoxy(3)-3-fluoroglucose-independent utilization of fructose</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10787">ptsG
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Deoxy(3)-3-fluoroglucose resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10789">ptsI
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Deoxy(2)galactitol resistance in galicitol utilizer</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG12414">gatACD
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Deoxy(2)galactose resistance</td><td>gal operon
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Deoxy(2)galactose resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10362">galE
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Deoxy(2)galactose resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10363">galK
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Deoxy(2)galactose resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG12148">galP
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Deoxy(2)glucose-resistant utilization of melibiose</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10578">melB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Deoxyglucose resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10567">manXYZ
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Deoxy(2)glucose-6-phosphate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11054">uhpT
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Deoxy(2)glucose-resistant fructose utilization</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10336">fruA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Deoxy(2)glucose-resistant fructose utilization</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10787">ptsG
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Deoxy(2)-2-iodoacetamidoglucose resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10635">nagE
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Diamino(2,6)purine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10051">apt
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Diamino(2,6)purine resistance in <em>pnp</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10051">apt
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Dicyclohexylcarbodiimide resistance in <em>rfa</em></td><td>rfb
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Dideoxy(2′,3′)thymidine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG12302">tmk
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Dihydroproline resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10802">putP
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Dihydroxybutylphosphonate resistance</td><td>cls
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Dihydroxybutylphosphonate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10401">glpT
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Dihydroxybutylphosphonate resistance in <em>glpT</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11046">ugpA,B
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Dihydroxybutylphosphonate resistance in <em>uhp</em>(Con)</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11054">uhpT
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Dimethyl sulfoxide resistance</td><td>pss
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Dinitropyrene resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11672">atoB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Dipeptide (valine containing) resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10248">dppA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Drug-resistant gene maintenance</td><td>dor
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Erythromycin growth dependence</td><td>rrn
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Erythromycin resistance</td><td>eryC
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Erythromycin resistance</td><td>eryD
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Erythromycin resistance</td><td>mac
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Erythromycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10867">rplD
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Erythromycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10882">rplV
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Erythromycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10866">rplC
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Ethanol resistance</td><td>pss
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Ethionine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10586">metG
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Ethionine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10588">metJ
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluoroacetate resistance</td><td>ack
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluoroacetate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG20173">pta
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluorocitrate resistance</td><td>tct
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluoro(5)cytosine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11326">codA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluoro(5)cytosine resistance</td><td>cod
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluorodeoxycytidine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10137">cdd
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluorodeoxycytidine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11971">nupC
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluorodeoxycytidine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10664">nupG
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluoro(5)deoxyuridine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10994">tdk
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluorodeoxyuridine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11971">nupC
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluorodeoxyuridine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10664">nupG
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluorodeoxyuridine + uracil resistance in <em>deoA</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10994">tdk
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluoro(2)-L-erythrocitrate resistance</td><td>tct
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluoro-3-hydroxyacetone resistance in <em>uhp</em>(Con)</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11054">uhpT
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluorohydroxyacetone phosphate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11046">ugpAB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluorohydroxyacetone phosphate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10401">glpT
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluoro(3)malate resistance</td><td>dct
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluoro(5)orotic acid resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10809">pyrF
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluoro(5)orotic acid resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11539">pyrH
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluoro(4)phenylalanine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11042">tyrR
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluoro(4)phenylalanine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10707">pheA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluoro(4)phenylalanine resistance</td><td>pheR
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluoro(4)phenylalanine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10709">pheS
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluoro(4)phenylalanine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG30064">pheU
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluorophenylalanine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10078">aroF
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluorophenylalanine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10079">aroG
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluorophenylalanine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10084">aroP
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluorophenylalanine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11039">tyrA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluoro(5)tryptophan resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10084">aroP
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluoro(6)tryptophan resistance</td><td>trp (generic)
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluorotyrosine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11042">tyrR
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluorotyrosine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11043">tyrS
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluoro(5)uracil + fluoro(5)uridine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10421">guaB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluoro(5)uracil + 5′-AMP resistance in <em>upp</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11060">ushA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluoro(5)uracil + 5-fluorouridine resistance in <em>udp</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11539">pyrH
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluoro(5)uracil + nucleotide resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10671">ompF
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluoro(5)uracil + nucleotide resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10672">ompR
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluoro(5)uracil + adenosine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11045">udp
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluorouracil + adenosine resistance in <em>upp deoD xapR</em>(Con)</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=G85">xapA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluoro(5)uracil + adenosine resistance in <em>upp, phoS,</em> or <em>phoT</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10727">phoA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluoro(5)uracil + adenosine resistance in <em>upp, phoS,</em> or <em>phoT</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10728">phoB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluoro(5)uracil and 3′-AMP resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10160">cpdB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluoro(5)uracil + 5′-AMP + 3′-AMP resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10164">crp
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluoro(5)uracil + carbamylaspartate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=G6365">ubiF
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluoro(5)uracil resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11332">upp
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluorouracil resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11971">nupC
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluorouracil resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10664">nupG
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluorouracil resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11539">pyrH
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluorouracil resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10894">rpoB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluoro(5)uridine + uracil resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11701">udk
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluoro(5)uridine resistance</td><td>udhA
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluoro(5)uridine resistance in <em>upp</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11701">udk
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluorouridine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11539">pyrH
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluorouridine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11971">nupC
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fluorouridine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10664">nupG
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fosfomycin resistance due to increased gene dosage</td><td>murZ
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fosfomycin + fructose-6-phosphate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10702">pgi
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fosfomycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10164">crp
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fosfomycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10170">cyaA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fosfomycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10401">glpT
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fosfomycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10443">hipA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fosfomycin resistance</td><td>mrb
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fosfomycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10789">ptsI
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fosfomycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11054">uhpT
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fucitol resistance in galactitol-utilizing strains</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG12414">gatACD
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fucose + arabinose-supported growth in <em>galP</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10592">mglA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fucose + arabinose-supported growth in <em>galP</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10593">mglB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fucose + arabinose-supported growth in <em>galP</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10594">mglC
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fucose resistance (arabinose as C source)</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10054">araC
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fusaric acid resistance of Tn<em>10</em> insertion mutants</td><td>tet
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Fusidic acid resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10360">fusA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Galactose resistance in <em>galE</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10363">galK
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Galactose resistance in <em>galT</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10363">galK
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Galactose resistance in <em>galU</em></td><td>gal operon
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Galactose resistance in <em>galU</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10363">galK
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Galactose resistance in <em>galT+K</em>+<em>E</em>+/<em>galU</em> <em>galT+K</em>+<em>E</em>+ merodiploid</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10364">galR
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Galactose utilization in presence of thiomethylglucoside</td><td>gal operon
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Gentamicin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10869">rplF
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Gentamicin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=G6365">ubiF
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Globomicin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10238">dnaE
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Globomicin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10544">lpp
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Glucarate (D) resistance in <em>ppc</em></td><td>garA
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Glucose + gluconate-independent motility</td><td>pts (general)
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Glucose-resistant satisfaction of <em>trp</em> by indole + 5-methyltryptophan</td><td>tna
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Glutamate (D) resistance in <em>gltS</em> (increased)</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10406">gltS
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Glutamine (D) resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10388">glnP
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Glutamyl(γ)hydrazide resistance</td><td>glnF
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Glutamyl(γ)hydrazide resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10386">glnH
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Glutamyl(γ)hydrazide resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10388">glnP
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Glutamyl(γ)methyl ester resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10407">gltX
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Glutamyl(γ)methyl ester resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10588">metJ
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Glutamyl(γ)methyl ester resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10589">metK
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Glyceraldehyde (DL) 3-phosphate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10401">glpT
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Glyceraldehyde (L) 3-phosphate resistance in <em>uhp</em>(Con)</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11054">uhpT
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Glyceraldehyde (L) resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10396">glpF
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Glyceraldehyde (L) resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10398">glpK
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Glycerol 3-phosphorothioate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10401">glpT
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Glycine tolerance</td><td>qmeACDE
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Glycylglycyl-<em>N</em>-phosphonoacetylornithine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10070">argR
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Glycylleucine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10493">ilvA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Glycylleucine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10494">ilvB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Glycylleucine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10499">ilvH
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Glycylleucine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10500">ilvI
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Glycylleucine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10674">oppA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Glycylleucine resistance in <em>ilv</em></td><td>gleR
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Glycylglycyl histidinol phosphate ester resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10674">oppA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Glycylglycyl-<em>N</em>-phosphonoacetylornithine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10674">oppA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Glyphosate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10073">aroA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Hexuronate resistance in <em>eda</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11065">uxaBC
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Hexuronate resistance in <em>eda</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11066">uxuAB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Homocysteic acid resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10406">gltS
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Hydrazino (α) imidazole propionic acid resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10072">argT
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Hydrazino (α) imidazole propionic acid resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10449">hisG
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Hydrazino (α) imidazole propionic acid resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG12124">hisJ
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Hydrazino (α) imidazole propionic acid resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG12125">hisQ
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Hydrazino (α) imidazole propionic acid resistance in <em>dhuA</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10452">hisP
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Hydroxy (β) norvaline resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10590">metL
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Hydroxyaspartate resistance</td><td>pan
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Hydroxybutylphosphonate resistance in <em>uhp</em>(Con)</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11054">uhpT
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Hydroxyurea resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10660">nrdA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Hydroxyurea resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10661">nrdB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Indole acrylic acid resistance</td><td>aroT
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Indolmycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11030">trpS
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Indospicine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10070">argR
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Iodoacetylglucosamine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10632">nagA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Iodoacetylglucosamine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10633">nagB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Iodoacetylglucosamine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10635">nagE
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Isoniazid resistance</td><td>pdx
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Kanamycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10098">atpA–atpE
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Kanamycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10163">cpxA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Kanamycin resistance</td><td>ecfB
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Kanamycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10427">hemA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Kanamycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10428">hemB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Kanamycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10432">hemL
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Kanamycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=G0-10473">topA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Kasugamycin resistance</td><td>ksgA
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Kasugamycin resistance</td><td>ksgB
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Kasugamycin resistance</td><td>ksgC
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Kasugamycin resistance</td><td>ksgD
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Kasugamycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10872">rplK
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Kasugamycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10865">rplB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Kasugamycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10913">rpsN
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Kasugamycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10912">rpsM
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Kasugamycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10917">rpsR
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Kasugamycin resistance and dependence</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10908">rpsI
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Keto(2)butyrate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10493">ilvA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Kirromycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11036">tufAB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Lambda <em>c</em>II phage + rifampin coresistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10894">rpoB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Lambda phage + nalidixic acid resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10164">crp
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Lambda phage + nalidixic acid resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10170">cyaA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Levallorphan resistance</td><td>lev
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Lincomycin resistance</td><td>linB
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Lincomycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10875">rplN
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Lincomycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10876">rplO
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Lincomycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10906">rpsG
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Lithium-resistant use of proline as C source</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10802">putP
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Lysine hydroxamate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10550">lysC
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Mecillinam resistance (see beta-lactam resistance)</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10034">alaS
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Mecillinam resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10071">argS
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Mecillinam resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10164">crp
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Mecillinam resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10170">cyaA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Mecillinam resistance</td><td>envB
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Mecillinam resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10606">mrdA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Mecillinam resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10607">mrdB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Mecillinam resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10608">mreB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Mecillinam resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10609">mreC
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Mecillinam resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10610">mreD
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Menadionine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11434">marA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Mercaptopurine resistance in <em>gpt</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG20098">hpt
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Methionine sulfoximine + methyl(α)methionine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10588">metJ
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Methionine sulfoximine resistance</td><td>asm
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Methionine sulfoximine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10388">glnP
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Methionine sulfoximine resistance</td><td>metP
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Methionine sulfoximine resistance in <em>glnG</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10383">glnA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Methylammonium resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10383">glnA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Methyl(3)anthranilate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11028">trpE
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Methyl(3)anthranilate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10079">aroG
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Methyl(α)glutamate resistance in <em>gltS</em> (increased)</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10406">gltS
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Methylglucoside(α)-resistant utilization of lactose</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10525">lacIO
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Methylglucoside(α)-resistant utilization of maltose</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10554">malE
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Methylglucoside(α)-resistant utilization of maltose</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10555">malF
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Methylglucoside(α)-resistant utilization of maltose</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10556">malG
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Methylglucoside(α)-resistant utilization of maltose</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10558">malK
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Methylglucoside(α)-resistant utilization of mannitol + lactose</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10787">ptsG
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Methylglucoside(α) resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10787">ptsG
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Methylglucoside(α)-resistant utilization of glycerol in ∆<em>ptsHI</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10165">crr
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Methylglucoside(α)-resistant utilization of lactose in <em>ptsH</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10525">lacI, operator
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Methylglucoside(α)-resistant utilization of melibiose</td><td>mel
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Methylglyoxal resistance</td><td>gsh
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Methyl(2)histidine resistance + aminotriazole resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG30042">hisR
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Methyl methanesulfonate resistance in <em>lon</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10984">sulA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Methyl methanesulfonate resistance in <em>lon</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10347">ftsZ
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Methyl(α)methionine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10581">metA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Methyl(α)methionine resistance</td><td>metD (metP)
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Methyl(α)methionine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10589">metK
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Methyl(6)purine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10051">apt
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Methyl(6)purine + hypoxanthine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10790">purA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Methyl(<em>o</em>)threonine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG12168">brnQ
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Methyl(4)tryptophan resistance in <em>aroP</em></td><td>aroT
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Methyl(5)tryptophan resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10617">mtr
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Methyl(5)tryptophan resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10084">aroP
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Methyl(5)tryptophan resistance</td><td>trp (generic)
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Methyl(5)tryptophan resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11028">trpE
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Methyl(5)tryptophan resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11029">trpR
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Methyl(5)tryptophan + thienylalanine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10084">aroP
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Methyl viologen resistance</td><td>mvrA
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Methyl viologen resistance (multicopy)</td><td>mvrC
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Methyl viologen resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10412">gor
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Metronidazole resistance</td><td>nar (general)
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Microcin B17 resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10928">sbmA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Microcin E492 resistance</td><td>semA
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Mitomycin C resistance of <em>lexA</em>(Con)</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10533">lexA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Mocimycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11036">tufA,B
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Nalidixic acid resistance</td><td>hisU
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Nalidixic acid resistance</td><td>icdE
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Nalidixic acid tolerance</td><td>sloB
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Nalidixic acid resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10423">gyrA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Nalidixic acid resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10424">gyrB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Nalidixic acid resistance</td><td>nalB
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Nalidixic acid resistance</td><td>nalD
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Nalidixic acid resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11314">purB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Nalidixic acid resistance due to gene dosage</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11439">emrB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Neamine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10911">rpsL
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Neamine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10916">rpsQ
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Neamine resistance</td><td>neaB
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Negamycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10762">prfB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Neomycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10098">atpA–atpE
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Neomycin resistance</td><td>ecfB
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Neomycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10427">hemA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Neomycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10429">hemC
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Neomycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10430">hemD
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Neomycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11543">hemE
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Neomycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11485">hemG
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Neomycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10583">metC (ecfA)
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Neomycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=G0-10473">topA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Neomycin resistance</td><td>ubi (generic)
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Neutrophil granule protein resistance</td><td>pmrA
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Nitro(4)pyridine <em>N</em>-oxide resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10767">proA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Nitro(4)pyridine <em>N</em>-oxide resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10768">proB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Nitro(<em>o</em>)-phenyl galactoside resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10527">lacZ
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Nitro(<em>o</em>)-phenylthiogalactoside resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10526">lacY
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Nitro(<em>o</em>)-phenylthiogalactoside resistance in <em>lacI</em>(Con) or <em>lacO</em>(Con) </td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10525">lacI, operator
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Nitrofurantoin resistance</td><td>nfnA
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Nitrofurantoin resistance</td><td>nfnB
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Nitrofurantoin resistance in <em>lon</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10984">sulA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Nitrofurantoin resistance in <em>lon</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10347">ftsZ
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Nitrofuranzone resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11261">nfsA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Nitrofuranzone resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG50005">nfsB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Nitrosoguanidine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10029">ada
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Nitrosoguanidine resistance</td><td>gsh (generic)
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Norfloxacin resistance</td><td>hipQ
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Norleucine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10588">metJ
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Norleucine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10589">metK
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Norleucine resistance</td><td>nol
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Norleucylglycyl glycine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10674">oppA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Novobiocin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10184">cysB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Novobiocin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10187">cysE
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Novobiocin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10424">gyrB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Novobiocin resistance</td><td>nov
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Novobiocin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10669">ompA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Oxolinic acid resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10423">gyrA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Paromycin resistance or dependence</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10911">rpsL
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Pentachlorophenol resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10098">atpA–atpE
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Penten(4)oate resistance in <em>atoC</em>(Con)</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11668">atoC
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Pentylpantothenamide resistance</td><td>pan
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Peptide (toxic, valine containing) resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10694">pepA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Peptide (toxic, valine containing) resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10695">pepD
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Peptide (toxic, valine containing) resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10696">pepN
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Peptide (toxic, valine containing) resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10698">pepQ
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Peroxide (organic) resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10681">oxyR
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Phaseolotoxin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10674">oppA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Phenethyl alcohol resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10236">dnaB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Phenethyl alcohol resistance</td><td>dnaP
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Phenethyl alcohol resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10936">secA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Phenethylyl galactoside resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10526">lacY
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Phenethylyl galactoside resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10527">lacZ
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Phenyl galactoside resistance of <em>lacP lacI</em>(nonsense) <em>supE</em></td><td>hisT
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Phenyl galactoside resistance of <em>lacP lacI</em>(nonsense) <em>supE</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10761">prfA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Phenyl galactoside resistance of <em>lacP lacI</em>(nonsense) <em>supE</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10911">rpsL
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Phenyl galactoside resistance of <em>lacI </em>q/<em>lacP lacI</em>(nonsense) <em>supL</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10552">lysS
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Phenyl galactoside resistance of <em>lacI </em>q/<em>lacP lacI</em>(nonsense) <em>supL</em></td><td>strM
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Phenyl galactoside resistance of <em>lacI </em>q/<em>lacP lacI</em>(nonsense) <em>supL</em></td><td>trmE
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Phenyl galactoside resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10526">lacY
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Phenyl galactoside resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10527">lacZ
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Phenylalanylleucine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10494">ilvB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Phenylalanylleucine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10499">ilvH
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Phenylalanylleucine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10500">ilvI
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Plasmid maintenance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10746">polA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Polymyxin resistance</td><td>pmrA
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Promethazine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10542">lon
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Pseudomonic acid resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10492">ileS
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Psicofuranine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10420">guaAB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Psoralen + UV irradiation resistance</td><td>puvA
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Pyrithiamine resistance</td><td>thi
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Quinaldic acid resistance of Tn<em>10</em> insertion mutants</td><td>tet
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Quinolone resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10423">gyrA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Ribitol resistance in <em>araC</em>(Con)</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10053">araB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Ribitol resistance in <em>araC</em>(Con)</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10054">araC
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Rifampin + kasugamycin dependence</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=G7877">ridA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Rifampin dependence</td><td>ridB
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Rifampin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10164">crp
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Rifampin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10170">cyaA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Rifampin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10894">rpoB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Rifampin resistance in <em>rpoB</em>+/<em>rpoB</em> (<em>rif</em>) diploids</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10894">rpoB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Salicylate resistance</td><td>pan
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Serine (D) resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG12504">cycA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Serine + methionine + glycine resistance in <em>relA</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10408">glyA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Serine hydroxamate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10944">serA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Serine hydroxamate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10947">serS
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Serine resistance</td><td>sbaA
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Serine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10998">thrA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Serine resistance in <em>relA</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10164">crp
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Serine resistance in <em>relA</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10170">cyaA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Serine resistance in <em>relA</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10835">relA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Serine resistance in <em>relA</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10894">rpoB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Serine resistance in <em>relA</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10895">rpoC
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Showdomycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11971">nupC
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Showdomycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10743">pnp
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Siderophore–beta-lactam conjugate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10155">cirA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Sorbitol + xylitol resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=G8210">srlA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Sorbose (L) resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10336">fruA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Sorbose resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10787">ptsG
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Spectinomycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10904">rpsE
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Spectinomycin resistance</td><td>rrn
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Spectinomycin resistance</td><td>spcB
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Spectinomycin resistance and sucrose dependence</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10902">rpsC
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Spectinomycin resistance and sucrose dependence</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10903">rpsD
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Spectinomycin resistance and sucrose dependence</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10904">rpsE
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Streptolydigin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10894">rpoB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Streptomycin + cAMP resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10164">crp
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Streptomycin resistance</td><td>hem (generic)
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Streptomycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10911">rpsL
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Streptomycin resistance</td><td>strB
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Streptomycin resistance</td><td>strC
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Streptomycin resistance</td><td>strM
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Streptomycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=G6365">ubiF
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Streptomycin resistance</td><td>mut (generic)
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Streptomycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=G0-10473">topA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Streptovaricin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10894">rpoB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Streptozotocin resistance</td><td>fba
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Streptozotocin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10366">galT
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Streptozotocin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10566">manA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Streptozotocin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10632">nagA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Streptozotocin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10635">nagE
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Streptozotocin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10699">pfkA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Streptozotocin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10789">ptsI
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Streptozotocin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10394">glpD
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Sulfanilamide + hypoxanthine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG20098">hpt
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Sulfanilamide + hypoxanthine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10776">pspABCDE
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Sulfanilamide resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG50011">folP
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Sulfanilamide resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10414">gpt
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Sulfometuron methyl + valine resistance</td><td>ilv structural genes
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Sulfometuron methyl + valine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10498">ilvG
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Sulfonamide resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10326">folA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Sulfonamide resistance</td><td>pab
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Tartrate resistance</td><td>dct
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Tetracycline resistance</td><td>cmlA
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Tetracycline resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11434">marA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Tetracycline resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10671">ompF
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Thiaisoleucine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10492">ileS
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Thiaisoleucine resistance</td><td>ilvU
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Thialysine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10998">thrA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Thiazolealanine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10449">hisG
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Thiazolealanine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG30042">hisR
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Thienylalanine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10084">aroP
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Thienylalanine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10079">aroG
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Thiodigalactoside resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10526">lacY
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Thio(5)glucose resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10165">crr
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Thiolactomycin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11439">emrB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Thiolactomycin resistance (due to gene dosage)</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10279">fadB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Thiolactomycin resistance (due to gene dosage)</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11439">emrB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Thiolutin resistance</td><td>tlnA
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Thiomaltose resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10528">lamB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Thiomaltose resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10554">malE
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Thiomaltose resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10555">malF
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Thiomaltose resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10556">malG
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Thiomaltose resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10558">malK
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Thiomaltose resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10562">malT
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Thiomethylgalactoside-independent utilization of galactose in <em>galR</em>(Con)</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10364">galR
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Thiopeptin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10868">rplE
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Thiosine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10490">argP
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Thymidine resistance in <em>deoC</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10219">deoA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Thymidine resistance in <em>deoC</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10220">deoB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Thymineless death prevention by low thymine levels in <em>thyA</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10220">deoB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Thymineless death prevention by low thymine levels in <em>thyA</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10221">deoC
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Thymineless death resistance in <em>thyA</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10828">recF
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Thymineless death resistance in <em>thyA</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10830">recJ
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Thymineless death resistance in <em>thyA</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10832">recO
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Thymineless death resistance in <em>thyA</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10833">recQ
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Thymineless death resistance in <em>thyA</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10034">alaS
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Thymineless death resistance in <em>thyA</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10709">pheS
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Thymineless death resistance in <em>thyA</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11067">valS
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Tiamulin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10866">rplC
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Tiamulin resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10867">rplD
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Triazole resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10184">cysB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Triazole resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10187">cysE
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Triazole resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10188">cysG
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Triazole resistance</td><td>trzA
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Triazole resistance in <em>cysM</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10184">cysB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Triazole resistance in <em>cysM</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10187">cysE
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Triazolealanine + aminotriazole resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10423">gyrA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Triazolealanine + aminotriazole resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10424">gyrB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Triazolealanine + aminotriazole resistance</td><td>his structural genes
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Triazolealanine + aminotriazole resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG30042">hisR
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Triazolealanine + aminotriazole resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10453">hisS
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Triazolealanine + aminotriazole resistance</td><td>hisT
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Triazolealanine + aminotriazole resistance</td><td>hisU
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Triazolealanine + aminotriazole resistance</td><td>hisW
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Trifluorocitrate resistance</td><td>tct
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Trifluoroleucine resistance</td><td>flrB
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Trifluoroleucine resistance</td><td>ileR
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Trifluoroleucine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11226">leuA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Trifluoroleucine resistance</td><td>leuJ
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Trifluoroleucine resistance</td><td>leu (generic)
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Trifluoroleucine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10532">leuS
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Trilysine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10674">oppA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Trimethoprim resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10326">folA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Trimethoprim resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11002">thyA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Triornithine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10674">oppA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Triornithine resistance in <em>opp</em></td><td>tpp
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Tripeptide (toxic amino acid containing) resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10674">oppAE
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Tripeptide (toxic amino acid containing) resistance</td><td>tppA
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Tripeptide (toxic amino acid containing) resistance</td><td>tppB
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Tryptophan analog resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10080">aroH
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Uncoupler resistances</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10098">atpA–atpE
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Uridine resistance of <em>thyA deoB</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10223">deoR
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Valine resistance</td><td>ilv structural genes
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Valine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10494">ilvB
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Valine resistance</td><td>ilvF
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Valine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10498">ilvG
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Valine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10499">ilvH
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Valine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10500">ilvI
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Valine resistance</td><td>ilvJ
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Valine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10537">livG
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Valine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10538">livH
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Valine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10539">livJ
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Valine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10540">livK
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Valine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG12168">brnQ
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Vinylglycolate resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10231">dld
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Vinylglycolate resistance</td><td>lct
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Xylitol resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10336">fruA
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Xylose resistance in <em>fda</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11076">xylE
</a></td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>Xylose (D) + cAMP resistance</td><td>cxm
</td><td>Resistance to biological and organic chemicals</td></tr>
<tr><td>9NA phage resistance</td><td>pmi
</td><td>Resistance to biological agents</td></tr>
<tr><td>9NA phage resistance in <em>galE</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10518">kdsA
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Bf23 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10126">btuB
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>C21 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10838">rfaD
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Chi phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10321">fliC (hag)
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Chi phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10601">motA
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Chi phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10602">motB
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>ES18 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10302">fhuA
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>ES18 phage resistance</td><td>prbA,B
</td><td>Resistance to biological agents</td></tr>
<tr><td>ES18 phage resistance</td><td>sidK
</td><td>Resistance to biological agents</td></tr>
<tr><td>ES18 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11012">tonB
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Felix O phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10362">galE
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Felix O phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11319">galU
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Felix O phage resistance</td><td>rfaC,D,E,F,H
</td><td>Resistance to biological agents</td></tr>
<tr><td>Filamentous phage tolerance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11011">tolR
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Filamentous phage tolerance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11010">tolQ
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>HK009 phage resistance</td><td>prh
</td><td>Resistance to biological agents</td></tr>
<tr><td>HK068 phage resistance</td><td>prk
</td><td>Resistance to biological agents</td></tr>
<tr><td>Host range phage (from <em>Serratia marcescens</em>) resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10670">ompC
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>K10 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10528">lamB
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>K3 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10669">ompA
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Lambda + 434 phage resistance</td><td>mopA
</td><td>Resistance to biological agents</td></tr>
<tr><td>Lambda + 434 phage resistance</td><td>mopB
</td><td>Resistance to biological agents</td></tr>
<tr><td>Lambda <em>c</em>II + rifampin coresistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10894">rpoB
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Lambda mutant infection resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10424">gyrB (hisW)
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Lambda mutant phage infection, survival of</td><td>himA
</td><td>Resistance to biological agents</td></tr>
<tr><td>Lambda mutant phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10435">hflC
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Lambda mutant phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10436">hflK
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Lambda mutant phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10437">hflX
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Lambda mutant prophage induction, survival of</td><td>grpD
</td><td>Resistance to biological agents</td></tr>
<tr><td>Lambda mutant prophage induction, survival of</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10416">grpE
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Lambda phage + nalidixic acid resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10164">crp
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Lambda phage + nalidixic acid resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10170">cyaA
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Lambda phage + 434 phage cross-resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10894">rpoB
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Lambda phage gamma mutant resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10746">polA
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Lambda phage induction, resistance to</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10240">dnaJ
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Lambda phage induction, resistance to</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10241">dnaK
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Lambda phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10269">envZ
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Lambda phage resistance</td><td>gprAB
</td><td>Resistance to biological agents</td></tr>
<tr><td>Lambda phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10528">lamB
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Lambda phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10558">malK
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Lambda phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10562">malT
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Lambda phage resistance</td><td>rap
</td><td>Resistance to biological agents</td></tr>
<tr><td>Lambda phage resistance with maltose and arabinose as C sources</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10164">crp
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Lambda phage resistance with maltose and arabinose as C sources</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10170">cyaA
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Lambda prophage induction deficiency during thymine deprivation</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10823">recA
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Lambda prophage induction, resistance to</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10909">rpsJ
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Lambda prophage induction, resistance to</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10912">rpsM
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Lambda prophage induction, survival of</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10665">nusA
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Lambda prophage induction, survival of</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10666">nusB
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Lambda <em>sus N7 nin-5</em> resistance in P2 lysogen</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10845">rho
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Lambda <em>vir</em> resistance after infection with heteromodified lambda <em>c</em>I857</td><td>hsd
</td><td>Resistance to biological agents</td></tr>
<tr><td>Lambdoid phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10666">nusB
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Lambdoid phage mixture coinfection resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10236">dnaB
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Lambdoid phage mixture coinfection resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10240">dnaJ
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Lambdoid phage mixture coinfection resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10241">dnaK
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Lambda <em>c</em>I71 resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10896">rpoD
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Lambda <em>Nmar</em> mutant phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10894">rpoB
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Male-specific phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10061">arcA (fexA)
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Me1 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10269">envZ
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Me1 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10670">ompC
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Me1 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10671">ompF
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Me1 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10672">ompR
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Mu phage lytic growth resistance</td><td>himA
</td><td>Resistance to biological agents</td></tr>
<tr><td>Mu phage lytic growth resistance</td><td>himD
</td><td>Resistance to biological agents</td></tr>
<tr><td>Mu prophage induction, survival of</td><td>himA
</td><td>Resistance to biological agents</td></tr>
<tr><td>Mu prophage induction, survival of</td><td>himD
</td><td>Resistance to biological agents</td></tr>
<tr><td>N4 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10567">manXYZ
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Ox2 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10669">ompA
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>P1 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10362">galE
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>P1 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11319">galU
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>P1 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=G6106">lpcA
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>P1 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10838">rfaD
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>P1 phage specialized transduction of drug resistance to <em>Salmonella</em> sp.</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10977">sspA
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>P2 <em>vir1</em> phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10893">rpoA
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>P2 phage + lambda phage coinfection resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10240">dnaJ
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>P2 phage + lambda phage coinfection resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10241">dnaK
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>P2 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10837">rep
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>P22 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10362">galE
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>P22 phage resistance</td><td>pmi
</td><td>Resistance to biological agents</td></tr>
<tr><td>P221 phage resistance</td><td>praAB
</td><td>Resistance to biological agents</td></tr>
<tr><td>PH105 phage resistance</td><td>ompB (ompR of E. coli)
</td><td>Resistance to biological agents</td></tr>
<tr><td>PH105 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10670">ompC
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>PH105 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10671">ompF
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>PH105 phage resistance</td><td>praAB
</td><td>Resistance to biological agents</td></tr>
<tr><td>PH51 phage resistance</td><td>ompB (ompR of E. coli)
</td><td>Resistance to biological agents</td></tr>
<tr><td>PH51 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10670">ompC
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>PH51 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10671">ompF
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>PH51 phage resistance</td><td>praAB
</td><td>Resistance to biological agents</td></tr>
<tr><td>PH51 phage resistance</td><td>prdB
</td><td>Resistance to biological agents</td></tr>
<tr><td>φX174 phage resistance</td><td>phxB
</td><td>Resistance to biological agents</td></tr>
<tr><td>φ80 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10302">fhuA
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>φ80 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11012">tonB
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Q phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10163">cpxA (ecfB, ssd, eup)
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>T-even phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10669">ompA
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>T1 phage (UV irradiated) resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10736">phr
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>T1 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10302">fhuA
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>T1 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11012">tonB
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>T2 phage resistance in <em>ompF</em>+ or <em>ompF</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10280">fadL
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>T3 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=G6106">lpcA
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>T4 mutant phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10535">lit
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>T4 phage <em>lig</em> mutant nibbling of colonies yielding <em>lig</em>-overproducing strain</td><td>lig
</td><td>Resistance to biological agents</td></tr>
<tr><td>T4 phage <em>lig</em> mutant resistance from <em>lig</em>-overproducing strain</td><td>lig
</td><td>Resistance to biological agents</td></tr>
<tr><td>T4 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=G6106">lpcA
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>T4 phage resistance</td><td>lpcB
</td><td>Resistance to biological agents</td></tr>
<tr><td>T4 phage resistance</td><td>mopA
</td><td>Resistance to biological agents</td></tr>
<tr><td>T4 phage resistance</td><td>mopB
</td><td>Resistance to biological agents</td></tr>
<tr><td>T4 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10845">rho
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>T4 phage resistance</td><td>tabC
</td><td>Resistance to biological agents</td></tr>
<tr><td>T4 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10535">lit
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>T4 phage (uracil containing) resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11058">ung
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>T5 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10302">fhuA
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>T6 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10164">crp
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>T6 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10170">cyaA
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>T6 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11035">tsx
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>T7 phage gene 1.2 mutant resistance</td><td>optA
</td><td>Resistance to biological agents</td></tr>
<tr><td>T7 phage gene 2 resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10895">rpoC
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>T7 phage resistance</td><td>groM
</td><td>Resistance to biological agents</td></tr>
<tr><td>T7 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=G6106">lpcA
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>T7 phage resistance</td><td>lpcB
</td><td>Resistance to biological agents</td></tr>
<tr><td>T7 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10894">rpoB
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>T7 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11031">trxA
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>TC45 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10728">phoB
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>TC45 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10729">phoE
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>TC45 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10733">phoR
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>TC45 phage resistance</td><td>phoS
</td><td>Resistance to biological agents</td></tr>
<tr><td>TC45 phage resistance</td><td>phoT
</td><td>Resistance to biological agents</td></tr>
<tr><td>TC45 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10782">pstABCS
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>TP1 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10269">envZ
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>TuIa phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10269">envZ
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>TuIa phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10670">ompC
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>TuIa phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10671">ompF
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>TuIa phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10672">ompR
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>TuIa phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11010">tolQAB
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>TuIb phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10671">ompF
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>TuII* phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10669">ompA
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>U3 phage + K3 phage coresistance</td><td>rfaP
</td><td>Resistance to biological agents</td></tr>
<tr><td>U3 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10362">galE
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>U3 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11319">galU
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>U3 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10702">pgi
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>U3 phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG12144">pgm
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>U3 phage resistance</td><td>rfa (general)
</td><td>Resistance to biological agents</td></tr>
<tr><td>UV-irradiated lytic phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11061">uvrA
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>UV-irradiated lytic phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11062">uvrB
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>UV-irradiated lytic phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11063">uvrC
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>UV-irradiated lytic phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11064">uvrD
</a></td><td>Resistance to biological agents</td></tr>
<tr><td>Cold resistance in <em>rpsE</em> cold-sensitive mutants</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10901">rpsB
</a></td><td>Resistance to physical extremes</td></tr>
<tr><td>Cold-resistant growth</td><td>crg
</td><td>Resistance to physical extremes</td></tr>
<tr><td>Filter retention</td><td>fts (generic)
</td><td>Resistance to physical extremes</td></tr>
<tr><td>Freeze-thaw resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10269">envZ
</a></td><td>Resistance to physical extremes</td></tr>
<tr><td>Gamma irradiation resistance</td><td>garA
</td><td>Resistance to physical extremes</td></tr>
<tr><td>Gamma irradiation resistance</td><td>garB
</td><td>Resistance to physical extremes</td></tr>
<tr><td>Growth at 42°C in <em>his</em>-overexpressing strain</td><td>his structural genes
</td><td>Resistance to physical extremes</td></tr>
<tr><td>Near-UV irradiation resistance</td><td>nuvA
</td><td>Resistance to physical extremes</td></tr>
<tr><td>Near-UV irradiation resistance</td><td>nuvC
</td><td>Resistance to physical extremes</td></tr>
<tr><td>Near-UV irradiation resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10835">relA
</a></td><td>Resistance to physical extremes</td></tr>
<tr><td>Osmotolerance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10164">crp
</a></td><td>Resistance to physical extremes</td></tr>
<tr><td>Osmotolerance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10170">cyaA
</a></td><td>Resistance to physical extremes</td></tr>
<tr><td>Osmotolerance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10679">osmB
</a></td><td>Resistance to physical extremes</td></tr>
<tr><td>Osmotolerance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10767">proA
</a></td><td>Resistance to physical extremes</td></tr>
<tr><td>Osmotolerance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10768">proB
</a></td><td>Resistance to physical extremes</td></tr>
<tr><td>Phage (UV-irradiated T1) resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10736">phr
</a></td><td>Resistance to physical extremes</td></tr>
<tr><td>Psoralen + UV irradiation resistance</td><td>puvA
</td><td>Resistance to physical extremes</td></tr>
<tr><td>Temperature resistance in <em>rpoD</em> (Ts)</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10897">rpoH
</a></td><td>Resistance to physical extremes</td></tr>
<tr><td>Thermotolerance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10423">gyrA
</a></td><td>Resistance to physical extremes</td></tr>
<tr><td>UV light resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10164">crp
</a></td><td>Resistance to physical extremes</td></tr>
<tr><td>UV light resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10170">cyaA
</a></td><td>Resistance to physical extremes</td></tr>
<tr><td>UV light resistance</td><td>envB
</td><td>Resistance to physical extremes</td></tr>
<tr><td>UV light resistance </td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10835">relA
</a></td><td>Resistance to physical extremes</td></tr>
<tr><td>UV irradiation resistance in <em>lon</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10984">sulA
</a></td><td>Resistance to physical extremes</td></tr>
<tr><td>UV irradiation resistance in <em>lon</em></td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10347">ftsZ
</a></td><td>Resistance to physical extremes</td></tr>
<tr><td>UV light + bromodeoxyuridine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10994">tdk
</a></td><td>Resistance to physical extremes</td></tr>
<tr><td>UV light + bromodeoxyuridine resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11058">ung
</a></td><td>Resistance to physical extremes</td></tr>
<tr><td>UV light + psoralen resistance</td><td>puvA
</td><td>Resistance to physical extremes</td></tr>
<tr><td>UV-irradiated lytic phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11061">uvrA
</a></td><td>Resistance to physical extremes</td></tr>
<tr><td>UV-irradiated lytic phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11062">uvrB
</a></td><td>Resistance to physical extremes</td></tr>
<tr><td>UV-irradiated lytic phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11063">uvrC
</a></td><td>Resistance to physical extremes</td></tr>
<tr><td>UV-irradiated lytic phage resistance</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11064">uvrD
</a></td><td>Resistance to physical extremes</td></tr>
<tr><td>Migration in a chemical gradient</td><td>che
</td><td>Movement</td></tr>
<tr><td>Static cultivation (prolonged)</td><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10321">fliC
</a></td><td>Movement</td></tr>
</table></div>

<div id="table2"><table>
<caption><span class="anchor" id="table2-genes-for-which-selections-exist"></span><h1>Genes for which selections exist in <i>E. coli</i> and <i>S. typhimurium</i></h1></caption>
<col class="gene">
<col class="organism">
<col class="selection">
<col class="references">
<col class="alteration">
<tr><th>Gene</th><th>Organism</th><th>Selection</th><th>References</th><th>Alteration<a href="#alterations">*</a></th></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10027">ackA</a></td><td>E, S</td><td>Fluoroacetate resistance</td><td><a href="#ref96">96</a> <a href="#ref281">281</a> <a href="#ref463">463</a> <a href="#ref797">797</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10029">ada</a></td><td>E</td><td>Nitrosoguanidine resistance</td><td><a href="#ref693">693</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10030">add</a></td><td>E</td><td>Deoxyadenosine + hypoxanthine satisfaction of <em>purB</em> requirement</td><td><a href="#ref370">370</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10030">add</a></td><td>S</td><td>Deoxyadenosine (low) satisfaction of <em>purA</em> requirement</td><td><a href="#ref326">326</a> </td><td>L</td></tr>
<tr><td>adhCE</td><td>E</td><td>Chloracetaldehyde resistance</td><td><a href="#ref157">157</a> </td><td> </td></tr>
<tr><td>adhCE</td><td>E</td><td>Chlorethanol resistance</td><td><a href="#ref157">157</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10031">adhE</a></td><td>E</td><td>Allyl alcohol resistance</td><td><a href="#ref476">476</a> </td><td>L</td></tr>
<tr><td>adh</td><td>E</td><td>Ethanol as C source</td><td><a href="#ref132">132</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10034">alaS</a></td><td>E</td><td>Thymineless death resistance in <em>thyA</em></td><td><a href="#ref394">394</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10034">alaS</a></td><td>E</td><td>Mecillinam resistance (beta-lactam resistance)</td><td><a href="#ref802">802</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10040">ampC</a></td><td>E</td><td>Ampicillin resistance</td><td><a href="#ref139">139</a> <a href="#ref570">570</a> </td><td>D</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10040">ampC</a></td><td>E</td><td>Ampicillin resistance</td><td><a href="#ref210">210</a> <a href="#ref212">212</a> <a href="#ref362">362</a> </td><td>C</td></tr>
<tr><td>amtAB</td><td>S</td><td>Aminotriazole resistance</td><td><a href="#ref828">828</a> </td><td> </td></tr>
<tr><td>apeA</td><td>E, S</td><td><em>N</em>-Acetyl-L-phenylalanine-β-naphthyl ester hydrolysis</td><td><a href="#ref421">421</a> <a href="#ref516">516</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10051">apt</a></td><td>E</td><td>6-Methylpurine resistance</td><td><a href="#ref60">60</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10051">apt</a></td><td>E</td><td>Adenine resistance in <em>hpt gpt</em></td><td><a href="#ref462">462</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10051">apt</a></td><td>E</td><td>2,6-Diaminopurine resistance in <em>pnp</em></td><td><a href="#ref470">470</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10051">apt</a></td><td>E, S</td><td>2,6-Diaminopurine resistance</td><td><a href="#ref107">107</a> <a href="#ref390">390</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10053">araB</a></td><td>E</td><td>Ribitol resistance from <em>araC</em>(Con)</td><td><a href="#ref396">396</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10054">araC</a></td><td>E</td><td>Fucose resistance (arabinose as C source)</td><td><a href="#ref67">67</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10054">araC</a></td><td>E</td><td>Ribitol resistance from <em>araC</em>(Con)</td><td><a href="#ref396">396</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10061">arcA</a></td><td>E</td><td>Male-specific phage resistance, also called <em>fexA</em></td><td><a href="#ref110">110</a> <a href="#ref651">651</a> <a href="#ref711">711</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10063">argA</a></td><td>E</td><td>Acetylornithine + uracil satisfaction of <em>car</em> (<em>pyrA</em>)</td><td><a href="#ref153">153</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10064">argB</a></td><td>E</td><td><em>N</em>-Acetylhistidine + ornithine satisfaction of <em>his</em></td><td><a href="#ref50">50</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10064">argB</a></td><td>E</td><td>Acetylornithine + uracil satisfaction of <em>car</em> (<em>pyrA</em>)</td><td><a href="#ref153">153</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10065">argC</a></td><td>E</td><td><em>N</em>-Acetylhistidine + ornithine satisfaction of <em>his</em></td><td><a href="#ref50">50</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10065">argC</a></td><td>E</td><td>Acetylornithine + uracil satisfaction of <em>car</em> (<em>pyrA</em>)</td><td><a href="#ref153">153</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10066">argD</a></td><td>E, S</td><td>Proline-independent growth of ∆<em>proAB</em></td><td><a href="#ref62">62</a> <a href="#ref403">403</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10066">argD</a></td><td>E</td><td>Acetylornithine + uracil satisfaction of <em>car</em> (<em>pyrA</em>)</td><td><a href="#ref153">153</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11286">argE</a></td><td>E</td><td><em>N</em>-Acetylhistidine + ornithine satisfaction of <em>his</em></td><td><a href="#ref50">50</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11286">argE</a></td><td>E</td><td>Acetylnorvaline resistance in <em>argR</em>(Con)</td><td><a href="#ref402">402</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11223">argH</a></td><td>E</td><td><em>N</em>-Acetylhistidine + ornithine satisfaction of <em>his</em></td><td><a href="#ref50">50</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10490">argP</a></td><td>E</td><td>Canavanine resistance</td><td><a href="#ref483">483</a> <a href="#ref655">655</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10490">argP</a></td><td>E</td><td>Thiosine resistance</td><td><a href="#ref293">293</a> <a href="#ref716">716</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10490">argP</a></td><td>E</td><td>Canavanine + thiouracil resistance</td><td><a href="#ref605">605</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10070">argR</a></td><td>E, S</td><td>Arginine-resistant, proline-independent growth of ∆<em>pro</em></td><td><a href="#ref62">62</a> <a href="#ref403">403</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10070">argR</a></td><td>E</td><td>Acetylhistidine satisfaction of <em>his</em></td><td><a href="#ref49">49</a> <a href="#ref50">50</a> <a href="#ref403">403</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10070">argR</a></td><td>E</td><td>Acetylmethionine satisfaction of <em>met</em></td><td><a href="#ref402">402</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10070">argR</a></td><td>E</td><td>Indospicine resistance</td><td><a href="#ref454">454</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10070">argR</a></td><td>E</td><td>Canavanine resistance</td><td><a href="#ref482">482</a> <a href="#ref523">523</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10070">argR</a></td><td>E</td><td>Glycylglycyl-<em>N</em>-phosphonoacetylornithine resistance</td><td><a href="#ref604">604</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10070">argR</a></td><td>E</td><td>Canavanine + azauracil resistance</td><td><a href="#ref605">605</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10070">argR</a></td><td>E</td><td>Canavanine + thiouracil resistance</td><td><a href="#ref605">605</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10070">argR</a></td><td>E</td><td>Acetylnorvaline resistance in <em>argR</em>(Con)</td><td><a href="#ref402">402</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10071">argS</a></td><td>E</td><td>Canavanine resistance</td><td><a href="#ref322">322</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10071">argS</a></td><td>E</td><td>Mecillinam resistance (beta-lactam resistance)</td><td><a href="#ref802">802</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10072">argT</a></td><td>S</td><td>Hydrazino (α) imidazole propionic acid resistance</td><td><a href="#ref432">432</a> <a href="#ref433">433</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10073">aroA</a></td><td>E, S</td><td>Glyphosate resistance</td><td><a href="#ref512">512</a> <a href="#ref731">731</a> <a href="#ref869">869</a> </td><td>C, D</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10078">aroF</a></td><td>E</td><td>Amino(4)phenylalanine resistance</td><td><a href="#ref504">504</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10078">aroF</a></td><td>S</td><td>Fluoro(4)phenylalanine resistance</td><td><a href="#ref270">270</a> <a href="#ref729">729</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10079">aroG</a></td><td>E, S</td><td>Methyl(3)anthranilate resistance</td><td><a href="#ref307">307</a> </td><td>FBI, S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10079">aroG</a></td><td>E</td><td>Thienylalanine resistance</td><td><a href="#ref215">215</a> </td><td>FBI, S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10079">aroG</a></td><td>E</td><td>Fluorophenylalanine resistance</td><td><a href="#ref351">351</a> </td><td>FBI, S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10080">aroH</a></td><td>E</td><td>Tryptophan analog resistance</td><td><a href="#ref637">637</a> </td><td>FBI, S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10084">aroP</a></td><td>E, S</td><td>Methyl(5)tryptophan resistance</td><td><a href="#ref12">12</a> <a href="#ref94">94</a> <a href="#ref288">288</a> <a href="#ref303">303</a> <a href="#ref393">393</a> <a href="#ref437">437</a> <a href="#ref438">438</a> <a href="#ref439">439</a> </td><td>L, C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10084">aroP</a></td><td>E, S</td><td>Fluoro(5)tryptophan resistance</td><td><a href="#ref12">12</a> <a href="#ref94">94</a> <a href="#ref288">288</a> <a href="#ref303">303</a> <a href="#ref393">393</a> <a href="#ref437">437</a> <a href="#ref438">438</a> <a href="#ref439">439</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10084">aroP</a></td><td>E, S</td><td>Azaserine resistance</td><td><a href="#ref12">12</a> <a href="#ref94">94</a> <a href="#ref288">288</a> <a href="#ref303">303</a> <a href="#ref393">393</a> <a href="#ref437">437</a> <a href="#ref438">438</a> <a href="#ref439">439</a> <a href="#ref836">836</a> <a href="#ref837">837</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10084">aroP</a></td><td>E, S</td><td>Thienylalanine resistance</td><td><a href="#ref12">12</a> <a href="#ref94">94</a> <a href="#ref288">288</a> <a href="#ref303">303</a> <a href="#ref393">393</a> <a href="#ref437">437</a> <a href="#ref438">438</a> <a href="#ref439">439</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10084">aroP</a></td><td>E, S</td><td>Fluorophenylalanine resistance</td><td><a href="#ref12">12</a> <a href="#ref94">94</a> <a href="#ref288">288</a> <a href="#ref303">303</a> <a href="#ref393">393</a> <a href="#ref437">437</a> <a href="#ref438">438</a> <a href="#ref439">439</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10084">aroP</a></td><td>E, S</td><td>Azaleucine resistance</td><td><a href="#ref12">12</a> <a href="#ref94">94</a> <a href="#ref288">288</a> <a href="#ref303">303</a> <a href="#ref393">393</a> <a href="#ref437">437</a> <a href="#ref438">438</a> <a href="#ref439">439</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10084">aroP</a></td><td>E, S</td><td>Methyl(5)tryptophan + thienylalanaine resistance</td><td><a href="#ref12">12</a> <a href="#ref94">94</a> <a href="#ref288">288</a> <a href="#ref303">303</a> <a href="#ref393">393</a> <a href="#ref437">437</a> <a href="#ref438">438</a> <a href="#ref439">439</a> </td><td>L</td></tr>
<tr><td>aroT</td><td>E, S</td><td>Indole acrylic acid resistance</td><td><a href="#ref769">769</a> </td><td>L</td></tr>
<tr><td>aroT</td><td>E</td><td>Methyl(4)tryptophan resistance in <em>aroP</em></td><td><a href="#ref594">594</a> </td><td>L</td></tr>
<tr><td>asm</td><td>S</td><td>Methionine sulfoxamine resistance</td><td><a href="#ref188">188</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11672">atoB</a></td><td>S</td><td>Dinitropyrene resistance</td><td><a href="#ref590">590</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11668">atoC</a></td><td>E</td><td>Butyrate as C source in <em>fadR</em>(Con)</td><td><a href="#ref602">602</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11668">atoC</a></td><td>E</td><td>Valerate as C source in <em>fadR</em>(Con)</td><td><a href="#ref602">602</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11668">atoC</a></td><td>E</td><td>Penten(4)oate resistance in <em>atoC</em>(Con)</td><td><a href="#ref669">669</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10098">atpA-atpE</a></td><td>E</td><td>Uncoupler resistances, including DCCD</td><td><a href="#ref225">225</a> <a href="#ref226">226</a> <a href="#ref229">229</a> <a href="#ref355">355</a> <a href="#ref537">537</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10098">atpA-atpE</a></td><td>E</td><td>Tributyl tin resistance</td><td><a href="#ref355">355</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10098">atpA-atpE</a></td><td>E</td><td>CCCP resistance</td><td><a href="#ref355">355</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10098">atpA-atpE</a></td><td>E</td><td>Pentachlorophenol resistance</td><td><a href="#ref355">355</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10098">atpA-atpE</a></td><td>E</td><td>Auroventin resistance</td><td><a href="#ref451">451</a> <a href="#ref685">685</a> <a href="#ref818">818</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10098">atpA-atpE</a></td><td>E</td><td>Kanamycin resistance</td><td><a href="#ref768">768</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10098">atpA-atpE</a></td><td>E</td><td>Neomycin resistance</td><td><a href="#ref392">392</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10098">atpA-atpE</a></td><td>E</td><td>Azide resistance</td><td><a href="#ref355">355</a> </td><td> </td></tr>
<tr><td>atr</td><td>S</td><td>Acid resistance</td><td><a href="#ref233">233</a> </td><td> </td></tr>
<tr><td>azaAB</td><td>E</td><td>Azaserine resistance</td><td><a href="#ref836">836</a> </td><td> </td></tr>
<tr><td>azl</td><td>E</td><td>Azaleucine resistance</td><td><a href="#ref610">610</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10114">bglBC</a></td><td>E</td><td>Arbutin as C source</td><td><a href="#ref523">523</a> <a href="#ref616">616</a> <a href="#ref640">640</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10114">bglBC</a></td><td>E</td><td>Salicin as C source</td><td><a href="#ref523">523</a> <a href="#ref616">616</a> <a href="#ref640">640</a> </td><td> </td></tr>
<tr><td>bglT</td><td>E</td><td>Methyl(β)glucoside as C source</td><td><a href="#ref686">686</a> </td><td>C</td></tr>
<tr><td>bio operon</td><td>E</td><td>Dehydrobiotin resistance</td><td><a href="#ref206">206</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11471">bioP</a></td><td>E</td><td>Dehydrobiotin resistance</td><td><a href="#ref206">206</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10123">birA</a></td><td>E</td><td>Dehydrobiotin resistance</td><td><a href="#ref206">206</a> <a href="#ref597">597</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG12168">brnQ</a></td><td>E</td><td>Methyl(<em>o</em>)threonine resistance</td><td><a href="#ref278">278</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG12168">brnQ</a></td><td>E</td><td>Valine resistance</td><td><a href="#ref277">277</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10126">btuB</a></td><td>E</td><td>Colicin E3 tolerance</td><td><a href="#ref46">46</a> <a href="#ref47">47</a> <a href="#ref109">109</a> <a href="#ref284">284</a> <a href="#ref388">388</a> <a href="#ref620">620</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10126">btuB</a></td><td>E, S</td><td>Bf23 phage resistance</td><td><a href="#ref46">46</a> <a href="#ref47">47</a> <a href="#ref109">109</a> <a href="#ref284">284</a> <a href="#ref388">388</a> <a href="#ref528">528</a> <a href="#ref620">620</a> </td><td> </td></tr>
<tr><td>bymA</td><td>E</td><td>Maltose utilization in <em>malT</em></td><td><a href="#ref327">327</a> </td><td> </td></tr>
<tr><td>cbt</td><td>E</td><td>Chloroacetate(β) resistance</td><td><a href="#ref68">68</a> </td><td>L</td></tr>
<tr><td>cbt</td><td>E</td><td>Colicin B tolerance</td><td><a href="#ref623">623</a> </td><td>L</td></tr>
<tr><td>cbt</td><td>E</td><td>Colicin D tolerance</td><td><a href="#ref623">623</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10137">cdd</a></td><td>E</td><td>Aza(5)-2′-deoxycytidine resistance</td><td><a href="#ref196">196</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10137">cdd</a></td><td>E, S</td><td>Fluoro(5)deoxycytidine resistance</td><td><a href="#ref53">53</a> <a href="#ref579">579</a> </td><td>L</td></tr>
<tr><td>celABCDF</td><td>E</td><td>Cellobiose as C source</td><td><a href="#ref426">426</a> </td><td> </td></tr>
<tr><td>cet</td><td>E</td><td>Colicin E2 tolerance</td><td><a href="#ref201">201</a> </td><td> </td></tr>
<tr><td>che</td><td>S</td><td>Migration in a chemotaxis gradient</td><td><a href="#ref34">34</a> </td><td> </td></tr>
<tr><td>chlABDEG</td><td>E, S</td><td>Chlorate resistance</td><td><a href="#ref2">2</a> <a href="#ref121">121</a> <a href="#ref122">122</a> <a href="#ref228">228</a> <a href="#ref266">266</a> <a href="#ref523">523</a> <a href="#ref743">743</a> <a href="#ref744">744</a> </td><td>L</td></tr>
<tr><td>chlABDE</td><td>E</td><td>Lactate + fumarate as anaerobic energy source</td><td><a href="#ref436">436</a> </td><td>L</td></tr>
<tr><td>chlC</td><td>S</td><td>Chlorate resistance</td><td><a href="#ref45">45</a> </td><td>L</td></tr>
<tr><td>chlE</td><td>E</td><td>Lactate + nitrate as anaerobic energy source</td><td><a href="#ref799">799</a> </td><td> </td></tr>
<tr><td>chlE</td><td>E</td><td>Biotin sulfoxide utilization in <em>chl bio</em></td><td><a href="#ref180">180</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10155">cirA</a></td><td>E</td><td>Colicin I resistance</td><td><a href="#ref77">77</a> <a href="#ref93">93</a> <a href="#ref620">620</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10155">cirA</a></td><td>E</td><td>Siderophore-beta-lactam conjugate resistance</td><td><a href="#ref93">93</a> </td><td>L</td></tr>
<tr><td>cit</td><td>E</td><td>Citrate as C source</td><td><a href="#ref290">290</a> </td><td> </td></tr>
<tr><td>cls</td><td>E</td><td>Dihydroxy(3,4)butyl-1-phosphonate resistance</td><td><a href="#ref345">345</a> </td><td>L</td></tr>
<tr><td>cmlA</td><td>E</td><td>Chloramphenicol resistance</td><td><a href="#ref48">48</a> </td><td> </td></tr>
<tr><td>cmlA</td><td>E</td><td>Tetracycline resistance</td><td><a href="#ref523">523</a> <a href="#ref638">638</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10922">coaA</a></td><td>E</td><td>Coenzyme A feedback resistance</td><td><a href="#ref789">789</a> <a href="#ref790">790</a> </td><td>FBI, S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11326">codA(cod)</a></td><td>E, S</td><td>Fluoro(5)cytosine resistance</td><td><a href="#ref4">4</a> <a href="#ref54">54</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11463">corABCD</a></td><td>E, S</td><td>Cobalt resistance</td><td><a href="#ref262">262</a> <a href="#ref600">600</a> <a href="#ref817">817</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11463">corABCD</a></td><td>E</td><td>Manganese resistance</td><td><a href="#ref600">600</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11463">corABCD</a></td><td>E</td><td>Nickel resistance</td><td><a href="#ref817">817</a> </td><td> </td></tr>
<tr><td>cpd</td><td>S</td><td>Succinate as improved C source</td><td><a href="#ref8">8</a> <a href="#ref530">530</a> </td><td>L</td></tr>
<tr><td>cpd</td><td>S</td><td>Citrate as C source in presence of low cAMP</td><td><a href="#ref8">8</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10160">cpdB</a></td><td>E</td><td>Fluorouracil(5) and 3′-AMP resistance</td><td><a href="#ref51">51</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10163">cpxA</a></td><td>E</td><td>Amikacin resistance</td><td><a href="#ref629">629</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10163">cpxA</a></td><td>E</td><td>Colicin A tolerance</td><td><a href="#ref629">629</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10163">cpxA</a></td><td>E</td><td>Kanamycin resistance</td><td><a href="#ref768">768</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10163">cpxA</a></td><td>E</td><td>Q phage resistance</td><td><a href="#ref508">508</a> <a href="#ref711">711</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10163">cpxA</a></td><td>E</td><td>Serine (L) as C source</td><td><a href="#ref567">567</a> </td><td> </td></tr>
<tr><td>crg</td><td>E</td><td>Cold-resistant growth</td><td><a href="#ref398">398</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10164">crp</a></td><td>S</td><td>Fosfomycin resistance</td><td><a href="#ref10">10</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10164">crp</a></td><td>E</td><td>Serine (L) resistance in <em>relA</em></td><td><a href="#ref167">167</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10164">crp</a></td><td>E</td><td>Lambda phage + nalidixic acid resistance</td><td><a href="#ref429">429</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10164">crp</a></td><td>E</td><td>Streptomycin + cAMP resistance</td><td><a href="#ref31">31</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10164">crp</a></td><td>E</td><td>Lambda phage resistance with maltose and arabinose as C sources</td><td><a href="#ref92">92</a> <a href="#ref766">766</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10164">crp</a></td><td>E</td><td>cAMP + glucose 6-phosphate resistance</td><td><a href="#ref1a">1a</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10164">crp</a></td><td>E</td><td>cAMP + glucose 6-phosphate + D-xylose + L-arabinose resistance</td><td><a href="#ref1a">1a</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10164">crp</a></td><td>E</td><td>Fluorouracil(5) + 5′-AMP + 3′-AMP resistance</td><td><a href="#ref52">52</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10164">crp</a></td><td>E</td><td>Osmotolerance</td><td><a href="#ref282">282</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10164">crp</a></td><td>E</td><td>Beta-lactam resistance</td><td><a href="#ref362">362</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10164">crp</a></td><td>E</td><td>Mecillinam resistance</td><td><a href="#ref25">25</a> <a href="#ref168">168</a> <a href="#ref857">857</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10164">crp</a></td><td>E</td><td>UV light resistance</td><td><a href="#ref627">627</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10164">crp</a></td><td>E</td><td>Rifampin resistance</td><td><a href="#ref413">413</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10164">crp</a></td><td>E</td><td>T6 phage resistance</td><td><a href="#ref6">6</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10165">crr</a></td><td>E</td><td>Thio(5)glucose resistance</td><td><a href="#ref419">419</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10165">crr</a></td><td>S</td><td>Methylglucoside(α)-resistant utilization of glycerol in <em>ptsHI</em></td><td><a href="#ref563">563</a> </td><td>L</td></tr>
<tr><td>cxm</td><td>E</td><td>Xylose (D) + cAMP resistance</td><td><a href="#ref1a">1a</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10170">cyaA</a></td><td>E</td><td>Lambda phage resistance with maltose and arabinose as C sources</td><td><a href="#ref92">92</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10170">cyaA</a></td><td>E</td><td>Lambda phage + nalidixic acid resistance</td><td><a href="#ref429">429</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10170">cyaA</a></td><td>E</td><td>Serine (L) resistance in <em>relA</em></td><td><a href="#ref167">167</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10170">cyaA</a></td><td>S</td><td>Fosfomycin resistance</td><td><a href="#ref10">10</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10170">cyaA</a></td><td>E</td><td>Osmotolerance</td><td><a href="#ref282">282</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10170">cyaA</a></td><td>E</td><td>Beta-lactam resistance</td><td><a href="#ref362">362</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10170">cyaA</a></td><td>E</td><td>Mecillinam resistance</td><td><a href="#ref25">25</a> <a href="#ref168">168</a> <a href="#ref857">857</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10170">cyaA</a></td><td>E</td><td>UV light resistance</td><td><a href="#ref627">627</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10170">cyaA</a></td><td>E</td><td>Rifampin resistance</td><td><a href="#ref413">413</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10170">cyaA</a></td><td>E</td><td>T6 phage resistance</td><td><a href="#ref6">6</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG12504">cycA</a></td><td>E</td><td>Cycloserine (D) resistance</td><td><a href="#ref150">150</a> <a href="#ref646">646</a> <a href="#ref813">813</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG12504">cycA</a></td><td>E</td><td>Serine (D) resistance</td><td><a href="#ref689">689</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10183">cysA</a></td><td>S</td><td>Chromate resistance</td><td><a href="#ref581">581</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10183">cysA</a></td><td>E</td><td>Selenate resistance</td><td><a href="#ref728">728</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10183">cysA</a></td><td>S</td><td>Azaserine resistance</td><td><a href="#ref340">340</a> <a href="#ref689">689</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10184">cysB</a></td><td>S</td><td>Chromate resistance</td><td><a href="#ref340">340</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10184">cysB</a></td><td>S</td><td>Azide resistance</td><td><a href="#ref227">227</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10184">cysB</a></td><td>S</td><td>Selenate resistance</td><td><a href="#ref339">339</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10184">cysB</a></td><td>E</td><td>Novobiocin resistance</td><td><a href="#ref631">631</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10184">cysB</a></td><td>S</td><td>Triazole resistance</td><td><a href="#ref227">227</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10184">cysB</a></td><td>S</td><td>Azaserine resistance</td><td><a href="#ref340">340</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10185">cysC</a></td><td>S</td><td>Azaserine resistance</td><td><a href="#ref340">340</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10185">cysC</a></td><td>S</td><td>Chromate resistance</td><td><a href="#ref581">581</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10186">cysD</a></td><td>S</td><td>Azaserine resistance</td><td><a href="#ref340">340</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10186">cysD</a></td><td>S</td><td>Chromate resistance</td><td><a href="#ref581">581</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10187">cysE</a></td><td>S</td><td>Triazole resistance in <em>cysM</em></td><td><a href="#ref717">717</a> <a href="#ref825">825</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10187">cysE</a></td><td>S</td><td>Triazole resistance</td><td><a href="#ref341">341</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10187">cysE</a></td><td>E</td><td>Novobiocin resistance</td><td><a href="#ref631">631</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10188">cysG</a></td><td>S</td><td>Azaserine resistance</td><td><a href="#ref340">340</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10189">cysH</a></td><td>S</td><td>Azaserine resistance</td><td><a href="#ref340">340</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10189">cysH</a></td><td>S</td><td>Chromate resistance</td><td><a href="#ref581">581</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10190">cysI</a></td><td>S</td><td>Azaserine resistance</td><td><a href="#ref340">340</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10190">cysI</a></td><td>S</td><td>Chromate resistance</td><td><a href="#ref581">581</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10191">cysJ</a></td><td>S</td><td>Azaserine resistance</td><td><a href="#ref340">340</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10191">cysJ</a></td><td>S</td><td>Chromate resistance</td><td><a href="#ref581">581</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10192">cysK</a></td><td>S, E</td><td>Azaserine resistance</td><td><a href="#ref340">340</a> <a href="#ref827">827</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10192">cysK</a></td><td>S</td><td>Azide resistance</td><td><a href="#ref147">147</a> <a href="#ref227">227</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10192">cysK</a></td><td>E</td><td>Selenate resistance</td><td><a href="#ref230">230</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10192">cysK</a></td><td>E, S</td><td>Triazole resistance</td><td><a href="#ref147">147</a> <a href="#ref227">227</a> <a href="#ref826">826</a> <a href="#ref827">827</a> </td><td>L, C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10192">cysK</a></td><td>E</td><td>Selenite resistance</td><td><a href="#ref230">230</a> </td><td>L</td></tr>
<tr><td>cysL</td><td>S</td><td>Selenate resistance</td><td><a href="#ref673">673</a> <a href="#ref674">674</a> </td><td> </td></tr>
<tr><td>cysL</td><td>S</td><td>Chromate + selenate resistance</td><td><a href="#ref338">338</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10193">cysM</a></td><td>S</td><td>Azaserine resistance</td><td><a href="#ref340">340</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10200">cytR</a></td><td>E</td><td>Uridine as C source, improved utilization</td><td><a href="#ref546">546</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11407">dadA</a></td><td>E, S</td><td>Chloro(β)-D-alanine resistance</td><td><a href="#ref832">832</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11407">dadA</a></td><td>E, S</td><td>D-Amino acid satisfaction of amino acid auxotrophy</td><td><a href="#ref428">428</a> <a href="#ref831">831</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10211">dcm</a></td><td>E</td><td>Aza(5)cytidine resistance</td><td><a href="#ref247">247</a> </td><td>L</td></tr>
<tr><td>dct</td><td>E</td><td>Fluoro(3)malate resistance</td><td><a href="#ref401">401</a> </td><td>L</td></tr>
<tr><td>dct</td><td>E</td><td>Tartrate resistance</td><td><a href="#ref687">687</a> </td><td>L</td></tr>
<tr><td>dct</td><td>S</td><td>Malate (L) as improved C source</td><td><a href="#ref745">745</a> </td><td>D</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10219">deoA</a></td><td>E</td><td>Deoxyadenosine + fluorouracil resistance in <em>upp</em></td><td><a href="#ref3">3</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10219">deoA</a></td><td>E</td><td>Thymidine resistance in <em>deoC</em></td><td><a href="#ref3">3</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10220">deoB</a></td><td>E</td><td>Thymidine resistance in <em>deoC</em></td><td><a href="#ref3">3</a> <a href="#ref651">651</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10220">deoB</a></td><td>E, S</td><td>Thymineless death prevention by low thymine levels in <em>thyA</em></td><td><a href="#ref71">71</a> <a href="#ref471">471</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10221">deoC</a></td><td>E</td><td>Thymineless death prevention by low thymine levels in <em>thyA</em></td><td><a href="#ref325">325</a> <a href="#ref471">471</a> <a href="#ref651">651</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10222">deoD</a></td><td>E</td><td>Deoxy(2)adenosine resistance in <em>deoC</em></td><td><a href="#ref651">651</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10222">deoD</a></td><td>E</td><td>Deoxyadenosine + fluorouracil resistance in <em>upp</em></td><td><a href="#ref3">3</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10223">deoR</a></td><td>E</td><td>Inosine as improved C source</td><td><a href="#ref546">546</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10223">deoR</a></td><td>S</td><td>Diamino(2,6)purine satisfaction of purine auxotrophy</td><td><a href="#ref251">251</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10223">deoR</a></td><td>S</td><td>Uridine resistance in <em>thyA deoB</em></td><td><a href="#ref71">71</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=G26">dgd</a></td><td>E</td><td>Arabitol (D) as C source in <em>fuc</em></td><td><a href="#ref849">849</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=G7789">dgoR</a></td><td>E</td><td>Keto(2)-3-deoxygalactonate as C source</td><td><a href="#ref144">144</a> </td><td>C</td></tr>
<tr><td>dgsA</td><td>E</td><td>Glucosamine as anaerobic C source in <em>ptsG</em></td><td><a href="#ref652">652</a> </td><td>L</td></tr>
<tr><td>dhuA</td><td>E, S</td><td>Histidine (D) satisfaction of <em>his</em> auxotrophs</td><td><a href="#ref422">422</a> <a href="#ref428">428</a> <a href="#ref546">546</a> </td><td> </td></tr>
<tr><td>dhuA</td><td>S</td><td>Azaserine + tryptophan resistance</td><td><a href="#ref432">432</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10231">dld</a></td><td>E</td><td>Vinylglycolate resistance</td><td><a href="#ref696">696</a> </td><td>L</td></tr>
<tr><td>dml</td><td>S</td><td>Malate (D) as C source</td><td><a href="#ref737">737</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10236">dnaB</a></td><td>E</td><td>Lambdoid phage mixture coinfection resistance</td><td><a href="#ref259">259</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10236">dnaB</a></td><td>E</td><td>Phenethyl alcohol resistance</td><td><a href="#ref494">494</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10238">dnaE</a></td><td>E</td><td>Acriflavin (acridine) resistance</td><td><a href="#ref553">553</a> <a href="#ref554">554</a> <a href="#ref555">555</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10238">dnaE</a></td><td>E</td><td>Globomycin resistance</td><td><a href="#ref668">668</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10240">dnaJ</a></td><td>E</td><td>Lambda phage induction, resistance to</td><td><a href="#ref667">667</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10240">dnaJ</a></td><td>E</td><td>Lambdoid phage mixture coinfection resistance</td><td><a href="#ref257">257</a> <a href="#ref259">259</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10240">dnaJ</a></td><td>E</td><td>P2 phage + lambda phage coinfection resistance</td><td><a href="#ref753">753</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10241">dnaK</a></td><td>E</td><td>Lambda phage induction, resistance to</td><td><a href="#ref667">667</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10241">dnaK</a></td><td>E</td><td>Lambdoid phage mixture coinfection resistance</td><td><a href="#ref257">257</a> <a href="#ref259">259</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10241">dnaK</a></td><td>E</td><td>P2 phage + lambda phage coninfection resistance</td><td><a href="#ref753">753</a> </td><td>S</td></tr>
<tr><td>dnaP</td><td>E</td><td>Phenethyl alcohol resistance</td><td><a href="#ref809">809</a> </td><td> </td></tr>
<tr><td>dor</td><td>S</td><td>Drug-resistant gene maintenance</td><td><a href="#ref814">814</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10248">dppA</a></td><td>E, S</td><td>Bacilysin resistance</td><td><a href="#ref1">1</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10248">dppA</a></td><td>E, S</td><td>Bialaphos resistance</td><td><a href="#ref1">1</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10248">dppA</a></td><td>E, S</td><td>Valine-containing dipeptide resistance</td><td><a href="#ref1">1</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10248">dppA</a></td><td>E</td><td>Lysyl-2-aminoxypropionate resistance</td><td><a href="#ref603">603</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10248">dppA</a></td><td>E</td><td>Glycylleucine resistance</td><td><a href="#ref269">269</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10248">dppA</a></td><td>E</td><td>Phenylalanylleucine resistance</td><td><a href="#ref269">269</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10248">dppA</a></td><td>E</td><td>Glycylvaline resistance in <em>opp</em></td><td><a href="#ref178">178</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10249">dsdA</a></td><td>E</td><td>Sucrose as C source</td><td><a href="#ref5">5</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG13161">dsdC</a></td><td>E</td><td>Sucrose as C source</td><td><a href="#ref5">5</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10252">ebgA</a></td><td>E</td><td>Lactose utilization in ∆<em>lacZ</em></td><td><a href="#ref118">118</a> </td><td>Q</td></tr>
<tr><td>ecfB</td><td>E</td><td>Serine (L) as C source</td><td><a href="#ref567">567</a> <a href="#ref768">768</a> </td><td> </td></tr>
<tr><td>ecfB</td><td>E</td><td>Kanamycin resistance</td><td><a href="#ref567">567</a> <a href="#ref768">768</a> </td><td> </td></tr>
<tr><td>ecfB</td><td>E</td><td>Neomycin resistance</td><td><a href="#ref608">608</a> </td><td> </td></tr>
<tr><td>ecfB</td><td>E</td><td>Colicin K resistance</td><td><a href="#ref608">608</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10257">edd</a></td><td>E</td><td>Gluconate as C source in <em>eda</em></td><td><a href="#ref239">239</a> <a href="#ref240">240</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11439">emrB</a></td><td>E</td><td>CCCP resistance due to gene dosage</td><td><a href="#ref472">472</a> </td><td>D</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11439">emrB</a></td><td>E</td><td>Nalidixic acid resistance due to gene dosage</td><td><a href="#ref472">472</a> </td><td>D</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11439">emrB</a></td><td>E</td><td>Thiolactomycin resistance due to gene dosage</td><td><a href="#ref250">250</a> </td><td>D</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11439">emrB</a></td><td>E</td><td>Thiolactomycin resistance</td><td><a href="#ref250">250</a> </td><td> </td></tr>
<tr><td>envB</td><td>E, S</td><td>Mecillinam resistance</td><td><a href="#ref359">359</a> <a href="#ref587">587</a> </td><td> </td></tr>
<tr><td>envB</td><td>E</td><td>UV light resistance</td><td><a href="#ref24">24</a> </td><td> </td></tr>
<tr><td>envM</td><td>E, S</td><td>Diazoborine resistance</td><td><a href="#ref63">63</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10269">envZ</a></td><td>E</td><td>Ampicillin resistance</td><td><a href="#ref362">362</a> <a href="#ref363">363</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10269">envZ</a></td><td>E</td><td>Colicin L tolerance</td><td><a href="#ref812">812</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10269">envZ</a></td><td>E</td><td>Lambda phage resistance</td><td><a href="#ref812">812</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10269">envZ</a></td><td>E</td><td>Me1 phage resistance</td><td><a href="#ref801">801</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10269">envZ</a></td><td>E</td><td>TPI phage resistance</td><td><a href="#ref812">812</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10269">envZ</a></td><td>E</td><td>TuIa phage resistance</td><td><a href="#ref801">801</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10269">envZ</a></td><td>E</td><td>Freeze-thaw resistance</td><td><a href="#ref112">112</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10269">envZ</a></td><td>E</td><td>Colicin (multiple) resistance</td><td><a href="#ref626">626</a> </td><td> </td></tr>
<tr><td>eryC</td><td>E</td><td>Erythromycin resistance</td><td><a href="#ref599">599</a> </td><td> </td></tr>
<tr><td>eryD</td><td>E</td><td>Erythromycin resistance</td><td><a href="#ref830">830</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10271">exbB</a></td><td>E</td><td>Albomycin resistance</td><td><a href="#ref205">205</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10271">exbB</a></td><td>E</td><td>Colicin I resistance</td><td><a href="#ref283">283</a> <a href="#ref624">624</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10271">exbB</a></td><td>E</td><td>Colicin (other) resistance</td><td><a href="#ref283">283</a> <a href="#ref624">624</a> </td><td> </td></tr>
<tr><td>exbC</td><td>E</td><td>Colicin B resistance</td><td><a href="#ref624">624</a> </td><td> </td></tr>
<tr><td>exbC</td><td>E</td><td>Colicin I resistance</td><td><a href="#ref283">283</a> <a href="#ref624">624</a> </td><td> </td></tr>
<tr><td>exbC</td><td>E</td><td>Colicin (other) resistance</td><td><a href="#ref283">283</a> <a href="#ref624">624</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG12739">exuR</a></td><td>E</td><td>Aldohexuronate resistance in <em>eda</em></td><td><a href="#ref612">612</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG12739">exuR</a></td><td>E</td><td>Hexuronate as C course in noninducible <em>exuR eda</em></td><td><a href="#ref612">612</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG12738">exuT</a></td><td>E</td><td>Aldohexuronate resistance in <em>eda</em></td><td><a href="#ref612">612</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10279">fadB</a></td><td>E</td><td>Thiolactomycin resistance (due to gene dosage)</td><td><a href="#ref778">778</a> </td><td>D</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10280">fadL</a></td><td>E</td><td>T2 phage resistance in <em>ompF</em>+ or <em>ompF</em></td><td><a href="#ref70">70</a> <a href="#ref534">534</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10281">fadR</a></td><td>E</td><td>Decanoate as C source</td><td><a href="#ref713">713</a> </td><td>L</td></tr>
<tr><td>fba</td><td>E</td><td>Streptozotocin resistance</td><td><a href="#ref456">456</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10293">fepABCDG</a></td><td>E</td><td>Colicin B resistance</td><td><a href="#ref623">623</a> <a href="#ref624">624</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10293">fepABDCG</a></td><td>E</td><td>Colicin D resistance</td><td><a href="#ref623">623</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10302">fhuA</a></td><td>E, S</td><td>Albomycin resistance</td><td><a href="#ref87">87</a> <a href="#ref387">387</a> <a href="#ref388">388</a> <a href="#ref478">478</a> <a href="#ref479">479</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10302">fhuA</a></td><td>S</td><td>ES18 phage resistance</td><td><a href="#ref87">87</a> <a href="#ref478">478</a> <a href="#ref479">479</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10302">fhuA</a></td><td>E</td><td>T1 phage resistance</td><td><a href="#ref93">93</a> <a href="#ref409">409</a> <a href="#ref870">870</a> </td><td>S, L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10302">fhuA</a></td><td>E</td><td>T5 phage resistance</td><td><a href="#ref93">93</a> <a href="#ref151">151</a> <a href="#ref387">387</a> <a href="#ref870">870</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10302">fhuA</a></td><td>E</td><td>Φ80 phage resistance</td><td><a href="#ref151">151</a> <a href="#ref387">387</a> <a href="#ref409">409</a> <a href="#ref870">870</a> </td><td>L, S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10302">fhuA</a></td><td>E</td><td>Colicin M resistance</td><td><a href="#ref151">151</a> <a href="#ref387">387</a> <a href="#ref409">409</a> </td><td>L, S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10302">fhuA</a></td><td>E</td><td>Beta-lactam conjugate resistance</td><td><a href="#ref93">93</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10303">fhuB</a></td><td>E</td><td>Albomycin resistance</td><td><a href="#ref618">618</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10321">fliC</a></td><td>E, S</td><td>Chi phage resistance</td><td><a href="#ref347">347</a> <a href="#ref349">349</a> <a href="#ref416">416</a> <a href="#ref417">417</a> <a href="#ref513">513</a> <a href="#ref710">710</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10321">fliC</a></td><td>S</td><td>Prolonged static cultivation</td><td><a href="#ref249">249</a> </td><td> </td></tr>
<tr><td>flrB</td><td>S</td><td>Trifluoroleucine resistance</td><td><a href="#ref115">115</a> <a href="#ref117">117</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10326">folA</a></td><td>E</td><td>Sulfonamide resistance</td><td><a href="#ref607">607</a> <a href="#ref697">697</a> </td><td>S, D</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10326">folA</a></td><td>S</td><td>Trimethoprim resistance</td><td><a href="#ref405">405</a> <a href="#ref654">654</a> <a href="#ref718">718</a> <a href="#ref719">719</a> </td><td>S, D</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG50011">folP</a></td><td>E</td><td>Sulfonamide resistance</td><td><a href="#ref756">756</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10336">fruA</a></td><td>E</td><td>Deoxy(2)glucose-independent fructose utilization</td><td><a href="#ref11">11</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10336">fruA</a></td><td>E</td><td>Xylitol resistance</td><td><a href="#ref614">614</a> <a href="#ref639">639</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10336">fruA</a></td><td>E</td><td>Sorbose (L) resistance</td><td><a href="#ref715">715</a> </td><td>L</td></tr>
<tr><td>fts (generic)</td><td>E</td><td>Filter retention</td><td><a href="#ref55">55</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10347">ftsZ</a></td><td>E</td><td>UV irradiation resistance in <em>lon</em></td><td><a href="#ref373">373</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10347">ftsZ</a></td><td>E</td><td>Methyl methanesulfonate resistance in <em>lon</em></td><td><a href="#ref373">373</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10347">ftsZ</a></td><td>E</td><td>Nitrofurantoin resistance in <em>lon</em></td><td><a href="#ref252">252</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10348">fucA</a></td><td>E</td><td>Propanediol as C source</td><td><a href="#ref287">287</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10348">fucA</a></td><td>E</td><td>Ethyleneglycol as C source in propanediol utilizer</td><td><a href="#ref79">79</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10348">fucA</a></td><td>E</td><td>Fucose (L) as C source in propanediol utilizer</td><td><a href="#ref287">287</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10348">fucA</a></td><td>E</td><td>Xylitol as C source in propanediol utilizer</td><td><a href="#ref848">848</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10348">fucA</a></td><td>E</td><td>Arabinose (D) as C source</td><td><a href="#ref450">450</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10359">fur</a></td><td>E</td><td>Manganese resistance</td><td><a href="#ref300">300</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10360">fusA</a></td><td>E</td><td>Fusidic acid resistance</td><td><a href="#ref65">65</a> <a href="#ref434">434</a> <a href="#ref695">695</a> <a href="#ref761">761</a> </td><td>S</td></tr>
<tr><td>gabC</td><td>E</td><td>Aminobutyrate as N source</td><td><a href="#ref197">197</a> </td><td> </td></tr>
<tr><td>gadS</td><td>E</td><td>Glutamate as C source</td><td><a href="#ref292">292</a> </td><td>L</td></tr>
<tr><td>gal operon</td><td>E</td><td>Galactose utilization in the presence of thiomethylglucoside</td><td><a href="#ref636">636</a> </td><td>D</td></tr>
<tr><td>gal operon</td><td>S</td><td>Deoxy(2)galactose resistance</td><td><a href="#ref9">9</a> <a href="#ref552">552</a> </td><td>L</td></tr>
<tr><td>gal operon</td><td>E</td><td>Galactose resistance in <em>galU</em></td><td><a href="#ref664">664</a> </td><td>L</td></tr>
<tr><td>galC</td><td>S</td><td>Glucose as C source in ∆<em>ptsHI</em></td><td><a href="#ref613">613</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10362">galE</a></td><td>E, S</td><td>P1 phage infection of <em>Salmonella</em> sp. monitored by drug resistance</td><td><a href="#ref527">527</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10362">galE</a></td><td>E</td><td>U3 phage resistance</td><td><a href="#ref816">816</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10362">galE</a></td><td>S</td><td>Deoxy(2)galactose resistance</td><td><a href="#ref408">408</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10362">galE</a></td><td>S</td><td>Felix O phage resistance</td><td><a href="#ref334">334</a> <a href="#ref527">527</a> <a href="#ref591">591</a> <a href="#ref833">833</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10362">galE</a></td><td>S</td><td>P22 phage resistance</td><td><a href="#ref334">334</a> <a href="#ref527">527</a> <a href="#ref591">591</a> <a href="#ref833">833</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10363">galK</a></td><td>S</td><td>Deoxy(2)galactose resistance</td><td><a href="#ref408">408</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10363">galK</a></td><td>E, S</td><td>Galactose resistance in <em>galE</em></td><td><a href="#ref569">569</a> <a href="#ref752">752</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10363">galK</a></td><td>E</td><td>Galactose resistance in <em>galT</em></td><td><a href="#ref859">859</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10363">galK</a></td><td>E</td><td>Galactose resistance in <em>galU</em></td><td><a href="#ref664">664</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG12148">galP</a></td><td>S</td><td>Deoxy(2)galactose resistance</td><td><a href="#ref552">552</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10364">galR</a></td><td>E</td><td>Thiomethylgalactoside-independent utilization of galactose in <em>galR</em>(Con)</td><td><a href="#ref108">108</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10364">galR</a></td><td>S</td><td>Glucose utilization in ∆<em>ptsHI</em></td><td><a href="#ref613">613</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10364">galR</a></td><td>E</td><td>Galactose resistance in <em>galT</em>+<em>K</em>+<em>E</em>+/<em>galT</em>+<em>K</em>+<em>E</em>+ <em>galU</em></td><td><a href="#ref664">664</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10366">galT</a></td><td>E</td><td>Streptozotocin resistance</td><td><a href="#ref456">456</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11319">galU</a></td><td>E</td><td>Ampicillin resistance</td><td><a href="#ref212">212</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11319">galU</a></td><td>E</td><td>P1 phage resistance</td><td><a href="#ref242">242</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11319">galU</a></td><td>E</td><td>U3 phage resistance</td><td><a href="#ref816">816</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11319">galU</a></td><td>S</td><td>Felix O phage resistance</td><td><a href="#ref334">334</a> </td><td>L</td></tr>
<tr><td>garA</td><td>S</td><td>Gamma irradiation resistance</td><td><a href="#ref346">346</a> </td><td> </td></tr>
<tr><td>garA</td><td>E</td><td>Glucarate (D) resistance in <em>ppc</em></td><td><a href="#ref647">647</a> </td><td>L</td></tr>
<tr><td>garB</td><td>S</td><td>Gamma irradiation resistance</td><td><a href="#ref346">346</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG12414">gatACD</a></td><td>E</td><td>Fucitol resistance in galactitol-utilizing strains</td><td><a href="#ref184">184</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG12414">gatACD</a></td><td>E</td><td>Deoxy(2)galactitol resistance in galactitol-utilizing strains</td><td><a href="#ref184">184</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG12414">gatACD</a></td><td>E</td><td>Arabitol utilization</td><td><a href="#ref466">466</a> <a href="#ref845">845</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG12414">gatACD</a></td><td>E</td><td>Ribitol utilization</td><td><a href="#ref466">466</a> <a href="#ref845">845</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG12414">gatACD</a></td><td>E</td><td>Arbitol (D) resistance</td><td><a href="#ref639">639</a> </td><td>L</td></tr>
<tr><td>gleR</td><td>S</td><td>Glycylleucine resistance in <em>ilv</em></td><td><a href="#ref580">580</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10383">glnA</a></td><td>S</td><td>Histidine(D) + glycylglutamine satisfaction of <em>his</em> auxotrophs</td><td><a href="#ref819">819</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10383">glnA</a></td><td>E</td><td>Methylammonium resistance</td><td><a href="#ref694">694</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10383">glnA</a></td><td>E</td><td>Methionine sulfoxamine resistance in <em>glnG</em></td><td><a href="#ref592">592</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10383">glnA</a></td><td>S</td><td>Glutamyl(γ)hydrazide resistance</td><td><a href="#ref518">518</a> <a href="#ref871">871</a> </td><td>S</td></tr>
<tr><td>glnF</td><td>S</td><td>Glutamyl(γ)hydrazide resistance</td><td><a href="#ref433">433</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10386">glnH</a></td><td>S</td><td>Glutamyl(γ)hydrazide resistance</td><td><a href="#ref433">433</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10388">glnP</a></td><td>E</td><td>Glutamine as C source</td><td><a href="#ref433">433</a> <a href="#ref501">501</a> <a href="#ref820">820</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10388">glnP</a></td><td>E, S</td><td>Methionine sulfoxamine resistance</td><td><a href="#ref37">37</a> <a href="#ref501">501</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10388">glnP</a></td><td>E</td><td>Glutamyl(γ)hydrazide resistance</td><td><a href="#ref501">501</a> <a href="#ref821">821</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10388">glnP</a></td><td>E</td><td>Glutamine (D) resistance</td><td><a href="#ref501">501</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG30029">glnV</a></td><td>E</td><td>Caffeine resistance</td><td><a href="#ref185">185</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10394">glpD</a></td><td>E</td><td>Streptozotocin resistance</td><td><a href="#ref456">456</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10396">glpF</a></td><td>E</td><td>Glyceraldehyde (L) resistance</td><td><a href="#ref762">762</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10398">glpK</a></td><td>E</td><td>Glyceraldehyde (L) resistance</td><td><a href="#ref762">762</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10398">glpK</a></td><td>E, S</td><td>Methyl(α)glucoside-resistant glycerol utilization in <em>ptsI</em></td><td><a href="#ref64">64</a> <a href="#ref665">665</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10401">glpT</a></td><td>E, S</td><td>Fosfomycin resistance</td><td><a href="#ref10">10</a> <a href="#ref29">29</a> <a href="#ref313">313</a> <a href="#ref707">707</a> <a href="#ref800">800</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10401">glpT</a></td><td>E</td><td>Glyceraldehyde (DL) 3-phosphate resistance</td><td><a href="#ref762">762</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10401">glpT</a></td><td>E</td><td>Arsenate resistance</td><td><a href="#ref838">838</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10401">glpT</a></td><td>E</td><td>Dihydroxybutyl phosphonate resistance</td><td><a href="#ref285">285</a> <a href="#ref453">453</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10401">glpT</a></td><td>E</td><td>Fluorohydroxyacetone phosphate resistance</td><td><a href="#ref515">515</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10401">glpT</a></td><td>E</td><td>Chlorohydroxyacetone phosphate resistance</td><td><a href="#ref515">515</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10401">glpT</a></td><td>E</td><td>Glycerol 3-phosphorothioate resistance</td><td><a href="#ref295">295</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10402">gltA</a></td><td>E</td><td>Improved growth in <em>icd</em></td><td><a href="#ref435">435</a> </td><td>L</td></tr>
<tr><td>gltH</td><td>E</td><td>Glutamate as C source</td><td><a href="#ref496">496</a> <a href="#ref497">497</a> <a href="#ref687">687</a> </td><td> </td></tr>
<tr><td>gltR</td><td>E</td><td>Glutamate as C source at 42°C</td><td><a href="#ref497">497</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10406">gltS</a></td><td>E</td><td>Glutamate as C source</td><td><a href="#ref496">496</a> <a href="#ref687">687</a> </td><td>Q</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10406">gltS</a></td><td>E</td><td>Glutamate as C source</td><td><a href="#ref496">496</a> <a href="#ref497">497</a> <a href="#ref687">687</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10406">gltS</a></td><td>E</td><td>Glutamate (D) resistance in <em>gltS</em> (increased)</td><td><a href="#ref525">525</a> <a href="#ref716">716</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10406">gltS</a></td><td>E</td><td>Methyl(α)glutamate resistance in <em>gltS</em> (increased)</td><td><a href="#ref391">391</a> <a href="#ref525">525</a> <a href="#ref716">716</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10406">gltS</a></td><td>E</td><td>Homocysteic acid resistance</td><td><a href="#ref213">213</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10407">gltX</a></td><td>E</td><td>Glutamyl-γ-methyl ester resistance</td><td><a href="#ref425">425</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10408">glyA</a></td><td>E</td><td>Serine + methionine + glycine resistance in <em>relA</em></td><td><a href="#ref783">783</a> </td><td> </td></tr>
<tr><td>gntM</td><td>E</td><td>Glycerol + gluconate as C source in <em>eda</em></td><td><a href="#ref217">217</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG12630">gntR</a></td><td>E</td><td>Acetate + gluconate as C source in <em>ppc</em></td><td><a href="#ref39">39</a> </td><td> </td></tr>
<tr><td>gntS</td><td>E</td><td>Acetate + gluconate as C source in <em>ppc</em></td><td><a href="#ref39">39</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10412">gor</a></td><td>E</td><td>Methyl viologen tolerance</td><td><a href="#ref430">430</a> </td><td>L</td></tr>
<tr><td>gprAB</td><td>E</td><td>Lambda phage resistance</td><td><a href="#ref578">578</a> <a href="#ref670">670</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10414">gpt</a></td><td>S</td><td>Aza(8)guanine resistance</td><td><a href="#ref272">272</a> <a href="#ref764">764</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10414">gpt</a></td><td>E</td><td>Sulfanilamide resistance in presence of guanine</td><td><a href="#ref97">97</a> </td><td>L</td></tr>
<tr><td>groM</td><td>E</td><td>T7 phage resistance</td><td><a href="#ref427">427</a> </td><td> </td></tr>
<tr><td>grpD</td><td>E</td><td>Lambda mutant prophage induction, survival of</td><td><a href="#ref667">667</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10416">grpE</a></td><td>E</td><td>Lambda mutant prophage induction, survival of</td><td><a href="#ref667">667</a> </td><td> </td></tr>
<tr><td>gsh (generic)</td><td>E</td><td>Nitrosoguanidine resistance</td><td><a href="#ref693">693</a> </td><td>L</td></tr>
<tr><td>gsh (generic)</td><td>E</td><td>Methylglyoxal resistance</td><td><a href="#ref548">548</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10418">gshAB</a></td><td>E, S</td><td>Selenite resistance</td><td><a href="#ref423">423</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10420">guaAB</a></td><td>E</td><td>Psicofuranine resistance</td><td><a href="#ref779">779</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10421">guaB</a></td><td>S</td><td>Fluoro(5)uracil + fluoro(5)uridine resistance</td><td><a href="#ref367">367</a> </td><td>L</td></tr>
<tr><td>gurBCD</td><td>E</td><td>Glycerol + methylglucuronide a C source in <em>eda</em></td><td><a href="#ref572">572</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10423">gyrA</a></td><td>E, S</td><td>Triazolealanine + aminotriazole resistance</td><td><a href="#ref660">660</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10423">gyrA</a></td><td>E, S</td><td>Nalidixic acid resistance</td><td><a href="#ref200">200</a> <a href="#ref298">298</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10423">gyrA</a></td><td>E</td><td>Quinolone resistance</td><td><a href="#ref860">860</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10423">gyrA</a></td><td>E</td><td>Oxolinic acid resistance</td><td><a href="#ref734">734</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10423">gyrA</a></td><td>E</td><td>Glucosides (β) as C source</td><td><a href="#ref194">194</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10423">gyrA</a></td><td>S</td><td>Thermotolerance</td><td><a href="#ref200">200</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10424">gyrB</a></td><td>S</td><td>Triaolealanine + aminotriazole resistance</td><td><a href="#ref660">660</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10424">gyrB</a></td><td>E</td><td>Aminotriazole resistance in <em>relA</em></td><td><a href="#ref772">772</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10424">gyrB</a></td><td>E</td><td>Novobiocin resistance</td><td><a href="#ref255">255</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10424">gyrB</a></td><td>E</td><td>Coumermycin resistance</td><td><a href="#ref181">181</a> <a href="#ref255">255</a> <a href="#ref593">593</a> </td><td>D, S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10424">gyrB</a></td><td>E</td><td>Chlorobiocin resistance</td><td><a href="#ref589">589</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10424">gyrB</a></td><td>E</td><td>Glucosides (β) as C source</td><td><a href="#ref194">194</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10424">gyrB</a></td><td>E</td><td>Lambda mutant infection resistance</td><td><a href="#ref521">521</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10424">gyrB</a></td><td>E</td><td>Nalidixic acid resistance</td><td><a href="#ref333">333</a> <a href="#ref855">855</a> </td><td>S</td></tr>
<tr><td>hem</td><td>E</td><td>Streptomycin resistance</td><td><a href="#ref57">57</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10427">hemA</a></td><td>E</td><td>Kanamycin resistance</td><td><a href="#ref585">585</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10427">hemA</a></td><td>E, S</td><td>Neomycin resistance</td><td><a href="#ref683">683</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10428">hemB</a></td><td>E</td><td>Kanamycin resistance</td><td><a href="#ref585">585</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10428">hemB</a></td><td>E</td><td>Kanamycin-resistant, hemin-supported growth of <em>hemA</em></td><td><a href="#ref507">507</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10428">hemB</a></td><td>E</td><td>Colicin K-type tolerance</td><td><a href="#ref83">83</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10429">hemC</a></td><td>S</td><td>Neomycin resistance</td><td><a href="#ref682">682</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10430">hemD</a></td><td>S</td><td>Neomycin resistance</td><td><a href="#ref681">681</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11543">hemE</a></td><td>S</td><td>Neomycin resistance</td><td><a href="#ref189">189</a> <a href="#ref680">680</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10428">hemBCF</a></td><td>E</td><td>Plate method</td><td><a href="#ref506">506</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11485">hemG</a></td><td>E</td><td>Neomycin resistance</td><td><a href="#ref679">679</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10432">hemL</a></td><td>E</td><td>Kanamycin resistance</td><td><a href="#ref350">350</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10435">hflC</a></td><td>E</td><td>Lambda mutant phage resistance</td><td><a href="#ref56">56</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10436">hflK</a></td><td>E</td><td>Lambda mutant phage resistance</td><td><a href="#ref56">56</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10437">hflX</a></td><td>E</td><td>Lambda mutant phage resistance</td><td><a href="#ref56">56</a> </td><td> </td></tr>
<tr><td>himA</td><td>E</td><td>Mu prophage induction, survival of</td><td><a href="#ref520">520</a> <a href="#ref522">522</a> </td><td> </td></tr>
<tr><td>himA</td><td>E</td><td>Mu phage lytic growth resistance</td><td><a href="#ref81">81</a> </td><td> </td></tr>
<tr><td>himA</td><td>E</td><td>Lambda mutant phage infection, survival of</td><td><a href="#ref519">519</a> <a href="#ref521">521</a> </td><td> </td></tr>
<tr><td>himA (himC)</td><td>E</td><td>Lambda mutant phage resistance</td><td><a href="#ref519">519</a> <a href="#ref521">521</a> </td><td> </td></tr>
<tr><td>himD</td><td>E</td><td>Mu prophage induction, survival of</td><td><a href="#ref520">520</a> <a href="#ref522">522</a> </td><td> </td></tr>
<tr><td>himD</td><td>E</td><td>Mu phage lytic growth resistance</td><td><a href="#ref81">81</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10443">hipA</a></td><td>E</td><td>Cycloserine resistance</td><td><a href="#ref541">541</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10443">hipA</a></td><td>E</td><td>Ampicillin resistance</td><td><a href="#ref541">541</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10443">hipA</a></td><td>E</td><td>Fosfomycin resistance</td><td><a href="#ref541">541</a> </td><td> </td></tr>
<tr><td>hipQ</td><td>E</td><td>Norfloxacin resistance</td><td><a href="#ref842">842</a> </td><td> </td></tr>
<tr><td>hipQ</td><td>E</td><td>Ampicillin resistance</td><td><a href="#ref842">842</a> </td><td> </td></tr>
<tr><td>his structural genes</td><td>E, S</td><td>Triazolealanine + aminotriazole resistance</td><td><a href="#ref98">98</a> <a href="#ref126">126</a> <a href="#ref658">658</a> </td><td>C, Q</td></tr>
<tr><td>his structural genes</td><td>S</td><td>Aminotriazole resistance</td><td><a href="#ref20">20</a> <a href="#ref21">21</a> </td><td>D</td></tr>
<tr><td>his structural genes</td><td>S</td><td>Growth at 42°C in <em>his</em>-overexpressing strain</td><td><a href="#ref804">804</a> </td><td>Q</td></tr>
<tr><td>his structural genes</td><td>S</td><td>Histidinal satisfaction of <em>his</em> auxotroph requirement at 30°C</td><td><a href="#ref374">374</a> <a href="#ref375">375</a> </td><td>Q</td></tr>
<tr><td>his structural genes</td><td>S</td><td>Histidine (D) satisfaction of <em>his</em> auxotroph requirement</td><td><a href="#ref374">374</a> <a href="#ref375">375</a> </td><td>Q</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10449">hisG</a></td><td>E, S</td><td>Thiazolealanine resistance</td><td><a href="#ref540">540</a> <a href="#ref698">698</a> <a href="#ref824">824</a> </td><td>S, FBI</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10449">hisG</a></td><td>S</td><td>Hydrazino (α) imidazole propionic acid resistance</td><td><a href="#ref699">699</a> </td><td>S, FBI</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10449">hisG</a></td><td>S</td><td>Histidinal satisfaction of <em>his</em> auxotroph requirement at 30°C</td><td><a href="#ref374">374</a> <a href="#ref375">375</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10449">hisG</a></td><td>S</td><td>Histidine (D) satisfaction of <em>his</em> auxotroph requirement</td><td><a href="#ref374">374</a> <a href="#ref375">375</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10449">hisG</a></td><td>E</td><td>Histidine-resistant adenine → guanine nucleotide</td><td><a href="#ref33">33</a> </td><td>S,FBI</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG12124">hisJ</a></td><td>S</td><td>Hydrazino (α) imidazole propionic acid resistance</td><td><a href="#ref13">13</a> <a href="#ref15">15</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG12124">hisJ</a></td><td>S</td><td>Azaserine + tryptophan resistance in <em>dhuA</em></td><td><a href="#ref15">15</a> <a href="#ref432">432</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10452">hisP</a></td><td>S</td><td>Hydrazino (α) imidazole propionic acid resistance in <em>dhuA</em></td><td><a href="#ref14">14</a> <a href="#ref15">15</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10452">hisP</a></td><td>S</td><td>Arginine as N source</td><td><a href="#ref13">13</a> <a href="#ref15">15</a> <a href="#ref432">432</a> </td><td>D</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG12125">hisQ</a></td><td>S</td><td>Hydrazino (α) imidazole propionic acid resistance</td><td><a href="#ref13">13</a> <a href="#ref15">15</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG30042">hisR</a></td><td>S</td><td>Triazolealanine + aminotriazole resistance</td><td><a href="#ref658">658</a> </td><td>Q</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG30042">hisR</a></td><td>S</td><td>Thiazolealanine resistance</td><td><a href="#ref706">706</a> </td><td>Q</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG30042">hisR</a></td><td>S</td><td>Methyl(2)histidine resistance + aminotriazole resistance</td><td><a href="#ref658">658</a> </td><td>Q</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10453">hisS</a></td><td>S</td><td>Triazolealanine + aminotriazole resistance</td><td><a href="#ref658">658</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10453">hisS</a></td><td>S</td><td>Thiazolealanine resistance</td><td><a href="#ref657">657</a> </td><td>S</td></tr>
<tr><td>hisT</td><td>E, S</td><td>Triazolealanine + aminotriazole resistance</td><td><a href="#ref98">98</a> <a href="#ref148">148</a> <a href="#ref658">658</a> <a href="#ref705">705</a> <a href="#ref784">784</a> </td><td>L</td></tr>
<tr><td>hisT</td><td>E</td><td>Azaleucine resistance</td><td><a href="#ref98">98</a> <a href="#ref784">784</a> </td><td>L</td></tr>
<tr><td>hisT</td><td>E</td><td>Serine resistance</td><td><a href="#ref98">98</a> <a href="#ref784">784</a> </td><td>L</td></tr>
<tr><td>hisT</td><td>S</td><td>Amino(3)tyrosine resistance</td><td><a href="#ref148">148</a> <a href="#ref658">658</a> <a href="#ref705">705</a> </td><td>L</td></tr>
<tr><td>hisT</td><td>S</td><td>Thialysine resistance</td><td><a href="#ref148">148</a> <a href="#ref658">658</a> <a href="#ref705">705</a> </td><td>L</td></tr>
<tr><td>hisT</td><td>S</td><td>Trifluoroleucine resistance</td><td><a href="#ref148">148</a> <a href="#ref658">658</a> <a href="#ref705">705</a> </td><td>L</td></tr>
<tr><td>hisT</td><td>S</td><td>Norleucine resistance</td><td><a href="#ref148">148</a> <a href="#ref658">658</a> <a href="#ref705">705</a> </td><td>L</td></tr>
<tr><td>hisT</td><td>S</td><td>Hydroxy (β) leucine resistance</td><td><a href="#ref148">148</a> <a href="#ref658">658</a> <a href="#ref705">705</a> </td><td>L</td></tr>
<tr><td>hisT</td><td>E</td><td>Phenyl galactoside resistance of <em>lacP lacI</em> (nonsense) <em>supE</em></td><td><a href="#ref747">747</a> </td><td> </td></tr>
<tr><td>hisU</td><td>S, E</td><td>Triazolealanine + aminotriazole resistance</td><td><a href="#ref23">23</a> <a href="#ref660">660</a> </td><td> </td></tr>
<tr><td>hisU</td><td>S</td><td>Nalidixic acid resistance</td><td><a href="#ref593">593</a> <a href="#ref660">660</a> </td><td> </td></tr>
<tr><td>hisW</td><td>S, E</td><td>Triazolealanine + aminotriazole resistance</td><td><a href="#ref23">23</a> </td><td> </td></tr>
<tr><td>hisW</td><td>S</td><td>Coumermycin resistance</td><td><a href="#ref593">593</a> <a href="#ref660">660</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG20098">hpt</a></td><td>E</td><td>Mercapto(6)purine resistance in <em>gpt</em></td><td><a href="#ref371">371</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG20098">hpt</a></td><td>E</td><td>Sulfanilamide + hypoxanthine resistance</td><td><a href="#ref97">97</a> </td><td>L</td></tr>
<tr><td>hsd</td><td>E</td><td>Lambda <em>vir</em> resistance after infection with</td><td><a href="#ref183">183</a> </td><td>L</td></tr>
<tr><td>hsd</td><td>S</td><td>Lactose utilization after conjugation of <em>F</em>′<em>lac</em> into <em>Salmonella</em> sp.</td><td><a href="#ref103">103</a> </td><td>L</td></tr>
<tr><td>hut</td><td>S</td><td>Histidine as sole N or C source</td><td><a href="#ref509">509</a> </td><td> </td></tr>
<tr><td>icdE</td><td>E, S</td><td>Nalidixic acid resistance</td><td><a href="#ref91">91</a> <a href="#ref311">311</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10491">iclR</a></td><td>E</td><td>Butyrate or valerate as better C source</td><td><a href="#ref602">602</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10491">iclR</a></td><td>E</td><td>Glucose or glycerol as C source in <em>ppc</em></td><td><a href="#ref488">488</a> </td><td> </td></tr>
<tr><td>ileR</td><td>E</td><td>Trifluoroleucine resistance</td><td><a href="#ref412">412</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10492">ileS</a></td><td>E</td><td>Pseudomonic acid resistance</td><td><a href="#ref858">858</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10492">ileS</a></td><td>E</td><td>Thiaisoleucine resistance</td><td><a href="#ref757">757</a> </td><td>S</td></tr>
<tr><td>ilv structural genes</td><td>S</td><td>Sulfometuron methyl + valine resistance</td><td><a href="#ref872">872</a> </td><td>C</td></tr>
<tr><td>ilv structural genes</td><td>E</td><td>Aminobutyrate resistance</td><td><a href="#ref633">633</a> <a href="#ref780">780</a> </td><td>C</td></tr>
<tr><td>ilv structural genes</td><td>E</td><td>Valine resistance</td><td><a href="#ref268">268</a> </td><td>C</td></tr>
<tr><td>ilv structural genes</td><td>S</td><td>Cyclopentane glycine resistance</td><td><a href="#ref586">586</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10493">ilvA</a></td><td>S</td><td>Cyclopentane glycine resistance</td><td><a href="#ref586">586</a> </td><td>S, FBI</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10493">ilvA</a></td><td>E</td><td>Glyclleucine resistance</td><td><a href="#ref806">806</a> </td><td>S, FBI</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10493">ilvA</a></td><td>S</td><td>Threonine as sole N source</td><td><a href="#ref106">106</a> </td><td>S, FBI</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10493">ilvA</a></td><td>E</td><td>Growth on minimal medium of <em>ilvA+</em> (multicopy) <em>ilvDC</em> <em>hisT</em> overcoming 2-ketobutyrate accumulation</td><td><a href="#ref231">231</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10494">ilvB</a></td><td>E</td><td>Aminobutyrate resistance</td><td><a href="#ref633">633</a> <a href="#ref780">780</a> </td><td>S, FBI</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10494">ilvB</a></td><td>E</td><td>Valine resistance</td><td><a href="#ref633">633</a> <a href="#ref755">755</a> </td><td>S, FBI</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10494">ilvB</a></td><td>E</td><td>Glyclleucine resistance</td><td><a href="#ref269">269</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10494">ilvB</a></td><td>E</td><td>Phenylalanylleucine resistance</td><td><a href="#ref269">269</a> </td><td>S</td></tr>
<tr><td>ilvF</td><td>E</td><td>Valine resistance</td><td><a href="#ref174">174</a> <a href="#ref610">610</a> </td><td>G</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10498">ilvG</a></td><td>E, S</td><td>Sulfometuron methyl + valine resistance</td><td><a href="#ref441">441</a> <a href="#ref851">851</a> <a href="#ref872">872</a> </td><td>S, Q</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10498">ilvG</a></td><td>E</td><td>Valine resistance</td><td><a href="#ref221">221</a> <a href="#ref448">448</a> <a href="#ref720">720</a> </td><td>G</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10499">ilvH</a></td><td>E</td><td>Aminobutyrate resistance</td><td><a href="#ref633">633</a> <a href="#ref780">780</a> </td><td>S, FBI</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10499">ilvH</a></td><td>E</td><td>Valine resistance</td><td><a href="#ref633">633</a> <a href="#ref755">755</a> </td><td>S, FBI</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10499">ilvH</a></td><td>E</td><td>Glyclleucine resistance</td><td><a href="#ref269">269</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10499">ilvH</a></td><td>E</td><td>Phenylalanylleucine resistance</td><td><a href="#ref269">269</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10500">ilvI</a></td><td>E</td><td>Aminobutyrate resistance</td><td><a href="#ref633">633</a> <a href="#ref780">780</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10500">ilvI</a></td><td>E</td><td>Valine resistance</td><td><a href="#ref633">633</a> <a href="#ref755">755</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10500">ilvI</a></td><td>E</td><td>Glyclleucine resistance</td><td><a href="#ref269">269</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10500">ilvI</a></td><td>E</td><td>Phenylalanylleucine resistance</td><td><a href="#ref269">269</a> </td><td> </td></tr>
<tr><td>ilvJ</td><td>E</td><td>Valine resistance</td><td><a href="#ref174">174</a> <a href="#ref361">361</a> <a href="#ref649">649</a> </td><td>G</td></tr>
<tr><td>ilvU</td><td>E</td><td>Thiaisoleucine resistance</td><td><a href="#ref222">222</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=G7003">kdgR</a></td><td>E</td><td>Keto(2)-3-deoxygluconate as C source</td><td><a href="#ref377">377</a> <a href="#ref615">615</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10518">kdsA</a></td><td>S</td><td>9NA phage resistance in <em>galE</em></td><td><a href="#ref452">452</a> <a href="#ref641">641</a> </td><td> </td></tr>
<tr><td>ksgA</td><td>E</td><td>Kasugamycin resistance</td><td><a href="#ref312">312</a> <a href="#ref726">726</a> <a href="#ref793">793</a> </td><td>L</td></tr>
<tr><td>ksgB</td><td>E</td><td>Kasugamycin resistance</td><td><a href="#ref238">238</a> <a href="#ref726">726</a> </td><td> </td></tr>
<tr><td>ksgC</td><td>E</td><td>Kasugamycin resistance</td><td><a href="#ref861">861</a> </td><td> </td></tr>
<tr><td>ksgD</td><td>E</td><td>Kasugamycin resistance</td><td><a href="#ref238">238</a> </td><td> </td></tr>
<tr><td>lac</td><td>E</td><td>Growth in lactose-limited chemostats</td><td><a href="#ref330">330</a> </td><td>D</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10525">lacI,operator</a></td><td>E</td><td>Methyl(α)glucoside-resistant utilization of lactose in <em>ptsH</em></td><td><a href="#ref665">665</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10525">lacI,operator</a></td><td>E</td><td>Raffinose as C source</td><td><a href="#ref459">459</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10525">lacI,operator</a></td><td>E</td><td>Neolactose as C source</td><td><a href="#ref459">459</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10525">lacI,operator</a></td><td>E</td><td>Phenyl galactoside as C source</td><td><a href="#ref523">523</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10525">lacI,operator</a></td><td>E</td><td>Lactobionic acid as C source</td><td><a href="#ref440">440</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10525">lacI,operator</a></td><td>E</td><td>Lactitol as C source</td><td><a href="#ref457">457</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10525">lacI,operator</a></td><td>E</td><td>Melibiose as C source</td><td><a href="#ref475">475</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10525">lacIoperator</a></td><td>E</td><td>Melibiose as C source and acetyl(<em>N</em>)lactonate as N source</td><td><a href="#ref475">475</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10525">lacI,operator</a></td><td>E</td><td>Lactobionic acid as C source</td><td><a href="#ref474">474</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10525">lacI,operator</a></td><td>E</td><td>Acetyl(<em>N</em>)lactonate as N source</td><td><a href="#ref474">474</a> </td><td>Q</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10525">lacI,operator</a></td><td>E</td><td>Nitro(<em>o</em>)-phenylthiogalactoside resistance in <em>lacI</em>(Con) or <em>lacO</em>(Con)</td><td><a href="#ref543">543</a> <a href="#ref571">571</a> </td><td>Q</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10525">lacI,operator</a></td><td>E</td><td>Methyl(α)glucoside-resistant lactose utilization</td><td><a href="#ref406">406</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10526">lacY</a></td><td>E</td><td>Nitro(<em>o</em>)-phenylthiogalactoside resistance</td><td><a href="#ref523">523</a> <a href="#ref543">543</a> <a href="#ref721">721</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10526">lacY</a></td><td>E</td><td>Phenyl galactoside resistance</td><td><a href="#ref376">376</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10526">lacY</a></td><td>E</td><td>Phenethylyl galactoside resistance</td><td><a href="#ref376">376</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10526">lacY</a></td><td>E</td><td>Arabinose (L) growth in the presence of <em>lac</em> operon inducer</td><td><a href="#ref812">812</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10526">lacY</a></td><td>E</td><td>Thiodigalactoside resistance</td><td><a href="#ref241">241</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10527">lacZ</a></td><td>E</td><td>Lactobionic acid as C source</td><td><a href="#ref440">440</a> </td><td>D</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10527">lacZ</a></td><td>E</td><td>Phenyl galactoside resistance</td><td><a href="#ref376">376</a> <a href="#ref500">500</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10527">lacZ</a></td><td>E</td><td>Phenethylyl galactoside resistance</td><td><a href="#ref376">376</a> <a href="#ref500">500</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10527">lacZ</a></td><td>E</td><td>Nitro(<em>o</em>)-phenyl galactoside resistance</td><td><a href="#ref376">376</a> <a href="#ref500">500</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10528">lamB</a></td><td>E</td><td>Lambda phage resistance</td><td><a href="#ref88">88</a> <a href="#ref134">134</a> <a href="#ref135">135</a> <a href="#ref253">253</a> <a href="#ref644">644</a> <a href="#ref766">766</a> </td><td>L, S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10528">lamB</a></td><td>E</td><td>K10 phage resistance</td><td><a href="#ref644">644</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10528">lamB</a></td><td>E</td><td>Thio(5)maltose resistance</td><td><a href="#ref223">223</a> </td><td>L</td></tr>
<tr><td>lct</td><td>E</td><td>Vinylglycolate resistance</td><td><a href="#ref696">696</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11226">leuA</a></td><td>S</td><td>Trifluoroleucine resistance</td><td><a href="#ref115">115</a> </td><td>FBI, S</td></tr>
<tr><td>leuJ</td><td>E</td><td>Trifluoroleucine resistance</td><td><a href="#ref575">575</a> </td><td> </td></tr>
<tr><td>leu(structural genes)</td><td>S</td><td>Trifluoroleucine resistance</td><td><a href="#ref115">115</a> <a href="#ref116">116</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10532">leuS</a></td><td>S, E</td><td>Azaleucine resistance</td><td><a href="#ref477">477</a> <a href="#ref514">514</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10532">leuS</a></td><td>S</td><td>Trifluoroleucine resistance</td><td><a href="#ref7">7</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG30052">leuW</a></td><td>E</td><td>Calmodulin inhibitor resistance</td><td><a href="#ref129">129</a> </td><td>S</td></tr>
<tr><td>lev</td><td>E</td><td>Levallorphan resistance</td><td><a href="#ref164">164</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10533">lexA</a></td><td>E</td><td>Mitomycin C resistance of <em>recA</em> (SOS-induced) <em>sfi</em> <em>lexA3</em>(Con)</td><td><a href="#ref538">538</a> </td><td>L</td></tr>
<tr><td>lig</td><td>E</td><td>T4 phage <em>lig</em> mutant nibbling of colonies yielding <em>lig</em> overproducer</td><td><a href="#ref254">254</a> </td><td>Q</td></tr>
<tr><td>lig</td><td>E</td><td>T4 phage <em>lig</em> mutant resistance from <em>lig</em> overproducer</td><td><a href="#ref254">254</a> </td><td>L</td></tr>
<tr><td>linB</td><td>E</td><td>Lincomycin resistance</td><td><a href="#ref26">26</a> <a href="#ref343">343</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10535">lit</a></td><td>E</td><td>T4 mutant phage resistance</td><td><a href="#ref143">143</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10537">livG</a></td><td>E</td><td>Azaleucine resistance</td><td><a href="#ref303">303</a> <a href="#ref561">561</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10538">livH</a></td><td>E</td><td>Valine resistance in <em>leu</em></td><td><a href="#ref17">17</a> <a href="#ref595">595</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10538">livH</a></td><td>E</td><td>Azaleucine resistance</td><td><a href="#ref561">561</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10539">livJ</a></td><td>E</td><td>Valine resistance in <em>leu</em></td><td><a href="#ref17">17</a> <a href="#ref595">595</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10540">livK</a></td><td>E</td><td>Valine resistance in <em>leu</em></td><td><a href="#ref17">17</a> <a href="#ref595">595</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10542">lon</a></td><td>E, S</td><td>Chlorpromazine resistance</td><td><a href="#ref198">198</a> <a href="#ref529">529</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10542">lon</a></td><td>E</td><td>Promethazine resistance</td><td><a href="#ref529">529</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10544">lpp</a></td><td>E</td><td>Globomycin resistance</td><td><a href="#ref140">140</a> <a href="#ref344">344</a> <a href="#ref854">854</a> <a href="#ref868">868</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=G6106">lpcA</a></td><td>E</td><td>T3 phage resistance</td><td><a href="#ref792">792</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=G6106">lpcA</a></td><td>E</td><td>T4 phage resistance</td><td><a href="#ref792">792</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=G6106">lpcA</a></td><td>E</td><td>T7 phage resistance</td><td><a href="#ref792">792</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=G6106">lpcA</a></td><td>E</td><td>P1 phage resistance</td><td><a href="#ref40">40</a> <a href="#ref41">41</a> </td><td> </td></tr>
<tr><td>lpcB</td><td>E</td><td>T4 phage resistance</td><td><a href="#ref760">760</a> </td><td> </td></tr>
<tr><td>lpcB</td><td>E</td><td>T7 phage resistance</td><td><a href="#ref760">760</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10547">lrp</a></td><td>E</td><td>Leucine (D) satisfaction of <em>leu</em> auxotrophs</td><td><a href="#ref18">18</a> <a href="#ref428">428</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10550">lysC</a></td><td>E</td><td>Aminoethylcysteine (thialysine) resistance</td><td><a href="#ref82">82</a> <a href="#ref765">765</a> </td><td>S, FBI</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10550">lysC</a></td><td>E</td><td>Lysine hydroxamate resistance</td><td><a href="#ref82">82</a> <a href="#ref765">765</a> </td><td>S, FBI</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11337">lysP</a></td><td>E</td><td>Aminoethylcysteine (thialysine) resistance</td><td><a href="#ref611">611</a> <a href="#ref736">736</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10552">lysS</a></td><td>E</td><td>Aminoethylcysteine (thialysine) resistance</td><td><a href="#ref323">323</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10552">lysS</a></td><td>E</td><td>Phenyl galactoside resistance of <em>lacI </em>q<em> lacI</em>(nonsense) lacP supL</td><td><a href="#ref748">748</a> </td><td> </td></tr>
<tr><td>lyt</td><td>E</td><td>Beta-lactam tolerance</td><td><a href="#ref700">700</a> </td><td> </td></tr>
<tr><td>mac</td><td>E</td><td>Erythromycin growth dependence</td><td><a href="#ref725">725</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10554">malE</a></td><td>E, S</td><td>Methylglucoside(α)-resistant utilization of maltose</td><td><a href="#ref665">665</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10554">malE</a></td><td>E</td><td>Thio(5)maltose resistance</td><td><a href="#ref223">223</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10555">malF</a></td><td>E, S</td><td>Methylglucoside(α)-resistant utilization of maltose</td><td><a href="#ref665">665</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10555">malF</a></td><td>E</td><td>Thio(5)maltose resistance</td><td><a href="#ref223">223</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10556">malG</a></td><td>E, S</td><td>Methylglucoside(α)-resistant utilization of maltose</td><td><a href="#ref665">665</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10556">malG</a></td><td>E</td><td>Thio(5)maltose resistance</td><td><a href="#ref223">223</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10558">malK</a></td><td>E</td><td>Lambda phage resistance</td><td><a href="#ref766">766</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10558">malK</a></td><td>E, S</td><td>Methylglucoside(α)-resistant utilization of maltose</td><td><a href="#ref177">177</a> <a href="#ref665">665</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10558">malK</a></td><td>E</td><td>Thio(5)maltose resistance</td><td><a href="#ref223">223</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10560">malP</a></td><td>E</td><td>Maltose as C source in <em>malT</em></td><td><a href="#ref327">327</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10561">malQ</a></td><td>E</td><td>Maltose as C source in <em>malT</em></td><td><a href="#ref327">327</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10562">malT</a></td><td>E</td><td>Lambda phage resistance</td><td><a href="#ref766">766</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10562">malT</a></td><td>E</td><td>Thiomaltose resistance</td><td><a href="#ref223">223</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10566">manA</a></td><td>E</td><td>Streptozotocin resistance</td><td><a href="#ref456">456</a> </td><td> </td></tr>
<tr><td>manC</td><td>E</td><td>Lyxose (D) as C source</td><td><a href="#ref738">738</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10567">manXYZ</a></td><td>E</td><td>Phage N4 resistance</td><td><a href="#ref410">410</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10567">manXYZ</a></td><td>E</td><td>Deoxyglucose resistance</td><td><a href="#ref158">158</a> <a href="#ref379">379</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11434">marA</a></td><td>E</td><td>Menadionine resistance</td><td><a href="#ref274">274</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11434">marA</a></td><td>E</td><td>Chloramphenicol resistance</td><td><a href="#ref286">286</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11434">marA</a></td><td>E</td><td>Tetracycline resistance</td><td><a href="#ref286">286</a> <a href="#ref329">329</a> </td><td> </td></tr>
<tr><td>mbrABCD</td><td>E</td><td>Camphor resistance</td><td><a href="#ref776">776</a> <a href="#ref777">777</a> </td><td> </td></tr>
<tr><td>mdoB</td><td>E</td><td>Arbutin resistance in <em>dgk</em></td><td><a href="#ref360">360</a> </td><td>L</td></tr>
<tr><td>mel (generic)</td><td>E</td><td>Raffinose as C source in <em>lacI</em></td><td><a href="#ref459">459</a> </td><td> </td></tr>
<tr><td>mel (generic)</td><td>S</td><td>Methylglucoside(α)-resistant utilization of melibiose</td><td><a href="#ref665">665</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10578">melB</a></td><td>E</td><td>Deoxy(2)glucose-resistant utilization of melibiose</td><td><a href="#ref431">431</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10578">melB</a></td><td>E</td><td>Lithium-resistant utilization of melibiose</td><td><a href="#ref397">397</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10581">metA</a></td><td>S</td><td>Methyl(α)methionine resistance</td><td><a href="#ref127">127</a> <a href="#ref447">447</a> </td><td>FBI, S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10583">metC (ecfA?)</a></td><td>E</td><td>Neomycin resistance</td><td><a href="#ref464">464</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10583">metC (ecfA?)</a></td><td>E</td><td>Colicin K resistance</td><td><a href="#ref465">465</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10583">metC (ecfA?)</a></td><td>E</td><td>Chloro(β)alanine resistance</td><td><a href="#ref832">832</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10586">metG</a></td><td>E</td><td>Ethionine resistance</td><td><a href="#ref28">28</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10588">metJ</a></td><td>E, S</td><td>Ethionine resistance</td><td><a href="#ref447">447</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10588">metJ</a></td><td>E</td><td>Glutamyl-γ-methyl ester resistance</td><td><a href="#ref425">425</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10588">metJ</a></td><td>E, S</td><td>Norleucine resistance</td><td><a href="#ref128">128</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10588">metJ</a></td><td>E</td><td>Methionine sulfoximine + methyl(α)methionine resistance</td><td><a href="#ref386">386</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10589">metK</a></td><td>E, S</td><td>Ethionine resistance</td><td><a href="#ref275">275</a> <a href="#ref289">289</a> <a href="#ref447">447</a> <a href="#ref735">735</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10589">metK</a></td><td>E</td><td>Glutamyl-γ-methyl ester resistance</td><td><a href="#ref425">425</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10589">metK</a></td><td>E, S</td><td>Norleucine resistance</td><td><a href="#ref128">128</a> <a href="#ref447">447</a> <a href="#ref735">735</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10589">metK</a></td><td>S</td><td>Methyl(α)methionine resistance</td><td><a href="#ref447">447</a> <a href="#ref735">735</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10590">metL</a></td><td>E</td><td>Hydroxy (β) norvaline resistance</td><td><a href="#ref138">138</a> </td><td>S, FBI</td></tr>
<tr><td>metPD</td><td>E, S</td><td>Methyl(α)methionine resistance</td><td><a href="#ref37">37</a> <a href="#ref38">38</a> <a href="#ref386">386</a> </td><td>L</td></tr>
<tr><td>metPD</td><td>E, S</td><td>Methionine sulfoximine resistance</td><td><a href="#ref37">37</a> <a href="#ref38">38</a> <a href="#ref386">386</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10592">mglA</a></td><td>E</td><td>Fucose + arabinose-supported growth in <em>galP</em></td><td><a href="#ref209">209</a> <a href="#ref613">613</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10593">mglB</a></td><td>E</td><td>Fucose + arabinose-supported growth in <em>galP</em></td><td><a href="#ref209">209</a> <a href="#ref613">613</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10594">mglC</a></td><td>E</td><td>Fucose + arabinose-supported growth in <em>galP</em></td><td><a href="#ref209">209</a> <a href="#ref613">613</a> </td><td> </td></tr>
<tr><td>mglD</td><td>E</td><td>Methyl-β-galactoside-supported growth</td><td><a href="#ref645">645</a> </td><td>C</td></tr>
<tr><td>mng</td><td>E</td><td>Manganese resistance</td><td><a href="#ref708">708</a> </td><td> </td></tr>
<tr><td>mopA</td><td>E</td><td>T4 phage resistance</td><td><a href="#ref258">258</a> </td><td>S</td></tr>
<tr><td>mopB</td><td>E</td><td>T4 phage resistance</td><td><a href="#ref258">258</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10601">motAB</a></td><td>E</td><td>Chi phage resistance</td><td><a href="#ref349">349</a> <a href="#ref709">709</a> <a href="#ref710">710</a> </td><td> </td></tr>
<tr><td>mrb</td><td>E</td><td>Fosfomycin resistance</td><td><a href="#ref800">800</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10606">mrdA</a></td><td>E</td><td>Mecillinam resistance</td><td><a href="#ref759">759</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10607">mrdB</a></td><td>E</td><td>Mecillinam resistance</td><td><a href="#ref358">358</a> <a href="#ref502">502</a> <a href="#ref759">759</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10608">mreBCD</a></td><td>E</td><td>Mecillinam resistance</td><td><a href="#ref582">582</a> <a href="#ref807">807</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10615">mtlA</a></td><td>E</td><td>Arabinitol (D) resistance</td><td><a href="#ref614">614</a> </td><td>L</td></tr>
<tr><td>mtlC</td><td>E</td><td>Mannitol (limiting) as C source</td><td><a href="#ref722">722</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10617">mtr</a></td><td>E</td><td>Methyl(5)tryptophan resistance</td><td><a href="#ref305">305</a> <a href="#ref320">320</a> <a href="#ref523">523</a> </td><td>L</td></tr>
<tr><td>murZ</td><td>E</td><td>Fosfomycin resistance due to increased gene dosage</td><td><a href="#ref499">499</a> </td><td>D</td></tr>
<tr><td>mut (generic)</td><td>E</td><td>Streptomycin resistance</td><td><a href="#ref309">309</a> </td><td> </td></tr>
<tr><td>mut (generic)</td><td>E</td><td>Azaserine resistance</td><td><a href="#ref703">703</a> </td><td> </td></tr>
<tr><td>mut (generic)</td><td>E</td><td>Chemostat growth</td><td><a href="#ref152">152</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10624">mutH</a></td><td>E</td><td>Amino(2)purine resistance in <em>dam</em></td><td><a href="#ref267">267</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11281">mutL</a></td><td>E</td><td>Amino(2)purine resistance in <em>dam</em></td><td><a href="#ref267">267</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10625">mutS</a></td><td>E</td><td>Amino(2)purine resistance in <em>dam</em></td><td><a href="#ref267">267</a> </td><td>L</td></tr>
<tr><td>mvrC</td><td>E</td><td>Methyl viologen resistance</td><td><a href="#ref532">532</a> </td><td>D</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10631">nadB</a></td><td>S</td><td>Amino(6)nicotinamide resistance</td><td><a href="#ref142">142</a> <a href="#ref336">336</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=G6350">nadD</a></td><td>S</td><td>Amino(6)nicotinamide resistance</td><td><a href="#ref336">336</a> <a href="#ref337">337</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10632">nagA</a></td><td>E</td><td>Streptozotocin resistance</td><td><a href="#ref456">456</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10632">nagA</a></td><td>E</td><td>Iodoacetylglucosamine resistance</td><td><a href="#ref823">823</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10633">nagB</a></td><td>E</td><td>Iodoacetylglucosamine resistance</td><td><a href="#ref823">823</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10635">nagE</a></td><td>E</td><td>Streptozotocin resistance</td><td><a href="#ref456">456</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10635">nagE</a></td><td>E</td><td>Iodoacetylglucosamine resistance</td><td><a href="#ref823">823</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10635">nagE</a></td><td>E</td><td>Deoxy(2)-2-iodoacetamideoglucose resistance</td><td><a href="#ref379">379</a> </td><td>L</td></tr>
<tr><td>nalB</td><td>E</td><td>Nalidixic acid resistance</td><td><a href="#ref298">298</a> </td><td> </td></tr>
<tr><td>nalD</td><td>E</td><td>Nalidixic acid resistance</td><td><a href="#ref333">333</a> </td><td> </td></tr>
<tr><td>nar (general)</td><td>E</td><td>Metronidazole resistance</td><td><a href="#ref677">677</a> </td><td>L</td></tr>
<tr><td>narC</td><td>E</td><td>Chlorate resistance</td><td><a href="#ref266">266</a> <a href="#ref280">280</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10638">narG</a></td><td>E</td><td>Chlorate resistance</td><td><a href="#ref739">739</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10639">narH</a></td><td>E</td><td>Chlorate resistance</td><td><a href="#ref739">739</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10650">ndk</a></td><td>S</td><td>Aza(8)guanine resistance</td><td><a href="#ref264">264</a> <a href="#ref650">650</a> </td><td> </td></tr>
<tr><td>neaB</td><td>E, S</td><td>Neamine resistance</td><td><a href="#ref119">119</a> <a href="#ref182">182</a> <a href="#ref495">495</a> </td><td> </td></tr>
<tr><td>nek</td><td>E</td><td>Aminoglycoside resistance</td><td><a href="#ref27">27</a> <a href="#ref342">342</a> </td><td> </td></tr>
<tr><td>nfnA</td><td>E</td><td>Nitrofurantoin resistance</td><td><a href="#ref684">684</a> </td><td> </td></tr>
<tr><td>nfnB</td><td>E</td><td>Nitrofurantoin resistance</td><td><a href="#ref684">684</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11261">nfsA</a></td><td>E</td><td>Nitrofuranzone resistance</td><td><a href="#ref90">90</a> <a href="#ref505">505</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG50005">nfsB</a></td><td>E</td><td>Nitrofuranzone resistance</td><td><a href="#ref90">90</a> <a href="#ref505">505</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10652">nhaA</a></td><td>E</td><td>Lithium resistance (multicopy)</td><td><a href="#ref628">628</a> </td><td>D</td></tr>
<tr><td>nol</td><td>S</td><td>Norleucine resistance</td><td><a href="#ref324">324</a> </td><td> </td></tr>
<tr><td>nov</td><td>E</td><td>Novobiocin resistance</td><td><a href="#ref630">630</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10660">nrdA</a></td><td>E</td><td>Hydroxyurea resistance</td><td><a href="#ref609">609</a> </td><td>D</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10661">nrdB</a></td><td>E</td><td>Hydroxyurea resistance</td><td><a href="#ref609">609</a> </td><td>D</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11971">nupC</a></td><td>E</td><td>Showdomycin resistance</td><td><a href="#ref415">415</a> <a href="#ref545">545</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11971">nupC</a></td><td>E</td><td>Fluorouracil resistance</td><td><a href="#ref544">544</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11971">nupC</a></td><td>E</td><td>Fluorodeoxyuridine resistance</td><td><a href="#ref545">545</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11971">nupC</a></td><td>E</td><td>Fluoruridine resistance</td><td><a href="#ref544">544</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11971">nupC</a></td><td>E</td><td>Fluorodeoxycytidine resistance</td><td><a href="#ref544">544</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10664">nupG</a></td><td>E</td><td>Fluorouracil resistance</td><td><a href="#ref544">544</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10664">nupG</a></td><td>E</td><td>Fluorodeoxyuridine resistance</td><td><a href="#ref545">545</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10664">nupG</a></td><td>E</td><td>Fluorouridine resistance</td><td><a href="#ref544">544</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10664">nupG</a></td><td>E</td><td>Fluorodeoxycytidine resistance</td><td><a href="#ref544">544</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10665">nusA</a></td><td>E</td><td>Lambda prophage induction, survival of</td><td><a href="#ref244">244</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10666">nusB</a></td><td>E</td><td>Lambda prophage induction, survival of</td><td><a href="#ref245">245</a> <a href="#ref260">260</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10666">nusB</a></td><td>E</td><td>Lambdoid phage resistance</td><td><a href="#ref260">260</a> </td><td> </td></tr>
<tr><td>nuvA</td><td>E, S</td><td>Near-UV irradiation resistance</td><td><a href="#ref424">424</a> <a href="#ref468">468</a> <a href="#ref767">767</a> </td><td>L</td></tr>
<tr><td>nuvC</td><td>E</td><td>Near-UV irradiation resistance</td><td><a href="#ref663">663</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10669">ompA</a></td><td>E</td><td>K3 phage resistance</td><td><a href="#ref306">306</a> <a href="#ref489">489</a> <a href="#ref491">491</a> <a href="#ref493">493</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10669">ompA</a></td><td>E</td><td>T-even-like phage resistance</td><td><a href="#ref199">199</a> <a href="#ref533">533</a> <a href="#ref535">535</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10669">ompA</a></td><td>E</td><td>TuII* phage resistance</td><td><a href="#ref171">171</a> <a href="#ref314">314</a> <a href="#ref493">493</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10669">ompA</a></td><td>E</td><td>Ox2 phage resistance</td><td><a href="#ref199">199</a> <a href="#ref493">493</a> <a href="#ref642">642</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10669">ompA</a></td><td>E</td><td>Bacteriocin JF246 resistance</td><td><a href="#ref124">124</a> <a href="#ref491">491</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10669">ompA</a></td><td>S</td><td>Bacteriocin 4-59 resistance</td><td><a href="#ref741">741</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10669">ompA</a></td><td>E</td><td>Colicin K tolerance</td><td><a href="#ref491">491</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10669">ompA</a></td><td>E</td><td>Colicin L tolerance</td><td><a href="#ref491">491</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10669">ompA</a></td><td>E</td><td>Chelator resistance</td><td><a href="#ref493">493</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10669">ompA</a></td><td>E</td><td>Novobiocin resistance</td><td><a href="#ref493">493</a> </td><td> </td></tr>
<tr><td>ompB</td><td>S</td><td>PH51 phage resistance</td><td><a href="#ref372">372</a> </td><td> </td></tr>
<tr><td>ompB</td><td>S</td><td>PH105 phage resistance</td><td><a href="#ref372">372</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10670">ompC</a></td><td>E</td><td>Me1 phage resistance</td><td><a href="#ref791">791</a> <a href="#ref801">801</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10670">ompC</a></td><td>E</td><td>TuIa phage resistance</td><td><a href="#ref801">801</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10670">ompC</a></td><td>S</td><td>PH51 phage resistance</td><td><a href="#ref372">372</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10670">ompC</a></td><td>S</td><td>PH105 phage resistance</td><td><a href="#ref372">372</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10670">ompC</a></td><td>E</td><td>Host range <em>Serratia marcescens</em> phage resistance</td><td><a href="#ref702">702</a> </td><td> </td></tr>
<tr><td>ompD</td><td>S</td><td>PH51 phage resistance</td><td><a href="#ref741">741</a> </td><td>L</td></tr>
<tr><td>ompD</td><td>S</td><td>Bacteriocin 4-59 resistance</td><td><a href="#ref741">741</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10671">ompF</a></td><td>E</td><td>Beta-lactam (ampicillin, cefoxitin) resistance</td><td><a href="#ref362">362</a> <a href="#ref363">363</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10671">ompF</a></td><td>E</td><td>Chloramphenicol resistance</td><td><a href="#ref235">235</a> <a href="#ref523">523</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10671">ompF</a></td><td>E</td><td>Tetracycline resistance</td><td><a href="#ref235">235</a> <a href="#ref523">523</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10671">ompF</a></td><td>E</td><td>Fluoro(5)uracil + nucleotide resistance</td><td><a href="#ref52">52</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10671">ompF</a></td><td>E</td><td>Colicin A tolerance</td><td><a href="#ref125">125</a> <a href="#ref235">235</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10671">ompF</a></td><td>E</td><td>Cloacin DF13 resistance</td><td><a href="#ref846">846</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10671">ompF</a></td><td>E</td><td>Colicin L tolerance</td><td><a href="#ref235">235</a> <a href="#ref291">291</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10671">ompF</a></td><td>E</td><td>Colicin K tolerance</td><td><a href="#ref235">235</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10671">ompF</a></td><td>E</td><td>Colicin E2 tolerance</td><td><a href="#ref235">235</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10671">ompF</a></td><td>E</td><td>Colicin E3 tolerance</td><td><a href="#ref235">235</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10671">ompF</a></td><td>E</td><td>Me1 phage resistance</td><td><a href="#ref801">801</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10671">ompF</a></td><td>E</td><td>TuIa phage resistance</td><td><a href="#ref801">801</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10671">ompF</a></td><td>E</td><td>TuIb phage resistance</td><td><a href="#ref237">237</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10671">ompF</a></td><td>S</td><td>PH51 phage resistance</td><td><a href="#ref372">372</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10671">ompF</a></td><td>S</td><td>PH105 phage resistance</td><td><a href="#ref372">372</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10671">ompF</a></td><td>E</td><td>Copper resistance in <em>ompC</em></td><td><a href="#ref480">480</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10672">ompR</a></td><td>E</td><td>Me1 phage resistance</td><td><a href="#ref801">801</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10672">ompR</a></td><td>E</td><td>TuIa phage resistance</td><td><a href="#ref801">801</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10672">ompR</a></td><td>E</td><td>Copper resistance</td><td><a href="#ref52">52</a> <a href="#ref625">625</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10672">ompR</a></td><td>E</td><td>Beta-lactam (ampicillin, cefoxitin) resistance</td><td><a href="#ref362">362</a> <a href="#ref363">363</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10672">ompR</a></td><td>E</td><td>Chloramphenicol resistance</td><td><a href="#ref625">625</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10672">ompR</a></td><td>E</td><td>Fluoro(5)uracil + nucleotide resistance</td><td><a href="#ref52">52</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10674">oppA</a></td><td>E, S</td><td>Triornithine resistance</td><td><a href="#ref43">43</a> <a href="#ref317">317</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10674">oppA</a></td><td>E</td><td>Glycylleucine resistance</td><td><a href="#ref806">806</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10674">oppA</a></td><td>E</td><td>Alanyl-2-aminopropionate resistance</td><td><a href="#ref603">603</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10674">oppA</a></td><td>E, S</td><td>Trilysine resistance</td><td><a href="#ref317">317</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10674">oppA</a></td><td>E, S</td><td>Norleucylglycyl glycine resistance</td><td><a href="#ref317">317</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10674">oppA</a></td><td>S</td><td>Glycylglycyl histidinol phosphate ester resistance</td><td><a href="#ref317">317</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10674">oppA</a></td><td>E, S</td><td>Phaseolotoxin resistance</td><td><a href="#ref733">733</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10674">oppA</a></td><td>E</td><td>Tripeptide (toxic amino acid containing) resistance</td><td><a href="#ref604">604</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10674">oppA</a></td><td>E</td><td>Glycylglycyl-<em>N</em>-<em>S</em>-(phosphonoacetyl)-L-ornithine resistance</td><td><a href="#ref604">604</a> </td><td>L</td></tr>
<tr><td>oppE</td><td>E</td><td>Tripeptide (toxic amino acid containing) resistance</td><td><a href="#ref22">22</a> </td><td>L</td></tr>
<tr><td>optA</td><td>E</td><td>T7 phage gene 1.2 mutant resistance</td><td><a href="#ref666">666</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10679">osmB</a></td><td>E</td><td>Osmotolerance</td><td><a href="#ref384">384</a> </td><td>L</td></tr>
<tr><td>osmZ</td><td>E</td><td>Arbutin as C source</td><td><a href="#ref179">179</a> </td><td> </td></tr>
<tr><td>osmZ</td><td>E</td><td>Salicin as C source</td><td><a href="#ref179">179</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10681">oxyR</a></td><td>E, S</td><td>Peroxide resistance</td><td><a href="#ref131">131</a> </td><td>C</td></tr>
<tr><td>pab</td><td>E</td><td>Sulfonamide resistance</td><td><a href="#ref97">97</a> </td><td>C</td></tr>
<tr><td>pan</td><td>E</td><td>Salicylate resistance</td><td><a href="#ref481">481</a> </td><td>C</td></tr>
<tr><td>pan</td><td>E</td><td>Hydroxyaspartate resistance</td><td><a href="#ref701">701</a> </td><td>C</td></tr>
<tr><td>pan</td><td>E</td><td>Pentylpantothenamide resistance</td><td><a href="#ref136">136</a> </td><td>C</td></tr>
<tr><td>pdx</td><td>E</td><td>Isoniazid resistance</td><td><a href="#ref186">186</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10694">pepA</a></td><td>S</td><td>Alafosfalin resistance</td><td><a href="#ref263">263</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10694">pepA</a></td><td>E, S</td><td>Peptide (toxic, valine containing) resistance in <em>pepN</em></td><td><a href="#ref517">517</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10695">pepD</a></td><td>E, S</td><td>Peptide (toxic, valine containing) resistance in <em>pepN</em> <em>pepA</em></td><td><a href="#ref517">517</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10696">pepN</a></td><td>E, S</td><td>Albomycin resistance</td><td><a href="#ref86">86</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10696">pepN</a></td><td>E, S</td><td>Peptide (toxic, valine containing) resistance</td><td><a href="#ref517">517</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10696">pepN</a></td><td>E, S</td><td>Indicator plate</td><td><a href="#ref445">445</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10698">pepQ</a></td><td>E, S</td><td>Peptide (toxic, valine containing) resistance</td><td><a href="#ref517">517</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10699">pfkA</a></td><td>E</td><td>Streptozotocin resistance</td><td><a href="#ref456">456</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10702">pgi</a></td><td>E</td><td>U3 phage resistance</td><td><a href="#ref816">816</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10702">pgi</a></td><td>E</td><td>Fosfomycin + fructose-6-phosphate resistance</td><td><a href="#ref243">243</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG12144">pgm</a></td><td>E</td><td>U3 phage resistance</td><td><a href="#ref816">816</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10707">pheA</a></td><td>E</td><td>Fluoro(4)phenylalanine resistance</td><td><a href="#ref351">351</a> <a href="#ref564">564</a> </td><td>C, FBI</td></tr>
<tr><td>pheR</td><td>S</td><td>Fluoro(4)phenylalanine resistance</td><td><a href="#ref271">271</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10709">pheS</a></td><td>E</td><td>Fluoro(4)phenylalanine resistance</td><td><a href="#ref78">78</a> <a href="#ref218">218</a> <a href="#ref395">395</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10709">pheS</a></td><td>E</td><td>Thymineless death resistance in <em>thyA</em></td><td><a href="#ref394">394</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG30064">pheU</a></td><td>S</td><td>Fluoro(4)phenylalanine resistance</td><td><a href="#ref729">729</a> </td><td> </td></tr>
<tr><td>pho</td><td>S</td><td>Glycerol(β)-phosphate as C source in the presence of high phosphate</td><td><a href="#ref407">407</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10727">phoA</a></td><td>E</td><td>Fluoro(5)uracil + adenosine resistance in <em>upp, phoS,</em> or <em>phoT</em></td><td><a href="#ref315">315</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10727">phoA</a></td><td>E</td><td>Glycerol(β)-phosphate as C source in the presence of high phosphate</td><td><a href="#ref773">773</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10727">phoA</a></td><td>E</td><td>Glycerol(β)-phosphate resistance in <em>glpD</em></td><td><a href="#ref678">678</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10728">phoB</a></td><td>E</td><td>Tellurite resistance</td><td><a href="#ref770">770</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10728">phoB</a></td><td>E</td><td>TC45 phage resistance</td><td><a href="#ref771">771</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10728">phoB</a></td><td>E</td><td>Fluoro(5)uracil + adenosine resistance in <em>upp, phoS,</em> or <em>phoT</em></td><td><a href="#ref316">316</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10729">phoE</a></td><td>E</td><td>Acid resistance</td><td><a href="#ref659">659</a> </td><td>D</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10729">phoE</a></td><td>E</td><td>TC45 phage resistance</td><td><a href="#ref420">420</a> <a href="#ref771">771</a> </td><td>S, L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10733">phoR</a></td><td>E</td><td>Arsenate resistance</td><td><a href="#ref852">852</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10733">phoR</a></td><td>E</td><td>Glycerol(β)-phosphate as C source in the presence of high phosphate</td><td><a href="#ref204">204</a> <a href="#ref838">838</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10733">phoR</a></td><td>E</td><td>TC45 phage resistance</td><td><a href="#ref621">621</a> </td><td>S</td></tr>
<tr><td>phoS</td><td>E</td><td>Arsenate resistance</td><td><a href="#ref852">852</a> </td><td> </td></tr>
<tr><td>phoS</td><td>E</td><td>Glycerol(β)-phosphate as C source in the presence of high phosphate</td><td><a href="#ref204">204</a> <a href="#ref838">838</a> </td><td> </td></tr>
<tr><td>phoS</td><td>E</td><td>TC45 phage resistance</td><td><a href="#ref621">621</a> </td><td>S</td></tr>
<tr><td>phoT</td><td>E</td><td>Arsenate resistance</td><td><a href="#ref852">852</a> </td><td> </td></tr>
<tr><td>phoT</td><td>E</td><td>Glycerol(β)-phosphate as C source in the presence of high phosphate</td><td><a href="#ref204">204</a> <a href="#ref838">838</a> </td><td> </td></tr>
<tr><td>phoT</td><td>E</td><td>TC45 phage resistance</td><td><a href="#ref621">621</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10736">phr</a></td><td>E</td><td>T1 phage (UV irradiated) resistance</td><td><a href="#ref332">332</a> </td><td>L</td></tr>
<tr><td>phxB</td><td>E</td><td>φX174 phage resistance</td><td><a href="#ref547">547</a> </td><td>L</td></tr>
<tr><td>pit</td><td>E</td><td>Arsenate resistance</td><td><a href="#ref59">59</a> <a href="#ref727">727</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10738">pldA</a></td><td>E</td><td>Actinomycin D resistance in the presence of EDTA</td><td><a href="#ref301">301</a> <a href="#ref805">805</a> </td><td>L</td></tr>
<tr><td>pmi</td><td>S</td><td>P22 phage resistance</td><td><a href="#ref656">656</a> <a href="#ref833">833</a> </td><td>L</td></tr>
<tr><td>pmi</td><td>S</td><td>9NA phage resistance</td><td><a href="#ref656">656</a> <a href="#ref833">833</a> </td><td>L</td></tr>
<tr><td>pmrA</td><td>E, S</td><td>Polymyxin resistance</td><td><a href="#ref219">219</a> <a href="#ref297">297</a> <a href="#ref485">485</a> <a href="#ref785">785</a> <a href="#ref786">786</a> <a href="#ref787">787</a> <a href="#ref788">788</a> </td><td> </td></tr>
<tr><td>pmrA</td><td>S</td><td>Neutrophil granule protein resistance</td><td><a href="#ref219">219</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11135">pncA</a></td><td>E, S</td><td>Amino(6)nicotinamide resistance</td><td><a href="#ref234">234</a> <a href="#ref455">455</a> <a href="#ref822">822</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11135">pncA</a></td><td>E</td><td>Nicotinamide as sole N source</td><td><a href="#ref598">598</a> </td><td>Q</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10742">pncB</a></td><td>E, S</td><td>Amino(6)nicotinamide resistance</td><td><a href="#ref234">234</a> <a href="#ref455">455</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10742">pncB</a></td><td>E, S</td><td>Amino(6)nicotinate resistance</td><td><a href="#ref234">234</a> <a href="#ref455">455</a> </td><td>L</td></tr>
<tr><td>pncX</td><td>S</td><td>Amino(6)nicotinamide resistance</td><td><a href="#ref336">336</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10743">pnp</a></td><td>E</td><td>Showdomycin resistance</td><td><a href="#ref58">58</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10746">polA</a></td><td>E</td><td>Lambda γ mutant phage resistance</td><td><a href="#ref867">867</a> </td><td>L</td></tr>
<tr><td>praA,B</td><td>S</td><td>P221 phage resistance</td><td><a href="#ref576">576</a> <a href="#ref674">674</a> </td><td>L</td></tr>
<tr><td>praA,B</td><td>S</td><td>PH51 phage resistance</td><td><a href="#ref576">576</a> </td><td>L</td></tr>
<tr><td>praA,B</td><td>S</td><td>PH105 phage resistance</td><td><a href="#ref576">576</a> </td><td>L</td></tr>
<tr><td>prbA,B</td><td>S</td><td>ES18 phage resistance</td><td><a href="#ref674">674</a> </td><td> </td></tr>
<tr><td>prdB</td><td>S</td><td>PH51 phage resistance</td><td><a href="#ref674">674</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10761">prfA</a></td><td>E</td><td>Phenylgalactoside resistance of <em>lacP lacI</em>(nonsense) <em>supE</em></td><td><a href="#ref747">747</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10762">prfB</a></td><td>E</td><td>Negamycin resistance</td><td><a href="#ref145">145</a> </td><td> </td></tr>
<tr><td>prh</td><td>S</td><td>HK009 phage resistance</td><td><a href="#ref674">674</a> </td><td> </td></tr>
<tr><td>prk</td><td>S</td><td>HK068 phage resistance</td><td><a href="#ref674">674</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10767">proAB</a></td><td>S</td><td>Osmotolerance</td><td><a href="#ref155">155</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10767">proA</a></td><td>E</td><td>Nitro(4)pyridine <em>N</em>-oxide resistance</td><td><a href="#ref62">62</a> <a href="#ref222">222</a> <a href="#ref321">321</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10768">proB</a></td><td>E, S</td><td>Dehydroproline resistance</td><td><a href="#ref62">62</a> <a href="#ref155">155</a> <a href="#ref635">635</a> </td><td>FBI, S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10768">proB</a></td><td>E</td><td>Nitro(4)pyridine <em>N</em>-oxide resistance</td><td><a href="#ref62">62</a> <a href="#ref222">222</a> <a href="#ref321">321</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10768">proB</a></td><td>E, S</td><td>Azetidine carboxylate resistance</td><td><a href="#ref62">62</a> <a href="#ref155">155</a> <a href="#ref635">635</a> </td><td>FBI, S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11612">proP</a></td><td>E, S</td><td>Dehydroproline resistance in <em>putA putP</em></td><td><a href="#ref156">156</a> <a href="#ref732">732</a> </td><td>L</td></tr>
<tr><td>proU</td><td>S</td><td>Dehydroproline + azetidine carboxylate resistance at high osmolarity in <em>putA putP</em></td><td><a href="#ref156">156</a> </td><td>L</td></tr>
<tr><td>prp</td><td>E</td><td>Propionate as C source</td><td><a href="#ref400">400</a> </td><td>Q</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10776">pspABCDE</a></td><td>E</td><td>Sulfanilamide + hypoxanthine resistance</td><td><a href="#ref97">97</a> </td><td> </td></tr>
<tr><td>pss</td><td>E</td><td>Ethanol resistance</td><td><a href="#ref133">133</a> </td><td> </td></tr>
<tr><td>pss</td><td>E</td><td>Dimethyl sulfoxide resistance</td><td><a href="#ref133">133</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10782">pstABCS</a></td><td>E</td><td>Arsenate resistance</td><td><a href="#ref727">727</a> <a href="#ref838">838</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10782">pstABCS</a></td><td>E</td><td>TC45 phage resistance</td><td><a href="#ref621">621</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG20173">pta</a></td><td>S</td><td>Fluoroacetate resistance</td><td><a href="#ref96">96</a> <a href="#ref281">281</a> <a href="#ref463">463</a> <a href="#ref797">797</a> </td><td> </td></tr>
<tr><td>pts (general)</td><td>E</td><td>Glucose + gluconate-independent motility</td><td><a href="#ref30">30</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10787">ptsG</a></td><td>E</td><td>Methyl(α)-glucoside resistance</td><td><a href="#ref102">102</a> <a href="#ref104">104</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10787">ptsG</a></td><td>E</td><td>Sorbose resistance</td><td><a href="#ref715">715</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10787">ptsG</a></td><td>E, S</td><td>Deoxy(2)glucose-independent utilization of fructose</td><td><a href="#ref418">418</a> <a href="#ref510">510</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10787">ptsG</a></td><td>E, S</td><td>Deoxy(3)-3-fluoroglucose-independent utilization of fructose</td><td><a href="#ref418">418</a> <a href="#ref510">510</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10787">ptsG</a></td><td>E</td><td>Methyl(α)-glucoside-independent lactose + mannitol utilization</td><td><a href="#ref80">80</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10787">ptsG</a></td><td>E</td><td>Glucosamine as C source</td><td><a href="#ref379">379</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10787">ptsG</a></td><td>E</td><td>Mannose as C source</td><td><a href="#ref379">379</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10788">ptsHI</a></td><td>E, S</td><td>Deoxy(3)-3-fluoroglucose-independent utilization of lactate</td><td><a href="#ref510">510</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10788">ptsHI</a></td><td>E, S</td><td>Fosfomycin resistance</td><td><a href="#ref146">146</a> <a href="#ref800">800</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10788">ptsHI</a></td><td>E</td><td>Streptozotocin resistance</td><td><a href="#ref16">16</a> <a href="#ref873">873</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10789">ptsI</a></td><td>E</td><td>Ampicillin resistance</td><td><a href="#ref212">212</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10789">ptsI</a></td><td>S</td><td>Deoxy(3)-3-fluoroglucose resistance</td><td><a href="#ref510">510</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10790">purA</a></td><td>E</td><td>Methyl(6)purine + hypoxanthine resistance</td><td><a href="#ref60">60</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11314">purB</a></td><td>E</td><td>Nalidixic acid resistance</td><td><a href="#ref310">310</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10800">purR</a></td><td>E</td><td>Adenine resistance in <em>hpt gpt</em></td><td><a href="#ref461">461</a> <a href="#ref462">462</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10801">putA</a></td><td>E, S</td><td>Azetidine carboxylic acid resistance</td><td><a href="#ref511">511</a> <a href="#ref635">635</a> <a href="#ref775">775</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10801">putAP</a></td><td>S</td><td>Baikiain resistance in constitutive background</td><td><a href="#ref187">187</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10801">putA</a></td><td>S</td><td>Proline as N source with glucose as C source</td><td><a href="#ref566">566</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10802">putP</a></td><td>E, S</td><td>Dehydroproline resistance</td><td><a href="#ref193">193</a> <a href="#ref511">511</a> <a href="#ref635">635</a> <a href="#ref732">732</a> <a href="#ref775">775</a> <a href="#ref843">843</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10802">putP</a></td><td>E, S</td><td>Azetidine carboxylic acid resistance</td><td><a href="#ref193">193</a> <a href="#ref511">511</a> <a href="#ref635">635</a> <a href="#ref775">775</a> <a href="#ref843">843</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10802">putP</a></td><td>S</td><td>Lithium-resistant proline utilization as C source</td><td><a href="#ref551">551</a> </td><td>S</td></tr>
<tr><td>puvA</td><td>E</td><td>Psoralen + UV irradiation resistance</td><td><a href="#ref328">328</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10805">pyrB</a></td><td>S</td><td>Arginine-independent growth of <em>pyrH</em> in the presence of uracil</td><td><a href="#ref366">366</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10806">pyrC</a></td><td>S</td><td>Arginine-independent growth of <em>pyrH</em> in the presence of uracil</td><td><a href="#ref366">366</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10807">pyrD</a></td><td>S</td><td>Arginine-independent growth of <em>pyrH</em> in the presence of uracil</td><td><a href="#ref366">366</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10809">pyrF</a></td><td>E</td><td>Fluoro(5)orotic acid resistance</td><td><a href="#ref74">74</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11539">pyrH</a></td><td>S</td><td>Fluoro(5)orotic acid resistance</td><td><a href="#ref865">865</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11539">pyrH</a></td><td>S</td><td>Fluorouracil resistance</td><td><a href="#ref385">385</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11539">pyrH</a></td><td>S</td><td>Fluorouridine resistance</td><td><a href="#ref385">385</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11539">pyrH</a></td><td>E</td><td>Fluoro(5)uracil + 5-fluorouridine resistance in <em>udp</em></td><td><a href="#ref606">606</a> </td><td> </td></tr>
<tr><td>qmeACDE</td><td>E</td><td>Glycine tolerance</td><td><a href="#ref829">829</a> </td><td> </td></tr>
<tr><td>rap</td><td>E</td><td>Lambda phage resistance</td><td><a href="#ref279">279</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10823">recA</a></td><td>E</td><td>Lambda prophage induction deficiency during thymine deprivation</td><td><a href="#ref190">190</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10828">recF</a></td><td>E</td><td>Thymineless death resistance in <em>thyA</em></td><td><a href="#ref558">558</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10830">recJ</a></td><td>E</td><td>Thymineless death resistance in <em>thyA</em></td><td><a href="#ref558">558</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10832">recO</a></td><td>E</td><td>Thymineless death resistance in <em>thyA</em></td><td><a href="#ref558">558</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10833">recQ</a></td><td>E</td><td>Thymineless death resistance in <em>thyA</em></td><td><a href="#ref557">557</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10835">relA</a></td><td>E</td><td>UV (near) irradiation resistance</td><td><a href="#ref632">632</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10835">relA</a></td><td>E</td><td>Serine resistance in <em>relA</em></td><td><a href="#ref167">167</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10837">rep</a></td><td>E</td><td>P2 phage resistance</td><td><a href="#ref113">113</a> </td><td> </td></tr>
<tr><td>rfa (general)</td><td>E, S</td><td>U3 and FO phage resistance</td><td><a href="#ref35">35</a> <a href="#ref484">484</a> <a href="#ref486">486</a> <a href="#ref588">588</a> <a href="#ref833">833</a> </td><td> </td></tr>
<tr><td>rfa (general)</td><td>E, S</td><td>Ampicillin resistance</td><td><a href="#ref212">212</a> <a href="#ref550">550</a> <a href="#ref562">562</a> </td><td> </td></tr>
<tr><td>rfa (general)</td><td>S</td><td>Hemin satisfaction of <em>hemA</em> auxotrophy</td><td><a href="#ref364">364</a> </td><td> </td></tr>
<tr><td>rfa (general)</td><td>S</td><td>Cephalothin resistance</td><td><a href="#ref562">562</a> </td><td> </td></tr>
<tr><td>rfa (general)</td><td>S</td><td>Bacitracin resistance</td><td><a href="#ref562">562</a> </td><td> </td></tr>
<tr><td>rfaCDEF</td><td>S</td><td>FO phage resistance</td><td><a href="#ref676">676</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10838">rfaD</a></td><td>E</td><td>C21 phage resistance</td><td><a href="#ref141">141</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10838">rfaD</a></td><td>E</td><td>P1 phage resistance</td><td><a href="#ref141">141</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10839">rfaH</a></td><td>S</td><td>FO phage resistance</td><td><a href="#ref458">458</a> <a href="#ref675">675</a> </td><td> </td></tr>
<tr><td>rfaP</td><td>E</td><td>U3 phage + K3 phage coresistance</td><td><a href="#ref601">601</a> </td><td> </td></tr>
<tr><td>rfb</td><td>S</td><td>DCCD resistance in <em>rfa</em></td><td><a href="#ref833">833</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10845">rho</a></td><td>E</td><td>T4 phage resistance</td><td><a href="#ref740">740</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10845">rho</a></td><td>E</td><td>Lambda <em>sus N7 nin-5</em> resistance in P2 lysogen</td><td><a href="#ref352">352</a> <a href="#ref353">353</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10845">rho</a></td><td>E, S</td><td>Polarity suppression of <em>gal</em> leader insertion</td><td><a href="#ref169">169</a> <a href="#ref331">331</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=G7877">ridA</a></td><td>E</td><td>Rifampin + kasugamycin dependence</td><td><a href="#ref162">162</a> </td><td> </td></tr>
<tr><td>ridB</td><td>E</td><td>Rifampin dependence</td><td><a href="#ref160">160</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10866">rplC</a></td><td>E</td><td>Tiamulin resistance</td><td><a href="#ref73">73</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10867">rplD</a></td><td>E</td><td>Erythromycin resistance</td><td><a href="#ref725">725</a> <a href="#ref840">840</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10868">rplE</a></td><td>E</td><td>Thiopeptin resistance</td><td><a href="#ref467">467</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10869">rplF</a></td><td>E</td><td>Gentamicin resistance</td><td><a href="#ref101">101</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10872">rplK</a></td><td>E</td><td>Kasugamycin resistance</td><td><a href="#ref159">159</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10872">rplK</a></td><td>E</td><td>DAP starvation of ∆(<em>mal-asd</em>) in the presence of serine, methionine, and glycine</td><td><a href="#ref165">165</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10875">rplN</a></td><td>E</td><td>Kasugamycin resistance</td><td><a href="#ref159">159</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10875">rplN</a></td><td>E</td><td>Lincomycin resistance</td><td><a href="#ref343">343</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10876">rplO</a></td><td>E</td><td>Lincomycin resistance</td><td><a href="#ref343">343</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10882">rplV</a></td><td>E</td><td>Erythromycin resistance</td><td><a href="#ref840">840</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10893">rpoA</a></td><td>E</td><td>P2 <em>vir1</em> phage resistance</td><td><a href="#ref754">754</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10894">rpoB</a></td><td>E</td><td>Lambda <em>c</em>II + rifampin coresistance</td><td><a href="#ref296">296</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10894">rpoB</a></td><td>E</td><td>Streptovaricin resistance</td><td><a href="#ref863">863</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10894">rpoB</a></td><td>E</td><td>Rifampin resistance</td><td><a href="#ref216">216</a> <a href="#ref359">359</a> <a href="#ref369">369</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10894">rpoB</a></td><td>E</td><td>Streptolydigin resistance</td><td><a href="#ref357">357</a> <a href="#ref469">469</a> <a href="#ref688">688</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10894">rpoB</a></td><td>S</td><td>Fluorouracil resistance</td><td><a href="#ref368">368</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10894">rpoB</a></td><td>E</td><td>Lambda phage + 434 phage cross-resistance</td><td><a href="#ref256">256</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10894">rpoB</a></td><td>E</td><td>Lambda <em>Nmar </em>mutant phage resistance</td><td><a href="#ref261">261</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10894">rpoB</a></td><td>E</td><td>Serine resistance in <em>relA</em></td><td><a href="#ref784">784</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10894">rpoB</a></td><td>E</td><td>Rifampin resistance in <em>rpoB</em>+/<em>rpoB</em> (<em>rif</em>) merodiploids</td><td><a href="#ref36">36</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10894">rpoB</a></td><td>E</td><td>T7 phage resistance</td><td><a href="#ref690">690</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10895">rpoC</a></td><td>E</td><td>T7 phage gene 2 resistance</td><td><a href="#ref100">100</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10896">rpoD</a></td><td>E</td><td>Lambda <em>c</em>I71 resistance</td><td><a href="#ref556">556</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10896">rpoD</a></td><td>E</td><td>Arabinose (L) as C source in <em>cya</em> or <em>crp</em></td><td><a href="#ref712">712</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10897">rpoH</a></td><td>E</td><td>Temperature resistance in <em>rpoD</em>(Ts)</td><td><a href="#ref577">577</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10901">rpsB</a></td><td>E</td><td>Kasugamycin resistance</td><td><a href="#ref583">583</a> <a href="#ref861">861</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10901">rpsB</a></td><td>E</td><td>Cold resistance in <em>rpsE</em> cold-sensitive mutants</td><td><a href="#ref559">559</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10902">rpsC</a></td><td>E</td><td>Spectinomycin resistance and sucrose dependence</td><td><a href="#ref195">195</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10903">rpsD</a></td><td>E</td><td>Spectinomycin resistance and sucrose dependence</td><td><a href="#ref195">195</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10903">rpsD</a></td><td>E</td><td>Streptomycin independence of streptomycin-dependent <em>rpsL</em></td><td><a href="#ref304">304</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10904">rpsE</a></td><td>E</td><td>Spectinomycin resistance and sucrose dependence</td><td><a href="#ref195">195</a> <a href="#ref526">526</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10904">rpsE</a></td><td>E, S</td><td>Spectinomycin resistance</td><td><a href="#ref172">172</a> <a href="#ref495">495</a> <a href="#ref634">634</a> <a href="#ref672">672</a> <a href="#ref853">853</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10904">rpsE</a></td><td>E</td><td>Streptomycin independence of streptomycin-dependent <em>rpsL</em></td><td><a href="#ref304">304</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10906">rpsG</a></td><td>E</td><td>Lincomycin resistance</td><td><a href="#ref343">343</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10908">rpsI</a></td><td>E</td><td>Kasugamycin resistance and dependence</td><td><a href="#ref161">161</a> <a href="#ref163">163</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10909">rpsJ</a></td><td>E</td><td>Lambda prophage induction, resistance to</td><td><a href="#ref170">170</a> <a href="#ref246">246</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10911">rpsL</a></td><td>E, S</td><td>Streptomycin resistance</td><td><a href="#ref89">89</a> <a href="#ref495">495</a> <a href="#ref853">853</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10911">rpsL</a></td><td>E</td><td>Neamine dependence</td><td><a href="#ref798">798</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10911">rpsL</a></td><td>E</td><td>Paromomycin resistance or dependence</td><td><a href="#ref853">853</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10911">rpsL</a></td><td>E</td><td>Phenyl galactoside resistance of <em>lacP lacI</em> (nonsense) <em>supE</em></td><td><a href="#ref747">747</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10912">rpsM</a></td><td>E</td><td>Kasugamycin resistance</td><td><a href="#ref159">159</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10916">rpsQ</a></td><td>E</td><td>Neamine resistance</td><td><a href="#ref76">76</a> <a href="#ref119">119</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10917">rpsR</a></td><td>E</td><td>Kasugamycin resistance</td><td><a href="#ref159">159</a> </td><td> </td></tr>
<tr><td>rrn</td><td>E</td><td>Erythromycin resistance</td><td><a href="#ref704">704</a> </td><td>S</td></tr>
<tr><td>rrn</td><td>E</td><td>Chloramphenicol resistance</td><td><a href="#ref704">704</a> </td><td>S</td></tr>
<tr><td>rrn</td><td>E</td><td>Spectinomycin resistance</td><td><a href="#ref704">704</a> </td><td>S</td></tr>
<tr><td>sbaA</td><td>E</td><td>Serine resistance</td><td><a href="#ref166">166</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10928">sbmA</a></td><td>E</td><td>Microcin B17 resistance</td><td><a href="#ref446">446</a> </td><td> </td></tr>
<tr><td>sdh</td><td>E</td><td>Succinate-independent growth in <em>lpd</em></td><td><a href="#ref154">154</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10936">secA</a></td><td>E, S</td><td>Azide resistance</td><td><a href="#ref227">227</a> <a href="#ref232">232</a> <a href="#ref584">584</a> <a href="#ref808">808</a> <a href="#ref864">864</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10936">secA</a></td><td>E</td><td>Phenethyl alcohol resistance</td><td><a href="#ref808">808</a> <a href="#ref864">864</a> </td><td> </td></tr>
<tr><td>semA</td><td>E</td><td>Microcin E492 resistance</td><td><a href="#ref622">622</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10944">serA</a></td><td>E</td><td>Serine hydroxamate resistance</td><td><a href="#ref774">774</a> </td><td>FBI, S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10947">serS</a></td><td>E</td><td>Serine hydroxamate resistance</td><td><a href="#ref774">774</a> </td><td>S</td></tr>
<tr><td>sidCF</td><td>S</td><td>Albomycin resistance</td><td><a href="#ref87">87</a> <a href="#ref478">478</a> <a href="#ref479">479</a> <a href="#ref674">674</a> </td><td> </td></tr>
<tr><td>sidK</td><td>S</td><td>Albomycin resistance</td><td><a href="#ref87">87</a> <a href="#ref478">478</a> <a href="#ref479">479</a> <a href="#ref674">674</a> </td><td> </td></tr>
<tr><td>sidK</td><td>S</td><td>ES18 phage resistance</td><td><a href="#ref87">87</a> <a href="#ref478">478</a> <a href="#ref479">479</a> <a href="#ref674">674</a> </td><td> </td></tr>
<tr><td>sloB</td><td>E</td><td>Nalidixic acid tolerance</td><td><a href="#ref473">473</a> </td><td> </td></tr>
<tr><td>sloB</td><td>E</td><td>Amidinopenicillin tolerance</td><td><a href="#ref473">473</a> </td><td> </td></tr>
<tr><td>sorAT</td><td>E</td><td>Sorbose (L) as C source in crosses with wild strains</td><td><a href="#ref844">844</a> </td><td> </td></tr>
<tr><td>spcB</td><td>S</td><td>Spectinomycin resistance</td><td><a href="#ref853">853</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=G8210">srlA</a></td><td>E</td><td>Sorbitol + xylitol resistance</td><td><a href="#ref639">639</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10971">srlD</a></td><td>E</td><td>Fructose as C source in <em>ptsF</em> or <em>ptsM</em></td><td><a href="#ref378">378</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10977">sspA</a></td><td>E</td><td>P1 phage resistance</td><td><a href="#ref835">835</a> </td><td>L</td></tr>
<tr><td>strB</td><td>S</td><td>Streptomycin resistance</td><td><a href="#ref273">273</a> <a href="#ref853">853</a> </td><td> </td></tr>
<tr><td>strC</td><td>S</td><td>Streptomycin resistance</td><td><a href="#ref648">648</a> </td><td> </td></tr>
<tr><td>strM</td><td>E</td><td>Streptomycin resistance</td><td><a href="#ref671">671</a> </td><td> </td></tr>
<tr><td>strM</td><td>E</td><td>Phenyl galactoside resistance of <em>lacI</em>q <em>lacI</em>(nonsense) <em>lacP supL</em></td><td><a href="#ref748">748</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10984">sulA</a></td><td>E</td><td>UV irradiation resistance in <em>lon</em></td><td><a href="#ref373">373</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10984">sulA</a></td><td>E</td><td>Methyl methanesulfonate resistance in <em>lon</em></td><td><a href="#ref373">373</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10984">sulA</a></td><td>E</td><td>Nitrofurantoin resistance in <em>lon</em></td><td><a href="#ref252">252</a> </td><td>L</td></tr>
<tr><td>tabC</td><td>E</td><td>T4 phage resistance</td><td><a href="#ref258">258</a> </td><td> </td></tr>
<tr><td>tct</td><td>S</td><td>Fluorocitrate resistance</td><td><a href="#ref724">724</a> </td><td>L</td></tr>
<tr><td>tct</td><td>S</td><td>Trifluorocitrate resistance</td><td><a href="#ref723">723</a> </td><td>L</td></tr>
<tr><td>tct</td><td>S</td><td>2-Fluoro-L-erythrocitrate resistance</td><td><a href="#ref32">32</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10994">tdk</a></td><td>E, S</td><td>Azidothymidine resistance</td><td><a href="#ref207">207</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10994">tdk</a></td><td>S</td><td>Fluorodeoxyuridine + uracil resistance in <em>deoA</em></td><td><a href="#ref54">54</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10994">tdk</a></td><td>E</td><td>Fluoro(5)deoxyuridine resistance</td><td><a href="#ref749">749</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10994">tdk</a></td><td>E</td><td>Bromo(5)deoxyuridine + UV irradiation resistance</td><td><a href="#ref319">319</a> <a href="#ref348">348</a> </td><td>L</td></tr>
<tr><td>tet</td><td>E, S</td><td>Fusaric acid resistance</td><td><a href="#ref72">72</a> <a href="#ref487">487</a> </td><td>L</td></tr>
<tr><td>tet</td><td>S</td><td>Quinaldic acid resistance</td><td><a href="#ref72">72</a> </td><td>L</td></tr>
<tr><td>thdA</td><td>E</td><td>Dapsone resistance</td><td><a href="#ref383">383</a> </td><td> </td></tr>
<tr><td>thi</td><td>E</td><td>Pyrithiamine resistance</td><td><a href="#ref399">399</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10998">thrA</a></td><td>E, S</td><td>Thialysine resistance</td><td><a href="#ref365">365</a> </td><td>S, FBI</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10998">thrA</a></td><td>E</td><td>Serine resistance</td><td><a href="#ref294">294</a> </td><td>S, FBI</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG10998">thrABC</a></td><td>E</td><td>Amino(2)-hydroxy(3)-pentoate resistance</td><td><a href="#ref847">847</a> </td><td>D</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11001">thrS</a></td><td>E</td><td>Borrelidin resistance</td><td><a href="#ref248">248</a> <a href="#ref276">276</a> <a href="#ref560">560</a> <a href="#ref596">596</a> </td><td>S, Q</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11002">thyA</a></td><td>E, S</td><td>Aminopterin resistance</td><td><a href="#ref66">66</a> <a href="#ref123">123</a> <a href="#ref325">325</a> <a href="#ref730">730</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11002">thyA</a></td><td>E</td><td>Trimethoprim resistance</td><td><a href="#ref66">66</a> <a href="#ref730">730</a> </td><td>L</td></tr>
<tr><td>tinA (tlr)</td><td>E, S</td><td>Thiolutin resistance</td><td><a href="#ref381">381</a> <a href="#ref382">382</a> <a href="#ref714">714</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG12302">tmk</a></td><td>E</td><td>Dideoxy(2′,3′)thymidine resistance</td><td><a href="#ref176">176</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11005">tnaA</a></td><td>E</td><td>Tryptophan-supported growth at 13°C</td><td><a href="#ref568">568</a> </td><td>C</td></tr>
<tr><td>tna</td><td>E</td><td>Indole + methyl(5)tryptophan-supported, glucose-resistant growth in ∆<em>trp</em></td><td><a href="#ref862">862</a> </td><td> </td></tr>
<tr><td>tol</td><td>E</td><td>Bacteriocin tolerance</td><td><a href="#ref85">85</a> <a href="#ref120">120</a> <a href="#ref173">173</a> <a href="#ref236">236</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11010">tolQAB</a></td><td>E</td><td>Colicin E1 resistance</td><td><a href="#ref449">449</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11010">tolQAB</a></td><td>E</td><td>TuIa phage resistance</td><td><a href="#ref449">449</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11008">tolB</a></td><td>E</td><td>Colicin E resistance</td><td><a href="#ref19">19</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11008">tolB</a></td><td>E</td><td>Azaleucine resistance</td><td><a href="#ref19">19</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11009">tolC</a></td><td>E</td><td>Colicin E1 tolerance</td><td><a href="#ref536">536</a> <a href="#ref653">653</a> </td><td>L</td></tr>
<tr><td>tolD</td><td>E</td><td>Colicin E2 and E3 tolerance</td><td><a href="#ref210">210</a> </td><td> </td></tr>
<tr><td>tolD</td><td>E</td><td>Ampicillin resistance</td><td><a href="#ref105">105</a> <a href="#ref210">210</a> </td><td> </td></tr>
<tr><td>tolE</td><td>E</td><td>Colicin E2 and E3 tolerance</td><td><a href="#ref210">210</a> <a href="#ref211">211</a> </td><td> </td></tr>
<tr><td>tolE</td><td>E</td><td>Ampicillin resistance</td><td><a href="#ref210">210</a> <a href="#ref211">211</a> </td><td> </td></tr>
<tr><td>tolI</td><td>E</td><td>Colicin Ia and Ib tolerance</td><td><a href="#ref120">120</a> </td><td> </td></tr>
<tr><td>tolJ</td><td>E</td><td>Colicin L, A, and S4 tolerance</td><td><a href="#ref173">173</a> </td><td> </td></tr>
<tr><td>tolM</td><td>E</td><td>Colicin M tolerance</td><td><a href="#ref302">302</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11010">tolQ</a></td><td>E</td><td>Colicin group A tolerance</td><td><a href="#ref84a">84a</a> <a href="#ref750">750</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11010">tolQ</a></td><td>E</td><td>Filamentous phage tolerance</td><td><a href="#ref84a">84a</a> <a href="#ref750">750</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11011">tolR</a></td><td>E</td><td>Colicin group A tolerance</td><td><a href="#ref84a">84a</a> <a href="#ref750">750</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11011">tolR</a></td><td>E</td><td>Filamentous phage tolerance</td><td><a href="#ref84a">84a</a> <a href="#ref750">750</a> <a href="#ref751">751</a> </td><td> </td></tr>
<tr><td>tolZ</td><td>E</td><td>Colicin E2, E3, D, 1a, and 1b tolerance</td><td><a href="#ref503">503</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11012">tonB</a></td><td>S</td><td>ES18 phage resistance</td><td><a href="#ref149">149</a> <a href="#ref741">741</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11012">tonB</a></td><td>E</td><td>T1 phage resistance</td><td><a href="#ref46">46</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11012">tonB</a></td><td>E</td><td>φ80 phage resistance</td><td><a href="#ref46">46</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11012">tonB</a></td><td>E</td><td>Colicin B resistance</td><td><a href="#ref46">46</a> <a href="#ref623">623</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11012">tonB</a></td><td>E, S</td><td>Albomycin resistance</td><td><a href="#ref85">85</a> <a href="#ref505">505</a> <a href="#ref741">741</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11012">tonB</a></td><td>E</td><td>Colicin M resistance</td><td><a href="#ref85">85</a> <a href="#ref523">523</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11012">tonB</a></td><td>S</td><td>Bacteriocin 4-59 resistance</td><td><a href="#ref741">741</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11012">tonB</a></td><td>E</td><td>Cephalosporin E-0702 resistance</td><td><a href="#ref815">815</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=G0-10473">topA</a></td><td>S</td><td>Kanamycin resistance</td><td><a href="#ref202">202</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=G0-10473">topA</a></td><td>S</td><td>Neomycin resistance</td><td><a href="#ref202">202</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=G0-10473">topA</a></td><td>S</td><td>Streptomycin resistance</td><td><a href="#ref202">202</a> </td><td> </td></tr>
<tr><td>tpp</td><td>E</td><td>Alafosfalin resistance</td><td><a href="#ref603">603</a> </td><td>L</td></tr>
<tr><td>tpp</td><td>E</td><td>Triornithine resistance in <em>opp</em></td><td><a href="#ref44">44</a> </td><td>L</td></tr>
<tr><td>tppA</td><td>S</td><td>Tripeptide (toxic) resistance</td><td><a href="#ref263">263</a> </td><td> </td></tr>
<tr><td>tppB</td><td>S</td><td>Tripeptide (toxic) resistance</td><td><a href="#ref263">263</a> </td><td> </td></tr>
<tr><td>trmE?</td><td>E</td><td>Phenyl galactoside resistance of <em>lacl</em>q <em>lacI</em>(nonsense) <em>lacP supL</em></td><td><a href="#ref748">748</a> </td><td> </td></tr>
<tr><td>trp (generic)</td><td>E, S</td><td>Fluoro(6)tryptophan resistance</td><td><a href="#ref42">42</a> <a href="#ref318">318</a> </td><td>C</td></tr>
<tr><td>trp (generic)</td><td>E, S</td><td>Methyl(5)tryptophan resistance</td><td><a href="#ref42">42</a> <a href="#ref318">318</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11028">trpE</a></td><td>E, S</td><td>Methyl(5)tryptophan resistance</td><td><a href="#ref114">114</a> <a href="#ref539">539</a> </td><td>S, FBI</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11028">trpE</a></td><td>E</td><td>Methyl(3)anthranilate resistance</td><td><a href="#ref308">308</a> </td><td>S, FBI</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11029">trpR</a></td><td>E, S</td><td>Methyl(5)tryptophan resistance</td><td><a href="#ref42">42</a> <a href="#ref137">137</a> <a href="#ref523">523</a> <a href="#ref746">746</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11029">trpR</a></td><td>E</td><td>Chuangxinmycin resistance</td><td><a href="#ref850">850</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11030">trpS</a></td><td>E</td><td>Indolmycin resistance</td><td><a href="#ref75">75</a> </td><td>D</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11031">trxA</a></td><td>E</td><td>T7 phage resistance</td><td><a href="#ref498">498</a> </td><td>L</td></tr>
<tr><td>trzA</td><td>S</td><td>Triazole resistance</td><td><a href="#ref339">339</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11035">tsx</a></td><td>E</td><td>T6 phage resistance</td><td><a href="#ref99">99</a> <a href="#ref299">299</a> <a href="#ref490">490</a> <a href="#ref492">492</a> <a href="#ref620">620</a> </td><td>L, S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11035">tsx</a></td><td>E</td><td>Colicin K resistance</td><td><a href="#ref99">99</a> <a href="#ref299">299</a> <a href="#ref490">490</a> <a href="#ref492">492</a> <a href="#ref620">620</a> </td><td>L, S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11035">tsx</a></td><td>E</td><td>Albicidin resistance</td><td><a href="#ref69">69</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11036">tufAB</a></td><td>E</td><td>Kirromycin resistance</td><td><a href="#ref220">220</a> <a href="#ref356">356</a> <a href="#ref763">763</a> <a href="#ref795">795</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11036">tufAB</a></td><td>E, S</td><td>Mocimycin resistance</td><td><a href="#ref335">335</a> <a href="#ref794">794</a> </td><td> </td></tr>
<tr><td>tyn</td><td>S</td><td>Tyramine as N source</td><td><a href="#ref549">549</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11039">tyrA</a></td><td>E, S</td><td>Fluorophenylalanine resistance</td><td><a href="#ref270">270</a> <a href="#ref729">729</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11039">tyrA</a></td><td>E</td><td>Amino(4)phenylalanine resistance</td><td><a href="#ref504">504</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11042">tyrR</a></td><td>E, S</td><td>Amino(4)phenylalanine resistance</td><td><a href="#ref95">95</a> <a href="#ref504">504</a> <a href="#ref811">811</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11042">tyrR</a></td><td>E</td><td>Fluorotyrosine resistance</td><td><a href="#ref130">130</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11042">tyrR</a></td><td>S</td><td>Fluoro(4)phenylalanine resistance</td><td><a href="#ref729">729</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11043">tyrS</a></td><td>E</td><td>Fluorotyrosine resistance</td><td><a href="#ref662">662</a> </td><td>S</td></tr>
<tr><td>ubi (generic)</td><td>E</td><td>Neomycin resistance</td><td><a href="#ref265">265</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=G6365">ubiF</a></td><td>S</td><td>Fluoro(5)uracil + carbamylaspartate resistance</td><td><a href="#ref404">404</a> <a href="#ref866">866</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=G6365">ubiF</a></td><td>E</td><td>Gentamicin resistance</td><td><a href="#ref542">542</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=G6365">ubiF</a></td><td>E</td><td>Streptomycin resistance</td><td><a href="#ref542">542</a> </td><td> </td></tr>
<tr><td>udhA</td><td>S</td><td>Fluoro(5)uridine resistance</td><td><a href="#ref834">834</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11701">udk</a></td><td>E</td><td>Fluoro(5)uridine + uracil resistance</td><td><a href="#ref380">380</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11701">udk</a></td><td>E</td><td>Fluoro(5)uridine resistance in <em>upp</em></td><td><a href="#ref565">565</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11045">udp</a></td><td>E</td><td>Fluoro(5)uracil + adenosine resistance</td><td><a href="#ref606">606</a> <a href="#ref617">617</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11046">ugpAB</a></td><td>E</td><td>Fluorohydroxyacetone phosphate resistance</td><td><a href="#ref515">515</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11046">ugpAB</a></td><td>E</td><td>Dihydroxybutylphosphonate resistance in <em>glpT</em></td><td><a href="#ref691">691</a> <a href="#ref692">692</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11046">ugpAB</a></td><td>E</td><td>Glycerol-3-phosphate as C source in <em>glpT</em></td><td><a href="#ref691">691</a> </td><td>C</td></tr>
<tr><td>uhpR</td><td>E</td><td>Glucosamine 6-phosphate as C source</td><td><a href="#ref191">191</a> </td><td>C</td></tr>
<tr><td>uhpR</td><td>S</td><td>Glucose 1-phosphate as C source</td><td><a href="#ref192">192</a> </td><td>C</td></tr>
<tr><td>uhpR</td><td>E</td><td>Fructose 1-phosphate as C source</td><td><a href="#ref224">224</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11054">uhpT</a></td><td>E</td><td>Chloro-3-hydroxyacetone resistance in <em>uhp</em>(Con)</td><td><a href="#ref515">515</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11054">uhpT</a></td><td>E</td><td>Fluoro-3-hydroxyacetone resistance in <em>uhp</em>(Con)</td><td><a href="#ref515">515</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11054">uhpT</a></td><td>E</td><td>Dihydroxybutylphosphonate resistance in <em>uhp</em>(Con)</td><td><a href="#ref285">285</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11054">uhpT</a></td><td>E</td><td>Hydroxybutylphosphonate resistance in <em>ubp</em>(Con)</td><td><a href="#ref285">285</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11054">uhpT</a></td><td>E</td><td>Deoxydihydroxyphosphonyl methyl fructose resistance</td><td><a href="#ref285">285</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11054">uhpT</a></td><td>E</td><td>L-Glyceraldehyde-3-phosphate resistance in <em>uhp</em>(Con)</td><td><a href="#ref285">285</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11054">uhpT</a></td><td>E</td><td>Fosfomycin resistance</td><td><a href="#ref230">230</a> <a href="#ref389">389</a> <a href="#ref800">800</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11054">uhpT</a></td><td>E</td><td>Deoxy(2)glucose-6-phosphate resistance</td><td><a href="#ref214">214</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11055">uidA</a></td><td>E</td><td>Galacturonide as C source</td><td><a href="#ref742">742</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11055">uidA</a></td><td>E</td><td>Methylgalacturonide as C source</td><td><a href="#ref573">573</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11055">uidA</a></td><td>E</td><td>Methylglucuronide + glycerol-supported growth in <em>eda</em></td><td><a href="#ref572">572</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=G6867">uidR</a></td><td>E</td><td>Galacturonide as C source</td><td><a href="#ref742">742</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=G6867">uidR</a></td><td>E</td><td>Methylgalacturonide as C source</td><td><a href="#ref573">573</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11058">ung</a></td><td>E</td><td>Bromodeoxyuridine + UV light resistance</td><td><a href="#ref856">856</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11058">ung</a></td><td>E</td><td>T4 phage (uracil containing) resistance</td><td><a href="#ref203">203</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11332">upp</a></td><td>E, S</td><td>Fluoro(5)uracil resistance</td><td><a href="#ref53">53</a> <a href="#ref605">605</a> <a href="#ref617">617</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11332">upp</a></td><td>E</td><td>Aza(6)uracil resistance</td><td><a href="#ref605">605</a> <a href="#ref617">617</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11332">upp</a></td><td>E</td><td>Canavanine + aza(6)uracil resistance</td><td><a href="#ref605">605</a> <a href="#ref617">617</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11060">ushA</a></td><td>E</td><td>Fluoro(5)uracil + 5′-AMP resistance in <em>upp</em></td><td><a href="#ref52">52</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11061">uvrA</a></td><td>E</td><td>UV-irradiated lytic phage resistance</td><td><a href="#ref332">332</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11062">uvrB</a></td><td>E</td><td>UV-irradiated lytic phage resistance</td><td><a href="#ref332">332</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11063">uvrC</a></td><td>E</td><td>UV-irradiated lytic phage resistance</td><td><a href="#ref332">332</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11064">uvrD</a></td><td>E</td><td>UV-irradiated lytic phage resistance</td><td><a href="#ref332">332</a> </td><td> </td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11065">uxaBC</a></td><td>E</td><td>Hexuronate resistance in <em>eda</em></td><td><a href="#ref742">742</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11066">uxuAB</a></td><td>E</td><td>Hexuronate resistance in <em>eda</em></td><td><a href="#ref742">742</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG20249">uxuR</a></td><td>E</td><td>Methylgalacturonide as C source</td><td><a href="#ref574">574</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11067">valS</a></td><td>E</td><td>Thymineless death resistance in <em>thyA</em></td><td><a href="#ref394">394</a> </td><td>S</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=G85">xapA</a></td><td>E</td><td>Fluoro(5)uracil + adenosine resistance in <em>upp deoD xapR(Con)</em></td><td><a href="#ref111">111</a> </td><td>L</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11146">xapR</a></td><td>E</td><td>Adenosine as C source in <em>upp deoD</em></td><td><a href="#ref111">111</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11146">xapR</a></td><td>E</td><td>Inosine as C source in <em>upp deoD</em></td><td><a href="#ref111">111</a> </td><td>C</td></tr>
<tr><td><a class="gene" href="http://ecocyc.org/gene?orgid=ECOLI&id=EG11076">xylE</a></td><td>E</td><td>Xylose resistance in <em>fda</em></td><td><a href="#ref175">175</a> </td><td>L</td></tr>
</table>
<p><span id="alterations" class="anchor"></span>* Alterations:<br> S, structure;<br> FBI, site of feedback inhibition;<br> D, increased gene dosage;<br> C, constitutive gene expression;<br> Q, altered amount of gene product;<br> G, gain of function;<br> L, loss of function.</p>
</div>

<div id="references"><table>
<caption><span class="anchor" id="references-table"></span><h1>References</h1></caption>
<col class="refnumber">
<col class="reftitle">
<col class="reflink">
<col class="refpdf">
<tr><th>Nº</th><th>title</th><th>link</th><th>pdf</th></tr>
<tr><td>1</td><td><a id="ref1" class="anchor"></a>Abouhamad, W. N., M. Manson, M. M. Gibson, and C. F. Higgins. <i>1991</i>. Peptide transport and chemotaxis in <i>Escherichia coli</i> and <i>Salmonella</i> <i>typhimurium:</i> characterization of the dipeptide permease <i>dpp</i> and the dipeptide-binding protein.  <span class="journal">Mol. Microbiol. 5:1035–1048.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Peptide%20transport%20and%20chemotaxis%20in%20Escherichia%20coli%20and%20Salmonella%20typhimurium:%20characterization%20of%20the%20dipeptide%20permease%20dpp%20and%20the%20dipeptide-binding%20protein">google scholar</a></td><td><a href="pdf/1.pdf">pdf</a></td></tr>
<tr><td>1a</td><td><a id="ref1a" class="anchor"></a>Ackerman, R. S., N. R. Cozzarelli, and W. Epstein. <i>1974</i>. Accumulation of toxic concentrations of methylglyoxal by <i>Escherichia coli</i> K-12.  <span class="journal">J. Bacteriol. 119:357–362.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Accumulation%20of%20toxic%20concentrations%20of%20methylglyoxal%20by%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/1a.pdf">pdf</a></td></tr>
<tr><td>2</td><td><a id="ref2" class="anchor"></a>Adhya, S., P. Cleary, and A. Campbell. <i>1968</i>. A deletion analysis of prophage lambda and adjacent genetic regions.  <span class="journal">Proc. Natl. Acad. Sci. USA 61:956–962.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20deletion%20analysis%20of%20prophage%20lambda%20and%20adjacent%20genetic%20regions">google scholar</a></td><td><a href="pdf/2.pdf">pdf</a></td></tr>
<tr><td>3</td><td><a id="ref3" class="anchor"></a>Ahmad, S. I., and R. H. Pritchard. <i>1969</i>. A map of four genes specifying enzymes involved in catabolism of nucleosides and deoxynucleosides in <i>Escherichia coli.</i>  <span class="journal">Mol. Gen. Genet. 104:351–359.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20map%20of%20four%20genes%20specifying%20enzymes%20involved%20in%20catabolism%20of%20nucleosides%20and%20deoxynucleosides%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/3.pdf">pdf</a></td></tr>
<tr><td>4</td><td><a id="ref4" class="anchor"></a>Ahmad, S. I., and R. H. Pritchard. <i>1972</i>. Location of the gene specifying cytosine deaminase in <i>Escherichia coli.</i>  <span class="journal">Mol. Gen. Genet. 118:323–325.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Location%20of%20the%20gene%20specifying%20cytosine%20deaminase%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/4.pdf">pdf</a></td></tr>
<tr><td>5</td><td><a id="ref5" class="anchor"></a>Alaeddinoglu, N. C., and H. P. Charles. <i>1979</i>. Transfer of a gene for sucrose utilization into <i>Escherichia coli</i> K12, and consequent failure of expression of genes for D-serine utilization.  <span class="journal">J. Gen. Microbiol. 110:47–59.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Transfer%20of%20a%20gene%20for%20sucrose%20utilization%20into%20Escherichia%20coli%20K12,%20and%20consequent%20failure%20of%20expression%20of%20genes%20for%20D-serine%20utilization">google scholar</a></td><td><a href="pdf/5.pdf">pdf</a></td></tr>
<tr><td>6</td><td><a id="ref6" class="anchor"></a>Alderman, E. M., S. S. Dills, T. Melton, and W. J. Dobrogosz. <i>1979</i>. Cyclic AMP regulation of the bacteriophage T6 colicin K receptor in <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 140:369–376.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Cyclic%20AMP%20regulation%20of%20the%20bacteriophage%20T6%20colicin%20K%20receptor%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/6.pdf">pdf</a></td></tr>
<tr><td>7</td><td><a id="ref7" class="anchor"></a>Alexander, R. R., J. M. Calvo, and M. Freundllch. <i>1971</i>. Mutants of <i>Salmonella typhimurium</i> with an altered leucyl-transfer ribonucleic acid synthetase.  <span class="journal">J. Bacteriol. 106:213–220.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutants%20of%20Salmonella%20typhimurium%20with%20an%20altered%20leucyl-transfer%20ribonucleic%20acid%20synthetase">google scholar</a></td><td><a href="pdf/7.pdf">pdf</a></td></tr>
<tr><td>8</td><td><a id="ref8" class="anchor"></a>Alper, M. D., and B. N. Ames. <i>1975</i>. Cyclic 3′,5′-adenosine monophosphate phosphodiesterase mutants of <i>Salmonella typhimurium.</i>  <span class="journal">J. Bacteriol. 122:1081–1090.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Cyclic%203′,5′-adenosine%20monophosphate%20phosphodiesterase%20mutants%20of%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/8.pdf">pdf</a></td></tr>
<tr><td>9</td><td><a id="ref9" class="anchor"></a>Alper, M. D., and B. N. Ames. <i>1975</i>. Positive selection of mutants with deletions of the <i>gal-chl</i> region of the <i>Salmonella</i> chromosome as a screening procedure for mutagens that cause deletions.  <span class="journal">J. Bacteriol. 121:259–266.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Positive%20selection%20of%20mutants%20with%20deletions%20of%20the%20gal-chl%20region%20of%20the%20Salmonella%20chromosome%20as%20a%20screening%20procedure%20for%20mutagens%20that%20cause%20deletions">google scholar</a></td><td><a href="pdf/9.pdf">pdf</a></td></tr>
<tr><td>10</td><td><a id="ref10" class="anchor"></a>Alper, M. D., and B. N. Ames. <i>1978</i>. Transport of antibiotics and metabolite analogs by systems under cyclic AMP control: positive selection of <i>Salmonella typhimurium cya</i> and <i>crp</i> mutants.  <span class="journal">J. Bacteriol. 133:149–157.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Transport%20of%20antibiotics%20and%20metabolite%20analogs%20by%20systems%20under%20cyclic%20AMP%20control:%20positive%20selection%20of%20Salmonella%20typhimurium%20cya%20and%20crp%20mutants">google scholar</a></td><td><a href="pdf/10.pdf">pdf</a></td></tr>
<tr><td>11</td><td><a id="ref11" class="anchor"></a>Amaral, D., and H. L. Kornberg. <i>1975</i>. Regulation of fructose uptake by glucose in <i>Escherichia</i> <i>coli.</i>  <span class="journal">J. Gen. Microbiol. 90:157–168.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation%20of%20fructose%20uptake%20by%20glucose%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/11.pdf">pdf</a></td></tr>
<tr><td>12</td><td><a id="ref12" class="anchor"></a>Ames, G. F.-L. <i>1964</i>. Uptake of amino acids by <i>Salmonella typhimurium.</i>  <span class="journal">Arch. Biochem. Biophys. 104:1–18.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Uptake%20of%20amino%20acids%20by%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/12.pdf">pdf</a></td></tr>
<tr><td>13</td><td><a id="ref13" class="anchor"></a>Ames, G. F.-L., D. P. Biek, and E. N. Spudich. <i>1978</i>. Duplications of histidine transport genes in <i>Salmonella typhimurium</i> and their use for the selection of deletion mutants.  <span class="journal">J. Bacteriol. 136:1094– 1108.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Duplications%20of%20histidine%20transport%20genes%20in%20Salmonella%20typhimurium%20and%20their%20use%20for%20the%20selection%20of%20deletion%20mutants">google scholar</a></td><td><a href="pdf/13.pdf">pdf</a></td></tr>
<tr><td>14</td><td><a id="ref14" class="anchor"></a>Ames, G. F.-L., and J. Lever. <i>1970</i>. Components of histidine transport: histidine-binding proteins and <i>hisP</i> protein.  <span class="journal">Proc. Natl. Acad. Sci. USA 66:1096–1103.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Components%20of%20histidine%20transport:%20histidine-binding%20proteins%20and%20hisP%20protein">google scholar</a></td><td><a href="pdf/14.pdf">pdf</a></td></tr>
<tr><td>15</td><td><a id="ref15" class="anchor"></a>Ames, G. F.-L., K. D. Noel, H. Tabor, E. N. Spudich, K. Nikaido, J. Afong, and F. Ardeshir. <i>1977</i>. Fine-structure map of the histidine transport genes in <i>Salmonella typhimurium.</i>  <span class="journal">J. Bacteriol. 129:1289–1297.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Fine-structure%20map%20of%20the%20histidine%20transport%20genes%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/15.pdf">pdf</a></td></tr>
<tr><td>16</td><td><a id="ref16" class="anchor"></a>Ammer, J., M. Brennenstuhl, P. Schindler, J.-V. Holtje, and H. Zahner. <i>1979</i>. Phosphorylation of streptozotocin during uptake via the phosphoenolpyruvate:sugar phosphotransferase system in <i>Escherichia coli.</i>  <span class="journal">Antimicrob. Agents Chemother. 16:801–807.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Phosphorylation%20of%20streptozotocin%20during%20uptake%20via%20the%20phosphoenolpyruvate:sugar%20phosphotransferase%20system%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/16.pdf">pdf</a></td></tr>
<tr><td>17</td><td><a id="ref17" class="anchor"></a>Anderson, J. J., and D. L. Oxender. <i>1977</i>. <i>Escherichia coli</i> transport mutants lacking binding protein and other components of the branched-chain amino acid transport systems.  <span class="journal">J. Bacteriol. 130:384–392.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Escherichia%20coli%20transport%20mutants%20lacking%20binding%20protein%20and%20other%20components%20of%20the%20branched-chain%20amino%20acid%20transport%20systems">google scholar</a></td><td><a href="pdf/17.pdf">pdf</a></td></tr>
<tr><td>18</td><td><a id="ref18" class="anchor"></a>Anderson, J. J., S. C. Quay, and D. L. Oxender. <i>1976</i>. Mapping of two loci affecting the regulation of branched-chain amino acid transport in <i>Escherichia coli</i> K-12.  <span class="journal">J. Bacteriol. 126:80–90.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mapping%20of%20two%20loci%20affecting%20the%20regulation%20of%20branched-chain%20amino%20acid%20transport%20in%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/18.pdf">pdf</a></td></tr>
<tr><td>19</td><td><a id="ref19" class="anchor"></a>Anderson, J. J., J. M. Wilson, and D. L. Oxender. <i>1979</i>. Defective transport and other phenotypes of a periplasmic “leaky”  mutant of <i>Escherichia coli</i> K-12.  <span class="journal">J. Bacteriol. 140:351–358.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Defective%20transport%20and%20other%20phenotypes%20of%20a%20periplasmic%20“leaky”%20%20mutant%20of%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/19.pdf">pdf</a></td></tr>
<tr><td>20</td><td><a id="ref20" class="anchor"></a>Anderson, R. P., and J. R. Roth. <i>1977</i>. Tandem genetic duplications in phage and bacteria.  <span class="journal">Annu. Rev. Microbiol. 31:473–505.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Tandem%20genetic%20duplications%20in%20phage%20and%20bacteria">google scholar</a></td><td><a href="pdf/20.pdf">pdf</a></td></tr>
<tr><td>21</td><td><a id="ref21" class="anchor"></a>Anderson, R. P., and J. R. Roth. <i>1978</i>. Tandem genetic duplications in <i>Salmonella typhimurium:</i> amplification of the histidine operon.  <span class="journal">J. Mol. Biol. 126:53–71.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Tandem%20genetic%20duplications%20in%20Salmonella%20typhimurium:%20amplification%20of%20the%20histidine%20operon">google scholar</a></td><td><a href="pdf/21.pdf">pdf</a></td></tr>
<tr><td>22</td><td><a id="ref22" class="anchor"></a>Andrews, J. C., and S. A. Short. <i>1985</i>. Genetic analysis of <i>Escherichia coli</i> oligopeptide transport mutants.  <span class="journal">J. Bacteriol. 161:484–492.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20analysis%20of%20Escherichia%20coli%20oligopeptide%20transport%20mutants">google scholar</a></td><td><a href="pdf/22.pdf">pdf</a></td></tr>
<tr><td>23</td><td><a id="ref23" class="anchor"></a>Anton, D. <i>1968</i>. Histidine regulatory mutants in <i>Salmonella typhimurium.</i> V. Two new classes of histidine regulatory mutants.  <span class="journal">J. Mol. Biol. 33:533–546.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Histidine%20regulatory%20mutants%20in%20Salmonella%20typhimurium.%20V.%20Two%20new%20classes%20of%20histidine%20regulatory%20mutants">google scholar</a></td><td><a href="pdf/23.pdf">pdf</a></td></tr>
<tr><td>24</td><td><a id="ref24" class="anchor"></a>Anton, D. N. <i>1981</i>. <i>envB</i> mutations confer UV sensitivity to <i>Salmonella typhimurium</i> and UV resistance to <i>Escherichia coli.</i>  <span class="journal">Mol. Gen. Genet. 181:150–152.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=envB%20mutations%20confer%20UV%20sensitivity%20to%20Salmonella%20typhimurium%20and%20UV%20resistance%20to%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/24.pdf">pdf</a></td></tr>
<tr><td>25</td><td><a id="ref25" class="anchor"></a>Aono, R., M. Yamasaki, and G. Tamura. <i>1979</i>. High and selective resistance to mecillinam in adenylate cyclase-deficient or cyclic adenosine 3′,5′-monophosphate receptor protein-deficient mutants of <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 137:839–845.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=High%20and%20selective%20resistance%20to%20mecillinam%20in%20adenylate%20cyclase-deficient%20or%20cyclic%20adenosine%203′,5′-monophosphate%20receptor%20protein-deficient%20mutants%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/25.pdf">pdf</a></td></tr>
<tr><td>26</td><td><a id="ref26" class="anchor"></a>Apirion, D. <i>1967</i>. Three genes that affect <i>Escherichia coli</i> ribosomes.  <span class="journal">J. Mol. Biol. 30:255–275.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Three%20genes%20that%20affect%20Escherichia%20coli%20ribosomes">google scholar</a></td><td><a href="pdf/26.pdf">pdf</a></td></tr>
<tr><td>27</td><td><a id="ref27" class="anchor"></a>Apirion, D., and D. Schlessinger. <i>1968</i>. Coresistance to neomycin and kanamycin by mutations in an <i>Escherichia coli</i> locus that affects ribosomes.  <span class="journal">J. Bacteriol. 96:768–776.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Coresistance%20to%20neomycin%20and%20kanamycin%20by%20mutations%20in%20an%20Escherichia%20coli%20locus%20that%20affects%20ribosomes">google scholar</a></td><td><a href="pdf/27.pdf">pdf</a></td></tr>
<tr><td>28</td><td><a id="ref28" class="anchor"></a>Archibold, E. R., and L. S. Williams. <i>1973</i>. Regulation of methionyl-transfer ribonucleic acid synthetase formation in <i>Escherichia coli</i> and <i>Salmonella typhimurium.</i>  <span class="journal">J. Bacteriol. 114:1007–1013.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation%20of%20methionyl-transfer%20ribonucleic%20acid%20synthetase%20formation%20in%20Escherichia%20coli%20and%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/28.pdf">pdf</a></td></tr>
<tr><td>29</td><td><a id="ref29" class="anchor"></a>Argast, M., D. Ludtke, T. J. Silhavy, and W. Boos. <i>1978</i>. A second transport system for <i>sn</i>- glycerol-3-phosphate in <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 136:1070–1083.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20second%20transport%20system%20for%20sn-%20glycerol-3-phosphate%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/29.pdf">pdf</a></td></tr>
<tr><td>30</td><td><a id="ref30" class="anchor"></a>Armstrong, G. D., and H. Yamazaki. <i>1977</i>. Isolation and characterization of catabolite repression resistant mutants of <i>Escherichia coli.</i>  <span class="journal">Can. J. Microbiol. 23:1384–1393.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20and%20characterization%20of%20catabolite%20repression%20resistant%20mutants%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/30.pdf">pdf</a></td></tr>
<tr><td>31</td><td><a id="ref31" class="anchor"></a>Artman, M., and S. Werthamer. <i>1979</i>. Use of streptomycin and cyclic adenosine 5′- monophosphate in the isolation of mutants deficient in CAP protein.  <span class="journal">J. Bacteriol. 120:542–544.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Use%20of%20streptomycin%20and%20cyclic%20adenosine%205′-%20monophosphate%20in%20the%20isolation%20of%20mutants%20deficient%20in%20CAP%20protein">google scholar</a></td><td><a href="pdf/31.pdf">pdf</a></td></tr>
<tr><td>32</td><td><a id="ref32" class="anchor"></a>Ashton, D. M., G. D. Sweet, J. M. Somers, and W. W. Kay. <i>1980</i>. Citrate transport in <i>Salmonella typhimurium:</i> studies with 2-fluoro-L-<i>erythro</i>-citrate as a substrate.  <span class="journal">Can. J. Biochem. 58:797–803.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Citrate%20transport%20in%20Salmonella%20typhimurium:%20studies%20with%202-fluoro-L-erythro-citrate%20as%20a%20substrate">google scholar</a></td><td><a href="pdf/32.pdf">pdf</a></td></tr>
<tr><td>33</td><td><a id="ref33" class="anchor"></a>Astvatsaturyants, G. V., A. F. Lysenkov, Y. V. Smirnov, R. and R. S. Shakulov. <i>1988</i>. Mutants of <i>Escherichia coli</i> with impaired feedback inhibition of histidine biosynthesis.  <span class="journal">Genetika 24:1928– 1934.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutants%20of%20Escherichia%20coli%20with%20impaired%20feedback%20inhibition%20of%20histidine%20biosynthesis">google scholar</a></td><td></td></tr>
<tr><td>34</td><td><a id="ref34" class="anchor"></a>Aswad, D., and D. E. Koshland, Jr. <i>1975</i>. Isolation, characterization and complementation of <i>Salmonella typhimurium</i> chemotaxis mutants.  <span class="journal">J. Mol. Biol. 97:225–235.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation,%20characterization%20and%20complementation%20of%20Salmonella%20typhimurium%20chemotaxis%20mutants">google scholar</a></td><td><a href="pdf/34.pdf">pdf</a></td></tr>
<tr><td>35</td><td><a id="ref35" class="anchor"></a>Austin, E. A., J. F. Graves, L. A. Hite, C. T. Parker, and C. A. Schnaitman. <i>1990</i>. Genetic analysis of lipopolysaccharide core biosynthesis by <i>Escherichia coli</i> K-12 insertion mutagenesis of the <i>rfa</i> locus.  <span class="journal">J. Bacteriol. 172:5312–5325.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20analysis%20of%20lipopolysaccharide%20core%20biosynthesis%20by%20Escherichia%20coli%20K-12%20insertion%20mutagenesis%20of%20the%20rfa%20locus">google scholar</a></td><td><a href="pdf/35.pdf">pdf</a></td></tr>
<tr><td>36</td><td><a id="ref36" class="anchor"></a>Austin, S., and J. Scaife. <i>1970</i>. A new method for selecting RNA polymerase mutants.  <span class="journal">J. Mol. Biol. 49:263–267.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20new%20method%20for%20selecting%20RNA%20polymerase%20mutants">google scholar</a></td><td><a href="pdf/36.pdf">pdf</a></td></tr>
<tr><td>37</td><td><a id="ref37" class="anchor"></a>Ayling, P. D. <i>1981</i>. Methionine sulfoxide is transported by high-affinity methionine and glutamine transport systems in <i>Salmonella typhimurium.</i>  <span class="journal">J. Bacteriol. 148:514–520.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Methionine%20sulfoxide%20is%20transported%20by%20high-affinity%20methionine%20and%20glutamine%20transport%20systems%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/37.pdf">pdf</a></td></tr>
<tr><td>38</td><td><a id="ref38" class="anchor"></a>Ayling, P. D., and E. S. Bridgeland. <i>1972</i>. Methionine transport in wild-type and transport- defective mutants of <i>Salmonella typhimurium.</i>  <span class="journal">J. Gen. Microbiol. 73:127–141.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Methionine%20transport%20in%20wild-type%20and%20transport-%20defective%20mutants%20of%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/38.pdf">pdf</a></td></tr>
<tr><td>39</td><td><a id="ref39" class="anchor"></a>Bachi, B., and H. L. Kornberg. <i>1975</i>. Genes involved in the uptake and catabolism of gluconate by <i>Escherichia coli.</i>  <span class="journal">J. Gen. Microbiol. 90:321–335.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genes%20involved%20in%20the%20uptake%20and%20catabolism%20of%20gluconate%20by%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/39.pdf">pdf</a></td></tr>
<tr><td>40</td><td><a id="ref40" class="anchor"></a>Bachmann, B. J. <i>1983</i>. Linkage map of <i>Escherichia coli</i> K-12, edition 7.  <span class="journal">Microbiol. Rev. 47:180–230.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Linkage%20map%20of%20Escherichia%20coli%20K-12,%20edition%207">google scholar</a></td><td><a href="pdf/40.pdf">pdf</a></td></tr>
<tr><td>41</td><td><a id="ref41" class="anchor"></a>Bachmann, B. J. <i>1990</i>. Linkage map of <i>Escherichia coli </i>K-12, edition 8.  <span class="journal">Microbiol. Rev. 54:130–197.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Linkage%20map%20of%20Escherichia%20coli%20K-12,%20edition%208">google scholar</a></td><td><a href="pdf/41.pdf">pdf</a></td></tr>
<tr><td>42</td><td><a id="ref42" class="anchor"></a>Balbinder, E., R. Callahan III, P. P. McCann, J. C. Cordero, A. R. Weber, A. M. Smith, and F. Angelosanto. <i>1970</i>. Regulatory mutants of the tryptophan operon of <i>Salmonella typhimurium</i>.  <span class="journal">Genetics 66:31–53.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulatory%20mutants%20of%20the%20tryptophan%20operon%20of%20Salmonella%20typhimurium">google scholar</a></td><td><a href="pdf/42.pdf">pdf</a></td></tr>
<tr><td>43</td><td><a id="ref43" class="anchor"></a>Barak, Z., and C. Gilvarg. <i>1974</i>. Triornithine-resistant strains of <i>Escherichia coli.</i> Isolation, definition, and genetic studies.  <span class="journal">J. Biol. Chem. 249:143–148.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Triornithine-resistant%20strains%20of%20Escherichia%20coli.%20Isolation,%20definition,%20and%20genetic%20studies">google scholar</a></td><td><a href="pdf/43.pdf">pdf</a></td></tr>
<tr><td>44</td><td><a id="ref44" class="anchor"></a>Barak, Z., and C. Gilvarg. <i>1975</i>. Specialized peptide transport system in <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 122:1200–1207.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Specialized%20peptide%20transport%20system%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/44.pdf">pdf</a></td></tr>
<tr><td>45</td><td><a id="ref45" class="anchor"></a>Barrett, E. L., and D. L. Riggs. <i>1982</i>. Evidence for a second nitrate reductase activity that is distinct from the respiratory enzyme in <i>Salmonella typhimurium.</i>  <span class="journal">J. Bacteriol. 150:563–571.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Evidence%20for%20a%20second%20nitrate%20reductase%20activity%20that%20is%20distinct%20from%20the%20respiratory%20enzyme%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/45.pdf">pdf</a></td></tr>
<tr><td>46</td><td><a id="ref46" class="anchor"></a>Bassford, P. J., Jr., C. Bradbeer, R. J. Kadner, and C. A. Schnaitman. <i>1976</i>. Transport of vitamin B12 in <i>tonB</i> mutants of <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 128:242–247.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Transport%20of%20vitamin%20B12%20in%20tonB%20mutants%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/46.pdf">pdf</a></td></tr>
<tr><td>47</td><td><a id="ref47" class="anchor"></a>Bassford, P. J., Jr., R. J. Kadner, and C. A. Schnaitman. <i>1977</i>. Biosynthesis of the outer membrane receptor for vitamin B12, E colicins, and bacteriophage BF23 by <i>Escherichia coli:</i> kinetics of phenotypic expression after the introduction of <i>bfe</i>+ and <i>bfe</i> alleles.  <span class="journal">J. Bacteriol. 129:265–275.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Biosynthesis%20of%20the%20outer%20membrane%20receptor%20for%20vitamin%20B12,%20E%20colicins,%20and%20bacteriophage%20BF23%20by%20Escherichia%20coli:%20kinetics%20of%20phenotypic%20expression%20after%20the%20introduction%20of%20bfe+%20and%20bfe%20alleles">google scholar</a></td><td><a href="pdf/47.pdf">pdf</a></td></tr>
<tr><td>48</td><td><a id="ref48" class="anchor"></a>Baughman, G. A., and S. R. Fahnestock. <i>1979</i>. Chloramphenicol resistance mutation in <i>Escherichia coli</i> which maps in the major ribosomal protein gene cluster.  <span class="journal">J. Bacteriol. 137:1315– 1323.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Chloramphenicol%20resistance%20mutation%20in%20Escherichia%20coli%20which%20maps%20in%20the%20major%20ribosomal%20protein%20gene%20cluster">google scholar</a></td><td><a href="pdf/48.pdf">pdf</a></td></tr>
<tr><td>49</td><td><a id="ref49" class="anchor"></a>Baumberg, S. <i>1970</i>. Acetylhistidine as substrate for acetylornithinase: a new system for the selection of arginine regulation mutants in <i>Escherichia coli.</i>  <span class="journal">Mol. Gen. Genet. 106:162–173.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Acetylhistidine%20as%20substrate%20for%20acetylornithinase:%20a%20new%20system%20for%20the%20selection%20of%20arginine%20regulation%20mutants%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/49.pdf">pdf</a></td></tr>
<tr><td>50</td><td><a id="ref50" class="anchor"></a>Baumberg, S. <i>1976</i>. Genetic control of arginine metabolism in prokaryotes. <span class="journal">Second International Symposium on the Genetics of Industrial Microorganisms. Academic Press (Inc.), London.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20control%20of%20arginine%20metabolism%20in%20prokaryotes">google scholar</a></td><td><a href="pdf/50.pdf">pdf</a></td></tr>
<tr><td>51</td><td><a id="ref51" class="anchor"></a>Beacham, I. R., and S. Garrett. <i>1980</i>. Isolation of <i>Escherichia coli</i> mutants <i>(cpdB)</i> deficient in periplasmic 2′:3′-cyclic phosphodiesterase and genetic mapping of the <i>cpdB</i> locus.  <span class="journal">J. Gen. Microbiol. 119:31–34.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20of%20Escherichia%20coli%20mutants%20(cpdB)%20deficient%20in%20periplasmic%202′:3′-cyclic%20phosphodiesterase%20and%20genetic%20mapping%20of%20the%20cpdB%20locus">google scholar</a></td><td><a href="pdf/51.pdf">pdf</a></td></tr>
<tr><td>52</td><td><a id="ref52" class="anchor"></a>Beacham, I. R., R. Kahana, L. Levy, and E. Yagil. <i>1973</i>. Mutants of <i>Escherichia coli</i> K-12 “cryptic” or deficient in 5′-nucleotidase (uridine diphosphate-sugar hydrolase) and 3′-nucleotidase (cyclic phosphodiesterase) activity.  <span class="journal">J. Bacteriol. 116:957–964.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutants%20of%20Escherichia%20coli%20K-12%20“cryptic”%20or%20deficient%20in%205′-nucleotidase%20(uridine%20diphosphate-sugar%20hydrolase)%20and%203′-nucleotidase%20(cyclic%20phosphodiesterase)%20activity">google scholar</a></td><td><a href="pdf/52.pdf">pdf</a></td></tr>
<tr><td>53</td><td><a id="ref53" class="anchor"></a>Beck, C. F., and J. L. Ingraham. <i>1971</i>. Location on the chromosome of <i>Salmonella typhimurium</i> of genes governing pyrimidine metabolism.  <span class="journal">Mol. Gen. Genet. 111:303–316.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Location%20on%20the%20chromosome%20of%20Salmonella%20typhimurium%20of%20genes%20governing%20pyrimidine%20metabolism">google scholar</a></td><td><a href="pdf/53.pdf">pdf</a></td></tr>
<tr><td>54</td><td><a id="ref54" class="anchor"></a>Beck, C. F., J. L. Ingraham, and J. Neuhard. <i>1972</i>. Location on the chromosome of <i>Salmonella</i> <i>typhimurium</i> of genes governing pyrimidine metabolism. II. Uridine kinase, cytosine deaminase, and thymidine kinase.  <span class="journal">Mol. Gen. Genet. 115:208–215.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Location%20on%20the%20chromosome%20of%20Salmonella%20typhimurium%20of%20genes%20governing%20pyrimidine%20metabolism.%20II.%20Uridine%20kinase,%20cytosine%20deaminase,%20and%20thymidine%20kinase">google scholar</a></td><td><a href="pdf/54.pdf">pdf</a></td></tr>
<tr><td>55</td><td><a id="ref55" class="anchor"></a>Begg, K. J., G. F. Hatfull, and W. D. Donachie. <i>1980</i>. Identification of new genes in a cell envelope-cell division gene cluster of <i>Escherichia coli:</i> cell division gene <i>ftsQ.</i>  <span class="journal">J. Bacteriol. 144:435–437.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Identification%20of%20new%20genes%20in%20a%20cell%20envelope-cell%20division%20gene%20cluster%20of%20Escherichia%20coli:%20cell%20division%20gene%20ftsQ.">google scholar</a></td><td><a href="pdf/55.pdf">pdf</a></td></tr>
<tr><td>56</td><td><a id="ref56" class="anchor"></a>Belfort, M., and D. L. Wulff. <i>1973</i>. Genetic and biochemical investigation of the <i>Escherichia coli</i> mutant <i>hfl-1</i> which is lysogenized at high frequency by bacteriophage lambda.  <span class="journal">J. Bacteriol. 115:299– 306.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20and%20biochemical%20investigation%20of%20the%20Escherichia%20coli%20mutant%20hfl-1%20which%20is%20lysogenized%20at%20high%20frequency%20by%20bacteriophage%20lambda">google scholar</a></td><td><a href="pdf/56.pdf">pdf</a></td></tr>
<tr><td>57</td><td><a id="ref57" class="anchor"></a>Beljanski, M., and M. Beljanski. <i>1957</i>. Sur la formation d’enzymes respiratoires chez un mutant d’<i>Escherichia coli</i> streptomycino-résistant et auxotrophe pour l’hemine.  <span class="journal">Ann. Inst. Pasteur  92:396–412.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Sur%20la%20formation%20d’enzymes%20respiratoires%20chez%20un%20mutant%20d’Escherichia%20coli%20streptomycino-résistant%20et%20auxotrophe%20pour%20l’hemine">google scholar</a></td><td><a href="pdf/57.pdf">pdf</a></td></tr>
<tr><td>58</td><td><a id="ref58" class="anchor"></a>Beljanski, M., P. Bourgarel, and M. Beljanski. <i>1970</i>. Showdomycine et biosynthèse d’ARN non complementaires de l’ADN. 1.  <span class="journal">Ann. Inst. Pasteur  118:253–276.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Showdomycine%20et%20biosynthèse%20d’ARN%20non%20complementaires%20de%20l’ADN.%201">google scholar</a></td><td><a href="pdf/58.pdf">pdf</a></td></tr>
<tr><td>59</td><td><a id="ref59" class="anchor"></a>Bennett, R. L., and M. H. Malamy. <i>1970</i>. Arsenate resistant mutants of <i>Escherichia coli</i> and phosphate transport.  <span class="journal">Biochem. Biophys. Res. Commun. 40:469–503.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Arsenate%20resistant%20mutants%20of%20Escherichia%20coli%20and%20phosphate%20transport">google scholar</a></td><td><a href="pdf/59.pdf">pdf</a></td></tr>
<tr><td>60</td><td><a id="ref60" class="anchor"></a>Benson, C. E., S. H. Love, and C. N. Remy. <i>1970</i>. Inhibition of de novo purine biosynthesis and interconversion by 6-methyl purine in <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 101:872–880.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Inhibition%20of%20de%20novo%20purine%20biosynthesis%20and%20interconversion%20by%206-methyl%20purine%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/60.pdf">pdf</a></td></tr>
<tr><td>61</td><td><a id="ref61" class="anchor"></a>Berg, C. M. <i>1990</i>. The branched chain amino acid transaminase genes and their products in <i>Escherichia coli,</i>. <span class="journal">Biosynthesis of Branched Chain Amino Acids. VCH Publishers, New York.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The%20branched%20chain%20amino%20acid%20transaminase%20genes%20and%20their%20products%20in%20Escherichia%20coli,">google scholar</a></td><td><a href="pdf/61.pdf">pdf</a></td></tr>
<tr><td>62</td><td><a id="ref62" class="anchor"></a>Berg, C. M., and J. J. Rossi. <i>1974</i>. Proline excretion and indirect suppression in <i>Escherichia coli</i> and <i>Salmonella typhimurium.</i>  <span class="journal">J. Bacteriol. 118:928–939.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Proline%20excretion%20and%20indirect%20suppression%20in%20Escherichia%20coli%20and%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/62.pdf">pdf</a></td></tr>
<tr><td>63</td><td><a id="ref63" class="anchor"></a>Bergler, H., G. Hogenauer, and F. Thurnowsky. <i>1992</i>. Sequences of the <i>envM</i> gene and of two mutated alleles in <i>Escherichia coli.</i>  <span class="journal">J. Gen. Microbiol. 138:2093–2100.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Sequences%20of%20the%20envM%20gene%20and%20of%20two%20mutated%20alleles%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/63.pdf">pdf</a></td></tr>
<tr><td>64</td><td><a id="ref64" class="anchor"></a>Berman-Kurtz, M., E. C. C. Lin, and D. P. Richey. <i>1971</i>. Promoter-like mutant with increased expression of the glycerol kinase operon of <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 106:724–731.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Promoter-like%20mutant%20with%20increased%20expression%20of%20the%20glycerol%20kinase%20operon%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/64.pdf">pdf</a></td></tr>
<tr><td>65</td><td><a id="ref65" class="anchor"></a>Bernardi, A., and P. Leder. <i>1970</i>. Protein biosynthesis in <i>Escherichia coli:</i> purification and characteristics of a mutant G factor.  <span class="journal">J. Biol. Chem. 245:4263–4268.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Protein%20biosynthesis%20in%20Escherichia%20coli:%20purification%20and%20characteristics%20of%20a%20mutant%20G%20factor">google scholar</a></td><td><a href="pdf/65.pdf">pdf</a></td></tr>
<tr><td>66</td><td><a id="ref66" class="anchor"></a>Bertino, J. B., and K. A. Stacey. <i>1966</i>. A suggested mechanism for the selective procedure for isolating thymine-requiring mutants of <i>Escherichia coli.</i>  <span class="journal">Biochem. J. 101:32c-33c.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20suggested%20mechanism%20for%20the%20selective%20procedure%20for%20isolating%20thymine-requiring%20mutants%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/66.pdf">pdf</a></td></tr>
<tr><td>67</td><td><a id="ref67" class="anchor"></a>Beverin, S., D. E. Sheppard, and S. S. Park. <i>1971</i>. D-Fucose as a gratuitous inducer of the L- arabinose operon in strains of <i>Escherichia coli</i> B/r mutant in gene <i>araC.</i>  <span class="journal">J. Bacteriol. 107:79–86.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=D-Fucose%20as%20a%20gratuitous%20inducer%20of%20the%20L-%20arabinose%20operon%20in%20strains%20of%20Escherichia%20coli%20B/r%20mutant%20in%20gene%20araC.">google scholar</a></td><td><a href="pdf/67.pdf">pdf</a></td></tr>
<tr><td>68</td><td><a id="ref68" class="anchor"></a>Bewick, M. A., and T. C. Y. Lo. <i>1980</i>. Localization of the dicarboxylate binding protein in the cell envelope of <i>Escherichia coli</i> K12.  <span class="journal">Can. J. Biochem. 58:885–897.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Localization%20of%20the%20dicarboxylate%20binding%20protein%20in%20the%20cell%20envelope%20of%20Escherichia%20coli%20K12">google scholar</a></td><td><a href="pdf/68.pdf">pdf</a></td></tr>
<tr><td>69</td><td><a id="ref69" class="anchor"></a>Birch, R. G., J. M. Pemberton, and W. V. S. Basnayake. <i>1990</i>. Stable albicidin resistance in <i>Escherichia coli</i> involves an altered outer-membrane nucleoside uptake system.  <span class="journal">J. Gen. Microbiol. 136:51–58.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Stable%20albicidin%20resistance%20in%20Escherichia%20coli%20involves%20an%20altered%20outer-membrane%20nucleoside%20uptake%20system">google scholar</a></td><td><a href="pdf/69.pdf">pdf</a></td></tr>
<tr><td>70</td><td><a id="ref70" class="anchor"></a>Black, P. N. <i>1988</i>. The <i>fadL</i> gene product of <i>Escherichia coli</i> is an outer membrane protein required for uptake of long-chain fatty acids and involved in sensitivity in bacteriophage T2.  <span class="journal">J. Bacteriol. 170:2850–2854.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The%20fadL%20gene%20product%20of%20Escherichia%20coli%20is%20an%20outer%20membrane%20protein%20required%20for%20uptake%20of%20long-chain%20fatty%20acids%20and%20involved%20in%20sensitivity%20in%20bacteriophage%20T2">google scholar</a></td><td><a href="pdf/70.pdf">pdf</a></td></tr>
<tr><td>71</td><td><a id="ref71" class="anchor"></a>Blank, J., and P. Hoffee. <i>1972</i>. Regulatory mutants of the <i>deo</i> regulon in <i>Salmonella typhimurium.</i>  <span class="journal">Mol. Gen. Genet. 116:291–298.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulatory%20mutants%20of%20the%20deo%20regulon%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/71.pdf">pdf</a></td></tr>
<tr><td>72</td><td><a id="ref72" class="anchor"></a>Bochner, B. R., H.-C. Huang, G. L. Schieven, and B. N. Ames. <i>1980</i>. Positive selection for loss of tetracycline resistance.  <span class="journal">J. Bacteriol. 143:926–933.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Positive%20selection%20for%20loss%20of%20tetracycline%20resistance">google scholar</a></td><td><a href="pdf/72.pdf">pdf</a></td></tr>
<tr><td>73</td><td><a id="ref73" class="anchor"></a>Böck, A., F. Turnowsky, and G. Hoegenauer. <i>1982</i>. Tiamulin resistance mutations in <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 151:1253–1256.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Tiamulin%20resistance%20mutations%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/73.pdf">pdf</a></td></tr>
<tr><td>74</td><td><a id="ref74" class="anchor"></a>Boeke, J. D., F. LaCroute, and G. R. Fink. <i>1984</i>. A positive selection for mutants lacking orotidine-5′-phosphate decarboxylase activity in yeast: 5-fluoro-orotic acid resistance.  <span class="journal">Mol. Gen. Genet. 197:345–346.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20positive%20selection%20for%20mutants%20lacking%20orotidine-5′-phosphate%20decarboxylase%20activity%20in%20yeast:%205-fluoro-orotic%20acid%20resistance">google scholar</a></td><td><a href="pdf/74.pdf">pdf</a></td></tr>
<tr><td>75</td><td><a id="ref75" class="anchor"></a>Bogosian, G., P. V. Haydock, and R. L. Somerville. <i>1983</i>. Indolmycin-mediated inhibition and stimulation of transcription at the <i>trp</i> promoter of <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 153:1120–1123.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Indolmycin-mediated%20inhibition%20and%20stimulation%20of%20transcription%20at%20the%20trp%20promoter%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/75.pdf">pdf</a></td></tr>
<tr><td>76</td><td><a id="ref76" class="anchor"></a>Bollen, A., T. Cabezon, M. De Wilde, R. Villarroel, and A. Herzog. <i>1975</i>. Alteration of ribosomal protein S17 by mutation linked to neamine resistance in <i>Escherichia coli.</i> I. General properties of <i>neaA</i> mutants.  <span class="journal">J. Mol. Biol. 99:795–806.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Alteration%20of%20ribosomal%20protein%20S17%20by%20mutation%20linked%20to%20neamine%20resistance%20in%20Escherichia%20coli.%20I.%20General%20properties%20of%20neaA%20mutants">google scholar</a></td><td><a href="pdf/76.pdf">pdf</a></td></tr>
<tr><td>77</td><td><a id="ref77" class="anchor"></a>Boos, W., C. Bantlow, D. Benner, and E. Roller. <i>1983</i>. <i>cir,</i> a gene conferring resistance to colicin 1, maps between <i>mgl</i> and <i>fpk</i> on the <i>Escherichia coli</i> chromosome.  <span class="journal">Mol. Gen. Genet. 191:401–406.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=cir,%20a%20gene%20conferring%20resistance%20to%20colicin%201,%20maps%20between%20mgl%20and%20fpk%20on%20the%20Escherichia%20coli%20chromosome">google scholar</a></td><td><a href="pdf/77.pdf">pdf</a></td></tr>
<tr><td>78</td><td><a id="ref78" class="anchor"></a>Borg-Oliver, S. A., D. Tarlington, and K. D. Brown. <i>1987</i>. Defective regulation of the phenylalanine biosynthetic operon in mutants of the phenylalanyl transfer RNA synthetase operon.  <span class="journal">J. Bacteriol. 169:1949–1953.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Defective%20regulation%20of%20the%20phenylalanine%20biosynthetic%20operon%20in%20mutants%20of%20the%20phenylalanyl%20transfer%20RNA%20synthetase%20operon">google scholar</a></td><td><a href="pdf/78.pdf">pdf</a></td></tr>
<tr><td>79</td><td><a id="ref79" class="anchor"></a>Boronat, A., E. Caballero, and J. Aguilar. <i>1983</i>. Experimental evolution of a metabolic pathway for ethylene glycol utilization by <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 153:134–139.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Experimental%20evolution%20of%20a%20metabolic%20pathway%20for%20ethylene%20glycol%20utilization%20by%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/79.pdf">pdf</a></td></tr>
<tr><td>80</td><td><a id="ref80" class="anchor"></a>Bourd, G. I., R. S. Erlagaeva, T. N. Bolshakova, and V. N. Gershanovitch. <i>1975</i>. Glucose catabolite repression in <i>Escherichia coli</i> K12 mutants defective in methyl-α-D-glucose transport.  <span class="journal">Eur. J. Biochem. 53:419–427.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Glucose%20catabolite%20repression%20in%20Escherichia%20coli%20K12%20mutants%20defective%20in%20methyl-α-D-glucose%20transport">google scholar</a></td><td><a href="pdf/80.pdf">pdf</a></td></tr>
<tr><td>81</td><td><a id="ref81" class="anchor"></a>Bourret, R. B., and M. S. Fox. <i>1988</i>. Lysogenization of <i>Escherichia coli him</i>+, <i>himA,</i> and <i>himD</i> hosts by bacteriophage Mu.  <span class="journal">J. Bacteriol. 170:1672–1682.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Lysogenization%20of%20Escherichia%20coli%20him+,%20himA,%20and%20himD%20hosts%20by%20bacteriophage%20Mu">google scholar</a></td><td><a href="pdf/81.pdf">pdf</a></td></tr>
<tr><td>82</td><td><a id="ref82" class="anchor"></a>Boy, E., and U. C. Patte. <i>1972</i>. Multivalent repression of aspartic semialdehyde dehydrogenase in <i>Escherichia coli</i> K-12.  <span class="journal">J. Bacteriol. 112:84–92.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Multivalent%20repression%20of%20aspartic%20semialdehyde%20dehydrogenase%20in%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/82.pdf">pdf</a></td></tr>
<tr><td>83</td><td><a id="ref83" class="anchor"></a>Brana, H., J. Hubacek, D. Michaljanicova, I. Holubova, and K. Cejka. <i>1977</i>. Membrane mutation affecting energy-linked functions in <i>Escherichia coli</i> K12.  <span class="journal">Folia Microbiol. 22:198–205.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Membrane%20mutation%20affecting%20energy-linked%20functions%20in%20Escherichia%20coli%20K12">google scholar</a></td><td><a href="pdf/83.pdf">pdf</a></td></tr>
<tr><td>84</td><td><a id="ref84" class="anchor"></a>Branes, L. V., J. M. Somers, and W. W. Kay. <i>1981</i>. Hydrophobic peptide auxotrophy in <i>Salmonella typhimurium.</i>  <span class="journal">J. Bacteriol. 147:986–996.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Hydrophobic%20peptide%20auxotrophy%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/84.pdf">pdf</a></td></tr>
<tr><td>84a</td><td><a id="ref84a" class="anchor"></a>Braun, V. <i>1989</i>. The structurally related <i>exbB</i> and <i>tolQ</i> genes are interchangeable in conferring <i>tonB</i>-dependent colicin, bacteriophage, and albomycin sensitivity.  <span class="journal">J. Bacteriol. 171: 6387–6390.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The%20structurally%20related%20exbB%20and%20tolQ%20genes%20are%20interchangeable%20in%20conferring%20tonB-dependent%20colicin,%20bacteriophage,%20and%20albomycin%20sensitivity">google scholar</a></td><td><a href="pdf/84a.pdf">pdf</a></td></tr>
<tr><td>85</td><td><a id="ref85" class="anchor"></a>Braun, V., J. Frenz, K. Hantke, and K. Schaller. <i>1980</i>. Penetration of colicin M into cells of <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 142:162–168.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Penetration%20of%20colicin%20M%20into%20cells%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/85.pdf">pdf</a></td></tr>
<tr><td>86</td><td><a id="ref86" class="anchor"></a>Braun, V., K. Guenthner, K. Hantke, and L. Zimmermann. <i>1983</i>. Intracellular activation of albomycin in <i>Escherichia coli</i> and <i>Salmonella typhimurium.</i>  <span class="journal">J. Bacteriol. 156:308–315.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Intracellular%20activation%20of%20albomycin%20in%20Escherichia%20coli%20and%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/86.pdf">pdf</a></td></tr>
<tr><td>87</td><td><a id="ref87" class="anchor"></a>Braun, V., K. Hantke, and W. Stauder. <i>1977</i>. Identification of the <i>sid</i> outer membrane receptor protein in <i>Salmonella typhimurium</i> SL1024.  <span class="journal">Mol. Gen. Genet. 155:227–229.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Identification%20of%20the%20sid%20outer%20membrane%20receptor%20protein%20in%20Salmonella%20typhimurium%20SL1024">google scholar</a></td><td><a href="pdf/87.pdf">pdf</a></td></tr>
<tr><td>88</td><td><a id="ref88" class="anchor"></a>Braun-Breton, C., and M. Hofnung. <i>1981</i>. In vivo and in vitro functional alterations of the bacteriophage lambda receptor in <i>lamB</i> missense mutants of <i>Escherichia coli</i> K-12.  <span class="journal">J. Bacteriol. 148:845–852.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=In%20vivo%20and%20in%20vitro%20functional%20alterations%20of%20the%20bacteriophage%20lambda%20receptor%20in%20lamB%20missense%20mutants%20of%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/88.pdf">pdf</a></td></tr>
<tr><td>89</td><td><a id="ref89" class="anchor"></a>Breckenridge, L., and L. Gorini. <i>1970</i>. Genetic analysis of streptomycin resistance in <i>Escherichia</i> <i>coli.</i>  <span class="journal">Genetics 65:9–25.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20analysis%20of%20streptomycin%20resistance%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/89.pdf">pdf</a></td></tr>
<tr><td>90</td><td><a id="ref90" class="anchor"></a>Breeze, A. S., and E. E. Obaseiki-Ebor. <i>1983</i>. Mutations to nitrofurantoin and nitrofurazone resistance in <i>Escherichia coli</i> K12.  <span class="journal">J. Gen. Microbiol. 129:99–103.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutations%20to%20nitrofurantoin%20and%20nitrofurazone%20resistance%20in%20Escherichia%20coli%20K12">google scholar</a></td><td><a href="pdf/90.pdf">pdf</a></td></tr>
<tr><td>91</td><td><a id="ref91" class="anchor"></a>Brenchley, J. E., C. A. Baker, and L. G. Patil. <i>1975</i>. Regulation of ammonia assimilatory enzymes in <i>Salmonella typhimurium.</i>  <span class="journal">J. Bacteriol. 124:182–189.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation%20of%20ammonia%20assimilatory%20enzymes%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/91.pdf">pdf</a></td></tr>
<tr><td>92</td><td><a id="ref92" class="anchor"></a>Brickman, E., L. Soll, and J. Beckwith. <i>1973</i>. Genetic characterization of mutations which affect catabolite-sensitive operons in <i>Escherichia coli,</i> including deletions of the gene for adenyl cyclase.  <span class="journal">J. Bacteriol. 116:582–587.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20characterization%20of%20mutations%20which%20affect%20catabolite-sensitive%20operons%20in%20Escherichia%20coli,%20including%20deletions%20of%20the%20gene%20for%20adenyl%20cyclase">google scholar</a></td><td><a href="pdf/92.pdf">pdf</a></td></tr>
<tr><td>93</td><td><a id="ref93" class="anchor"></a>Brochu, A., N. Brochu, T. I. Nicas, T. R. Parr, Jr., A. A. Minnick, Jr., E. K. Dolence, J. A. McKee, M. J. Miller, M. C. Lavoie, and F. Malouin. <i>1992</i>. Modes of action and inhibitory activities of new siderophore-beta lactam conjugates that use specific iron uptake pathways for entry into bacteria.  <span class="journal">Antimicrob. Agents Chemother. 36:2166–2175.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Modes%20of%20action%20and%20inhibitory%20activities%20of%20new%20siderophore-beta%20lactam%20conjugates%20that%20use%20specific%20iron%20uptake%20pathways%20for%20entry%20into%20bacteria">google scholar</a></td><td><a href="pdf/93.pdf">pdf</a></td></tr>
<tr><td>94</td><td><a id="ref94" class="anchor"></a>Brown, K. D. <i>1970</i>. Formation of aromatic amino acid pools in <i>Escherichia coli</i> K-12.  <span class="journal">J. Bacteriol. 104:177–188.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Formation%20of%20aromatic%20amino%20acid%20pools%20in%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/94.pdf">pdf</a></td></tr>
<tr><td>95</td><td><a id="ref95" class="anchor"></a>Brown, K. D., and R. L. Somerville. <i>1971</i>. Repression of aromatic amino acid biosynthesis in <i>Escherichia coli</i> K-12.  <span class="journal">J. Bacteriol. 108:386–399.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Repression%20of%20aromatic%20amino%20acid%20biosynthesis%20in%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/95.pdf">pdf</a></td></tr>
<tr><td>96</td><td><a id="ref96" class="anchor"></a>Brown, T. D. K., M. C. Jones-Mortimer, and H. L. Kornberg. <i>1977</i>. The enzymic interconversion of acetate and acetyl-coenzyme A in <i>Escherichia coli.</i>  <span class="journal">J. Gen. Microbiol. 102:327– 336.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The%20enzymic%20interconversion%20of%20acetate%20and%20acetyl-coenzyme%20A%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/96.pdf">pdf</a></td></tr>
<tr><td>97</td><td><a id="ref97" class="anchor"></a>Bruce, I., J. Hardy, and K. A. Stacey. <i>1984</i>. Potentiation by purines of the growth-inhibitory effects of sulphonamides on <i>Escherichia coli</i> and the location of the gene which mediates this effect.  <span class="journal">J. Gen. Microbiol. 130:2489–2495.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Potentiation%20by%20purines%20of%20the%20growth-inhibitory%20effects%20of%20sulphonamides%20on%20Escherichia%20coli%20and%20the%20location%20of%20the%20gene%20which%20mediates%20this%20effect">google scholar</a></td><td><a href="pdf/97.pdf">pdf</a></td></tr>
<tr><td>98</td><td><a id="ref98" class="anchor"></a>Bruni, C. B., V. Colantuoni, L. Sbordone, R. Cortese, and F. Blasi. <i>1977</i>. Biochemical and regulatory properties of <i>Escherichia coli</i> K-12 <i>hisT</i> mutants.  <span class="journal">J. Bacteriol. 130:4–10.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Biochemical%20and%20regulatory%20properties%20of%20Escherichia%20coli%20K-12%20hisT%20mutants">google scholar</a></td><td><a href="pdf/98.pdf">pdf</a></td></tr>
<tr><td>99</td><td><a id="ref99" class="anchor"></a>Brunner, D. P., G. S. Graham, and R. W. Treick. <i>1982</i>. Exchange of colicin receptor capacity between strains of <i>Escherichia coli</i> sensitive or resistant to colicin K-K235.  <span class="journal">Arch. Microbiol. 131:229–234.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Exchange%20of%20colicin%20receptor%20capacity%20between%20strains%20of%20Escherichia%20coli%20sensitive%20or%20resistant%20to%20colicin%20K-K235">google scholar</a></td><td><a href="pdf/99.pdf">pdf</a></td></tr>
<tr><td>100</td><td><a id="ref100" class="anchor"></a>Buchstein, S. R., and D. C. Hinkle. <i>1982</i>. Genetic analysis of two bacterial RNA polymerase mutants that inhibit the growth of bacteriophage T7.  <span class="journal">Mol. Gen. Genet. 188:211–218.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20analysis%20of%20two%20bacterial%20RNA%20polymerase%20mutants%20that%20inhibit%20the%20growth%20of%20bacteriophage%20T7">google scholar</a></td><td><a href="pdf/100.pdf">pdf</a></td></tr>
<tr><td>101</td><td><a id="ref101" class="anchor"></a>Buckel, P., A. Buchberger, A. Boeck, and H. G. Wittmann. <i>1977</i>. Alteration of ribosomal protein L6 in mutants of <i>Escherichia coli</i> resistant to gentamicin.  <span class="journal">Mol. Gen. Genet. 158:47–54.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Alteration%20of%20ribosomal%20protein%20L6%20in%20mutants%20of%20Escherichia%20coli%20resistant%20to%20gentamicin">google scholar</a></td><td><a href="pdf/101.pdf">pdf</a></td></tr>
<tr><td>102</td><td><a id="ref102" class="anchor"></a>Buhr, A., G. A. Daniels, and B. Erin. <i>1992</i>. The glucose transporter of <i>Escherichia coli</i> mutants with impaired translocation activity that retain phosphorylation activity.  <span class="journal">J. Biol. Chem. 267:3847– 3851.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The%20glucose%20transporter%20of%20Escherichia%20coli%20mutants%20with%20impaired%20translocation%20activity%20that%20retain%20phosphorylation%20activity">google scholar</a></td><td><a href="pdf/102.pdf">pdf</a></td></tr>
<tr><td>103</td><td><a id="ref103" class="anchor"></a>Bullas, L. R., and C. Colson. <i>1975</i>. DNA restriction and modification systems in <i>Salmonella.</i> III. SP, a <i>Salmonella potsdam</i> system allelic to the SB system in <i>Salmonella typhimurium.</i>  <span class="journal">Mol. Gen. Genet. 139:177–188.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=DNA%20restriction%20and%20modification%20systems%20in%20Salmonella.%20III.%20SP,%20a%20Salmonella%20potsdam%20system%20allelic%20to%20the%20SB%20system%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/103.pdf">pdf</a></td></tr>
<tr><td>104</td><td><a id="ref104" class="anchor"></a>Burd, G. I., G. I. Erlagaeva, T. N. Bol’shakova, and V. N. Gershanovich. <i>1974</i>. Separation of the α-methylglucoside transport and phosphorylation system in a mutant of <i>Escherichia coli</i> K12 resistant to catabolic repression by glucose.  <span class="journal">Dokl. Akad. Nauk SSSR 215:1243–1246.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Separation%20of%20the%20α-methylglucoside%20transport%20and%20phosphorylation%20system%20in%20a%20mutant%20of%20Escherichia%20coli%20K12%20resistant%20to%20catabolic%20repression%20by%20glucose">google scholar</a></td><td></td></tr>
<tr><td>105</td><td><a id="ref105" class="anchor"></a>Burman, L. G., and K. Nordstrom. <i>1971</i>. Colicin tolerance induced by ampicillin resistance in a strain of <i>Escherichia coli</i> K-12.  <span class="journal">J. Bacteriol. 106:1–13.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Colicin%20tolerance%20induced%20by%20ampicillin%20resistance%20in%20a%20strain%20of%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/105.pdf">pdf</a></td></tr>
<tr><td>106</td><td><a id="ref106" class="anchor"></a>Burns, R. O., J. G. Hofler, and G. H. Luginbuhl. <i>1979</i>. Threonine deaminase from <i>Salmonella</i> <i>typhimurium.</i> Substrate-specific patterns of inhibition in an activation site-deficient form of the enzyme.  <span class="journal">J. Biol. Chem. 254:1074–1079.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Threonine%20deaminase%20from%20Salmonella%20typhimurium.%20Substrate-specific%20patterns%20of%20inhibition%20in%20an%20activation%20site-deficient%20form%20of%20the%20enzyme">google scholar</a></td><td><a href="pdf/106.pdf">pdf</a></td></tr>
<tr><td>107</td><td><a id="ref107" class="anchor"></a>Burton, K. <i>1977</i>. Transport of adenine, hypoxanthine and uracil into <i>Escherichia coli.</i>  <span class="journal">Biochem. J. 168:195–204.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Transport%20of%20adenine,%20hypoxanthine%20and%20uracil%20into%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/107.pdf">pdf</a></td></tr>
<tr><td>108</td><td><a id="ref108" class="anchor"></a>Buttin, G. <i>1963</i>. Mécanismes regulateurs dans la biosynthèse des enzymes du metabolisme du galactose chez <i>Escherichia coli</i> K12.  <span class="journal">J. Mol. Biol. 7:183–205.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mécanismes%20regulateurs%20dans%20la%20biosynthèse%20des%20enzymes%20du%20metabolisme%20du%20galactose%20chez%20Escherichia%20coli%20K12">google scholar</a></td><td><a href="pdf/108.pdf">pdf</a></td></tr>
<tr><td>109</td><td><a id="ref109" class="anchor"></a>Buxton, R. S. <i>1971</i>. Genetic analysis of <i>Escherichia coli</i> K-12 mutants resistant to bacteriophage BF23 and the E-group colicins.  <span class="journal">Mol. Gen. Genet. 113:154–156.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20analysis%20of%20Escherichia%20coli%20K-12%20mutants%20resistant%20to%20bacteriophage%20BF23%20and%20the%20E-group%20colicins">google scholar</a></td><td><a href="pdf/109.pdf">pdf</a></td></tr>
<tr><td>110</td><td><a id="ref110" class="anchor"></a>Buxton, R. S., K. Hammer-Jespersen, and T. D. Hansen. <i>1978</i>. Insertion of bacteriophage lambda into the <i>deo</i> operon of <i>Escherichia coli</i> K-12 and isolation of plaque-forming λ <i>deo</i>+ transducing bacteriophages.  <span class="journal">J. Bacteriol. 136:668–681.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Insertion%20of%20bacteriophage%20lambda%20into%20the%20deo%20operon%20of%20Escherichia%20coli%20K-12%20and%20isolation%20of%20plaque-forming%20λ%20deo+%20transducing%20bacteriophages">google scholar</a></td><td><a href="pdf/110.pdf">pdf</a></td></tr>
<tr><td>111</td><td><a id="ref111" class="anchor"></a>Buxton, R. S., K. Hammer-Jespersen, and P. Valentin-Hansen. <i>1980</i>. A second purine nucleoside phosphorylase in <i>Escherichia coli</i> K-12. I. Xanthosine phosphorylase regulatory mutants isolated as secondary-site revertants of a <i>deoD</i> mutant.  <span class="journal">Mol. Gen. Genet. 179:331–340.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20second%20purine%20nucleoside%20phosphorylase%20in%20Escherichia%20coli%20K-12.%20I.%20Xanthosine%20phosphorylase%20regulatory%20mutants%20isolated%20as%20secondary-site%20revertants%20of%20a%20deoD%20mutant">google scholar</a></td><td><a href="pdf/111.pdf">pdf</a></td></tr>
<tr><td>112</td><td><a id="ref112" class="anchor"></a>Calcott, P. H., and K. N. Calcott. <i>1984</i>. Involvement of outer membrane proteins in freeze thaw resistance of <i>Escherichia coli.</i>  <span class="journal">Can. J. Microbiol. 30:339–344.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Involvement%20of%20outer%20membrane%20proteins%20in%20freeze%20thaw%20resistance%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/112.pdf">pdf</a></td></tr>
<tr><td>113</td><td><a id="ref113" class="anchor"></a>Calendar, R., B. Lindquist, G. Sironi, and A. J. Clark. <i>1970</i>. Characterization of REP– mutants and their interaction with P2 phage.  <span class="journal">Virology 40:72–83.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Characterization%20of%20REP–%20mutants%20and%20their%20interaction%20with%20P2%20phage">google scholar</a></td><td><a href="pdf/113.pdf">pdf</a></td></tr>
<tr><td>114</td><td><a id="ref114" class="anchor"></a>Caligiuri, M. G., and R. Bauerle. <i>1991</i>. Identification of amino acid residues involved in feedback regulation of the anthranilate synthase complex from <i>Salmonella typhimurium:</i> evidence for an amino-terminal regulatory site.  <span class="journal">J. Biol. Chem. 266:8328–8335.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Identification%20of%20amino%20acid%20residues%20involved%20in%20feedback%20regulation%20of%20the%20anthranilate%20synthase%20complex%20from%20Salmonella%20typhimurium:%20evidence%20for%20an%20amino-terminal%20regulatory%20site">google scholar</a></td><td><a href="pdf/114.pdf">pdf</a></td></tr>
<tr><td>115</td><td><a id="ref115" class="anchor"></a>Calvo, J. M., M. Freundlich, and H. E. Umbarger. <i>1969</i>. Regulation of branched-chain amino acid biosynthesis in <i>Salmonella typhimurium:</i> isolation of regulatory mutants.  <span class="journal">J. Bacteriol. 97:1272– 1282.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation%20of%20branched-chain%20amino%20acid%20biosynthesis%20in%20Salmonella%20typhimurium:%20isolation%20of%20regulatory%20mutants">google scholar</a></td><td><a href="pdf/115.pdf">pdf</a></td></tr>
<tr><td>116</td><td><a id="ref116" class="anchor"></a>Calvo, J. M., P. Margolin, and H. E. Umbarger. <i>1969</i>. Operator constitutive mutations in the leucine operon of <i>Salmonella typhimurium.</i>  <span class="journal">Genetics 61:777–787.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Operator%20constitutive%20mutations%20in%20the%20leucine%20operon%20of%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/116.pdf">pdf</a></td></tr>
<tr><td>117</td><td><a id="ref117" class="anchor"></a>Calvo, R. A., and J. M. Calvo. <i>1967</i>. Lack of end-product inhibition and repression of leucine synthesis in a strain of <i>Salmonella typhimurium.</i>  <span class="journal">Science 156:1107–1109.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Lack%20of%20end-product%20inhibition%20and%20repression%20of%20leucine%20synthesis%20in%20a%20strain%20of%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/117.pdf">pdf</a></td></tr>
<tr><td>118</td><td><a id="ref118" class="anchor"></a>Campbell, J. H., J. Lengyel, and J. Langridge. <i>1973</i>. Evolution of a second gene for β- galactosidase in <i>Escherichia coli.</i>  <span class="journal">Proc. Natl. Acad. Sci. USA 70:1841–1845.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Evolution%20of%20a%20second%20gene%20for%20β-%20galactosidase%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/118.pdf">pdf</a></td></tr>
<tr><td>119</td><td><a id="ref119" class="anchor"></a>Cannon, M., T. Cabezon, and A. Bollen. <i>1979</i>. Mapping of neamine resistance: identification of two genetic loci, <i>neaA</i> and <i>neaB.</i>  <span class="journal">Mol. Gen. Genet. 130:321–326.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mapping%20of%20neamine%20resistance:%20identification%20of%20two%20genetic%20loci,%20neaA%20and%20neaB.">google scholar</a></td><td><a href="pdf/119.pdf">pdf</a></td></tr>
<tr><td>120</td><td><a id="ref120" class="anchor"></a>Cardelli, J., and J. Konisky. <i>1974</i>. Isolation and characterization of an <i>Escherichia coli</i> mutant tolerant to colicins Ia and Ib.  <span class="journal">J. Bacteriol. 119:379–385.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20and%20characterization%20of%20an%20Escherichia%20coli%20mutant%20tolerant%20to%20colicins%20Ia%20and%20Ib">google scholar</a></td><td><a href="pdf/120.pdf">pdf</a></td></tr>
<tr><td>121</td><td><a id="ref121" class="anchor"></a>Casse, F. <i>1970</i>. Mapping of the gene <i>chlB</i> controlling membrane bound nitrate reductase and formic hydrogen-lyase activities in <i>Escherichia coli</i> K-12.  <span class="journal">Biochem. Biophys. Res. Commun. 39:429– 436.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mapping%20of%20the%20gene%20chlB%20controlling%20membrane%20bound%20nitrate%20reductase%20and%20formic%20hydrogen-lyase%20activities%20in%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/121.pdf">pdf</a></td></tr>
<tr><td>122</td><td><a id="ref122" class="anchor"></a>Casse, F., M. C. Pascal, M. Chippaux, and J. Ratouchniak. <i>1972</i>. Mapping of the <i>chlB</i> gene in <i>Salmonella typhimurium</i> LT2.  <span class="journal">Mol. Gen. Genet. 119:67–70.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mapping%20of%20the%20chlB%20gene%20in%20Salmonella%20typhimurium%20LT2">google scholar</a></td><td><a href="pdf/122.pdf">pdf</a></td></tr>
<tr><td>123</td><td><a id="ref123" class="anchor"></a>Caster, J. H. <i>1967</i>. Selection of thymine-requiring strains from <i>Escherichia coli</i> on solid medium.  <span class="journal">J. Bacteriol. 94:1804.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Selection%20of%20thymine-requiring%20strains%20from%20Escherichia%20coli%20on%20solid%20medium">google scholar</a></td><td><a href="pdf/123.pdf">pdf</a></td></tr>
<tr><td>124</td><td><a id="ref124" class="anchor"></a>Chai, T., and J. Foulds. <i>1974</i>. Missing outer membrane protein in <i>tolG</i> mutants of <i>Escherichia coli.</i>  <span class="journal">J. Mol. Biol. 85:465–474.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Missing%20outer%20membrane%20protein%20in%20tolG%20mutants%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/124.pdf">pdf</a></td></tr>
<tr><td>125</td><td><a id="ref125" class="anchor"></a>Chai, T.-J., V. Wu, and J. Foulds. <i>1982</i>. Colicin A receptor role of two <i>Escherichia coli</i> outer membrane proteins (OmpF protein and <i>btuB</i> gene product) and lipopolysaccharide.  <span class="journal">J. Bacteriol. 151:983–988.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Colicin%20A%20receptor%20role%20of%20two%20Escherichia%20coli%20outer%20membrane%20proteins%20(OmpF%20protein%20and%20btuB%20gene%20product)%20and%20lipopolysaccharide">google scholar</a></td><td><a href="pdf/125.pdf">pdf</a></td></tr>
<tr><td>126</td><td><a id="ref126" class="anchor"></a>Chang, G. W., D. Straus, and B. N. Ames. <i>1971</i>. Enriched selection of dominant mutations: histidine operator mutations.  <span class="journal">J. Bacteriol. 107:578–579.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Enriched%20selection%20of%20dominant%20mutations:%20histidine%20operator%20mutations">google scholar</a></td><td><a href="pdf/126.pdf">pdf</a></td></tr>
<tr><td>127</td><td><a id="ref127" class="anchor"></a>Chater, K. F., and R. J. Rowbury. <i>1970</i>. A genetical study of the feedback-sensitive enzyme of methionine synthesis in <i>Salmonella typhimurium.</i>  <span class="journal">J. Gen. Microbiol. 63:111–120.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20genetical%20study%20of%20the%20feedback-sensitive%20enzyme%20of%20methionine%20synthesis%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/127.pdf">pdf</a></td></tr>
<tr><td>128</td><td><a id="ref128" class="anchor"></a>Chattopadhyay, M. K., A. K. Ghosh, and S. Sengupta. <i>1991</i>. Control of methionine biosynthesis in <i>Escherichia coli</i> K12: a closer study with analog resistant mutants.  <span class="journal">J. Gen. Microbiol. 137:685–691.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Control%20of%20methionine%20biosynthesis%20in%20Escherichia%20coli%20K12:%20a%20closer%20study%20with%20analog%20resistant%20mutants">google scholar</a></td><td><a href="pdf/128.pdf">pdf</a></td></tr>
<tr><td>129</td><td><a id="ref129" class="anchor"></a>Chen, M. X., N. Bouquin, V. Norris, S. Casaregola, S. J. Seror, J. Simone, and I. B. Holland. <i>1991</i>. A single base change in the acceptor stem of tRNA Leu 3 confers resistance upon <i>Escherichia</i> <i>coli</i> to the calmodulin inhibitor 48/80.  <span class="journal">EMBO J. 10:3113–3122.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20single%20base%20change%20in%20the%20acceptor%20stem%20of%20tRNA%20Leu%203%20confers%20resistance%20upon%20Escherichia%20coli%20to%20the%20calmodulin%20inhibitor%2048/80">google scholar</a></td><td><a href="pdf/129.pdf">pdf</a></td></tr>
<tr><td>130</td><td><a id="ref130" class="anchor"></a>Chippaux, M., D. Giudici, A. Abou-Jaoude, and M. C. Pascal. <i>1978</i>. A mutation leading to the total lack of nitrate reductase activity in <i>Escherichia</i> coli K12.  <span class="journal">Mol. Gen. Genet. 160:225–229.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20mutation%20leading%20to%20the%20total%20lack%20of%20nitrate%20reductase%20activity%20in%20Escherichia%20coli%20K12">google scholar</a></td><td><a href="pdf/130.pdf">pdf</a></td></tr>
<tr><td>131</td><td><a id="ref131" class="anchor"></a>Christman, M. F., R. W. Morgan, F. S. Jacobson, and B. N. Ames. <i>1985</i>. Positive control of a regulon for defenses against oxidative stress and some heat-shock proteins in <i>Salmonella typhimurium.</i>  <span class="journal">Cell 41:753–762.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Positive%20control%20of%20a%20regulon%20for%20defenses%20against%20oxidative%20stress%20and%20some%20heat-shock%20proteins%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/131.pdf">pdf</a></td></tr>
<tr><td>132</td><td><a id="ref132" class="anchor"></a>Clark, D., and J. E. Cronan, Jr. <i>1980</i>. <i>Escherichia coli</i> mutants with altered control of alcohol dehydrogenase and nitrate reductase.  <span class="journal">J. Bacteriol. 141:177–183.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Escherichia%20coli%20mutants%20with%20altered%20control%20of%20alcohol%20dehydrogenase%20and%20nitrate%20reductase">google scholar</a></td><td><a href="pdf/132.pdf">pdf</a></td></tr>
<tr><td>133</td><td><a id="ref133" class="anchor"></a>Clark, D. P., and J. P. Beard. <i>1979</i>. Altered phospholipid composition in mutants of <i>Escherichia</i> <i>coli</i> sensitive or resistant to organic solvents.  <span class="journal">J. Gen. Microbiol. 113:267–279.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Altered%20phospholipid%20composition%20in%20mutants%20of%20Escherichia%20coli%20sensitive%20or%20resistant%20to%20organic%20solvents">google scholar</a></td><td><a href="pdf/133.pdf">pdf</a></td></tr>
<tr><td>134</td><td><a id="ref134" class="anchor"></a>Clement, J. M., C. Braun-Breton, E. Lepouce, C. Marchal, D. Perrin, H. Villarroya, and M. Hofnung. <i>1982</i>. A system for genetic analysis in gene <i>lamB:</i> first results with lambda resistant tight mutants.  <span class="journal">Ann. Microbiol. 133A:9–20.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20system%20for%20genetic%20analysis%20in%20gene%20lamB:%20first%20results%20with%20lambda%20resistant%20tight%20mutants">google scholar</a></td><td><a href="pdf/134.pdf">pdf</a></td></tr>
<tr><td>135</td><td><a id="ref135" class="anchor"></a>Clement, J.-M., E. Lepouce, C. Marchal, and M. Hofnung. <i>1983</i>. Genetic study of a membrane protein: DNA sequence alterations due to 17 <i>lamB</i> point mutations affecting adsorption of phage lambda.  <span class="journal">EMBO J. 2:77–80.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20study%20of%20a%20membrane%20protein:%20DNA%20sequence%20alterations%20due%20to%2017%20lamB%20point%20mutations%20affecting%20adsorption%20of%20phage%20lambda">google scholar</a></td><td><a href="pdf/135.pdf">pdf</a></td></tr>
<tr><td>136</td><td><a id="ref136" class="anchor"></a>Clifton, G., S. R. Bryant, and C. G. Skinner. <i>1970</i>. N1-(substituted) pantothenamides, antimetabolites of pantothenic acid.  <span class="journal">Arch. Biochem. Biophys. 137:523–528.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=N1-(substituted)%20pantothenamides,%20antimetabolites%20of%20pantothenic%20acid">google scholar</a></td><td><a href="pdf/136.pdf">pdf</a></td></tr>
<tr><td>137</td><td><a id="ref137" class="anchor"></a>Cohen, G., and F. Jacob. <i>1959</i>. Sur la répression de la synthese des enzymes intervenant dans la formation du tryptophane chez <i>Escherichia coli.</i>  <span class="journal">C.R. Acad. Sci. Ser. D 248:3490–3492.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Sur%20la%20répression%20de%20la%20synthese%20des%20enzymes%20intervenant%20dans%20la%20formation%20du%20tryptophane%20chez%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/137.pdf">pdf</a></td></tr>
<tr><td>138</td><td><a id="ref138" class="anchor"></a>Cohen, G. N., J.-C. Patte, P. Truffa-Bachi, C. Sawas, and M. Doudoroff. <i>1965</i>. Repression and end-product inhibition in a branched biosynthetic pathway. <span class="journal">Mécanismes de Regulation des Activitiés Cellulaires chez les Micro-organismes. Centre National de la Recherche Scientifique, Paris.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Repression%20and%20end-product%20inhibition%20in%20a%20branched%20biosynthetic%20pathway">google scholar</a></td><td><a href="pdf/138.pdf">pdf</a></td></tr>
<tr><td>139</td><td><a id="ref139" class="anchor"></a>Cole, S. T., and J. R Guest. <i>1979</i>. Production of a soluble form of fumarate reductase by multiple gene duplications in <i>Escherichia coli</i> K12.  <span class="journal">Eur. J. Biochem. 102:65–71.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Production%20of%20a%20soluble%20form%20of%20fumarate%20reductase%20by%20multiple%20gene%20duplications%20in%20Escherichia%20coli%20K12">google scholar</a></td><td><a href="pdf/139.pdf">pdf</a></td></tr>
<tr><td>140</td><td><a id="ref140" class="anchor"></a>Coleman, J., S. Inouye, and M. Inouye. <i>1983</i>. Isolation of mutants of the major outer membrane lipoprotein of <i>Escherichia coli</i> for the study of its assembly.  <span class="journal">Methods Enzymol. 97:124–129.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20of%20mutants%20of%20the%20major%20outer%20membrane%20lipoprotein%20of%20Escherichia%20coli%20for%20the%20study%20of%20its%20assembly">google scholar</a></td><td><a href="pdf/140.pdf">pdf</a></td></tr>
<tr><td>141</td><td><a id="ref141" class="anchor"></a>Coleman, W. G., and L. Leive. <i>1979</i>. Two mutations which affect the barrier function of the <i>Escherichia coli</i> K-12 outer membrane.  <span class="journal">J. Bacteriol. 139:899–910.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Two%20mutations%20which%20affect%20the%20barrier%20function%20of%20the%20Escherichia%20coli%20K-12%20outer%20membrane">google scholar</a></td><td><a href="pdf/141.pdf">pdf</a></td></tr>
<tr><td>142</td><td><a id="ref142" class="anchor"></a>Cookson, B. T., B. M. Olivera, and J. R. Roth. <i>1987</i>. Genetic characterization and regulation of the <i>nadB</i> locus of <i>Salmonella typhimurium.</i>  <span class="journal">J. Bacteriol. 169:4285–4293.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20characterization%20and%20regulation%20of%20the%20nadB%20locus%20of%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/142.pdf">pdf</a></td></tr>
<tr><td>143</td><td><a id="ref143" class="anchor"></a>Cooley, W., K. Sirotkin, R. Green, and L. Snyder. <i>1979</i>. A new gene of <i>Escherichia coli</i> K-12 whose product participates in T4 bacteriophage late gene expression: interaction of <i>lit</i> with the T4- induced polynucleotide 5′-kinase 3′-phosphatase.  <span class="journal">J. Bacteriol. 140:83–91.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20new%20gene%20of%20Escherichia%20coli%20K-12%20whose%20product%20participates%20in%20T4%20bacteriophage%20late%20gene%20expression:%20interaction%20of%20lit%20with%20the%20T4-%20induced%20polynucleotide%205′-kinase%203′-phosphatase">google scholar</a></td><td><a href="pdf/143.pdf">pdf</a></td></tr>
<tr><td>144</td><td><a id="ref144" class="anchor"></a>Cooper, R. A. <i>1978</i>. The utilization of D-galactonate and D-2-oxo-3-deoxygalactonate by <i>Escherichia coli</i> K-12.  <span class="journal">Arch. Microbiol. 118:199–206.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The%20utilization%20of%20D-galactonate%20and%20D-2-oxo-3-deoxygalactonate%20by%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/144.pdf">pdf</a></td></tr>
<tr><td>145</td><td><a id="ref145" class="anchor"></a>Corchuelo, M. C., A. Herzog, L. Desmarez, R. Lavalle, and A. Bollen. <i>1981</i>. Resistance to the peptide-like antibiotic negamycin in <i>Escherichia coli.</i>  <span class="journal">Biochem. Biophys. Res. Commun. 100:1497– 1503.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Resistance%20to%20the%20peptide-like%20antibiotic%20negamycin%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/145.pdf">pdf</a></td></tr>
<tr><td>146</td><td><a id="ref146" class="anchor"></a>Cordaro, J. C., T. Melton, J. P. Stratis, M. Atagun, C. Gladding, P. E. Hartman, and S. Roseman. <i>1976</i>. Fosfomycin resistance: selection method for internal and extended deletions of the phosphoenolpyruvate:sugar phosphotransferase genes of <i>Salmonella typhimurium.</i>  <span class="journal">J. Bacteriol. 128:785–793.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Fosfomycin%20resistance:%20selection%20method%20for%20internal%20and%20extended%20deletions%20of%20the%20phosphoenolpyruvate:sugar%20phosphotransferase%20genes%20of%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/146.pdf">pdf</a></td></tr>
<tr><td>147</td><td><a id="ref147" class="anchor"></a>Cordaro, J. C., and S. Roseman. <i>1972</i>. Deletion mapping of the genes coding for HPr and enzyme I of the phosphoenolpyruvate:sugar phosphotransferase system in <i>Salmonella typhimurium.</i>  <span class="journal">J. Bacteriol. 112:17–29.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Deletion%20mapping%20of%20the%20genes%20coding%20for%20HPr%20and%20enzyme%20I%20of%20the%20phosphoenolpyruvate:sugar%20phosphotransferase%20system%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/147.pdf">pdf</a></td></tr>
<tr><td>148</td><td><a id="ref148" class="anchor"></a>Cortese, R., R. Landsberg, R. A. VanderHaar, H. E. Umbarger, and B. N. Ames. <i>1974</i>. Pleiotrophy of <i>hisT</i> mutants blocked in pseudouridine synthesis in tRNA: <i>leucine</i> and <i>isoleucine- valine</i> operons.  <span class="journal">Proc. Natl. Acad. Sci. USA 71:1857–1861.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Pleiotrophy%20of%20hisT%20mutants%20blocked%20in%20pseudouridine%20synthesis%20in%20tRNA:%20leucine%20and%20isoleucine-%20valine%20operons">google scholar</a></td><td><a href="pdf/148.pdf">pdf</a></td></tr>
<tr><td>149</td><td><a id="ref149" class="anchor"></a>Corwin, L. M., G. R. Fanning, F. Feldman, and P. Margolin. <i>1966</i>. Mutation leading to increased sensitivity to chromium ion in <i>Salmonella typhimurium.</i>  <span class="journal">J. Bacteriol. 91:1509–1515.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutation%20leading%20to%20increased%20sensitivity%20to%20chromium%20ion%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/149.pdf">pdf</a></td></tr>
<tr><td>150</td><td><a id="ref150" class="anchor"></a>Cosloy, S. D. <i>1973</i>. D-Serine transport system in <i>Escherichia coli</i> K-12.  <span class="journal">J. Bacteriol. 114:679–684.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=D-Serine%20transport%20system%20in%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/150.pdf">pdf</a></td></tr>
<tr><td>151</td><td><a id="ref151" class="anchor"></a>Coulton, J. W., P. Mason, D. R. Cameron, G. Carmel, R. Jean, and H. N. Rode. <i>1986</i>. Protein fusions of β-galactosidase to the ferrichrome-iron receptor of <i>Escherichia coli</i> K-12.  <span class="journal">J. Bacteriol. 165:181–192.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Protein%20fusions%20of%20β-galactosidase%20to%20the%20ferrichrome-iron%20receptor%20of%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/151.pdf">pdf</a></td></tr>
<tr><td>152</td><td><a id="ref152" class="anchor"></a>Cox, E. C., and T. C. Gibson. <i>1974</i>. Selection for high mutation rates in chemostats.  <span class="journal">Genetics 77:169–184.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Selection%20for%20high%20mutation%20rates%20in%20chemostats">google scholar</a></td><td><a href="pdf/152.pdf">pdf</a></td></tr>
<tr><td>153</td><td><a id="ref153" class="anchor"></a>Crabeel, M., D. Charlier, R. Cunin, A. Bogen, N. Glansdorf, and A. Pierard. <i>1975</i>. Accumulation of arginine precursors in <i>Escherichia</i> coli: effects on growth, enzyme repression, and application to the forward selection of arginine auxotrophs.  <span class="journal">J. Bacteriol. 123:898–904.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Accumulation%20of%20arginine%20precursors%20in%20Escherichia%20coli:%20effects%20on%20growth,%20enzyme%20repression,%20and%20application%20to%20the%20forward%20selection%20of%20arginine%20auxotrophs">google scholar</a></td><td><a href="pdf/153.pdf">pdf</a></td></tr>
<tr><td>154</td><td><a id="ref154" class="anchor"></a>Creaghan, I. T., and J. R. Guest. <i>1977</i>. Suppression of the succinate requirement of lipoamide dehydrogenase mutants of <i>Escherichia coli</i> by mutations affecting succinate dehydrogenase activity.  <span class="journal">J. Gen. Microbiol. 102:183–194.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Suppression%20of%20the%20succinate%20requirement%20of%20lipoamide%20dehydrogenase%20mutants%20of%20Escherichia%20coli%20by%20mutations%20affecting%20succinate%20dehydrogenase%20activity">google scholar</a></td><td><a href="pdf/154.pdf">pdf</a></td></tr>
<tr><td>155</td><td><a id="ref155" class="anchor"></a>Csonka, L. N. <i>1981</i>. Proline over-production results in enhanced osmotolerance in <i>Salmonella</i> <i>typhimurium.</i>  <span class="journal">Mol. Gen. Genet. 182:82–86.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Proline%20over-production%20results%20in%20enhanced%20osmotolerance%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/155.pdf">pdf</a></td></tr>
<tr><td>156</td><td><a id="ref156" class="anchor"></a>Csonka, L. N. <i>1982</i>. A third L-proline permease in <i>Salmonella typhimurium</i> which functions in media of elevated osmotic strength.  <span class="journal">J. Bacteriol. 151:1433–1443.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20third%20L-proline%20permease%20in%20Salmonella%20typhimurium%20which%20functions%20in%20media%20of%20elevated%20osmotic%20strength">google scholar</a></td><td><a href="pdf/156.pdf">pdf</a></td></tr>
<tr><td>157</td><td><a id="ref157" class="anchor"></a>Cunningham, P. R., and D. P. Clark. <i>1986</i>. The use of suicide substrates to select mutants of <i>Escherichia coli</i> lacking enzymes of alcohol fermentation.  <span class="journal">Mol. Gen. Genet. 205:487–493.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The%20use%20of%20suicide%20substrates%20to%20select%20mutants%20of%20Escherichia%20coli%20lacking%20enzymes%20of%20alcohol%20fermentation">google scholar</a></td><td><a href="pdf/157.pdf">pdf</a></td></tr>
<tr><td>158</td><td><a id="ref158" class="anchor"></a>Curtis, S. J., and W. Epstein. <i>1975</i>. Phosphorylation of D-glucose in <i>Escherichia coli</i> mutants defective in glucosephosphotransferase, mannosephosphotransferase, and glucokinase.  <span class="journal">J. Bacteriol. 122:1189–1199.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Phosphorylation%20of%20D-glucose%20in%20Escherichia%20coli%20mutants%20defective%20in%20glucosephosphotransferase,%20mannosephosphotransferase,%20and%20glucokinase">google scholar</a></td><td><a href="pdf/158.pdf">pdf</a></td></tr>
<tr><td>159</td><td><a id="ref159" class="anchor"></a>Dabbs, E. R. <i>1978</i>. Kasugamycin-dependent mutants of <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 136:994– 1001.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Kasugamycin-dependent%20mutants%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/159.pdf">pdf</a></td></tr>
<tr><td>160</td><td><a id="ref160" class="anchor"></a>Dabbs, E. R. <i>1982</i>. Three additional loci of rifampicin dependence in <i>Escherichia</i> <i>coli.</i> <i>Mol. Gen.</i>  <span class="journal">Genet. 187:519–522.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Three%20additional%20loci%20of%20rifampicin%20dependence%20in%20Escherichia%20coli.%20Mol.%20Gen.">google scholar</a></td><td><a href="pdf/160.pdf">pdf</a></td></tr>
<tr><td>161</td><td><a id="ref161" class="anchor"></a>Dabbs, E. R. <i>1983</i>. <i>Escherichia coli</i> kasugamycin dependence arising from mutation at the <i>rpsI</i> locus.  <span class="journal">J. Bacteriol. 153:709–715.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Escherichia%20coli%20kasugamycin%20dependence%20arising%20from%20mutation%20at%20the%20rpsI%20locus">google scholar</a></td><td><a href="pdf/161.pdf">pdf</a></td></tr>
<tr><td>162</td><td><a id="ref162" class="anchor"></a>Dabbs, E. R., and K. Looman. <i>1981</i>. An antibiotic dependent conditional lethal mutant with a lesion affecting transcription and translation.  <span class="journal">Mol. Gen. Genet. 184:224–229.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=An%20antibiotic%20dependent%20conditional%20lethal%20mutant%20with%20a%20lesion%20affecting%20transcription%20and%20translation">google scholar</a></td><td><a href="pdf/162.pdf">pdf</a></td></tr>
<tr><td>163</td><td><a id="ref163" class="anchor"></a>Dabbs, E. R., B. Poldermans, H. Bakker, and P. H. Van Knippenberg. <i>1980</i>. Biochemical characterization of ribosomes of kasugamycin-dependent mutants of <i>Escherichia coli.</i>  <span class="journal">FEBS Lett. 117:164–166.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Biochemical%20characterization%20of%20ribosomes%20of%20kasugamycin-dependent%20mutants%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/163.pdf">pdf</a></td></tr>
<tr><td>164</td><td><a id="ref164" class="anchor"></a>Dame, J. B., and B. M. Shapiro. <i>1976</i>. Use of polymyxin B, levallorphan, and tetracaine to isolate novel mutants of <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 127:961–972.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Use%20of%20polymyxin%20B,%20levallorphan,%20and%20tetracaine%20to%20isolate%20novel%20mutants%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/164.pdf">pdf</a></td></tr>
<tr><td>165</td><td><a id="ref165" class="anchor"></a>Danchin, A. <i>1977</i>. A new technique for selection of sensitive and auxotrophic mutants of <i>E. coli:</i> isolation of a strain sensitive to an excess of one-carbon metabolites.  <span class="journal">Mol. Gen. Genet. 150:293–299.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20new%20technique%20for%20selection%20of%20sensitive%20and%20auxotrophic%20mutants%20of%20E.%20coli:%20isolation%20of%20a%20strain%20sensitive%20to%20an%20excess%20of%20one-carbon%20metabolites">google scholar</a></td><td><a href="pdf/165.pdf">pdf</a></td></tr>
<tr><td>166</td><td><a id="ref166" class="anchor"></a>Danchin, A., and L. Dondon. <i>1980</i>. Serine sensitivity of <i>Escherichia coli</i> K-12: partial characterization of a serine resistant mutant that is extremely sensitive to 2-ketobutyrate.  <span class="journal">Mol. Gen. Genet. 178:155–164.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Serine%20sensitivity%20of%20Escherichia%20coli%20K-12:%20partial%20characterization%20of%20a%20serine%20resistant%20mutant%20that%20is%20extremely%20sensitive%20to%202-ketobutyrate">google scholar</a></td><td><a href="pdf/166.pdf">pdf</a></td></tr>
<tr><td>167</td><td><a id="ref167" class="anchor"></a>Daniel, J., and A. Danchin. <i>1979</i>. Involvement of cyclic AMP and its receptor protein in the sensitivity of <i>Escherichia coli</i> K12 toward serine.  <span class="journal">Mol. Gen. Genet. 176:343–350.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Involvement%20of%20cyclic%20AMP%20and%20its%20receptor%20protein%20in%20the%20sensitivity%20of%20Escherichia%20coli%20K12%20toward%20serine">google scholar</a></td><td><a href="pdf/167.pdf">pdf</a></td></tr>
<tr><td>168</td><td><a id="ref168" class="anchor"></a>D’Ari, R., A. Jaffé, P. Bouloc, and A. Robin. <i>1988</i>. Cyclic AMP and cell division in <i>Escherichia</i> <i>coli.</i>  <span class="journal">J. Bacteriol. 170:65–70.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Cyclic%20AMP%20and%20cell%20division%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/168.pdf">pdf</a></td></tr>
<tr><td>169</td><td><a id="ref169" class="anchor"></a>Das, A., D. Court, and S. Adhya. <i>1976</i>. Isolation and characteristics of conditional-lethal mutants of <i>Escherichia coli</i> defective in transcription termination factor rho.  <span class="journal">Proc. Natl. Acad. Sci. USA 73:1959–1963.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20and%20characteristics%20of%20conditional-lethal%20mutants%20of%20Escherichia%20coli%20defective%20in%20transcription%20termination%20factor%20rho">google scholar</a></td><td><a href="pdf/169.pdf">pdf</a></td></tr>
<tr><td>170</td><td><a id="ref170" class="anchor"></a>Das, A., B. Ghosh, S. Barik, and K. Wolska. <i>1985</i>. Evidence that ribosomal protein S-10 itself is a cellular component necessary for transcription antitermination by phage lambda N protein.  <span class="journal">Proc. Natl. Acad. Sci. USA 82:4070–4074.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Evidence%20that%20ribosomal%20protein%20S-10%20itself%20is%20a%20cellular%20component%20necessary%20for%20transcription%20antitermination%20by%20phage%20lambda%20N%20protein">google scholar</a></td><td><a href="pdf/170.pdf">pdf</a></td></tr>
<tr><td>171</td><td><a id="ref171" class="anchor"></a>Datta, D. B., B. Arden, and U. Henning. <i>1977</i>. Major proteins of the <i>Escherichia coli</i> outer cell envelope membrane as bacteriophage receptors.  <span class="journal">J. Bacteriol. 131:821–829.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Major%20proteins%20of%20the%20Escherichia%20coli%20outer%20cell%20envelope%20membrane%20as%20bacteriophage%20receptors">google scholar</a></td><td><a href="pdf/171.pdf">pdf</a></td></tr>
<tr><td>172</td><td><a id="ref172" class="anchor"></a>Davies, J., P. Anderson, and B. D. Davis. <i>1965</i>. Inhibition of protein synthesis by spectinomycin.  <span class="journal">Science 149:1096–1098.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Inhibition%20of%20protein%20synthesis%20by%20spectinomycin">google scholar</a></td><td><a href="pdf/172.pdf">pdf</a></td></tr>
<tr><td>173</td><td><a id="ref173" class="anchor"></a>Davies, J. K., and P. Reeves. <i>1975</i>. Genetics of resistance to colicins in <i>Escherichia coli</i> K-12: cross-resistance among colicins of group A.  <span class="journal">J. Bacteriol. 123:102–117.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetics%20of%20resistance%20to%20colicins%20in%20Escherichia%20coli%20K-12:%20cross-resistance%20among%20colicins%20of%20group%20A">google scholar</a></td><td><a href="pdf/173.pdf">pdf</a></td></tr>
<tr><td>174</td><td><a id="ref174" class="anchor"></a>Davis, E. J., J. M. Blatt, E. K. Henderson, J. J. Whittaker, and J. H. Jackson. <i>1977</i>. Valine- sensitive acetohydroxy acid synthetases in <i>Escherichia coli</i> K-12: unique regulation modulated by multiple genetic sites.  <span class="journal">Mol. Gen. Genet. 156:239–249.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Valine-%20sensitive%20acetohydroxy%20acid%20synthetases%20in%20Escherichia%20coli%20K-12:%20unique%20regulation%20modulated%20by%20multiple%20genetic%20sites">google scholar</a></td><td><a href="pdf/174.pdf">pdf</a></td></tr>
<tr><td>175</td><td><a id="ref175" class="anchor"></a>Davis, E. O., M. J. Jones-Mortimer, and P. J. F. Henderson. <i>1984</i>. Location of a structural gene for xylose proton symport at 91 minutes on the linkage map of <i>Escherichia coli</i> K12.  <span class="journal">J. Biol. Chem. 259:1520–1525.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Location%20of%20a%20structural%20gene%20for%20xylose%20proton%20symport%20at%2091%20minutes%20on%20the%20linkage%20map%20of%20Escherichia%20coli%20K12">google scholar</a></td><td><a href="pdf/175.pdf">pdf</a></td></tr>
<tr><td>176</td><td><a id="ref176" class="anchor"></a>Daws, T. D., and J. A. Fuchs. <i>1984</i>. Isolation and characterization of an <i>Escherichia</i> <i>coli</i> mutant deficient in dTMP kinase activity.  <span class="journal">J. Bacteriol. 157:440–444.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20and%20characterization%20of%20an%20Escherichia%20coli%20mutant%20deficient%20in%20dTMP%20kinase%20activity">google scholar</a></td><td><a href="pdf/176.pdf">pdf</a></td></tr>
<tr><td>177</td><td><a id="ref177" class="anchor"></a>Dean, D. A., J. Reizer, H. Nikaido, and M. H. Saier, Jr. <i>1990</i>. Regulation of the maltose transport system of <i>Escherichia</i> <i>coli</i> by the glucose-specific enzyme III of the phosphoenolpyruvate- sugar phosphotransferase system. Characterization of inducer exclusion resistant mutants and reconstitution of inducer exlusion in proteoliposomes.  <span class="journal">J. Biol. Chem. 265:21005–21010.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation%20of%20the%20maltose%20transport%20system%20of%20Escherichia%20coli%20by%20the%20glucose-specific%20enzyme%20III%20of%20the%20phosphoenolpyruvate-%20sugar%20phosphotransferase%20system.%20Characterization%20of%20inducer%20exclusion%20resistant%20mutants%20and%20reconstitution%20of%20inducer%20exlusion%20in%20proteoliposomes">google scholar</a></td><td><a href="pdf/177.pdf">pdf</a></td></tr>
<tr><td>178</td><td><a id="ref178" class="anchor"></a>DeFelice, M., J. Guardiola, A. Lamberti, and M. Iaccarino. <i>1973</i>. <i>Escherichia coli</i> K-12 mutants altered in the transport systems for oligo- and dipeptides.  <span class="journal">J. Bacteriol. 116:751–756.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Escherichia%20coli%20K-12%20mutants%20altered%20in%20the%20transport%20systems%20for%20oligo-%20and%20dipeptides">google scholar</a></td><td><a href="pdf/178.pdf">pdf</a></td></tr>
<tr><td>179</td><td><a id="ref179" class="anchor"></a>Defez, R., and M. DeFelice. <i>1981</i>. Cryptic operon for β-glucoside metabolism in <i>Escherichia coli</i> K12: genetic evidence for a regulatory protein.  <span class="journal">Genetics 97:11–25.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Cryptic%20operon%20for%20β-glucoside%20metabolism%20in%20Escherichia%20coli%20K12:%20genetic%20evidence%20for%20a%20regulatory%20protein">google scholar</a></td><td><a href="pdf/179.pdf">pdf</a></td></tr>
<tr><td>180</td><td><a id="ref180" class="anchor"></a>Del Campillo-Campbell, A., and A. Campbell. <i>1982</i>. Molybdenum cofactor requirement for biotin sulfoxide reduction in <i>Escherichia coli</i>.  <span class="journal">J. Bacteriol. 149:469–478.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Molybdenum%20cofactor%20requirement%20for%20biotin%20sulfoxide%20reduction%20in%20Escherichia%20coli">google scholar</a></td><td><a href="pdf/180.pdf">pdf</a></td></tr>
<tr><td>181</td><td><a id="ref181" class="anchor"></a>Del Castillo, I., J. L. Vizan, M. Del Carmen Rodriguez-Sainz, and F. Moreno. <i>1991</i>. An unusual mechanism for resistance to the antibiotic coumermycin A-1.  <span class="journal">Proc. Natl. Acad. Sci. USA 88:8860–8864.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=An%20unusual%20mechanism%20for%20resistance%20to%20the%20antibiotic%20coumermycin%20A-1">google scholar</a></td><td><a href="pdf/181.pdf">pdf</a></td></tr>
<tr><td>182</td><td><a id="ref182" class="anchor"></a>Delcuve, G., T. Cabezon, A. Herzog, M. Cannon, and A. Bollen. <i>1978</i>. Resistance to the aminoglycoside antibiotic neamine in <i>Escherichia</i> <i>coli.</i> A new mutant whose NeaR phenotype results from the cumulative effects of two distinct mutations.  <span class="journal">Biochem. J. 174:1–7.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Resistance%20to%20the%20aminoglycoside%20antibiotic%20neamine%20in%20Escherichia%20coli.%20A%20new%20mutant%20whose%20NeaR%20phenotype%20results%20from%20the%20cumulative%20effects%20of%20two%20distinct%20mutations">google scholar</a></td><td><a href="pdf/182.pdf">pdf</a></td></tr>
<tr><td>183</td><td><a id="ref183" class="anchor"></a>DelGiudice, L. <i>1979</i>. Method for isolating restriction- and modification-less mutants of <i>Escherichia coli</i> K-12.  <span class="journal">J. Bacteriol. 137:673–676.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Method%20for%20isolating%20restriction-%20and%20modification-less%20mutants%20of%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/183.pdf">pdf</a></td></tr>
<tr><td>184</td><td><a id="ref184" class="anchor"></a>Delidakis, C. E., M. C. Jones-Mortimer, and H. L. Kornberg. <i>1982</i>. A mutant inducible for galactitol utilization in <i>Escherichia coli</i> K12.  <span class="journal">J. Gen. Microbiol. 128:601–604.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20mutant%20inducible%20for%20galactitol%20utilization%20in%20Escherichia%20coli%20K12">google scholar</a></td><td><a href="pdf/184.pdf">pdf</a></td></tr>
<tr><td>185</td><td><a id="ref185" class="anchor"></a>Delvaux, A. M., and R. Devoret. <i>1969</i>. Occurrence of suppressors in caffeine-resistant mutants from <i>Escherichia coli</i> K12.  <span class="journal">Mutat. Res. 7:273–285.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Occurrence%20of%20suppressors%20in%20caffeine-resistant%20mutants%20from%20Escherichia%20coli%20K12">google scholar</a></td><td><a href="pdf/185.pdf">pdf</a></td></tr>
<tr><td>186</td><td><a id="ref186" class="anchor"></a>Dempsey, W. B., and L. J. Arcement. <i>1971</i>. Identification of the forms of vitamin B6 present in the culture media of “vitamin B6 control” mutants.  <span class="journal">J. Bacteriol. 107:580-582.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Identification%20of%20the%20forms%20of%20vitamin%20B6%20present%20in%20the%20culture%20media%20of%20“vitamin%20B6%20control”%20mutants">google scholar</a></td><td><a href="pdf/186.pdf">pdf</a></td></tr>
<tr><td>187</td><td><a id="ref187" class="anchor"></a>Dendinger, S., and W. J. Brill. <i>1972</i>. Effect of the proline analogue baikiain on proline metabolism in <i>Salmonella typhimurium.</i>  <span class="journal">J. Bacteriol. 112:1134–1141.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Effect%20of%20the%20proline%20analogue%20baikiain%20on%20proline%20metabolism%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/187.pdf">pdf</a></td></tr>
<tr><td>188</td><td><a id="ref188" class="anchor"></a>Dendinger, S. M., L. G. Patil, and J. E. Brenchley. <i>1980</i>. <i>Salmonella typhimurium</i> mutants with altered glutamate dehydrogenase and glutamate synthase activities.  <span class="journal">J. Bacteriol. 141:190–198.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Salmonella%20typhimurium%20mutants%20with%20altered%20glutamate%20dehydrogenase%20and%20glutamate%20synthase%20activities">google scholar</a></td><td><a href="pdf/188.pdf">pdf</a></td></tr>
<tr><td>189</td><td><a id="ref189" class="anchor"></a>Desrochers, M., L. Peloquin, and A. Sasarman. <i>1978</i>. Mapping of the <i>hemE</i> locus in <i>Salmonella</i> <i>typhimurium.</i>  <span class="journal">J. Bacteriol. 135:1151–1153.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mapping%20of%20the%20hemE%20locus%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/189.pdf">pdf</a></td></tr>
<tr><td>190</td><td><a id="ref190" class="anchor"></a>Devoret, R., and M. Blanco. <i>1970</i>. Mutants of <i>Escherichia coli</i> K12 λ+ noninducible by thymine deprivation. I. Method of isolation and classes of mutants obtained.  <span class="journal">Mol. Gen. Genet. 107:272–280.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutants%20of%20Escherichia%20coli%20K12%20λ+%20noninducible%20by%20thymine%20deprivation.%20I.%20Method%20of%20isolation%20and%20classes%20of%20mutants%20obtained">google scholar</a></td><td><a href="pdf/190.pdf">pdf</a></td></tr>
<tr><td>191</td><td><a id="ref191" class="anchor"></a>Dietz, G. W., Jr. <i>1978</i>. Growth of <i>Escherichia coli</i> on glucosamine 6-phosphate selection of a constitutive hexose phosphate transport system mutant.  <span class="journal">Can. J. Microbiol. 24:203–208.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Growth%20of%20Escherichia%20coli%20on%20glucosamine%206-phosphate%20selection%20of%20a%20constitutive%20hexose%20phosphate%20transport%20system%20mutant">google scholar</a></td><td><a href="pdf/191.pdf">pdf</a></td></tr>
<tr><td>192</td><td><a id="ref192" class="anchor"></a>Dietz, G. W., and L. A. Heppel. <i>1971</i>. Studies on the uptake of hexose phosphates. III. Mechanism of uptake of glucose 1-phosphate in  <span class="journal">Escherichia coli. J. Biol. Chem. 246:2891–2897.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Studies%20on%20the%20uptake%20of%20hexose%20phosphates.%20III.%20Mechanism%20of%20uptake%20of%20glucose%201-phosphate%20in">google scholar</a></td><td><a href="pdf/192.pdf">pdf</a></td></tr>
<tr><td>193</td><td><a id="ref193" class="anchor"></a>Dila, D. K., and S. R. Maloy. <i>1986</i>. Proline transport in <i>Salmonella typhimurium</i> <i>putP</i> permease mutants with altered substrate specificity.  <span class="journal">J. Bacteriol. 168:590–594.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Proline%20transport%20in%20Salmonella%20typhimurium%20putP%20permease%20mutants%20with%20altered%20substrate%20specificity">google scholar</a></td><td><a href="pdf/193.pdf">pdf</a></td></tr>
<tr><td>194</td><td><a id="ref194" class="anchor"></a>DiNardo, S., K. A. Voelkel, R. Sternglanz, A. E. Reynolds, and A. Wright. <i>1982</i>. <i>Escherichia</i> <i>coli</i> DNA topoisomerase I mutants have compensatory mutations in DNA gyrase genes.  <span class="journal">Cell 31:43– 51.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Escherichia%20coli%20DNA%20topoisomerase%20I%20mutants%20have%20compensatory%20mutations%20in%20DNA%20gyrase%20genes">google scholar</a></td><td><a href="pdf/194.pdf">pdf</a></td></tr>
<tr><td>195</td><td><a id="ref195" class="anchor"></a>Dombou, M., T. Mizuno, and S. Mizushima. <i>1977</i>. Interaction of the cytoplasmic membrane and ribosomes in <i>Escherichia coli</i> altered ribosomal proteins in sucrose-dependent spectinomycin- resistant mutants.  <span class="journal">Mol. Gen. Genet. 155:53–60.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Interaction%20of%20the%20cytoplasmic%20membrane%20and%20ribosomes%20in%20Escherichia%20coli%20altered%20ribosomal%20proteins%20in%20sucrose-dependent%20spectinomycin-%20resistant%20mutants">google scholar</a></td><td><a href="pdf/195.pdf">pdf</a></td></tr>
<tr><td>196</td><td><a id="ref196" class="anchor"></a>Doskocil, J., and F. Sorm. <i>1970</i>. The mode of action of 5-aza-2′-deoxycytidine in <i>Escherichia</i> <i>coli.</i>  <span class="journal">Eur. J. Biochem. 13:180–187.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The%20mode%20of%20action%20of%205-aza-2′-deoxycytidine%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/196.pdf">pdf</a></td></tr>
<tr><td>197</td><td><a id="ref197" class="anchor"></a>Dover, S., and Y. S. Halpern. <i>1974</i>. Genetic analysis of the γ-aminobutyrate utilization pathway in <i>Escherichia coli</i> K-12.  <span class="journal">J. Bacteriol. 117:494–501.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20analysis%20of%20the%20γ-aminobutyrate%20utilization%20pathway%20in%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/197.pdf">pdf</a></td></tr>
<tr><td>198</td><td><a id="ref198" class="anchor"></a>Downs, D., L. Waxman, A. L. Goldberg, and J. Roth. <i>1986</i>. Isolation and characterization of <i>lon</i> mutants in <i>Salmonella</i> <i>typhimurium.</i>  <span class="journal">J. Bacteriol. 165:193–197.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20and%20characterization%20of%20lon%20mutants%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/198.pdf">pdf</a></td></tr>
<tr><td>199</td><td><a id="ref199" class="anchor"></a>Drexler, K., I. Riede, D. Montag, M.-L. Eschbach, and U. Henning. <i>1989</i>. Receptor specificity of the <i>Escherichia coli</i> T-even type phage Ox2: mutational alterations in host range mutants.  <span class="journal">J. Mol. Biol. 207:797–804.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Receptor%20specificity%20of%20the%20Escherichia%20coli%20T-even%20type%20phage%20Ox2:%20mutational%20alterations%20in%20host%20range%20mutants">google scholar</a></td><td><a href="pdf/199.pdf">pdf</a></td></tr>
<tr><td>200</td><td><a id="ref200" class="anchor"></a>Droffner, M. L., and N. Yamamoto. <i>1992</i>. Role of nalidixic acid in isolation of <i>Salmonella</i> <i>typhimurium</i> strains capable of growth at 48°C.  <span class="journal">Curr. Microbiol. 25:257–260.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Role%20of%20nalidixic%20acid%20in%20isolation%20of%20Salmonella%20typhimurium%20strains%20capable%20of%20growth%20at%2048°C">google scholar</a></td><td><a href="pdf/200.pdf">pdf</a></td></tr>
<tr><td>201</td><td><a id="ref201" class="anchor"></a>Drury, L. S., and R. S. Buxton. <i>1988</i>. Identification and sequencing of the <i>Escherichia</i> <i>coli cet</i> gene which codes for an inner membrane protein, mutation of which causes tolerance to colicin E2.  <span class="journal">Mol. Microbiol. 2:109–119.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Identification%20and%20sequencing%20of%20the%20Escherichia%20coli%20cet%20gene%20which%20codes%20for%20an%20inner%20membrane%20protein,%20mutation%20of%20which%20causes%20tolerance%20to%20colicin%20E2">google scholar</a></td><td><a href="pdf/201.pdf">pdf</a></td></tr>
<tr><td>202</td><td><a id="ref202" class="anchor"></a>Dubnau, E., and P. Margolin. <i>1972</i>. Suppression of promoter mutations by the pleiotrophic <i>supX</i> mutations.  <span class="journal">Mol. Gen. Genet. 117:91–112.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Suppression%20of%20promoter%20mutations%20by%20the%20pleiotrophic%20supX%20mutations">google scholar</a></td><td><a href="pdf/202.pdf">pdf</a></td></tr>
<tr><td>203</td><td><a id="ref203" class="anchor"></a>Duncan, B. K., P. A. Rockstroh, and H. R. Warner. <i>1978</i>. <i>Escherichia coli</i> K-12 mutants deficient in uracil-DNA glycosylase.  <span class="journal">J. Bacteriol. 134:1039–1045.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Escherichia%20coli%20K-12%20mutants%20deficient%20in%20uracil-DNA%20glycosylase">google scholar</a></td><td><a href="pdf/203.pdf">pdf</a></td></tr>
<tr><td>204</td><td><a id="ref204" class="anchor"></a>Echols, H., A. Garen, S. Garen, and A. Torriani. <i>1961</i>. Genetic control of repression of alkaline phosphatase in <i>E. coli.</i>  <span class="journal">J. Mol. Biol. 3:425–438.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20control%20of%20repression%20of%20alkaline%20phosphatase%20in%20E.%20coli.">google scholar</a></td><td><a href="pdf/204.pdf">pdf</a></td></tr>
<tr><td>205</td><td><a id="ref205" class="anchor"></a>Eick-Helmerich, K., and V. Braun. <i>1989</i>. Import of biopolymers into <i>Escherichia coli</i> nucleotide sequences of the <i>exbB</i> and <i>exbD</i> genes are homologous to those of the <i>tolQ</i> and <i>tolR</i> genes, respectively.  <span class="journal">J. Bacteriol. 171:5117–5126.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Import%20of%20biopolymers%20into%20Escherichia%20coli%20nucleotide%20sequences%20of%20the%20exbB%20and%20exbD%20genes%20are%20homologous%20to%20those%20of%20the%20tolQ%20and%20tolR%20genes,%20respectively">google scholar</a></td><td><a href="pdf/205.pdf">pdf</a></td></tr>
<tr><td>206</td><td><a id="ref206" class="anchor"></a>Eisenberg, M. A., B. Mee, O. Prakash, and M. R. Eisenberg. <i>1975</i>. Properties of α- dehydrobiotin-resistant mutant of <i>Escherichia</i> <i>coli</i> K-12.  <span class="journal">J. Bacteriol. 122:66–72.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Properties%20of%20α-%20dehydrobiotin-resistant%20mutant%20of%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/206.pdf">pdf</a></td></tr>
<tr><td>207</td><td><a id="ref207" class="anchor"></a>Elwell, L. P., R. Ferone, G. A. Freeman, J. A. Fyfe, J. A. Hill, P. H. Ray, C. A. Richards, S. C. Singer, V. B. Knick, J. L. Rideout, and T. P. Zimmerman. <i>1987</i>. Antibacterial activity and mechanism of action of 3′ azido-3′-deoxythymidine BWA-509U.  <span class="journal">Antimicrob. Agents Chemother. 31:274–280.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Antibacterial%20activity%20and%20mechanism%20of%20action%20of%203′%20azido-3′-deoxythymidine%20BWA-509U">google scholar</a></td><td><a href="pdf/207.pdf">pdf</a></td></tr>
<tr><td>208</td><td><a id="ref208" class="anchor"></a>Ely, B., D. B. Fankhauser, and P. E. Hartman. <i>1974</i>. A fine structure map of the <i>Salmonella</i> histidine operator-promoter.  <span class="journal">Genetics 78:607–631.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20fine%20structure%20map%20of%20the%20Salmonella%20histidine%20operator-promoter">google scholar</a></td><td><a href="pdf/208.pdf">pdf</a></td></tr>
<tr><td>209</td><td><a id="ref209" class="anchor"></a>Englesberg, E., J. Irr, J. Power, and N. Lee. <i>1965</i>. Positive control of enzyme synthesis by gene C in the L-arabinose system.  <span class="journal">J. Bacteriol. 90:946–957.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Positive%20control%20of%20enzyme%20synthesis%20by%20gene%20C%20in%20the%20L-arabinose%20system">google scholar</a></td><td><a href="pdf/209.pdf">pdf</a></td></tr>
<tr><td>210</td><td><a id="ref210" class="anchor"></a>Eriksson-Grennberg, K. G., H. G. Boman, J. A. T. Jansson, and S. Thoren. <i>1965</i>. Resistance of <i>Escherichia coli</i> to penicillins. I. Genetic study of some ampicillin-resistant mutants.  <span class="journal">J. Bacteriol. 90:54–62.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Resistance%20of%20Escherichia%20coli%20to%20penicillins.%20I.%20Genetic%20study%20of%20some%20ampicillin-resistant%20mutants">google scholar</a></td><td><a href="pdf/210.pdf">pdf</a></td></tr>
<tr><td>211</td><td><a id="ref211" class="anchor"></a>Eriksson-Grennberg, K. G., and K. Nordstrom. <i>1973</i>. Genetics and physiology of a <i>tolE</i> mutant of <i>Escherichia coli</i> K-12 and phenotypic suppression of its phenotype by galactose.  <span class="journal">J. Bacteriol. 115:1219–1222.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetics%20and%20physiology%20of%20a%20tolE%20mutant%20of%20Escherichia%20coli%20K-12%20and%20phenotypic%20suppression%20of%20its%20phenotype%20by%20galactose">google scholar</a></td><td><a href="pdf/211.pdf">pdf</a></td></tr>
<tr><td>212</td><td><a id="ref212" class="anchor"></a>Eriksson-Grennberg, K. G., K. Nordstrom, and P. Englund. <i>1971</i>. Resistance of <i>Escherichia</i> <i>coli</i> to penicillins. IX. Genetics and physiology of class II ampicillin-resistant mutants that are galactose negative or sensitive to bacteriophage C21, or both.  <span class="journal">J. Bacteriol. 108:1210–1223.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Resistance%20of%20Escherichia%20coli%20to%20penicillins.%20IX.%20Genetics%20and%20physiology%20of%20class%20II%20ampicillin-resistant%20mutants%20that%20are%20galactose%20negative%20or%20sensitive%20to%20bacteriophage%20C21,%20or%20both">google scholar</a></td><td><a href="pdf/212.pdf">pdf</a></td></tr>
<tr><td>213</td><td><a id="ref213" class="anchor"></a>Essenberg, R. C. <i>1984</i>. Use of homocysteic acid for selecting mutants at the <i>gltS</i> locus of <i>Escherichia coli</i> K12.  <span class="journal">J. Gen. Microbiol. 130:1311–1314.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Use%20of%20homocysteic%20acid%20for%20selecting%20mutants%20at%20the%20gltS%20locus%20of%20Escherichia%20coli%20K12">google scholar</a></td><td><a href="pdf/213.pdf">pdf</a></td></tr>
<tr><td>214</td><td><a id="ref214" class="anchor"></a>Essenberg, R. C., and H. L. Kornberg. <i>1977</i>. Location of the gene for hexose phosphate transport <i>(uhp)</i> on the chromosome of <i>Escherichia coli.</i>  <span class="journal">J. Gen. Microbiol. 99:157–169.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Location%20of%20the%20gene%20for%20hexose%20phosphate%20transport%20(uhp)%20on%20the%20chromosome%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/214.pdf">pdf</a></td></tr>
<tr><td>215</td><td><a id="ref215" class="anchor"></a>Ezekiel, D. H. <i>1965</i>. False feedback inhibition of aromatic amino acid biosynthesis by β-2-thienyl- alanine.  <span class="journal">Biochim. Biophys. Acta 95:54–62.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=False%20feedback%20inhibition%20of%20aromatic%20amino%20acid%20biosynthesis%20by%20β-2-thienyl-%20alanine">google scholar</a></td><td><a href="pdf/215.pdf">pdf</a></td></tr>
<tr><td>216</td><td><a id="ref216" class="anchor"></a>Ezekiel, D., and J. E. Hutchins. <i>1968</i>. Mutations affecting RNA polymerase associated with rifampicin resistance in <i>Escherichia coli.</i>  <span class="journal">Nature  220:276–277.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutations%20affecting%20RNA%20polymerase%20associated%20with%20rifampicin%20resistance%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/216.pdf">pdf</a></td></tr>
<tr><td>217</td><td><a id="ref217" class="anchor"></a>Faik, P., and H. L. Kornberg. <i>1973</i>. Isolation and properties of <i>E. coli</i> mutants affected in gluconate uptake.  <span class="journal">FEBS Lett. 32:260–263.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20and%20properties%20of%20E.%20coli%20mutants%20affected%20in%20gluconate%20uptake">google scholar</a></td><td><a href="pdf/217.pdf">pdf</a></td></tr>
<tr><td>218</td><td><a id="ref218" class="anchor"></a>Fangman, W. L., and F. C. Neidhardt. <i>1964</i>. Demonstration of an altered aminoacyl ribonucleic acid synthetase in a mutant of <i>Escherichia coli.</i>  <span class="journal">J. Biol. Chem. 239:1839–1843.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Demonstration%20of%20an%20altered%20aminoacyl%20ribonucleic%20acid%20synthetase%20in%20a%20mutant%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/218.pdf">pdf</a></td></tr>
<tr><td>219</td><td><a id="ref219" class="anchor"></a>Farley, M. M., W. M. Shafer, and J. K. Spitznagel. <i>1988</i>. Lipopolysaccharide structure determines ionic and hydrophobic binding of a cationic antimicrobial neutrophil granule protein.  <span class="journal">Infect. Immun. 56:1589–1592.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Lipopolysaccharide%20structure%20determines%20ionic%20and%20hydrophobic%20binding%20of%20a%20cationic%20antimicrobial%20neutrophil%20granule%20protein">google scholar</a></td><td><a href="pdf/219.pdf">pdf</a></td></tr>
<tr><td>220</td><td><a id="ref220" class="anchor"></a>Fasano, O., and A. Parmeggiani. <i>1981</i>. Altered regulation of the GTPase activity in a kirromycin resistant elongation factor Tu.  <span class="journal">Biochemistry 20:1361–1366.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Altered%20regulation%20of%20the%20GTPase%20activity%20in%20a%20kirromycin%20resistant%20elongation%20factor%20Tu">google scholar</a></td><td><a href="pdf/220.pdf">pdf</a></td></tr>
<tr><td>221</td><td><a id="ref221" class="anchor"></a>Favre, R., A. Wiater, S. Puppo, and M. Iaccarino. <i>1976</i>. Expression of a valine resistant acetolactate synthase activity mediated by the <i>ilvO</i> and <i>ilvG</i> genes of <i>Escherichia coli</i> K12.  <span class="journal">Mol. Gen. Genet. 143:243–252.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Expression%20of%20a%20valine%20resistant%20acetolactate%20synthase%20activity%20mediated%20by%20the%20ilvO%20and%20ilvG%20genes%20of%20Escherichia%20coli%20K12">google scholar</a></td><td><a href="pdf/221.pdf">pdf</a></td></tr>
<tr><td>222</td><td><a id="ref222" class="anchor"></a>Fayerman, J. T., M. C. Vann, L S. Williams, and H. E. Umbarger. <i>1979</i>. <i>ilvU,</i> a locus in <i>Escherichia coli</i> affecting the derepression of isoleucyl-tRNA synthetase and the RPC-5 chromatographic profiles of tRNAile and tRNAval.  <span class="journal">J. Biol. Chem. 254:9429–9440.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=ilvU,%20a%20locus%20in%20Escherichia%20coli%20affecting%20the%20derepression%20of%20isoleucyl-tRNA%20synthetase%20and%20the%20RPC-5%20chromatographic%20profiles%20of%20tRNAile%20and%20tRNAval">google scholar</a></td><td><a href="pdf/222.pdf">pdf</a></td></tr>
<tr><td>223</td><td><a id="ref223" class="anchor"></a>Ferenci, T. <i>1980</i>. Methyl-α-maltoside and 5-thiomaltose: analogs transported by the <i>Escherichia</i> <i>coli</i> maltose transport system.  <span class="journal">J. Bacteriol. 144:7–11.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Methyl-α-maltoside%20and%205-thiomaltose:%20analogs%20transported%20by%20the%20Escherichia%20coli%20maltose%20transport%20system">google scholar</a></td><td><a href="pdf/223.pdf">pdf</a></td></tr>
<tr><td>224</td><td><a id="ref224" class="anchor"></a>Ferenci, T., H. L. Kornberg, and J. Smith. <i>1971</i>. Isolation and properties of a regulatory mutant in the hexose phosphate transport system of <i>Escherichia coli.</i>  <span class="journal">FEBS Lett. 13:133–136.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20and%20properties%20of%20a%20regulatory%20mutant%20in%20the%20hexose%20phosphate%20transport%20system%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/224.pdf">pdf</a></td></tr>
<tr><td>225</td><td><a id="ref225" class="anchor"></a>Fillingame, R. H. <i>1975</i>. Identification of the dicyclohexylcarbodiimide-reactive protein component of the adenosine 5′-triphosphate transducing system of <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 124:870–883.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Identification%20of%20the%20dicyclohexylcarbodiimide-reactive%20protein%20component%20of%20the%20adenosine%205′-triphosphate%20transducing%20system%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/225.pdf">pdf</a></td></tr>
<tr><td>226</td><td><a id="ref226" class="anchor"></a>Fillingame, R. H., M. Oldenburg, and D. Fraga. <i>1991</i>. Mutation of alanine-24 to serine in subunit C of the <i>Escherichia coli</i> F-1 F-O ATP synthase reduces reactivity of aspartyl 61 with dicyclohexylcarbodiimide.  <span class="journal">J. Biol. Chem. 266:20934–20939.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutation%20of%20alanine-24%20to%20serine%20in%20subunit%20C%20of%20the%20Escherichia%20coli%20F-1%20F-O%20ATP%20synthase%20reduces%20reactivity%20of%20aspartyl%2061%20with%20dicyclohexylcarbodiimide">google scholar</a></td><td><a href="pdf/226.pdf">pdf</a></td></tr>
<tr><td>227</td><td><a id="ref227" class="anchor"></a>Filutowicz, M., Z. Ciesla, and T. Klopotowski. <i>1979</i>. Interference of azide with cysteine biosynthesis in <i>Salmonella typhimurium.</i>  <span class="journal">J. Gen. Microbiol. 113:45–55.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Interference%20of%20azide%20with%20cysteine%20biosynthesis%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/227.pdf">pdf</a></td></tr>
<tr><td>228</td><td><a id="ref228" class="anchor"></a>Fimmel, A., and B. Haddock. <i>1979</i>. Use of <i>chlC-lac</i> fusions to determine regulation of gene <i>chlC</i> in <i>Escherichia coli</i> K-12.  <span class="journal">J. Bacteriol. 138:726–730.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Use%20of%20chlC-lac%20fusions%20to%20determine%20regulation%20of%20gene%20chlC%20in%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/228.pdf">pdf</a></td></tr>
<tr><td>229</td><td><a id="ref229" class="anchor"></a>Fimmel, A. L., D. A. Jans, L. Hatch, L. B. James, F. Gibson, and G. B. Cox. <i>1985</i>. The F-1 F-0 ATPase of <i>Escherichia coli</i> : the substitution of alanine by threonine at position 25 in the C-subunit affects function but not assembly.  <span class="journal">Biochim. Biophys. Acta 808:252–258.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The%20F-1%20F-0%20ATPase%20of%20Escherichia%20coli%20:%20the%20substitution%20of%20alanine%20by%20threonine%20at%20position%2025%20in%20the%20C-subunit%20affects%20function%20but%20not%20assembly">google scholar</a></td><td><a href="pdf/229.pdf">pdf</a></td></tr>
<tr><td>230</td><td><a id="ref230" class="anchor"></a>Fimmel, A. L., and R. E. Loughlin. <i>1977</i>. Isolation and characterization of <i>cysK</i> mutants of <i>Escherichia coli.</i>  <span class="journal">J. Gen. Microbiol. 103:37–43.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20and%20characterization%20of%20cysK%20mutants%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/230.pdf">pdf</a></td></tr>
<tr><td>231</td><td><a id="ref231" class="anchor"></a>Fisher, K. E., and E. Eisenstein. <i>1993</i>. An efficient approach to identify <i>ilvA</i> mutations reveals an amino-terminal catalytic domain in biosynthetic threonine deaminase from <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 175:6605–6613.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=An%20efficient%20approach%20to%20identify%20ilvA%20mutations%20reveals%20an%20amino-terminal%20catalytic%20domain%20in%20biosynthetic%20threonine%20deaminase%20from%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/231.pdf">pdf</a></td></tr>
<tr><td>232</td><td><a id="ref232" class="anchor"></a>Fortin, T., P. Phoenix, and G. R. Drapeau. <i>1990</i>. Mutations conferring resistance to azide in <i>Escherichia coli</i> occur primarily in the <i>secA</i> gene.  <span class="journal">J. Bacteriol. 172:6607–6610.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutations%20conferring%20resistance%20to%20azide%20in%20Escherichia%20coli%20occur%20primarily%20in%20the%20secA%20gene">google scholar</a></td><td><a href="pdf/232.pdf">pdf</a></td></tr>
<tr><td>233</td><td><a id="ref233" class="anchor"></a>Foster, J. W., and H. K. Hall. <i>1990</i>. Adaptive acidification tolerance response of <i>Salmonella</i> <i>typhimurium.</i>  <span class="journal">J. Bacteriol. 172:771–778.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Adaptive%20acidification%20tolerance%20response%20of%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/233.pdf">pdf</a></td></tr>
<tr><td>234</td><td><a id="ref234" class="anchor"></a>Foster, J. W., D. M. Kinney, and A. G. Moat. <i>1979</i>. Pyridine nucleotide cycle of <i>Salmonella</i> <i>typhimurium:</i> isolation and characterization of <i>pncA, pncB,</i> and <i>pncC</i> mutants and utilization of exogenous nicotinamide adenine dinucleotide.  <span class="journal">J. Bacteriol. 137:1165–1175.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Pyridine%20nucleotide%20cycle%20of%20Salmonella%20typhimurium:%20isolation%20and%20characterization%20of%20pncA,%20pncB,%20and%20pncC%20mutants%20and%20utilization%20of%20exogenous%20nicotinamide%20adenine%20dinucleotide">google scholar</a></td><td><a href="pdf/234.pdf">pdf</a></td></tr>
<tr><td>235</td><td><a id="ref235" class="anchor"></a>Foulds, J. <i>1976</i>. <i>tolF</i> locus in <i>Escherichia coli:</i> chromosomal location and relationship to loci <i>cmlB</i> and <i>tolD.</i>  <span class="journal">J. Bacteriol. 128:604–608.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=tolF%20locus%20in%20Escherichia%20coli:%20chromosomal%20location%20and%20relationship%20to%20loci%20cmlB%20and%20tolD.">google scholar</a></td><td><a href="pdf/235.pdf">pdf</a></td></tr>
<tr><td>236</td><td><a id="ref236" class="anchor"></a>Foulds, J., and C. Barrett. <i>1973</i>. Characterization of <i>Escherichia coli</i> mutants tolerant to bacteriocin JF246: two new classes of tolerant mutants.  <span class="journal">J. Bacteriol. 116:885–892.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Characterization%20of%20Escherichia%20coli%20mutants%20tolerant%20to%20bacteriocin%20JF246:%20two%20new%20classes%20of%20tolerant%20mutants">google scholar</a></td><td><a href="pdf/236.pdf">pdf</a></td></tr>
<tr><td>237</td><td><a id="ref237" class="anchor"></a>Foulds, J., and T.-J. Chai. <i>1978</i>. New major outer membrane protein found in an <i>Escherichia coli</i> <i>tolF</i> mutant resistant to bacteriophage TuIb.  <span class="journal">J. Bacteriol. 133:1478–1483.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=New%20major%20outer%20membrane%20protein%20found%20in%20an%20Escherichia%20coli%20tolF%20mutant%20resistant%20to%20bacteriophage%20TuIb">google scholar</a></td><td><a href="pdf/237.pdf">pdf</a></td></tr>
<tr><td>238</td><td><a id="ref238" class="anchor"></a>Fouts, K. E., and S. D. Barbour. <i>1981</i>. Transductional mapping of <i>ksgB</i> and a new Tn<i>5</i>-induced kasugamycin resistance gene, <i>ksgD,</i> in <i>Escherichia coli</i> K-12.  <span class="journal">J. Bacteriol. 145:914–919.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Transductional%20mapping%20of%20ksgB%20and%20a%20new%20Tn5-induced%20kasugamycin%20resistance%20gene,%20ksgD,%20in%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/238.pdf">pdf</a></td></tr>
<tr><td>239</td><td><a id="ref239" class="anchor"></a>Fradkin, J. E., and D. G. Fraenkel. <i>1971</i>. 2-Keto-3-deoxygluconate 6-phosphate aldolase mutants of <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 108:1277–1283.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=2-Keto-3-deoxygluconate%206-phosphate%20aldolase%20mutants%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/239.pdf">pdf</a></td></tr>
<tr><td>240</td><td><a id="ref240" class="anchor"></a>Fraenkel, D. G., and S. Banerjee. <i>1972</i>. Deletion mapping of <i>zwf,</i> the gene for a constitutive enzyme, glucose 6-phosphate dehydrogenase in <i>Escherichia coli.</i>  <span class="journal">Genetics 71:481–489.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Deletion%20mapping%20of%20zwf,%20the%20gene%20for%20a%20constitutive%20enzyme,%20glucose%206-phosphate%20dehydrogenase%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/240.pdf">pdf</a></td></tr>
<tr><td>241</td><td><a id="ref241" class="anchor"></a>Franco, P. J., J. A. Eelkema, and R. J. Brooker. <i>1989</i>. Isolation and characterization of thiodigalactoside resistant mutants of the lactose permease which possess an enhanced recognition for maltose.  <span class="journal">J. Biol. Chem. 264:15988–15992.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20and%20characterization%20of%20thiodigalactoside%20resistant%20mutants%20of%20the%20lactose%20permease%20which%20possess%20an%20enhanced%20recognition%20for%20maltose">google scholar</a></td><td><a href="pdf/241.pdf">pdf</a></td></tr>
<tr><td>242</td><td><a id="ref242" class="anchor"></a>Franklin, N. C. <i>1969</i>. Mutation in <i>galU</i> gene of <i>E. coli</i> blocks phage Pl infection.  <span class="journal">Virology 38:189–191.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutation%20in%20galU%20gene%20of%20E.%20coli%20blocks%20phage%20Pl%20infection">google scholar</a></td><td><a href="pdf/242.pdf">pdf</a></td></tr>
<tr><td>243</td><td><a id="ref243" class="anchor"></a>Friedberg, I. <i>1972</i>. Localization of phosphoglucose isomerase in <i>Escherichia coli</i> and its relation to the induction of the hexose phosphate transport system.  <span class="journal">J. Bacteriol. 112:1201–1205.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Localization%20of%20phosphoglucose%20isomerase%20in%20Escherichia%20coli%20and%20its%20relation%20to%20the%20induction%20of%20the%20hexose%20phosphate%20transport%20system">google scholar</a></td><td><a href="pdf/243.pdf">pdf</a></td></tr>
<tr><td>244</td><td><a id="ref244" class="anchor"></a>Friedman, D. I. <i>1971</i>. A bacterial mutant affecting lambda development. <span class="journal">The Bacteriophage Lambda. Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20bacterial%20mutant%20affecting%20lambda%20development">google scholar</a></td><td></td></tr>
<tr><td>245</td><td><a id="ref245" class="anchor"></a>Friedman, D. I., and M. Baumann. <i>1976</i>. Cooperative effects of bacterial mutations affecting λ N gene expression. 1. Isolation and characterization of a <i>nusB</i> mutant.  <span class="journal">Virology 73:119–127.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Cooperative%20effects%20of%20bacterial%20mutations%20affecting%20λ%20N%20gene%20expression.%201.%20Isolation%20and%20characterization%20of%20a%20nusB%20mutant">google scholar</a></td><td><a href="pdf/245.pdf">pdf</a></td></tr>
<tr><td>246</td><td><a id="ref246" class="anchor"></a>Friedman, D. I., and M. Gottesman. <i>1983</i>. Lytic mode of λ development. <span class="journal">Lambda II. Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Lytic%20mode%20of%20λ%20development">google scholar</a></td><td></td></tr>
<tr><td>247</td><td><a id="ref247" class="anchor"></a>Friedman, S. <i>1982</i>. Bactericidal effect of 5-azacytidine on <i>Escherichia coli</i> carrying <i>Eco</i>RII restriction-modification enzymes.  <span class="journal">J. Bacteriol. 151:262–268.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Bactericidal%20effect%20of%205-azacytidine%20on%20Escherichia%20coli%20carrying%20EcoRII%20restriction-modification%20enzymes">google scholar</a></td><td><a href="pdf/247.pdf">pdf</a></td></tr>
<tr><td>248</td><td><a id="ref248" class="anchor"></a>Froehler, J., A. Rechenmacher, J. Thomale, G. Nass, and A. Boeck. <i>1980</i>. Genetic analysis of mutations causing borrelidin resistance by overproduction of threonyl-transfer ribonucleic acid synthetase.  <span class="journal">J. Bacteriol. 143:1135–1141.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20analysis%20of%20mutations%20causing%20borrelidin%20resistance%20by%20overproduction%20of%20threonyl-transfer%20ribonucleic%20acid%20synthetase">google scholar</a></td><td><a href="pdf/248.pdf">pdf</a></td></tr>
<tr><td>249</td><td><a id="ref249" class="anchor"></a>Fujita, H., S. Yamaguchi, T. Taira, and T. Iino. <i>1981</i>. A simple method for the isolation of flagellar shape mutants in <i>Salmonella.</i>  <span class="journal">J. Gen. Microbiol. 125:213–216.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20simple%20method%20for%20the%20isolation%20of%20flagellar%20shape%20mutants%20in%20Salmonella.">google scholar</a></td><td><a href="pdf/249.pdf">pdf</a></td></tr>
<tr><td>250</td><td><a id="ref250" class="anchor"></a>Furukawa, H., J.-T. Tsay, S. Jackowski, Y. Takamura, and C. O. Rock. <i>1993</i>. Thiolactomycin resistance in <i>Escherichia coli</i> is associated with the multidrug resistance efflux pump encoded by <i>emrAB.</i>  <span class="journal">J. Bacteriol. 175:3723–3729.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Thiolactomycin%20resistance%20in%20Escherichia%20coli%20is%20associated%20with%20the%20multidrug%20resistance%20efflux%20pump%20encoded%20by%20emrAB.">google scholar</a></td><td><a href="pdf/250.pdf">pdf</a></td></tr>
<tr><td>251</td><td><a id="ref251" class="anchor"></a>Garber, B. B., and J. S. Gots. <i>1980</i>. Utilization of 2,6-diaminopurine by <i>Salmonella</i> <i>typhimurium.</i>  <span class="journal">J. Bacteriol. 143:864–871.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Utilization%20of%202,6-diaminopurine%20by%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/251.pdf">pdf</a></td></tr>
<tr><td>252</td><td><a id="ref252" class="anchor"></a>Gayda, R. C., L. T. Yamamoto, and A. Markovitz. <i>1976</i>. Second-site mutations in <i>capR(lon)</i> strains of <i>Escherichia coli</i> K-12 that prevent radiation sensitivity and allow bacteriophage lambda to lysogenize.  <span class="journal">J. Bacteriol. 127:1208–1216.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Second-site%20mutations%20in%20capR(lon)%20strains%20of%20Escherichia%20coli%20K-12%20that%20prevent%20radiation%20sensitivity%20and%20allow%20bacteriophage%20lambda%20to%20lysogenize">google scholar</a></td><td><a href="pdf/252.pdf">pdf</a></td></tr>
<tr><td>253</td><td><a id="ref253" class="anchor"></a>Gehring, K., A. Charbit, E. Brissaud, and M. Hofnung. <i>1987</i>. Bacteriophage lambda receptor site on the <i>Escherichia coli</i> K-12 <i>lamB</i> protein.  <span class="journal">J. Bacteriol. 169:2103–2106.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Bacteriophage%20lambda%20receptor%20site%20on%20the%20Escherichia%20coli%20K-12%20lamB%20protein">google scholar</a></td><td><a href="pdf/253.pdf">pdf</a></td></tr>
<tr><td>254</td><td><a id="ref254" class="anchor"></a>Gellert, M., and M. L. Bullock. <i>1970</i>. DNA ligase mutants of <i>Escherichia coli.</i> <i>Proc. Natl. Acad.</i>  <span class="journal">Sci. USA 67:1580–1587.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=DNA%20ligase%20mutants%20of%20Escherichia%20coli.%20Proc.%20Natl.%20Acad.">google scholar</a></td><td><a href="pdf/254.pdf">pdf</a></td></tr>
<tr><td>255</td><td><a id="ref255" class="anchor"></a>Gellert, M., M. H. O’Dea, T. Itoh, and J.-I. Tomizawa. <i>1976</i>. Novobiocin and coumermycin inhibit DNA supercoiling catalyzed by DNA gyrase.  <span class="journal">Proc. Natl. Acad. Sci. USA 73:4474–4478.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Novobiocin%20and%20coumermycin%20inhibit%20DNA%20supercoiling%20catalyzed%20by%20DNA%20gyrase">google scholar</a></td><td><a href="pdf/255.pdf">pdf</a></td></tr>
<tr><td>256</td><td><a id="ref256" class="anchor"></a>Georgopoulos, C. P. <i>1971</i>. Bacterial mutants in which the gene N function of bacteriophage lambda is blocked have an altered RNA polymerase.  <span class="journal">Proc. Natl. Acad. Sci. USA 68:2977–2981.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Bacterial%20mutants%20in%20which%20the%20gene%20N%20function%20of%20bacteriophage%20lambda%20is%20blocked%20have%20an%20altered%20RNA%20polymerase">google scholar</a></td><td><a href="pdf/256.pdf">pdf</a></td></tr>
<tr><td>257</td><td><a id="ref257" class="anchor"></a>Georgopoulos, C. P. <i>1977</i>. A new bacterial gene <i>groPC</i> which affects phage lambda DNA replication.  <span class="journal">Mol. Gen. Genet. 151:35–40.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20new%20bacterial%20gene%20groPC%20which%20affects%20phage%20lambda%20DNA%20replication">google scholar</a></td><td><a href="pdf/257.pdf">pdf</a></td></tr>
<tr><td>258</td><td><a id="ref258" class="anchor"></a>Georgopoulos, C. P., R. W. Hendrix, and A. D. Kaiser. <i>1972</i>. Role of the host cell in bacteriophage morphogenesis: effects of a bacterial mutation on T4 head assembly.  <span class="journal">Nature  New Biol. 239:38–41.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Role%20of%20the%20host%20cell%20in%20bacteriophage%20morphogenesis:%20effects%20of%20a%20bacterial%20mutation%20on%20T4%20head%20assembly">google scholar</a></td><td><a href="pdf/258.pdf">pdf</a></td></tr>
<tr><td>259</td><td><a id="ref259" class="anchor"></a>Georgopoulos, C. P., and I. Herskowitz. <i>1971</i>. <i>Escherichia coli</i> mutants blocked in lambda DNA synthesis. <span class="journal">The Bacteriophage Lambda. Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Escherichia%20coli%20mutants%20blocked%20in%20lambda%20DNA%20synthesis">google scholar</a></td><td><a href="pdf/259.pdf">pdf</a></td></tr>
<tr><td>260</td><td><a id="ref260" class="anchor"></a>Georgopoulos, C. P., J. Swindle, F. Keppel, M. Ballivet, R. Bisig, and H. Eisen. <i>1980</i>. Studies on the <i>E.</i> coli <i>groNB (nusB)</i> gene which affects bacteriophage λ N gene function.  <span class="journal">Mol. Gen. Genet. 179:55–61.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Studies%20on%20the%20E.%20coli%20groNB%20(nusB)%20gene%20which%20affects%20bacteriophage%20λ%20N%20gene%20function">google scholar</a></td><td><a href="pdf/260.pdf">pdf</a></td></tr>
<tr><td>261</td><td><a id="ref261" class="anchor"></a>Ghysen, A., and M. Pirionio. <i>1972</i>. Relationship between the <i>N</i> function of bacteriophage λ and host RNA polymerase.  <span class="journal">J. Mol. Biol. 65:259—272.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Relationship%20between%20the%20N%20function%20of%20bacteriophage%20λ%20and%20host%20RNA%20polymerase">google scholar</a></td><td><a href="pdf/261.pdf">pdf</a></td></tr>
<tr><td>262</td><td><a id="ref262" class="anchor"></a>Gibson, M. M., D. A. Bagga, C. G. Miller, and M. E. Maguire. <i>1991</i>. Magnesium transport in <i>Salmonella typhimurium:</i> the influence of new mutations conferring Co2+ resistance on the CorA Mg2+transport system.  <span class="journal">Mol. Microbiol. 5:2753–2762.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Magnesium%20transport%20in%20Salmonella%20typhimurium:%20the%20influence%20of%20new%20mutations%20conferring%20Co2+%20resistance%20on%20the%20CorA%20Mg2+transport%20system">google scholar</a></td><td><a href="pdf/262.pdf">pdf</a></td></tr>
<tr><td>263</td><td><a id="ref263" class="anchor"></a>Gibson, M. M., M. Price, and C. F. Higgins. <i>1984</i>. Genetic characterization and molecular cloning of the tripeptide permease (<i>tpp</i>) genes of <i>Salmonella typhimurium.</i>  <span class="journal">J. Bacteriol. 160:122–130.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20characterization%20and%20molecular%20cloning%20of%20the%20tripeptide%20permease%20(tpp)%20genes%20of%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/263.pdf">pdf</a></td></tr>
<tr><td>264</td><td><a id="ref264" class="anchor"></a>Ginther, C. L., and J. L. Ingraham. <i>1974</i>. Cold-sensitive mutant of <i>Salmonella typhimurium</i> defective in nucleoside diphosphokinase.  <span class="journal">J. Bacteriol. 118:1020–1026.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Cold-sensitive%20mutant%20of%20Salmonella%20typhimurium%20defective%20in%20nucleoside%20diphosphokinase">google scholar</a></td><td><a href="pdf/264.pdf">pdf</a></td></tr>
<tr><td>265</td><td><a id="ref265" class="anchor"></a>Giordano, G., L. Grillet, R. Rosset, J. H. Dou, and E. Azoulay. <i>1978</i>. Characterization of an <i>E.</i> <i>coli</i> mutant that is sensitive to chlorate when grown aerobically.  <span class="journal">Biochem. J. 176:553–561.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Characterization%20of%20an%20E.%20coli%20mutant%20that%20is%20sensitive%20to%20chlorate%20when%20grown%20aerobically">google scholar</a></td><td><a href="pdf/265.pdf">pdf</a></td></tr>
<tr><td>266</td><td><a id="ref266" class="anchor"></a>Glaser, J. H., and J. A. DeMoss. <i>1972</i>. Comparison of nitrate reductase mutants of <i>Escherichia</i> <i>coli</i> selected by alternative procedures.  <span class="journal">Mol. Gen. Genet. 116:1–10.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Comparison%20of%20nitrate%20reductase%20mutants%20of%20Escherichia%20coli%20selected%20by%20alternative%20procedures">google scholar</a></td><td><a href="pdf/266.pdf">pdf</a></td></tr>
<tr><td>267</td><td><a id="ref267" class="anchor"></a>Glickman, B. W., and M. Radman. <i>1980</i>. <i>Escherichia coli</i> mutator mutants deficient in methylation instructed DNA mismatch correction.  <span class="journal">Proc. Natl. Acad. Sci. USA 77:1063–1067.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Escherichia%20coli%20mutator%20mutants%20deficient%20in%20methylation%20instructed%20DNA%20mismatch%20correction">google scholar</a></td><td><a href="pdf/267.pdf">pdf</a></td></tr>
<tr><td>268</td><td><a id="ref268" class="anchor"></a>Glover, S. W. <i>1962</i>. Valine-resistant mutants of <i>Escherichia coli</i> K-12.  <span class="journal">Genet. Res. 3:448–460.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Valine-resistant%20mutants%20of%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/268.pdf">pdf</a></td></tr>
<tr><td>269</td><td><a id="ref269" class="anchor"></a>Gollop, N., H. Tavori, and Z. Barak. <i>1982</i>. Acetohydroxy acid synthase is a target for leucine- containing peptide toxicity in <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 149:387–390.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Acetohydroxy%20acid%20synthase%20is%20a%20target%20for%20leucine-%20containing%20peptide%20toxicity%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/269.pdf">pdf</a></td></tr>
<tr><td>270</td><td><a id="ref270" class="anchor"></a>Gollub, E., and D. B. Sprinson. <i>1969</i>. A regulatory mutation in tyrosine biosynthesis. <i>Biochem.</i>  <span class="journal">Biophys. Res. Commun. 35:389–395.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20regulatory%20mutation%20in%20tyrosine%20biosynthesis.%20Biochem.">google scholar</a></td><td><a href="pdf/270.pdf">pdf</a></td></tr>
<tr><td>271</td><td><a id="ref271" class="anchor"></a>Gollub, E. G., K. P. Liu, and D. B. Sprinson. <i>1973</i>. A regulatory gene of phenylalanine biosynthesis <i>(pheR)</i> in <i>Salmonella typhimurium.</i>  <span class="journal">J. Bacteriol. 115:121–128.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20regulatory%20gene%20of%20phenylalanine%20biosynthesis%20(pheR)%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/271.pdf">pdf</a></td></tr>
<tr><td>272</td><td><a id="ref272" class="anchor"></a>Gots, J. S., C. E. Benson, and S. R. Shumas. <i>1972</i>. Genetic separation of hypoxanthine and guanine-xanthine phosphoribosyltransferase activities by deletion mutations in <i>Salmonella typhimurium.</i>  <span class="journal">J. Bacteriol. 112:910–916.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20separation%20of%20hypoxanthine%20and%20guanine-xanthine%20phosphoribosyltransferase%20activities%20by%20deletion%20mutations%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/272.pdf">pdf</a></td></tr>
<tr><td>273</td><td><a id="ref273" class="anchor"></a>Green, L., and C. G. Mlller. <i>1980</i>. Genetic mapping of the <i>Salmonella typhimurium pepB</i> locus.  <span class="journal">J. Bacteriol. 143:1524–1526.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20mapping%20of%20the%20Salmonella%20typhimurium%20pepB%20locus">google scholar</a></td><td><a href="pdf/273.pdf">pdf</a></td></tr>
<tr><td>274</td><td><a id="ref274" class="anchor"></a>Greenberg, J. T., J. H. Chou, P. A. Monach, and B. Demple. <i>1991</i>. Activation of oxidative stress genes by mutations at the <i>soxQ</i>/<i>cfxB</i>/<i>marA</i> locus of <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 173:4433– 4439.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Activation%20of%20oxidative%20stress%20genes%20by%20mutations%20at%20the%20soxQ/cfxB/marA%20locus%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/274.pdf">pdf</a></td></tr>
<tr><td>275</td><td><a id="ref275" class="anchor"></a>Greene, R. C., J. V. Hunter, and E. H. Coch. <i>1973</i>. Properties of <i>metK</i> mutants of <i>Escherichia</i> <i>coli</i> K-12.  <span class="journal">J. Bacteriol. 115:57–67.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Properties%20of%20metK%20mutants%20of%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/275.pdf">pdf</a></td></tr>
<tr><td>276</td><td><a id="ref276" class="anchor"></a>Gruell, J. M., H. Hennecke, J. Froehler, J. Thomale, G. Nass, and A. Böck. <i>1979</i>. <i>Escherichia</i> <i>coli</i> mutants overproducing phenylalanyl transfer RNA synthetase and threonyl transfer RNA synthetase.  <span class="journal">J. Bacteriol. 137:480–489.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Escherichia%20coli%20mutants%20overproducing%20phenylalanyl%20transfer%20RNA%20synthetase%20and%20threonyl%20transfer%20RNA%20synthetase">google scholar</a></td><td><a href="pdf/276.pdf">pdf</a></td></tr>
<tr><td>277</td><td><a id="ref277" class="anchor"></a>Guardiola, J., M. DeFelice, T. Klopotowski, and M. Iaccarino. <i>1974</i>. Mutations affecting the different transport systems for isoleucine, leucine, and valine in <i>Escherichia coli</i> K-12.  <span class="journal">J. Bacteriol. 117:393 405.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutations%20affecting%20the%20different%20transport%20systems%20for%20isoleucine,%20leucine,%20and%20valine%20in%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/277.pdf">pdf</a></td></tr>
<tr><td>278</td><td><a id="ref278" class="anchor"></a>Guardiola, J., and M. Iaccarino. <i>1971</i>. <i>Escherichia coli</i> K-12 mutants altered in the transport of branched-chain amino acids.  <span class="journal">J. Bacteriol. 108:1034–1044.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Escherichia%20coli%20K-12%20mutants%20altered%20in%20the%20transport%20of%20branched-chain%20amino%20acids">google scholar</a></td><td><a href="pdf/278.pdf">pdf</a></td></tr>
<tr><td>279</td><td><a id="ref279" class="anchor"></a>Guarneros, G., G. Machado, P. Guzman and E. Garay. <i>1987</i>. Genetic and physical location of the <i>Escherichia coli</i> <i>rap</i> locus, which is essential for growth of bacteriophage lambda.  <span class="journal">J. Bacteriol. 169:5188–5192.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20and%20physical%20location%20of%20the%20Escherichia%20coli%20rap%20locus,%20which%20is%20essential%20for%20growth%20of%20bacteriophage%20lambda">google scholar</a></td><td><a href="pdf/279.pdf">pdf</a></td></tr>
<tr><td>280</td><td><a id="ref280" class="anchor"></a>Guest, J. R. <i>1969</i>. Biochemical and genetic studies with nitrate reductase C-gene mutants of <i>Escherichia coli.</i>  <span class="journal">Mol. Gen. Genet. 105:285–297.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Biochemical%20and%20genetic%20studies%20with%20nitrate%20reductase%20C-gene%20mutants%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/280.pdf">pdf</a></td></tr>
<tr><td>281</td><td><a id="ref281" class="anchor"></a>Guest, J. R. <i>1979</i>. Anaerobic growth of <i>Escherichia coli</i> K12 with fumarate as terminal electron acceptor. Genetic studies with menaquinone and fluoroacetate-resistant mutants.  <span class="journal">J. Gen. Microbiol. 115:259–271.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Anaerobic%20growth%20of%20Escherichia%20coli%20K12%20with%20fumarate%20as%20terminal%20electron%20acceptor.%20Genetic%20studies%20with%20menaquinone%20and%20fluoroacetate-resistant%20mutants">google scholar</a></td><td><a href="pdf/281.pdf">pdf</a></td></tr>
<tr><td>282</td><td><a id="ref282" class="anchor"></a>Gupta, A., and S. Kumar. <i>1985</i>. Roles of internally and externally supplied cyclic AMP in the growth response of <i>Escherichia coli</i> to sodium chloride.  <span class="journal">Indian J. Exp. Biol. 23:647–652.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Roles%20of%20internally%20and%20externally%20supplied%20cyclic%20AMP%20in%20the%20growth%20response%20of%20Escherichia%20coli%20to%20sodium%20chloride">google scholar</a></td><td><a href="pdf/282.pdf">pdf</a></td></tr>
<tr><td>283</td><td><a id="ref283" class="anchor"></a>Guterman, S. K., and L. Dann. <i>1973</i>. Excretion of enterochelin by <i>exbA</i> and <i>exbB</i> mutants of <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 114:1225–1230.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Excretion%20of%20enterochelin%20by%20exbA%20and%20exbB%20mutants%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/283.pdf">pdf</a></td></tr>
<tr><td>284</td><td><a id="ref284" class="anchor"></a>Guterman, S. K., A. Wright, and D. H. Boyd. <i>1975</i>. Genes affecting coliphage BF23 and E colicin sensitivity in <i>Salmonella typhimurium.</i>  <span class="journal">J. Bacteriol. 124:1351–1358.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genes%20affecting%20coliphage%20BF23%20and%20E%20colicin%20sensitivity%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/284.pdf">pdf</a></td></tr>
<tr><td>285</td><td><a id="ref285" class="anchor"></a>Guth, A., R. Engel, and B. E. Tropp. <i>1980</i>. Uptake of glycerol 3-phosphate and some of its analogs by the hexose phosphate transport system of <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 143:538–539.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Uptake%20of%20glycerol%203-phosphate%20and%20some%20of%20its%20analogs%20by%20the%20hexose%20phosphate%20transport%20system%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/285.pdf">pdf</a></td></tr>
<tr><td>286</td><td><a id="ref286" class="anchor"></a>Hachler, H., S. P. Cohen, and S. B. Levy. <i>1991</i>. <i>marA,</i> a regulated locus which controls expression of chromosomal multiple antibiotic resistance in <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 173: 5532– 5538.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=marA,%20a%20regulated%20locus%20which%20controls%20expression%20of%20chromosomal%20multiple%20antibiotic%20resistance%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/286.pdf">pdf</a></td></tr>
<tr><td>287</td><td><a id="ref287" class="anchor"></a>Hacking, A. J., and E. C. C. Lin. <i>1976</i>. Disruption of the fucose pathway as a consequence of genetic adaptation to propanediol as a carbon source in <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 126:1166–1172.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Disruption%20of%20the%20fucose%20pathway%20as%20a%20consequence%20of%20genetic%20adaptation%20to%20propanediol%20as%20a%20carbon%20source%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/287.pdf">pdf</a></td></tr>
<tr><td>288</td><td><a id="ref288" class="anchor"></a>Hadar, R., and J. Kuhn. <i>1977</i>. A mutant constitutive for aromatic permease.  <span class="journal">FEBS Lett. 76:77–80.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20mutant%20constitutive%20for%20aromatic%20permease">google scholar</a></td><td><a href="pdf/288.pdf">pdf</a></td></tr>
<tr><td>289</td><td><a id="ref289" class="anchor"></a>Hafner, E. W., C. W. Tabor, and H. Tabor. <i>1977</i>. Isolation of a <i>metK</i> mutant with a temperature- sensitive S-adenosylmethionine synthetase.  <span class="journal">J. Bacteriol. 132:832–840.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20of%20a%20metK%20mutant%20with%20a%20temperature-%20sensitive%20S-adenosylmethionine%20synthetase">google scholar</a></td><td><a href="pdf/289.pdf">pdf</a></td></tr>
<tr><td>290</td><td><a id="ref290" class="anchor"></a>Hall, B. G. <i>1982</i>. Chromosomal mutation for citrate utilization by <i>Escherichia coli</i> K-12.  <span class="journal">J. Bacteriol. 151:269–273.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Chromosomal%20mutation%20for%20citrate%20utilization%20by%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/290.pdf">pdf</a></td></tr>
<tr><td>291</td><td><a id="ref291" class="anchor"></a>Hall, M. N., and T. J. Silhavy. <i>1981</i>. The <i>ompB</i> locus and the regulation of the major outer membrane porin proteins of <i>Escherichia coli</i> K12.  <span class="journal">J. Mol. Biol. 146:23–44.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The%20ompB%20locus%20and%20the%20regulation%20of%20the%20major%20outer%20membrane%20porin%20proteins%20of%20Escherichia%20coli%20K12">google scholar</a></td><td><a href="pdf/291.pdf">pdf</a></td></tr>
<tr><td>292</td><td><a id="ref292" class="anchor"></a>Halpern, Y. S., and H. E. Umbarger. <i>1961</i>. Utilization of L-glutamic acid and 2-oxoglutaric acid as sole sources of carbon by <i>Escherichia coli.</i>  <span class="journal">J. Gen. Microbiol. 26:175–183.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Utilization%20of%20L-glutamic%20acid%20and%202-oxoglutaric%20acid%20as%20sole%20sources%20of%20carbon%20by%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/292.pdf">pdf</a></td></tr>
<tr><td>293</td><td><a id="ref293" class="anchor"></a>Halsall, D. M. <i>1975</i>. Strains of <i>E. coli</i> with defective lysine transport systems.  <span class="journal">Biochem. Genet. 13:109–124.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Strains%20of%20E.%20coli%20with%20defective%20lysine%20transport%20systems">google scholar</a></td><td><a href="pdf/293.pdf">pdf</a></td></tr>
<tr><td>294</td><td><a id="ref294" class="anchor"></a>Hama, H., Y. Sumita, Y. Kakutani, M. Tsuda, and T. Tsuchiya. <i>1990</i>. Target of serine inhibition in <i>Escherichia coli.</i>  <span class="journal">Biochem. Biophys. Res. Commun. 168:1211–1216.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Target%20of%20serine%20inhibition%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/294.pdf">pdf</a></td></tr>
<tr><td>295</td><td><a id="ref295" class="anchor"></a>Hammelburger, J. W., and G. A. Orr. <i>1983</i>. Interaction of <i>sn</i>-glycerol 3-phosphorothioate with <i>Escherichia coli:</i> effect on cell growth and metabolism.  <span class="journal">J. Bacteriol. 156:789–799.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Interaction%20of%20sn-glycerol%203-phosphorothioate%20with%20Escherichia%20coli:%20effect%20on%20cell%20growth%20and%20metabolism">google scholar</a></td><td><a href="pdf/295.pdf">pdf</a></td></tr>
<tr><td>296</td><td><a id="ref296" class="anchor"></a>Hammer, K., K. F. Jensen, P. Poulsen, A. B. Oppenheim, and M. Gottesman. <i>1987</i>. Isolation of <i>Escherichia coli</i> <i>rpoB</i> mutants resistant to killing by lambda cII protein and altered in <i>pyrE</i> gene attenuation.  <span class="journal">J. Bacteriol. 169:5289–5297.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20of%20Escherichia%20coli%20rpoB%20mutants%20resistant%20to%20killing%20by%20lambda%20cII%20protein%20and%20altered%20in%20pyrE%20gene%20attenuation">google scholar</a></td><td><a href="pdf/296.pdf">pdf</a></td></tr>
<tr><td>297</td><td><a id="ref297" class="anchor"></a>Hancock, R. E. W., S. W. Farmer, Z. Li, and K. Poole. <i>1991</i>. Interaction of aminoglycosides with the outer membranes and purified lipopolysaccharide and <i>ompF</i> porin of <i>Escherichia coli</i>.  <span class="journal">Antimicrob. Agents Chemother. 35:1309–1314.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Interaction%20of%20aminoglycosides%20with%20the%20outer%20membranes%20and%20purified%20lipopolysaccharide%20and%20ompF%20porin%20of%20Escherichia%20coli">google scholar</a></td><td><a href="pdf/297.pdf">pdf</a></td></tr>
<tr><td>298</td><td><a id="ref298" class="anchor"></a>Hane, M. W., and T. H. Wood. <i>1969</i>. <i>Escherichia coli</i> K-12 mutants resistant to nalidixic acid: genetic mapping and dominance studies.  <span class="journal">J. Bacteriol. 99:238–241.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Escherichia%20coli%20K-12%20mutants%20resistant%20to%20nalidixic%20acid:%20genetic%20mapping%20and%20dominance%20studies">google scholar</a></td><td><a href="pdf/298.pdf">pdf</a></td></tr>
<tr><td>299</td><td><a id="ref299" class="anchor"></a>Hantke, K. <i>1976</i>. Phage T6-colicin K receptor and nucleoside transport in <i>Escherichia coli.</i> <i>FEBS</i>  <span class="journal">Lett. 70:109–112.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Phage%20T6-colicin%20K%20receptor%20and%20nucleoside%20transport%20in%20Escherichia%20coli.%20FEBS">google scholar</a></td><td><a href="pdf/299.pdf">pdf</a></td></tr>
<tr><td>300</td><td><a id="ref300" class="anchor"></a>Hantke, K. <i>1987</i>. Selection procedure for deregulated iron transport mutants (<i>fur</i>) in <i>Escherichia</i> <i>coli</i> K12: <i>fur</i> not only affects iron metabolism.  <span class="journal">Mol. Gen. Genet. 210:135–139.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Selection%20procedure%20for%20deregulated%20iron%20transport%20mutants%20(fur)%20in%20Escherichia%20coli%20K12:%20fur%20not%20only%20affects%20iron%20metabolism">google scholar</a></td><td><a href="pdf/300.pdf">pdf</a></td></tr>
<tr><td>301</td><td><a id="ref301" class="anchor"></a>Hardaway, K. L., and C. S. Buller. <i>1979</i>. Effects of ethylenediaminetetraacetate on phospholipids and outer membrane function in <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 137:62–68.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Effects%20of%20ethylenediaminetetraacetate%20on%20phospholipids%20and%20outer%20membrane%20function%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/301.pdf">pdf</a></td></tr>
<tr><td>302</td><td><a id="ref302" class="anchor"></a>Harkness, R. E., and V. Braun. <i>1990</i>. In vitro peptidoglycan synthesis by envelopes from <i>Escherichia coli tolM</i> mutants is inhibited by colicin M.  <span class="journal">J. Bacteriol. 172:498–500.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=In%20vitro%20peptidoglycan%20synthesis%20by%20envelopes%20from%20Escherichia%20coli%20tolM%20mutants%20is%20inhibited%20by%20colicin%20M">google scholar</a></td><td><a href="pdf/302.pdf">pdf</a></td></tr>
<tr><td>303</td><td><a id="ref303" class="anchor"></a>Harrison, L. I., H. N. Christiansen, M. E. Handlogten, D. L. Oxender, and S. C. Quay. <i>1975</i>. Transport of L-4-azaleucine in <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 122:957–965.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Transport%20of%20L-4-azaleucine%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/303.pdf">pdf</a></td></tr>
<tr><td>304</td><td><a id="ref304" class="anchor"></a>Hasenbank, R., C. Guthrie, G. Stoffler, H. G. Wittmann, L. Rosen, and D. Apirion. <i>1973</i>. Electrophoretic and immunological studies on ribosomal proteins of 100 <i>Escherichia coli</i> revertants from streptomycin dependence.  <span class="journal">Mol. Gen. Genet. 127:1–18.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Electrophoretic%20and%20immunological%20studies%20on%20ribosomal%20proteins%20of%20100%20Escherichia%20coli%20revertants%20from%20streptomycin%20dependence">google scholar</a></td><td><a href="pdf/304.pdf">pdf</a></td></tr>
<tr><td>305</td><td><a id="ref305" class="anchor"></a>Heatwole, V. M., and R. L. Somerville. <i>1991</i>. Cloning, nucleotide sequence, and characterization of <i>mtr,</i> the structural gene for a tryptophan-specific permease of <i>Escherichia</i> <i>coli</i> K-12.  <span class="journal">J. Bacteriol. 173:108–115.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Cloning,%20nucleotide%20sequence,%20and%20characterization%20of%20mtr,%20the%20structural%20gene%20for%20a%20tryptophan-specific%20permease%20of%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/305.pdf">pdf</a></td></tr>
<tr><td>306</td><td><a id="ref306" class="anchor"></a>Hedges, R. W., and K. P. Shannon. <i>1984</i>. Resistance to apramycin in <i>Escherichia coli</i> isolated from animals: detection of a novel aminoglycoside-modifying enzyme.  <span class="journal">J. Gen. Microbiol. 130:473– 482.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Resistance%20to%20apramycin%20in%20Escherichia%20coli%20isolated%20from%20animals:%20detection%20of%20a%20novel%20aminoglycoside-modifying%20enzyme">google scholar</a></td><td><a href="pdf/306.pdf">pdf</a></td></tr>
<tr><td>307</td><td><a id="ref307" class="anchor"></a>Held, W. A., and O. H. Smith. <i>1970</i>. Regulation of the <i>Escherichia coli</i> tryptophan operon by early reactions in the aromatic pathway.  <span class="journal">J. Bacteriol. 101:202–208.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation%20of%20the%20Escherichia%20coli%20tryptophan%20operon%20by%20early%20reactions%20in%20the%20aromatic%20pathway">google scholar</a></td><td><a href="pdf/307.pdf">pdf</a></td></tr>
<tr><td>308</td><td><a id="ref308" class="anchor"></a>Held, W. A., and O. H. Smith. <i>1970</i>. Mechanism of 3-methylanthranilic acid derepression of the tryptophan operon in <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 101:209–217.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mechanism%20of%203-methylanthranilic%20acid%20derepression%20of%20the%20tryptophan%20operon%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/308.pdf">pdf</a></td></tr>
<tr><td>309</td><td><a id="ref309" class="anchor"></a>Helling, R. B. <i>1968</i>. Selection of a mutant of <i>Escherichia coli</i> which has high mutation rates. <i>J.</i>  <span class="journal">Bacteriol. 96:975–980.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Selection%20of%20a%20mutant%20of%20Escherichia%20coli%20which%20has%20high%20mutation%20rates.%20J.">google scholar</a></td><td><a href="pdf/309.pdf">pdf</a></td></tr>
<tr><td>310</td><td><a id="ref310" class="anchor"></a>Helling, R. B., and B. S. Adams. <i>1970</i>. Nalidixic acid-resistant auxotrophs of <i>Escherichia coli.</i> <i>J.</i>  <span class="journal">Bacteriol. 104:1027–1029.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Nalidixic%20acid-resistant%20auxotrophs%20of%20Escherichia%20coli.%20J.">google scholar</a></td><td><a href="pdf/310.pdf">pdf</a></td></tr>
<tr><td>311</td><td><a id="ref311" class="anchor"></a>Helling, R. B., and J. Kukora. <i>1971</i>. Nalidixic acid-resistant mutants of <i>Escherichia coli</i> deficient in isocitrate dehydrogenase.  <span class="journal">J. Bacteriol. 105:1224–1226.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Nalidixic%20acid-resistant%20mutants%20of%20Escherichia%20coli%20deficient%20in%20isocitrate%20dehydrogenase">google scholar</a></td><td><a href="pdf/311.pdf">pdf</a></td></tr>
<tr><td>312</td><td><a id="ref312" class="anchor"></a>Helser, T. L., J. E. Davies, and J. E. Dahlberg. <i>1972</i>. Mechanism of kasugamycin resistance in <i>Escherichia coli.</i> <i>Nature</i>   <span class="journal">New Biol. 235:6–9.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mechanism%20of%20kasugamycin%20resistance%20in%20Escherichia%20coli.%20Nature">google scholar</a></td><td><a href="pdf/312.pdf">pdf</a></td></tr>
<tr><td>313</td><td><a id="ref313" class="anchor"></a>Hengge, R., T. J. Larson, and W. Boos. <i>1983</i>. <i>sn</i>-Glycerol-3-phosphate transport in <i>Salmonella</i> <i>typhimurium.</i>  <span class="journal">J. Bacteriol. 155:186–195.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=sn-Glycerol-3-phosphate%20transport%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/313.pdf">pdf</a></td></tr>
<tr><td>314</td><td><a id="ref314" class="anchor"></a>Henning, U., I. Hindennach, and I. Haller. <i>1976</i>. The major proteins of the <i>Escherichia coli</i> outer cell envelope membrane: evidence for the structural gene of protein II.  <span class="journal">FEBS Lett. 61:46–48.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The%20major%20proteins%20of%20the%20Escherichia%20coli%20outer%20cell%20envelope%20membrane:%20evidence%20for%20the%20structural%20gene%20of%20protein%20II">google scholar</a></td><td><a href="pdf/314.pdf">pdf</a></td></tr>
<tr><td>315</td><td><a id="ref315" class="anchor"></a>Heyde, M., and R. Portalier. <i>1982</i>. New pleiotropic alkaline phosphatase-negative mutants of <i>Escherichia coli</i> K-12.  <span class="journal">J. Bacteriol. 151:529–533.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=New%20pleiotropic%20alkaline%20phosphatase-negative%20mutants%20of%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/315.pdf">pdf</a></td></tr>
<tr><td>316</td><td><a id="ref316" class="anchor"></a>Heyde, M., and R. Portalier. <i>1983</i>. Isolation and characterization of a new type of <i>Escherichia</i> <i>coli</i> K12 <i>phoB</i> mutants.  <span class="journal">Mol. Gen. Genet. 190:122–127.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20and%20characterization%20of%20a%20new%20type%20of%20Escherichia%20coli%20K12%20phoB%20mutants">google scholar</a></td><td><a href="pdf/316.pdf">pdf</a></td></tr>
<tr><td>317</td><td><a id="ref317" class="anchor"></a>Higgins, C. F., M. M. Hardie, D. Jamieson, and L. M. Powell. <i>1983</i>. Genetic map of the <i>opp</i> (oligopeptide permease) locus of <i>Salmonella typhimurium.</i>  <span class="journal">J. Bacteriol. 153:830–836.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20map%20of%20the%20opp%20(oligopeptide%20permease)%20locus%20of%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/317.pdf">pdf</a></td></tr>
<tr><td>318</td><td><a id="ref318" class="anchor"></a>Hiraga, S. <i>1969</i>. Operator mutants of the tryptophan operon in <i>Escherichia coli.</i>  <span class="journal">J. Mol. Biol. 39:159–179.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Operator%20mutants%20of%20the%20tryptophan%20operon%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/318.pdf">pdf</a></td></tr>
<tr><td>319</td><td><a id="ref319" class="anchor"></a>Hiraga, S., K. Igarashi, and T. Yura. <i>1967</i>. A deoxythymidine kinase deficient mutant of <i>Escherichia coli.</i> I. Isolation and some properties.  <span class="journal">Biochem. Biophys. Acta 145:41–51.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20deoxythymidine%20kinase%20deficient%20mutant%20of%20Escherichia%20coli.%20I.%20Isolation%20and%20some%20properties">google scholar</a></td><td><a href="pdf/319.pdf">pdf</a></td></tr>
<tr><td>320</td><td><a id="ref320" class="anchor"></a>Hiraga, S., K. Ito, T. Matsuyama, H. Ozaki, and T. Yura. <i>1968</i>. 5-Methyltryptophan-resistant mutations linked with the arginine G marker in <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 96:1880–1881.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=5-Methyltryptophan-resistant%20mutations%20linked%20with%20the%20arginine%20G%20marker%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/320.pdf">pdf</a></td></tr>
<tr><td>321</td><td><a id="ref321" class="anchor"></a>Hirota, Y., M. Inuzuka, and M. Tomoeda. <i>1966</i>. Elective selection of proline-requiring mutants.  <span class="journal">J. Bacteriol. 91:2392.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Elective%20selection%20of%20proline-requiring%20mutants">google scholar</a></td><td><a href="pdf/321.pdf">pdf</a></td></tr>
<tr><td>322</td><td><a id="ref322" class="anchor"></a>Hirshfield, I. N., R. Dedeken, P. C. Horn, D. A. Hopwood, and W. K. Maas. <i>1968</i>. Studies on the mechanism of repression of arginine biosynthesis in <i>Escherichia coli.</i> III. Repression of enzymes of arginine biosyntheses in arginyl-tRNA synthetase mutants.  <span class="journal">J. Mol. Biol. 35:83–93.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Studies%20on%20the%20mechanism%20of%20repression%20of%20arginine%20biosynthesis%20in%20Escherichia%20coli.%20III.%20Repression%20of%20enzymes%20of%20arginine%20biosyntheses%20in%20arginyl-tRNA%20synthetase%20mutants">google scholar</a></td><td><a href="pdf/322.pdf">pdf</a></td></tr>
<tr><td>323</td><td><a id="ref323" class="anchor"></a>Hirshfield, I. N., and P. C. Zamecnik. <i>1972</i>. Thiosine-resistant mutants of <i>Escherichia coli</i> K-12 with growth medium-dependent lysyl-tRNA synthetase activity. I. Isolation and physiological characterization.  <span class="journal">Biochim. Biophys. Acta 259:330–343.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Thiosine-resistant%20mutants%20of%20Escherichia%20coli%20K-12%20with%20growth%20medium-dependent%20lysyl-tRNA%20synthetase%20activity.%20I.%20Isolation%20and%20physiological%20characterization">google scholar</a></td><td><a href="pdf/323.pdf">pdf</a></td></tr>
<tr><td>324</td><td><a id="ref324" class="anchor"></a>Hobson, A. C. <i>1974</i>. A norleucine-resistant mutant of <i>Salmonella typhimurium</i> with a possible defect in valine uptake or regulation.  <span class="journal">J. Gen. Microbiol. 82:425–429.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20norleucine-resistant%20mutant%20of%20Salmonella%20typhimurium%20with%20a%20possible%20defect%20in%20valine%20uptake%20or%20regulation">google scholar</a></td><td><a href="pdf/324.pdf">pdf</a></td></tr>
<tr><td>325</td><td><a id="ref325" class="anchor"></a>Hoffee, P. A. <i>1968</i>. 2-Deoxyribose gene-enzyme complex in <i>Salmonella typhimurium.</i> <i>J.</i>  <span class="journal">Bacteriol. 95:449–457.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=2-Deoxyribose%20gene-enzyme%20complex%20in%20Salmonella%20typhimurium.%20J.">google scholar</a></td><td><a href="pdf/325.pdf">pdf</a></td></tr>
<tr><td>326</td><td><a id="ref326" class="anchor"></a>Hoffmeyer, J., and J. Neuhard. <i>1971</i>. Metabolism of exogenous purine bases and nucleosides by <i>Salmonella typhimurium.</i>  <span class="journal">J. Bacteriol. 106:14–24.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Metabolism%20of%20exogenous%20purine%20bases%20and%20nucleosides%20by%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/326.pdf">pdf</a></td></tr>
<tr><td>327</td><td><a id="ref327" class="anchor"></a>Hofnung, M., and M. Schwartz. <i>1971</i>. Mutations allowing growth on maltose of <i>Escherichia coli</i> K-12 strains with a deleted <i>malT</i> gene.  <span class="journal">Mol. Gen. Genet. 112:117–132.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutations%20allowing%20growth%20on%20maltose%20of%20Escherichia%20coli%20K-12%20strains%20with%20a%20deleted%20malT%20gene">google scholar</a></td><td><a href="pdf/327.pdf">pdf</a></td></tr>
<tr><td>328</td><td><a id="ref328" class="anchor"></a>Holland, J., I. B. Holland, and S. I. Ahmad. <i>1991</i>. DNA damage by 8-methoxypsoralen plus near UV light, <i>puvA</i> and its repair in <i>Escherichia coli:</i> genetic analysis.  <span class="journal">Mutat. Res. 254:289–298.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=DNA%20damage%20by%208-methoxypsoralen%20plus%20near%20UV%20light,%20puvA%20and%20its%20repair%20in%20Escherichia%20coli:%20genetic%20analysis">google scholar</a></td><td><a href="pdf/328.pdf">pdf</a></td></tr>
<tr><td>329</td><td><a id="ref329" class="anchor"></a>Hooper, D. C., J. S. Wolfson, M. A. Bozza, and E. Y. Ng. <i>1992</i>. Genetics and regulation of outer membrane protein expression by quinolone resistance loci <i>nfxB,</i> <i>nfxC,</i> and <i>cfxB.</i>  <span class="journal">Antimicrob. Agents Chemother. 36:1151–1154.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetics%20and%20regulation%20of%20outer%20membrane%20protein%20expression%20by%20quinolone%20resistance%20loci%20nfxB,%20nfxC,%20and%20cfxB.">google scholar</a></td><td><a href="pdf/329.pdf">pdf</a></td></tr>
<tr><td>330</td><td><a id="ref330" class="anchor"></a>Horiuchi, T., S. Horiuchi, and A. Novick. <i>1963</i>. The genetic basis of hyper-synthesis of β- galactoside.  <span class="journal">Genetics 48:157–169.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The%20genetic%20basis%20of%20hyper-synthesis%20of%20β-%20galactoside">google scholar</a></td><td><a href="pdf/330.pdf">pdf</a></td></tr>
<tr><td>331</td><td><a id="ref331" class="anchor"></a>Housley, P. R., A. D. Leavitt, and H. J. Whitfield. <i>1981</i>. Genetic analysis of a temperature- sensitive <i>Salmonella typhimurium</i> <i>rho</i> mutant with an altered Rho-associated polycytidylate- dependent adenosine triphosphatase activity.  <span class="journal">J. Bacteriol. 147:13–24.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20analysis%20of%20a%20temperature-%20sensitive%20Salmonella%20typhimurium%20rho%20mutant%20with%20an%20altered%20Rho-associated%20polycytidylate-%20dependent%20adenosine%20triphosphatase%20activity">google scholar</a></td><td><a href="pdf/331.pdf">pdf</a></td></tr>
<tr><td>332</td><td><a id="ref332" class="anchor"></a>Howard-Flanders, P., and L. Theriot. <i>1962</i>. A method for selecting radiation-sensitive mutants of <i>Escherichia coli.</i>  <span class="journal">Genetics 47:1219–1224.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20method%20for%20selecting%20radiation-sensitive%20mutants%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/332.pdf">pdf</a></td></tr>
<tr><td>333</td><td><a id="ref333" class="anchor"></a>Hrebenda, J., H. Heleszko, K. Brzostek, and J. Bielecki. <i>1985</i>. Mutation affecting resistance of <i>Escherichia coli</i> K12 to nalidixic acid.  <span class="journal">J. Gen. Microbiol. 131:2285–2292.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutation%20affecting%20resistance%20of%20Escherichia%20coli%20K12%20to%20nalidixic%20acid">google scholar</a></td><td><a href="pdf/333.pdf">pdf</a></td></tr>
<tr><td>334</td><td><a id="ref334" class="anchor"></a>Hudson, H. P., A. A. Lindberg, and B. A. D. Stocker. <i>1978</i>. Lipopolysaccharide core defects in <i>Salmonella typhimurium</i> mutants which are resistant to Felix O phage but retain smooth character.  <span class="journal">J. Gen. Microbiol. 109:97–112.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Lipopolysaccharide%20core%20defects%20in%20Salmonella%20typhimurium%20mutants%20which%20are%20resistant%20to%20Felix%20O%20phage%20but%20retain%20smooth%20character">google scholar</a></td><td><a href="pdf/334.pdf">pdf</a></td></tr>
<tr><td>335</td><td><a id="ref335" class="anchor"></a>Hughes, D. <i>1986</i>. The isolation and mapping of EF-Tu mutations in <i>Salmonella typhimurium.</i>  <span class="journal">Mol. Gen. Genet. 202:108–111.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The%20isolation%20and%20mapping%20of%20EF-Tu%20mutations%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/335.pdf">pdf</a></td></tr>
<tr><td>336</td><td><a id="ref336" class="anchor"></a>Hughes, K. T., B. T. Cookson, D. Ladika, B. M. Olivera, and J. R. Roth. <i>1983</i>. 6- Aminonicotinamide-resistant mutants of <i>Salmonella typhimurium.</i>  <span class="journal">J. Bacteriol. 154:1126–1136.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=6-%20Aminonicotinamide-resistant%20mutants%20of%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/336.pdf">pdf</a></td></tr>
<tr><td>337</td><td><a id="ref337" class="anchor"></a>Hughes, K. T., D. Ladika, J. R. Roth, and B. M. Olivera. <i>1983</i>. An indispensable gene for NAD biosynthesis in <i>Salmonella typhimurium.</i>  <span class="journal">J. Bacteriol. 155:213–221.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=An%20indispensable%20gene%20for%20NAD%20biosynthesis%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/337.pdf">pdf</a></td></tr>
<tr><td>338</td><td><a id="ref338" class="anchor"></a>Hulanicka, D. <i>1972</i>. Resistance to sulfate analogs in <i>Salmonella typhimurium.</i>  <span class="journal">Acta Biochim. Pol. 19:367–376.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Resistance%20to%20sulfate%20analogs%20in%20Salmonella%20typhimurium.">google scholar</a></td><td></td></tr>
<tr><td>339</td><td><a id="ref339" class="anchor"></a>Hulanicka, D., and T. Klopotowsi. <i>1972</i>. Mutants of <i>Salmonella typhimurium</i> resistant to triazole.  <span class="journal">Acta Biochim. Pol. 19:251–260.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutants%20of%20Salmonella%20typhimurium%20resistant%20to%20triazole">google scholar</a></td><td><a href="pdf/339.pdf">pdf</a></td></tr>
<tr><td>340</td><td><a id="ref340" class="anchor"></a>Hulanicka, M. D., S. G. Hallquist, N. M. Kredich, and T. Mojica-A. <i>1979</i>. Regulation of O- acetylserine sulfhydrylase B by L-cysteine in <i>Salmonella typhimurium.</i>  <span class="journal">J. Bacteriol. 140:141–146.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation%20of%20O-%20acetylserine%20sulfhydrylase%20B%20by%20L-cysteine%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/340.pdf">pdf</a></td></tr>
<tr><td>341</td><td><a id="ref341" class="anchor"></a>Hulanicka, M. D., and N. M. Kredich. <i>1976</i>. A mutation affecting expression of the gene coding for serine transacetylase in <i>Salmonella typhimurium.</i>  <span class="journal">Mol. Gen. Genet. 148:143–148.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20mutation%20affecting%20expression%20of%20the%20gene%20coding%20for%20serine%20transacetylase%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/341.pdf">pdf</a></td></tr>
<tr><td>342</td><td><a id="ref342" class="anchor"></a>Hull, R., J. D. Klinger, and E. E. M. Moody. <i>1976</i>. Isolation and characterization of mutants of <i>Escherichia</i> <i>coli</i> K12 resistant to the new aminoglycoside antibiotic, amikacin.  <span class="journal">J. Gen. Microbiol. 94:389–394.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20and%20characterization%20of%20mutants%20of%20Escherichia%20coli%20K12%20resistant%20to%20the%20new%20aminoglycoside%20antibiotic,%20amikacin">google scholar</a></td><td><a href="pdf/342.pdf">pdf</a></td></tr>
<tr><td>343</td><td><a id="ref343" class="anchor"></a>Hummel, H., W. Piepersberg, and A. Böck. <i>1979</i>. Analysis of lincomycin resistance mutations in <i>Escherichia coli.</i>  <span class="journal">Mol. Gen. Genet. 169:345–347.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Analysis%20of%20lincomycin%20resistance%20mutations%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/343.pdf">pdf</a></td></tr>
<tr><td>344</td><td><a id="ref344" class="anchor"></a>Hussain, M., S. Ichihara, and S. Mizushima. <i>1980</i>. Accumulation of glyceride containing precursor of the outer membrane lipoprotein in the cytoplasmic membrane of <i>Escherichia coli</i> treated with globomycin.  <span class="journal">J. Biol. Chem. 255:3707–3712.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Accumulation%20of%20glyceride%20containing%20precursor%20of%20the%20outer%20membrane%20lipoprotein%20in%20the%20cytoplasmic%20membrane%20of%20Escherichia%20coli%20treated%20with%20globomycin">google scholar</a></td><td><a href="pdf/344.pdf">pdf</a></td></tr>
<tr><td>345</td><td><a id="ref345" class="anchor"></a>Hwang, Y.-W., R. Engel, and B. E. Tropp. <i>1984</i>. Correlation of 3,4-dihydroxybutyl 1- phosphonate resistance with a defect in cardiolipin synthesis in <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 157:846–856.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Correlation%20of%203,4-dihydroxybutyl%201-%20phosphonate%20resistance%20with%20a%20defect%20in%20cardiolipin%20synthesis%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/345.pdf">pdf</a></td></tr>
<tr><td>346</td><td><a id="ref346" class="anchor"></a>Ibe, S. N., A. J. Sinskey, and D. Botstein. <i>1982</i>. Genetic mapping of mutations in a highly radiation-resistant mutant of <i>Salmonella typhimurium</i> LT2.  <span class="journal">J. Bacteriol. 152:260–268.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20mapping%20of%20mutations%20in%20a%20highly%20radiation-resistant%20mutant%20of%20Salmonella%20typhimurium%20LT2">google scholar</a></td><td><a href="pdf/346.pdf">pdf</a></td></tr>
<tr><td>347</td><td><a id="ref347" class="anchor"></a>Icho, T., and T. Iino. <i>1978</i>. Isolation and characterization of motile <i>Escherichia coli</i> mutants resistant to bacteriophage χ.  <span class="journal">J. Bacteriol. 134:854–860.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20and%20characterization%20of%20motile%20Escherichia%20coli%20mutants%20resistant%20to%20bacteriophage%20χ">google scholar</a></td><td><a href="pdf/347.pdf">pdf</a></td></tr>
<tr><td>348</td><td><a id="ref348" class="anchor"></a>Igarishi, K., S. Hiraga, and T. Yura. <i>1967</i>. A deoxythymidine kinase deficient mutant of <i>Escherichia coli.</i> II. Mapping and transduction studies with phage φ80.  <span class="journal">Genetics 57:643–654.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20deoxythymidine%20kinase%20deficient%20mutant%20of%20Escherichia%20coli.%20II.%20Mapping%20and%20transduction%20studies%20with%20phage%20φ80">google scholar</a></td><td><a href="pdf/348.pdf">pdf</a></td></tr>
<tr><td>349</td><td><a id="ref349" class="anchor"></a>Iino, T. <i>1977</i>. Genetics of structure and function of bacterial flagella.  <span class="journal">Annu. Rev. Genet. 11:161–182.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetics%20of%20structure%20and%20function%20of%20bacterial%20flagella">google scholar</a></td><td><a href="pdf/349.pdf">pdf</a></td></tr>
<tr><td>350</td><td><a id="ref350" class="anchor"></a>Ilag, L. L., D. Jahn, G. Eggertsson, and D. Söll. <i>1991</i>. The <i>Escherichia coli</i> <i>hemL</i> gene encodes glutamate 1-semialdehyde aminotransferase.  <span class="journal">J. Bacteriol. 173:3408–3413.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The%20Escherichia%20coli%20hemL%20gene%20encodes%20glutamate%201-semialdehyde%20aminotransferase">google scholar</a></td><td><a href="pdf/350.pdf">pdf</a></td></tr>
<tr><td>351</td><td><a id="ref351" class="anchor"></a>Im, S. W. K., and J. Pittard. <i>1971</i>. Phenylalanine biosynthesis in <i>Escherichia coli</i> K-12: mutants derepressed for chorismate mutase P-prephenate dehydratase.  <span class="journal">J. Bacteriol. 106:784–790.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Phenylalanine%20biosynthesis%20in%20Escherichia%20coli%20K-12:%20mutants%20derepressed%20for%20chorismate%20mutase%20P-prephenate%20dehydratase">google scholar</a></td><td><a href="pdf/351.pdf">pdf</a></td></tr>
<tr><td>352</td><td><a id="ref352" class="anchor"></a>Inoko, H., and M. Imai. <i>1976</i>. Isolation and genetic characterization of the <i>nitA</i> mutants of <i>Escherichia coli</i> affecting the termination factor rho.  <span class="journal">Mol. Gen. Genet. 143:211–221.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20and%20genetic%20characterization%20of%20the%20nitA%20mutants%20of%20Escherichia%20coli%20affecting%20the%20termination%20factor%20rho">google scholar</a></td><td><a href="pdf/352.pdf">pdf</a></td></tr>
<tr><td>353</td><td><a id="ref353" class="anchor"></a>Inoko, H., K. Shigesada, and M. Imai. <i>1977</i>. Isolation and characterization of conditional-lethal <i>rho</i> mutants of <i>Escherichia coli.</i>  <span class="journal">Proc. Natl. Acad. Sci. USA 74:1162–1166.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20and%20characterization%20of%20conditional-lethal%20rho%20mutants%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/353.pdf">pdf</a></td></tr>
<tr><td>354</td><td><a id="ref354" class="anchor"></a>Ishidsu, J.-I. <i>1973</i>. Device of a method for isolation of arginine sensitive mutants in <i>Salmonella</i> <i>typhimurium.</i>  <span class="journal">Jpn. J. Genet. 48:377–379.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Device%20of%20a%20method%20for%20isolation%20of%20arginine%20sensitive%20mutants%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/354.pdf">pdf</a></td></tr>
<tr><td>355</td><td><a id="ref355" class="anchor"></a>Ito, M., and Y. Ohnishi. <i>1981</i>. Isolation of <i>Escherichia coli</i> mutants which are resistant to an inhibitor of proton-ATPase, tributyltin and also to uncouplers of oxidative phosphorylation.  <span class="journal">FEBS Lett. 136:225–230.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20of%20Escherichia%20coli%20mutants%20which%20are%20resistant%20to%20an%20inhibitor%20of%20proton-ATPase,%20tributyltin%20and%20also%20to%20uncouplers%20of%20oxidative%20phosphorylation">google scholar</a></td><td><a href="pdf/355.pdf">pdf</a></td></tr>
<tr><td>356</td><td><a id="ref356" class="anchor"></a>Ivell, R., O. Fasano, J.-B. Crechet, and A. Parmeggiani. <i>1981</i>. Characterization of a kirromycin resistant elongation factor Tu from <i>Escherichia coli.</i>  <span class="journal">Biochemistry 20:1355–1361.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Characterization%20of%20a%20kirromycin%20resistant%20elongation%20factor%20Tu%20from%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/356.pdf">pdf</a></td></tr>
<tr><td>357</td><td><a id="ref357" class="anchor"></a>Iwakura, Y., A. Ishihama, and T. Yura. <i>1973</i>. RNA polymerase mutants of <i>Escherichia coli.</i> II. Streptolydigin resistance and its relation to rifampicin resistance.  <span class="journal">Mol. Gen. Genet. 121:181–196.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=RNA%20polymerase%20mutants%20of%20Escherichia%20coli.%20II.%20Streptolydigin%20resistance%20and%20its%20relation%20to%20rifampicin%20resistance">google scholar</a></td><td><a href="pdf/357.pdf">pdf</a></td></tr>
<tr><td>358</td><td><a id="ref358" class="anchor"></a>Iwaya, M., R. Goldman, D. J. Tipper, B. Feingold, and J. L. Strominger. <i>1978</i>. Morphology of an <i>Escherichia coli</i> mutant with a temperature-dependent round cell shape.  <span class="journal">J. Bacteriol. 136:1143– 1158.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Morphology%20of%20an%20Escherichia%20coli%20mutant%20with%20a%20temperature-dependent%20round%20cell%20shape">google scholar</a></td><td><a href="pdf/358.pdf">pdf</a></td></tr>
<tr><td>359</td><td><a id="ref359" class="anchor"></a>Iwaya, M., C. W. Jones, J. Khorana, and J. L. Strominger. <i>1978</i>. Mapping of the mecillinam- resistant, round morphological mutants of <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 133:196–202.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mapping%20of%20the%20mecillinam-%20resistant,%20round%20morphological%20mutants%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/359.pdf">pdf</a></td></tr>
<tr><td>360</td><td><a id="ref360" class="anchor"></a>Jackson, B. J., J.-P. Bohin, and E. P. Kennedy. <i>1984</i>. Biosynthesis of membrane-derived oligosaccharides: characterization of <i>mdoB</i> mutants defective in phosphoglycerol transferase I activity.  <span class="journal">J. Bacteriol. 160:976–981.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Biosynthesis%20of%20membrane-derived%20oligosaccharides:%20characterization%20of%20mdoB%20mutants%20defective%20in%20phosphoglycerol%20transferase%20I%20activity">google scholar</a></td><td><a href="pdf/360.pdf">pdf</a></td></tr>
<tr><td>361</td><td><a id="ref361" class="anchor"></a>Jackson, J. H., E. J. Davis, A. C. Madu, and S. E. Braxter. <i>1981</i>. Three-factor reciprocal cross mapping of a gene that causes expression of feedback resistant acetohydroxy acid synthase in <i>Escherichia coli</i> K-12.  <span class="journal">Mol. Gen. Genet. 181:417–419.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Three-factor%20reciprocal%20cross%20mapping%20of%20a%20gene%20that%20causes%20expression%20of%20feedback%20resistant%20acetohydroxy%20acid%20synthase%20in%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/361.pdf">pdf</a></td></tr>
<tr><td>362</td><td><a id="ref362" class="anchor"></a>Jaffe, A., Y. A. Chabbert, and E. Derlot. <i>1983</i>. Selection and characterization of beta-lactam- resistant <i>Escherichia coli</i> K-12 mutants.  <span class="journal">Antimicrob. Agents Chemother. 23:622–625.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Selection%20and%20characterization%20of%20beta-lactam-%20resistant%20Escherichia%20coli%20K-12%20mutants">google scholar</a></td><td><a href="pdf/362.pdf">pdf</a></td></tr>
<tr><td>363</td><td><a id="ref363" class="anchor"></a>Jaffe, A., Y. A. Chabbert, and O. Semonin. <i>1982</i>. Role of porin proteins <i>ompF</i> and <i>ompC</i> in the permeation of beta lactams.  <span class="journal">Antimicrob. Agents Chemother. 22:942–948.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Role%20of%20porin%20proteins%20ompF%20and%20ompC%20in%20the%20permeation%20of%20beta%20lactams">google scholar</a></td><td><a href="pdf/363.pdf">pdf</a></td></tr>
<tr><td>364</td><td><a id="ref364" class="anchor"></a>Janzer, J. J., H. Stan-Lotter, and K. E. Sanderson. <i>1981</i>. Isolation and characterization of hemin-permeable envelope-defective mutants of <i>Salmonella typhimurium.</i>  <span class="journal">Can. J. Microbiol. 27:226– 237.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20and%20characterization%20of%20hemin-permeable%20envelope-defective%20mutants%20of%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/364.pdf">pdf</a></td></tr>
<tr><td>365</td><td><a id="ref365" class="anchor"></a>Jegede, V. A., F. Spencer, and J. E. Brenchley. <i>1976</i>. Thialysine-resistant mutant of <i>Salmonella</i> <i>typhimurium</i> with a lesion in the <i>thrA</i> gene.  <span class="journal">Genetics 83:619–632.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Thialysine-resistant%20mutant%20of%20Salmonella%20typhimurium%20with%20a%20lesion%20in%20the%20thrA%20gene">google scholar</a></td><td><a href="pdf/365.pdf">pdf</a></td></tr>
<tr><td>366</td><td><a id="ref366" class="anchor"></a>Jenness, D. D., and H. K. Schachman. <i>1980</i>. <i>pyrB</i> mutations as suppressors of arginine auxotrophy in <i>Salmonella</i> <i>typhimurium.</i>  <span class="journal">J. Bacteriol. 141:33–40.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=pyrB%20mutations%20as%20suppressors%20of%20arginine%20auxotrophy%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/366.pdf">pdf</a></td></tr>
<tr><td>367</td><td><a id="ref367" class="anchor"></a>Jensen, K. F. <i>1979</i>. Apparent involvement of purines in the control of expression of <i>Salmonella</i> <i>typhimurium pyr</i> genes: analysis of a leaky <i>guaB</i> mutant resistant to pyrimidine analogs.  <span class="journal">J. Bacteriol. 138:731–738.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Apparent%20involvement%20of%20purines%20in%20the%20control%20of%20expression%20of%20Salmonella%20typhimurium%20pyr%20genes:%20analysis%20of%20a%20leaky%20guaB%20mutant%20resistant%20to%20pyrimidine%20analogs">google scholar</a></td><td><a href="pdf/367.pdf">pdf</a></td></tr>
<tr><td>368</td><td><a id="ref368" class="anchor"></a>Jensen, K. F., J. Neuhard, and L. Shack. <i>1982</i>. RNA polymerase involvement in the control of <i>Salmonella typhimurium pyr</i> gene expression. Isolation and characteristics of a fluorouracil resistant mutant with high, constitutive expression of <i>pyrB</i> and <i>pyrE</i> due to a mutation in <i>rpoBC.</i>  <span class="journal">EMBO J. 1:69–74.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=RNA%20polymerase%20involvement%20in%20the%20control%20of%20Salmonella%20typhimurium%20pyr%20gene%20expression.%20Isolation%20and%20characteristics%20of%20a%20fluorouracil%20resistant%20mutant%20with%20high,%20constitutive%20expression%20of%20pyrB%20and%20pyrE%20due%20to%20a%20mutation%20in%20rpoBC.">google scholar</a></td><td><a href="pdf/368.pdf">pdf</a></td></tr>
<tr><td>369</td><td><a id="ref369" class="anchor"></a>Jin, D. J., and C. A. Gross. <i>1988</i>. Mapping and sequencing of mutations in the <i>Escherichia coli</i> <i>rpoB</i> gene that lead to rifampicin resistance.  <span class="journal">J. Mol. Biol. 202:45–58.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mapping%20and%20sequencing%20of%20mutations%20in%20the%20Escherichia%20coli%20rpoB%20gene%20that%20lead%20to%20rifampicin%20resistance">google scholar</a></td><td><a href="pdf/369.pdf">pdf</a></td></tr>
<tr><td>370</td><td><a id="ref370" class="anchor"></a>Jochimsen, B., P. Nygaard, and T. Vestergaard. <i>1975</i>. Location on the chromosome of <i>Escherichia</i> coli of genes governing purine metabolism: adenosine deaminase <i>(add),</i> guanosine kinase <i>(gsk)</i> and hypoxanthine phosphoribosyltransferase <i>(hpt).</i>  <span class="journal">Mol. Gen. Genet. 143:85–91.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Location%20on%20the%20chromosome%20of%20Escherichia%20coli%20of%20genes%20governing%20purine%20metabolism:%20adenosine%20deaminase%20(add),%20guanosine%20kinase%20(gsk)%20and%20hypoxanthine%20phosphoribosyltransferase%20(hpt).">google scholar</a></td><td><a href="pdf/370.pdf">pdf</a></td></tr>
<tr><td>371</td><td><a id="ref371" class="anchor"></a>Jochimsen, B. U., B. Hove-Jensen, B. B. Carber, and J. S. Gots. <i>1985</i>. Characteristics of a <i>Salmonella typhimurium</i> mutant defective in phosphoribosylpyrophosphate synthetase.  <span class="journal">J. Gen. Microbiol. 131:245–252.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Characteristics%20of%20a%20Salmonella%20typhimurium%20mutant%20defective%20in%20phosphoribosylpyrophosphate%20synthetase">google scholar</a></td><td><a href="pdf/371.pdf">pdf</a></td></tr>
<tr><td>372</td><td><a id="ref372" class="anchor"></a>Johansson, V., A. Aarti, M. Nurminen, and P. H. Makela. <i>1978</i>. Outer membrane protein- specific bacteriophages of <i>Salmonella typhimurium.</i>  <span class="journal">J. Gen. Microbiol. 107:183–187.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Outer%20membrane%20protein-%20specific%20bacteriophages%20of%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/372.pdf">pdf</a></td></tr>
<tr><td>373</td><td><a id="ref373" class="anchor"></a>Johnson, B. F. <i>1977</i>. Fine structure mapping and properties of mutations suppressing the <i>lon</i> mutation in <i>Escherichia coli</i> K-12 and B strains.  <span class="journal">Genet. Res. 30:273–286.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Fine%20structure%20mapping%20and%20properties%20of%20mutations%20suppressing%20the%20lon%20mutation%20in%20Escherichia%20coli%20K-12%20and%20B%20strains">google scholar</a></td><td><a href="pdf/373.pdf">pdf</a></td></tr>
<tr><td>374</td><td><a id="ref374" class="anchor"></a>Johnston, H. M., and J. R. Roth. <i>1979</i>. Histidine mutants requiring adenine: selection of mutants with reduced <i>hisG</i> expression in <i>Salmonella typhimurium.</i>  <span class="journal">Genetics 92:1–15.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Histidine%20mutants%20requiring%20adenine:%20selection%20of%20mutants%20with%20reduced%20hisG%20expression%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/374.pdf">pdf</a></td></tr>
<tr><td>375</td><td><a id="ref375" class="anchor"></a>Johnston, H. M., and J. R. Roth. <i>1981</i>. Genetic analysis of the histidine operon control region of <i>Salmonella typhimurium.</i>  <span class="journal">J. Mol. Biol. 145:713—734.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20analysis%20of%20the%20histidine%20operon%20control%20region%20of%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/375.pdf">pdf</a></td></tr>
<tr><td>376</td><td><a id="ref376" class="anchor"></a>Johnston, M. A., and H. Pivnick. <i>1970</i>. Use of autocytotoxic β-D-galactosides for selective growth of <i>Salmonella typhimurium</i> in the presence of coliforms.  <span class="journal">Can. J. Microbiol. 16:83–89.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Use%20of%20autocytotoxic%20β-D-galactosides%20for%20selective%20growth%20of%20Salmonella%20typhimurium%20in%20the%20presence%20of%20coliforms">google scholar</a></td><td><a href="pdf/376.pdf">pdf</a></td></tr>
<tr><td>377</td><td><a id="ref377" class="anchor"></a>Jones-Mortimer, M., and H. L. Kornberg. <i>1974</i>. Genetical analysis of fructose utilization by <i>Escherichia coli.</i>  <span class="journal">Proc. R. Soc. London Ser. B 187:121–131.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetical%20analysis%20of%20fructose%20utilization%20by%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/377.pdf">pdf</a></td></tr>
<tr><td>378</td><td><a id="ref378" class="anchor"></a>Jones-Mortimer, M. C., and H. L. Kornberg. <i>1976</i>. Uptake of fructose by the sorbitol phosphotransferase of <i>Escherichia coli</i> K12.  <span class="journal">J. Gen. Microbiol. 96:383–391.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Uptake%20of%20fructose%20by%20the%20sorbitol%20phosphotransferase%20of%20Escherichia%20coli%20K12">google scholar</a></td><td><a href="pdf/378.pdf">pdf</a></td></tr>
<tr><td>379</td><td><a id="ref379" class="anchor"></a>Jones-Mortimer, M. C., and H. L Kornberg. <i>1980</i>. Amino-sugar transport systems of <i>Escherichia coli</i> K12.  <span class="journal">J. Gen. Microbiol. 117:369–376.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Amino-sugar%20transport%20systems%20of%20Escherichia%20coli%20K12">google scholar</a></td><td><a href="pdf/379.pdf">pdf</a></td></tr>
<tr><td>380</td><td><a id="ref380" class="anchor"></a>Josephsen, J., K. Hammer-Jespersen, and T. D. Hansen. <i>1983</i>. Mapping of the gene for cytidine deaminase <i>(cdd)</i> in <i>Escherichia coli</i> K-12.  <span class="journal">J. Bacteriol. 154:72–75.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mapping%20of%20the%20gene%20for%20cytidine%20deaminase%20(cdd)%20in%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/380.pdf">pdf</a></td></tr>
<tr><td>381</td><td><a id="ref381" class="anchor"></a>Joshi, A., J. Z. Siddiqui, M. Verma, and M. Chakravorty. <i>1982</i>. Participation of host proteins in the morphogenesis of bacteriophage P22.  <span class="journal">Mol. Gen. Genet. 186:44–49.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Participation%20of%20host%20proteins%20in%20the%20morphogenesis%20of%20bacteriophage%20P22">google scholar</a></td><td><a href="pdf/381.pdf">pdf</a></td></tr>
<tr><td>382</td><td><a id="ref382" class="anchor"></a>Joshi, A., M. Verma, and M. Chakravorty. <i>1982</i>. Thiolutin-resistant mutants of <i>Salmonella</i> <i>typhimurium.</i>  <span class="journal">Antimicrob. Agents Chemother. 22:541–547.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Thiolutin-resistant%20mutants%20of%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/382.pdf">pdf</a></td></tr>
<tr><td>383</td><td><a id="ref383" class="anchor"></a>Juhl, M. J., and D. P. Clark. <i>1990</i>. Thiophene-degrading <i>Escherichia coli</i> mutants possess sulfone oxidase activity and show altered resistance to sulfur-containing antibiotics.  <span class="journal">Appl. Environ. Microbiol. 56:3179–3185.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Thiophene-degrading%20Escherichia%20coli%20mutants%20possess%20sulfone%20oxidase%20activity%20and%20show%20altered%20resistance%20to%20sulfur-containing%20antibiotics">google scholar</a></td><td><a href="pdf/383.pdf">pdf</a></td></tr>
<tr><td>384</td><td><a id="ref384" class="anchor"></a>Jung, J. U., C. Gutierrez, and M. R. Villarejo. <i>1989</i>. Sequence of an osmotically inducible lipoprotein gene.  <span class="journal">J. Bacteriol. 171:511–520.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Sequence%20of%20an%20osmotically%20inducible%20lipoprotein%20gene">google scholar</a></td><td><a href="pdf/384.pdf">pdf</a></td></tr>
<tr><td>385</td><td><a id="ref385" class="anchor"></a>Justesen, J., and J. Neuhard. <i>1975</i>. <i>pyrR</i> identical to <i>pyrH</i> in <i>Salmonella typhimurium:</i> control of expression of the <i>pyr</i> genes.  <span class="journal">J. Bacteriol. 123:851–854.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=pyrR%20identical%20to%20pyrH%20in%20Salmonella%20typhimurium:%20control%20of%20expression%20of%20the%20pyr%20genes">google scholar</a></td><td><a href="pdf/385.pdf">pdf</a></td></tr>
<tr><td>386</td><td><a id="ref386" class="anchor"></a>Kadner, R. J. <i>1977</i>. Transport and utilization of D-methionine and other methionine sources in <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 129:207–216.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Transport%20and%20utilization%20of%20D-methionine%20and%20other%20methionine%20sources%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/386.pdf">pdf</a></td></tr>
<tr><td>387</td><td><a id="ref387" class="anchor"></a>Kadner, R. J., K. Heller, J. W. Coulton, and V. Braun. <i>1980</i>. Genetic control of hydroxamate- mediated iron uptake in <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 143:256–264.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20control%20of%20hydroxamate-%20mediated%20iron%20uptake%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/387.pdf">pdf</a></td></tr>
<tr><td>388</td><td><a id="ref388" class="anchor"></a>Kadner, R. J., and C. L Liggins. <i>1973</i>. Transport of vitamin B12 in <i>Escherichia coli:</i> genetic studies.  <span class="journal">J. Bacteriol. 115:514–521.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Transport%20of%20vitamin%20B12%20in%20Escherichia%20coli:%20genetic%20studies">google scholar</a></td><td><a href="pdf/388.pdf">pdf</a></td></tr>
<tr><td>389</td><td><a id="ref389" class="anchor"></a>Kadner, R. J., and D. M. Shattuck-Eidens. <i>1983</i>. Genetic control of the hexose phosphate transport system of <i>Escherichia coli:</i> mapping of deletion and insertion mutations in the <i>uhp</i> region.  <span class="journal">J. Bacteriol. 155:1052–1061.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20control%20of%20the%20hexose%20phosphate%20transport%20system%20of%20Escherichia%20coli:%20mapping%20of%20deletion%20and%20insertion%20mutations%20in%20the%20uhp%20region">google scholar</a></td><td><a href="pdf/389.pdf">pdf</a></td></tr>
<tr><td>390</td><td><a id="ref390" class="anchor"></a>Kalle, C. P., and J. S. Gots. <i>1961</i>. Alterations in purine nucleotide pyrophosphorylases and resistance to purine analogues.  <span class="journal">Biochim. Biophys. Acta 53:66–173.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Alterations%20in%20purine%20nucleotide%20pyrophosphorylases%20and%20resistance%20to%20purine%20analogues">google scholar</a></td><td><a href="pdf/390.pdf">pdf</a></td></tr>
<tr><td>391</td><td><a id="ref391" class="anchor"></a>Kalman, M., D. R. Gentry, and M. Cashel. <i>1991</i>. Characterization of the <i>Escherichia coli</i> K12 <i>gltS</i> glutamate permease gene.  <span class="journal">Mol. Gen. Genet. 225:379–386.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Characterization%20of%20the%20Escherichia%20coli%20K12%20gltS%20glutamate%20permease%20gene">google scholar</a></td><td><a href="pdf/391.pdf">pdf</a></td></tr>
<tr><td>392</td><td><a id="ref392" class="anchor"></a>Kanner, B. I., and D. L. Gutnick. <i>1972</i>. Use of neomycin in the isolation of mutants blocked in energy conservation in <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 111:287–289.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Use%20of%20neomycin%20in%20the%20isolation%20of%20mutants%20blocked%20in%20energy%20conservation%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/392.pdf">pdf</a></td></tr>
<tr><td>393</td><td><a id="ref393" class="anchor"></a>Kaplan, L., H. C. Reilly, and C. C. Stock. <i>1959</i>. Action of azaserine on <i>Escherichia coli.</i> <i>J.</i>  <span class="journal">Bacteriol. 78:511–519.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Action%20of%20azaserine%20on%20Escherichia%20coli.%20J.">google scholar</a></td><td><a href="pdf/393.pdf">pdf</a></td></tr>
<tr><td>394</td><td><a id="ref394" class="anchor"></a>Kaplan, S., and D. Anderson. <i>1968</i>. Selection of temperature-sensitive activating enzyme mutants in <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 95:991–997.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Selection%20of%20temperature-sensitive%20activating%20enzyme%20mutants%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/394.pdf">pdf</a></td></tr>
<tr><td>395</td><td><a id="ref395" class="anchor"></a>Kast, P., and H. Hennecke. <i>1991</i>. Amino acid substrate specificity of <i>Escherichia coli</i> phenylalanyl-tRNA synthetase altered by distinct mutations.  <span class="journal">J. Mol. Biol. 222:99–124.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Amino%20acid%20substrate%20specificity%20of%20Escherichia%20coli%20phenylalanyl-tRNA%20synthetase%20altered%20by%20distinct%20mutations">google scholar</a></td><td><a href="pdf/395.pdf">pdf</a></td></tr>
<tr><td>396</td><td><a id="ref396" class="anchor"></a>Katz, L. <i>1970</i>. Selection of <i>AraB</i> and <i>AraC</i> mutants of <i>Escherichia coli</i> B/r by resistance to ribitol.  <span class="journal">J. Bacteriol. 102:593–595.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Selection%20of%20AraB%20and%20AraC%20mutants%20of%20Escherichia%20coli%20B/r%20by%20resistance%20to%20ribitol">google scholar</a></td><td><a href="pdf/396.pdf">pdf</a></td></tr>
<tr><td>397</td><td><a id="ref397" class="anchor"></a>Kawakami, T., Y. Akizawa, T. Ishikawa, T. Shimamoto, M. Tsuda, and T. Tsuchiya. <i>1988</i>. Amino acid substitutions and alteration in cation specificity in the melibiose carrier of <i>Escherichia coli.</i>  <span class="journal">J. Biol. Chem. 263:14276–14280.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Amino%20acid%20substitutions%20and%20alteration%20in%20cation%20specificity%20in%20the%20melibiose%20carrier%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/397.pdf">pdf</a></td></tr>
<tr><td>398</td><td><a id="ref398" class="anchor"></a>Kawamoto, S., S. Tokuyama, S. Yashima, and Y. Eguchi. <i>1984</i>. Genetic mapping of cold resistance gene of <i>Escherichia coli.</i>  <span class="journal">Agric. Biol. Chem. 48:2067–2071.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20mapping%20of%20cold%20resistance%20gene%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/398.pdf">pdf</a></td></tr>
<tr><td>399</td><td><a id="ref399" class="anchor"></a>Kawasaki, T., and Y. Nose. <i>1969</i>. Thiamine regulatory mutants in <i>Escherichia coli.</i>  <span class="journal">J. Biochem. 65:417–425.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Thiamine%20regulatory%20mutants%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/399.pdf">pdf</a></td></tr>
<tr><td>400</td><td><a id="ref400" class="anchor"></a>Kay, W. W. <i>1972</i>. Genetic control of the metabolism of propionate by <i>Escherichia</i> <i>coli</i> K12.  <span class="journal">Biochim. Biophys. Acta 264:508–521.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20control%20of%20the%20metabolism%20of%20propionate%20by%20Escherichia%20coli%20K12">google scholar</a></td><td><a href="pdf/400.pdf">pdf</a></td></tr>
<tr><td>401</td><td><a id="ref401" class="anchor"></a>Kay, W. W., and H. L. Kornberg. <i>1969</i>. Genetic control of the uptake of Ci-dicarboxylic acids by <i>Escherichia coli.</i>  <span class="journal">FEBS Lett. 3:93–96.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20control%20of%20the%20uptake%20of%20Ci-dicarboxylic%20acids%20by%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/401.pdf">pdf</a></td></tr>
<tr><td>402</td><td><a id="ref402" class="anchor"></a>Kelker, N. E., and W. K. Maas. <i>1974</i>. Selection for genetically repressible (Arg R+) strains of <i>Escherichia coli</i> K12 from genetically derepressed (Arg R-) mutants using acetylnorvaline.  <span class="journal">Mol. Gen. Genet. 131:131–136.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Selection%20for%20genetically%20repressible%20(Arg%20R+)%20strains%20of%20Escherichia%20coli%20K12%20from%20genetically%20derepressed%20(Arg%20R-)%20mutants%20using%20acetylnorvaline">google scholar</a></td><td><a href="pdf/402.pdf">pdf</a></td></tr>
<tr><td>403</td><td><a id="ref403" class="anchor"></a>Kelln, R. A., and G. A. O’Donovan. <i>1976</i>. Isolation and partial characteristics of an <i>argR</i> mutant of <i>Salmonella typhimurium.</i>  <span class="journal">J. Bacteriol. 128:528–535.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20and%20partial%20characteristics%20of%20an%20argR%20mutant%20of%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/403.pdf">pdf</a></td></tr>
<tr><td>404</td><td><a id="ref404" class="anchor"></a>Kelln, R. A., and V. L. Zak. <i>1980</i>. A mutation in <i>Salmonella typhimurium</i> imparting conditional resistance to 5-fluorouracil and a bioenergetic defect mapping of <i>cad.</i>  <span class="journal">Mol. Gen. Genet. 179:678–682.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20mutation%20in%20Salmonella%20typhimurium%20imparting%20conditional%20resistance%20to%205-fluorouracil%20and%20a%20bioenergetic%20defect%20mapping%20of%20cad.">google scholar</a></td><td><a href="pdf/404.pdf">pdf</a></td></tr>
<tr><td>405</td><td><a id="ref405" class="anchor"></a>Kemper, J. <i>1974</i>. Gene order and co-transduction in the <i>leu-ara-fol-pyrA</i> region of the <i>Salmonella</i> <i>typhimurium</i> linkage map.  <span class="journal">J. Bacteriol. 117:94–99.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Gene%20order%20and%20co-transduction%20in%20the%20leu-ara-fol-pyrA%20region%20of%20the%20Salmonella%20typhimurium%20linkage%20map">google scholar</a></td><td><a href="pdf/405.pdf">pdf</a></td></tr>
<tr><td>406</td><td><a id="ref406" class="anchor"></a>Kessler, D. P., and H. V. Rickenberg. <i>1964</i>. A new method for the selection of mutants of <i>Escherichia coli</i> forming β-galactosidase constitutively.  <span class="journal">Biochim. Biophys. Acta 90:609–610.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20new%20method%20for%20the%20selection%20of%20mutants%20of%20Escherichia%20coli%20forming%20β-galactosidase%20constitutively">google scholar</a></td><td><a href="pdf/406.pdf">pdf</a></td></tr>
<tr><td>407</td><td><a id="ref407" class="anchor"></a>Kier, L. D., R. M. Weppelman, and B. N. Ames. <i>1977</i>. Regulation of two phosphatases and a cyclic phosphodiesterase of <i>Salmonella typhimurium.</i>  <span class="journal">J. Bacteriol. 130:420–428.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation%20of%20two%20phosphatases%20and%20a%20cyclic%20phosphodiesterase%20of%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/407.pdf">pdf</a></td></tr>
<tr><td>408</td><td><a id="ref408" class="anchor"></a>Kier, L. D., R. M. Weppelman, and B. N. Ames. <i>1979</i>. Regulation of nonspecific acid phosphatase in <i>Salmonella: phoN</i> and <i>phoP</i> genes.  <span class="journal">J. Bacteriol. 138:155–161.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation%20of%20nonspecific%20acid%20phosphatase%20in%20Salmonella:%20phoN%20and%20phoP%20genes">google scholar</a></td><td><a href="pdf/408.pdf">pdf</a></td></tr>
<tr><td>409</td><td><a id="ref409" class="anchor"></a>Killmann, H., and V. Braun. <i>1992</i>. An aspartate deletion mutation defines a binding site of the multifunctional <i>fhuA</i> outer membrane receptor of <i>Escherichia coli</i> K-12.  <span class="journal">J. Bacteriol. 174:3479–3486.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=An%20aspartate%20deletion%20mutation%20defines%20a%20binding%20site%20of%20the%20multifunctional%20fhuA%20outer%20membrane%20receptor%20of%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/409.pdf">pdf</a></td></tr>
<tr><td>410</td><td><a id="ref410" class="anchor"></a>Kim, K. T., and O. J. Yoo. <i>1987</i>. Inhibition of coliphage N4 infection of <i>Escherichia coli</i> mutant defective in mannose permease.  <span class="journal">Korean J. Microbiol. 25:184–188.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Inhibition%20of%20coliphage%20N4%20infection%20of%20Escherichia%20coli%20mutant%20defective%20in%20mannose%20permease">google scholar</a></td><td></td></tr>
<tr><td>411</td><td><a id="ref411" class="anchor"></a>Kleckner, N., J. Roth, and D. Botstein. <i>1977</i>. Genetic engineering <i>in vivo</i> using translocatable drug-resistance elements. New methods in bacterial genetics.  <span class="journal">J. Mol. Biol. 116:125–159.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20engineering%20in%20vivo%20using%20translocatable%20drug-resistance%20elements.%20New%20methods%20in%20bacterial%20genetics">google scholar</a></td><td><a href="pdf/411.pdf">pdf</a></td></tr>
<tr><td>412</td><td><a id="ref412" class="anchor"></a>Kline, E. L. <i>1972</i>. New amino acid regulatory locus having unusual properties in heterozygous merodiploids.  <span class="journal">J. Bacteriol. 110:1127–1134.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=New%20amino%20acid%20regulatory%20locus%20having%20unusual%20properties%20in%20heterozygous%20merodiploids">google scholar</a></td><td><a href="pdf/412.pdf">pdf</a></td></tr>
<tr><td>413</td><td><a id="ref413" class="anchor"></a>Kocharyan, S. M. <i>1979</i>. Positive method for selection of adenylate cyclase and cyclic adenosine 3′, 5′-monophosphate receptor protein deficient mutants of <i>Escherichia coli</i> K12.  <span class="journal">Biol. Zh. Arm. 32:346–351.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Positive%20method%20for%20selection%20of%20adenylate%20cyclase%20and%20cyclic%20adenosine%203′,%205′-monophosphate%20receptor%20protein%20deficient%20mutants%20of%20Escherichia%20coli%20K12">google scholar</a></td><td></td></tr>
<tr><td>414</td><td><a id="ref414" class="anchor"></a>Kohara, Y., K. Akiyama, and K. Isono. <i>1987</i>. The physical map of the whole <i>E. coli</i> chromosome: application of a new strategy for the rapid analysis and sorting of a large genomic library.  <span class="journal">Cell 50:495–508.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The%20physical%20map%20of%20the%20whole%20E.%20coli%20chromosome:%20application%20of%20a%20new%20strategy%20for%20the%20rapid%20analysis%20and%20sorting%20of%20a%20large%20genomic%20library">google scholar</a></td><td><a href="pdf/414.pdf">pdf</a></td></tr>
<tr><td>415</td><td><a id="ref415" class="anchor"></a>Komatsu, Y., and K. Tanaka. <i>1972</i>. A showdomycin-resistant mutant of <i>Escherichia coli</i> K-12 with altered nucleoside transport character.  <span class="journal">Biochim. Biophys. Acta 288:390–403.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20showdomycin-resistant%20mutant%20of%20Escherichia%20coli%20K-12%20with%20altered%20nucleoside%20transport%20character">google scholar</a></td><td><a href="pdf/415.pdf">pdf</a></td></tr>
<tr><td>416</td><td><a id="ref416" class="anchor"></a>Komeda, Y. <i>1982</i>. Fusions of flagellar operons to lactose genes on a Mu <i>lac</i> bacteriophage. <i>J.</i>  <span class="journal">Bacteriol. 150:16–26.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Fusions%20of%20flagellar%20operons%20to%20lactose%20genes%20on%20a%20Mu%20lac%20bacteriophage.%20J.">google scholar</a></td><td><a href="pdf/416.pdf">pdf</a></td></tr>
<tr><td>417</td><td><a id="ref417" class="anchor"></a>Komeda, Y., M. Silverman, and M. Simon. <i>1977</i>. Genetic analysis of <i>Escherichia coli</i> K-12 region I flagellar mutants.  <span class="journal">J. Bacteriol. 131:801–808.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20analysis%20of%20Escherichia%20coli%20K-12%20region%20I%20flagellar%20mutants">google scholar</a></td><td><a href="pdf/417.pdf">pdf</a></td></tr>
<tr><td>418</td><td><a id="ref418" class="anchor"></a>Kornberg, H. L., and J. Smith. <i>1972</i>. Genetic control of glucose uptake by <i>Escherichia coli.</i>  <span class="journal">FEBS Lett. 20:270–272.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20control%20of%20glucose%20uptake%20by%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/418.pdf">pdf</a></td></tr>
<tr><td>419</td><td><a id="ref419" class="anchor"></a>Kornberg, H. L., and P. D. Watts. <i>1978</i>. Roles of <i>crr</i>-gene products in regulating carbohydrate uptake by <i>Escherichia coli.</i>  <span class="journal">FEBS Lett. 89:329–332.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Roles%20of%20crr-gene%20products%20in%20regulating%20carbohydrate%20uptake%20by%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/419.pdf">pdf</a></td></tr>
<tr><td>420</td><td><a id="ref420" class="anchor"></a>Korteland, J., N. Overbeeke, P. De Graaff, P. Overduin, and B. Lugtenberg. <i>1985</i>. Role of the Arg158 residue of the outer membrane <i>phoE</i> pore protein of <i>Escherichia coli</i> K12 in bacteriophage TC45 recognition and in channel characteristics.  <span class="journal">Eur. J. Biochem. 152:691–697.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Role%20of%20the%20Arg158%20residue%20of%20the%20outer%20membrane%20phoE%20pore%20protein%20of%20Escherichia%20coli%20K12%20in%20bacteriophage%20TC45%20recognition%20and%20in%20channel%20characteristics">google scholar</a></td><td><a href="pdf/420.pdf">pdf</a></td></tr>
<tr><td>421</td><td><a id="ref421" class="anchor"></a>Kowit, J. D., W.-N. Choy, S. P. Champe, and A. L. Goldberg. <i>1976</i>. Role and location of “protease 1” from <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 128:776–784.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Role%20and%20location%20of%20“protease%201”%20from%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/421.pdf">pdf</a></td></tr>
<tr><td>422</td><td><a id="ref422" class="anchor"></a>Krajewska-Grynkiewicz, K., W. Walczak, and T. Klopotowski. <i>1971</i>. Mutants of <i>Salmonella</i> <i>typhimurium</i> able to utilize D-histidine as a source of L-histidine.  <span class="journal">J. Bacteriol. 105:28–37.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutants%20of%20Salmonella%20typhimurium%20able%20to%20utilize%20D-histidine%20as%20a%20source%20of%20L-histidine">google scholar</a></td><td><a href="pdf/422.pdf">pdf</a></td></tr>
<tr><td>423</td><td><a id="ref423" class="anchor"></a>Kramer, G. F., and B. N. Ames. <i>1988</i>. Mechanisms of mutagenicity and toxicity of sodium selenite in <i>Salmonella typhimurium.</i>  <span class="journal">Mutat. Res. 201:169–180.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mechanisms%20of%20mutagenicity%20and%20toxicity%20of%20sodium%20selenite%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/423.pdf">pdf</a></td></tr>
<tr><td>424</td><td><a id="ref424" class="anchor"></a>Kramer, G. F., J. C. Baker, and B. N. Ames. <i>1988</i>. Near-UV stress in <i>Salmonella typhimurium:</i> 4-thiouridine in tRNA, ppGpp, and ApppGpp as components of an adaptive response.  <span class="journal">J. Bacteriol. 170: 2344–2351.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Near-UV%20stress%20in%20Salmonella%20typhimurium:%204-thiouridine%20in%20tRNA,%20ppGpp,%20and%20ApppGpp%20as%20components%20of%20an%20adaptive%20response">google scholar</a></td><td><a href="pdf/424.pdf">pdf</a></td></tr>
<tr><td>425</td><td><a id="ref425" class="anchor"></a>Kraus, J., D. Söll, and K. B. Low. <i>1979</i>. Glutamyl-γ-methyl ester acts as a methionine analogue in <i>Escherichia coli:</i> analogue resistant mutants map at the <i>metJ</i> and <i>metK</i> loci.  <span class="journal">Genet. Res. 33:49–55.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Glutamyl-γ-methyl%20ester%20acts%20as%20a%20methionine%20analogue%20in%20Escherichia%20coli:%20analogue%20resistant%20mutants%20map%20at%20the%20metJ%20and%20metK%20loci">google scholar</a></td><td><a href="pdf/425.pdf">pdf</a></td></tr>
<tr><td>426</td><td><a id="ref426" class="anchor"></a>Kricker, M., and B. G. Hall. <i>1984</i>. Directed evolution of cellobiose utilization in <i>Escherichia coli</i> K12.  <span class="journal">Mol. Biol. Evol. 1:171–182.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Directed%20evolution%20of%20cellobiose%20utilization%20in%20Escherichia%20coli%20K12">google scholar</a></td><td><a href="pdf/426.pdf">pdf</a></td></tr>
<tr><td>427</td><td><a id="ref427" class="anchor"></a>Kuhn, A. H. U., M. L. J. Moncany, E. Kellenberger, and R. Hausmann. <i>1982</i>. Involvement of the bacterial <i>groM</i> gene product in bacteriophage T7 reproduction.  <span class="journal">J. Virol. 41:657–673.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Involvement%20of%20the%20bacterial%20groM%20gene%20product%20in%20bacteriophage%20T7%20reproduction">google scholar</a></td><td><a href="pdf/427.pdf">pdf</a></td></tr>
<tr><td>428</td><td><a id="ref428" class="anchor"></a>Kuhn, J., and R. L. Somerville. <i>1971</i>. Mutant strains of <i>Escherichia coli</i> K12 that can use D- amino acids.  <span class="journal">Proc. Natl. Acad. Sci. USA 68:2484 2487.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutant%20strains%20of%20Escherichia%20coli%20K12%20that%20can%20use%20D-%20amino%20acids">google scholar</a></td><td><a href="pdf/428.pdf">pdf</a></td></tr>
<tr><td>429</td><td><a id="ref429" class="anchor"></a>Kumar, S. <i>1976</i>. Properties of adenyl cyclase and cyclic adenosine 3′,5′-monophosphate receptor protein-deficient mutants of <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 125:545–555.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Properties%20of%20adenyl%20cyclase%20and%20cyclic%20adenosine%203′,5′-monophosphate%20receptor%20protein-deficient%20mutants%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/429.pdf">pdf</a></td></tr>
<tr><td>430</td><td><a id="ref430" class="anchor"></a>Kunert, K. J., C. F. Cresswell, A. Schmidt, P. M. Mullineaux, and C. H. Foyer. <i>1990</i>. Variations in the activity of glutathione reductase and the cellular glutathione content in relation to sensitivity to methyl viologen in <i>Escherichia coli.</i>  <span class="journal">Arch. Biochem. Biophys. 282:233–238.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Variations%20in%20the%20activity%20of%20glutathione%20reductase%20and%20the%20cellular%20glutathione%20content%20in%20relation%20to%20sensitivity%20to%20methyl%20viologen%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/430.pdf">pdf</a></td></tr>
<tr><td>431</td><td><a id="ref431" class="anchor"></a>Kuroda, M., S. De Waard, K. Mizushima, M. Tsuda, P. Postma, and T. Tsuchiya. <i>1992</i>. Resistance of the melibiose carrier to inhibition by the phosphotransferase system due to substitutions of amino acid residues in the carrier of <i>Salmonella typhimurium.</i>  <span class="journal">J. Biol. Chem. 267:18336–18341.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Resistance%20of%20the%20melibiose%20carrier%20to%20inhibition%20by%20the%20phosphotransferase%20system%20due%20to%20substitutions%20of%20amino%20acid%20residues%20in%20the%20carrier%20of%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/431.pdf">pdf</a></td></tr>
<tr><td>432</td><td><a id="ref432" class="anchor"></a>Kustu, S. G., and G. F.-L. Ames. <i>1973</i>. The <i>hisP</i> protein, a known histidine transport component in <i>Salmonella typhimurium,</i> is also an arginine transport component.  <span class="journal">J. Bacteriol. 116:107–113.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The%20hisP%20protein,%20a%20known%20histidine%20transport%20component%20in%20Salmonella%20typhimurium,%20is%20also%20an%20arginine%20transport%20component">google scholar</a></td><td><a href="pdf/432.pdf">pdf</a></td></tr>
<tr><td>433</td><td><a id="ref433" class="anchor"></a>Kustu, S. G., N. C. McFarland, S. P. Hui, B. Esmon, and G. F.-L. Ames. <i>1979</i>. Nitrogen control in <i>Salmonella typhimurium:</i> coregulation of synthesis of glutamine synthetase and amino acid transport systems.  <span class="journal">J. Bacteriol. 138:218–234.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Nitrogen%20control%20in%20Salmonella%20typhimurium:%20coregulation%20of%20synthesis%20of%20glutamine%20synthetase%20and%20amino%20acid%20transport%20systems">google scholar</a></td><td><a href="pdf/433.pdf">pdf</a></td></tr>
<tr><td>434</td><td><a id="ref434" class="anchor"></a>Kuwano, M., D. Schlessinger, G. Rinaldi, L. Felicetti, and G. P. Tocchini-Valentini. <i>1971</i>. G factor mutants of <i>Escherichia coli:</i> map location and properties.  <span class="journal">Biochem. Biophys. Res. Commun. 42:441–444.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=G%20factor%20mutants%20of%20Escherichia%20coli:%20map%20location%20and%20properties">google scholar</a></td><td><a href="pdf/434.pdf">pdf</a></td></tr>
<tr><td>435</td><td><a id="ref435" class="anchor"></a>Lakshmi, T. M., and R. B. Helling. <i>1976</i>. Selection for citrate synthease deficiency in <i>icd</i> mutants of <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 127:76–83.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Selection%20for%20citrate%20synthease%20deficiency%20in%20icd%20mutants%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/435.pdf">pdf</a></td></tr>
<tr><td>436</td><td><a id="ref436" class="anchor"></a>Lambden, P. R., and J. R. Guest. <i>1976</i>. A novel method for isolating chlorate-resistant mutants of <i>Escherichia coli</i> K12 by anaerobic selection on a lactate plus fumarate medium.  <span class="journal">J. Gen. Microbiol. 93:173–176.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20novel%20method%20for%20isolating%20chlorate-resistant%20mutants%20of%20Escherichia%20coli%20K12%20by%20anaerobic%20selection%20on%20a%20lactate%20plus%20fumarate%20medium">google scholar</a></td><td><a href="pdf/436.pdf">pdf</a></td></tr>
<tr><td>437</td><td><a id="ref437" class="anchor"></a>Langley, D., and J. R. Guest. <i>1974</i>. Biochemical and genetic characteristics of deletion and other mutant strains of <i>Salmonella typhimurium</i> LT2 lacking α-ketoacid dehydrogenase complex activities.  <span class="journal">J. Gen. Microbiol. 82:319–335.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Biochemical%20and%20genetic%20characteristics%20of%20deletion%20and%20other%20mutant%20strains%20of%20Salmonella%20typhimurium%20LT2%20lacking%20α-ketoacid%20dehydrogenase%20complex%20activities">google scholar</a></td><td><a href="pdf/437.pdf">pdf</a></td></tr>
<tr><td>438</td><td><a id="ref438" class="anchor"></a>Langley, D., and J. R. Guest. <i>1977</i>. Biochemical studies of the α-ketoacid dehydrogenase complexes of <i>Escherichia coli</i> K12: isolation and biochemical properties of deletion mutants.  <span class="journal">J. Gen. Microbiol. 99:263–276.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Biochemical%20studies%20of%20the%20α-ketoacid%20dehydrogenase%20complexes%20of%20Escherichia%20coli%20K12:%20isolation%20and%20biochemical%20properties%20of%20deletion%20mutants">google scholar</a></td><td></td></tr>
<tr><td>439</td><td><a id="ref439" class="anchor"></a>Langley, D., and J. R. Guest. <i>1978</i>. Biochemical genetics of the α-keto acid dehydrogenase complexes of <i>Escherichia coli</i> K12: genetic characterization and regulatory properties of deletion mutants.  <span class="journal">J. Gen. Microbiol. 106:103–117.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Biochemical%20genetics%20of%20the%20α-keto%20acid%20dehydrogenase%20complexes%20of%20Escherichia%20coli%20K12:%20genetic%20characterization%20and%20regulatory%20properties%20of%20deletion%20mutants">google scholar</a></td><td><a href="pdf/439.pdf">pdf</a></td></tr>
<tr><td>440</td><td><a id="ref440" class="anchor"></a>Langridge, J. <i>1969</i>. Mutations conferring quantitative and qualitative increases in β-galactosidase activity in <i>Escherichia coli.</i>  <span class="journal">Mol. Gen. Genet. 105:74–83.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutations%20conferring%20quantitative%20and%20qualitative%20increases%20in%20β-galactosidase%20activity%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/440.pdf">pdf</a></td></tr>
<tr><td>441</td><td><a id="ref441" class="anchor"></a>LaRossa, R. A., and J. V. Schloss. <i>1984</i>. The sulfonylurea herbicide sulfometuron methyl is an extremely potent and selective inhibitor of acetolactate synthase in <i>Salmonella typhimurium.</i>  <span class="journal">J. Biol. Chem. 259:8753–8757.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The%20sulfonylurea%20herbicide%20sulfometuron%20methyl%20is%20an%20extremely%20potent%20and%20selective%20inhibitor%20of%20acetolactate%20synthase%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/441.pdf">pdf</a></td></tr>
<tr><td>442</td><td><a id="ref442" class="anchor"></a>LaRossa, R. A., and D. R. Smulski. <i>1984</i>. <i>ilvB</i>-encloded acetolactate synthase is resistant to the herbicide sulfometuron methyl.  <span class="journal">J. Bacteriol. 160:391–394.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=ilvB-encloded%20acetolactate%20synthase%20is%20resistant%20to%20the%20herbicide%20sulfometuron%20methyl">google scholar</a></td><td><a href="pdf/442.pdf">pdf</a></td></tr>
<tr><td>443</td><td><a id="ref443" class="anchor"></a>LaRossa, R. A., T. K. Van Dyk, and D. R. Smulski. <i>1987</i>. Toxic accumulation of α-ketobutyrate caused by inhibition of the branched-chain amino acid biosynthetic enzyme acetolactate synthase in <i>Salmonella typhimurium.</i>  <span class="journal">J. Bacteriol. 169:1372–1378.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Toxic%20accumulation%20of%20α-ketobutyrate%20caused%20by%20inhibition%20of%20the%20branched-chain%20amino%20acid%20biosynthetic%20enzyme%20acetolactate%20synthase%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/443.pdf">pdf</a></td></tr>
<tr><td>444</td><td><a id="ref444" class="anchor"></a>LaRossa, R. A., T. K. Van Dyk, and D. R. Smulski. <i>1990</i>. A need for metabolic insulation: lessons from sulfonylurea genetics. <span class="journal">Biosynthesis of Branched Chain Amino Acids. VCH Publishers, New York.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20need%20for%20metabolic%20insulation:%20lessons%20from%20sulfonylurea%20genetics">google scholar</a></td><td><a href="pdf/444.pdf">pdf</a></td></tr>
<tr><td>445</td><td><a id="ref445" class="anchor"></a>Latil, M., M. Murgier, A. Lazdunski, and C. Lazdunski. <i>1976</i>. Isolation and genetic mapping of <i>Escherichia coli</i> aminopeptidase mutants.  <span class="journal">Mol. Gen. Genet. 148:43–47.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20and%20genetic%20mapping%20of%20Escherichia%20coli%20aminopeptidase%20mutants">google scholar</a></td><td><a href="pdf/445.pdf">pdf</a></td></tr>
<tr><td>446</td><td><a id="ref446" class="anchor"></a>Lavina, M., A. P. Pugsley, and F. Moreno. <i>1986</i>. Identification, mapping, cloning, and characterization of a gene (<i>sbmA</i>) required for microcin B17 action on <i>Escherichia coli</i> K-12.  <span class="journal">J. Gen. Microbiol. 132:1685–1693.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Identification,%20mapping,%20cloning,%20and%20characterization%20of%20a%20gene%20(sbmA)%20required%20for%20microcin%20B17%20action%20on%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/446.pdf">pdf</a></td></tr>
<tr><td>447</td><td><a id="ref447" class="anchor"></a>Lawrence, D. A., D. A. Smith, and R. J. Rowbury. <i>1968</i>. Regulation of methionine synthesis in <i>Salmonella typhimurium:</i> mutants resistant to inhibition by analogues of methionine.  <span class="journal">Genetics 58:473–492.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation%20of%20methionine%20synthesis%20in%20Salmonella%20typhimurium:%20mutants%20resistant%20to%20inhibition%20by%20analogues%20of%20methionine">google scholar</a></td><td><a href="pdf/447.pdf">pdf</a></td></tr>
<tr><td>448</td><td><a id="ref448" class="anchor"></a>Lawther, R. P., D. H. Calhoun, C. W. Adams, C. A. Hauser, J. Gray, and G. W. Hatfield. <i>1981</i>. Molecular basis of valine resistance in <i>Escherichia coli</i> K-12.  <span class="journal">Proc. Natl. Acad. Sci. USA 78:922–925.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Molecular%20basis%20of%20valine%20resistance%20in%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/448.pdf">pdf</a></td></tr>
<tr><td>449</td><td><a id="ref449" class="anchor"></a>Lazzaroni, J. C., and R. C. Portalier. <i>1981</i>. Genetic and biochemical characterization of periplasmic-leaky mutants of <i>Escherichia coli</i> K-12.  <span class="journal">J. Bacteriol. 145:1351–1358.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20and%20biochemical%20characterization%20of%20periplasmic-leaky%20mutants%20of%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/449.pdf">pdf</a></td></tr>
<tr><td>450</td><td><a id="ref450" class="anchor"></a>LeBlanc, D. J., and R. P. Mortlock. <i>1971</i>. Metabolism of D-arabinose: a new pathway in <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 106:90–96.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Metabolism%20of%20D-arabinose:%20a%20new%20pathway%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/450.pdf">pdf</a></td></tr>
<tr><td>451</td><td><a id="ref451" class="anchor"></a>Lee, R. S.-F., J. Pagan, S. Wilke-Mounts, and A. E. Senior. <i>1991</i>. Characterization of <i>Escherichia coli</i> ATP synthase beta-subunit mutations using a chromosomal deletion strain.  <span class="journal">Biochemistry 30:6842–6847.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Characterization%20of%20Escherichia%20coli%20ATP%20synthase%20beta-subunit%20mutations%20using%20a%20chromosomal%20deletion%20strain">google scholar</a></td><td><a href="pdf/451.pdf">pdf</a></td></tr>
<tr><td>452</td><td><a id="ref452" class="anchor"></a>Lehmann, V., E. Rupprecht, and M. J. Osborn. <i>1977</i>. Isolation of mutants conditionally blocked in the biosynthesis of the 3-deoxy-D-manno-octulosonic acid-lipid A part of lipopolysaccharides derived from <i>Salmonella</i> <i>typhimurium.</i>  <span class="journal">Eur. J. Biochem. 76:41–49.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20of%20mutants%20conditionally%20blocked%20in%20the%20biosynthesis%20of%20the%203-deoxy-D-manno-octulosonic%20acid-lipid%20A%20part%20of%20lipopolysaccharides%20derived%20from%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/452.pdf">pdf</a></td></tr>
<tr><td>453</td><td><a id="ref453" class="anchor"></a>Leifer, Z., R. Engel, and B. E. Tropp. <i>1977</i>. Transport of 3,4-dihydroxybutyl-l-phosphonate, an analogue of <i>sn</i>-glycerol 3-phosphate.  <span class="journal">J. Bacteriol. 130:968–971.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Transport%20of%203,4-dihydroxybutyl-l-phosphonate,%20an%20analogue%20of%20sn-glycerol%203-phosphate">google scholar</a></td><td><a href="pdf/453.pdf">pdf</a></td></tr>
<tr><td>454</td><td><a id="ref454" class="anchor"></a>Leisinger, T., D. Haas, and M. P. Hegarty. <i>1972</i>. Indospicine as an arginine antagonist in <i>Escherichia coli</i> and <i>Pseudomonas aeruginosa.</i>  <span class="journal">Biochim. Biophys. Acta 262:214–219.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Indospicine%20as%20an%20arginine%20antagonist%20in%20Escherichia%20coli%20and%20Pseudomonas%20aeruginosa.">google scholar</a></td><td><a href="pdf/454.pdf">pdf</a></td></tr>
<tr><td>455</td><td><a id="ref455" class="anchor"></a>Lemmon, R. D., J. J. Rowe, and G. J. Tritz. <i>1980</i>. Isolation and characterization of mutants of <i>Escherichia coli</i> defective in pyridine nucleotide cycle enyzmes.  <span class="journal">Curr. Microbiol. 4:31–35.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20and%20characterization%20of%20mutants%20of%20Escherichia%20coli%20defective%20in%20pyridine%20nucleotide%20cycle%20enyzmes">google scholar</a></td><td><a href="pdf/455.pdf">pdf</a></td></tr>
<tr><td>456</td><td><a id="ref456" class="anchor"></a>Lengeler, J. <i>1980</i>. Characterization of mutants of <i>Escherichia coli</i> K12, selected by resistance to streptozotocin.  <span class="journal">Mol. Gen. Genet. 179:49–54.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Characterization%20of%20mutants%20of%20Escherichia%20coli%20K12,%20selected%20by%20resistance%20to%20streptozotocin">google scholar</a></td><td><a href="pdf/456.pdf">pdf</a></td></tr>
<tr><td>457</td><td><a id="ref457" class="anchor"></a>Lengeler, J., and H. Steinberger. <i>1978</i>. Analysis of the regulatory mechanisms controlling the synthesis of the hexitol transport systems in <i>Escherichia coli</i> K12.  <span class="journal">Mol. Gen. Genet. 164:163–169.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Analysis%20of%20the%20regulatory%20mechanisms%20controlling%20the%20synthesis%20of%20the%20hexitol%20transport%20systems%20in%20Escherichia%20coli%20K12">google scholar</a></td><td><a href="pdf/457.pdf">pdf</a></td></tr>
<tr><td>458</td><td><a id="ref458" class="anchor"></a>Lerman, R. D., and B. A. D. Stocker. <i>1981</i>. Mutations in <i>Salmonella typhimurium</i> affecting synthesis of lipopolysaccharide core at high temperature.  <span class="journal">Wasmann J. Biol. 39:42–49.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutations%20in%20Salmonella%20typhimurium%20affecting%20synthesis%20of%20lipopolysaccharide%20core%20at%20high%20temperature">google scholar</a></td><td><a href="pdf/458.pdf">pdf</a></td></tr>
<tr><td>459</td><td><a id="ref459" class="anchor"></a>Lester, B., and D. M. Bonner. <i>1957</i>. Genetic control of raffinose utilization in <i>Escherichia coli.</i> <i>J.</i>  <span class="journal">Bacteriol. 73:544–552.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20control%20of%20raffinose%20utilization%20in%20Escherichia%20coli.%20J.">google scholar</a></td><td><a href="pdf/459.pdf">pdf</a></td></tr>
<tr><td>460</td><td><a id="ref460" class="anchor"></a>Levin, D. E., L. J. Marnett, and B. N. Ames. <i>1984</i>. Spontaneous and mutagen-induced deletions: mechanistic studies in <i>Salmonella</i> tester strain TA102.  <span class="journal">Proc. Natl. Acad. Sci. USA 81:4457–4461.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Spontaneous%20and%20mutagen-induced%20deletions:%20mechanistic%20studies%20in%20Salmonella%20tester%20strain%20TA102">google scholar</a></td><td><a href="pdf/460.pdf">pdf</a></td></tr>
<tr><td>461</td><td><a id="ref461" class="anchor"></a>Levine, R. A., and M. W. Taylor. <i>1981</i>. Selection for purine regulatory mutants in an <i>E. coli</i> hypoxanthine phosphoribosyl transferase-guanine phosphoribosyl transferase double mutant.  <span class="journal">Mol. Gen. Genet. 181:313–318.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Selection%20for%20purine%20regulatory%20mutants%20in%20an%20E.%20coli%20hypoxanthine%20phosphoribosyl%20transferase-guanine%20phosphoribosyl%20transferase%20double%20mutant">google scholar</a></td><td><a href="pdf/461.pdf">pdf</a></td></tr>
<tr><td>462</td><td><a id="ref462" class="anchor"></a>Levine, R. A., and M. W. Taylor. <i>1982</i>. Mechanism of adenine toxicity in <i>Escherichia coli.</i> <i>J.</i>  <span class="journal">Bacteriol. 149:923–930.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mechanism%20of%20adenine%20toxicity%20in%20Escherichia%20coli.%20J.">google scholar</a></td><td><a href="pdf/462.pdf">pdf</a></td></tr>
<tr><td>463</td><td><a id="ref463" class="anchor"></a>LeVine, S. M., F. Ardeshir, and G. F.-L. Ames. <i>1980</i>. Isolation and characterization of acetate kinase and phosphotransacetylase mutants of <i>Escherichia coli</i> and <i>Salmonella typhimurium.</i>  <span class="journal">J. Bacteriol. 143:1081–1085.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20and%20characterization%20of%20acetate%20kinase%20and%20phosphotransacetylase%20mutants%20of%20Escherichia%20coli%20and%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/463.pdf">pdf</a></td></tr>
<tr><td>464</td><td><a id="ref464" class="anchor"></a>Lieberman, M. A., and J.-S. Hong. <i>1974</i>. A mutant of <i>Escherichia coli</i> defective in the coupling of metabolic energy to active transport.  <span class="journal">Proc. Natl. Acad. Sci. USA 71:4395–4399.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20mutant%20of%20Escherichia%20coli%20defective%20in%20the%20coupling%20of%20metabolic%20energy%20to%20active%20transport">google scholar</a></td><td><a href="pdf/464.pdf">pdf</a></td></tr>
<tr><td>465</td><td><a id="ref465" class="anchor"></a>Lieberman, M. A., M. Simon, and J.-S. Hong. <i>1977</i>. Characterization of <i>Escherichia coli</i> mutant incapable of maintaining a transmembrane potential.  <span class="journal">J. Biol. Chem. 252:4056–4067.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Characterization%20of%20Escherichia%20coli%20mutant%20incapable%20of%20maintaining%20a%20transmembrane%20potential">google scholar</a></td><td><a href="pdf/465.pdf">pdf</a></td></tr>
<tr><td>466</td><td><a id="ref466" class="anchor"></a>Link, C. D., and A. M. Reiner. <i>1983</i>. Genotypic exclusion: a novel relationship between the ribitol-arabitol and galactitol genes of <i>E. coli.</i>  <span class="journal">Mol. Gen. Genet. 189:337–339.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genotypic%20exclusion:%20a%20novel%20relationship%20between%20the%20ribitol-arabitol%20and%20galactitol%20genes%20of%20E.%20coli.">google scholar</a></td><td><a href="pdf/466.pdf">pdf</a></td></tr>
<tr><td>467</td><td><a id="ref467" class="anchor"></a>Liou, T.-F., M. Yoshikawa, and N. Tanaka. <i>1975</i>. Alteration of ribosomal protein L5 in a thiopeptin resistant mutant of <i>Escherichia coli.</i>  <span class="journal">Biochem. Biophys. Res. Commun. 65:1096–1101.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Alteration%20of%20ribosomal%20protein%20L5%20in%20a%20thiopeptin%20resistant%20mutant%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/467.pdf">pdf</a></td></tr>
<tr><td>468</td><td><a id="ref468" class="anchor"></a>Lipsett, M. N. <i>1978</i>. Enzymes producing 4-thiouridine in <i>Escherichia coli</i> tRNA: approximate chromosomal locations of the genes and enzyme activities in a 4-thiouridine-deficient mutant.  <span class="journal">J. Bacteriol. 135:993–997.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Enzymes%20producing%204-thiouridine%20in%20Escherichia%20coli%20tRNA:%20approximate%20chromosomal%20locations%20of%20the%20genes%20and%20enzyme%20activities%20in%20a%204-thiouridine-deficient%20mutant">google scholar</a></td><td><a href="pdf/468.pdf">pdf</a></td></tr>
<tr><td>469</td><td><a id="ref469" class="anchor"></a>Lisitsyn, N. A., E. D. Sverdlov, E. P. Moiseeva, and V. G. Nikiforov. <i>1985</i>. Localization of a mutation leading to antibiotic streptolydigin resistance of <i>E. coli</i> RNA polymerase in the <i>rpoB</i> gene coding for beta-subunit of the enzyme.  <span class="journal">Bioorg. Khim. 11:132–134.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Localization%20of%20a%20mutation%20leading%20to%20antibiotic%20streptolydigin%20resistance%20of%20E.%20coli%20RNA%20polymerase%20in%20the%20rpoB%20gene%20coding%20for%20beta-subunit%20of%20the%20enzyme">google scholar</a></td><td><a href="pdf/469.pdf">pdf</a></td></tr>
<tr><td>470</td><td><a id="ref470" class="anchor"></a>Livshits, V. A. <i>1976</i>. Effect of 2,6-diaminopurine resistant mutations on the uptake of adenine and adenosine by adenine-requiring strains of <i>Escherichia coli</i> K12.  <span class="journal">Genetika 12:180–182.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Effect%20of%202,6-diaminopurine%20resistant%20mutations%20on%20the%20uptake%20of%20adenine%20and%20adenosine%20by%20adenine-requiring%20strains%20of%20Escherichia%20coli%20K12">google scholar</a></td><td></td></tr>
<tr><td>471</td><td><a id="ref471" class="anchor"></a>Lomax, M. S., and G. R. Greenberg. <i>1968</i>. Characteristics of the <i>deo</i> operon: role in thymine utilization and sensitivity to deoxyribonucleosides.  <span class="journal">J. Bacteriol. 96:501–514.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Characteristics%20of%20the%20deo%20operon:%20role%20in%20thymine%20utilization%20and%20sensitivity%20to%20deoxyribonucleosides">google scholar</a></td><td><a href="pdf/471.pdf">pdf</a></td></tr>
<tr><td>472</td><td><a id="ref472" class="anchor"></a>Lomovskaya, O., and K. Lewis. <i>1992</i>. <i>emr,</i> an <i>Escherichia coli</i> locus for multidrug resistance.  <span class="journal">Proc. Natl. Acad. Sci. USA 89:8938–8942.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=emr,%20an%20Escherichia%20coli%20locus%20for%20multidrug%20resistance">google scholar</a></td><td><a href="pdf/472.pdf">pdf</a></td></tr>
<tr><td>473</td><td><a id="ref473" class="anchor"></a>Long, W. S., C. L. Slayman, and K. B. Low. <i>1978</i>. Production of giant cells of <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 133:995–1007.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Production%20of%20giant%20cells%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/473.pdf">pdf</a></td></tr>
<tr><td>474</td><td><a id="ref474" class="anchor"></a>Loomis, W. F., Jr., and B. Magasanik. <i>1965</i>. Genetic control of catabolite repression of the <i>lac</i> operon in <i>Escherichia coli.</i>  <span class="journal">Biochem. Biophys. Res. Commun. 20:230–234.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20control%20of%20catabolite%20repression%20of%20the%20lac%20operon%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/474.pdf">pdf</a></td></tr>
<tr><td>475</td><td><a id="ref475" class="anchor"></a>Loomis, W. F., Jr., and B. Magasanik. <i>1967</i>. The catabolite repression gene of the <i>lac</i> operon in <i>Escherichia coli.</i>  <span class="journal">J. Mol. Biol. 23:487–494.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The%20catabolite%20repression%20gene%20of%20the%20lac%20operon%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/475.pdf">pdf</a></td></tr>
<tr><td>476</td><td><a id="ref476" class="anchor"></a>Lorowitz, W., and D. Clark. <i>1982</i>. <i>Escherichia coli</i> mutants with a temperature-sensitive alcohol dehydrogenase.  <span class="journal">J. Bacteriol. 152:935–938.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Escherichia%20coli%20mutants%20with%20a%20temperature-sensitive%20alcohol%20dehydrogenase">google scholar</a></td><td><a href="pdf/476.pdf">pdf</a></td></tr>
<tr><td>477</td><td><a id="ref477" class="anchor"></a>Low, K. B., F. Gates, T. Goldstein, and D. Söll. <i>1971</i>. Isolation and partial characterization of temperature-sensitive <i>Escherichia coli</i> mutants with altered leucyl- and seryl-transfer ribonucleic acid synthetases.  <span class="journal">J. Bacteriol. 108:742–750.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20and%20partial%20characterization%20of%20temperature-sensitive%20Escherichia%20coli%20mutants%20with%20altered%20leucyl-%20and%20seryl-transfer%20ribonucleic%20acid%20synthetases">google scholar</a></td><td><a href="pdf/477.pdf">pdf</a></td></tr>
<tr><td>478</td><td><a id="ref478" class="anchor"></a>Luckey, M., and J. B. Neilands. <i>1976</i>. Iron transport in <i>Salmonella typhimurium</i> LT-2: prevention, by ferrichrome, of adsorption of bacteriophages ES18 and ES18.hl to a common cell envelope receptor.  <span class="journal">J. Bacteriol. 127:1036–1037.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Iron%20transport%20in%20Salmonella%20typhimurium%20LT-2:%20prevention,%20by%20ferrichrome,%20of%20adsorption%20of%20bacteriophages%20ES18%20and%20ES18.hl%20to%20a%20common%20cell%20envelope%20receptor">google scholar</a></td><td><a href="pdf/478.pdf">pdf</a></td></tr>
<tr><td>479</td><td><a id="ref479" class="anchor"></a>Luckey, M., J. R. Pollack, R. Wayne, B. N. Ames, and J. B. Neilands. <i>1972</i>. Iron uptake in <i>Salmonella typhimurium:</i> utilization of exogenous siderochromes as iron carriers.  <span class="journal">J. Bacteriol. 111:731–738.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Iron%20uptake%20in%20Salmonella%20typhimurium:%20utilization%20of%20exogenous%20siderochromes%20as%20iron%20carriers">google scholar</a></td><td><a href="pdf/479.pdf">pdf</a></td></tr>
<tr><td>480</td><td><a id="ref480" class="anchor"></a>Lutkenhaus, J. F. <i>1977</i>. Role of a major outer membrane protein in <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 131:631–637.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Role%20of%20a%20major%20outer%20membrane%20protein%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/480.pdf">pdf</a></td></tr>
<tr><td>481</td><td><a id="ref481" class="anchor"></a>Maas, W. K. <i>1952</i>. Pantothenate studies. II. Evidence from mutants for interference by salicylate with pantoate synthesis.  <span class="journal">J. Bacteriol. 63:227–232.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Pantothenate%20studies.%20II.%20Evidence%20from%20mutants%20for%20interference%20by%20salicylate%20with%20pantoate%20synthesis">google scholar</a></td><td><a href="pdf/481.pdf">pdf</a></td></tr>
<tr><td>482</td><td><a id="ref482" class="anchor"></a>Maas, W. K. <i>1961</i>. Repression of arginine formation.  <span class="journal">Cold Spring Harbor Symp. Quant. Biol. 26:183–190.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Repression%20of%20arginine%20formation">google scholar</a></td><td><a href="pdf/482.pdf">pdf</a></td></tr>
<tr><td>483</td><td><a id="ref483" class="anchor"></a>Maas, W. K. <i>1965</i>. Genetic defect affecting an arginine permease and repression of arginine synthesis in <i>E. coli.</i>  <span class="journal">Fed. Proc. 24:1239.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20defect%20affecting%20an%20arginine%20permease%20and%20repression%20of%20arginine%20synthesis%20in%20E.%20coli.">google scholar</a></td><td><a href="pdf/483.pdf">pdf</a></td></tr>
<tr><td>484</td><td><a id="ref484" class="anchor"></a>MacPhee, D. G., V. Krishnapillai, R. J. Roantree, and B. A. D. Stocker. <i>1975</i>. Mutations in <i>Salmonella typhimurium</i> conferring resistance to Felix O phage without loss of smooth character.  <span class="journal">J. Gen. Microbiol. 87:1–10.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutations%20in%20Salmonella%20typhimurium%20conferring%20resistance%20to%20Felix%20O%20phage%20without%20loss%20of%20smooth%20character">google scholar</a></td><td><a href="pdf/484.pdf">pdf</a></td></tr>
<tr><td>485</td><td><a id="ref485" class="anchor"></a>Makela, P. H., M. Sarvas, S. Calcagno, and K. Lounatmaa. <i>1978</i>. Isolation and genetic characterization of polymyxin resistant mutants of <i>Salmonella.</i>  <span class="journal">FEMS Microbiol. Lett. 3:323–326.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20and%20genetic%20characterization%20of%20polymyxin%20resistant%20mutants%20of%20Salmonella.">google scholar</a></td><td><a href="pdf/485.pdf">pdf</a></td></tr>
<tr><td>486</td><td><a id="ref486" class="anchor"></a>Makela, P. H., and B. A. D. Stocker. <i>1969</i>. Genetics of polysaccharide biosynthesis.  <span class="journal">Annu. Rev. Genet. 3:291–322.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetics%20of%20polysaccharide%20biosynthesis">google scholar</a></td><td><a href="pdf/486.pdf">pdf</a></td></tr>
<tr><td>487</td><td><a id="ref487" class="anchor"></a>Maloy, S. R., and W. D. Nunn. <i>1981</i>. Selection for loss of tetracycline resistance by <i>Escherichia</i> <i>coli.</i>  <span class="journal">J. Bacteriol. 145:1110–1112.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Selection%20for%20loss%20of%20tetracycline%20resistance%20by%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/487.pdf">pdf</a></td></tr>
<tr><td>488</td><td><a id="ref488" class="anchor"></a>Maloy, S. R., and W. D. Nunn. <i>1982</i>. Genetic regulation of the glyoxylate shunt in <i>Escherichia</i> <i>coli</i> K-12.  <span class="journal">J. Bacteriol. 149:173–180.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20regulation%20of%20the%20glyoxylate%20shunt%20in%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/488.pdf">pdf</a></td></tr>
<tr><td>489</td><td><a id="ref489" class="anchor"></a>Manning, P. A., A. Puspurs, and P. Reeves. <i>1976</i>. Outer membrane of <i>Escherichia coli</i> K12: isolation of mutants with altered protein 3A by using host range mutants of bacteriophage K3.  <span class="journal">J. Bacteriol. 127:1080–1084.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Outer%20membrane%20of%20Escherichia%20coli%20K12:%20isolation%20of%20mutants%20with%20altered%20protein%203A%20by%20using%20host%20range%20mutants%20of%20bacteriophage%20K3">google scholar</a></td><td><a href="pdf/489.pdf">pdf</a></td></tr>
<tr><td>490</td><td><a id="ref490" class="anchor"></a>Manning, P. A., and P. Reeves. <i>1976</i>. Outer membrane of <i>Escherichia coli</i> K12: <i>tsx</i> mutants (resistant to bacteriophage T6 and colicin K) lack an outer membrane protein.  <span class="journal">Biochem. Biophys. Res. Commun. 71:466–471.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Outer%20membrane%20of%20Escherichia%20coli%20K12:%20tsx%20mutants%20(resistant%20to%20bacteriophage%20T6%20and%20colicin%20K)%20lack%20an%20outer%20membrane%20protein">google scholar</a></td><td><a href="pdf/490.pdf">pdf</a></td></tr>
<tr><td>491</td><td><a id="ref491" class="anchor"></a>Manning, P. A., and P. Reeves. <i>1976</i>. Outer membrane of <i>Escherichia coli</i> K12: differentiation of proteins 3A and 3B on acrylamide gels and further characterization of <i>con</i> <i>tolG</i> mutants.  <span class="journal">J. Bacteriol. 127:1070–1079.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Outer%20membrane%20of%20Escherichia%20coli%20K12:%20differentiation%20of%20proteins%203A%20and%203B%20on%20acrylamide%20gels%20and%20further%20characterization%20of%20con%20tolG%20mutants">google scholar</a></td><td><a href="pdf/491.pdf">pdf</a></td></tr>
<tr><td>492</td><td><a id="ref492" class="anchor"></a>Manning, P. A., and P. Reeves. <i>1978</i>. Outer membrane proteins of <i>Escherichia coli</i> K12. Isolation of a common receptor protein for bacteriophage T-6 and colicin K.  <span class="journal">Mol. Gen. Genet. 158:279–286.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Outer%20membrane%20proteins%20of%20Escherichia%20coli%20K12.%20Isolation%20of%20a%20common%20receptor%20protein%20for%20bacteriophage%20T-6%20and%20colicin%20K">google scholar</a></td><td><a href="pdf/492.pdf">pdf</a></td></tr>
<tr><td>493</td><td><a id="ref493" class="anchor"></a>Manoil, C., and J. P. Rosenbusch. <i>1982</i>. Conjugation deficient mutants of <i>Escherichia coli</i> distinguish classes of functions of the outer membrane <i>ompA</i> protein.  <span class="journal">Mol. Gen. Genet. 187:148–156.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Conjugation%20deficient%20mutants%20of%20Escherichia%20coli%20distinguish%20classes%20of%20functions%20of%20the%20outer%20membrane%20ompA%20protein">google scholar</a></td><td><a href="pdf/493.pdf">pdf</a></td></tr>
<tr><td>494</td><td><a id="ref494" class="anchor"></a>Mao, Y., and Z. Sheng. <i>1986</i>. A phenylethyl alcohol resistant temperature sensitive <i>dnaB</i> mutant in <i>Escherichia coli.</i>  <span class="journal">Acta Genet. Sin. 13:81–88.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20phenylethyl%20alcohol%20resistant%20temperature%20sensitive%20dnaB%20mutant%20in%20Escherichia%20coli.">google scholar</a></td><td></td></tr>
<tr><td>495</td><td><a id="ref495" class="anchor"></a>Marakusha, B. I., V. G. Petrovskaya, and K. I. Volkovoi. <i>1986</i>. Obtaining <i>Salmonella</i> <i>typhimurium</i> and <i>Salmonella</i> <i>dublin</i> neamine resistant mutants suitable for the development of vaccines and the characterization of their biological and genetic properties.  <span class="journal">Zh. Mikrobiol. Epidemiol. Immunobiol. 1986:3–8.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Obtaining%20Salmonella%20typhimurium%20and%20Salmonella%20dublin%20neamine%20resistant%20mutants%20suitable%20for%20the%20development%20of%20vaccines%20and%20the%20characterization%20of%20their%20biological%20and%20genetic%20properties">google scholar</a></td><td><a href="pdf/495.pdf">pdf</a></td></tr>
<tr><td>496</td><td><a id="ref496" class="anchor"></a>Marcus, M., and Y. S. Halpern. <i>1967</i>. Genetic analysis of glutamate transport in <i>Escherichia</i> <i>coli.</i>  <span class="journal">J. Bacteriol. 93:1409–1415.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20analysis%20of%20glutamate%20transport%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/496.pdf">pdf</a></td></tr>
<tr><td>497</td><td><a id="ref497" class="anchor"></a>Marcus, M., and Y. S. Halpern. <i>1969</i>. Genetic analysis of the glutamate permease in <i>Escherichia</i> <i>coli</i> K-12.  <span class="journal">J. Bacteriol. 97:1118–1128.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20analysis%20of%20the%20glutamate%20permease%20in%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/497.pdf">pdf</a></td></tr>
<tr><td>498</td><td><a id="ref498" class="anchor"></a>Mark, D. F., J. W. Chang, and C. F. Richardson. <i>1977</i>. Genetic mapping of <i>trxA,</i> a gene affecting thioredoxin in <i>Escherichia coli</i> K12.  <span class="journal">Mol. Gen. Genet. 155:145–152.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20mapping%20of%20trxA,%20a%20gene%20affecting%20thioredoxin%20in%20Escherichia%20coli%20K12">google scholar</a></td><td><a href="pdf/498.pdf">pdf</a></td></tr>
<tr><td>499</td><td><a id="ref499" class="anchor"></a>Marquardt, J. L., D. A. Siegele, R. Kolter, and C. T. Walsh. <i>1992</i>. Cloning and sequencing of <i>Escherichia coli</i> <i>murZ</i> and purification of its product, a UDP-<i>N</i>-acetylglucosamine enolpyruvyl transferase.  <span class="journal">J. Bacteriol. 174:5748–5752.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Cloning%20and%20sequencing%20of%20Escherichia%20coli%20murZ%20and%20purification%20of%20its%20product,%20a%20UDP-N-acetylglucosamine%20enolpyruvyl%20transferase">google scholar</a></td><td><a href="pdf/499.pdf">pdf</a></td></tr>
<tr><td>500</td><td><a id="ref500" class="anchor"></a>Martinez, D., and F. Whitehouse, Jr. <i>1973</i>. Selective autocytotoxicity in a model system of <i>Escherichia coli</i> recombinants.  <span class="journal">J. Bacteriol. 114:882–884.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Selective%20autocytotoxicity%20in%20a%20model%20system%20of%20Escherichia%20coli%20recombinants">google scholar</a></td><td><a href="pdf/500.pdf">pdf</a></td></tr>
<tr><td>501</td><td><a id="ref501" class="anchor"></a>Masters, P. S., and J.-S. Hong. <i>1981</i>. Genetics of the glutamine transport system in <i>Escherichia</i> <i>coli.</i>  <span class="journal">J. Bacteriol. 147:805–819.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetics%20of%20the%20glutamine%20transport%20system%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/501.pdf">pdf</a></td></tr>
<tr><td>502</td><td><a id="ref502" class="anchor"></a>Matsuzawa, H., S. Asoh, T. Ohta, S. Tamaki, and M. Matsuhashi. <i>1980</i>. Further studies on <i>rodA</i> mutant: a round morphological mutant of <i>Escherichia coli</i> K12 with wild-type penicillin-binding protein 2.  <span class="journal">Agric. Biol. Chem. 44:2937–2941.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Further%20studies%20on%20rodA%20mutant:%20a%20round%20morphological%20mutant%20of%20Escherichia%20coli%20K12%20with%20wild-type%20penicillin-binding%20protein%202">google scholar</a></td><td><a href="pdf/502.pdf">pdf</a></td></tr>
<tr><td>503</td><td><a id="ref503" class="anchor"></a>Matsuzawa, H., S. Ushiyama, Y. Koyama, and T. Ohta. <i>1984</i>. <i>Escherichia coli</i> K-12 <i>tolZ</i> mutants tolerant to colicins E2, E3, D, Ia and Ib: defect in generation of the electrochemical proton gradient.  <span class="journal">J. Bacteriol. 160:733–739.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Escherichia%20coli%20K-12%20tolZ%20mutants%20tolerant%20to%20colicins%20E2,%20E3,%20D,%20Ia%20and%20Ib:%20defect%20in%20generation%20of%20the%20electrochemical%20proton%20gradient">google scholar</a></td><td><a href="pdf/503.pdf">pdf</a></td></tr>
<tr><td>504</td><td><a id="ref504" class="anchor"></a>Mattern, I. E., and J. Pittard. <i>1971</i>. Regulation of tyrosine biosynthesis in <i>Escherichia coli</i> K- 12: isolation and characterization of operator mutants.  <span class="journal">J. Bacteriol. 107:8–15.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation%20of%20tyrosine%20biosynthesis%20in%20Escherichia%20coli%20K-%2012:%20isolation%20and%20characterization%20of%20operator%20mutants">google scholar</a></td><td><a href="pdf/504.pdf">pdf</a></td></tr>
<tr><td>505</td><td><a id="ref505" class="anchor"></a>McCalla, D. R., C. Kaiser, and M. H. L. Green. <i>1978</i>. Genetics of nitrofurazone resistance in <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 133:10–16.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetics%20of%20nitrofurazone%20resistance%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/505.pdf">pdf</a></td></tr>
<tr><td>506</td><td><a id="ref506" class="anchor"></a>McConville, M. L., and H. P. Charles. <i>1979</i>. Mutants of <i>Escherichia coli</i> K12 accumulating porphobilinogen: a new locus, <i>hemC.</i>  <span class="journal">J. Gen. Microbiol. 111:193–200.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutants%20of%20Escherichia%20coli%20K12%20accumulating%20porphobilinogen:%20a%20new%20locus,%20hemC.">google scholar</a></td><td><a href="pdf/506.pdf">pdf</a></td></tr>
<tr><td>507</td><td><a id="ref507" class="anchor"></a>McConville, M. L., and H. P. Charles. <i>1979</i>. Isolation of haemin-requiring mutants of <i>Escherichia coli</i> K12.  <span class="journal">J. Gen. Microbiol. 113:155–164.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20of%20haemin-requiring%20mutants%20of%20Escherichia%20coli%20K12">google scholar</a></td><td><a href="pdf/507.pdf">pdf</a></td></tr>
<tr><td>508</td><td><a id="ref508" class="anchor"></a>McEwen, J., and P. Silverman. <i>1980</i>. Chromosomal mutations of <i>Escherichia coli</i> that alter expression of conjugative plasmid functions.  <span class="journal">Proc. Natl. Acad. Sci. USA 77:513–517.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Chromosomal%20mutations%20of%20Escherichia%20coli%20that%20alter%20expression%20of%20conjugative%20plasmid%20functions">google scholar</a></td><td><a href="pdf/508.pdf">pdf</a></td></tr>
<tr><td>509</td><td><a id="ref509" class="anchor"></a>Meiss, H. K., W. J. Brill, and B. Magasanik. <i>1969</i>. Genetic control of histidine degradation in <i>Salmonella typhimurium</i> LT-2.  <span class="journal">J. Biol. Chem. 244:5382–5391.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20control%20of%20histidine%20degradation%20in%20Salmonella%20typhimurium%20LT-2">google scholar</a></td><td><a href="pdf/509.pdf">pdf</a></td></tr>
<tr><td>510</td><td><a id="ref510" class="anchor"></a>Melton, T., W. Kundig, P. E. Hartman, and N. Meadow. <i>1976</i>. 3-Deoxy-3-fluoro-D-glucose- resistant <i>Salmonella typhimurium</i> mutants defective in the phosphoenolpyruvate:glycose phosphotransferase system.  <span class="journal">J. Bacteriol. 128:794–800.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=3-Deoxy-3-fluoro-D-glucose-%20resistant%20Salmonella%20typhimurium%20mutants%20defective%20in%20the%20phosphoenolpyruvate:glycose%20phosphotransferase%20system">google scholar</a></td><td><a href="pdf/510.pdf">pdf</a></td></tr>
<tr><td>511</td><td><a id="ref511" class="anchor"></a>Menzel, R., and J. Roth. <i>1981</i>. Regulation of the genes for proline utilization in <i>Salmonella</i> <i>typhimurium:</i> autogenous repression by the <i>putA</i> gene product.  <span class="journal">J. Mol. Biol. 148:21–44.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation%20of%20the%20genes%20for%20proline%20utilization%20in%20Salmonella%20typhimurium:%20autogenous%20repression%20by%20the%20putA%20gene%20product">google scholar</a></td><td><a href="pdf/511.pdf">pdf</a></td></tr>
<tr><td>512</td><td><a id="ref512" class="anchor"></a>Mett, V. L., F. I. Urmeeva, N. S. Kobets, T. V. Kolganova, K. A. Aliev, and E. S. Piruzyan. <i>1991</i>. Cloning and expression of a mutant EPSP-synthase gene of <i>E. coli</i> in transgenic plants.  <span class="journal">Biotekhnologiya 3:19–22.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Cloning%20and%20expression%20of%20a%20mutant%20EPSP-synthase%20gene%20of%20E.%20coli%20in%20transgenic%20plants">google scholar</a></td><td></td></tr>
<tr><td>513</td><td><a id="ref513" class="anchor"></a>Meynell, E. W. <i>1961</i>. A phage χ, which attacks motile bacteria.  <span class="journal">J. Gen. Microbiol. 25:253–290.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20phage%20χ,%20which%20attacks%20motile%20bacteria">google scholar</a></td><td><a href="pdf/513.pdf">pdf</a></td></tr>
<tr><td>514</td><td><a id="ref514" class="anchor"></a>Mikulka, T. W., B. I. Stieglitz, and J. M. Calvo. <i>1972</i>. Leucyltransfer-ribonucleic acid synthetase from a wild-type and temperature-sensitive mutant of <i>Salmonella typhimurium.</i>  <span class="journal">J. Bacteriol. 19:584–593.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Leucyltransfer-ribonucleic%20acid%20synthetase%20from%20a%20wild-type%20and%20temperature-sensitive%20mutant%20of%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/514.pdf">pdf</a></td></tr>
<tr><td>515</td><td><a id="ref515" class="anchor"></a>Mildener, B., T. P. Fondy, R. Engel, and B. E. Tropp. <i>1981</i>. Effects of halo analogs of glycerol 3-phosphate and dihydroxyacetone phosphate upon <i>Escherichia coli.</i>  <span class="journal">Antimicrob. Agents Chemother. 19:678–681.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Effects%20of%20halo%20analogs%20of%20glycerol%203-phosphate%20and%20dihydroxyacetone%20phosphate%20upon%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/515.pdf">pdf</a></td></tr>
<tr><td>516</td><td><a id="ref516" class="anchor"></a>Miller, C. G., C. Heiman, and C. Yen. <i>1976</i>. Mutants of <i>Salmonella typhimurium</i> deficient in an endopeptidase.  <span class="journal">J. Bacteriol. 127:490–497.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutants%20of%20Salmonella%20typhimurium%20deficient%20in%20an%20endopeptidase">google scholar</a></td><td><a href="pdf/516.pdf">pdf</a></td></tr>
<tr><td>517</td><td><a id="ref517" class="anchor"></a>Miller, C. G., and G. Schwartz. <i>1978</i>. Peptidase-deficient mutants of <i>Escherichia coli.</i> <i>J.</i>  <span class="journal">Bacteriol. 135:603–611.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Peptidase-deficient%20mutants%20of%20Escherichia%20coli.%20J.">google scholar</a></td><td><a href="pdf/517.pdf">pdf</a></td></tr>
<tr><td>518</td><td><a id="ref518" class="anchor"></a>Miller, E. S., and J. E. Brenchley. <i>1981</i>. L-methionine SR-sulfoximine resistant glutamine synthetase from mutants of <i>Salmonella typhimurium.</i>  <span class="journal">J. Biol. Chem. 256:11307–11312.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=L-methionine%20SR-sulfoximine%20resistant%20glutamine%20synthetase%20from%20mutants%20of%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/518.pdf">pdf</a></td></tr>
<tr><td>519</td><td><a id="ref519" class="anchor"></a>Miller, H. I., and D. I. Friedman. <i>1977</i>. Isolation of <i>Escherichia coli</i> mutants unable to support lambda integrative recombination. <span class="journal">DNA Insertion Elements, Plasmids, and Episomes. Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20of%20Escherichia%20coli%20mutants%20unable%20to%20support%20lambda%20integrative%20recombination">google scholar</a></td><td><a href="pdf/519.pdf">pdf</a></td></tr>
<tr><td>520</td><td><a id="ref520" class="anchor"></a>Miller, H. I., and D. I. Friedman. <i>1980</i>. An <i>E. coli</i> gene product required for λ site specific recombination.  <span class="journal">Cell 20:711–719.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=An%20E.%20coli%20gene%20product%20required%20for%20λ%20site%20specific%20recombination">google scholar</a></td><td><a href="pdf/520.pdf">pdf</a></td></tr>
<tr><td>521</td><td><a id="ref521" class="anchor"></a>Miller, H. I., A. Kikuchi, H. A. Nash, R. A. Weisberg, and D. I. Friedman. <i>1978</i>. Site-specific recombination of bacteriophage λ: the role of host gene products.  <span class="journal">Cold Spring Harbor Symp. Quant. Biol. 43:1121–1126.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Site-specific%20recombination%20of%20bacteriophage%20λ:%20the%20role%20of%20host%20gene%20products">google scholar</a></td><td><a href="pdf/521.pdf">pdf</a></td></tr>
<tr><td>522</td><td><a id="ref522" class="anchor"></a>Miller, H. I., M. Kirk, and H. Echols. <i>1981</i>. SOS induction and autoregulation of the <i>himA</i> gene for site-specific recombination in <i>Escherichia coli.</i>  <span class="journal">Proc. Natl. Acad. Sci. USA 78:6754–6758.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=SOS%20induction%20and%20autoregulation%20of%20the%20himA%20gene%20for%20site-specific%20recombination%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/522.pdf">pdf</a></td></tr>
<tr><td>523</td><td><a id="ref523" class="anchor"></a>Miller, J. H. <i>1972</i>.  <span class="journal">Experiments in Molecular Genetics. Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=">google scholar</a></td><td></td></tr>
<tr><td>524</td><td><a id="ref524" class="anchor"></a>Miller, J. H. <i>1992</i>.  <span class="journal">A Short Course in Bacterial Genetics. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=">google scholar</a></td><td></td></tr>
<tr><td>525</td><td><a id="ref525" class="anchor"></a>Miner, K. M., and L. Frank. <i>1974</i>. Sodium-stimulated glutamate transport in osmotically shocked cells and membrane vesicles of <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 117:1093–1098.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Sodium-stimulated%20glutamate%20transport%20in%20osmotically%20shocked%20cells%20and%20membrane%20vesicles%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/525.pdf">pdf</a></td></tr>
<tr><td>526</td><td><a id="ref526" class="anchor"></a>Miyoshi, Y., and H. Yamagata. <i>1976</i>. Sucrose-dependent spectinomycin-resistant mutants of  <span class="journal">Escherichia coli. J. Bacteriol. 125:142–148.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Sucrose-dependent%20spectinomycin-resistant%20mutants%20of">google scholar</a></td><td><a href="pdf/526.pdf">pdf</a></td></tr>
<tr><td>527</td><td><a id="ref527" class="anchor"></a>Mojica, A. T. <i>1975</i>. Transduction by phage PICM <i>clr-100</i> in <i>Salmonella typhimurium.</i> <i>Mol. Gen.</i>  <span class="journal">Genet. 138:113 126.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Transduction%20by%20phage%20PICM%20clr-100%20in%20Salmonella%20typhimurium.%20Mol.%20Gen.">google scholar</a></td><td><a href="pdf/527.pdf">pdf</a></td></tr>
<tr><td>528</td><td><a id="ref528" class="anchor"></a>Mojica, A. T., and E. Garcia. <i>1976</i>. Growth of coliphage BF-23 on rough strains of <i>Salmonella</i> <i>typhimurium.</i> the <i>bfe</i> locus.  <span class="journal">Mol. Gen. Genet. 147:195–202.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Growth%20of%20coliphage%20BF-23%20on%20rough%20strains%20of%20Salmonella%20typhimurium.%20the%20bfe%20locus">google scholar</a></td><td><a href="pdf/528.pdf">pdf</a></td></tr>
<tr><td>529</td><td><a id="ref529" class="anchor"></a>Molnar, J., I. B. Holland, and Y. Mandi. <i>1977</i>. Selection of <i>lon</i> mutants in <i>Escherichia coli</i> by treatment with phenothiazines.  <span class="journal">Genet. Res. 30:13–20.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Selection%20of%20lon%20mutants%20in%20Escherichia%20coli%20by%20treatment%20with%20phenothiazines">google scholar</a></td><td><a href="pdf/529.pdf">pdf</a></td></tr>
<tr><td>530</td><td><a id="ref530" class="anchor"></a>Monard, D., J. Janecek, and H. V. Rickenberg. <i>1969</i>. The enzymatic degradation of 3′,5′ cyclic AMP in strains of <i>E. coli</i> sensitive and resistant to catabolite repression.  <span class="journal">Biochem. Biophys. Res. Commun. 35:584–591.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The%20enzymatic%20degradation%20of%203′,5′%20cyclic%20AMP%20in%20strains%20of%20E.%20coli%20sensitive%20and%20resistant%20to%20catabolite%20repression">google scholar</a></td><td><a href="pdf/530.pdf">pdf</a></td></tr>
<tr><td>531</td><td><a id="ref531" class="anchor"></a>Morimoto, R. I., A. Tissières, and C. Georgopoulos . <i>1994</i>. The Biology of Heat Shock Proteins and Molecular Chaperones.  <span class="journal">Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The%20Biology%20of%20Heat%20Shock%20Proteins%20and%20Molecular%20Chaperones">google scholar</a></td><td></td></tr>
<tr><td>532</td><td><a id="ref532" class="anchor"></a>Morimyo, M., E. Hongo, H. Hama-Inaba, and I. Machida. <i>1992</i>. Cloning and characterization of the <i>mvrC</i> gene of <i>Escherichia coli</i> K12 which confers resistance against methyl viologen toxicity.  <span class="journal">Nucleic Acids Res. 20:3159–3165.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Cloning%20and%20characterization%20of%20the%20mvrC%20gene%20of%20Escherichia%20coli%20K12%20which%20confers%20resistance%20against%20methyl%20viologen%20toxicity">google scholar</a></td><td><a href="pdf/532.pdf">pdf</a></td></tr>
<tr><td>533</td><td><a id="ref533" class="anchor"></a>Morona, R., and U. Henning. <i>1984</i>. Host range mutants of bacteriophage OX-2 can use two different outer membrane proteins of <i>Escherichia coli</i> K-12 as receptors.  <span class="journal">J. Bacteriol. 159:579–582.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Host%20range%20mutants%20of%20bacteriophage%20OX-2%20can%20use%20two%20different%20outer%20membrane%20proteins%20of%20Escherichia%20coli%20K-12%20as%20receptors">google scholar</a></td><td><a href="pdf/533.pdf">pdf</a></td></tr>
<tr><td>534</td><td><a id="ref534" class="anchor"></a>Morona, R., and U. Henning. <i>1986</i>. New locus (<i>ttr</i>) in <i>Escherichia coli</i> K-12 affecting sensitivity to bacteriophage T2 and growth on oleate as the sole carbon source.  <span class="journal">J. Bacteriol. 168:534–540.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=New%20locus%20(ttr)%20in%20Escherichia%20coli%20K-12%20affecting%20sensitivity%20to%20bacteriophage%20T2%20and%20growth%20on%20oleate%20as%20the%20sole%20carbon%20source">google scholar</a></td><td><a href="pdf/534.pdf">pdf</a></td></tr>
<tr><td>535</td><td><a id="ref535" class="anchor"></a>Morona, R., M. Klose, and U. Henning. <i>1984</i>. <i>Escherichia coli</i> K-12 outer membrane protein OmpA as a bacteriophage receptor: analysis of mutant genes expressing altered proteins.  <span class="journal">J. Bacteriol. 159:570–578.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Escherichia%20coli%20K-12%20outer%20membrane%20protein%20OmpA%20as%20a%20bacteriophage%20receptor:%20analysis%20of%20mutant%20genes%20expressing%20altered%20proteins">google scholar</a></td><td><a href="pdf/535.pdf">pdf</a></td></tr>
<tr><td>536</td><td><a id="ref536" class="anchor"></a>Morona, R., and P. Reeves. <i>1982</i>. The <i>tolC</i> locus of <i>Escherichia coli</i> affects the expression of three major outer membrane proteins.  <span class="journal">J. Bacteriol. 150:1016–1023.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The%20tolC%20locus%20of%20Escherichia%20coli%20affects%20the%20expression%20of%20three%20major%20outer%20membrane%20proteins">google scholar</a></td><td><a href="pdf/536.pdf">pdf</a></td></tr>
<tr><td>537</td><td><a id="ref537" class="anchor"></a>Mosher, M. E., L. K. White, J. Hermolin, and R. H. Fillingame. <i>1985</i>. Proton-translocating ATPase of <i>Escherichia coli:</i> an <i>uncE</i> mutation impairing coupling between F-1 and F-0 but not F-0 mediated proton translocation.  <span class="journal">J. Biol. Chem. 260:4807–4814.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Proton-translocating%20ATPase%20of%20Escherichia%20coli:%20an%20uncE%20mutation%20impairing%20coupling%20between%20F-1%20and%20F-0%20but%20not%20F-0%20mediated%20proton%20translocation">google scholar</a></td><td></td></tr>
<tr><td>538</td><td><a id="ref538" class="anchor"></a>Mount, D. W. <i>1977</i>. A mutant of <i>Escherichia coli</i> showing constitutive expression of the lysogenic induction and error-prone DNA repair pathways.  <span class="journal">Proc. Natl. Acad. Sci. USA 74:300–304.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20mutant%20of%20Escherichia%20coli%20showing%20constitutive%20expression%20of%20the%20lysogenic%20induction%20and%20error-prone%20DNA%20repair%20pathways">google scholar</a></td><td><a href="pdf/538.pdf">pdf</a></td></tr>
<tr><td>539</td><td><a id="ref539" class="anchor"></a>Moyed, H. S. <i>1960</i>. False feedback inhibition: inhibition of tryptophan biosynthesis by 5- methyltryptophan.  <span class="journal">J. Biol. Chem. 235:1098–1102.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=False%20feedback%20inhibition:%20inhibition%20of%20tryptophan%20biosynthesis%20by%205-%20methyltryptophan">google scholar</a></td><td><a href="pdf/539.pdf">pdf</a></td></tr>
<tr><td>540</td><td><a id="ref540" class="anchor"></a>Moyed, H. S. <i>1961</i>. Interference with the feed-back control of histidine biosynthesis.  <span class="journal">J. Biol. Chem. 236:2261–2267.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Interference%20with%20the%20feed-back%20control%20of%20histidine%20biosynthesis">google scholar</a></td><td><a href="pdf/540.pdf">pdf</a></td></tr>
<tr><td>541</td><td><a id="ref541" class="anchor"></a>Moyed, H. S., and K. P. Bertrand. <i>1983</i>. <i>hipA,</i> a newly recognized gene of <i>Escherichia coli</i> K-12 that affects frequency of persistence after inhibition of murein synthesis.  <span class="journal">J. Bacteriol. 155:768–775.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=hipA,%20a%20newly%20recognized%20gene%20of%20Escherichia%20coli%20K-12%20that%20affects%20frequency%20of%20persistence%20after%20inhibition%20of%20murein%20synthesis">google scholar</a></td><td><a href="pdf/541.pdf">pdf</a></td></tr>
<tr><td>542</td><td><a id="ref542" class="anchor"></a>Muir, M. E., D. R. Hanwell, and B. J. Wallace. <i>1981</i>. Characterization of a respiratory mutant of <i>Escherichia coli</i> with reduced uptake of aminoglycoside antibiotics.  <span class="journal">Biochim. Biophys. Acta 638:234– 241.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Characterization%20of%20a%20respiratory%20mutant%20of%20Escherichia%20coli%20with%20reduced%20uptake%20of%20aminoglycoside%20antibiotics">google scholar</a></td><td><a href="pdf/542.pdf">pdf</a></td></tr>
<tr><td>543</td><td><a id="ref543" class="anchor"></a>Muller-Hill, B., L. Crapo, and W. Gilbert. <i>1968</i>. Mutants that make more lac repressor.  <span class="journal">Proc. Natl. Acad. Sci. USA 59:1259–1264.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutants%20that%20make%20more%20lac%20repressor">google scholar</a></td><td><a href="pdf/543.pdf">pdf</a></td></tr>
<tr><td>544</td><td><a id="ref544" class="anchor"></a>Munch-Petersen, A., and B. Mygind. <i>1976</i>. Nucleoside transport systems in <i>Escherichia coli</i> K12: specificity and regulation.  <span class="journal">J. Cell. Physiol. 89:551–560.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Nucleoside%20transport%20systems%20in%20Escherichia%20coli%20K12:%20specificity%20and%20regulation">google scholar</a></td><td><a href="pdf/544.pdf">pdf</a></td></tr>
<tr><td>545</td><td><a id="ref545" class="anchor"></a>Munch-Petersen, A., B. Mygind, A. Nicolaisen, and N. J. Pihl. <i>1979</i>. Nucleoside transport in cells and membrane vesicles from <i>Escherichia coli</i> K12.  <span class="journal">J. Biol. Chem. 254:3730–3737.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Nucleoside%20transport%20in%20cells%20and%20membrane%20vesicles%20from%20Escherichia%20coli%20K12">google scholar</a></td><td><a href="pdf/545.pdf">pdf</a></td></tr>
<tr><td>546</td><td><a id="ref546" class="anchor"></a>Munch-Petersen, A., P. Nygaard, K. Hammer-Jespersen, and N. Fiil. <i>1972</i>. Mutants constitutive for nucleoside-catabolizing enzymes in <i>Escherichia coli</i> K12.  <span class="journal">Eur. J. Biochem. 27:208– 215.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutants%20constitutive%20for%20nucleoside-catabolizing%20enzymes%20in%20Escherichia%20coli%20K12">google scholar</a></td><td><a href="pdf/546.pdf">pdf</a></td></tr>
<tr><td>547</td><td><a id="ref547" class="anchor"></a>Munekiyo, R., T. Tsuzuki, and M. Sekiguchi. <i>1979</i>. A new locus of <i>Escherichia coli</i> that determines sensitivity to bacteriophage φX174.  <span class="journal">J. Bacteriol. 138:1038–1040.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20new%20locus%20of%20Escherichia%20coli%20that%20determines%20sensitivity%20to%20bacteriophage%20φX174">google scholar</a></td><td><a href="pdf/547.pdf">pdf</a></td></tr>
<tr><td>548</td><td><a id="ref548" class="anchor"></a>Murata, K., K. Tani, J. Kato, and I. Chibata. <i>1980</i>. Excretion of glutathione by methylglyoxal- resistant <i>Escherichia coli.</i>  <span class="journal">J. Gen. Microbiol. 120:545–547.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Excretion%20of%20glutathione%20by%20methylglyoxal-%20resistant%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/548.pdf">pdf</a></td></tr>
<tr><td>549</td><td><a id="ref549" class="anchor"></a>Murooka, Y., and T. Harada. <i>1981</i>. Regulation of derepressed synthesis of arylsulfatase by tyramine oxidase in <i>Salmonella typhimurium.</i>  <span class="journal">J. Bacteriol. 145:796–802.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation%20of%20derepressed%20synthesis%20of%20arylsulfatase%20by%20tyramine%20oxidase%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/549.pdf">pdf</a></td></tr>
<tr><td>550</td><td><a id="ref550" class="anchor"></a>Myers, D. E., B. A. D. Stocker, and R. J. Roantree. <i>1980</i>. Mapping of genes determining penicillin-resistance and serum-sensitivity in <i>Salmonella enteritidis.</i>  <span class="journal">J. Gen. Microbiol. 118:367–376.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mapping%20of%20genes%20determining%20penicillin-resistance%20and%20serum-sensitivity%20in%20Salmonella%20enteritidis.">google scholar</a></td><td><a href="pdf/550.pdf">pdf</a></td></tr>
<tr><td>551</td><td><a id="ref551" class="anchor"></a>Myers, R. S., and S. R. Maloy. <i>1988</i>. Mutations of <i>putP</i> that alter the lithium sensitivity of <i>Salmonella typhimurium.</i>  <span class="journal">Mol. Microbiol. 2:749–755.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutations%20of%20putP%20that%20alter%20the%20lithium%20sensitivity%20of%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/551.pdf">pdf</a></td></tr>
<tr><td>552</td><td><a id="ref552" class="anchor"></a>Nagelkerke, F., and P. W. Postma. <i>1978</i>. 2-Deoxygalactose, a specific substrate of the <i>Salmonella typhimurium</i> galactose permease: its use for the isolation of <i>galP</i> mutants.  <span class="journal">J. Bacteriol. 133:607–613.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=2-Deoxygalactose,%20a%20specific%20substrate%20of%20the%20Salmonella%20typhimurium%20galactose%20permease:%20its%20use%20for%20the%20isolation%20of%20galP%20mutants">google scholar</a></td><td><a href="pdf/552.pdf">pdf</a></td></tr>
<tr><td>553</td><td><a id="ref553" class="anchor"></a>Nakamura, H. <i>1979</i>. Novel acriflavine resistance genes, <i>acrC</i> and <i>acrD,</i> in <i>Escherichia coli</i> K-12.  <span class="journal">J. Bacteriol. 139:8–12.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Novel%20acriflavine%20resistance%20genes,%20acrC%20and%20acrD,%20in%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/553.pdf">pdf</a></td></tr>
<tr><td>554</td><td><a id="ref554" class="anchor"></a>Nakamura, H. <i>1980</i>. Complementation analysis between genes determining cell division, <i>arcC,</i> and DNA polymerase III, <i>dnaE,</i> in <i>Escherichia coli</i> K12.  <span class="journal">Mem. Konan Univ. Sci. Ser. 25:31–33.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Complementation%20analysis%20between%20genes%20determining%20cell%20division,%20arcC,%20and%20DNA%20polymerase%20III,%20dnaE,%20in%20Escherichia%20coli%20K12">google scholar</a></td><td><a href="pdf/554.pdf">pdf</a></td></tr>
<tr><td>555</td><td><a id="ref555" class="anchor"></a>Nakamura, H., A. Hase, and K. Hirayoshi. <i>1987</i>. Temperature-sensitive phase in the cell cycle of ACR-C mutants of Escherichia coli K12.  <span class="journal">Mem. Konan Univ. Sci. Ser. 34:87–98.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Temperature-sensitive%20phase%20in%20the%20cell%20cycle%20of%20ACR-C%20mutants%20of%20Escherichia%20coli%20K12">google scholar</a></td><td><a href="pdf/555.pdf">pdf</a></td></tr>
<tr><td>556</td><td><a id="ref556" class="anchor"></a>Nakamura, Y., T. Kurihara, H. Saito, and H. Uchida. <i>1979</i>. σ-subunit of <i>Escherichia coli</i> RNA polymerase affects the function of λ N gene.  <span class="journal">Proc. Natl. Acad. Sci. USA 76:4593–4597.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=σ-subunit%20of%20Escherichia%20coli%20RNA%20polymerase%20affects%20the%20function%20of%20λ%20N%20gene">google scholar</a></td><td><a href="pdf/556.pdf">pdf</a></td></tr>
<tr><td>557</td><td><a id="ref557" class="anchor"></a>Nakayama, H., K. Nakayama, R. Nakayama, N. Irino, Y. Nakayama, and P. C. Hanawalt. <i>1984</i>. Isolation and genetic characterization of a thymineless death-resistant mutant of <i>Escherichia coli</i> K12: identification of a new mutation (<i>recQ1</i>) that blocks the RecF recombination pathway.  <span class="journal">Mol. Gen. Genet. 195:474–480.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20and%20genetic%20characterization%20of%20a%20thymineless%20death-resistant%20mutant%20of%20Escherichia%20coli%20K12:%20identification%20of%20a%20new%20mutation%20(recQ1)%20that%20blocks%20the%20RecF%20recombination%20pathway">google scholar</a></td><td><a href="pdf/557.pdf">pdf</a></td></tr>
<tr><td>558</td><td><a id="ref558" class="anchor"></a>Nakayama, K., S. Shiota, and H. Nakayama. <i>1988</i>. Thymineless death in <i>Escherichia coli</i> mutants deficient in the <i>recF</i> recombination pathway.  <span class="journal">Can. J. Microbiol. 34:905–907.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Thymineless%20death%20in%20Escherichia%20coli%20mutants%20deficient%20in%20the%20recF%20recombination%20pathway">google scholar</a></td><td><a href="pdf/558.pdf">pdf</a></td></tr>
<tr><td>559</td><td><a id="ref559" class="anchor"></a>Nashimoto, H., and H. Uchida. <i>1975</i>. Late steps in the assembly of 30s ribosomal proteins <i>in vivo</i> in a spectinomycin-resistant mutant of <i>Escherichia coli.</i>  <span class="journal">J. Mol. Biol. 96:443–453.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Late%20steps%20in%20the%20assembly%20of%2030s%20ribosomal%20proteins%20in%20vivo%20in%20a%20spectinomycin-resistant%20mutant%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/559.pdf">pdf</a></td></tr>
<tr><td>560</td><td><a id="ref560" class="anchor"></a>Nass, G., and J. Thomale. <i>1974</i>. Alteration of structure or level of threonyl-tRNA synthetase in borrelidin-resistant mutants of <i>Escherichia coli.</i>  <span class="journal">FEBS Lett. 39:182–186.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Alteration%20of%20structure%20or%20level%20of%20threonyl-tRNA%20synthetase%20in%20borrelidin-resistant%20mutants%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/560.pdf">pdf</a></td></tr>
<tr><td>561</td><td><a id="ref561" class="anchor"></a>Nazos, P. M., M. M. Mayo, T. Z. Su, J. J. Anderson, and D. L. Oxender. <i>1985</i>. Identification of <i>livG,</i> a membrane-associated component of the branched-chain amino acid transport in <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 163:1196–1202.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Identification%20of%20livG,%20a%20membrane-associated%20component%20of%20the%20branched-chain%20amino%20acid%20transport%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/561.pdf">pdf</a></td></tr>
<tr><td>562</td><td><a id="ref562" class="anchor"></a>Nelson, B. W., and R. J. Roantree. <i>1967</i>. Analysis of lipopolysaccharides extracted from penicillin-resistant, serum-sensitive <i>Salmonella</i> mutants.  <span class="journal">J. Gen. Microbiol. 48:179–188.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Analysis%20of%20lipopolysaccharides%20extracted%20from%20penicillin-resistant,%20serum-sensitive%20Salmonella%20mutants">google scholar</a></td><td><a href="pdf/562.pdf">pdf</a></td></tr>
<tr><td>563</td><td><a id="ref563" class="anchor"></a>Nelson, S. O., B. J. Scholte, and P. W. Postma. <i>1982</i>. Phosphoenolpyruvate:sugar phosphotransferase system-mediated regulation of carbohydrate metabolism in <i>Salmonella typhimurium.</i>  <span class="journal">J. Bacteriol. 150:604 615.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Phosphoenolpyruvate:sugar%20phosphotransferase%20system-mediated%20regulation%20of%20carbohydrate%20metabolism%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/563.pdf">pdf</a></td></tr>
<tr><td>564</td><td><a id="ref564" class="anchor"></a>Nelms, J., R. M. Edwards, J. Warwick, and I. Fotheringham. <i>1992</i>. Novel mutations in the <i>pheA</i> gene of <i>Escherichia coli</i> K-12 which result in highly feedback inhibition resistant variants of chorismate mutase/prephenate dehydratase.  <span class="journal">Appl. Environ. Microbiol. 58:2592–2598.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Novel%20mutations%20in%20the%20pheA%20gene%20of%20Escherichia%20coli%20K-12%20which%20result%20in%20highly%20feedback%20inhibition%20resistant%20variants%20of%20chorismate%20mutase/prephenate%20dehydratase">google scholar</a></td><td><a href="pdf/564.pdf">pdf</a></td></tr>
<tr><td>565</td><td><a id="ref565" class="anchor"></a>Neuhard, J., and K. Thomassen. <i>1976</i>. Altered deoxyribonucleotide pools in P2 eductants of <i>Escherichia coli</i> K-12 due to deletion of the <i>dcd</i> gene.  <span class="journal">J. Bacteriol. 126:999–1001.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Altered%20deoxyribonucleotide%20pools%20in%20P2%20eductants%20of%20Escherichia%20coli%20K-12%20due%20to%20deletion%20of%20the%20dcd%20gene">google scholar</a></td><td><a href="pdf/565.pdf">pdf</a></td></tr>
<tr><td>566</td><td><a id="ref566" class="anchor"></a>Newell, S. L., and W. J. Brill. <i>1972</i>. Mutants of <i>Salmonella typhimurium</i> that are insensitive to catabolite repression of proline degradation.  <span class="journal">J. Bacteriol. 111:375–382.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutants%20of%20Salmonella%20typhimurium%20that%20are%20insensitive%20to%20catabolite%20repression%20of%20proline%20degradation">google scholar</a></td><td><a href="pdf/566.pdf">pdf</a></td></tr>
<tr><td>567</td><td><a id="ref567" class="anchor"></a>Newman, E. B., N. Malik, and C. Walter. <i>1982</i>. L-Serine degradation in <i>Escherichia coli</i> K-12: directly isolated <i>ssd</i> mutants and their intragenic revertants.  <span class="journal">J. Bacteriol. 150:710–715.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=L-Serine%20degradation%20in%20Escherichia%20coli%20K-12:%20directly%20isolated%20ssd%20mutants%20and%20their%20intragenic%20revertants">google scholar</a></td><td><a href="pdf/567.pdf">pdf</a></td></tr>
<tr><td>568</td><td><a id="ref568" class="anchor"></a>Ng, H., and T. K. Gartner. <i>1963</i>. Selection of mutants of <i>Escherichia coli</i> constitutive for tryptophanase.  <span class="journal">J. Bacteriol. 85:245–246.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Selection%20of%20mutants%20of%20Escherichia%20coli%20constitutive%20for%20tryptophanase">google scholar</a></td><td><a href="pdf/568.pdf">pdf</a></td></tr>
<tr><td>569</td><td><a id="ref569" class="anchor"></a>Nikaido, H. <i>1961</i>. Galactose sensitive mutants of <i>Salmonella.</i> I. Metabolism of galactoside.  <span class="journal">Biochim. Biophys. Acta 48:460 469.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Galactose%20sensitive%20mutants%20of%20Salmonella.%20I.%20Metabolism%20of%20galactoside">google scholar</a></td><td><a href="pdf/569.pdf">pdf</a></td></tr>
<tr><td>570</td><td><a id="ref570" class="anchor"></a>Normark, S., T. Edlund, T. Grundstrom, S. Bergstrom, and H. Wolf-Watz. <i>1977</i>. <i>Escherichia</i> <i>coli</i> K-12 mutants hyperproducing chromosomal beta-lactamase by gene repetitions.  <span class="journal">J. Bacteriol. 132:912–922.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Escherichia%20coli%20K-12%20mutants%20hyperproducing%20chromosomal%20beta-lactamase%20by%20gene%20repetitions">google scholar</a></td><td><a href="pdf/570.pdf">pdf</a></td></tr>
<tr><td>571</td><td><a id="ref571" class="anchor"></a>Norwood, W. I., and J. R. Sadler. <i>1977</i>. Pseudoreversion of lactose operator-constitutive mutants.  <span class="journal">J. Bacteriol. 130:100–106.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Pseudoreversion%20of%20lactose%20operator-constitutive%20mutants">google scholar</a></td><td><a href="pdf/571.pdf">pdf</a></td></tr>
<tr><td>572</td><td><a id="ref572" class="anchor"></a>Novel, G., and M. Novel. <i>1973</i>. Mutants d’<i>Escherichia coli</i> K12 affectés pour leur croissance sur méthyl-β-D-glucuronide: localisation du gène de structure de la β-D-glucuronidase (<i>uidA</i>).  <span class="journal">Mol. Gen. Genet. 120:319–335.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutants%20d’Escherichia%20coli%20K12%20affectés%20pour%20leur%20croissance%20sur%20méthyl-β-D-glucuronide:%20localisation%20du%20gène%20de%20structure%20de%20la%20β-D-glucuronidase%20(uidA)">google scholar</a></td><td><a href="pdf/572.pdf">pdf</a></td></tr>
<tr><td>573</td><td><a id="ref573" class="anchor"></a>Novel, M., and G. Novel. <i>1976</i>. Regulation of β-glucuronidase synthesis in <i>Escherichia coli</i> K-12: constitutive mutants specifically derepressed for <i>uidA</i> expression.  <span class="journal">J. Bacteriol. 127:406–417.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation%20of%20β-glucuronidase%20synthesis%20in%20Escherichia%20coli%20K-12:%20constitutive%20mutants%20specifically%20derepressed%20for%20uidA%20expression">google scholar</a></td><td><a href="pdf/573.pdf">pdf</a></td></tr>
<tr><td>574</td><td><a id="ref574" class="anchor"></a>Novel, M., and G. Novel. <i>1976</i>. Regulation of β-glucuronidase synthesis in <i>Escherichia coli</i> K-12: pleiotropic constitutive mutations affecting <i>uxu</i> and <i>uidA</i> expression.  <span class="journal">J. Bacteriol. 127:418–432.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation%20of%20β-glucuronidase%20synthesis%20in%20Escherichia%20coli%20K-12:%20pleiotropic%20constitutive%20mutations%20affecting%20uxu%20and%20uidA%20expression">google scholar</a></td><td><a href="pdf/574.pdf">pdf</a></td></tr>
<tr><td>575</td><td><a id="ref575" class="anchor"></a>Nummer, B. A., S. F. Barefoot, and E. L. Kline. <i>1992</i>. Effects of the <i>flrA</i> regulatory locus on biosynthesis and excretion of amino acids in <i>Escherichia coli</i> B/r.  <span class="journal">Biochem. Biophys. Res. Commun. 183:343–349.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Effects%20of%20the%20flrA%20regulatory%20locus%20on%20biosynthesis%20and%20excretion%20of%20amino%20acids%20in%20Escherichia%20coli%20B/r">google scholar</a></td><td><a href="pdf/575.pdf">pdf</a></td></tr>
<tr><td>576</td><td><a id="ref576" class="anchor"></a>Nurminen, M., K. Lounatmaa, M. Sarvas, P. H. Makela, and T. Nakae. <i>1976</i>. Bacteriophage- resistant mutants of <i>Salmonella typhimurium</i> deficient in two major outer membrane proteins.  <span class="journal">J. Bacteriol. 127:941–955.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Bacteriophage-%20resistant%20mutants%20of%20Salmonella%20typhimurium%20deficient%20in%20two%20major%20outer%20membrane%20proteins">google scholar</a></td><td><a href="pdf/576.pdf">pdf</a></td></tr>
<tr><td>577</td><td><a id="ref577" class="anchor"></a>Obukowicz, M. G., N. R. Staten, and G. G. Krivi. <i>1992</i>. Enhanced heterologous gene expression in novel <i>rpoH</i> mutants of <i>Escherichia coli.</i>  <span class="journal">Appl. Environ. Microbiol. 58:1511–1523.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Enhanced%20heterologous%20gene%20expression%20in%20novel%20rpoH%20mutants%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/577.pdf">pdf</a></td></tr>
<tr><td>578</td><td><a id="ref578" class="anchor"></a>Odoevskaya, E. R., and S. P. Sineolsii. <i>1987</i>. Isolation and genetic study of bacterial mutations <i>gpr</i> blocking the replication of certain lambdoid phages.  <span class="journal">Sov. Genet. 23:432–440.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20and%20genetic%20study%20of%20bacterial%20mutations%20gpr%20blocking%20the%20replication%20of%20certain%20lambdoid%20phages">google scholar</a></td><td></td></tr>
<tr><td>579</td><td><a id="ref579" class="anchor"></a>O’Donovan, G. A., G. Edlin, J. A. Fuchs, J. Neuhard, and E. Thomassen. <i>1971</i>. Deoxycytidine triphosphate deaminase: characterization of an <i>Escherichia coli</i> mutant deficient in the enzyme.  <span class="journal">J. Bacteriol. 105:666–672.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Deoxycytidine%20triphosphate%20deaminase:%20characterization%20of%20an%20Escherichia%20coli%20mutant%20deficient%20in%20the%20enzyme">google scholar</a></td><td><a href="pdf/579.pdf">pdf</a></td></tr>
<tr><td>580</td><td><a id="ref580" class="anchor"></a>Ohnishi, K., and K. Kiritani. <i>1978</i>. Glycyl-L-leucine-resistance mutation affecting transport of branched-chain amino acids in <i>Salmonella typhimurium.</i>  <span class="journal">Jpn. J. Genet. 53:275–283.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Glycyl-L-leucine-resistance%20mutation%20affecting%20transport%20of%20branched-chain%20amino%20acids%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/580.pdf">pdf</a></td></tr>
<tr><td>581</td><td><a id="ref581" class="anchor"></a>Ohta, N., P. R. Galsworthy, and A. B. Pardee. <i>1971</i>. Genetics of sulfate transport by <i>Salmonella</i> <i>typhimurium.</i>  <span class="journal">J. Bacteriol. 105:1053–1062.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetics%20of%20sulfate%20transport%20by%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/581.pdf">pdf</a></td></tr>
<tr><td>582</td><td><a id="ref582" class="anchor"></a>Okada, Y., M. Wachi, K. Nagai, and M. Matsuhashi. <i>1992</i>. Change of the quantity of penicillin-binding proteins and other cytoplasmic and membrane proteins by mutations of the cell shape-determination genes <i>mreB</i> <i>mrec</i> and <i>mreD</i> of <i>Escherichia coli.</i>  <span class="journal">J. Gen. Appl. Microbiol. 38:157–163.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Change%20of%20the%20quantity%20of%20penicillin-binding%20proteins%20and%20other%20cytoplasmic%20and%20membrane%20proteins%20by%20mutations%20of%20the%20cell%20shape-determination%20genes%20mreB%20mrec%20and%20mreD%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/582.pdf">pdf</a></td></tr>
<tr><td>583</td><td><a id="ref583" class="anchor"></a>Okuyama, A., M. Yoshikawa, and N. Tanaka. <i>1974</i>. Alteration of ribosomal protein S2 in kasugamycin resistant mutant derived from <i>Escherichia coli</i> AB312.  <span class="journal">Biochem. Biophys. Res. Commun. 60:1163–1169.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Alteration%20of%20ribosomal%20protein%20S2%20in%20kasugamycin%20resistant%20mutant%20derived%20from%20Escherichia%20coli%20AB312">google scholar</a></td><td><a href="pdf/583.pdf">pdf</a></td></tr>
<tr><td>584</td><td><a id="ref584" class="anchor"></a>Oliver, D. B., R. J. Cabelli, K. M. Dolan, and G. P. Jarosik. <i>1990</i>. Azide resistant mutants of <i>Escherichia coli</i> alter the SecA protein, an azide-sensitive component of the protein export machinery.  <span class="journal">Proc. Natl. Acad. Sci. USA 87:8227–8231.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Azide%20resistant%20mutants%20of%20Escherichia%20coli%20alter%20the%20SecA%20protein,%20an%20azide-sensitive%20component%20of%20the%20protein%20export%20machinery">google scholar</a></td><td><a href="pdf/584.pdf">pdf</a></td></tr>
<tr><td>585</td><td><a id="ref585" class="anchor"></a>O’Neill, G. P., S. Thorbjanardottir, U. Michelsen, S. Palsson, D. Söll, and G. Eggertsson. <i>1991</i>. ∆-Aminolevulinic acid dehydratase deficiency can cause ∆-aminolevulinate auxotrophy in <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 173:94–100.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=∆-Aminolevulinic%20acid%20dehydratase%20deficiency%20can%20cause%20∆-aminolevulinate%20auxotrophy%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/585.pdf">pdf</a></td></tr>
<tr><td>586</td><td><a id="ref586" class="anchor"></a>O’Neill, J. B., and M. Freundlich. <i>1972</i>. Effects of cyclopentane glycine on metabolism in <i>Salmonella typhimurium.</i>  <span class="journal">J. Bacteriol. 111:510–515.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Effects%20of%20cyclopentane%20glycine%20on%20metabolism%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/586.pdf">pdf</a></td></tr>
<tr><td>587</td><td><a id="ref587" class="anchor"></a>Oppezzo, O. J., B. Avanzati, and D. N. Anton. <i>1991</i>. Increased susceptibility to beta lactam antibiotics and decreased porin content caused by <i>envB</i> mutations of  <span class="journal">Salmonella typhimurium. Antimicrob. Agents Chemother. 35:1203–1207.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Increased%20susceptibility%20to%20beta%20lactam%20antibiotics%20and%20decreased%20porin%20content%20caused%20by%20envB%20mutations%20of">google scholar</a></td><td><a href="pdf/587.pdf">pdf</a></td></tr>
<tr><td>588</td><td><a id="ref588" class="anchor"></a>Ornellas, E. P., and B. A. D. Stocker. <i>1974</i>. Relation of lipopolysaccharide character to Pl sensitivity in <i>Salmonella typhimurium.</i>  <span class="journal">Virology 60:491–502.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Relation%20of%20lipopolysaccharide%20character%20to%20Pl%20sensitivity%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/588.pdf">pdf</a></td></tr>
<tr><td>589</td><td><a id="ref589" class="anchor"></a>Orr, E., N. F. Fairweather, I. B. Holland, and R. H. Pritchard. <i>1979</i>. Isolation and characterization of a strain carrying a conditional lethal mutation in the <i>cou</i> gene of <i>Escherichia coli </i>K12.  <span class="journal">Mol. Gen. Genet. 177:103–112.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20and%20characterization%20of%20a%20strain%20carrying%20a%20conditional%20lethal%20mutation%20in%20the%20cou%20gene%20of%20Escherichia%20coli%20K12">google scholar</a></td><td><a href="pdf/589.pdf">pdf</a></td></tr>
<tr><td>590</td><td><a id="ref590" class="anchor"></a>Orr, J. C., D. W. Bryant, D. R. McCalla, and M. A. Quilliam. <i>1985</i>. Dinitropyrene resistant <i>Salmonella typhimurium</i> are deficient in an acetyl-CoA acetyltransferase.  <span class="journal">Chem. Biol. Interact. 54:281–288.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Dinitropyrene%20resistant%20Salmonella%20typhimurium%20are%20deficient%20in%20an%20acetyl-CoA%20acetyltransferase">google scholar</a></td><td><a href="pdf/590.pdf">pdf</a></td></tr>
<tr><td>591</td><td><a id="ref591" class="anchor"></a>Osborn, M. J., S. M. Rosen, L. Rothfield, and B. L. Horecker. <i>1962</i>. Biosynthesis of bacterial lipopolysaccharide. I. Enzymatic incorporation of galactose in a mutant strain of <i>Salmonella.</i>  <span class="journal">Proc. Natl. Acad. Sci. USA 48:1831–1838.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Biosynthesis%20of%20bacterial%20lipopolysaccharide.%20I.%20Enzymatic%20incorporation%20of%20galactose%20in%20a%20mutant%20strain%20of%20Salmonella.">google scholar</a></td><td><a href="pdf/591.pdf">pdf</a></td></tr>
<tr><td>592</td><td><a id="ref592" class="anchor"></a>Osorio, A. V., L. Servin-Gonzalez, M. Rocha, A. A. Covarrubias, and F. Bastarrachea. <i>1984</i>. <i>cis</i>-dominant, glutamine synthetase constitutive mutations of <i>Escherichia coli</i> independent of activation by the <i>glnG</i> and <i>glnF</i> products.  <span class="journal">Mol. Gen. Genet. 194:114–123.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=cis-dominant,%20glutamine%20synthetase%20constitutive%20mutations%20of%20Escherichia%20coli%20independent%20of%20activation%20by%20the%20glnG%20and%20glnF%20products">google scholar</a></td><td><a href="pdf/592.pdf">pdf</a></td></tr>
<tr><td>593</td><td><a id="ref593" class="anchor"></a>Ostrowski, J., and D. Hulanicka. <i>1981</i>. Effect of DNA gyrase inhibitors on gene expression of the cysteine region.  <span class="journal">Mol. Gen. Genet. 181:363–366.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Effect%20of%20DNA%20gyrase%20inhibitors%20on%20gene%20expression%20of%20the%20cysteine%20region">google scholar</a></td><td><a href="pdf/593.pdf">pdf</a></td></tr>
<tr><td>594</td><td><a id="ref594" class="anchor"></a>Oxender, D. L. <i>1972</i>. Membrane transport.  <span class="journal">Annu. Rev. Biochem. 41:777–814.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Membrane%20transport">google scholar</a></td><td><a href="pdf/594.pdf">pdf</a></td></tr>
<tr><td>595</td><td><a id="ref595" class="anchor"></a>Oxender, D. L., J. J. Anderson, M. M. Mayo, and S. C. Quay. <i>1977</i>. Leucine binding protein and regulation of transport in <i>Escherichia coli.</i>  <span class="journal">J. Supramol. Struct. 6:419–432.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Leucine%20binding%20protein%20and%20regulation%20of%20transport%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/595.pdf">pdf</a></td></tr>
<tr><td>596</td><td><a id="ref596" class="anchor"></a>Paetz, W., and G. Nass. <i>1973</i>. Biochemical and immunological characterization of threonyl- tRNA synthetase of two borrelidin resistant mutants of <i>Escherichia coli</i> K12.  <span class="journal">Eur. J. Biochem. 35:331–337.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Biochemical%20and%20immunological%20characterization%20of%20threonyl-%20tRNA%20synthetase%20of%20two%20borrelidin%20resistant%20mutants%20of%20Escherichia%20coli%20K12">google scholar</a></td><td><a href="pdf/596.pdf">pdf</a></td></tr>
<tr><td>597</td><td><a id="ref597" class="anchor"></a>Pai, C. H. <i>1974</i>. Biochemical and genetic characterization of dehydrobiotin resistant mutants of <i>Escherichia coli.</i>  <span class="journal">Mol. Gen. Genet. 134:345–357.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Biochemical%20and%20genetic%20characterization%20of%20dehydrobiotin%20resistant%20mutants%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/597.pdf">pdf</a></td></tr>
<tr><td>598</td><td><a id="ref598" class="anchor"></a>Pardee, A. B., E. J. Benz, Jr., D. A. St. Peter, J. N. Krieger, M. Meuth, and H. W. Trieshmann, Jr. <i>1971</i>. Hyperproduction and purification of nicotinamide deamidase, a microconstitutive enzyme of <i>Escherichia coli.</i>  <span class="journal">J. Biol. Chem. 246:6792–6796.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Hyperproduction%20and%20purification%20of%20nicotinamide%20deamidase,%20a%20microconstitutive%20enzyme%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/598.pdf">pdf</a></td></tr>
<tr><td>599</td><td><a id="ref599" class="anchor"></a>Pardo, D., and R. Rosset. <i>1977</i>. Properties of ribosomes from erythromycin resistant mutants of <i>Escherichia coli.</i>  <span class="journal">Mol. Gen. Genet. 156:267–271.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Properties%20of%20ribosomes%20from%20erythromycin%20resistant%20mutants%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/599.pdf">pdf</a></td></tr>
<tr><td>600</td><td><a id="ref600" class="anchor"></a>Park, M. H., B. B. Wong, and J. E. Lusk. <i>1976</i>. Mutants in three genes affecting transport of magnesium in <i>Escherichia coli:</i> genetics and physiology.  <span class="journal">J. Bacteriol. 126:1096–1103.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutants%20in%20three%20genes%20affecting%20transport%20of%20magnesium%20in%20Escherichia%20coli:%20genetics%20and%20physiology">google scholar</a></td><td><a href="pdf/600.pdf">pdf</a></td></tr>
<tr><td>601</td><td><a id="ref601" class="anchor"></a>Parker, C. T., A. W. Kloser, C. A. Schnaitman, M. A. Stein, S. Gottesman, and B. W. Gibson. <i>1992</i>. Role of the <i>rfaG</i> and <i>rfaP</i> genes in determining the lipopolysaccharide core structure and cell surface properties of <i>Escherichia coli</i> K-12.  <span class="journal">J. Bacteriol. 174:2525–2538.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Role%20of%20the%20rfaG%20and%20rfaP%20genes%20in%20determining%20the%20lipopolysaccharide%20core%20structure%20and%20cell%20surface%20properties%20of%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/601.pdf">pdf</a></td></tr>
<tr><td>602</td><td><a id="ref602" class="anchor"></a>Pauli, G., and P. Overath. <i>1972</i>. <i>ato</i> operon: a highly inducible system for acetoacetate and butyrate degradation in <i>Escherichia coli.</i>  <span class="journal">Eur. J. Biochem. 29:553–562.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=ato%20operon:%20a%20highly%20inducible%20system%20for%20acetoacetate%20and%20butyrate%20degradation%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/602.pdf">pdf</a></td></tr>
<tr><td>603</td><td><a id="ref603" class="anchor"></a>Payne, J. W., J. S. Morley, P. Armitage, and G. M. Payne. <i>1984</i>. Transport and hydrolysis of antibacterial peptide analogues in <i>Escherichia coli:</i> backbone-modified aminoxy peptides.  <span class="journal">J. Gen. Microbiol. 130:2253–2265.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Transport%20and%20hydrolysis%20of%20antibacterial%20peptide%20analogues%20in%20Escherichia%20coli:%20backbone-modified%20aminoxy%20peptides">google scholar</a></td><td><a href="pdf/603.pdf">pdf</a></td></tr>
<tr><td>604</td><td><a id="ref604" class="anchor"></a>Penninckx, M., and D. Gigot. <i>1979</i>. Synthesis of a peptide form of N-δ-(phosphonoacetyl)-L- ornithine. Its antibacterial effect through the specific inhibition of <i>Escherichia coli</i> L-ornithine carbamoyltransferase.  <span class="journal">J. Biol. Chem. 254:6392–6395.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Synthesis%20of%20a%20peptide%20form%20of%20N-δ-(phosphonoacetyl)-L-%20ornithine.%20Its%20antibacterial%20effect%20through%20the%20specific%20inhibition%20of%20Escherichia%20coli%20L-ornithine%20carbamoyltransferase">google scholar</a></td><td><a href="pdf/604.pdf">pdf</a></td></tr>
<tr><td>605</td><td><a id="ref605" class="anchor"></a>Pierard, A., and N. Glansdorf. <i>1972</i>. Mutations affecting uridine monophosphate pyrophosphorylase or the <i>argR</i> gene in <i>Escherichia coli.</i>  <span class="journal">Mol. Gen. Genet. 118:235–245.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutations%20affecting%20uridine%20monophosphate%20pyrophosphorylase%20or%20the%20argR%20gene%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/605.pdf">pdf</a></td></tr>
<tr><td>606</td><td><a id="ref606" class="anchor"></a>Pierard, A., N. Glansdorf, D. Gigot, M. Crabeel, P. Halleux, and L. Thiry. <i>1976</i>. Repression of <i>Escherichia coli</i> carbamoylphosphate synthase: relationships with enzyme synthesis in the arginine and pyrimidine pathways.  <span class="journal">J. Bacteriol. 127:291–301.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Repression%20of%20Escherichia%20coli%20carbamoylphosphate%20synthase:%20relationships%20with%20enzyme%20synthesis%20in%20the%20arginine%20and%20pyrimidine%20pathways">google scholar</a></td><td><a href="pdf/606.pdf">pdf</a></td></tr>
<tr><td>607</td><td><a id="ref607" class="anchor"></a>Pinn, P. A., K. J. Towner, and F. W. O’Grady. <i>1983</i>. Genetic analysis of chromosomal resistance to trimethoprim derived from clinical isolates of <i>Escherichia coli.</i>  <span class="journal">J. Gen. Microbiol. 128:85–92.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20analysis%20of%20chromosomal%20resistance%20to%20trimethoprim%20derived%20from%20clinical%20isolates%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/607.pdf">pdf</a></td></tr>
<tr><td>608</td><td><a id="ref608" class="anchor"></a>Plate, C. A., and J. L. Suit. <i>1981</i>. The <i>eup</i> genetic locus of <i>Escherichia coli</i> and its role in proton solute symport.  <span class="journal">J. Biol. Chem. 256:12974–12980.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The%20eup%20genetic%20locus%20of%20Escherichia%20coli%20and%20its%20role%20in%20proton%20solute%20symport">google scholar</a></td><td><a href="pdf/608.pdf">pdf</a></td></tr>
<tr><td>609</td><td><a id="ref609" class="anchor"></a>Platz, A., and B.-M. Sjoberg. <i>1980</i>. Construction and characterization of hybrid plasmids containing the <i>Escherichia coli nrd</i> region.  <span class="journal">J. Bacteriol. 143:561–568.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Construction%20and%20characterization%20of%20hybrid%20plasmids%20containing%20the%20Escherichia%20coli%20nrd%20region">google scholar</a></td><td><a href="pdf/609.pdf">pdf</a></td></tr>
<tr><td>610</td><td><a id="ref610" class="anchor"></a>Pledger, W. J., and H. E. Umbarger. <i>1973</i>. Isoleucine and valine metabolism in <i>Escherichia coli.</i> XXI. Mutations affecting derepression and valine resistance.  <span class="journal">J. Bacteriol. 114:183–194.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isoleucine%20and%20valine%20metabolism%20in%20Escherichia%20coli.%20XXI.%20Mutations%20affecting%20derepression%20and%20valine%20resistance">google scholar</a></td><td><a href="pdf/610.pdf">pdf</a></td></tr>
<tr><td>611</td><td><a id="ref611" class="anchor"></a>Popkin, P. S., and W. K. Maas. <i>1980</i>. <i>Escherichia coli</i> regulatory mutation affecting lysine transport and lysine decarboxylase.  <span class="journal">J. Bacteriol. 141:485–492.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Escherichia%20coli%20regulatory%20mutation%20affecting%20lysine%20transport%20and%20lysine%20decarboxylase">google scholar</a></td><td><a href="pdf/611.pdf">pdf</a></td></tr>
<tr><td>612</td><td><a id="ref612" class="anchor"></a>Portalier, R., J. Robert-Baudouy, and F. Stoeber. <i>1980</i>. Regulation of <i>Escherichia coli</i> K-12 hexuronate system genes: <i>exu</i> regulon.  <span class="journal">J. Bacteriol. 143:1095–1107.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation%20of%20Escherichia%20coli%20K-12%20hexuronate%20system%20genes:%20exu%20regulon">google scholar</a></td><td><a href="pdf/612.pdf">pdf</a></td></tr>
<tr><td>613</td><td><a id="ref613" class="anchor"></a>Postma, P. W. <i>1977</i>. Galactose transport in <i>Salmonella typhimurium.</i>  <span class="journal">J. Bacteriol. 129:630–639.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Galactose%20transport%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/613.pdf">pdf</a></td></tr>
<tr><td>614</td><td><a id="ref614" class="anchor"></a>Postma, P. W., and J. W. Lengeler. <i>1985</i>. Phosphoenolpyruvate:carbohydrate phosphotransferase system of bacteria.  <span class="journal">Microbiol. Rev. 49:232–269.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Phosphoenolpyruvate:carbohydrate%20phosphotransferase%20system%20of%20bacteria">google scholar</a></td><td><a href="pdf/614.pdf">pdf</a></td></tr>
<tr><td>615</td><td><a id="ref615" class="anchor"></a>Pouyssegur, J. M., and F. R. Stoeber. <i>1972</i>. Rameau degradatif commun des hexuronates chez <i>Escherichia coli</i> K12. Mécanisme d’induction dex enzymes assurant le métabolisme du 2-céto-3- desoxy-gluconate.  <span class="journal">Eur. J. Biochem. 30:479–494.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Rameau%20degradatif%20commun%20des%20hexuronates%20chez%20Escherichia%20coli%20K12.%20Mécanisme%20d’induction%20dex%20enzymes%20assurant%20le%20métabolisme%20du%202-céto-3-%20desoxy-gluconate">google scholar</a></td><td><a href="pdf/615.pdf">pdf</a></td></tr>
<tr><td>616</td><td><a id="ref616" class="anchor"></a>Prasad, I., B. Young, and S. Schaefler. <i>1973</i>. Genetic determination of the constitutive biosynthesis of phospho-β-glucosidase A in <i>Escherichia coli</i> K-12.  <span class="journal">J. Bacteriol. 114:909–915.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20determination%20of%20the%20constitutive%20biosynthesis%20of%20phospho-β-glucosidase%20A%20in%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/616.pdf">pdf</a></td></tr>
<tr><td>617</td><td><a id="ref617" class="anchor"></a>Pritchard, R. H., and S. I. Ahmad. <i>1971</i>. Fluorouracil and the isolation of mutants lacking uridine phosphorylase in <i>E. coli:</i> location of the gene.  <span class="journal">Mol. Gen. Genet. 111:84–88.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Fluorouracil%20and%20the%20isolation%20of%20mutants%20lacking%20uridine%20phosphorylase%20in%20E.%20coli:%20location%20of%20the%20gene">google scholar</a></td><td><a href="pdf/617.pdf">pdf</a></td></tr>
<tr><td>618</td><td><a id="ref618" class="anchor"></a>Prody, C. A., and J. B. Nellands. <i>1984</i>. Genetic and biochemical characterization of the <i>Escherichia coli</i> K-12 <i>fhuB</i> mutation.  <span class="journal">J. Bacteriol. 157:874–880.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20and%20biochemical%20characterization%20of%20the%20Escherichia%20coli%20K-12%20fhuB%20mutation">google scholar</a></td><td><a href="pdf/618.pdf">pdf</a></td></tr>
<tr><td>619</td><td><a id="ref619" class="anchor"></a>Pueyo, C., and J. Lopez-Barea. <i>1979</i>. The L-arabinose-resistance test with <i>Salmonella</i> <i>typhimurium</i> strain SV3 selects forward mutations at several <i>ara</i> genes.  <span class="journal">Mutat. Res. 64:249–258.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The%20L-arabinose-resistance%20test%20with%20Salmonella%20typhimurium%20strain%20SV3%20selects%20forward%20mutations%20at%20several%20ara%20genes">google scholar</a></td><td><a href="pdf/619.pdf">pdf</a></td></tr>
<tr><td>620</td><td><a id="ref620" class="anchor"></a>Pugsley, A. P. <i>1985</i>. <i>Escherichia coli</i> K12 strains for use in the identification and characterization of colicins.  <span class="journal">J. Gen. Microbiol. 131:369–376.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Escherichia%20coli%20K12%20strains%20for%20use%20in%20the%20identification%20and%20characterization%20of%20colicins">google scholar</a></td><td><a href="pdf/620.pdf">pdf</a></td></tr>
<tr><td>621</td><td><a id="ref621" class="anchor"></a>Pugsley, A. P., D. R. Lee, and C. A. Schnaitman. <i>1980</i>. Genes affecting the major outer membrane proteins of <i>Escherichia coli</i> K12: mutations at <i>nmpA</i> and <i>nmpB.</i>  <span class="journal">Mol. Gen. Genet. 677:681–690.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genes%20affecting%20the%20major%20outer%20membrane%20proteins%20of%20Escherichia%20coli%20K12:%20mutations%20at%20nmpA%20and%20nmpB.">google scholar</a></td><td><a href="pdf/621.pdf">pdf</a></td></tr>
<tr><td>622</td><td><a id="ref622" class="anchor"></a>Pugsley, A. P., F. Moreno and V. de Lorenzo. <i>1986</i>. Microcin-E492-insensitive mutants of <i>Escherichia coli.</i>  <span class="journal">J. Gen. Microbiol. 132:3253–3260.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Microcin-E492-insensitive%20mutants%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/622.pdf">pdf</a></td></tr>
<tr><td>623</td><td><a id="ref623" class="anchor"></a>Pugsley, A. P., and P. Reeves. <i>1976</i>. Iron uptake in colicin B-resistant mutants of <i>Escherichia coli</i> K-12.  <span class="journal">J. Bacteriol. 126:1052–1062.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Iron%20uptake%20in%20colicin%20B-resistant%20mutants%20of%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/623.pdf">pdf</a></td></tr>
<tr><td>624</td><td><a id="ref624" class="anchor"></a>Pugsley, A. P., and P. Reeves. <i>1976</i>. Characterization of group B colicin-resistant mutants of <i>Escherichia coli</i> K-12: colicin resistance and the role of enterochelin.  <span class="journal">J. Bacteriol. 127:218–228.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Characterization%20of%20group%20B%20colicin-resistant%20mutants%20of%20Escherichia%20coli%20K-12:%20colicin%20resistance%20and%20the%20role%20of%20enterochelin">google scholar</a></td><td><a href="pdf/624.pdf">pdf</a></td></tr>
<tr><td>625</td><td><a id="ref625" class="anchor"></a>Pugsley, A. P., and C. A. Schnaitman. <i>1978</i>. Outer membrane proteins of <i>Escherichia coli.</i> VII. Evidence that bacteriophage-directed protein 2 functions as a pore.  <span class="journal">J. Bacteriol. 133:1181–1189.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Outer%20membrane%20proteins%20of%20Escherichia%20coli.%20VII.%20Evidence%20that%20bacteriophage-directed%20protein%202%20functions%20as%20a%20pore">google scholar</a></td><td><a href="pdf/625.pdf">pdf</a></td></tr>
<tr><td>626</td><td><a id="ref626" class="anchor"></a>Pugsley, A. P., and C. A. Schnaitman. <i>1978</i>. Identification of three genes controlling production of new outer membrane pore proteins in <i>Escherichia coli</i> K-12.  <span class="journal">J. Bacteriol. 135:1118–1129.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Identification%20of%20three%20genes%20controlling%20production%20of%20new%20outer%20membrane%20pore%20proteins%20in%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/626.pdf">pdf</a></td></tr>
<tr><td>627</td><td><a id="ref627" class="anchor"></a>Puyo, M. F., P. Calsou, and B. Salles. <i>1992</i>. UV resistance of <i>E. coli</i> K12 deficient in cAMP/CRP regulation.  <span class="journal">Mutat. Res. 282:247–252.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=UV%20resistance%20of%20E.%20coli%20K12%20deficient%20in%20cAMP/CRP%20regulation">google scholar</a></td><td><a href="pdf/627.pdf">pdf</a></td></tr>
<tr><td>628</td><td><a id="ref628" class="anchor"></a>Rahav-Manor, O., O. Carmel, R. Karpel, D. Taglicht, G. Glaser, S. Schuldiner, and E. Padan. <i>1992</i>. NhaR, a protein homologous to a family of bacterial regulatory proteins (LysR), regulates <i>nhaA,</i> the sodium proton antiporter gene in <i>Escherichia coli.</i>  <span class="journal">J. Biol. Chem. 267:10433– 10438.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=NhaR,%20a%20protein%20homologous%20to%20a%20family%20of%20bacterial%20regulatory%20proteins%20(LysR),%20regulates%20nhaA,%20the%20sodium%20proton%20antiporter%20gene%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/628.pdf">pdf</a></td></tr>
<tr><td>629</td><td><a id="ref629" class="anchor"></a>Rainwater, S., and P. M. Silverman. <i>1990</i>. The Cpx proteins of <i>Escherichia coli</i> K-12: evidence that <i>cpxA,</i> <i>ecfB,</i> <i>ssd,</i> and <i>eup</i> mutations all identify the same gene.  <span class="journal">J. Bacteriol. 172:2456–2461.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The%20Cpx%20proteins%20of%20Escherichia%20coli%20K-12:%20evidence%20that%20cpxA,%20ecfB,%20ssd,%20and%20eup%20mutations%20all%20identify%20the%20same%20gene">google scholar</a></td><td><a href="pdf/629.pdf">pdf</a></td></tr>
<tr><td>630</td><td><a id="ref630" class="anchor"></a>Rakonjac, J., M. Milic, D. Ajdic-Predic, D. Santos, R. Ivanisevic, and D. J. Savic. <i>1992</i>. A new genetic locus that affects the response of <i>Escherichia coli</i> K12 to novobiocin.  <span class="journal">Mol. Microbiol. 6:1547–1553.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20new%20genetic%20locus%20that%20affects%20the%20response%20of%20Escherichia%20coli%20K12%20to%20novobiocin">google scholar</a></td><td><a href="pdf/630.pdf">pdf</a></td></tr>
<tr><td>631</td><td><a id="ref631" class="anchor"></a>Rakonjac, J., M. Milic, and D. J. Savic. <i>1991</i>. <i>cysB</i> and <i>cysE</i> mutants of <i>Escherichia coli</i> K12 show increased resistance to novobiocin.  <span class="journal">Mol. Gen. Genet. 228:307–311.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=cysB%20and%20cysE%20mutants%20of%20Escherichia%20coli%20K12%20show%20increased%20resistance%20to%20novobiocin">google scholar</a></td><td><a href="pdf/631.pdf">pdf</a></td></tr>
<tr><td>632</td><td><a id="ref632" class="anchor"></a>Ramabhadran, T. V. <i>1976</i>. Method for the isolation of <i>Escherichia coli</i> relaxed mutants, utilizing near-ultraviolet irradiation.  <span class="journal">J. Bacteriol. 127:1587–1589.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Method%20for%20the%20isolation%20of%20Escherichia%20coli%20relaxed%20mutants,%20utilizing%20near-ultraviolet%20irradiation">google scholar</a></td><td><a href="pdf/632.pdf">pdf</a></td></tr>
<tr><td>633</td><td><a id="ref633" class="anchor"></a>Ramakrishnan, T., and E. A. Adelberg. <i>1965</i>. Regulatory mechanisms in the biosynthesis of isoleucine and valine. II. Identification of two operator genes.  <span class="journal">J. Bacteriol. 89:654–660.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulatory%20mechanisms%20in%20the%20biosynthesis%20of%20isoleucine%20and%20valine.%20II.%20Identification%20of%20two%20operator%20genes">google scholar</a></td><td><a href="pdf/633.pdf">pdf</a></td></tr>
<tr><td>634</td><td><a id="ref634" class="anchor"></a>Ramakrishnan, V., and S. W. White. <i>1992</i>. The structure of ribosomal protein S5 reveals sites of interaction with 16s RRNA.  <span class="journal">Nature  358:768–771.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The%20structure%20of%20ribosomal%20protein%20S5%20reveals%20sites%20of%20interaction%20with%2016s%20RRNA">google scholar</a></td><td><a href="pdf/634.pdf">pdf</a></td></tr>
<tr><td>635</td><td><a id="ref635" class="anchor"></a>Rancourt, D. E., J. T. Stephenson, G. A. Vickell, and J. M. Wood. <i>1984</i>. Proline excretion by <i>Escherichia coli</i> K12.  <span class="journal">Biotechnol. Bioeng. 26:74–80.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Proline%20excretion%20by%20Escherichia%20coli%20K12">google scholar</a></td><td><a href="pdf/635.pdf">pdf</a></td></tr>
<tr><td>636</td><td><a id="ref636" class="anchor"></a>Raney, M. E., and R. W. Elliott. <i>1978</i>. The biochemical and genetic basis for high frequency thiomethylgalactoside resistance in lambda-lambda-dg lysogens of <i>Escherichia coli.</i>  <span class="journal">J. Gen. Microbiol. 104:287–298.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The%20biochemical%20and%20genetic%20basis%20for%20high%20frequency%20thiomethylgalactoside%20resistance%20in%20lambda-lambda-dg%20lysogens%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/636.pdf">pdf</a></td></tr>
<tr><td>637</td><td><a id="ref637" class="anchor"></a>Ray, J. M., C. Yanofsky, and R. Bauerle. <i>1988</i>. Mutational analysis of the catalytic and feedback sites of the tryptophan-sensitive 3-deoxy-D-<i>arabino</i>-heptulosonate-7-phosphate synthase of <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 170:5500–5506.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutational%20analysis%20of%20the%20catalytic%20and%20feedback%20sites%20of%20the%20tryptophan-sensitive%203-deoxy-D-arabino-heptulosonate-7-phosphate%20synthase%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/637.pdf">pdf</a></td></tr>
<tr><td>638</td><td><a id="ref638" class="anchor"></a>Reeve, E. C. R. <i>1968</i>. Genetic analysis of some mutations causing resistance to tetracycline in <i>Escherichia coli</i> K12.  <span class="journal">Genet. Res. 11:303–309.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20analysis%20of%20some%20mutations%20causing%20resistance%20to%20tetracycline%20in%20Escherichia%20coli%20K12">google scholar</a></td><td><a href="pdf/638.pdf">pdf</a></td></tr>
<tr><td>639</td><td><a id="ref639" class="anchor"></a>Reiner, A. M. <i>1977</i>. Xylitol and D-arabitol toxicities due to derepressed fructose, galactitol, and sorbitol phosphotransferases of <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 132:166–173.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Xylitol%20and%20D-arabitol%20toxicities%20due%20to%20derepressed%20fructose,%20galactitol,%20and%20sorbitol%20phosphotransferases%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/639.pdf">pdf</a></td></tr>
<tr><td>640</td><td><a id="ref640" class="anchor"></a>Reynolds, A. E., J. Felton, and A. Wright. <i>1981</i>. Insertion of DNA activates the cryptic <i>bgl</i> operon in <i>E. coli</i> K-12.  <span class="journal">Nature  293:625–629.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Insertion%20of%20DNA%20activates%20the%20cryptic%20bgl%20operon%20in%20E.%20coli%20K-12">google scholar</a></td><td><a href="pdf/640.pdf">pdf</a></td></tr>
<tr><td>641</td><td><a id="ref641" class="anchor"></a>Rick, P. D., and M. J. Osborn. <i>1972</i>. Isolation of a mutant of <i>Salmonella typhimurium</i> dependent on D-arabinose-5-P for growth and synthesis of 3-deoxy-D-mannooctulosonate (ketodeoxyoctonate).  <span class="journal">Proc. Natl. Acad. Sci. USA 69:3756–3760.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20of%20a%20mutant%20of%20Salmonella%20typhimurium%20dependent%20on%20D-arabinose-5-P%20for%20growth%20and%20synthesis%20of%203-deoxy-D-mannooctulosonate%20(ketodeoxyoctonate)">google scholar</a></td><td><a href="pdf/641.pdf">pdf</a></td></tr>
<tr><td>642</td><td><a id="ref642" class="anchor"></a>Ried, G., I. Hindennach, and U. Henning. <i>1990</i>. Role of lipopolysaccharide in assembly of <i>Escherichia coli</i> outer membrane proteins OmpA, OmpC, and OmpF.  <span class="journal">J. Bacteriol. 172:6048–6053.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Role%20of%20lipopolysaccharide%20in%20assembly%20of%20Escherichia%20coli%20outer%20membrane%20proteins%20OmpA,%20OmpC,%20and%20OmpF">google scholar</a></td><td><a href="pdf/642.pdf">pdf</a></td></tr>
<tr><td>643</td><td><a id="ref643" class="anchor"></a>Riley, M. <i>1993</i>. Functions of the gene products of <i>Escherichia coli.</i>  <span class="journal">Microbiol. Rev. 57:862–952.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Functions%20of%20the%20gene%20products%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/643.pdf">pdf</a></td></tr>
<tr><td>644</td><td><a id="ref644" class="anchor"></a>Roa, M. <i>1979</i>. Interaction of bacteriophage K10 with its receptor, the <i>lamB</i> protein of <i>Escherichia</i> <i>coli.</i>  <span class="journal">J. Bacteriol. 140:680–686.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Interaction%20of%20bacteriophage%20K10%20with%20its%20receptor,%20the%20lamB%20protein%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/644.pdf">pdf</a></td></tr>
<tr><td>645</td><td><a id="ref645" class="anchor"></a>Robbins, A. R. <i>1975</i>. Regulation of the <i>Escherichia coli</i> methylgalactoside transport system by gene <i>mglD.</i>  <span class="journal">J. Bacteriol. 123:69–74.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation%20of%20the%20Escherichia%20coli%20methylgalactoside%20transport%20system%20by%20gene%20mglD.">google scholar</a></td><td><a href="pdf/645.pdf">pdf</a></td></tr>
<tr><td>646</td><td><a id="ref646" class="anchor"></a>Robbins, J. C., and D. L Oxender. <i>1973</i>. Transport systems for alanine, serine, and glycine in <i>Escherichia coli</i> K-12.  <span class="journal">J. Bacteriol. 116:12–18.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Transport%20systems%20for%20alanine,%20serine,%20and%20glycine%20in%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/646.pdf">pdf</a></td></tr>
<tr><td>647</td><td><a id="ref647" class="anchor"></a>Roberton, A. M., P. A. Sullivan, M. C. Jones-Mortimer, and H. L. Kornberg. <i>1980</i>. Two genes affecting glucarate utilization in <i>Escherichia coli</i> K12.  <span class="journal">J. Gen. Microbiol. 117:377–382.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Two%20genes%20affecting%20glucarate%20utilization%20in%20Escherichia%20coli%20K12">google scholar</a></td><td><a href="pdf/647.pdf">pdf</a></td></tr>
<tr><td>648</td><td><a id="ref648" class="anchor"></a>Roberts, L. M., and E. C. R. Reeve. <i>1970</i>. Two mutations giving low-level streptomycin resistance in <i>Escherichia coli</i> K12.  <span class="journal">Genet. Res. 16:359–365.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Two%20mutations%20giving%20low-level%20streptomycin%20resistance%20in%20Escherichia%20coli%20K12">google scholar</a></td><td><a href="pdf/648.pdf">pdf</a></td></tr>
<tr><td>649</td><td><a id="ref649" class="anchor"></a>Robinson, C. L., and J. H. Jackson. <i>1982</i>. New acetohydroxy acid synthetase activity from mutational activation of a cryptic gene in <i>Escherichia coli</i> K-12.  <span class="journal">Mol. Gen. Genet. 186:240–246.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=New%20acetohydroxy%20acid%20synthetase%20activity%20from%20mutational%20activation%20of%20a%20cryptic%20gene%20in%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/649.pdf">pdf</a></td></tr>
<tr><td>650</td><td><a id="ref650" class="anchor"></a>Rodriguez, S. B., and J. L. Ingraham. <i>1983</i>. Location on the <i>Salmonella typhimurium</i> chromosome of the gene encoding nucleoside diphosphokinase <i>(ndk).</i>  <span class="journal">J. Bacteriol. 153:1101–1103.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Location%20on%20the%20Salmonella%20typhimurium%20chromosome%20of%20the%20gene%20encoding%20nucleoside%20diphosphokinase%20(ndk).">google scholar</a></td><td><a href="pdf/650.pdf">pdf</a></td></tr>
<tr><td>651</td><td><a id="ref651" class="anchor"></a>Roeder, N., and R. L Somerville. <i>1979</i>. Cloning the <i>trpR</i> gene.  <span class="journal">Mol. Gen. Genet. 176:361–368.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Cloning%20the%20trpR%20gene">google scholar</a></td><td><a href="pdf/651.pdf">pdf</a></td></tr>
<tr><td>652</td><td><a id="ref652" class="anchor"></a>Roehl, R. A., and R. T. Vinopal. <i>1980</i>. Genetic locus, distant from <i>ptsM,</i> affecting enzyme IIA/IIB function in <i>Escherichia</i> coli K-12.  <span class="journal">J. Bacteriol. 142:120–130.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20locus,%20distant%20from%20ptsM,%20affecting%20enzyme%20IIA/IIB%20function%20in%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/652.pdf">pdf</a></td></tr>
<tr><td>653</td><td><a id="ref653" class="anchor"></a>Rolfe, B., and K. Onodera. <i>1971</i>. Demonstration of missing membrane proteins in a colicin- tolerant mutant of <i>Escherichia.</i>  <span class="journal">Biochem. Biophys. Res. Commun. 44:767–773.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Demonstration%20of%20missing%20membrane%20proteins%20in%20a%20colicin-%20tolerant%20mutant%20of%20Escherichia.">google scholar</a></td><td><a href="pdf/653.pdf">pdf</a></td></tr>
<tr><td>654</td><td><a id="ref654" class="anchor"></a>Rood, J. I., A. J. Laird, and J. W. Williams. <i>1980</i>. Cloning of the <i>Escherichia coli</i> K12 dihydrofolate reductase EC-1.5.1.3 gene following phage Mu mediated transposition.  <span class="journal">Gene 8:255– 266.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Cloning%20of%20the%20Escherichia%20coli%20K12%20dihydrofolate%20reductase%20EC-1.5.1.3%20gene%20following%20phage%20Mu%20mediated%20transposition">google scholar</a></td><td><a href="pdf/654.pdf">pdf</a></td></tr>
<tr><td>655</td><td><a id="ref655" class="anchor"></a>Rosen, B. P. <i>1973</i>. Basic amino acid transport in <i>Escherichia coli:</i> properties of canavanine- resistant mutants.  <span class="journal">J. Bacteriol. 116:627–635.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Basic%20amino%20acid%20transport%20in%20Escherichia%20coli:%20properties%20of%20canavanine-%20resistant%20mutants">google scholar</a></td><td><a href="pdf/655.pdf">pdf</a></td></tr>
<tr><td>656</td><td><a id="ref656" class="anchor"></a>Rosen, S. M., L. D. Zeleznick, D. Fraenkel, I. M. Wiener, M. J. Osborn, and B. L. Horecker. <i>1965</i>. Characterization of the cell wall lipopolysaccharide of a mutant of <i>Salmonella typhimurium </i>lacking phosphomannose isomerase.  <span class="journal">Biochem. Z. 342:375–386.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Characterization%20of%20the%20cell%20wall%20lipopolysaccharide%20of%20a%20mutant%20of%20Salmonella%20typhimurium%20lacking%20phosphomannose%20isomerase">google scholar</a></td><td><a href="pdf/656.pdf">pdf</a></td></tr>
<tr><td>657</td><td><a id="ref657" class="anchor"></a>Roth, J. R., and B. N. Ames. <i>1966</i>. Histidine regulatory mutants in <i>Salmonella typhimurium.</i> II. Histidine regulatory mutants having altered histidyl-tRNA synthetase.  <span class="journal">J. Mol. Biol. 22:325–334.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Histidine%20regulatory%20mutants%20in%20Salmonella%20typhimurium.%20II.%20Histidine%20regulatory%20mutants%20having%20altered%20histidyl-tRNA%20synthetase">google scholar</a></td><td><a href="pdf/657.pdf">pdf</a></td></tr>
<tr><td>658</td><td><a id="ref658" class="anchor"></a>Roth, J. R., D. N. Anton, and P. E. Hartman. <i>1966</i>. Histidine regulatory mutants in <i>Salmonella</i> <i>typhimurium.</i> I. Isolation and general properties.  <span class="journal">J. Mol. Biol. 22:305–323.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Histidine%20regulatory%20mutants%20in%20Salmonella%20typhimurium.%20I.%20Isolation%20and%20general%20properties">google scholar</a></td><td><a href="pdf/658.pdf">pdf</a></td></tr>
<tr><td>659</td><td><a id="ref659" class="anchor"></a>Rowbury, R. J., M. Goodson, and A. D. Wallace. <i>1992</i>. The <i>phoE</i> porin and transmission of the chemical stimulus for induction of acid resistance acid habituation in <i>Escherichia coli.</i>  <span class="journal">J. Appl. Bacteriol. 72:233–243.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The%20phoE%20porin%20and%20transmission%20of%20the%20chemical%20stimulus%20for%20induction%20of%20acid%20resistance%20acid%20habituation%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/659.pdf">pdf</a></td></tr>
<tr><td>660</td><td><a id="ref660" class="anchor"></a>Rudd, K. E., and R. Menzel. <i>1987</i>. <i>his</i> operons of <i>Escherichia coli</i> and <i>Salmonella typhimurium</i> are regulated by DNA supercoiling.  <span class="journal">Proc. Natl. Acad. Sci. USA 84:517–521.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=his%20operons%20of%20Escherichia%20coli%20and%20Salmonella%20typhimurium%20are%20regulated%20by%20DNA%20supercoiling">google scholar</a></td><td><a href="pdf/660.pdf">pdf</a></td></tr>
<tr><td>661</td><td><a id="ref661" class="anchor"></a>Rudd, K. E., W. Miller, C. Werner, J. Ostell, C. Tolstoshev, and S. G. Satterfield. <i>1991</i>. Mapping sequenced <i>E. coli</i> genes by computer: software, strategies and examples.  <span class="journal">Nucleic Acids Res. 19:637–647.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mapping%20sequenced%20E.%20coli%20genes%20by%20computer:%20software,%20strategies%20and%20examples">google scholar</a></td><td><a href="pdf/661.pdf">pdf</a></td></tr>
<tr><td>662</td><td><a id="ref662" class="anchor"></a>Russell, R. R. B., and A. J. Pittard. <i>1971</i>. Mutants of <i>Escherichia</i> coli unable to make protein at 42°C.  <span class="journal">J. Bacteriol. 108:790–798.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutants%20of%20Escherichia%20coli%20unable%20to%20make%20protein%20at%2042°C">google scholar</a></td><td><a href="pdf/662.pdf">pdf</a></td></tr>
<tr><td>663</td><td><a id="ref663" class="anchor"></a>Ryals, J., R.-Y. Hsu, M. N. Lipsett, and H. Bremer. <i>1982</i>. Isolation of single-site <i>Escherichia</i> <i>coli</i> mutants deficient in thiamine and 4-thiouridine syntheses: identification of a <i>nuvC</i> mutant.  <span class="journal">J. Bacteriol. 151:899–904.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20of%20single-site%20Escherichia%20coli%20mutants%20deficient%20in%20thiamine%20and%204-thiouridine%20syntheses:%20identification%20of%20a%20nuvC%20mutant">google scholar</a></td><td><a href="pdf/663.pdf">pdf</a></td></tr>
<tr><td>664</td><td><a id="ref664" class="anchor"></a>Saedler, H., A. Gullon, L. Fiethen, and P. Starlinger. <i>1968</i>. Negative control of the galactose operon in <i>E. coli.</i>  <span class="journal">Mol. Gen. Genet. 102:79–88.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Negative%20control%20of%20the%20galactose%20operon%20in%20E.%20coli.">google scholar</a></td><td><a href="pdf/664.pdf">pdf</a></td></tr>
<tr><td>665</td><td><a id="ref665" class="anchor"></a>Saier, M. H., Jr., H. Straud, L. S. Massman, J. J. Judice, M. J. Newman, and B. U. Feucht. <i>1978</i>. Permease-specific mutations in <i>Salmonella typhimurium</i> and <i>Escherichia coli</i> that release the glycerol, maltose, melibiose, and lactose transport systems from regulation by the phosphoenolpyruvate:sugar phosphotransferase system.  <span class="journal">J. Bacteriol. 133:1358–1367.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Permease-specific%20mutations%20in%20Salmonella%20typhimurium%20and%20Escherichia%20coli%20that%20release%20the%20glycerol,%20maltose,%20melibiose,%20and%20lactose%20transport%20systems%20from%20regulation%20by%20the%20phosphoenolpyruvate:sugar%20phosphotransferase%20system">google scholar</a></td><td><a href="pdf/665.pdf">pdf</a></td></tr>
<tr><td>666</td><td><a id="ref666" class="anchor"></a>Saito, H., and C. C. Richardson. <i>1981</i>. Genetic analysis of gene 1.2 of bacteriophage T7: isolation of a mutant of <i>Escherichia coli</i> unable to support the growth of T7 gene 1.2 mutants.  <span class="journal">Virology 37:343–351.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20analysis%20of%20gene%201.2%20of%20bacteriophage%20T7:%20isolation%20of%20a%20mutant%20of%20Escherichia%20coli%20unable%20to%20support%20the%20growth%20of%20T7%20gene%201.2%20mutants">google scholar</a></td><td><a href="pdf/666.pdf">pdf</a></td></tr>
<tr><td>667</td><td><a id="ref667" class="anchor"></a>Saito, H., and H. Uchida. <i>1977</i>. Initiation of the DNA replication of bacteriophage lambda in <i>Escherichia coli</i> K12.  <span class="journal">J. Mol. Biol. 113:1–25.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Initiation%20of%20the%20DNA%20replication%20of%20bacteriophage%20lambda%20in%20Escherichia%20coli%20K12">google scholar</a></td><td><a href="pdf/667.pdf">pdf</a></td></tr>
<tr><td>668</td><td><a id="ref668" class="anchor"></a>Sakka, K., T. Watanabe, R. Beers, and H. C. Wu. <i>1987</i>. Isolation and characterization of a new globomycin-resistant <i>dnaE</i> mutant of <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 169:3400–3408.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20and%20characterization%20of%20a%20new%20globomycin-resistant%20dnaE%20mutant%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/668.pdf">pdf</a></td></tr>
<tr><td>669</td><td><a id="ref669" class="anchor"></a>Salanitro, J. P., and W. S. Wegener. <i>1971</i>. Growth of <i>Escherichia</i> coli on short-chain fatty acids: nature of the uptake system.  <span class="journal">J. Bacteriol. 18:893–900.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Growth%20of%20Escherichia%20coli%20on%20short-chain%20fatty%20acids:%20nature%20of%20the%20uptake%20system">google scholar</a></td><td><a href="pdf/669.pdf">pdf</a></td></tr>
<tr><td>670</td><td><a id="ref670" class="anchor"></a>Samsonov, V. V., E. R. Odoevskaia, and S. P. Sineokii. <i>1992</i>. Cloning and complementation analysis of the <i>Escherichia coli</i> <i>gpr</i> locus, influencing DNA replication of certain lambdoid phages.  <span class="journal">Genetika 28:39–45.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Cloning%20and%20complementation%20analysis%20of%20the%20Escherichia%20coli%20gpr%20locus,%20influencing%20DNA%20replication%20of%20certain%20lambdoid%20phages">google scholar</a></td><td><a href="pdf/670.pdf">pdf</a></td></tr>
<tr><td>671</td><td><a id="ref671" class="anchor"></a>Sanchez-Anzaldo, F. J., and F. Bastarrachea. <i>1974</i>. Genetic characterization of streptomycin- resistant and dependent mutants of <i>Escherichia coli</i> K-12.  <span class="journal">Mol. Gen. Genet. 130:47–64.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20characterization%20of%20streptomycin-%20resistant%20and%20dependent%20mutants%20of%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/671.pdf">pdf</a></td></tr>
<tr><td>672</td><td><a id="ref672" class="anchor"></a>Sanchez-Anzaldo, F. J., R. Gomez, and F. Bastarrachea. <i>1979</i>. Paromoycin resistant mutants of <i>Escherichia coli</i> K12. I. Cross-resistance to streptomycin and synergism of the mixture of both antibiotics.  <span class="journal">Rev. Latinoam. Microbiol. 21:121–128.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Paromoycin%20resistant%20mutants%20of%20Escherichia%20coli%20K12.%20I.%20Cross-resistance%20to%20streptomycin%20and%20synergism%20of%20the%20mixture%20of%20both%20antibiotics">google scholar</a></td><td><a href="pdf/672.pdf">pdf</a></td></tr>
<tr><td>673</td><td><a id="ref673" class="anchor"></a>Sanderson, K. E., and J. R. Roth. <i>1983</i>. Linkage map of <i>Salmonella typhimurium,</i> edition VI.  <span class="journal">Microbiol. Rev. 47:410–453.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Linkage%20map%20of%20Salmonella%20typhimurium,%20edition%20VI">google scholar</a></td><td><a href="pdf/673.pdf">pdf</a></td></tr>
<tr><td>674</td><td><a id="ref674" class="anchor"></a>Sanderson, K. E., and J. R. Roth. <i>1988</i>. Linkage map of <i>Salmonella typhimurium,</i> edition VII.  <span class="journal">Microbiol. Rev. 52:485–532.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Linkage%20map%20of%20Salmonella%20typhimurium,%20edition%20VII">google scholar</a></td><td><a href="pdf/674.pdf">pdf</a></td></tr>
<tr><td>675</td><td><a id="ref675" class="anchor"></a>Sanderson, K. E., and B. A. D. Stocker. <i>1981</i>. Gene <i>rfaH,</i> which affects lipopolysaccharide core structure in <i>Salmonella typhimurium,</i> is required also for expression of F-factor functions.  <span class="journal">J. Bacteriol. 146:535–541.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Gene%20rfaH,%20which%20affects%20lipopolysaccharide%20core%20structure%20in%20Salmonella%20typhimurium,%20is%20required%20also%20for%20expression%20of%20F-factor%20functions">google scholar</a></td><td><a href="pdf/675.pdf">pdf</a></td></tr>
<tr><td>676</td><td><a id="ref676" class="anchor"></a>Sanderson, K. E., J. Van Wyngaarten, O. Luderitz, and B. A. D. Stocker. <i>1974</i>. Rough mutants of <i>Salmonella typhimurium</i> with defects in the heptose region of the lipopolysaccharide core.  <span class="journal">Can. J. Microbiol. 20:1127–1134.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Rough%20mutants%20of%20Salmonella%20typhimurium%20with%20defects%20in%20the%20heptose%20region%20of%20the%20lipopolysaccharide%20core">google scholar</a></td><td><a href="pdf/676.pdf">pdf</a></td></tr>
<tr><td>677</td><td><a id="ref677" class="anchor"></a>Santangelo, J. D., D. T. Jones, and D. R. Woods. <i>1991</i>. Metronidazole activation and isolation of <i>Clostridium</i> <i>acetobutylicum</i> electron transport genes.  <span class="journal">J. Bacteriol. 173:1088–1095.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Metronidazole%20activation%20and%20isolation%20of%20Clostridium%20acetobutylicum%20electron%20transport%20genes">google scholar</a></td><td><a href="pdf/677.pdf">pdf</a></td></tr>
<tr><td>678</td><td><a id="ref678" class="anchor"></a>Sarrhy, A., S. Michaels, and J. Beckwith. <i>1981</i>. Deletion map of the <i>Escherichia coli</i> structural gene for alkaline phosphatase, <i>phoA.</i>  <span class="journal">J. Bacteriol. 145:288–292.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Deletion%20map%20of%20the%20Escherichia%20coli%20structural%20gene%20for%20alkaline%20phosphatase,%20phoA.">google scholar</a></td><td><a href="pdf/678.pdf">pdf</a></td></tr>
<tr><td>679</td><td><a id="ref679" class="anchor"></a>Sasarman, A., P. Chartrand, M. Lavoie, D. Tardif, R. Proschek, and C. Lapointe. <i>1979</i>. Mapping of a new <i>hem</i> gene in <i>Escherichia coli</i> K12.  <span class="journal">J. Gen. Microbiol. 113:297–303.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mapping%20of%20a%20new%20hem%20gene%20in%20Escherichia%20coli%20K12">google scholar</a></td><td><a href="pdf/679.pdf">pdf</a></td></tr>
<tr><td>680</td><td><a id="ref680" class="anchor"></a>Sasarman, A., P. Chartrand, R. Proschek, M. Desrochers, D. Tardif, and C. Lapointe. <i>1975</i>. Uroporphyrin-accumulating mutant of <i>Escherichia coli</i> K-12.  <span class="journal">J. Bacteriol. 124:1205–1212.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Uroporphyrin-accumulating%20mutant%20of%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/680.pdf">pdf</a></td></tr>
<tr><td>681</td><td><a id="ref681" class="anchor"></a>Sasarman, A., and M. Desrochers. <i>1976</i>. Uroporphyrinogen III cosynthase-deficient mutant of <i>Salmonella typhimurium</i> LT2.  <span class="journal">J. Bacteriol. 128:717–721.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Uroporphyrinogen%20III%20cosynthase-deficient%20mutant%20of%20Salmonella%20typhimurium%20LT2">google scholar</a></td><td><a href="pdf/681.pdf">pdf</a></td></tr>
<tr><td>682</td><td><a id="ref682" class="anchor"></a>Sasarman, A., M. Desrochers, S. Sonea, K. E. Sanderson, and M. Surdeanu. <i>1976</i>. Porphobilogen-accumulating mutants of <i>Salmonella typhimurium</i> LT2.  <span class="journal">J. Gen. Microbiol. 94:359– 366.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Porphobilogen-accumulating%20mutants%20of%20Salmonella%20typhimurium%20LT2">google scholar</a></td><td><a href="pdf/682.pdf">pdf</a></td></tr>
<tr><td>683</td><td><a id="ref683" class="anchor"></a>Sasarman, A., K. E. Sanderson, M. Surdeanu, and S. Sonea. <i>1970</i>. Hemin-deficient mutants of <i>Salmonella typhimurium.</i>  <span class="journal">J. Bacteriol. 102:531–536.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Hemin-deficient%20mutants%20of%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/683.pdf">pdf</a></td></tr>
<tr><td>684</td><td><a id="ref684" class="anchor"></a>Sastry, S. S., and R. Jayaraman. <i>1984</i>. Nitrofurantoin-resistant mutants of <i>Escherichia coli:</i> isolation and mapping.  <span class="journal">Mol. Gen. Genet. 196:379–380.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Nitrofurantoin-resistant%20mutants%20of%20Escherichia%20coli:%20isolation%20and%20mapping">google scholar</a></td><td><a href="pdf/684.pdf">pdf</a></td></tr>
<tr><td>685</td><td><a id="ref685" class="anchor"></a>Satre, M., C. Klein, and P. V. Vignais. <i>1978</i>. Isolation of <i>Escherichia coli</i> mutants with an adenosine triphosphatase insensitive to aurovertin.  <span class="journal">J. Bacteriol. 134:17–23.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20of%20Escherichia%20coli%20mutants%20with%20an%20adenosine%20triphosphatase%20insensitive%20to%20aurovertin">google scholar</a></td><td><a href="pdf/685.pdf">pdf</a></td></tr>
<tr><td>686</td><td><a id="ref686" class="anchor"></a>Schaefler, S. <i>1967</i>. Inducible system for the utilization of β-glucosides in <i>Escherichia coli.</i> I. Active transport and utilization of β-glucosides.  <span class="journal">J. Bacteriol. 93:254–263.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Inducible%20system%20for%20the%20utilization%20of%20β-glucosides%20in%20Escherichia%20coli.%20I.%20Active%20transport%20and%20utilization%20of%20β-glucosides">google scholar</a></td><td><a href="pdf/686.pdf">pdf</a></td></tr>
<tr><td>687</td><td><a id="ref687" class="anchor"></a>Schellenberg, G. D., and C. E. Furlong. <i>1977</i>. Resolution of the multiplicity of the glutamate and aspartate transport systems of <i>Escherichia coli.</i>  <span class="journal">J. Biol. Chem. 252:9055–9064.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Resolution%20of%20the%20multiplicity%20of%20the%20glutamate%20and%20aspartate%20transport%20systems%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/687.pdf">pdf</a></td></tr>
<tr><td>688</td><td><a id="ref688" class="anchor"></a>Schleif, R. <i>1969</i>. Isolation and characterization of a streptolydigin resistant RNA polymerase.  <span class="journal">Nature  223:1068–1069.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20and%20characterization%20of%20a%20streptolydigin%20resistant%20RNA%20polymerase">google scholar</a></td><td><a href="pdf/688.pdf">pdf</a></td></tr>
<tr><td>689</td><td><a id="ref689" class="anchor"></a>Schwartz, J. H., W. K. Mass, and E. J. Simon. <i>1959</i>. An impaired concentrating mechanism for amino acids in mutants of <i>Escherichia coli</i> resistant to L-canavanine and D-serine.  <span class="journal">Biochim. Biophys. Acta 32:582–583.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=An%20impaired%20concentrating%20mechanism%20for%20amino%20acids%20in%20mutants%20of%20Escherichia%20coli%20resistant%20to%20L-canavanine%20and%20D-serine">google scholar</a></td><td><a href="pdf/689.pdf">pdf</a></td></tr>
<tr><td>690</td><td><a id="ref690" class="anchor"></a>Schwarz, T. F. R., S. M. Yeats, and P. Connolly. <i>1981</i>. Altered transcriptional termination in a rifampicin-resistant mutant of <i>Escherichia coli</i> which inhibits the growth of bacteriophage T7.  <span class="journal">Mol. Gen. Genet. 183:181–186.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Altered%20transcriptional%20termination%20in%20a%20rifampicin-resistant%20mutant%20of%20Escherichia%20coli%20which%20inhibits%20the%20growth%20of%20bacteriophage%20T7">google scholar</a></td><td><a href="pdf/690.pdf">pdf</a></td></tr>
<tr><td>691</td><td><a id="ref691" class="anchor"></a>Schweizer, H., M. Argast, and W. Boos. <i>1982</i>. Characteristics of a binding protein-dependent transport system for <i>sn</i>-glycerol 3-phosphate in <i>Escherichia coli</i> that is part of the <i>pho</i> regulon.  <span class="journal">J. Bacteriol. 150:1154–1163.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Characteristics%20of%20a%20binding%20protein-dependent%20transport%20system%20for%20sn-glycerol%203-phosphate%20in%20Escherichia%20coli%20that%20is%20part%20of%20the%20pho%20regulon">google scholar</a></td><td><a href="pdf/691.pdf">pdf</a></td></tr>
<tr><td>692</td><td><a id="ref692" class="anchor"></a>Schweizer, H., T. Grussenmeyer, and W. Boos. <i>1982</i>. Mapping of two <i>ugp</i> genes coding for the <i>pho</i> regulon-dependent sn-glycerol 3-phosphate transport system of <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 150:1164–1171.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mapping%20of%20two%20ugp%20genes%20coding%20for%20the%20pho%20regulon-dependent%20sn-glycerol%203-phosphate%20transport%20system%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/692.pdf">pdf</a></td></tr>
<tr><td>693</td><td><a id="ref693" class="anchor"></a>Sedgwick, B., and P. Robins. <i>1980</i>. Isolation of mutants of <i>Escherichia coli</i> with increased resistance to alkylating agents: mutants deficient in thiols and mutants constitutive for the adaptive response.  <span class="journal">Mol. Gen. Genet. 180:85–90.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20of%20mutants%20of%20Escherichia%20coli%20with%20increased%20resistance%20to%20alkylating%20agents:%20mutants%20deficient%20in%20thiols%20and%20mutants%20constitutive%20for%20the%20adaptive%20response">google scholar</a></td><td><a href="pdf/693.pdf">pdf</a></td></tr>
<tr><td>694</td><td><a id="ref694" class="anchor"></a>Servin-Gonzalez, L., M. Ortiz, A. Gonzalez, and F. Bastarrachea. <i>1987</i>. <i>glnA</i> mutations conferring resistance to methylammonium in <i>Escherichia coli</i> K12.  <span class="journal">J. Gen. Microbiol. 133:1631– 1639.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=glnA%20mutations%20conferring%20resistance%20to%20methylammonium%20in%20Escherichia%20coli%20K12">google scholar</a></td><td><a href="pdf/694.pdf">pdf</a></td></tr>
<tr><td>695</td><td><a id="ref695" class="anchor"></a>Shakulov, R. S., V. A. Livshits, G. G. Zaigraeva, L. F. Lideman, and T. T. D. Mukhamed. <i>1978</i>. Partial suppression of the action of the gene <i>relA</i> in cells of <i>fus</i>r mutants of <i>Escherichia coli </i>K12.  <span class="journal">Sov. Genet. 14:632–640.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Partial%20suppression%20of%20the%20action%20of%20the%20gene%20relA%20in%20cells%20of%20fusr%20mutants%20of%20Escherichia%20coli%20K12">google scholar</a></td><td></td></tr>
<tr><td>696</td><td><a id="ref696" class="anchor"></a>Shaw, L, F. Grau, H. R. Kaback, J. S. Hong, and C. Walsh. <i>1975</i>. Vinylglycolate resistance in <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 121:1047–1055.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Vinylglycolate%20resistance%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/696.pdf">pdf</a></td></tr>
<tr><td>697</td><td><a id="ref697" class="anchor"></a>Sheldon, R., and S. Brenner. <i>1976</i>. Regulatory mutants of dihydrofolate reductase in <i>Escherichia</i> <i>coli</i> K12.  <span class="journal">Mol. Gen. Genet. 147:91–97.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulatory%20mutants%20of%20dihydrofolate%20reductase%20in%20Escherichia%20coli%20K12">google scholar</a></td><td><a href="pdf/697.pdf">pdf</a></td></tr>
<tr><td>698</td><td><a id="ref698" class="anchor"></a>Sheppard, D. E. <i>1964</i>. Mutants of <i>Salmonella typhimurium</i> resistant to feedback inhibition by L- histidine.  <span class="journal">Genetics 50:611–623.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutants%20of%20Salmonella%20typhimurium%20resistant%20to%20feedback%20inhibition%20by%20L-%20histidine">google scholar</a></td><td><a href="pdf/698.pdf">pdf</a></td></tr>
<tr><td>699</td><td><a id="ref699" class="anchor"></a>Shifrin, S., B. N. Ames, and G. F.-L. Ames. <i>1966</i>. Effect of the α-hydrazino analogue of histidine on histidine transport and arginine biosynthesis.  <span class="journal">J. Biol. Chem. 241:3424–3429.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Effect%20of%20the%20α-hydrazino%20analogue%20of%20histidine%20on%20histidine%20transport%20and%20arginine%20biosynthesis">google scholar</a></td><td><a href="pdf/699.pdf">pdf</a></td></tr>
<tr><td>700</td><td><a id="ref700" class="anchor"></a>Shimmin, L. C., D. Vanderwel, R. E. Harkness, B. R. Currie, C. A. Galloway, and E. E. Ishiguro. <i>1984</i>. Temperature-sensitive beta-lactam-tolerant mutants of <i>Escherichia coli.</i>  <span class="journal">J. Gen.Microbiol. 122:351–354.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Temperature-sensitive%20beta-lactam-tolerant%20mutants%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/700.pdf">pdf</a></td></tr>
<tr><td>701</td><td><a id="ref701" class="anchor"></a>Shive, W., and J. Macow. <i>1946</i>. Biochemical transformations as determined by competitive analogue-metabolite growth inhibitions. I. Some transformations involving aspartic acid.  <span class="journal">J. Biol. Chem. 162:451–462.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Biochemical%20transformations%20as%20determined%20by%20competitive%20analogue-metabolite%20growth%20inhibitions.%20I.%20Some%20transformations%20involving%20aspartic%20acid">google scholar</a></td><td><a href="pdf/701.pdf">pdf</a></td></tr>
<tr><td>702</td><td><a id="ref702" class="anchor"></a>Siden, I., and H. G. Boman. <i>1983</i>. <i>Escherichia coli</i> mutants with an altered sensitivity to cecropin D.  <span class="journal">J. Bacteriol. 154:170–176.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Escherichia%20coli%20mutants%20with%20an%20altered%20sensitivity%20to%20cecropin%20D">google scholar</a></td><td><a href="pdf/702.pdf">pdf</a></td></tr>
<tr><td>703</td><td><a id="ref703" class="anchor"></a>Siegel, E. C., and V. Bryson. <i>1967</i>. Mutator gene of <i>Escherichia coli</i> B.  <span class="journal">J. Bacteriol. 94:38–47.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutator%20gene%20of%20Escherichia%20coli%20B">google scholar</a></td><td><a href="pdf/703.pdf">pdf</a></td></tr>
<tr><td>704</td><td><a id="ref704" class="anchor"></a>Sigmund, C. D., M. Ettayebi, and E. A. Morgan. <i>1984</i>. Antibiotic resistance mutations in 16s and 23s ribosomal RNA genes of <i>Escherichia coli.</i>  <span class="journal">Nucleic Acids Res. 12:4653–4663.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Antibiotic%20resistance%20mutations%20in%2016s%20and%2023s%20ribosomal%20RNA%20genes%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/704.pdf">pdf</a></td></tr>
<tr><td>705</td><td><a id="ref705" class="anchor"></a>Signer, C. E., G. R. Smith, R. Cortese, and B. N. Ames. <i>1972</i>. Mutant tRNAhis ineffective in repression and lacking two pseudouridine modifications. <i>Nature</i>   <span class="journal">New Biol. 238:72–74.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutant%20tRNAhis%20ineffective%20in%20repression%20and%20lacking%20two%20pseudouridine%20modifications.%20Nature">google scholar</a></td><td><a href="pdf/705.pdf">pdf</a></td></tr>
<tr><td>706</td><td><a id="ref706" class="anchor"></a>Silbert, D. F., G. R. Fink, and B. N. Ames. <i>1966</i>. Histidine regulatory mutants in <i>Salmonella</i> <i>typhimurium.</i> III. A class of regulatory mutants deficient in tRNA for histidine.  <span class="journal">J. Mol. Biol. 22:335– 347.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Histidine%20regulatory%20mutants%20in%20Salmonella%20typhimurium.%20III.%20A%20class%20of%20regulatory%20mutants%20deficient%20in%20tRNA%20for%20histidine">google scholar</a></td><td><a href="pdf/706.pdf">pdf</a></td></tr>
<tr><td>707</td><td><a id="ref707" class="anchor"></a>Silhavy, T. J., I. Hartig-Beecken, and W. Boos. <i>1976</i>. Periplasmic protein related to the <i>sn</i>- glycerol 3-phosphate transport system of <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 126:951–958.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Periplasmic%20protein%20related%20to%20the%20sn-%20glycerol%203-phosphate%20transport%20system%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/707.pdf">pdf</a></td></tr>
<tr><td>708</td><td><a id="ref708" class="anchor"></a>Silver, S., P. Johnseine, E. Whitney, and D. Clark. <i>1972</i>. Manganese-resistant mutants of <i>Escherichia coli:</i> physiological and genetic studies.  <span class="journal">J. Bacteriol. 110:186–195.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Manganese-resistant%20mutants%20of%20Escherichia%20coli:%20physiological%20and%20genetic%20studies">google scholar</a></td><td><a href="pdf/708.pdf">pdf</a></td></tr>
<tr><td>709</td><td><a id="ref709" class="anchor"></a>Silverman, M., and M. Simon. <i>1973</i>. Genetic analysis of flagellar mutants in <i>Escherichia coli.</i> <i>J.</i>  <span class="journal">Bacteriol. 113:105–113.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20analysis%20of%20flagellar%20mutants%20in%20Escherichia%20coli.%20J.">google scholar</a></td><td><a href="pdf/709.pdf">pdf</a></td></tr>
<tr><td>710</td><td><a id="ref710" class="anchor"></a>Silverman, M., and M. I. Simon. <i>1977</i>. Bacterial flagella.  <span class="journal">Annu. Rev. Microbiol. 31:397–419.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Bacterial%20flagella">google scholar</a></td><td></td></tr>
<tr><td>711</td><td><a id="ref711" class="anchor"></a>Silverman, P., K. Nat, J. McEwen, and R. Birchman. <i>1980</i>. Selection of <i>Escherichia coli</i> K-12 mutants that prevent expression of F-plasmid functions.  <span class="journal">J. Bacteriol. 143:1519–1523.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Selection%20of%20Escherichia%20coli%20K-12%20mutants%20that%20prevent%20expression%20of%20F-plasmid%20functions">google scholar</a></td><td><a href="pdf/711.pdf">pdf</a></td></tr>
<tr><td>712</td><td><a id="ref712" class="anchor"></a>Silverstone, A. E., M. Goman, and J. G. Scaife. <i>1972</i>. ALT: a new factor involved in the synthesis of RNA by <i>Escherichia coli.</i>  <span class="journal">Mol. Gen. Genet. 118:223–234.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=ALT:%20a%20new%20factor%20involved%20in%20the%20synthesis%20of%20RNA%20by%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/712.pdf">pdf</a></td></tr>
<tr><td>713</td><td><a id="ref713" class="anchor"></a>Simons, R W., P. A. Egan, H. T. Chute, and W. D. Nunn. <i>1980</i>. Regulation of fatty acid degradation in <i>Escherichia coli:</i> isolation and characterization of strains bearing insertion and temperature-sensitive mutations in <i>fadR.</i>  <span class="journal">J. Bacteriol. 142:621–632.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation%20of%20fatty%20acid%20degradation%20in%20Escherichia%20coli:%20isolation%20and%20characterization%20of%20strains%20bearing%20insertion%20and%20temperature-sensitive%20mutations%20in%20fadR.">google scholar</a></td><td><a href="pdf/713.pdf">pdf</a></td></tr>
<tr><td>714</td><td><a id="ref714" class="anchor"></a>Sivasubramanian, N., and R. Jayaraman. <i>1980</i>. Mapping of two transcription mutations <i>(tlnI</i> and <i>tlnII</i>) conferring thiolutin resistance, adjacent to <i>dnaZ</i> and <i>rho</i> in <i>Escherichia coli.</i>  <span class="journal">Mol. Gen. Genet. 180:609–615.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mapping%20of%20two%20transcription%20mutations%20(tlnI%20and%20tlnII)%20conferring%20thiolutin%20resistance,%20adjacent%20to%20dnaZ%20and%20rho%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/714.pdf">pdf</a></td></tr>
<tr><td>715</td><td><a id="ref715" class="anchor"></a>Slater, A. C., M. C. Jones-Mortimer, and H. L Kornberg. <i>1981</i>. L-sorbose phosphorylation in <i>Escherichia coli</i> K-12.  <span class="journal">Biochim. Biophys. Acta 646:365–367.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=L-sorbose%20phosphorylation%20in%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/715.pdf">pdf</a></td></tr>
<tr><td>716</td><td><a id="ref716" class="anchor"></a>Slayman, C. W. <i>1973</i>. The genetic control of membrane transport.  <span class="journal">Curr. Top. Membr. Transp. 4:1–174.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The%20genetic%20control%20of%20membrane%20transport">google scholar</a></td><td><a href="pdf/716.pdf">pdf</a></td></tr>
<tr><td>717</td><td><a id="ref717" class="anchor"></a>Sledziewska, E., and D. Hulanicka. <i>1978</i>. Method of isolation of cysteine constitutive mutants of the cysteine regulon in <i>Salmonella typhimurium.</i>  <span class="journal">Mol. Gen. Genet. 165:289–293.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Method%20of%20isolation%20of%20cysteine%20constitutive%20mutants%20of%20the%20cysteine%20regulon%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/717.pdf">pdf</a></td></tr>
<tr><td>718</td><td><a id="ref718" class="anchor"></a>Smith, D. R., and J. M. Calvo. <i>1979</i>. Regulation of dihydrofolate reductase EC-1.5.1.3 synthesis in <i>Escherichia coli.</i>  <span class="journal">Mol. Gen. Genet. 175:31–38.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation%20of%20dihydrofolate%20reductase%20EC-1.5.1.3%20synthesis%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/718.pdf">pdf</a></td></tr>
<tr><td>719</td><td><a id="ref719" class="anchor"></a>Smith, D. R., J. I. Rood, P. I. Bird, M. K, Sneddon, J. M. Calvo, and J. F. Morrison. <i>1982</i>. Amplification and modification of dihydrofolate reductase EC-1.5.1.3 in <i>Escherichia coli</i> nucleotide sequence of <i>fol</i> genes from mutationally altered plasmids.  <span class="journal">J. Biol. Chem. 257:9043–9048.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Amplification%20and%20modification%20of%20dihydrofolate%20reductase%20EC-1.5.1.3%20in%20Escherichia%20coli%20nucleotide%20sequence%20of%20fol%20genes%20from%20mutationally%20altered%20plasmids">google scholar</a></td><td><a href="pdf/719.pdf">pdf</a></td></tr>
<tr><td>720</td><td><a id="ref720" class="anchor"></a>Smith, J. M., F. J. Smith, and H. E. Umbarger. <i>1979</i>. Mutations affecting the formation of acetohydroxyacid synthase II in <i>Escherichia coli</i> K12.  <span class="journal">Mol. Gen. Genet. 169:299–314.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutations%20affecting%20the%20formation%20of%20acetohydroxyacid%20synthase%20II%20in%20Escherichia%20coli%20K12">google scholar</a></td><td><a href="pdf/720.pdf">pdf</a></td></tr>
<tr><td>721</td><td><a id="ref721" class="anchor"></a>Smith, T. F., and J. R. Sadler. <i>1971</i>. The nature of lactose operator constitutive mutations. <i>J.</i>  <span class="journal">Mol. Biol. 59:273–305.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The%20nature%20of%20lactose%20operator%20constitutive%20mutations.%20J.">google scholar</a></td><td><a href="pdf/721.pdf">pdf</a></td></tr>
<tr><td>722</td><td><a id="ref722" class="anchor"></a>Solomon, E., and E. C. C. Lin. <i>1972</i>. Mutations affecting the dissimilation of mannitol by <i>Escherichia coli</i> K-12.  <span class="journal">J. Bacteriol. 111:566–574.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutations%20affecting%20the%20dissimilation%20of%20mannitol%20by%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/722.pdf">pdf</a></td></tr>
<tr><td>723</td><td><a id="ref723" class="anchor"></a>Somers, J. M., and W. W. Kay. <i>1983</i>. Genetic fine structure of the tricarboxylate transport (<i>tct</i>) locus of <i>Salmonella typhimurium.</i>  <span class="journal">Mol. Gen. Genet. 190:20–26.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20fine%20structure%20of%20the%20tricarboxylate%20transport%20(tct)%20locus%20of%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/723.pdf">pdf</a></td></tr>
<tr><td>724</td><td><a id="ref724" class="anchor"></a>Somers, J. M., G. D. Sweet, and W. W. Kay. <i>1981</i>. Fluorocitrate resistant tricarboxylate transport mutants of <i>Salmonella typhimurium.</i>  <span class="journal">Mol. Gen. Genet. 181:338–345.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Fluorocitrate%20resistant%20tricarboxylate%20transport%20mutants%20of%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/724.pdf">pdf</a></td></tr>
<tr><td>725</td><td><a id="ref725" class="anchor"></a>Sparling, P. F., and E. Blackman. <i>1973</i>. Mutation to erythromycin dependence in <i>Escherichia</i> <i>coli</i> K-12.  <span class="journal">J. Bacteriol. 116:74–83.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutation%20to%20erythromycin%20dependence%20in%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/725.pdf">pdf</a></td></tr>
<tr><td>726</td><td><a id="ref726" class="anchor"></a>Sparling, P. F., Y. Ikeya, and D. Elliot. <i>1973</i>. Two genetic loci for resistance to kasugamycin in <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 113:704–710.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Two%20genetic%20loci%20for%20resistance%20to%20kasugamycin%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/726.pdf">pdf</a></td></tr>
<tr><td>727</td><td><a id="ref727" class="anchor"></a>Sprague, G. F., Jr., R. M. Bell, and J. E. Cronan, Jr. <i>1975</i>. A mutant of <i>Escherichia coli</i> auxotrophic for organic phosphates: evidence for two defects in inorganic phosphate transport.  <span class="journal">Mol. Gen. Genet. 143:71–77.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20mutant%20of%20Escherichia%20coli%20auxotrophic%20for%20organic%20phosphates:%20evidence%20for%20two%20defects%20in%20inorganic%20phosphate%20transport">google scholar</a></td><td><a href="pdf/727.pdf">pdf</a></td></tr>
<tr><td>728</td><td><a id="ref728" class="anchor"></a>Springer, S. E., and R. E. Huber. <i>1973</i>. Sulfate and selenate uptake and transport in wild and in two selenate-tolerant strains of <i>Escherichia coli</i> K-12.  <span class="journal">Arch. Biochem. Biophys. 156:595–603.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Sulfate%20and%20selenate%20uptake%20and%20transport%20in%20wild%20and%20in%20two%20selenate-tolerant%20strains%20of%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/728.pdf">pdf</a></td></tr>
<tr><td>729</td><td><a id="ref729" class="anchor"></a>Sprinson, D. B., E. G. Gollub, R. C. Hu, and K.-P. Liu. <i>1976</i>. Regulation of tyrosine and phenylalanine biosynthesis in <i>Salmonella.</i>  <span class="journal">Acta Microbiol. Acad. Sci. Hung. 23:167–170.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation%20of%20tyrosine%20and%20phenylalanine%20biosynthesis%20in%20Salmonella.">google scholar</a></td><td></td></tr>
<tr><td>730</td><td><a id="ref730" class="anchor"></a>Stacey, K. A., and E. Simson. <i>1965</i>. Improved method for the isolation of thymine-requiring mutants of <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 90:554 555.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Improved%20method%20for%20the%20isolation%20of%20thymine-requiring%20mutants%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/730.pdf">pdf</a></td></tr>
<tr><td>731</td><td><a id="ref731" class="anchor"></a>Stalker, D. M., W. R. Hiatt, and L. Comai. <i>1985</i>. A single amino acid substitution in the enzyme 5-enolpyruvylshikimate-3-phosphate synthase confers resistance to the herbicide glyphosate.  <span class="journal">J. Biol. Chem. 260:4724–4728.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20single%20amino%20acid%20substitution%20in%20the%20enzyme%205-enolpyruvylshikimate-3-phosphate%20synthase%20confers%20resistance%20to%20the%20herbicide%20glyphosate">google scholar</a></td><td><a href="pdf/731.pdf">pdf</a></td></tr>
<tr><td>732</td><td><a id="ref732" class="anchor"></a>Stalmach, M. E., S. Grothe, and J. M. Wood. <i>1983</i>. Two proline porters in <i>Escherichia coli</i> K-12.  <span class="journal">J. Bacteriol. 156:481–486.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Two%20proline%20porters%20in%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/732.pdf">pdf</a></td></tr>
<tr><td>733</td><td><a id="ref733" class="anchor"></a>Staskawicz, B. J., and N. J. Panopoulos. <i>1980</i>. Phaseolotoxin transport in <i>Escherichia coli</i> and <i>Salmonella typhimurium</i> via the oligopeptide permease.  <span class="journal">J. Bacteriol. 142:474–479.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Phaseolotoxin%20transport%20in%20Escherichia%20coli%20and%20Salmonella%20typhimurium%20via%20the%20oligopeptide%20permease">google scholar</a></td><td><a href="pdf/733.pdf">pdf</a></td></tr>
<tr><td>734</td><td><a id="ref734" class="anchor"></a>Staudenbauer, W. L. <i>1976</i>. Replication of <i>Escherichia coli</i> DNA <i>in vitro:</i> inhibition by oxolinic acid.  <span class="journal">Eur. J. Biochem. 62:491–497.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Replication%20of%20Escherichia%20coli%20DNA%20in%20vitro:%20inhibition%20by%20oxolinic%20acid">google scholar</a></td><td><a href="pdf/734.pdf">pdf</a></td></tr>
<tr><td>735</td><td><a id="ref735" class="anchor"></a>Stauffer, G. V., and J. E. Brenchley. <i>1977</i>. Influence of methionine biosynthesis on serine transhydroxymethylase (EC 2.1.2.1) regulation in <i>Salmonella typhimurium</i> LT2.  <span class="journal">J. Bacteriol. 129:740–749.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Influence%20of%20methionine%20biosynthesis%20on%20serine%20transhydroxymethylase%20(EC%202.1.2.1)%20regulation%20in%20Salmonella%20typhimurium%20LT2">google scholar</a></td><td><a href="pdf/735.pdf">pdf</a></td></tr>
<tr><td>736</td><td><a id="ref736" class="anchor"></a>Steffes, C., J. Ellis, J. Wu, and B. P. Rosen. <i>1992</i>. The <i>lysP</i> gene encodes the lysine-specific permease.  <span class="journal">J. Bacteriol. 174:3242–3249.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The%20lysP%20gene%20encodes%20the%20lysine-specific%20permease">google scholar</a></td><td><a href="pdf/736.pdf">pdf</a></td></tr>
<tr><td>737</td><td><a id="ref737" class="anchor"></a>Stern, J. R., and R. W. O’Brien. <i>1969</i>. Oxidation of D-malic and β-alkyl malic acids by wild- type and mutant strains of <i>Salmonella typhimurium</i> and by <i>Aerobacter aerogenes.</i>  <span class="journal">J. Bacteriol. 98:147–151.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Oxidation%20of%20D-malic%20and%20β-alkyl%20malic%20acids%20by%20wild-%20type%20and%20mutant%20strains%20of%20Salmonella%20typhimurium%20and%20by%20Aerobacter%20aerogenes.">google scholar</a></td><td><a href="pdf/737.pdf">pdf</a></td></tr>
<tr><td>738</td><td><a id="ref738" class="anchor"></a>Stevens, F. J., and T. T. Wu. <i>1976</i>. Growth on D-lyxose of a mutant strain of <i>Escherichia coli</i> K12 using a novel isomerase and enzymes related to D-xylose metabolism.  <span class="journal">J. Gen. Microbiol. 97:257–265.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Growth%20on%20D-lyxose%20of%20a%20mutant%20strain%20of%20Escherichia%20coli%20K12%20using%20a%20novel%20isomerase%20and%20enzymes%20related%20to%20D-xylose%20metabolism">google scholar</a></td><td><a href="pdf/738.pdf">pdf</a></td></tr>
<tr><td>739</td><td><a id="ref739" class="anchor"></a>Stewart, V., and C. H. MacGregor. <i>1982</i>. Nitrate reductase in <i>Escherichia coli</i> K-12: involvement of <i>chlC, chlE,</i> and <i>chlG</i> loci.  <span class="journal">J. Bacteriol. 151:788–799.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Nitrate%20reductase%20in%20Escherichia%20coli%20K-12:%20involvement%20of%20chlC,%20chlE,%20and%20chlG%20loci">google scholar</a></td><td><a href="pdf/739.pdf">pdf</a></td></tr>
<tr><td>740</td><td><a id="ref740" class="anchor"></a>Stitt, B. L., H. R. Revel, I. Lielausis, and W. B. Wood. <i>1980</i>. Role of the host cell in bacteriophage T4 development. II. Characterization of host mutants that have pleiotropic effects on T4 growth.  <span class="journal">J. Virol. 35:775–789.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Role%20of%20the%20host%20cell%20in%20bacteriophage%20T4%20development.%20II.%20Characterization%20of%20host%20mutants%20that%20have%20pleiotropic%20effects%20on%20T4%20growth">google scholar</a></td><td><a href="pdf/740.pdf">pdf</a></td></tr>
<tr><td>741</td><td><a id="ref741" class="anchor"></a>Stocker, B. A. D., M. Nurminen, and P. H. Makela. <i>1979</i>. Mutants defective in the 33K outer membrane protein of <i>Salmonella typhimurium.</i>  <span class="journal">J. Bacteriol. 139:376–383.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutants%20defective%20in%20the%2033K%20outer%20membrane%20protein%20of%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/741.pdf">pdf</a></td></tr>
<tr><td>742</td><td><a id="ref742" class="anchor"></a>Stoeber, F., A. Lagarde, G. Nemoz, G. Novel, M. Novel, R Portalier, J. Pouyssegur, and J. Robert-Baudouy. <i>1974</i>. Le métabolisme des hexuronides et des hexuronates chez <i>Escherichia coli </i>K12: aspects physiologiques et génétiques de sa régulation.  <span class="journal">Biochimie 56:199–213.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Le%20métabolisme%20des%20hexuronides%20et%20des%20hexuronates%20chez%20Escherichia%20coli%20K12:%20aspects%20physiologiques%20et%20génétiques%20de%20sa%20régulation">google scholar</a></td><td><a href="pdf/742.pdf">pdf</a></td></tr>
<tr><td>743</td><td><a id="ref743" class="anchor"></a>Stouthamer, A. H. <i>1969</i>. A genetical and biochemical study of chlorate-resistant mutants of <i>Salmonella typhimurium.</i>  <span class="journal">Antonie van Leeuwenhoek J. Microbiol. Serol. 35:505–521.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20genetical%20and%20biochemical%20study%20of%20chlorate-resistant%20mutants%20of%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/743.pdf">pdf</a></td></tr>
<tr><td>744</td><td><a id="ref744" class="anchor"></a>Stouthamer, A. H., and C. W. Bettenhaussen. <i>1970</i>. Mapping a gene causing resistance to chlorate in <i>Salmonella typhimurium.</i>  <span class="journal">Antonie van Leeuwenhoek J. Microbiol. Serol. 36:555–565.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mapping%20a%20gene%20causing%20resistance%20to%20chlorate%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/744.pdf">pdf</a></td></tr>
<tr><td>745</td><td><a id="ref745" class="anchor"></a>Straus, D. S., and G. R. Hoffman. <i>1975</i>. Selection for a large genetic duplication in <i>Salmonella</i> <i>typhimurium.</i>  <span class="journal">Genetics 80:227–237.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Selection%20for%20a%20large%20genetic%20duplication%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/745.pdf">pdf</a></td></tr>
<tr><td>746</td><td><a id="ref746" class="anchor"></a>Stuttard, C. <i>1972</i>. Location of <i>trpR</i> mutations in the <i>serB-thr</i> region of <i>Salmonella typhimurium.</i>  <span class="journal">J. Bacteriol. 111:368–374.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Location%20of%20trpR%20mutations%20in%20the%20serB-thr%20region%20of%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/746.pdf">pdf</a></td></tr>
<tr><td>747</td><td><a id="ref747" class="anchor"></a>Sullivan, M. A., and R. M. Bock. <i>1985</i>. Isolation and characterization of antisuppressor mutations in <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 161:377–384.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20and%20characterization%20of%20antisuppressor%20mutations%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/747.pdf">pdf</a></td></tr>
<tr><td>748</td><td><a id="ref748" class="anchor"></a>Sullivan, M. A., J. F. Cannon, F. H. Webb, and R. M. Bock. <i>1985</i>. Antisuppressor mutation in <i>Escherichia coli</i> defective in biosynthesis of 5-methylaminomethyl-2-thiouridine.  <span class="journal">J. Bacteriol. 161:368–376.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Antisuppressor%20mutation%20in%20Escherichia%20coli%20defective%20in%20biosynthesis%20of%205-methylaminomethyl-2-thiouridine">google scholar</a></td><td><a href="pdf/748.pdf">pdf</a></td></tr>
<tr><td>749</td><td><a id="ref749" class="anchor"></a>Summers, W. C., and P. Raskin. <i>1993</i>. A method for selection of mutations at the <i>tdk</i> locus in <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 175:6049–6051.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20method%20for%20selection%20of%20mutations%20at%20the%20tdk%20locus%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/749.pdf">pdf</a></td></tr>
<tr><td>750</td><td><a id="ref750" class="anchor"></a>Sun, T.-P., and R. E. Webster. <i>1986</i>. <i>fii,</i> a bacterial locus required for filamentous phage infection, and its relation to colicin-tolerant <i>tolA</i> and <i>tolB.</i>  <span class="journal">J. Bacteriol. 165:107–115.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=fii,%20a%20bacterial%20locus%20required%20for%20filamentous%20phage%20infection,%20and%20its%20relation%20to%20colicin-tolerant%20tolA%20and%20tolB.">google scholar</a></td><td><a href="pdf/750.pdf">pdf</a></td></tr>
<tr><td>751</td><td><a id="ref751" class="anchor"></a>Sun, T.-P., and R. E. Webster. <i>1987</i>. Nucleotide sequence of a gene cluster involved in entry of E colicins and single-stranded DNA of infecting filamentous bacteriophages into <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 169:2667–2674.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Nucleotide%20sequence%20of%20a%20gene%20cluster%20involved%20in%20entry%20of%20E%20colicins%20and%20single-stranded%20DNA%20of%20infecting%20filamentous%20bacteriophages%20into%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/751.pdf">pdf</a></td></tr>
<tr><td>752</td><td><a id="ref752" class="anchor"></a>Sundararajan, T. A. <i>1963</i>. Interference with glycerokinase induction in mutants of <i>E. coli</i> accumulating gal-1-P.  <span class="journal">Proc. Natl. Acad. Sci. USA 50:463–469.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Interference%20with%20glycerokinase%20induction%20in%20mutants%20of%20E.%20coli%20accumulating%20gal-1-P">google scholar</a></td><td><a href="pdf/752.pdf">pdf</a></td></tr>
<tr><td>753</td><td><a id="ref753" class="anchor"></a>Sunshine, M., M. Feiss, J. Stuart, and J. Yochem. <i>1977</i>. A new host gene <i>(groPC)</i> necessary for lambda DNA replication.  <span class="journal">Mol. Gen. Genet. 151:27–34.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20new%20host%20gene%20(groPC)%20necessary%20for%20lambda%20DNA%20replication">google scholar</a></td><td><a href="pdf/753.pdf">pdf</a></td></tr>
<tr><td>754</td><td><a id="ref754" class="anchor"></a>Sunshine, M. G., and B. Sauer. <i>1975</i>. A bacterial mutation blocking P2 phage late gene expression.  <span class="journal">Proc. Natl. Acad. Sci. USA 72:2770–2774.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20bacterial%20mutation%20blocking%20P2%20phage%20late%20gene%20expression">google scholar</a></td><td><a href="pdf/754.pdf">pdf</a></td></tr>
<tr><td>755</td><td><a id="ref755" class="anchor"></a>Sutton, A., T. Newman, M. Francis, and M. Freundlich. <i>1981</i>. Valine-resistant <i>Escherichia coli</i> K-12 strains with mutations in the <i>ilvB</i> operon.  <span class="journal">J. Bacteriol. 148:998–1001.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Valine-resistant%20Escherichia%20coli%20K-12%20strains%20with%20mutations%20in%20the%20ilvB%20operon">google scholar</a></td><td><a href="pdf/755.pdf">pdf</a></td></tr>
<tr><td>756</td><td><a id="ref756" class="anchor"></a>Swedberg, C., S. Castensson, and O. Skold. <i>1979</i>. Characterization of mutationally altered dihydropteroate synthase and its ability to form a sulfonamide-containing dihydrofolate analog.  <span class="journal">J. Bacteriol. 137:129–136.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Characterization%20of%20mutationally%20altered%20dihydropteroate%20synthase%20and%20its%20ability%20to%20form%20a%20sulfonamide-containing%20dihydrofolate%20analog">google scholar</a></td><td><a href="pdf/756.pdf">pdf</a></td></tr>
<tr><td>757</td><td><a id="ref757" class="anchor"></a>Szentirmai, A., M. Szentirmai, and H. E. Umbarger. <i>1968</i>. Isoleucine and valine metabolism of <i>Escherichia coli.</i> XV. Biochemical properties of mutants resistant to thiaisoleucine.  <span class="journal">J. Bacteriol. 95:1672–1679.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isoleucine%20and%20valine%20metabolism%20of%20Escherichia%20coli.%20XV.%20Biochemical%20properties%20of%20mutants%20resistant%20to%20thiaisoleucine">google scholar</a></td><td><a href="pdf/757.pdf">pdf</a></td></tr>
<tr><td>758</td><td><a id="ref758" class="anchor"></a>Takahashi, H. <i>1978</i>. Genetic and physiological characterization of <i>Escherichia coli</i> K12 mutants <i>(tabC)</i> which induce the abortive infection of bacteriophage T4.  <span class="journal">Virology 87:256–265.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20and%20physiological%20characterization%20of%20Escherichia%20coli%20K12%20mutants%20(tabC)%20which%20induce%20the%20abortive%20infection%20of%20bacteriophage%20T4">google scholar</a></td><td><a href="pdf/758.pdf">pdf</a></td></tr>
<tr><td>759</td><td><a id="ref759" class="anchor"></a>Tamaki, S., H. Matsuzawa, and M. Matsuhashi. <i>1980</i>. Cluster of <i>mrdA</i> and <i>mrdB</i> genes responsible for the rod shape and mecillinam sensitivity of <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 141:52–57.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Cluster%20of%20mrdA%20and%20mrdB%20genes%20responsible%20for%20the%20rod%20shape%20and%20mecillinam%20sensitivity%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/759.pdf">pdf</a></td></tr>
<tr><td>760</td><td><a id="ref760" class="anchor"></a>Tamaki, S., T. Sato, and M. Matsuhashi. <i>1971</i>. Role of lipopolysaccharides in antibiotic resistance and bacteriophage adsorption of <i>Escherichia coli</i> K-12.  <span class="journal">J. Bacteriol. 105:968–975.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Role%20of%20lipopolysaccharides%20in%20antibiotic%20resistance%20and%20bacteriophage%20adsorption%20of%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/760.pdf">pdf</a></td></tr>
<tr><td>761</td><td><a id="ref761" class="anchor"></a>Tanaka, N., G. Kawano, and T. Kinoshita. <i>1971</i>. Chromosomal location of a fusidic acid resistant marker in <i>Escherichia coli.</i>  <span class="journal">Biochem. Biophys. Res. Commun. 42:564–567.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Chromosomal%20location%20of%20a%20fusidic%20acid%20resistant%20marker%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/761.pdf">pdf</a></td></tr>
<tr><td>762</td><td><a id="ref762" class="anchor"></a>Tang, C.-T., R. Engel, and B. E. Tropp. <i>1977</i>. L-Glyceraldehyde 3-phosphate, a bactericidal agent.  <span class="journal">Antimicrob. Agents Chemother. 11:147–153.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=L-Glyceraldehyde%203-phosphate,%20a%20bactericidal%20agent">google scholar</a></td><td><a href="pdf/762.pdf">pdf</a></td></tr>
<tr><td>763</td><td><a id="ref763" class="anchor"></a>Tapio, S., and L. A. Isaksson. <i>1990</i>. Antisuppression by mutations in elongation factor Tu. <i>Eur.</i>  <span class="journal">J. Biochem. 188:339–346.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Antisuppression%20by%20mutations%20in%20elongation%20factor%20Tu.%20Eur.">google scholar</a></td><td><a href="pdf/763.pdf">pdf</a></td></tr>
<tr><td>764</td><td><a id="ref764" class="anchor"></a>Thakar, J. H., and G. P. Kalle. <i>1968</i>. Defective guanine uptake in an 8-azaguanine-resistant mutant of <i>Salmonella typhimurium.</i>  <span class="journal">J. Bacteriol. 95:458–464.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Defective%20guanine%20uptake%20in%20an%208-azaguanine-resistant%20mutant%20of%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/764.pdf">pdf</a></td></tr>
<tr><td>765</td><td><a id="ref765" class="anchor"></a>Thèze, J., D. Margarita, G. N. Cohen, F. Borne, and J. C. Patte. <i>1974</i>. Mapping of the structural genes of the three aspartokinases and the two homoserine dehydrogenases of <i>Escherichia coli</i> K-12.  <span class="journal">J. Bacteriol. 117:133–143.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mapping%20of%20the%20structural%20genes%20of%20the%20three%20aspartokinases%20and%20the%20two%20homoserine%20dehydrogenases%20of%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/765.pdf">pdf</a></td></tr>
<tr><td>766</td><td><a id="ref766" class="anchor"></a>Thirion, J. P., and M. Hofnung. <i>1982</i>. On some genetic aspects of phage λ resistance in <i>E. coli</i> K12.  <span class="journal">Genetics 71:207–216.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=On%20some%20genetic%20aspects%20of%20phage%20λ%20resistance%20in%20E.%20coli%20K12">google scholar</a></td><td><a href="pdf/766.pdf">pdf</a></td></tr>
<tr><td>767</td><td><a id="ref767" class="anchor"></a>Thomas, G., and A. Favre. <i>1977</i>. Genetic mapping of a mutant exhibiting no near ultraviolet induced growth delay and lacking 4-thiouridine in its tRNA.  <span class="journal">C. R. Acad. Sci. 284:2285–2288.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20mapping%20of%20a%20mutant%20exhibiting%20no%20near%20ultraviolet%20induced%20growth%20delay%20and%20lacking%204-thiouridine%20in%20its%20tRNA">google scholar</a></td><td><a href="pdf/767.pdf">pdf</a></td></tr>
<tr><td>768</td><td><a id="ref768" class="anchor"></a>Thorbjarnardottir, S. H., R. A. Magnusdottir, and C. Eggertsson. <i>1978</i>. Mutations determining generalized resistance to aminoglycoside antibiotics in <i>Escherichia coli.</i>  <span class="journal">Mol. Gen. Genet. 161:89–98.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutations%20determining%20generalized%20resistance%20to%20aminoglycoside%20antibiotics%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/768.pdf">pdf</a></td></tr>
<tr><td>769</td><td><a id="ref769" class="anchor"></a>Thorne, C. M., and L. M. Corwin. <i>1975</i>. Mutations affecting aromatic amino acid transport in <i>Escherichia coli</i> and <i>Salmonella typhimurium.</i>  <span class="journal">J. Gen. Microbiol. 90:203–216.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutations%20affecting%20aromatic%20amino%20acid%20transport%20in%20Escherichia%20coli%20and%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/769.pdf">pdf</a></td></tr>
<tr><td>770</td><td><a id="ref770" class="anchor"></a>Tomas, J. M., and W. W. Kay. <i>1986</i>. Tellurite susceptibility and non-plasmid-mediated resistance in <i>Escherichia coli.</i>  <span class="journal">Antimicrob. Agents Chemother. 30:127–131.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Tellurite%20susceptibility%20and%20non-plasmid-mediated%20resistance%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/770.pdf">pdf</a></td></tr>
<tr><td>771</td><td><a id="ref771" class="anchor"></a>Tommassen, J., and B. Lugtenberg. <i>1981</i>. Localization of <i>phoE,</i> the structural gene for outer membrane protein E in <i>Escherichia coli</i> K-12.  <span class="journal">J. Bacteriol. 147:118–123.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Localization%20of%20phoE,%20the%20structural%20gene%20for%20outer%20membrane%20protein%20E%20in%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/771.pdf">pdf</a></td></tr>
<tr><td>772</td><td><a id="ref772" class="anchor"></a>Toone, W. M., K. E. Rudd, and J. D. Friesen. <i>1992</i>. Mutations causing aminotriazole resistance and temperature sensitivity reside in <i>gyrB,</i> which encodes the β subunit of DNA gyrase.  <span class="journal">J. Bacteriol. 174:5479–5481.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutations%20causing%20aminotriazole%20resistance%20and%20temperature%20sensitivity%20reside%20in%20gyrB,%20which%20encodes%20the%20β%20subunit%20of%20DNA%20gyrase">google scholar</a></td><td><a href="pdf/772.pdf">pdf</a></td></tr>
<tr><td>773</td><td><a id="ref773" class="anchor"></a>Torriani, A., and F. Rothman. <i>1961</i>. Mutants of <i>Escherichia coli</i> constitutive for alkaline phosphatase.  <span class="journal">J. Bacteriol. 81:835–836.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutants%20of%20Escherichia%20coli%20constitutive%20for%20alkaline%20phosphatase">google scholar</a></td><td><a href="pdf/773.pdf">pdf</a></td></tr>
<tr><td>774</td><td><a id="ref774" class="anchor"></a>Tosa, T., and L. I. Pizer. <i>1971</i>. Biochemical bases for the antimetabolite action of L-serine hydroxamate.  <span class="journal">J. Bacteriol. 106:972–982.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Biochemical%20bases%20for%20the%20antimetabolite%20action%20of%20L-serine%20hydroxamate">google scholar</a></td><td><a href="pdf/774.pdf">pdf</a></td></tr>
<tr><td>775</td><td><a id="ref775" class="anchor"></a>Tristram, H., and S. Neale. <i>1968</i>. The activity and specificity of the proline permease in wild- type and analogue-resistant strains of <i>Escherichia coli.</i>  <span class="journal">J. Gen. Microbiol. 50:121–137.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The%20activity%20and%20specificity%20of%20the%20proline%20permease%20in%20wild-%20type%20and%20analogue-resistant%20strains%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/775.pdf">pdf</a></td></tr>
<tr><td>776</td><td><a id="ref776" class="anchor"></a>Trun, N. J., and S. Gottesman. <i>1990</i>. On the bacterial cell cycle: <i>Escherichia coli</i> mutants with altered ploidy.  <span class="journal">Genes Dev. 4:2036–2047.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=On%20the%20bacterial%20cell%20cycle:%20Escherichia%20coli%20mutants%20with%20altered%20ploidy">google scholar</a></td><td><a href="pdf/776.pdf">pdf</a></td></tr>
<tr><td>777</td><td><a id="ref777" class="anchor"></a>Trun, N. J., and S. Gottesman. <i>1991</i>. Characterization of <i>Escherichia coli</i> mutants with altered ploidy.  <span class="journal">Res. Microbiol. 142:195–200.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Characterization%20of%20Escherichia%20coli%20mutants%20with%20altered%20ploidy">google scholar</a></td><td><a href="pdf/777.pdf">pdf</a></td></tr>
<tr><td>778</td><td><a id="ref778" class="anchor"></a>Tsay, J.-T., C. O. Rock, and S. Jackowski. <i>1992</i>. Overproduction of beta ketoacyl-acyl carrier protein synthase I imparts thiolactomycin resistance to <i>Escherichia coli</i> K-12.  <span class="journal">J. Bacteriol. 174:508– 513.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Overproduction%20of%20beta%20ketoacyl-acyl%20carrier%20protein%20synthase%20I%20imparts%20thiolactomycin%20resistance%20to%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/778.pdf">pdf</a></td></tr>
<tr><td>779</td><td><a id="ref779" class="anchor"></a>Udaka, S., and H. S. Moyed. <i>1963</i>. Inhibition of parental and mutant xanthosine 5′-phosphate aminases by psicofuranine.  <span class="journal">J. Biol. Chem. 238:2797–2803.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Inhibition%20of%20parental%20and%20mutant%20xanthosine%205′-phosphate%20aminases%20by%20psicofuranine">google scholar</a></td><td><a href="pdf/779.pdf">pdf</a></td></tr>
<tr><td>780</td><td><a id="ref780" class="anchor"></a>Umbarger, H. E. <i>1969</i>. Regulation of the biosynthesis of the branched-chain amino acids.  <span class="journal">Curr. Top. Cell. Regul. 1:57–76.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation%20of%20the%20biosynthesis%20of%20the%20branched-chain%20amino%20acids">google scholar</a></td><td><a href="pdf/780.pdf">pdf</a></td></tr>
<tr><td>781</td><td><a id="ref781" class="anchor"></a>Umbarger, H. E. <i>1971</i>. Metabolite analogs as genetic and biochemical probes.  <span class="journal">Adv. Genet. 16:119–140.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Metabolite%20analogs%20as%20genetic%20and%20biochemical%20probes">google scholar</a></td><td><a href="pdf/781.pdf">pdf</a></td></tr>
<tr><td>782</td><td><a id="ref782" class="anchor"></a>Umbarger, H. E. <i>1987</i>. Biosynthesis of the branched-chain amino acids. <span class="journal">Escherichia coli and Salmonella typhimurium: Cellular and Molecular Biology. American Society for Microbiology, Washington, D.C.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Biosynthesis%20of%20the%20branched-chain%20amino%20acids">google scholar</a></td><td><a href="pdf/782.pdf">pdf</a></td></tr>
<tr><td>783</td><td><a id="ref783" class="anchor"></a>Uzan, M., and A. Danchin. <i>1976</i>. A rapid test for the <i>relA</i> mutation in <i>E. coli.</i>  <span class="journal">Biochem. Biophys. Res. Commun. 69:751–758.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20rapid%20test%20for%20the%20relA%20mutation%20in%20E.%20coli.">google scholar</a></td><td><a href="pdf/783.pdf">pdf</a></td></tr>
<tr><td>784</td><td><a id="ref784" class="anchor"></a>Uzan, M., and A. Danchin. <i>1978</i>. Correlation between the serine sensitivity and the derepressibility of the <i>ilv</i> genes in <i>Escherichia coli relA</i> mutants.  <span class="journal">Mol. Gen. Genet. 165:21–30.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Correlation%20between%20the%20serine%20sensitivity%20and%20the%20derepressibility%20of%20the%20ilv%20genes%20in%20Escherichia%20coli%20relA%20mutants">google scholar</a></td><td><a href="pdf/784.pdf">pdf</a></td></tr>
<tr><td>785</td><td><a id="ref785" class="anchor"></a>Vaara, M. <i>1981</i>. Increased outer membrane resistance to ethylenediamine-tetraacetate and cations in novel lipid A mutants.  <span class="journal">J. Bacteriol. 148: 426–434.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Increased%20outer%20membrane%20resistance%20to%20ethylenediamine-tetraacetate%20and%20cations%20in%20novel%20lipid%20A%20mutants">google scholar</a></td><td><a href="pdf/785.pdf">pdf</a></td></tr>
<tr><td>786</td><td><a id="ref786" class="anchor"></a>Vaara, M., and T. Vaara. <i>1981</i>. Outer membrane permeability barrier disruption by polymyxin in polymyxin-susceptible and resistant <i>Salmonella typhimurium.</i>  <span class="journal">Antimicrob. Agents Chemother. 19:578–583.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Outer%20membrane%20permeability%20barrier%20disruption%20by%20polymyxin%20in%20polymyxin-susceptible%20and%20resistant%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/786.pdf">pdf</a></td></tr>
<tr><td>787</td><td><a id="ref787" class="anchor"></a>Vaara, M., T. Vaara, M. Jensen, I. Helander, M. Nurminen, E. T. Rietschel, and P. H. Makela. <i>1981</i>. Characterization of the lipopolysaccharide from the polymyxin-resistant <i>pmrA</i> mutants of <i>Salmonella typhimurium.</i>  <span class="journal">FEBS Lett. 129:145–149.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Characterization%20of%20the%20lipopolysaccharide%20from%20the%20polymyxin-resistant%20pmrA%20mutants%20of%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/787.pdf">pdf</a></td></tr>
<tr><td>788</td><td><a id="ref788" class="anchor"></a>Vaara, M., T. Vaara, and M. Sarvas. <i>1979</i>. Decreased binding of polymyxin by polymyxin- resistant mutants of <i>Salmonella typhimurium.</i>  <span class="journal">J. Bacteriol. 139:664–667.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Decreased%20binding%20of%20polymyxin%20by%20polymyxin-%20resistant%20mutants%20of%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/788.pdf">pdf</a></td></tr>
<tr><td>789</td><td><a id="ref789" class="anchor"></a>Vallari, D. S., and S. Jackowski. <i>1988</i>. Biosynthesis and degradation both contribute to the regulation of coenzyme A content in <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 170:3961–3966.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Biosynthesis%20and%20degradation%20both%20contribute%20to%20the%20regulation%20of%20coenzyme%20A%20content%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/789.pdf">pdf</a></td></tr>
<tr><td>790</td><td><a id="ref790" class="anchor"></a>Vallari, D. S., and C. O. Rock. <i>1987</i>. Isolation and characterization of temperature-sensitive pantothenate kinase (<i>coaA</i>) mutants of <i>Escherichia</i> <i>coli.</i>  <span class="journal">J. Bacteriol. 169:5795–5800.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20and%20characterization%20of%20temperature-sensitive%20pantothenate%20kinase%20(coaA)%20mutants%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/790.pdf">pdf</a></td></tr>
<tr><td>791</td><td><a id="ref791" class="anchor"></a>Van Alphen, L., B. Lugtenberg, R. Van Boxtel, A.-M. Hack, C. Verhoef, and L. Havekes. <i>1979</i>. <i>meoA</i> is the structural gene for outer membrane protein C of <i>Escherichia coli</i> K12.  <span class="journal">Mol. Gen. Genet. 169:147–156.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=meoA%20is%20the%20structural%20gene%20for%20outer%20membrane%20protein%20C%20of%20Escherichia%20coli%20K12">google scholar</a></td><td><a href="pdf/791.pdf">pdf</a></td></tr>
<tr><td>792</td><td><a id="ref792" class="anchor"></a>Van Alphen, W., B. Lugtenberg, and W. Berendsen. <i>1976</i>. Heptose-deficient mutants of <i>Escherichia coli</i> K12 deficient in up to three major outer membrane proteins.  <span class="journal">Mol. Gen. Genet.147:263–269.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Heptose-deficient%20mutants%20of%20Escherichia%20coli%20K12%20deficient%20in%20up%20to%20three%20major%20outer%20membrane%20proteins">google scholar</a></td><td><a href="pdf/792.pdf">pdf</a></td></tr>
<tr><td>793</td><td><a id="ref793" class="anchor"></a>Van Buul, C. P. J. J., and P. H. Van Knippenberg. <i>1985</i>. Nucleotide sequence of the <i>ksgA</i> gene of <i>Escherichia coli</i>—comparison of methyltransferases effecting dimethylation of adenosine in ribosomal RNA.  <span class="journal">Gene 38:65–72.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Nucleotide%20sequence%20of%20the%20ksgA%20gene%20of%20Escherichia%20coli—comparison%20of%20methyltransferases%20effecting%20dimethylation%20of%20adenosine%20in%20ribosomal%20RNA">google scholar</a></td><td><a href="pdf/793.pdf">pdf</a></td></tr>
<tr><td>794</td><td><a id="ref794" class="anchor"></a>Van de Klundert, J. A. M., E. Den Turk, A. H. Borman, P. H. Van der Meide, and L. Bosch. <i>1977</i>. Isolation and characterization of a mocimycin resistant mutant of <i>Escherichia coli</i> with an altered elongation factor EF-Tu.  <span class="journal">FEBS Lett. 81:303–307.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20and%20characterization%20of%20a%20mocimycin%20resistant%20mutant%20of%20Escherichia%20coli%20with%20an%20altered%20elongation%20factor%20EF-Tu">google scholar</a></td><td><a href="pdf/794.pdf">pdf</a></td></tr>
<tr><td>795</td><td><a id="ref795" class="anchor"></a>Van de Klundert, J. A. M., P. H. Van der Meide, P. Van de Putte, and L. Bosch. <i>1978</i>. Mutants of <i>Escherichia coli</i> altered in both genes coding for the elongation factor Tu.  <span class="journal">Proc. Natl. Acad. Sci. USA 75:4470–4473.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutants%20of%20Escherichia%20coli%20altered%20in%20both%20genes%20coding%20for%20the%20elongation%20factor%20Tu">google scholar</a></td><td><a href="pdf/795.pdf">pdf</a></td></tr>
<tr><td>796</td><td><a id="ref796" class="anchor"></a>Van Dyk, T. K., and R. A. LaRossa. <i>1986</i>. Sensitivity of a <i>Salmonella</i> <i>typhimurium</i> <i>aspC</i> mutant to sulfometuron methyl, a potent inhibitor of acetolactate synthase II.  <span class="journal">J. Bacteriol. 165:386–392.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Sensitivity%20of%20a%20Salmonella%20typhimurium%20aspC%20mutant%20to%20sulfometuron%20methyl,%20a%20potent%20inhibitor%20of%20acetolactate%20synthase%20II">google scholar</a></td><td><a href="pdf/796.pdf">pdf</a></td></tr>
<tr><td>797</td><td><a id="ref797" class="anchor"></a>Van Dyk, T. K., and R. A. LaRossa. <i>1987</i>. Involvement of <i>ack-pta</i> operon products in α- ketobutyrate metabolism by <i>Salmonella typhimurium.</i>  <span class="journal">Mol. Gen. Genet. 207:435–440.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Involvement%20of%20ack-pta%20operon%20products%20in%20α-%20ketobutyrate%20metabolism%20by%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/797.pdf">pdf</a></td></tr>
<tr><td>798</td><td><a id="ref798" class="anchor"></a>Vasiljevic, B., and L. Topisirovic. <i>1987</i>. Properties of ribosomes from neamine-dependent (NeaD) mutants of <i>Escherichia coli.</i>  <span class="journal">Period. Biol. 89:89–93.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Properties%20of%20ribosomes%20from%20neamine-dependent%20(NeaD)%20mutants%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/798.pdf">pdf</a></td></tr>
<tr><td>799</td><td><a id="ref799" class="anchor"></a>Venables, W. A., and J. R. Guest. <i>1968</i>. Transduction of nitrate reductase loci of <i>Escherichia coli</i> by phages Pl and λ.  <span class="journal">Mol. Gen. Genet. 103:127–140.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Transduction%20of%20nitrate%20reductase%20loci%20of%20Escherichia%20coli%20by%20phages%20Pl%20and%20λ">google scholar</a></td><td><a href="pdf/799.pdf">pdf</a></td></tr>
<tr><td>800</td><td><a id="ref800" class="anchor"></a>Venkateswaran, P. S., and H. C. Wu. <i>1972</i>. Isolation and characterization of a phosphonomycin- resistant mutant of <i>Escherichia coli</i> K-12.  <span class="journal">J. Bacteriol. 110:935–944.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20and%20characterization%20of%20a%20phosphonomycin-%20resistant%20mutant%20of%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/800.pdf">pdf</a></td></tr>
<tr><td>801</td><td><a id="ref801" class="anchor"></a>Verhof, C., B. Lugtenberg, R. van Boxtel, P. de Craaff, and H. Verheij. <i>1979</i>. Genetics and biochemistry of the peptidoglycan-associated proteins b and c of <i>Escherichia coli</i> K12.  <span class="journal">Mol. Gen. Genet. 169:137–146.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetics%20and%20biochemistry%20of%20the%20peptidoglycan-associated%20proteins%20b%20and%20c%20of%20Escherichia%20coli%20K12">google scholar</a></td><td><a href="pdf/801.pdf">pdf</a></td></tr>
<tr><td>802</td><td><a id="ref802" class="anchor"></a>Vinella, D., R. D’Ari, and P. Bouloc. <i>1992</i>. Penicillin-binding protein 2 is dispensable in <i>Escherichia coli</i> when ppGpp synthesis is induced.  <span class="journal">EMBO J. 11:1493–1501.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Penicillin-binding%20protein%202%20is%20dispensable%20in%20Escherichia%20coli%20when%20ppGpp%20synthesis%20is%20induced">google scholar</a></td><td><a href="pdf/802.pdf">pdf</a></td></tr>
<tr><td>803</td><td><a id="ref803" class="anchor"></a>Vinopal, R. T. <i>1987</i>. Selectable phenotypes. <span class="journal">Escherichia coli and Salmonella typhimurium: Cellular and Molecular Biology. American Society for Microbiology, Washington, D.C.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Selectable%20phenotypes">google scholar</a></td><td></td></tr>
<tr><td>804</td><td><a id="ref804" class="anchor"></a>Voll, M. J. <i>1967</i>. Polarity in the histidine operon. III. The isolation of prototrophic polar mutations.  <span class="journal">J. Mol. Biol. 30:109–124.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Polarity%20in%20the%20histidine%20operon.%20III.%20The%20isolation%20of%20prototrophic%20polar%20mutations">google scholar</a></td><td><a href="pdf/804.pdf">pdf</a></td></tr>
<tr><td>805</td><td><a id="ref805" class="anchor"></a>Voll, M. J., and L. Leive. <i>1970</i>. Release of lipopolysaccharide in <i>Escherichia coli</i> resistant to the permeability increase induced by ethylenediaminetetraacetate.  <span class="journal">J. Biol. Chem. 245:1108–1114.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Release%20of%20lipopolysaccharide%20in%20Escherichia%20coli%20resistant%20to%20the%20permeability%20increase%20induced%20by%20ethylenediaminetetraacetate">google scholar</a></td><td><a href="pdf/805.pdf">pdf</a></td></tr>
<tr><td>806</td><td><a id="ref806" class="anchor"></a>Vonder Haar, R. A., and H. E. Umbarger. <i>1972</i>. Isoleucine and valine metabolism in <i>Escherichia coli.</i> XIX. Inhibition of isoleucine biosynthesis by glycyl-leucine.  <span class="journal">J. Bacteriol. 112:142– 147.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isoleucine%20and%20valine%20metabolism%20in%20Escherichia%20coli.%20XIX.%20Inhibition%20of%20isoleucine%20biosynthesis%20by%20glycyl-leucine">google scholar</a></td><td><a href="pdf/806.pdf">pdf</a></td></tr>
<tr><td>807</td><td><a id="ref807" class="anchor"></a>Wachi, M., M. Doi, S. Tamaki, W. Park, S. Nakajima-Iijima, and M. Matsuhashi. <i>1987</i>. Mutant isolation and molecular cloning of <i>mre</i> genes which determine cell shape sensitivity to mecillinam and amount of penicillin-binding proteins in <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 169:4935– 4940.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutant%20isolation%20and%20molecular%20cloning%20of%20mre%20genes%20which%20determine%20cell%20shape%20sensitivity%20to%20mecillinam%20and%20amount%20of%20penicillin-binding%20proteins%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/807.pdf">pdf</a></td></tr>
<tr><td>808</td><td><a id="ref808" class="anchor"></a>Wada, C., and T. Yura. <i>1971</i>. Phenethyl alcohol resistance in <i>Escherichia coli.</i> II. Replication of F (fertility) factor in the resistant strain C600.  <span class="journal">Genetics 69:275–287.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Phenethyl%20alcohol%20resistance%20in%20Escherichia%20coli.%20II.%20Replication%20of%20F%20(fertility)%20factor%20in%20the%20resistant%20strain%20C600">google scholar</a></td><td><a href="pdf/808.pdf">pdf</a></td></tr>
<tr><td>809</td><td><a id="ref809" class="anchor"></a>Wada, C., and T. Yura. <i>1974</i>. Phenethyl alcohol resistance in <i>Escherichia coli.</i> III. A temperature-sensitive mutation <i>(dnaP)</i> affecting DNA replication.  <span class="journal">Genetics 77:199–220.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Phenethyl%20alcohol%20resistance%20in%20Escherichia%20coli.%20III.%20A%20temperature-sensitive%20mutation%20(dnaP)%20affecting%20DNA%20replication">google scholar</a></td><td><a href="pdf/809.pdf">pdf</a></td></tr>
<tr><td>810</td><td><a id="ref810" class="anchor"></a>Walker, G. C. <i>1987</i>. The SOS response of <i>Escherichia coli,</i>. <span class="journal">Escherichia coli and Salmonella typhimurium: Cellular and Molecular Biology. American Society for Microbiology, Washington, D.C.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The%20SOS%20response%20of%20Escherichia%20coli,">google scholar</a></td><td><a href="pdf/810.pdf">pdf</a></td></tr>
<tr><td>811</td><td><a id="ref811" class="anchor"></a>Wallace, B. J., and J. Pittard. <i>1969</i>. Regulator gene controlling enzymes concerned in tyrosine biosynthesis in <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 97:1234–1241.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulator%20gene%20controlling%20enzymes%20concerned%20in%20tyrosine%20biosynthesis%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/811.pdf">pdf</a></td></tr>
<tr><td>812</td><td><a id="ref812" class="anchor"></a>Wandersman, C., F. Moreno, and M. Schwartz. <i>1980</i>. Pleiotropic mutations rendering <i>Escherichia coli</i> K-12 resistant to bacteriophage TP1.  <span class="journal">J. Bacteriol. 143:1374–1383.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Pleiotropic%20mutations%20rendering%20Escherichia%20coli%20K-12%20resistant%20to%20bacteriophage%20TP1">google scholar</a></td><td><a href="pdf/812.pdf">pdf</a></td></tr>
<tr><td>813</td><td><a id="ref813" class="anchor"></a>Wargel, R. J., C. A. Shadur, and F. C. Neuhaus. <i>1971</i>. Mechanism of D-cycloserine action: transport mutants for D-alanine, D-cycloserine, and glycine.  <span class="journal">J. Bacteriol. 105:1028–1035.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mechanism%20of%20D-cycloserine%20action:%20transport%20mutants%20for%20D-alanine,%20D-cycloserine,%20and%20glycine">google scholar</a></td><td><a href="pdf/813.pdf">pdf</a></td></tr>
<tr><td>814</td><td><a id="ref814" class="anchor"></a>Watanabe, H., H. Hashimoto, and S. Mitsuhashi. <i>1980</i>. <i>Salmonella typhimurium</i> LT2 mutation affecting the deletion of resistance determinants on R plasmids.  <span class="journal">J. Bacteriol. 142:145–152.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Salmonella%20typhimurium%20LT2%20mutation%20affecting%20the%20deletion%20of%20resistance%20determinants%20on%20R%20plasmids">google scholar</a></td><td><a href="pdf/814.pdf">pdf</a></td></tr>
<tr><td>815</td><td><a id="ref815" class="anchor"></a>Watanabe, N.-A., T. Nagasu, K. Katsu, and K. Kitoh. <i>1987</i>. E-0702, a new cephalosporin, is incorporated into <i>Escherichia coli</i> cells via the <i>tonB</i>-dependent iron transport system.  <span class="journal">Antimicrob. Agents Chemother. 31:497–504.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=E-0702,%20a%20new%20cephalosporin,%20is%20incorporated%20into%20Escherichia%20coli%20cells%20via%20the%20tonB-dependent%20iron%20transport%20system">google scholar</a></td><td><a href="pdf/815.pdf">pdf</a></td></tr>
<tr><td>816</td><td><a id="ref816" class="anchor"></a>Watson, C., and K. Paigen. <i>1971</i>. Isolation and characterization of an <i>Escherichia coli</i> bacteriophage requiring cell wall galactose.  <span class="journal">J. Virol. 8:669–674.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation%20and%20characterization%20of%20an%20Escherichia%20coli%20bacteriophage%20requiring%20cell%20wall%20galactose">google scholar</a></td><td><a href="pdf/816.pdf">pdf</a></td></tr>
<tr><td>817</td><td><a id="ref817" class="anchor"></a>Webb, M. <i>1970</i>. The mechanism of acquired resistance to Co2+ and Ni2+ in gram-positive and gram-negative bacteria.  <span class="journal">Biochim. Biophys. Acta 222:440–446.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The%20mechanism%20of%20acquired%20resistance%20to%20Co2+%20and%20Ni2+%20in%20gram-positive%20and%20gram-negative%20bacteria">google scholar</a></td><td><a href="pdf/817.pdf">pdf</a></td></tr>
<tr><td>818</td><td><a id="ref818" class="anchor"></a>Weber, J., R. S. F. Lee, E. Grell, and A. E. Senior. <i>1992</i>. Investigation of the aurovertin binding site of <i>Escherichia coli</i> F-1 ATPase by fluorescence spectroscopy and site-directed mutagenesis.  <span class="journal">Biochemistry 31:5112–5116.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Investigation%20of%20the%20aurovertin%20binding%20site%20of%20Escherichia%20coli%20F-1%20ATPase%20by%20fluorescence%20spectroscopy%20and%20site-directed%20mutagenesis">google scholar</a></td><td><a href="pdf/818.pdf">pdf</a></td></tr>
<tr><td>819</td><td><a id="ref819" class="anchor"></a>Wei, C. R., and S. Kustu. <i>1981</i>. Glutamine auxotrophs with mutations in a nitrogen regulatory gene, <i>ntrC,</i> that is near <i>glnA.</i>  <span class="journal">Mol. Gen. Genet. 183:392–399.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Glutamine%20auxotrophs%20with%20mutations%20in%20a%20nitrogen%20regulatory%20gene,%20ntrC,%20that%20is%20near%20glnA.">google scholar</a></td><td><a href="pdf/819.pdf">pdf</a></td></tr>
<tr><td>820</td><td><a id="ref820" class="anchor"></a>Weiner, J. H., C. E. Furlong, and L. A. Heppel. <i>1971</i>. A binding protein for L-glutamine and its relation to active transport in <i>E.</i>  <span class="journal">coli. Arch. Biochem. Biophys. 142:715–717.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20binding%20protein%20for%20L-glutamine%20and%20its%20relation%20to%20active%20transport%20in%20E.">google scholar</a></td><td><a href="pdf/820.pdf">pdf</a></td></tr>
<tr><td>821</td><td><a id="ref821" class="anchor"></a>Weiner, J. H., and L. A. Heppel. <i>1971</i>. A binding protein for glutamine and its relation to active transport in <i>E. coli.</i>  <span class="journal">J. Biol. Chem. 246:6933–6941.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20binding%20protein%20for%20glutamine%20and%20its%20relation%20to%20active%20transport%20in%20E.%20coli.">google scholar</a></td><td><a href="pdf/821.pdf">pdf</a></td></tr>
<tr><td>822</td><td><a id="ref822" class="anchor"></a>White, B. J., S. J. Hochhauser, N. M. Cintron, and B. Weiss. <i>1976</i>. Genetic mapping of <i>xthA,</i> the structural gene for exonuclease III in <i>Escherichia coli</i> strain K12.  <span class="journal">J. Bacteriol. 126:1082–1088.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20mapping%20of%20xthA,%20the%20structural%20gene%20for%20exonuclease%20III%20in%20Escherichia%20coli%20strain%20K12">google scholar</a></td><td><a href="pdf/822.pdf">pdf</a></td></tr>
<tr><td>823</td><td><a id="ref823" class="anchor"></a>White, R. J., and P. W. West. <i>1970</i>. An examination of the inhibitory effects of N- iodoacetylglucosamine on <i>Escherichia coli</i> and isolation of resistant mutants.  <span class="journal">Biochem. J. 118:81–87.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=An%20examination%20of%20the%20inhibitory%20effects%20of%20N-%20iodoacetylglucosamine%20on%20Escherichia%20coli%20and%20isolation%20of%20resistant%20mutants">google scholar</a></td><td><a href="pdf/823.pdf">pdf</a></td></tr>
<tr><td>824</td><td><a id="ref824" class="anchor"></a>Whitfield, H. J., Jr. <i>1971</i>. Purification and properties of the wild type and a feedback resistant phosphoribosyladenosine triphosphate pyrophosphate phosphoribosyltransferase, the first enzyme of histidine biosynthesis in <i>Salmonella typhimurium.</i>  <span class="journal">J. Biol. Chem. 246:899–908.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Purification%20and%20properties%20of%20the%20wild%20type%20and%20a%20feedback%20resistant%20phosphoribosyladenosine%20triphosphate%20pyrophosphate%20phosphoribosyltransferase,%20the%20first%20enzyme%20of%20histidine%20biosynthesis%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/824.pdf">pdf</a></td></tr>
<tr><td>825</td><td><a id="ref825" class="anchor"></a>Wiater, A., M. Filutowicz, and D. Hulanicka. <i>1982</i>. A new class of mutants of the <i>cysB</i> regulatory gene for cysteine biosynthesis in <i>Salmonella typhimurium.</i>  <span class="journal">J. Gen. Microbiol. 128:1785– 1790.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20new%20class%20of%20mutants%20of%20the%20cysB%20regulatory%20gene%20for%20cysteine%20biosynthesis%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/825.pdf">pdf</a></td></tr>
<tr><td>826</td><td><a id="ref826" class="anchor"></a>Wiater, A., and D. Hulanicka. <i>1978</i>. The regulatory <i>cysK</i> mutant of <i>S. typhimurium.</i>  <span class="journal">Acta Biochim. Pol. 25:281–287.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The%20regulatory%20cysK%20mutant%20of%20S.%20typhimurium.">google scholar</a></td><td></td></tr>
<tr><td>827</td><td><a id="ref827" class="anchor"></a>Wiater, A., and D. Hulanicka. <i>1979</i>. Properties of <i>cysK</i> mutants of <i>Escherichia coli</i> K12. <i> <span class="journal">Acta Biochim. Pol. 26:21–28.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Properties%20of%20cysK%20mutants%20of%20Escherichia%20coli%20K12.%20">google scholar</a></td><td><a href="pdf/827.pdf">pdf</a></td></tr>
<tr><td>828</td><td><a id="ref828" class="anchor"></a>Wiater, A., and T. Klopotowski. <i>1972</i>. Mutations rendering <i>Salmonella typhimurium</i> resistant to 3-aminotriazole in the presence of histidine.  <span class="journal">Acta Biochim. Pol. 19:191–199.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutations%20rendering%20Salmonella%20typhimurium%20resistant%20to%203-aminotriazole%20in%20the%20presence%20of%20histidine">google scholar</a></td><td><a href="pdf/828.pdf">pdf</a></td></tr>
<tr><td>829</td><td><a id="ref829" class="anchor"></a>Wijsman, H. J. W., and H. C. Pafort. <i>1974</i>. Pleiotropic mutations in <i>Escherichia coli</i> conferring tolerance to glycine and sensitivity to penicillin.  <span class="journal">Mol. Gen. Genet. 128:349–357.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Pleiotropic%20mutations%20in%20Escherichia%20coli%20conferring%20tolerance%20to%20glycine%20and%20sensitivity%20to%20penicillin">google scholar</a></td><td><a href="pdf/829.pdf">pdf</a></td></tr>
<tr><td>830</td><td><a id="ref830" class="anchor"></a>Wild, D. G. <i>1988</i>. Reversion from erythromycin dependence in <i>Escherichia coli:</i> strains altered in ribosomal sub-unit association and ribosome assembly.  <span class="journal">J. Gen. Microbiol. 134:1251–1263.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Reversion%20from%20erythromycin%20dependence%20in%20Escherichia%20coli:%20strains%20altered%20in%20ribosomal%20sub-unit%20association%20and%20ribosome%20assembly">google scholar</a></td><td><a href="pdf/830.pdf">pdf</a></td></tr>
<tr><td>831</td><td><a id="ref831" class="anchor"></a>Wild, J., and T. Klopotowski. <i>1975</i>. Insensitivity of D-amino acid dehydrogenase synthesis to catabolite repression in <i>dadR</i> mutants of <i>Salmonella typhimurium.</i>  <span class="journal">Mol. Gen. Genet. 136:63–73.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Insensitivity%20of%20D-amino%20acid%20dehydrogenase%20synthesis%20to%20catabolite%20repression%20in%20dadR%20mutants%20of%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/831.pdf">pdf</a></td></tr>
<tr><td>832</td><td><a id="ref832" class="anchor"></a>Wild, J., and T. Klopotowski. <i>1981</i>. D-amino acid dehydrogenase of <i>Escherichia coli</i> K12: positive selection of mutants defective in the enzyme activity and localization of the structural gene.  <span class="journal">Mol. Gen. Genet. 181:373–378.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=D-amino%20acid%20dehydrogenase%20of%20Escherichia%20coli%20K12:%20positive%20selection%20of%20mutants%20defective%20in%20the%20enzyme%20activity%20and%20localization%20of%20the%20structural%20gene">google scholar</a></td><td><a href="pdf/832.pdf">pdf</a></td></tr>
<tr><td>833</td><td><a id="ref833" class="anchor"></a>Wilkinson, R. C., P. Gemski, Jr., and B. A. D. Stocker. <i>1972</i>. Non-smooth mutants of <i>Salmonella typhimurium:</i> differentiation by phage sensitivity and genetic mapping.  <span class="journal">J. Gen. Microbiol. 70:527–554.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Non-smooth%20mutants%20of%20Salmonella%20typhimurium:%20differentiation%20by%20phage%20sensitivity%20and%20genetic%20mapping">google scholar</a></td><td><a href="pdf/833.pdf">pdf</a></td></tr>
<tr><td>834</td><td><a id="ref834" class="anchor"></a>Williams, J. C., C. E. Lee, and J. R. Wild. <i>1980</i>. Genetic and biochemical characterization of distinct transport systems for uracil, uridine, and cytidine in <i>Salmonella typhimurium.</i>  <span class="journal">Mol. Gen. Genet. 178:121–130.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic%20and%20biochemical%20characterization%20of%20distinct%20transport%20systems%20for%20uracil,%20uridine,%20and%20cytidine%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/834.pdf">pdf</a></td></tr>
<tr><td>835</td><td><a id="ref835" class="anchor"></a>Williams, M. D., J. A. Fuchs, and M. C. Flickinger. <i>1991</i>. Null mutations in the stringent starvation protein of <i>Escherichia coli</i> disrupt lytic development of bacteriophage P1.  <span class="journal">Gene 109:21–30.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Null%20mutations%20in%20the%20stringent%20starvation%20protein%20of%20Escherichia%20coli%20disrupt%20lytic%20development%20of%20bacteriophage%20P1">google scholar</a></td><td><a href="pdf/835.pdf">pdf</a></td></tr>
<tr><td>836</td><td><a id="ref836" class="anchor"></a>Williams, M. V., T. J. Kerr, R. D. Lemmon, and C. J. Tritz. <i>1980</i>. Azaserine resistance in <i>Escherichia coli:</i> chromosomal location of multiple genes.  <span class="journal">J. Bacteriol. 143:383–388.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Azaserine%20resistance%20in%20Escherichia%20coli:%20chromosomal%20location%20of%20multiple%20genes">google scholar</a></td><td><a href="pdf/836.pdf">pdf</a></td></tr>
<tr><td>837</td><td><a id="ref837" class="anchor"></a>Williams, M. V., J. J. Rowe, T. J. Kerr, and G. J. Tritz. <i>1977</i>. Studies on the modes of action of azaserine in <i>Escherichia coli</i> mechanism of resistance to azaserine.  <span class="journal">Microbios 19:181–190.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Studies%20on%20the%20modes%20of%20action%20of%20azaserine%20in%20Escherichia%20coli%20mechanism%20of%20resistance%20to%20azaserine">google scholar</a></td><td><a href="pdf/837.pdf">pdf</a></td></tr>
<tr><td>838</td><td><a id="ref838" class="anchor"></a>Willsky, G. R., R. L. Bennett, and M. H. Malamy. <i>1973</i>. Inorganic phosphate transport in <i>Escherichia coli:</i> involvement of two genes which play a role in alkaline phosphatase regulation.  <span class="journal">J. Bacteriol. 113:529–539.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Inorganic%20phosphate%20transport%20in%20Escherichia%20coli:%20involvement%20of%20two%20genes%20which%20play%20a%20role%20in%20alkaline%20phosphatase%20regulation">google scholar</a></td><td><a href="pdf/838.pdf">pdf</a></td></tr>
<tr><td>839</td><td><a id="ref839" class="anchor"></a>Wittenbach, V. A., D. Rayner, and J. V. Schloss. <i>1992</i>. Pressure points in the biosynthetic pathway for branched chain amino acids. <span class="journal">Biosynthesis and Molecular Regulation of Amino Acids in Plants. American Society of Plant Physiologists, Rockville, Md.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Pressure%20points%20in%20the%20biosynthetic%20pathway%20for%20branched%20chain%20amino%20acids">google scholar</a></td><td><a href="pdf/839.pdf">pdf</a></td></tr>
<tr><td>840</td><td><a id="ref840" class="anchor"></a>Wittman, H. G., C. Stoffler, D. Apirion, L. Rosen, K. Tanaka, M. Tamaki, R. Takata, S. Dekio, E. Otaka, and S. Osawa. <i>1973</i>. Biochemical and genetic studies on two different types of erythromycin resistant mutants of <i>Escherichia coli</i> with altered ribosomal proteins.  <span class="journal">Mol. Gen. Genet. 127:175–189.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Biochemical%20and%20genetic%20studies%20on%20two%20different%20types%20of%20erythromycin%20resistant%20mutants%20of%20Escherichia%20coli%20with%20altered%20ribosomal%20proteins">google scholar</a></td><td><a href="pdf/840.pdf">pdf</a></td></tr>
<tr><td>841</td><td><a id="ref841" class="anchor"></a>Wolfner, M. D. Yep, F. Messenguy, and G. Fink. <i>1975</i>. Integration of amino acid biosynthesis into the cell cycle of <i>Saccharomyces</i> <i>cerevisiae.</i>  <span class="journal">J. Mol. Biol. 96:273–290.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Integration%20of%20amino%20acid%20biosynthesis%20into%20the%20cell%20cycle%20of%20Saccharomyces%20cerevisiae.">google scholar</a></td><td><a href="pdf/841.pdf">pdf</a></td></tr>
<tr><td>842</td><td><a id="ref842" class="anchor"></a>Wolfson, J. S., D. C. Hooper, G. L. McHugh, M. A. Bozza, and M. N. Swartz. <i>1990</i>. Mutants of <i>Escherichia coli</i> K12 exhibiting reduced killing by both quinolone and beta-lactam antimicrobial agents.  <span class="journal">Antimicrob. Agents Chemother. 34:1938–1943.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutants%20of%20Escherichia%20coli%20K12%20exhibiting%20reduced%20killing%20by%20both%20quinolone%20and%20beta-lactam%20antimicrobial%20agents">google scholar</a></td><td><a href="pdf/842.pdf">pdf</a></td></tr>
<tr><td>843</td><td><a id="ref843" class="anchor"></a>Wood, J. M. <i>1981</i>. Genetics of L-proline utilization in <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 146:895–901.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetics%20of%20L-proline%20utilization%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/843.pdf">pdf</a></td></tr>
<tr><td>844</td><td><a id="ref844" class="anchor"></a>Woodward, M. J., and H. P. Charles. <i>1982</i>. Genes for L-sorbose utilization in <i>Escherichia coli.</i>  <span class="journal">J. Gen. Microbiol. 128:1969–1980.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genes%20for%20L-sorbose%20utilization%20in%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/844.pdf">pdf</a></td></tr>
<tr><td>845</td><td><a id="ref845" class="anchor"></a>Woodward, M. J., and H. P. Charles. <i>1983</i>. Polymorphism in <i>Escherichia coli: rtl, atl,</i> and <i>gat</i> regions behave as chromosomal alterations.  <span class="journal">J. Gen. Microbiol. 129:75–84.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Polymorphism%20in%20Escherichia%20coli:%20rtl,%20atl,%20and%20gat%20regions%20behave%20as%20chromosomal%20alterations">google scholar</a></td><td><a href="pdf/845.pdf">pdf</a></td></tr>
<tr><td>846</td><td><a id="ref846" class="anchor"></a>Wooldridge, K. G., and P. H. Williams. <i>1991</i>. Sensitivity of <i>Escherichia coli</i> to cloacin DF13 involves the major outer membrane protein OmpF.  <span class="journal">J. Bacteriol. 173:2420–2424.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Sensitivity%20of%20Escherichia%20coli%20to%20cloacin%20DF13%20involves%20the%20major%20outer%20membrane%20protein%20OmpF">google scholar</a></td><td><a href="pdf/846.pdf">pdf</a></td></tr>
<tr><td>847</td><td><a id="ref847" class="anchor"></a>Wu, R., S. Yang, K. Jin, X. Chu, and M. Li. <i>1987</i>. Cloning and mutagenesis of threonine operon.  <span class="journal">Shengwu Gongcheng Xuebao 3:177–182.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Cloning%20and%20mutagenesis%20of%20threonine%20operon">google scholar</a></td><td><a href="pdf/847.pdf">pdf</a></td></tr>
<tr><td>848</td><td><a id="ref848" class="anchor"></a>Wu, T. T. <i>1976</i>. Growth of a mutant of <i>Escherichia coli</i> K12 on xylitol by recruiting enzymes for D-xylose and L-1,2-propanediol.  <span class="journal">Biochim. Biophys. Acta 428:656–666.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Growth%20of%20a%20mutant%20of%20Escherichia%20coli%20K12%20on%20xylitol%20by%20recruiting%20enzymes%20for%20D-xylose%20and%20L-1,2-propanediol">google scholar</a></td><td><a href="pdf/848.pdf">pdf</a></td></tr>
<tr><td>849</td><td><a id="ref849" class="anchor"></a>Wu, T. T. <i>1976</i>. Growth on D-arabitol of a mutant strain of <i>Escherichia coli</i> K12 using a novel dehydrogenase and enzymes related to L-1,2-propanediol and D-xylose metabolism.  <span class="journal">J. Gen. Microbiol. 94:246–256.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Growth%20on%20D-arabitol%20of%20a%20mutant%20strain%20of%20Escherichia%20coli%20K12%20using%20a%20novel%20dehydrogenase%20and%20enzymes%20related%20to%20L-1,2-propanediol%20and%20D-xylose%20metabolism">google scholar</a></td><td><a href="pdf/849.pdf">pdf</a></td></tr>
<tr><td>850</td><td><a id="ref850" class="anchor"></a>Xu, S., and T. Qi. <i>1990</i>. Enhancing effect of chuangxinmycin on synthesis of enzymes in tryptophan synthesis pathway of chuangxinmycin resistant mutant strain of <i>E. coli.</i>  <span class="journal">Zhongguo Yixue Kexueyuan Xuebao 12:25–30.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Enhancing%20effect%20of%20chuangxinmycin%20on%20synthesis%20of%20enzymes%20in%20tryptophan%20synthesis%20pathway%20of%20chuangxinmycin%20resistant%20mutant%20strain%20of%20E.%20coli.">google scholar</a></td><td></td></tr>
<tr><td>851</td><td><a id="ref851" class="anchor"></a>Yadav, N., R. E. McDevitt, S. Benard, and S. C. Falco. <i>1986</i>. Single amino acid substitutions in the enzyme acetolactate synthase confer resistance to the herbicide sulfometuron methyl.  <span class="journal">Proc. Natl. Acad. Sci. USA 83:4418–4422.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Single%20amino%20acid%20substitutions%20in%20the%20enzyme%20acetolactate%20synthase%20confer%20resistance%20to%20the%20herbicide%20sulfometuron%20methyl">google scholar</a></td><td><a href="pdf/851.pdf">pdf</a></td></tr>
<tr><td>852</td><td><a id="ref852" class="anchor"></a>Yagil, E., and H. Be’eri. <i>1977</i>. Arsenate-resistant alkaline phosphatase-constitutive mutants of <i>Escherichia coli.</i>  <span class="journal">Mol. Gen. Genet. 154:185–189.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Arsenate-resistant%20alkaline%20phosphatase-constitutive%20mutants%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/852.pdf">pdf</a></td></tr>
<tr><td>853</td><td><a id="ref853" class="anchor"></a>Yamada, T., and J. Davies. <i>1971</i>. A genetic and biochemical study of streptomycin- and spectinomycin-resistance in <i>Salmonella typhimurium.</i>  <span class="journal">Mol. Gen. Genet. 110:197–210.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20genetic%20and%20biochemical%20study%20of%20streptomycin-%20and%20spectinomycin-resistance%20in%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/853.pdf">pdf</a></td></tr>
<tr><td>854</td><td><a id="ref854" class="anchor"></a>Yamagata, H., C. Ippolito, M. Inukai, and M. Inouye. <i>1982</i>. Temperature-sensitive processing of outer membrane lipoprotein in an <i>Escherichia coli</i> mutant.  <span class="journal">J. Bacteriol. 152:1163–1168.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Temperature-sensitive%20processing%20of%20outer%20membrane%20lipoprotein%20in%20an%20Escherichia%20coli%20mutant">google scholar</a></td><td><a href="pdf/854.pdf">pdf</a></td></tr>
<tr><td>855</td><td><a id="ref855" class="anchor"></a>Yamagishi, J., H. Yoshida, M. Yamayoshi, and S. Nakamura. <i>1986</i>. Nalidixic acid resistant mutations of the <i>gyrB</i> gene of <i>Escherichia coli.</i>  <span class="journal">Mol. Gen. Genet. 204:367–373.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Nalidixic%20acid%20resistant%20mutations%20of%20the%20gyrB%20gene%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/855.pdf">pdf</a></td></tr>
<tr><td>856</td><td><a id="ref856" class="anchor"></a>Yamamoto, T., and Y. Fujiwara. <i>1990</i>. Uracil DNA glycosylase causes 5-bromodeoxyuridine photosensitization in <i>Escherichia coli</i> K-12.  <span class="journal">J. Bacteriol. 172:5278–5285.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Uracil%20DNA%20glycosylase%20causes%205-bromodeoxyuridine%20photosensitization%20in%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/856.pdf">pdf</a></td></tr>
<tr><td>857</td><td><a id="ref857" class="anchor"></a>Yamasaki, M., R. Aono, and G. Tamura. <i>1976</i>. FL-1060 binding protein of <i>Escherichia coli</i> is probably under the control of cyclic AMP.  <span class="journal">Agric. Biol. Chem. 40:1665–1667.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=FL-1060%20binding%20protein%20of%20Escherichia%20coli%20is%20probably%20under%20the%20control%20of%20cyclic%20AMP">google scholar</a></td><td><a href="pdf/857.pdf">pdf</a></td></tr>
<tr><td>858</td><td><a id="ref858" class="anchor"></a>Yanagisawa, T., J. T. Lee, H. C. Wu, and M. Kawakam. <i>1994</i>. Relationship of protein structure of isoleucyl-tRNA synthetase with pseudomonic acid resistance of <i>Escherichia coli:</i> a proposed mode of action of pseudomonic acid as an inhibitor of isoleucyl-tRNA synthetase.  <span class="journal">J. Biol. Chem. 269:24304–24309.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Relationship%20of%20protein%20structure%20of%20isoleucyl-tRNA%20synthetase%20with%20pseudomonic%20acid%20resistance%20of%20Escherichia%20coli:%20a%20proposed%20mode%20of%20action%20of%20pseudomonic%20acid%20as%20an%20inhibitor%20of%20isoleucyl-tRNA%20synthetase">google scholar</a></td><td><a href="pdf/858.pdf">pdf</a></td></tr>
<tr><td>859</td><td><a id="ref859" class="anchor"></a>Yarmolinsky, M. B., H. Wiesmeyer, H. M. Kalckar, and E. Jordan. <i>1959</i>. Hereditary defects in galactose metabolism in <i>Escherichia coli</i> mutants. II. Galactose-induced sensitivity.  <span class="journal">Proc. Natl. Acad. Sci. USA 45:1786–1791.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Hereditary%20defects%20in%20galactose%20metabolism%20in%20Escherichia%20coli%20mutants.%20II.%20Galactose-induced%20sensitivity">google scholar</a></td><td><a href="pdf/859.pdf">pdf</a></td></tr>
<tr><td>860</td><td><a id="ref860" class="anchor"></a>Yoshida, H., M. Bogaki, M. Nakamura, and S. Nakamura. <i>1990</i>. Quinolone resistance- determining region in the DNA gyrase <i>gyrA</i> gene of <i>Escherichia coli.</i>  <span class="journal">Antimicrob. Agents Chemother. 34:1271–1272.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Quinolone%20resistance-%20determining%20region%20in%20the%20DNA%20gyrase%20gyrA%20gene%20of%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/860.pdf">pdf</a></td></tr>
<tr><td>861</td><td><a id="ref861" class="anchor"></a>Yoshikawa, M., A. Okuyuma, and N. Tanaka. <i>1975</i>. A third kasugamycin resistance locus, <i>ksgC,</i> affecting ribosomal protein S2 in <i>Escherichia coli</i> K-12.  <span class="journal">J. Bacteriol. 122:796–797.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20third%20kasugamycin%20resistance%20locus,%20ksgC,%20affecting%20ribosomal%20protein%20S2%20in%20Escherichia%20coli%20K-12">google scholar</a></td><td><a href="pdf/861.pdf">pdf</a></td></tr>
<tr><td>862</td><td><a id="ref862" class="anchor"></a>Yudkin, M. D. <i>1977</i>. Unstable mutations that relieve catabolite repression of tryptophanase synthesis by <i>Escherichia coli.</i>  <span class="journal">J. Bacteriol. 130:57–61.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Unstable%20mutations%20that%20relieve%20catabolite%20repression%20of%20tryptophanase%20synthesis%20by%20Escherichia%20coli.">google scholar</a></td><td><a href="pdf/862.pdf">pdf</a></td></tr>
<tr><td>863</td><td><a id="ref863" class="anchor"></a>Yura, T., and K. Igarishi. <i>1968</i>. RNA polymerase mutants of <i>Escherichia coli.</i> I. Mutants resistant to streptovaricin.  <span class="journal">Proc. Natl. Acad. Sci. USA 61:1313–1319.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=RNA%20polymerase%20mutants%20of%20Escherichia%20coli.%20I.%20Mutants%20resistant%20to%20streptovaricin">google scholar</a></td><td><a href="pdf/863.pdf">pdf</a></td></tr>
<tr><td>864</td><td><a id="ref864" class="anchor"></a>Yura, T., and C. Wada. <i>1968</i>. Phenethyl alcohol resistance in <i>Escherichia coli.</i> I. Resistance of strain C600 and its relation to azide resistance.  <span class="journal">Genetics 59:177–190.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Phenethyl%20alcohol%20resistance%20in%20Escherichia%20coli.%20I.%20Resistance%20of%20strain%20C600%20and%20its%20relation%20to%20azide%20resistance">google scholar</a></td><td><a href="pdf/864.pdf">pdf</a></td></tr>
<tr><td>865</td><td><a id="ref865" class="anchor"></a>Zak, V. L., and R. A. Kelln. <i>1978</i>. 5-Fluoroorotate-resistant mutants of <i>Salmonella typhimurium.</i>  <span class="journal">Can. J. Microbiol. 24:1339–1345.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=5-Fluoroorotate-resistant%20mutants%20of%20Salmonella%20typhimurium.">google scholar</a></td><td><a href="pdf/865.pdf">pdf</a></td></tr>
<tr><td>866</td><td><a id="ref866" class="anchor"></a>Zak, V. L., and R. A. Kelln. <i>1981</i>. A <i>Salmonella typhimurium</i> mutant dependent upon carbamyl aspartate for resistance to 5-fluorouracil is specifically altered in ubiquinone biosynthesis.  <span class="journal">J. Bacteriol. 145:1095–1098.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A%20Salmonella%20typhimurium%20mutant%20dependent%20upon%20carbamyl%20aspartate%20for%20resistance%20to%205-fluorouracil%20is%20specifically%20altered%20in%20ubiquinone%20biosynthesis">google scholar</a></td><td><a href="pdf/866.pdf">pdf</a></td></tr>
<tr><td>867</td><td><a id="ref867" class="anchor"></a>Zissler, J., E. Signer, and F. Schaefer. <i>1971</i>. The role of recombination in growth of bacteriophage lambda. I. The gamma gene. <span class="journal">The Bacteriophage Lambda. Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The%20role%20of%20recombination%20in%20growth%20of%20bacteriophage%20lambda.%20I.%20The%20gamma%20gene">google scholar</a></td><td><a href="pdf/867.pdf">pdf</a></td></tr>
<tr><td>868</td><td><a id="ref868" class="anchor"></a>Zwiebel, L. J., M. Inukai, K. Nakamura, and M. Inouye. <i>1981</i>. Preferential selection of deletion mutations of the outer membrane lipoprotein gene of <i>Escherichia coli</i> by globomycin.  <span class="journal">J. Bacteriol. 145:654–656. </span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Preferential%20selection%20of%20deletion%20mutations%20of%20the%20outer%20membrane%20lipoprotein%20gene%20of%20Escherichia%20coli%20by%20globomycin">google scholar</a></td><td><a href="pdf/868.pdf">pdf</a></td></tr>
<tr><td>869</td><td><a id="ref869" class="anchor"></a>Comai, L.. <i>1985</i>. Inhibition resistant 5-enolpyruvyl-3-phosphoshikimate synthetase, production and use. <span class="journal">US Patent US4535060.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Inhibition%20resistant%205-enolpyruvyl-3-phosphoshikimate%20synthetase,%20production%20and%20use">google scholar</a></td><td><a href="pdf/869.pdf">pdf</a></td></tr>
<tr><td>870</td><td><a id="ref870" class="anchor"></a>Lin, S. C. L., D. T. Palmer, and H. I. Miller. <i>1986</i>. Phage resistant E. Coli bacterium and method for making the same. <span class="journal">European Patent EP0183469A1.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Phage%20resistant%20E.%20Coli%20bacterium%20and%20method%20for%20making%20the%20same">google scholar</a></td><td><a href="pdf/870.pdf">pdf</a></td></tr>
<tr><td>871</td><td><a id="ref871" class="anchor"></a>Miller, E. S.. <i>1984</i>. PhD Thesis. <span class="journal">Purdue University.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=PhD%20Thesis">google scholar</a></td><td></td></tr>
<tr><td>872</td><td><a id="ref872" class="anchor"></a>Epelbaum, S., Z. Barak, R. Larossa, and D. Chipman. <i>1994</i>. Unpublished data. <span class="journal"></span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Unpublished%20data">google scholar</a></td><td></td></tr>
<tr><td>873</td><td><a id="ref873" class="anchor"></a>Freitag, C. S.. <i>1982</i>. The application of streptozotocin resistance and transposon mutagenesis in the selection of pts mutations in Salmonella Typhimurium (PhD Thesis). <span class="journal">North Carolina State University.</span></td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The%20application%20of%20streptozotocin%20resistance%20and%20transposon%20mutagenesis%20in%20the%20selection%20of%20pts%20mutations%20in%20Salmonella%20Typhimurium%20(PhD%20Thesis)">google scholar</a></td><td></td></tr>
</table></div>

<div id="contribute">
<h1><span class="anchor" id="how-to-contribute"></span>Contribute to free <em>E. coli</em> (and <em>Salmonella</em>) knowledge!</h1>

<p>
These data were automatically extracted and parsed from the pdf, using a few bash and python scripts. 
Many mistakes may remain, either introduced by our parsing or present in the original book. Moreover finding a pdf for each reference requires manual work in many cases. You are very welcome to contribute! <br> 
</p>

<p>
Start by cloning the git repository from <a href="https://github.com/afrenoy/EcoliSelectableGenes">https://github.com/afrenoy/EcoliSelectableGenes</a>. You can then easily:
</p>

<ul>
  <li>Fix a mistake in one of the tables: just edit the corresponding csv file (<code>table1.csv</code>, <code>table2.csv</code> or <code>references.csv</code>), and the webpage will be updated when running <code>./generatewebpage.py</code></li>
  <li>Add a missing pdf: just add it in the <code>pdf/</code> directory, naming it <code>refnumber.pdf</code> (eg <code>84a.pdf</code> or <code>26.pdf</code>), and a link will automatically be generated when running <code>./generatewebpage.py</code></li>
  <li>Add a missing link to <a href="https://ecocyc.org/">EcoCyc</a>: add a new line to <code>ecocyc.csv</code> linking the name used in our tables with and an EcoCyc ID, and a link will automatically be generated when running <code>./generatewebpage.py</code></li>
</ul>

<p>
After doing any of these changes, you can just run <code>./generatewebpage.py</code> to update the main file <code>index.html</code>. If you are satisfied with the outcome, send a patch / a pull request. <br>
Other contributions and suggestions of improvement are also very welcome!
</p>
</div>


</div>
<footer>
<hr>
<p>C'est amusant, pensa le petit prince. C'est assez poétique. Mais ce n'est pas très sérieux.</p>
<p>Le petit prince avait sur les choses sérieuses des idées très différentes des idées des grandes personnes.</p>
<p>- Moi, dit-il encore, je possède une fleur que j'arrose tous les jours. Je possède trois volcans que je ramone toutes les semaines. Car je ramone aussi celui qui est éteint. On ne sait jamais. C'est utile à mes volcans, et c'est utile à ma fleur, que je les possède. Mais tu n'es pas utile aux étoiles.</p>
<p><em><a href="http://www.saintexupery-domainepublic.be/">Le Petit Prince, Antoine de Saint-Exupéry</a></em></p>

</footer>

</body>
</html>
