<!DOCTYPE html>
<html>
<head>
	<meta charset="UTF-8">
	<title>Larossa</title>
	<link rel="stylesheet" href="style.css">
</head>
<body>

<div id="menu"><nav><ul><li><a href="#about">About</a></li><li><a href="#tab1">Selections giving rise to mutants</a></li><li><a href="#tab2">Genes for which selections exist</a></li><li><a href="#ref">References</a></li></ul></nav></div>
<div id="main">
<h1 id="about">About this document</h1><p>This is data from chapter 139 of E coli and Salmonella, reproduced without permission. </br>The pdf has been converted to html and parsed using a few bash and python scripts availabe on <a href="http://github.com/frenoy/ecosal">github</a>.</p><p>The raw data can be downloaded as csv files:</p><ul><li><a href="table1-4.csv">Table 1</a>: Selections giving rise to mutants</li><li><a href="table2-4.csv">Table 2</a>: Genes for which selections exist</li><li><a href="references-4.csv">Table 3</a>: References</li></ul>
<h1><a name="tab1"></a><strong>TABLE 1</strong> Selections giving rise to mutants of <em>E.coli</em> and <em>S.typhimurium</em></h1>
<table>
<tr>
<td>
Type
</td>
<td>
Selection purpose and technique
</td>
<td>
Mutant gene
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Acetate + gluconate as C source in <em>ppc</em>
</td>
<td>
gntR
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Acetate + gluconate as C source in <em>ppc</em>
</td>
<td>
gntS
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Acetyl(<em>N</em>)histidine + ornithine satisfaction of <em>his</em>
</td>
<td>
argB
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Acetyl(<em>N</em>)histidine + ornithine satisfaction of <em>his</em>
</td>
<td>
argC
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Acetyl(<em>N</em>)histidine + ornithine satisfaction of <em>his</em>
</td>
<td>
argE
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Acetyl(<em>N</em>)histidine + ornithine satisfaction of <em>his</em>
</td>
<td>
argH
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Acetylhistidine satisfaction of <em>his</em>
</td>
<td>
argR
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Acetyl(<em>N</em>)lactonate as sole N source 
</td>
<td>
<em>lacI,</em> operator
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Acetylmethionine satisfaction of <em>met</em>
</td>
<td>
argR
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Acetylornithine + uracil satisfaction of <em>car</em> (<em>pyrA</em>)
</td>
<td>
argA
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Acetylornithine + uracil satisfaction of <em>car</em> (<em>pyrA</em>)
</td>
<td>
argB
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Acetylornithine + uracil satisfaction of <em>car</em> (<em>pyrA</em>)
</td>
<td>
argC
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Acetylornithine + uracil satisfaction of <em>car</em> (<em>pyrA</em>)
</td>
<td>
argD
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Adenosine as C source in <em>upp deoD</em>
</td>
<td>
xapR
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Aminobutyrate as N source
</td>
<td>
gabC
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Arabinitol (D) resistance
</td>
<td>
mtlA
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Arabinose (L) as C source in <em>cya</em> or <em>crp</em>
</td>
<td>
rpoD
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Arabinose (L) as C source in presence of <em>lac</em> operon inducer
</td>
<td>
lacY
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Arabinose (D) as C source
</td>
<td>
fucA
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Arabitol as C source
</td>
<td>
gatACD
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Arabitol (D) as C source in <em>fuc</em>
</td>
<td>
dgd
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Arbutin as a C source
</td>
<td>
bglBC
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Arbutin as C source)
</td>
<td>
<em>osmZ</em> (<em>bglY)</em>
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Arginine as N source
</td>
<td>
hisP
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Arginine-independent growth of <em>pyrH</em> in presence of uracil
</td>
<td>
pyrB
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Arginine-independent growth of <em>pyrH</em> in presence of uracil
</td>
<td>
pyrC
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Arginine-independent growth of <em>pyrH</em> in presence of uracil
</td>
<td>
pyrD
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Arginine-resistant, proline-independent growth of ∆<em>pro</em>
</td>
<td>
argR
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
β-Glycerol-phosphate as C source in presence of high phosphate
</td>
<td>
phoA
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
β-Glycerol-phosphate as C source in presence of high phosphate
</td>
<td>
phoR
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
β-Glycerol-phosphate as C source in presence of high phosphate
</td>
<td>
phoS
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
β-Glycerol-phosphate as C source in presence of high phosphate
</td>
<td>
phoT
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Biotin sulfoxide utilization in <em>chl bio</em>
</td>
<td>
chlE
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Branched-chain amino acid satisfaction of ∆<em>ilvDC hisT</em>
</td>
<td>
ilvA
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Butyrate as C source in <em>fadR</em>(Con)
</td>
<td>
atoC
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Butyrate or valerate as better C source
</td>
<td>
iclR
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Cellobiose as C source
</td>
<td>
celABCDF
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Chemostat growth improved
</td>
<td>
mut
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Citrate as C source
</td>
<td>
cit
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Citrate as C source in presence of low cAMP
</td>
<td>
cpd
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
D-Amino acid satisfaction of amino acid auxotrophy
</td>
<td>
dadA
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
DAP-independent growth of ∆(<em>mal-asd</em>) in the presence of serine, methionine,and glycine D-Histidine + glycylglutamine satisfaction of <em>his</em> auxotrophs
</td>
<td>
glnA
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Decanoate as C source
</td>
<td>
fadR
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Deoxyadenosine (low) satisfaction of purine requirement
</td>
<td>
add
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Deoxyadenosine + hypoxanthine satisfaction of purine requirement
</td>
<td>
add
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Diamino (2,6)purine satisfaction of purine auxotrophy
</td>
<td>
deoR
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Erythromycin growth dependence
</td>
<td>
mac
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Ethanol as C source
</td>
<td>
adh
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Ethyleneglycol as C source in propanediol utilizer
</td>
<td>
fucA
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Fructose as C source in <em>ptsF</em> or <em>ptsM</em>
</td>
<td>
srlD
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Fructose 1-phosphate as C source
</td>
<td>
uhpR
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Fucose (L) as C source in propanediol utilizer
</td>
<td>
fucA
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Fucose + arabinose-supported growth in <em>galP</em>
</td>
<td>
mglA
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Fucose + arabinose-supported growth in <em>galP</em>
</td>
<td>
mglB
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Fucose + arabinose-supported growth in <em>galP</em>
</td>
<td>
mglC
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Galactose as C source for <em>gal</em> (leader)::IS
</td>
<td>
rho
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Galacturonide as C source
</td>
<td>
uidA
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Galacturonide as C source
</td>
<td>
uidR
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Glucanate as C source in <em>eda</em>
</td>
<td>
edd
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Glucosamine as C source
</td>
<td>
ptsG
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Glucosamine 6-phosphate as C source
</td>
<td>
uhpR
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Glucosamine as anaerobic C source in <em>ptsG</em>
</td>
<td>
dgsA
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Glucose 1-phosphate as C source
</td>
<td>
uhpR
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Glucose as C source in ∆<em>ptsHI</em>
</td>
<td>
galC
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Glucose as C source in <em>ppc</em>
</td>
<td>
iclR
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Glucose as C source in ∆<em>ptsHI</em>
</td>
<td>
galR
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Glucosides (β) as C source
</td>
<td>
<em>gyrA</em> (<em>hisU</em>)
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Glucosides (β) as C source
</td>
<td>
<em>gyrB</em> (<em>hisW)</em>
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Glutamate as C source
</td>
<td>
gadS
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Glutamate as C source
</td>
<td>
gltH
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Glutamate as C source
</td>
<td>
gltS
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Glutamate as C source at 42°C
</td>
<td>
gltR
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Glutamine as C source
</td>
<td>
glnP
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Glycerol as C source in <em>ppc</em>
</td>
<td>
iclR
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Glycerol + gluconate as C source in <em>eda</em>
</td>
<td>
gntM
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Glycerol + methylglucuronide as C source in <em>eda</em>
</td>
<td>
gurBCD
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Glycerol 3-phosphate as C source in <em>glpT ugpAB</em>
</td>
<td>
Hemin satisfaction of <em>hemA</em> auxotroph
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Hemin satisfaction of <em>hemA</em> auxotroph
</td>
<td>
rfa
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Hexuronate as C source in noninducible strain
</td>
<td>
exuR
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Histidinal satisfaction of <em>his</em> auxotroph requirement at 30°C
</td>
<td>
<em>his</em> structural genes
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Histidinal satisfaction of <em>his</em> auxotroph requirement at 30°C
</td>
<td>
hisG
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Histidine (D) satisfaction of <em>his</em> auxotroph requirement
</td>
<td>
<em>his</em> structural genes
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Histidine (D) satisfaction of <em>his</em> auxotroph requirement
</td>
<td>
hisG
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Histidine (D) satisfaction of <em>his</em> auxotroph requirement
</td>
<td>
dhuA
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Histidine as C source
</td>
<td>
hut
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Histidine as N source
</td>
<td>
hut
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Improved growth in <em>icd</em>
</td>
<td>
gltA
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Inosine as C source in <em>upp deoD</em>
</td>
<td>
xapR
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Inosine as improved C source
</td>
<td>
deoR
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Keto(2)-3-deoxygalactonate as C source
</td>
<td>
kdgR
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Keto(2)-3-deoxygalactonate as C source
</td>
<td>
dgoR
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Lactate + fumarate as anaerobic energy source
</td>
<td>
chlABDE
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Lactate + nitrate as anaerobic energy source (<em>chl</em> to <em>chl</em>+)
</td>
<td>
chlE
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Lactitol as C source
</td>
<td>
<em>lacI,</em> operator
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Lactobionic acid as C source
</td>
<td>
<em>lacI,</em> operator
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Lactobionic acid as C source
</td>
<td>
lacZ
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Lactose utilization after conjugation of F′<em>lac</em> into <em>Salmonella</em> sp.
</td>
<td>
hsd
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Lactose utilization in ∆<em>lacZ</em>
</td>
<td>
ebgA
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Leucine (D) satisfaction of <em>leu</em> auxotrophs
</td>
<td>
<em>lrp</em> (<em>livR, lstR, lss, mbl, oppl)</em>
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Lyxose (D) as C source
</td>
<td>
manC
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Malate (L) as improved C source
</td>
<td>
dct
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Malate (D) as C source
</td>
<td>
dml
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Maltose as C source in <em>malT</em>
</td>
<td>
malP
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Maltose as C source in <em>malT</em>
</td>
<td>
malQ
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Maltose utilization in <em>malT</em>
</td>
<td>
bymA
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Mannitol (limiting) as C source
</td>
<td>
mtlC
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Mannose as C source
</td>
<td>
ptdG
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Melibiose as C source
</td>
<td>
<em>lacI,</em> operator
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Melibiose as C source and acetyl(<em>N</em>)lactonate as N source
</td>
<td>
<em>lacI,</em> operator
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Methyl-β-galactoside-supported growth
</td>
<td>
mglD
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Methylgalacturonide as C source
</td>
<td>
uidA
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Methylgalacturonide as C source
</td>
<td>
uidR
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Methylgalacturonide as C source
</td>
<td>
uxuR
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Methyl(β)glucoside as C source
</td>
<td>
bglT
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Methylglucuronide + glycerol-supported growth
</td>
<td>
uidA
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Neolactose as C source
</td>
<td>
<em>lacI,</em> operator
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Nicotinamide as sole N source
</td>
<td>
pncA
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Phenylgalactoside as C source
</td>
<td>
<em>lacI,</em> operator
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Proline as sole N source with glucose as C source
</td>
<td>
putA
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Proline-independent growth of <em>proAB</em>
</td>
<td>
argD
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Propanediol as C source
</td>
<td>
fucA
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Propionate as C source
</td>
<td>
prp
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Raffinose as C source in <em>lacI</em>
</td>
<td>
<em>mel</em> (generic)
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Raffinose as C source
</td>
<td>
<em>lacI,</em> operator
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Ribitol utilization
</td>
<td>
gatACD
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Salicin as C source
</td>
<td>
<em>bglB,C</em>
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Salicin as C source
</td>
<td>
osmZ
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Serine (L) as C source
</td>
<td>
cpxA
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Serine (L) as C source
</td>
<td>
ecfB
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Sorbose (L) as C source in crosses with wild strains
</td>
<td>
sorAT
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Streptomycin independence of <em>rpsL</em>
</td>
<td>
rpsD
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Streptomycin independence of <em>rpsL</em>
</td>
<td>
rpsE
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Succinate as improved C source
</td>
<td>
cpd
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Succinate-independent growth in <em>lpd</em>
</td>
<td>
sdh
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Sucrose as C source
</td>
<td>
dsdA
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Sucrose as C source
</td>
<td>
dsdC
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Threonine as N source
</td>
<td>
ilvA
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Tyramine as N source
</td>
<td>
tyn
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Uridine as C source, improved utilization
</td>
<td>
cytR
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Valerate as C source in <em>fadR</em>(Con)
</td>
<td>
atoC
</td>
</tr>
<tr>
<td>
Expansion of metabolic capacity
</td>
<td>
Xylitol as C source in propanediol utilizer
</td>
<td>
fucA
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Acid resistance
</td>
<td>
atr
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Acid resistance
</td>
<td>
phoE
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Arsenate resistance
</td>
<td>
glpT
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Arsenate resistance
</td>
<td>
phoR
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Arsenate resistance
</td>
<td>
phoS
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Arsenate resistance
</td>
<td>
phoT
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Arsenate resistance
</td>
<td>
pstABCS
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Arsenate resistance
</td>
<td>
pit
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Azide resistance
</td>
<td>
<em>atpA–atpE</em>
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Azide resistance
</td>
<td>
cysB
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Azide resistance
</td>
<td>
cysK
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Azide resistance
</td>
<td>
secA
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Chlorate resistance
</td>
<td>
chlABCDEG
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Chlorate resistance
</td>
<td>
narCG
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Chlorate resistance
</td>
<td>
narH
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Chromate + selenate resistance
</td>
<td>
cysL
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Chromate resistance
</td>
<td>
cysA
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Chromate resistance
</td>
<td>
cysB
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Chromate resistance
</td>
<td>
cysC
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Chromate resistance
</td>
<td>
cysD
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Chromate resistance
</td>
<td>
cysH
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Chromate resistance
</td>
<td>
cysI
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Chromate resistance
</td>
<td>
cysJ
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Copper resistance in <em>ompC</em>
</td>
<td>
ompF
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Copper resistance
</td>
<td>
ompR
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Diazoborine resistance
</td>
<td>
envM
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Lithium resistance (multicopy)
</td>
<td>
nhaA
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Lithium-resistant utilization of melibiose
</td>
<td>
melB
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Manganese resistance
</td>
<td>
corABCD
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Manganese resistance
</td>
<td>
fur
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Manganese resistance
</td>
<td>
mng
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Nickel resistance
</td>
<td>
corABCD
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Osmotolerance
</td>
<td>
crp
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Osmotolerance
</td>
<td>
cyaA
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Osmotolerance
</td>
<td>
osmB
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Osmotolerance
</td>
<td>
proA
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Osmotolerance
</td>
<td>
proB
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Peroxide resistance
</td>
<td>
oxyR
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Selenate resistance
</td>
<td>
cysA
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Selenate resistance
</td>
<td>
cysB
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Selenate resistance in <em>cysM</em>
</td>
<td>
cysB
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Selenate resistance
</td>
<td>
cysK
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Selenate resistance
</td>
<td>
cysL
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Selenite resistance
</td>
<td>
cysK
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Selenite resistance
</td>
<td>
gshA
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Selenite resistance
</td>
<td>
gshB
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Tellurite resistance
</td>
<td>
phoB
</td>
</tr>
<tr>
<td>
Resistance to inorganic chemicals
</td>
<td>
Tributyl tin resistance
</td>
<td>
<em>atpA–atpE</em>
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Acetylnorvaline resistance in <em>argR</em>(Con)
</td>
<td>
argE
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Acetylnorvaline resistance in <em>argR</em>(Con)
</td>
<td>
argR
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Acridine resistance
</td>
<td>
dnaE
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Actinomycin D resistance in the presence of EDTA
</td>
<td>
pldA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Adenine resistance in <em>hpt gpt</em>
</td>
<td>
apt
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Adenine resistance in <em>hpt gpt</em>
</td>
<td>
purR
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Alafosfalin resistance
</td>
<td>
pepA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Alafosfalin resistance
</td>
<td>
tpp
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Alanyl-2-aminopropionate resistance
</td>
<td>
oppA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Albicidin resistance
</td>
<td>
tsx
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Albomycin resistance
</td>
<td>
exbB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Albomycin resistance
</td>
<td>
fhuA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Albomycin resistance
</td>
<td>
fhuB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Albomycin resistance
</td>
<td>
pepN
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Albomycin resistance
</td>
<td>
sidCF
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Albomycin resistance
</td>
<td>
sidK
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Albomycin resistance
</td>
<td>
tonB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Aldohexuronate resistance in <em>eda</em>
</td>
<td>
exuT
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Aldohexuronate resistance in <em>eda</em>
</td>
<td>
exuR
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Allyl alcohol resistance
</td>
<td>
adhE
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Amidinopenicillin tolerance
</td>
<td>
sloB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Amikacin resistance
</td>
<td>
cpxA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Amino(2)purine resistance in <em>dam</em>
</td>
<td>
mutH
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Amino(2)purine resistance in <em>dam</em>
</td>
<td>
mutL
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Amino(2)purine resistance in <em>dam</em>
</td>
<td>
mutS
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Amino(4)phenylalanine resistance
</td>
<td>
aroF
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Amino(4)phenylalanine resistance
</td>
<td>
tyrA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Amino(4)phenylalanine resistance
</td>
<td>
tyrR
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Amino(6)nicotinamide resistance
</td>
<td>
nadA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Amino(6)nicotinamide resistance
</td>
<td>
nadD
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Amino(6)nicotinamide resistance
</td>
<td>
pncA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Amino(6)nicotinamide resistance
</td>
<td>
pncB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Amino(6)nicotinamide resistance
</td>
<td>
pncX
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Amino(6)nicotinate resistance
</td>
<td>
pncB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Aminobutyrate resistance
</td>
<td>
<em>ilv</em> structural genes
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Aminobutyrate resistance
</td>
<td>
ilvB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Aminobutyrate resistance
</td>
<td>
ilvH
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Aminobutyrate resistance
</td>
<td>
ilvI
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Aminoethylcysteine (thialysine) resistance
</td>
<td>
lysC
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Aminoethylcysteine (thialysine) resistance
</td>
<td>
lysP
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Aminoethylcysteine (thialysine) resistance
</td>
<td>
lysS
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Aminoglycoside resistance
</td>
<td>
nek
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Amino(2)hydroxy(3)pentoate resistance
</td>
<td>
<em>thr</em> structural genes
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Aminopterin resistance
</td>
<td>
thyA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Aminotriazole resistance
</td>
<td>
<em>amtA, amtB</em>
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Aminotriazole resistance
</td>
<td>
<em>his</em> structural genes
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Aminotriazole resistance in <em>relA</em>
</td>
<td>
gyrB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Ampicillin resistance
</td>
<td>
ampC
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Ampicillin resistance
</td>
<td>
ampD
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Ampicillin resistance
</td>
<td>
ampE
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Ampicillin resistance
</td>
<td>
envZ
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Ampicillin resistance
</td>
<td>
galU
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Ampicillin resistance
</td>
<td>
hipA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Ampicillin resistance
</td>
<td>
hipQ
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Ampicillin resistance
</td>
<td>
ompC
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Ampicillin resistance
</td>
<td>
ompF
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Ampicillin resistance
</td>
<td>
ompR
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Ampicillin resistance
</td>
<td>
ptsI
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Ampicillin resistance
</td>
<td>
<em>rfa</em> (general)
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Ampicillin resistance
</td>
<td>
tolD
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Ampicillin resistance
</td>
<td>
tolE
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Arabinitol (D) resistance
</td>
<td>
mtlA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Arbitol (D) resistance
</td>
<td>
gatACD
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Arbutin resistance in <em>dgk</em>
</td>
<td>
mdoB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Arginine-resistant, proline-independent growth of ∆<em>pro</em>
</td>
<td>
argR
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Auroventin resistance
</td>
<td>
<em>atpA–atpE</em>
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Aza(5)cytidine resistance
</td>
<td>
dcm
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Azadeoxycytidine resistance
</td>
<td>
cdd
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Aza(6)uracil resistance
</td>
<td>
gpt
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Aza(8)guanine resistance
</td>
<td>
ndk
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Aza(8)guanine resistance
</td>
<td>
upp
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Azaleucine resistance
</td>
<td>
aroP
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Azaleucine resistance
</td>
<td>
azl
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Azaleucine resistance
</td>
<td>
tolB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Azaleucine resistance
</td>
<td>
leuS
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Azaleucine resistance
</td>
<td>
livG
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Azaleucine resistance
</td>
<td>
livH
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Azaserine + tryptophan resistance
</td>
<td>
dhuA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Azaserine + tryptophan resistance in <em>dhuA</em>
</td>
<td>
hisJ
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Azaserine resistance
</td>
<td>
aroP
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Azaserine resistance
</td>
<td>
azaAB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Azaserine resistance
</td>
<td>
cysA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Azaserine resistance
</td>
<td>
cysB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Azaserine resistance
</td>
<td>
cysC
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Azaserine resistance
</td>
<td>
cysD
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Azaserine resistance
</td>
<td>
cysE
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Azaserine resistance
</td>
<td>
cysG
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Azaserine resistance
</td>
<td>
cysH
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Azaserine resistance
</td>
<td>
cysI
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Azaserine resistance
</td>
<td>
cysJ
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Azaserine resistance
</td>
<td>
cysK
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Azaserine resistance
</td>
<td>
cysM
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Azaserine resistance
</td>
<td>
<em>mut</em> (generic)
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Azetidine carboxylate resistance
</td>
<td>
proB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Azetidine carboxylate resistance
</td>
<td>
putA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Azetidine carboxylate resistance
</td>
<td>
putP
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Azidothymidine resistance
</td>
<td>
tdk
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Bacilysin resistance
</td>
<td>
dppA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Bacitracin resistance
</td>
<td>
rfa
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Bacteriocin resistance
</td>
<td>
tol
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Bacteriocin 4-59 resistance
</td>
<td>
ompA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Bacteriocin 4-59 resistance
</td>
<td>
tonB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Bacteriocin JF246 resistance
</td>
<td>
ompA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Baikiain resistance in constitutive background
</td>
<td>
putA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
β-Glycerol-phosphate resistance in <em>glpD</em>
</td>
<td>
phoA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Beta-lactam conjugate resistance
</td>
<td>
fhuA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Beta-lactam resistance (see mecillinam resistance)
</td>
<td>
crp
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Beta-lactam resistance
</td>
<td>
cyaA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Beta-lactam resistance
</td>
<td>
lyt
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Beta-lactam resistance
</td>
<td>
alaS
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Beta-lactam resistance
</td>
<td>
argS
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Bialaphos resistance
</td>
<td>
dppA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Borrelidin resistance
</td>
<td>
thrS
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Bromodeoxyuridine + UV light resistance
</td>
<td>
ung
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Cadaverine resistance
</td>
<td>
cadB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Caffeine resistance
</td>
<td>
glnV
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Calmodulin inhibitor resistance
</td>
<td>
leuW
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
cAMP + glucose 6-phosphate + D-xylose + L-arabinose resistance
</td>
<td>
crp
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
cAMP + glucose 6-phosphate resistance
</td>
<td>
crp
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Camphor resistance
</td>
<td>
mbrABCD
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Canavanine + azauracil resistance
</td>
<td>
argR
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Canavanine + thiouracil resistance
</td>
<td>
upp
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Canavanine + thiouracil resistance
</td>
<td>
argP
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Canavanine resistance
</td>
<td>
argS
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Canavanine resistance
</td>
<td>
argP
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Canavanine resistance
</td>
<td>
argR
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
CCCP resistance
</td>
<td>
<em>atpA–atpE</em>
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
CCCP resistance due to gene dosage
</td>
<td>
emrB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Cephalosporin E-0702 resistance
</td>
<td>
tonB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Cephalothin resistance
</td>
<td>
rfa
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Chelator resistance
</td>
<td>
ompA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Chloramphenicol resistance
</td>
<td>
cmlA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Chloramphenicol resistance
</td>
<td>
marA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Chloramphenicol resistance
</td>
<td>
rrn
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Chloramphenicol resistance
</td>
<td>
ompF
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Chloramphenicol resistance
</td>
<td>
ompR
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Chloro(β)-D-alanine resistance
</td>
<td>
dadA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Chloro(β)-D-alanine resistance
</td>
<td>
<em>metC</em> (<em>ecfA</em>)
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Chloro-3-hydroxyacetone resistance in <em>uhp</em>(Con)
</td>
<td>
uhpT
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Chloroacetaldehyde resistance
</td>
<td>
adhCE
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Chloroacetate(β) resistance
</td>
<td>
cbt
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Chlorobiocin resistance
</td>
<td>
gyrB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Chloroethanol resistance
</td>
<td>
adhCE
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Chlorohydroxyacetone phosphate resistance
</td>
<td>
glpT
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Chlorpromazine resistance
</td>
<td>
lon
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Chuangxinmycin resistance
</td>
<td>
trpR
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Cloacin DF13 resistance
</td>
<td>
ompF
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Coenzyme A feedback resistance
</td>
<td>
coaA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Colicin “ K-type” tolerance
</td>
<td>
hemB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Colicin (multiple) resistance
</td>
<td>
envZ
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Colicin (other) resistance
</td>
<td>
exbB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Colicin (other) resistance
</td>
<td>
exbC
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Colicin A tolerance
</td>
<td>
cpxA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Colicin A tolerance
</td>
<td>
ompF
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Colicin A group tolerance
</td>
<td>
cirA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Colicin A group tolerance
</td>
<td>
tolQ
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Colicin B resistance
</td>
<td>
exbC
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Colicin B resistance
</td>
<td>
fepABCDG
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Colicin B resistance
</td>
<td>
tonB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Colicin B tolerance
</td>
<td>
cbt
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Colicin D resistance
</td>
<td>
cbt
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Colicin D tolerance
</td>
<td>
tolB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Colicin E resistance
</td>
<td>
tolC
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Colicin E1 tolerance
</td>
<td>
cet
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Colicin E1 tolerance
</td>
<td>
tolQAB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Colicin E2 tolerance
</td>
<td>
ompF
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Colicin E3 tolerance
</td>
<td>
btuB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Colicin E3 tolerance
</td>
<td>
tolR
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Colicin E2 and E3 tolerance
</td>
<td>
tolE
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Colicin E2 and E3 tolerance
</td>
<td>
tolZ
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Colicin E2, E3, D, Ia, and Ib tolerance
</td>
<td>
tolI
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Colicin I resistance
</td>
<td>
exbB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Colicin I resistance
</td>
<td>
exbC
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Colicin I resistance
</td>
<td>
ecfB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Colicin Ia and Ib tolerance
</td>
<td>
tolJ
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Colicin K resistance
</td>
<td>
<em>metC</em> (<em>ecfA</em>)
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Colicin K resistance
</td>
<td>
tsx
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Colicin K resistance
</td>
<td>
ompA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Colicin K tolerance
</td>
<td>
ompA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Colicin K tolerance
</td>
<td>
ompF
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Colicin L tolerance
</td>
<td>
envZ
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Colicin L tolerance
</td>
<td>
ompF
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Colicin L tolerance
</td>
<td>
fhuA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Colicin L, A, and S4 tolerance
</td>
<td>
fepABCDG
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Colicin M resistance
</td>
<td>
tonB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Colicin M resistance
</td>
<td>
tolM
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Colicin M tolerance
</td>
<td>
tolD
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Coumermycin resistance
</td>
<td>
gyrB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Coumermycin resistance
</td>
<td>
hisW
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Cyclopentane glycine resistance
</td>
<td>
<em>ilv</em> structural genes
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Cyclopentane glycine resistance
</td>
<td>
ilvA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Cycloserine resistance
</td>
<td>
hipA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Cycloserine (D) resistance
</td>
<td>
cycA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Dapsone resistance
</td>
<td>
thdA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Dehydrobiotin resistance
</td>
<td>
<em>bio </em>operon
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Dehydrobiotin resistance
</td>
<td>
bioP
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Dehydrobiotin resistance
</td>
<td>
birA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Dehydroproline + azetidine carboxylate resistance at high osmolarity in <em>putA</em><em>putP</em>
</td>
<td>
proU
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Dehydroproline resistance
</td>
<td>
proB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Dehydroproline resistance in <em>putA putP</em>
</td>
<td>
proP
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Deoxy(2)adenosine resistance in <em>deoC</em>
</td>
<td>
deoD
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Deoxyadenosine + fluorouracil resistance in <em>upp</em>
</td>
<td>
deoA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Deoxyadenosine + fluorouracil resistance in <em>upp</em>
</td>
<td>
deoD
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Deoxydihydroxyphosphonyl methyl fructose resistance
</td>
<td>
uhpT
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Deoxy(3)-3-fluoroglucose-independent utilization of lactate
</td>
<td>
ptsHIG
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Deoxy(3)-3-fluoroglucose-independent utilization of fructose
</td>
<td>
ptsG
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Deoxy(3)-3-fluoroglucose resistance
</td>
<td>
ptsI
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Deoxy(2)galactitol resistance in galicitol utilizer
</td>
<td>
gatACD
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Deoxy(2)galactose resistance
</td>
<td>
<em>gal</em> operon
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Deoxy(2)galactose resistance
</td>
<td>
galE
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Deoxy(2)galactose resistance
</td>
<td>
galK
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Deoxy(2)galactose resistance
</td>
<td>
galP
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Deoxy(2)glucose-resistant utilization of melibiose
</td>
<td>
melB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Deoxyglucose resistance
</td>
<td>
manXYZ
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Deoxy(2)glucose-6-phosphate resistance
</td>
<td>
uhpT
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Deoxy(2)glucose-resistant fructose utilization
</td>
<td>
fruA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Deoxy(2)glucose-resistant fructose utilization
</td>
<td>
ptsG
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Deoxy(2)-2-iodoacetamidoglucose resistance
</td>
<td>
nagE
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Diamino(2,6)purine resistance
</td>
<td>
apt
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Diamino(2,6)purine resistance in <em>pnp</em>
</td>
<td>
apt
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Dicyclohexylcarbodiimide resistance in <em>rfa</em>
</td>
<td>
rfb
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Dideoxy(2′,3′)thymidine resistance
</td>
<td>
tmk
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Dihydroproline resistance
</td>
<td>
putP
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Dihydroxybutylphosphonate resistance
</td>
<td>
cls
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Dihydroxybutylphosphonate resistance
</td>
<td>
glpT
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Dihydroxybutylphosphonate resistance in <em>glpT</em>
</td>
<td>
<em>ugpA,B</em>
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Dihydroxybutylphosphonate resistance in <em>uhp</em>(Con)
</td>
<td>
uhpT
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Dimethyl sulfoxide resistance
</td>
<td>
pss
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Dinitropyrene resistance
</td>
<td>
atoB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Dipeptide (valine containing) resistance
</td>
<td>
dppA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Drug-resistant gene maintenance
</td>
<td>
dor
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Erythromycin growth dependence
</td>
<td>
rrn
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Erythromycin resistance
</td>
<td>
eryC
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Erythromycin resistance
</td>
<td>
eryD
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Erythromycin resistance
</td>
<td>
mac
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Erythromycin resistance
</td>
<td>
rplD
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Erythromycin resistance
</td>
<td>
rplV
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Erythromycin resistance
</td>
<td>
rplC
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Ethanol resistance
</td>
<td>
pss
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Ethionine resistance
</td>
<td>
metG
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Ethionine resistance
</td>
<td>
metJ
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluoroacetate resistance
</td>
<td>
ack
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluoroacetate resistance
</td>
<td>
pta
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluorocitrate resistance
</td>
<td>
tct
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluoro(5)cytosine resistance
</td>
<td>
codA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluoro(5)cytosine resistance
</td>
<td>
cod
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluorodeoxycytidine resistance
</td>
<td>
cdd
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluorodeoxycytidine resistance
</td>
<td>
nupC
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluorodeoxycytidine resistance
</td>
<td>
nupG
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluoro(5)deoxyuridine resistance
</td>
<td>
tdk
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluorodeoxyuridine resistance
</td>
<td>
nupC
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluorodeoxyuridine resistance
</td>
<td>
nupG
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluorodeoxyuridine + uracil resistance in <em>deoA</em>
</td>
<td>
tdk
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluoro(2)-L-erythrocitrate resistance
</td>
<td>
tct
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluoro-3-hydroxyacetone resistance in <em>uhp</em>(Con)
</td>
<td>
uhpT
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluorohydroxyacetone phosphate resistance
</td>
<td>
ugpAB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluorohydroxyacetone phosphate resistance
</td>
<td>
glpT
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluoro(3)malate resistance
</td>
<td>
dct
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluoro(5)orotic acid resistance
</td>
<td>
pyrF
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluoro(5)orotic acid resistance
</td>
<td>
pyrH
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluoro(4)phenylalanine resistance
</td>
<td>
tyrR
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluoro(4)phenylalanine resistance
</td>
<td>
pheA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluoro(4)phenylalanine resistance
</td>
<td>
pheR
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluoro(4)phenylalanine resistance
</td>
<td>
pheS
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluoro(4)phenylalanine resistance
</td>
<td>
pheU
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluorophenylalanine resistance
</td>
<td>
aroF
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluorophenylalanine resistance
</td>
<td>
aroG
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluorophenylalanine resistance
</td>
<td>
aroP
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluorophenylalanine resistance
</td>
<td>
tyrA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluoro(5)tryptophan resistance
</td>
<td>
aroP
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluoro(6)tryptophan resistance
</td>
<td>
<em>trp</em> (generic)
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluorotyrosine resistance
</td>
<td>
tyrR
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluorotyrosine resistance
</td>
<td>
tyrS
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluoro(5)uracil + fluoro(5)uridine resistance
</td>
<td>
guaB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluoro(5)uracil + 5′-AMP resistance in <em>upp</em>
</td>
<td>
ushA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluoro(5)uracil + 5-fluorouridine resistance in <em>udp</em>
</td>
<td>
pyrH
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluoro(5)uracil + nucleotide resistance
</td>
<td>
ompF
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluoro(5)uracil + nucleotide resistance
</td>
<td>
ompR
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluoro(5)uracil + adenosine resistance
</td>
<td>
udp
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluorouracil + adenosine resistance in <em>upp deoD xapR</em>(Con)
</td>
<td>
xapA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluoro(5)uracil + adenosine resistance in <em>upp, phoS,</em> or <em>phoT</em>
</td>
<td>
phoA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluoro(5)uracil + adenosine resistance in <em>upp, phoS,</em> or <em>phoT</em>
</td>
<td>
phoB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluoro(5)uracil and 3′-AMP resistance
</td>
<td>
cpdB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluoro(5)uracil + 5′-AMP + 3′-AMP resistance
</td>
<td>
crp
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluoro(5)uracil + carbamylaspartate resistance
</td>
<td>
ubiF
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluoro(5)uracil resistance
</td>
<td>
upp
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluorouracil resistance
</td>
<td>
nupC
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluorouracil resistance
</td>
<td>
nupG
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluorouracil resistance
</td>
<td>
pyrH
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluorouracil resistance
</td>
<td>
rpoB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluoro(5)uridine + uracil resistance
</td>
<td>
udk
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluoro(5)uridine resistance
</td>
<td>
udhA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluoro(5)uridine resistance in <em>upp</em>
</td>
<td>
udk
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluorouridine resistance
</td>
<td>
pyrH
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluorouridine resistance
</td>
<td>
nupC
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fluorouridine resistance
</td>
<td>
nupG
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fosfomycin resistance due to increased gene dosage
</td>
<td>
murZ
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fosfomycin + fructose-6-phosphate resistance
</td>
<td>
pgi
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fosfomycin resistance
</td>
<td>
crp
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fosfomycin resistance
</td>
<td>
cyaA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fosfomycin resistance
</td>
<td>
glpT
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fosfomycin resistance
</td>
<td>
hipA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fosfomycin resistance
</td>
<td>
mrb
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fosfomycin resistance
</td>
<td>
ptsI
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fosfomycin resistance
</td>
<td>
uhpT
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fucitol resistance in galactitol-utilizing strains
</td>
<td>
gatACD
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fucose + arabinose-supported growth in <em>galP</em>
</td>
<td>
mglA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fucose + arabinose-supported growth in <em>galP</em>
</td>
<td>
mglB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fucose + arabinose-supported growth in <em>galP</em>
</td>
<td>
mglC
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fucose resistance (arabinose as C source)
</td>
<td>
araC
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fusaric acid resistance of Tn<em>10</em> insertion mutants
</td>
<td>
tet
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Fusidic acid resistance
</td>
<td>
fusA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Galactose resistance in <em>galE</em>
</td>
<td>
galK
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Galactose resistance in <em>galT</em>
</td>
<td>
galK
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Galactose resistance in <em>galU</em>
</td>
<td>
<em>gal</em> operon
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Galactose resistance in <em>galU</em>
</td>
<td>
galK
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Galactose resistance in <em>galT+K</em>+<em>E</em>+/<em>galU</em> <em>galT+K</em>+<em>E</em>+ merodiploid
</td>
<td>
galR
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Galactose utilization in presence of thiomethylglucoside
</td>
<td>
<em>gal</em> operon
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Gentamicin resistance
</td>
<td>
rplF
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Gentamicin resistance
</td>
<td>
ubiF
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Globomicin resistance
</td>
<td>
dnaE
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Globomicin resistance
</td>
<td>
lpp
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Glucarate (D) resistance in <em>ppc</em>
</td>
<td>
garA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Glucose + gluconate-independent motility
</td>
<td>
<em>pts</em> (general)
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Glucose-resistant satisfaction of <em>trp</em> by indole + 5-methyltryptophan
</td>
<td>
tna
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Glutamate (D) resistance in <em>gltS</em> (increased)
</td>
<td>
gltS
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Glutamine (D) resistance
</td>
<td>
glnP
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Glutamyl(γ)hydrazide resistance
</td>
<td>
glnF
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Glutamyl(γ)hydrazide resistance
</td>
<td>
glnH
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Glutamyl(γ)hydrazide resistance
</td>
<td>
glnP
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Glutamyl(γ)methyl ester resistance
</td>
<td>
gltX
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Glutamyl(γ)methyl ester resistance
</td>
<td>
metJ
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Glutamyl(γ)methyl ester resistance
</td>
<td>
metK
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Glyceraldehyde (DL) 3-phosphate resistance
</td>
<td>
glpT
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Glyceraldehyde (L) 3-phosphate resistance in <em>uhp</em>(Con)
</td>
<td>
uhpT
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Glyceraldehyde (L) resistance
</td>
<td>
glpF
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Glyceraldehyde (L) resistance
</td>
<td>
glpK
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Glycerol 3-phosphorothioate resistance
</td>
<td>
glpT
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Glycine tolerance
</td>
<td>
qmeACDE
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Glycylglycyl-<em>N</em>-phosphonoacetylornithine resistance
</td>
<td>
argR
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Glycylleucine resistance
</td>
<td>
ilvA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Glycylleucine resistance
</td>
<td>
ilvB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Glycylleucine resistance
</td>
<td>
ilvH
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Glycylleucine resistance
</td>
<td>
ilvI
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Glycylleucine resistance
</td>
<td>
oppA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Glycylleucine resistance in <em>ilv</em>
</td>
<td>
gleR
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Glycylglycyl histidinol phosphate ester resistance
</td>
<td>
oppA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Glycylglycyl-<em>N</em>-phosphonoacetylornithine resistance
</td>
<td>
oppA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Glyphosate resistance
</td>
<td>
aroA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Hexuronate resistance in <em>eda</em>
</td>
<td>
uxaBC
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Hexuronate resistance in <em>eda</em>
</td>
<td>
uxuAB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Homocysteic acid resistance
</td>
<td>
gltS
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Hydrazino (α) imidazole propionic acid resistance
</td>
<td>
argT
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Hydrazino (α) imidazole propionic acid resistance
</td>
<td>
hisG
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Hydrazino (α) imidazole propionic acid resistance
</td>
<td>
hisJ
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Hydrazino (α) imidazole propionic acid resistance
</td>
<td>
hisQ
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Hydrazino (α) imidazole propionic acid resistance in <em>dhuA</em>
</td>
<td>
hisP
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Hydroxy (β) norvaline resistance
</td>
<td>
metL
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Hydroxyaspartate resistance
</td>
<td>
pan
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Hydroxybutylphosphonate resistance in <em>uhp</em>(Con)
</td>
<td>
uhpT
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Hydroxyurea resistance
</td>
<td>
nrdA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Hydroxyurea resistance
</td>
<td>
nrdB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Indole acrylic acid resistance
</td>
<td>
aroT
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Indolmycin resistance
</td>
<td>
trpS
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Indospicine resistance
</td>
<td>
argR
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Iodoacetylglucosamine resistance
</td>
<td>
nagA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Iodoacetylglucosamine resistance
</td>
<td>
nagB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Iodoacetylglucosamine resistance
</td>
<td>
nagE
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Isoniazid resistance
</td>
<td>
pdx
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Kanamycin resistance
</td>
<td>
<em>atpA–atpE</em>
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Kanamycin resistance
</td>
<td>
cpxA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Kanamycin resistance
</td>
<td>
ecfB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Kanamycin resistance
</td>
<td>
hemA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Kanamycin resistance
</td>
<td>
hemB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Kanamycin resistance
</td>
<td>
hemL
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Kanamycin resistance
</td>
<td>
topA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Kasugamycin resistance
</td>
<td>
ksgA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Kasugamycin resistance
</td>
<td>
ksgB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Kasugamycin resistance
</td>
<td>
ksgC
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Kasugamycin resistance
</td>
<td>
ksgD
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Kasugamycin resistance
</td>
<td>
rplK
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Kasugamycin resistance
</td>
<td>
rplB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Kasugamycin resistance
</td>
<td>
rpsN
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Kasugamycin resistance
</td>
<td>
rpsM
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Kasugamycin resistance
</td>
<td>
rpsR
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Kasugamycin resistance and dependence
</td>
<td>
rpsI
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Keto(2)butyrate resistance
</td>
<td>
ilvA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Kirromycin resistance
</td>
<td>
tufAB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Lambda <em>c</em>II phage + rifampin coresistance
</td>
<td>
rpoB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Lambda phage + nalidixic acid resistance
</td>
<td>
crp
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Lambda phage + nalidixic acid resistance
</td>
<td>
cyaA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Levallorphan resistance
</td>
<td>
lev
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Lincomycin resistance
</td>
<td>
linB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Lincomycin resistance
</td>
<td>
rplN
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Lincomycin resistance
</td>
<td>
rplO
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Lincomycin resistance
</td>
<td>
rpsG
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Lithium-resistant use of proline as C source
</td>
<td>
putP
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Lysine hydroxamate resistance
</td>
<td>
lysC
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Mecillinam resistance (see beta-lactam resistance)
</td>
<td>
alaS
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Mecillinam resistance
</td>
<td>
argS
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Mecillinam resistance
</td>
<td>
crp
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Mecillinam resistance
</td>
<td>
cyaA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Mecillinam resistance
</td>
<td>
envB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Mecillinam resistance
</td>
<td>
mrdA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Mecillinam resistance
</td>
<td>
mrdB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Mecillinam resistance
</td>
<td>
mreB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Mecillinam resistance
</td>
<td>
mreC
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Mecillinam resistance
</td>
<td>
mreD
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Menadionine resistance
</td>
<td>
marA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Mercaptopurine resistance in <em>gpt</em>
</td>
<td>
hpt
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Methionine sulfoximine + methyl(α)methionine resistance
</td>
<td>
metJ
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Methionine sulfoximine resistance
</td>
<td>
asm
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Methionine sulfoximine resistance
</td>
<td>
glnP
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Methionine sulfoximine resistance
</td>
<td>
metP
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Methionine sulfoximine resistance in <em>glnG</em>
</td>
<td>
glnA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Methylammonium resistance
</td>
<td>
glnA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Methyl(3)anthranilate resistance
</td>
<td>
trpE
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Methyl(3)anthranilate resistance
</td>
<td>
aroG
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Methyl(α)glutamate resistance in <em>gltS</em> (increased)
</td>
<td>
gltS
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Methylglucoside(α)-resistant utilization of lactose
</td>
<td>
lacIO
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Methylglucoside(α)-resistant utilization of maltose
</td>
<td>
malE
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Methylglucoside(α)-resistant utilization of maltose
</td>
<td>
malF
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Methylglucoside(α)-resistant utilization of maltose
</td>
<td>
malG
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Methylglucoside(α)-resistant utilization of maltose
</td>
<td>
malK
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Methylglucoside(α)-resistant utilization of mannitol + lactose
</td>
<td>
ptsG
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Methylglucoside(α) resistance
</td>
<td>
ptsG
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Methylglucoside(α)-resistant utilization of glycerol in ∆<em>ptsHI</em>
</td>
<td>
crr
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Methylglucoside(α)-resistant utilization of lactose in <em>ptsH</em>
</td>
<td>
<em>lacI,</em> operator
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Methylglucoside(α)-resistant utilization of melibiose
</td>
<td>
mel
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Methylglyoxal resistance
</td>
<td>
gsh
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Methyl(2)histidine resistance + aminotriazole resistance
</td>
<td>
hisR
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Methyl methanesulfonate resistance in <em>lon</em>
</td>
<td>
sulA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Methyl methanesulfonate resistance in <em>lon</em>
</td>
<td>
ftsZ
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Methyl(α)methionine resistance
</td>
<td>
metA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Methyl(α)methionine resistance
</td>
<td>
<em>metD</em> (<em>metP</em>)
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Methyl(α)methionine resistance
</td>
<td>
metK
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Methyl(6)purine resistance
</td>
<td>
apt
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Methyl(6)purine + hypoxanthine resistance
</td>
<td>
purA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Methyl(<em>o</em>)threonine resistance
</td>
<td>
brnQ
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Methyl(4)tryptophan resistance in <em>aroP</em>
</td>
<td>
aroT
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Methyl(5)tryptophan resistance
</td>
<td>
mtr
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Methyl(5)tryptophan resistance
</td>
<td>
aroP
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Methyl(5)tryptophan resistance
</td>
<td>
<em>trp</em> (generic)
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Methyl(5)tryptophan resistance
</td>
<td>
trpE
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Methyl(5)tryptophan resistance
</td>
<td>
trpR
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Methyl(5)tryptophan + thienylalanine resistance
</td>
<td>
aroP
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Methyl viologen resistance
</td>
<td>
mvrA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Methyl viologen resistance (multicopy)
</td>
<td>
mvrC
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Methyl viologen resistance
</td>
<td>
gor
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Metronidazole resistance
</td>
<td>
<em>nar</em> (general)
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Microcin B17 resistance
</td>
<td>
sbmA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Microcin E492 resistance
</td>
<td>
semA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Mitomycin C resistance of <em>lexA</em>(Con)
</td>
<td>
lexA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Mocimycin resistance
</td>
<td>
<em>tufA,B</em>
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Nalidixic acid resistance
</td>
<td>
hisU
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Nalidixic acid resistance
</td>
<td>
icdE
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Nalidixic acid tolerance
</td>
<td>
sloB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Nalidixic acid resistance
</td>
<td>
gyrA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Nalidixic acid resistance
</td>
<td>
gyrB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Nalidixic acid resistance
</td>
<td>
nalB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Nalidixic acid resistance
</td>
<td>
nalD
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Nalidixic acid resistance
</td>
<td>
purB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Nalidixic acid resistance due to gene dosage
</td>
<td>
emrB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Neamine resistance
</td>
<td>
rpsL
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Neamine resistance
</td>
<td>
rpsQ
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Neamine resistance
</td>
<td>
neaB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Negamycin resistance
</td>
<td>
prfB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Neomycin resistance
</td>
<td>
<em>atpA–atpE</em>
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Neomycin resistance
</td>
<td>
ecfB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Neomycin resistance
</td>
<td>
hemA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Neomycin resistance
</td>
<td>
hemC
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Neomycin resistance
</td>
<td>
hemD
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Neomycin resistance
</td>
<td>
hemE
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Neomycin resistance
</td>
<td>
hemG
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Neomycin resistance
</td>
<td>
<em>metC</em> (<em>ecfA</em>)
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Neomycin resistance
</td>
<td>
topA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Neomycin resistance
</td>
<td>
<em>ubi</em> (generic)
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Neutrophil granule protein resistance
</td>
<td>
pmrA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Nitro(4)pyridine <em>N</em>-oxide resistance
</td>
<td>
proA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Nitro(4)pyridine <em>N</em>-oxide resistance
</td>
<td>
proB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Nitro(<em>o</em>)-phenyl galactoside resistance
</td>
<td>
lacZ
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Nitro(<em>o</em>)-phenylthiogalactoside resistance
</td>
<td>
lacY
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Nitro(<em>o</em>)-phenylthiogalactoside resistance in <em>lacI</em>(Con) or <em>lacO</em>(Con) 
</td>
<td>
<em>lacI,</em> operator
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Nitrofurantoin resistance
</td>
<td>
nfnA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Nitrofurantoin resistance
</td>
<td>
nfnB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Nitrofurantoin resistance in <em>lon</em>
</td>
<td>
sulA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Nitrofurantoin resistance in <em>lon</em>
</td>
<td>
ftsZ
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Nitrofuranzone resistance
</td>
<td>
nfsA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Nitrofuranzone resistance
</td>
<td>
nfsB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Nitrosoguanidine resistance
</td>
<td>
ada
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Nitrosoguanidine resistance
</td>
<td>
<em>gsh</em> (generic)
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Norfloxacin resistance
</td>
<td>
hipQ
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Norleucine resistance
</td>
<td>
metJ
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Norleucine resistance
</td>
<td>
metK
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Norleucine resistance
</td>
<td>
nol
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Norleucylglycyl glycine resistance
</td>
<td>
oppA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Novobiocin resistance
</td>
<td>
cysB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Novobiocin resistance
</td>
<td>
cysE
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Novobiocin resistance
</td>
<td>
gyrB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Novobiocin resistance
</td>
<td>
nov
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Novobiocin resistance
</td>
<td>
ompA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Oxolinic acid resistance
</td>
<td>
gyrA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Paromycin resistance or dependence
</td>
<td>
rpsL
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Pentachlorophenol resistance
</td>
<td>
<em>atpA–atpE</em>
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Penten(4)oate resistance in <em>atoC</em>(Con)
</td>
<td>
atoC
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Pentylpantothenamide resistance
</td>
<td>
pan
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Peptide (toxic, valine containing) resistance
</td>
<td>
pepA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Peptide (toxic, valine containing) resistance
</td>
<td>
pepD
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Peptide (toxic, valine containing) resistance
</td>
<td>
pepN
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Peptide (toxic, valine containing) resistance
</td>
<td>
pepQ
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Peroxide (organic) resistance
</td>
<td>
oxyR
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Phaseolotoxin resistance
</td>
<td>
oppA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Phenethyl alcohol resistance
</td>
<td>
dnaB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Phenethyl alcohol resistance
</td>
<td>
dnaP
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Phenethyl alcohol resistance
</td>
<td>
secA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Phenethylyl galactoside resistance
</td>
<td>
lacY
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Phenethylyl galactoside resistance
</td>
<td>
lacZ
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Phenyl galactoside resistance of <em>lacP lacI</em>(nonsense) <em>supE</em>
</td>
<td>
hisT
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Phenyl galactoside resistance of <em>lacP lacI</em>(nonsense) <em>supE</em>
</td>
<td>
prfA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Phenyl galactoside resistance of <em>lacP lacI</em>(nonsense) <em>supE</em>
</td>
<td>
rpsL
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Phenyl galactoside resistance of <em>lacI </em>q/<em>lacP lacI</em>(nonsense) <em>supL</em>
</td>
<td>
lysS
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Phenyl galactoside resistance of <em>lacI </em>q/<em>lacP lacI</em>(nonsense) <em>supL</em>
</td>
<td>
strM
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Phenyl galactoside resistance of <em>lacI </em>q/<em>lacP lacI</em>(nonsense) <em>supL</em>
</td>
<td>
trmE
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Phenyl galactoside resistance
</td>
<td>
lacY
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Phenyl galactoside resistance
</td>
<td>
lacZ
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Phenylalanylleucine resistance
</td>
<td>
ilvB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Phenylalanylleucine resistance
</td>
<td>
ilvH
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Phenylalanylleucine resistance
</td>
<td>
ilvI
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Plasmid maintenance
</td>
<td>
polA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Polymyxin resistance
</td>
<td>
pmrA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Promethazine resistance
</td>
<td>
lon
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Pseudomonic acid resistance
</td>
<td>
ileS
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Psicofuranine resistance
</td>
<td>
guaAB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Psoralen + UV irradiation resistance
</td>
<td>
puvA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Pyrithiamine resistance
</td>
<td>
thi
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Quinaldic acid resistance of Tn<em>10</em> insertion mutants
</td>
<td>
tet
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Quinolone resistance
</td>
<td>
gyrA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Ribitol resistance in <em>araC</em>(Con)
</td>
<td>
araB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Ribitol resistance in <em>araC</em>(Con)
</td>
<td>
araC
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Rifampin + kasugamycin dependence
</td>
<td>
ridA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Rifampin dependence
</td>
<td>
ridB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Rifampin resistance
</td>
<td>
crp
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Rifampin resistance
</td>
<td>
cyaA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Rifampin resistance
</td>
<td>
rpoB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Rifampin resistance in <em>rpoB</em>+/<em>rpoB</em> (<em>rif</em>) diploids
</td>
<td>
rpoB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Salicylate resistance
</td>
<td>
pan
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Serine (D) resistance
</td>
<td>
cycA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Serine + methionine + glycine resistance in <em>relA</em>
</td>
<td>
glyA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Serine hydroxamate resistance
</td>
<td>
serA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Serine hydroxamate resistance
</td>
<td>
serS
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Serine resistance
</td>
<td>
sbaA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Serine resistance
</td>
<td>
thrA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Serine resistance in <em>relA</em>
</td>
<td>
crp
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Serine resistance in <em>relA</em>
</td>
<td>
cyaA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Serine resistance in <em>relA</em>
</td>
<td>
relA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Serine resistance in <em>relA</em>
</td>
<td>
rpoB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Serine resistance in <em>relA</em>
</td>
<td>
rpoC
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Showdomycin resistance
</td>
<td>
nupC
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Showdomycin resistance
</td>
<td>
pnp
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Siderophore–beta-lactam conjugate resistance
</td>
<td>
cirA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Sorbitol + xylitol resistance
</td>
<td>
srlA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Sorbose (L) resistance
</td>
<td>
fruA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Sorbose resistance
</td>
<td>
ptsG
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Spectinomycin resistance
</td>
<td>
rpsE
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Spectinomycin resistance
</td>
<td>
rrn
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Spectinomycin resistance
</td>
<td>
spcB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Spectinomycin resistance and sucrose dependence
</td>
<td>
rpsC
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Spectinomycin resistance and sucrose dependence
</td>
<td>
rpsD
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Spectinomycin resistance and sucrose dependence
</td>
<td>
rpsE
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Streptolydigin resistance
</td>
<td>
rpoB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Streptomycin + cAMP resistance
</td>
<td>
crp
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Streptomycin resistance
</td>
<td>
<em>hem</em> (generic)
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Streptomycin resistance
</td>
<td>
rpsL
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Streptomycin resistance
</td>
<td>
strB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Streptomycin resistance
</td>
<td>
strC
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Streptomycin resistance
</td>
<td>
strM
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Streptomycin resistance
</td>
<td>
ubiF
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Streptomycin resistance
</td>
<td>
<em>mut</em> (generic)
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Streptomycin resistance
</td>
<td>
topA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Streptovaricin resistance
</td>
<td>
rpoB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Streptozotocin resistance
</td>
<td>
fba
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Streptozotocin resistance
</td>
<td>
galT
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Streptozotocin resistance
</td>
<td>
manA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Streptozotocin resistance
</td>
<td>
nagA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Streptozotocin resistance
</td>
<td>
nagE
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Streptozotocin resistance
</td>
<td>
pfkA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Streptozotocin resistance
</td>
<td>
ptsI
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Streptozotocin resistance
</td>
<td>
glpD
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Sulfanilamide + hypoxanthine resistance
</td>
<td>
hpt
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Sulfanilamide + hypoxanthine resistance
</td>
<td>
pspABCDE
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Sulfanilamide resistance
</td>
<td>
folP
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Sulfanilamide resistance
</td>
<td>
gpt
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Sulfometuron methyl + valine resistance
</td>
<td>
<em>ilv</em> structural genes
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Sulfometuron methyl + valine resistance
</td>
<td>
ilvG
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Sulfonamide resistance
</td>
<td>
folA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Sulfonamide resistance
</td>
<td>
pab
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Tartrate resistance
</td>
<td>
dct
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Tetracycline resistance
</td>
<td>
cmlA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Tetracycline resistance
</td>
<td>
marA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Tetracycline resistance
</td>
<td>
ompF
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Thiaisoleucine resistance
</td>
<td>
ileS
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Thiaisoleucine resistance
</td>
<td>
ilvU
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Thialysine resistance
</td>
<td>
thrA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Thiazolealanine resistance
</td>
<td>
hisG
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Thiazolealanine resistance
</td>
<td>
hisR
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Thienylalanine resistance
</td>
<td>
aroP
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Thienylalanine resistance
</td>
<td>
aroG
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Thiodigalactoside resistance
</td>
<td>
lacY
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Thio(5)glucose resistance
</td>
<td>
crr
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Thiolactomycin resistance
</td>
<td>
emrB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Thiolactomycin resistance (due to gene dosage)
</td>
<td>
fadB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Thiolactomycin resistance (due to gene dosage)
</td>
<td>
emrB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Thiolutin resistance
</td>
<td>
tlnA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Thiomaltose resistance
</td>
<td>
lamB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Thiomaltose resistance
</td>
<td>
malE
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Thiomaltose resistance
</td>
<td>
malF
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Thiomaltose resistance
</td>
<td>
malG
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Thiomaltose resistance
</td>
<td>
malK
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Thiomaltose resistance
</td>
<td>
malT
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Thiomethylgalactoside-independent utilization of galactose in <em>galR</em>(Con)
</td>
<td>
galR
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Thiopeptin resistance
</td>
<td>
rplE
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Thiosine resistance
</td>
<td>
argP
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Thymidine resistance in <em>deoC</em>
</td>
<td>
deoA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Thymidine resistance in <em>deoC</em>
</td>
<td>
deoB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Thymineless death prevention by low thymine levels in <em>thyA</em>
</td>
<td>
deoB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Thymineless death prevention by low thymine levels in <em>thyA</em>
</td>
<td>
deoC
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Thymineless death resistance in <em>thyA</em>
</td>
<td>
recF
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Thymineless death resistance in <em>thyA</em>
</td>
<td>
recJ
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Thymineless death resistance in <em>thyA</em>
</td>
<td>
recO
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Thymineless death resistance in <em>thyA</em>
</td>
<td>
recQ
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Thymineless death resistance in <em>thyA</em>
</td>
<td>
alaS
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Thymineless death resistance in <em>thyA</em>
</td>
<td>
pheS
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Thymineless death resistance in <em>thyA</em>
</td>
<td>
valS
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Tiamulin resistance
</td>
<td>
rplC
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Tiamulin resistance
</td>
<td>
rplD
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Triazole resistance
</td>
<td>
cysB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Triazole resistance
</td>
<td>
cysE
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Triazole resistance
</td>
<td>
cysG
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Triazole resistance
</td>
<td>
trzA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Triazole resistance in <em>cysM</em>
</td>
<td>
cysB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Triazole resistance in <em>cysM</em>
</td>
<td>
cysE
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Triazolealanine + aminotriazole resistance
</td>
<td>
gyrA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Triazolealanine + aminotriazole resistance
</td>
<td>
gyrB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Triazolealanine + aminotriazole resistance
</td>
<td>
<em>his</em> structural genes
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Triazolealanine + aminotriazole resistance
</td>
<td>
hisR
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Triazolealanine + aminotriazole resistance
</td>
<td>
hisS
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Triazolealanine + aminotriazole resistance
</td>
<td>
hisT
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Triazolealanine + aminotriazole resistance
</td>
<td>
hisU
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Triazolealanine + aminotriazole resistance
</td>
<td>
hisW
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Trifluorocitrate resistance
</td>
<td>
tct
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Trifluoroleucine resistance
</td>
<td>
flrB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Trifluoroleucine resistance
</td>
<td>
ileR
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Trifluoroleucine resistance
</td>
<td>
leuA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Trifluoroleucine resistance
</td>
<td>
leuJ
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Trifluoroleucine resistance
</td>
<td>
<em>leu</em> (generic)
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Trifluoroleucine resistance
</td>
<td>
leuS
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Trilysine resistance
</td>
<td>
oppA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Trimethoprim resistance
</td>
<td>
folA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Trimethoprim resistance
</td>
<td>
thyA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Triornithine resistance
</td>
<td>
oppA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Triornithine resistance in <em>opp</em>
</td>
<td>
tpp
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Tripeptide (toxic amino acid containing) resistance
</td>
<td>
oppAE
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Tripeptide (toxic amino acid containing) resistance
</td>
<td>
tppA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Tripeptide (toxic amino acid containing) resistance
</td>
<td>
tppB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Tryptophan analog resistance
</td>
<td>
aroH
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Uncoupler resistances
</td>
<td>
<em>atpA–atpE</em>
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Uridine resistance of <em>thyA deoB</em>
</td>
<td>
deoR
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Valine resistance
</td>
<td>
<em>ilv</em> structural genes
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Valine resistance
</td>
<td>
ilvB
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Valine resistance
</td>
<td>
ilvF
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Valine resistance
</td>
<td>
ilvG
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Valine resistance
</td>
<td>
ilvH
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Valine resistance
</td>
<td>
ilvI
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Valine resistance
</td>
<td>
ilvJ
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Valine resistance
</td>
<td>
livG
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Valine resistance
</td>
<td>
livH
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Valine resistance
</td>
<td>
livJ
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Valine resistance
</td>
<td>
livK
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Valine resistance
</td>
<td>
brnQ
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Vinylglycolate resistance
</td>
<td>
dld
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Vinylglycolate resistance
</td>
<td>
lct
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Xylitol resistance
</td>
<td>
fruA
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Xylose resistance in <em>fda</em>
</td>
<td>
xylE
</td>
</tr>
<tr>
<td>
Resistance to biological and organic chemicals
</td>
<td>
Xylose (D) + cAMP resistance
</td>
<td>
cxm
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
9NA phage resistance
</td>
<td>
pmi
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
9NA phage resistance in <em>galE</em>
</td>
<td>
kdsA
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Bf23 phage resistance
</td>
<td>
btuB
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
C21 phage resistance
</td>
<td>
rfaD
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Chi phage resistance
</td>
<td>
<em>fliC (hag)</em>
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Chi phage resistance
</td>
<td>
motA
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Chi phage resistance
</td>
<td>
motB
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
ES18 phage resistance
</td>
<td>
fhuA
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
ES18 phage resistance
</td>
<td>
<em>prbA,B</em>
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
ES18 phage resistance
</td>
<td>
sidK
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
ES18 phage resistance
</td>
<td>
tonB
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Felix O phage resistance
</td>
<td>
galE
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Felix O phage resistance
</td>
<td>
galU
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Felix O phage resistance
</td>
<td>
<em>rfaC,D,E,F,H</em>
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Filamentous phage tolerance
</td>
<td>
tolR
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Filamentous phage tolerance
</td>
<td>
tolQ
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
HK009 phage resistance
</td>
<td>
prh
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
HK068 phage resistance
</td>
<td>
prk
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Host range phage (from <em>Serratia marcescens</em>) resistance
</td>
<td>
ompC
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
K10 phage resistance
</td>
<td>
lamB
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
K3 phage resistance
</td>
<td>
ompA
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Lambda + 434 phage resistance
</td>
<td>
mopA
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Lambda + 434 phage resistance
</td>
<td>
mopB
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Lambda <em>c</em>II + rifampin coresistance
</td>
<td>
rpoB
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Lambda mutant infection resistance
</td>
<td>
<em>gyrB</em> (<em>hisW</em>)
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Lambda mutant phage infection, survival of
</td>
<td>
himA
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Lambda mutant phage resistance
</td>
<td>
hflC
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Lambda mutant phage resistance
</td>
<td>
hflK
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Lambda mutant phage resistance
</td>
<td>
hflX
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Lambda mutant prophage induction, survival of
</td>
<td>
grpD
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Lambda mutant prophage induction, survival of
</td>
<td>
grpE
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Lambda phage + nalidixic acid resistance
</td>
<td>
crp
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Lambda phage + nalidixic acid resistance
</td>
<td>
cyaA
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Lambda phage + 434 phage cross-resistance
</td>
<td>
rpoB
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Lambda phage gamma mutant resistance
</td>
<td>
polA
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Lambda phage induction, resistance to
</td>
<td>
dnaJ
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Lambda phage induction, resistance to
</td>
<td>
dnaK
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Lambda phage resistance
</td>
<td>
envZ
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Lambda phage resistance
</td>
<td>
gprAB
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Lambda phage resistance
</td>
<td>
lamB
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Lambda phage resistance
</td>
<td>
malK
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Lambda phage resistance
</td>
<td>
malT
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Lambda phage resistance
</td>
<td>
rap
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Lambda phage resistance with maltose and arabinose as C sources
</td>
<td>
crp
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Lambda phage resistance with maltose and arabinose as C sources
</td>
<td>
cyaA
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Lambda prophage induction deficiency during thymine deprivation
</td>
<td>
recA
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Lambda prophage induction, resistance to
</td>
<td>
rpsJ
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Lambda prophage induction, resistance to
</td>
<td>
rpsM
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Lambda prophage induction, survival of
</td>
<td>
nusA
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Lambda prophage induction, survival of
</td>
<td>
nusB
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Lambda <em>sus N7 nin-5</em> resistance in P2 lysogen
</td>
<td>
rho
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Lambda <em>vir</em> resistance after infection with heteromodified lambda <em>c</em>I857
</td>
<td>
hsd
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Lambdoid phage resistance
</td>
<td>
nusB
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Lambdoid phage mixture coinfection resistance
</td>
<td>
dnaB
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Lambdoid phage mixture coinfection resistance
</td>
<td>
dnaJ
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Lambdoid phage mixture coinfection resistance
</td>
<td>
dnaK
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Lambda <em>c</em>I71 resistance
</td>
<td>
rpoD
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Lambda <em>Nmar</em> mutant phage resistance
</td>
<td>
rpoB
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Male-specific phage resistance
</td>
<td>
<em>arcA</em> (<em>fexA</em>)
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Me1 phage resistance
</td>
<td>
envZ
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Me1 phage resistance
</td>
<td>
ompC
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Me1 phage resistance
</td>
<td>
ompF
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Me1 phage resistance
</td>
<td>
ompR
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Mu phage lytic growth resistance
</td>
<td>
himA
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Mu phage lytic growth resistance
</td>
<td>
himD
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Mu prophage induction, survival of
</td>
<td>
himA
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Mu prophage induction, survival of
</td>
<td>
himD
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
N4 phage resistance
</td>
<td>
manXYZ
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Ox2 phage resistance
</td>
<td>
ompA
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
P1 phage resistance
</td>
<td>
galE
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
P1 phage resistance
</td>
<td>
galU
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
P1 phage resistance
</td>
<td>
lpcA
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
P1 phage resistance
</td>
<td>
rfaD
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
P1 phage specialized transduction of drug resistance to <em>Salmonella</em> sp.
</td>
<td>
sspA
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
P2 <em>vir1</em> phage resistance
</td>
<td>
rpoA
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
P2 phage + lambda phage coinfection resistance
</td>
<td>
dnaJ
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
P2 phage + lambda phage coinfection resistance
</td>
<td>
dnaK
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
P2 phage resistance
</td>
<td>
rep
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
P22 phage resistance
</td>
<td>
galE
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
P22 phage resistance
</td>
<td>
pmi
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
P221 phage resistance
</td>
<td>
praAB
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
PH105 phage resistance
</td>
<td>
<em>ompB</em> (<em>ompR</em> of <em>E. coli</em>)
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
PH105 phage resistance
</td>
<td>
ompC
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
PH105 phage resistance
</td>
<td>
ompF
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
PH105 phage resistance
</td>
<td>
praAB
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
PH51 phage resistance
</td>
<td>
<em>ompB</em> (<em>ompR</em> of <em>E. coli</em>)
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
PH51 phage resistance
</td>
<td>
ompC
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
PH51 phage resistance
</td>
<td>
ompF
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
PH51 phage resistance
</td>
<td>
praAB
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
PH51 phage resistance
</td>
<td>
prdB
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
φX174 phage resistance
</td>
<td>
phxB
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
φ80 phage resistance
</td>
<td>
fhuA
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
φ80 phage resistance
</td>
<td>
tonB
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
Q phage resistance
</td>
<td>
<em>cpxA</em> (<em>ecfB, ssd, eup</em>)
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
T-even phage resistance
</td>
<td>
ompA
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
T1 phage (UV irradiated) resistance
</td>
<td>
phr
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
T1 phage resistance
</td>
<td>
fhuA
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
T1 phage resistance
</td>
<td>
tonB
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
T2 phage resistance in <em>ompF</em>+ or <em>ompF</em>
</td>
<td>
fadL
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
T3 phage resistance
</td>
<td>
lpcA
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
T4 mutant phage resistance
</td>
<td>
lit
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
T4 phage <em>lig</em> mutant nibbling of colonies yielding <em>lig</em>-overproducing strain
</td>
<td>
lig
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
T4 phage <em>lig</em> mutant resistance from <em>lig</em>-overproducing strain
</td>
<td>
lig
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
T4 phage resistance
</td>
<td>
lpcA
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
T4 phage resistance
</td>
<td>
lpcB
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
T4 phage resistance
</td>
<td>
mopA
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
T4 phage resistance
</td>
<td>
mopB
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
T4 phage resistance
</td>
<td>
rho
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
T4 phage resistance
</td>
<td>
tabC
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
T4 phage resistance
</td>
<td>
lit
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
T4 phage (uracil containing) resistance
</td>
<td>
ung
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
T5 phage resistance
</td>
<td>
fhuA
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
T6 phage resistance
</td>
<td>
crp
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
T6 phage resistance
</td>
<td>
cyaA
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
T6 phage resistance
</td>
<td>
tsx
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
T7 phage gene 1.2 mutant resistance
</td>
<td>
optA
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
T7 phage gene 2 resistance
</td>
<td>
rpoC
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
T7 phage resistance
</td>
<td>
groM
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
T7 phage resistance
</td>
<td>
lpcA
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
T7 phage resistance
</td>
<td>
lpcB
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
T7 phage resistance
</td>
<td>
rpoB
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
T7 phage resistance
</td>
<td>
trxA
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
TC45 phage resistance
</td>
<td>
phoB
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
TC45 phage resistance
</td>
<td>
phoE
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
TC45 phage resistance
</td>
<td>
phoR
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
TC45 phage resistance
</td>
<td>
phoS
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
TC45 phage resistance
</td>
<td>
phoT
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
TC45 phage resistance
</td>
<td>
pstABCS
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
TP1 phage resistance
</td>
<td>
envZ
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
TuIa phage resistance
</td>
<td>
envZ
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
TuIa phage resistance
</td>
<td>
ompC
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
TuIa phage resistance
</td>
<td>
ompF
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
TuIa phage resistance
</td>
<td>
ompR
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
TuIa phage resistance
</td>
<td>
tolQAB
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
TuIb phage resistance
</td>
<td>
ompF
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
TuII* phage resistance
</td>
<td>
ompA
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
U3 phage + K3 phage coresistance
</td>
<td>
rfaP
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
U3 phage resistance
</td>
<td>
galE
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
U3 phage resistance
</td>
<td>
galU
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
U3 phage resistance
</td>
<td>
pgi
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
U3 phage resistance
</td>
<td>
pgm
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
U3 phage resistance
</td>
<td>
<em>rfa</em> (general)
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
UV-irradiated lytic phage resistance
</td>
<td>
uvrA
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
UV-irradiated lytic phage resistance
</td>
<td>
uvrB
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
UV-irradiated lytic phage resistance
</td>
<td>
uvrC
</td>
</tr>
<tr>
<td>
Resistance to biological agents
</td>
<td>
UV-irradiated lytic phage resistance
</td>
<td>
uvrD
</td>
</tr>
<tr>
<td>
Resistance to physical extremes
</td>
<td>
Cold resistance in <em>rpsE</em> cold-sensitive mutants
</td>
<td>
rpsB
</td>
</tr>
<tr>
<td>
Resistance to physical extremes
</td>
<td>
Cold-resistant growth
</td>
<td>
crg
</td>
</tr>
<tr>
<td>
Resistance to physical extremes
</td>
<td>
Filter retention
</td>
<td>
<em>fts</em> (generic)
</td>
</tr>
<tr>
<td>
Resistance to physical extremes
</td>
<td>
Freeze-thaw resistance
</td>
<td>
envZ
</td>
</tr>
<tr>
<td>
Resistance to physical extremes
</td>
<td>
Gamma irradiation resistance
</td>
<td>
garA
</td>
</tr>
<tr>
<td>
Resistance to physical extremes
</td>
<td>
Gamma irradiation resistance
</td>
<td>
garB
</td>
</tr>
<tr>
<td>
Resistance to physical extremes
</td>
<td>
Growth at 42°C in <em>his</em>-overexpressing strain
</td>
<td>
<em>his</em> structural genes
</td>
</tr>
<tr>
<td>
Resistance to physical extremes
</td>
<td>
Near-UV irradiation resistance
</td>
<td>
nuvA
</td>
</tr>
<tr>
<td>
Resistance to physical extremes
</td>
<td>
Near-UV irradiation resistance
</td>
<td>
nuvC
</td>
</tr>
<tr>
<td>
Resistance to physical extremes
</td>
<td>
Near-UV irradiation resistance
</td>
<td>
relA
</td>
</tr>
<tr>
<td>
Resistance to physical extremes
</td>
<td>
Osmotolerance
</td>
<td>
crp
</td>
</tr>
<tr>
<td>
Resistance to physical extremes
</td>
<td>
Osmotolerance
</td>
<td>
cyaA
</td>
</tr>
<tr>
<td>
Resistance to physical extremes
</td>
<td>
Osmotolerance
</td>
<td>
osmB
</td>
</tr>
<tr>
<td>
Resistance to physical extremes
</td>
<td>
Osmotolerance
</td>
<td>
proA
</td>
</tr>
<tr>
<td>
Resistance to physical extremes
</td>
<td>
Osmotolerance
</td>
<td>
proB
</td>
</tr>
<tr>
<td>
Resistance to physical extremes
</td>
<td>
Phage (UV-irradiated T1) resistance
</td>
<td>
phr
</td>
</tr>
<tr>
<td>
Resistance to physical extremes
</td>
<td>
Psoralen + UV irradiation resistance
</td>
<td>
puvA
</td>
</tr>
<tr>
<td>
Resistance to physical extremes
</td>
<td>
Temperature resistance in <em>rpoD</em> (Ts)
</td>
<td>
rpoH
</td>
</tr>
<tr>
<td>
Resistance to physical extremes
</td>
<td>
Thermotolerance
</td>
<td>
gyrA
</td>
</tr>
<tr>
<td>
Resistance to physical extremes
</td>
<td>
UV light resistance
</td>
<td>
crp
</td>
</tr>
<tr>
<td>
Resistance to physical extremes
</td>
<td>
UV light resistance
</td>
<td>
cyaA
</td>
</tr>
<tr>
<td>
Resistance to physical extremes
</td>
<td>
UV light resistance
</td>
<td>
envB
</td>
</tr>
<tr>
<td>
Resistance to physical extremes
</td>
<td>
UV light resistance 
</td>
<td>
relA
</td>
</tr>
<tr>
<td>
Resistance to physical extremes
</td>
<td>
UV irradiation resistance in <em>lon</em>
</td>
<td>
sulA
</td>
</tr>
<tr>
<td>
Resistance to physical extremes
</td>
<td>
UV irradiation resistance in <em>lon</em>
</td>
<td>
ftsZ
</td>
</tr>
<tr>
<td>
Resistance to physical extremes
</td>
<td>
UV light + bromodeoxyuridine resistance
</td>
<td>
tdk
</td>
</tr>
<tr>
<td>
Resistance to physical extremes
</td>
<td>
UV light + bromodeoxyuridine resistance
</td>
<td>
ung
</td>
</tr>
<tr>
<td>
Resistance to physical extremes
</td>
<td>
UV light + psoralen resistance
</td>
<td>
puvA
</td>
</tr>
<tr>
<td>
Resistance to physical extremes
</td>
<td>
UV-irradiated lytic phage resistance
</td>
<td>
uvrA
</td>
</tr>
<tr>
<td>
Resistance to physical extremes
</td>
<td>
UV-irradiated lytic phage resistance
</td>
<td>
uvrB
</td>
</tr>
<tr>
<td>
Resistance to physical extremes
</td>
<td>
UV-irradiated lytic phage resistance
</td>
<td>
uvrC
</td>
</tr>
<tr>
<td>
Resistance to physical extremes
</td>
<td>
UV-irradiated lytic phage resistance
</td>
<td>
uvrD
</td>
</tr>
<tr>
<td>
Movement
</td>
<td>
Migration in a chemical gradient
</td>
<td>
che
</td>
</tr>
<tr>
<td>
Movement
</td>
<td>
Static cultivation (prolonged)
</td>
<td>
fliC
</td>
</tr>
</table>

<h1><a name="tab2"></a><strong>TABLE 2</strong> Genes for which selections exist in <i>E. coli</i> and <i>S. typhimurium</i>
</h1>
<table>
<tr><td>Gene</td><td>Organism</td><td>Selection</td><td>References</td><td>Alteration
</td></tr><tr>
<td>ackA</td><td>E, S</td><td>Fluoroacetate resistance</td><td><a href="#ref96">96</a> <a href="#ref281">281</a> <a href="#ref463">463</a> <a href="#ref797">797</a> </td><td>L
</td></tr>
<tr>
<td>ada</td><td>E</td><td>Nitrosoguanidine resistance</td><td><a href="#ref693">693</a> </td><td> 
</td></tr>
<tr>
<td>add</td><td>E</td><td>Deoxyadenosine + hypoxanthine satisfaction of <i>purB</i> requirement</td><td><a href="#ref370">370</a> </td><td>L
</td></tr>
<tr>
<td>add</td><td>S</td><td>Deoxyadenosine (low) satisfaction of <i>purA</i> requirement</td><td><a href="#ref326">326</a> </td><td>L
</td></tr>
<tr>
<td>adhCE</td><td>E</td><td>Chloracetaldehyde resistance</td><td><a href="#ref157">157</a> </td><td> 
</td></tr>
<tr>
<td>adhCE</td><td>E</td><td>Chlorethanol resistance</td><td><a href="#ref157">157</a> </td><td> 
</td></tr>
<tr>
<td>adhE</td><td>E</td><td>Allyl alcohol resistance</td><td><a href="#ref476">476</a> </td><td>L
</td></tr>
<tr>
<td>adh</td><td>E</td><td>Ethanol as C source</td><td><a href="#ref132">132</a> </td><td>C
</td></tr>
<tr>
<td>alaS</td><td>E</td><td>Thymineless death resistance in <i>thyA</i></td><td><a href="#ref394">394</a> </td><td>S
</td></tr>
<tr>
<td>alaS</td><td>E</td><td>Mecillinam resistance (beta-lactam resistance)</td><td><a href="#ref802">802</a> </td><td>S
</td></tr>
<tr>
<td>ampC</td><td>E</td><td>Ampicillin resistance</td><td><a href="#ref139">139</a> <a href="#ref570">570</a> </td><td>D
</td></tr>
<tr>
<td>ampC</td><td>E</td><td>Ampicillin resistance</td><td><a href="#ref210">210</a> <a href="#ref212">212</a> <a href="#ref362">362</a> </td><td>C
</td></tr>
<tr>
<td>amtAB</td><td>S</td><td>Aminotriazole resistance</td><td><a href="#ref828">828</a> </td><td> 
</td></tr>
<tr>
<td>apeA</td><td>E, S</td><td><i>N</i>-Acetyl-L-phenylalanine-β-naphthyl ester hydrolysis</td><td><a href="#ref421">421</a> <a href="#ref516">516</a> </td><td>L
</td></tr>
<tr>
<td>apt</td><td>E</td><td>6-Methylpurine resistance</td><td><a href="#ref60">60</a> </td><td>L
</td></tr>
<tr>
<td>apt</td><td>E</td><td>Adenine resistance in <i>hpt gpt</i></td><td><a href="#ref462">462</a> </td><td> 
</td></tr>
<tr>
<td>apt</td><td>E</td><td>2,6-Diaminopurine resistance in <i>pnp</i></td><td><a href="#ref470">470</a> </td><td>L
</td></tr>
<tr>
<td>apt</td><td>E, S</td><td>2,6-Diaminopurine resistance</td><td><a href="#ref107">107</a> <a href="#ref390">390</a> </td><td>L
</td></tr>
<tr>
<td>araB</td><td>E</td><td>Ribitol resistance from <i>araC</i>(Con)</td><td><a href="#ref396">396</a> </td><td>L
</td></tr>
<tr>
<td>araC</td><td>E</td><td>Fucose resistance (arabinose as C source)</td><td><a href="#ref67">67</a> </td><td>C
</td></tr>
<tr>
<td>araC</td><td>E</td><td>Ribitol resistance from <i>araC</i>(Con)</td><td><a href="#ref396">396</a> </td><td>L
</td></tr>
<tr>
<td>arcA</td><td>E</td><td>Male-specific phage resistance, also called <i>fexA</i></td><td><a href="#ref110">110</a> <a href="#ref651">651</a> <a href="#ref711">711</a> </td><td>L
</td></tr>
<tr>
<td>argA</td><td>E</td><td>Acetylornithine + uracil satisfaction of <i>car</i> (<i>pyrA</i>)</td><td><a href="#ref153">153</a> </td><td>L
</td></tr>
<tr>
<td>argB</td><td>E</td><td><i>N</i>-Acetylhistidine + ornithine satisfaction of <i>his</i></td><td><a href="#ref50">50</a> </td><td>C
</td></tr>
<tr>
<td>argB</td><td>E</td><td>Acetylornithine + uracil satisfaction of <i>car</i> (<i>pyrA</i>)</td><td><a href="#ref153">153</a> </td><td>L
</td></tr>
<tr>
<td>argC</td><td>E</td><td><i>N</i>-Acetylhistidine + ornithine satisfaction of <i>his</i></td><td><a href="#ref50">50</a> </td><td>C
</td></tr>
<tr>
<td>argC</td><td>E</td><td>Acetylornithine + uracil satisfaction of <i>car</i> (<i>pyrA</i>)</td><td><a href="#ref153">153</a> </td><td>L
</td></tr>
<tr>
<td>argD</td><td>E, S</td><td>Proline-independent growth of ∆<i>proAB</i></td><td><a href="#ref62">62</a> <a href="#ref403">403</a> </td><td>L
</td></tr>
<tr>
<td>argD</td><td>E</td><td>Acetylornithine + uracil satisfaction of <i>car</i> (<i>pyrA</i>)</td><td><a href="#ref153">153</a> </td><td>L
</td></tr>
<tr>
<td>argE</td><td>E</td><td><i>N</i>-Acetylhistidine + ornithine satisfaction of <i>his</i></td><td><a href="#ref50">50</a> </td><td>C
</td></tr>
<tr>
<td>argE</td><td>E</td><td>Acetylnorvaline resistance in <i>argR</i>(Con)</td><td><a href="#ref402">402</a> </td><td>L
</td></tr>
<tr>
<td>argH</td><td>E</td><td><i>N</i>-Acetylhistidine + ornithine satisfaction of <i>his</i></td><td><a href="#ref50">50</a> </td><td>C
</td></tr>
<tr>
<td>argP</td><td>E</td><td>Canavanine resistance</td><td><a href="#ref483">483</a> <a href="#ref655">655</a> </td><td>L
</td></tr>
<tr>
<td>argP</td><td>E</td><td>Thiosine resistance</td><td><a href="#ref293">293</a> <a href="#ref716">716</a> </td><td>L
</td></tr>
<tr>
<td>argP</td><td>E</td><td>Canavanine + thiouracil resistance</td><td><a href="#ref605">605</a> </td><td>L
</td></tr>
<tr>
<td>argR</td><td>E, S</td><td>Arginine-resistant, proline-independent growth of ∆<i>pro</i></td><td><a href="#ref62">62</a> <a href="#ref403">403</a> </td><td>C
</td></tr>
<tr>
<td>argR</td><td>E</td><td>Acetylhistidine satisfaction of <i>his</i></td><td><a href="#ref49">49</a> <a href="#ref50">50</a> <a href="#ref403">403</a> </td><td>C
</td></tr>
<tr>
<td>argR</td><td>E</td><td>Acetylmethionine satisfaction of <i>met</i></td><td><a href="#ref402">402</a> </td><td>C
</td></tr>
<tr>
<td>argR</td><td>E</td><td>Indospicine resistance</td><td><a href="#ref454">454</a> </td><td>C
</td></tr>
<tr>
<td>argR</td><td>E</td><td>Canavanine resistance</td><td><a href="#ref482">482</a> <a href="#ref523">523</a> </td><td>C
</td></tr>
<tr>
<td>argR</td><td>E</td><td>Glycylglycyl-<i>N</i>-phosphonoacetylornithine resistance</td><td><a href="#ref604">604</a> </td><td>C
</td></tr>
<tr>
<td>argR</td><td>E</td><td>Canavanine + azauracil resistance</td><td><a href="#ref605">605</a> </td><td>C
</td></tr>
<tr>
<td>argR</td><td>E</td><td>Canavanine + thiouracil resistance</td><td><a href="#ref605">605</a> </td><td>C
</td></tr>
<tr>
<td>argR</td><td>E</td><td>Acetylnorvaline resistance in <i>argR</i>(Con)</td><td><a href="#ref402">402</a> </td><td> 
</td></tr>
<tr>
<td>argS</td><td>E</td><td>Canavanine resistance</td><td><a href="#ref322">322</a> </td><td>S
</td></tr>
<tr>
<td>argS</td><td>E</td><td>Mecillinam resistance (beta-lactam resistance)</td><td><a href="#ref802">802</a> </td><td>S
</td></tr>
<tr>
<td>argT</td><td>S</td><td>Hydrazino (α) imidazole propionic acid resistance</td><td><a href="#ref432">432</a> <a href="#ref433">433</a> </td><td> 
</td></tr>
<tr>
<td>aroA</td><td>E, S</td><td>Glyphosate resistance</td><td><a href="#ref512">512</a> <a href="#ref731">731</a> <a href="#ref869">869</a> </td><td>C, D
</td></tr>
<tr>
<td>aroF</td><td>E</td><td>Amino(4)phenylalanine resistance</td><td><a href="#ref504">504</a> </td><td>C
</td></tr>
<tr>
<td>aroF</td><td>S</td><td>Fluoro(4)phenylalanine resistance</td><td><a href="#ref270">270</a> <a href="#ref729">729</a> </td><td>C
</td></tr>
<tr>
<td>aroG</td><td>E, S</td><td>Methyl(3)anthranilate resistance</td><td><a href="#ref307">307</a> </td><td>FBI, S
</td></tr>
<tr>
<td>aroG</td><td>E</td><td>Thienylalanine resistance</td><td><a href="#ref215">215</a> </td><td>FBI, S
</td></tr>
<tr>
<td>aroG</td><td>E</td><td>Fluorophenylalanine resistance</td><td><a href="#ref351">351</a> </td><td>FBI, S
</td></tr>
<tr>
<td>aroH</td><td>E</td><td>Tryptophan analog resistance</td><td><a href="#ref637">637</a> </td><td>FBI, S
</td></tr>
<tr>
<td>aroP</td><td>E, S</td><td>Methyl(5)tryptophan resistance</td><td><a href="#ref12">12</a> <a href="#ref94">94</a> <a href="#ref288">288</a> <a href="#ref303">303</a> <a href="#ref393">393</a> <a href="#ref437">437</a> <a href="#ref438">438</a> <a href="#ref439">439</a> </td><td>L, C
</td></tr>
<tr>
<td>aroP</td><td>E, S</td><td>Fluoro(5)tryptophan resistance</td><td><a href="#ref12">12</a> <a href="#ref94">94</a> <a href="#ref288">288</a> <a href="#ref303">303</a> <a href="#ref393">393</a> <a href="#ref437">437</a> <a href="#ref438">438</a> <a href="#ref439">439</a> </td><td>L
</td></tr>
<tr>
<td>aroP</td><td>E, S</td><td>Azaserine resistance</td><td><a href="#ref12">12</a> <a href="#ref94">94</a> <a href="#ref288">288</a> <a href="#ref303">303</a> <a href="#ref393">393</a> <a href="#ref437">437</a> <a href="#ref438">438</a> <a href="#ref439">439</a> <a href="#ref836">836</a> <a href="#ref837">837</a> </td><td>L
</td></tr>
<tr>
<td>aroP</td><td>E, S</td><td>Thienylalanine resistance</td><td><a href="#ref12">12</a> <a href="#ref94">94</a> <a href="#ref288">288</a> <a href="#ref303">303</a> <a href="#ref393">393</a> <a href="#ref437">437</a> <a href="#ref438">438</a> <a href="#ref439">439</a> </td><td>L
</td></tr>
<tr>
<td>aroP</td><td>E, S</td><td>Fluorophenylalanine resistance</td><td><a href="#ref12">12</a> <a href="#ref94">94</a> <a href="#ref288">288</a> <a href="#ref303">303</a> <a href="#ref393">393</a> <a href="#ref437">437</a> <a href="#ref438">438</a> <a href="#ref439">439</a> </td><td>L
</td></tr>
<tr>
<td>aroP</td><td>E, S</td><td>Azaleucine resistance</td><td><a href="#ref12">12</a> <a href="#ref94">94</a> <a href="#ref288">288</a> <a href="#ref303">303</a> <a href="#ref393">393</a> <a href="#ref437">437</a> <a href="#ref438">438</a> <a href="#ref439">439</a> </td><td>L
</td></tr>
<tr>
<td>aroP</td><td>E, S</td><td>Methyl(5)tryptophan + thienylalanaine resistance</td><td><a href="#ref12">12</a> <a href="#ref94">94</a> <a href="#ref288">288</a> <a href="#ref303">303</a> <a href="#ref393">393</a> <a href="#ref437">437</a> <a href="#ref438">438</a> <a href="#ref439">439</a> </td><td>L
</td></tr>
<tr>
<td>aroT</td><td>E, S</td><td>Indole acrylic acid resistance</td><td><a href="#ref769">769</a> </td><td>L
</td></tr>
<tr>
<td>aroT</td><td>E</td><td>Methyl(4)tryptophan resistance in <i>aroP</i></td><td><a href="#ref594">594</a> </td><td>L
</td></tr>
<tr>
<td>asm</td><td>S</td><td>Methionine sulfoxamine resistance</td><td><a href="#ref188">188</a> </td><td> 
</td></tr>
<tr>
<td>atoB</td><td>S</td><td>Dinitropyrene resistance</td><td><a href="#ref590">590</a> </td><td>L
</td></tr>
<tr>
<td>atoC</td><td>E</td><td>Butyrate as C source in <i>fadR</i>(Con)</td><td><a href="#ref602">602</a> </td><td>C
</td></tr>
<tr>
<td>atoC</td><td>E</td><td>Valerate as C source in <i>fadR</i>(Con)</td><td><a href="#ref602">602</a> </td><td>C
</td></tr>
<tr>
<td>atoC</td><td>E</td><td>Penten(4)oate resistance in <i>atoC</i>(Con)</td><td><a href="#ref669">669</a> </td><td>L
</td></tr>
<tr>
<td><i>atpA-atpE</i></td><td>E</td><td>Uncoupler resistances, including DCCD</td><td><a href="#ref225">225</a> <a href="#ref226">226</a> <a href="#ref229">229</a> <a href="#ref355">355</a> <a href="#ref537">537</a> </td><td>L
</td></tr>
<tr>
<td><i>atpA-atpE</i></td><td>E</td><td>Tributyl tin resistance</td><td><a href="#ref355">355</a> </td><td> 
</td></tr>
<tr>
<td><i>atpA-atpE</i></td><td>E</td><td>CCCP resistance</td><td><a href="#ref355">355</a> </td><td> 
</td></tr>
<tr>
<td><i>atpA-atpE</i></td><td>E</td><td>Pentachlorophenol resistance</td><td><a href="#ref355">355</a> </td><td> 
</td></tr>
<tr>
<td><i>atpA-atpE</i></td><td>E</td><td>Auroventin resistance</td><td><a href="#ref451">451</a> <a href="#ref685">685</a> <a href="#ref818">818</a> </td><td>L
</td></tr>
<tr>
<td><i>atpA-atpE</i></td><td>E</td><td>Kanamycin resistance</td><td><a href="#ref768">768</a> </td><td>L
</td></tr>
<tr>
<td><i>atpA-atpE</i></td><td>E</td><td>Neomycin resistance</td><td><a href="#ref392">392</a> </td><td>L
</td></tr>
<tr>
<td><i>atpA-atpE</i></td><td>E</td><td>Azide resistance</td><td><a href="#ref355">355</a> </td><td> 
</td></tr>
<tr>
<td>atr</td><td>S</td><td>Acid resistance</td><td><a href="#ref233">233</a> </td><td> 
</td></tr>
<tr>
<td>azaAB</td><td>E</td><td>Azaserine resistance</td><td><a href="#ref836">836</a> </td><td> 
</td></tr>
<tr>
<td>azl</td><td>E</td><td>Azaleucine resistance</td><td><a href="#ref610">610</a> </td><td> 
</td></tr>
<tr>
<td>bglBC</td><td>E</td><td>Arbutin as C source</td><td><a href="#ref523">523</a> <a href="#ref616">616</a> <a href="#ref640">640</a> </td><td> 
</td></tr>
<tr>
<td>bglBC</td><td>E</td><td>Salicin as C source</td><td><a href="#ref523">523</a> <a href="#ref616">616</a> <a href="#ref640">640</a> </td><td> 
</td></tr>
<tr>
<td>bglT</td><td>E</td><td>Methyl(β)glucoside as C source</td><td><a href="#ref686">686</a> </td><td>C
</td></tr>
<tr>
<td>bio operon</td><td>E</td><td>Dehydrobiotin resistance</td><td><a href="#ref206">206</a> </td><td> 
</td></tr>
<tr>
<td>bioP</td><td>E</td><td>Dehydrobiotin resistance</td><td><a href="#ref206">206</a> </td><td>L
</td></tr>
<tr>
<td>birA</td><td>E</td><td>Dehydrobiotin resistance</td><td><a href="#ref206">206</a> <a href="#ref597">597</a> </td><td> 
</td></tr>
<tr>
<td>brnQ</td><td>E</td><td>Methyl(<i>o</i>)threonine resistance</td><td><a href="#ref278">278</a> </td><td>L
</td></tr>
<tr>
<td>brnQ</td><td>E</td><td>Valine resistance</td><td><a href="#ref277">277</a> </td><td>L
</td></tr>
<tr>
<td>btuB</td><td>E</td><td>Colicin E3 tolerance</td><td><a href="#ref46">46</a> <a href="#ref47">47</a> <a href="#ref109">109</a> <a href="#ref284">284</a> <a href="#ref388">388</a> <a href="#ref620">620</a> </td><td> 
</td></tr>
<tr>
<td>btuB</td><td>E, S</td><td>Bf23 phage resistance</td><td><a href="#ref46">46</a> <a href="#ref47">47</a> <a href="#ref109">109</a> <a href="#ref284">284</a> <a href="#ref388">388</a> <a href="#ref528">528</a> <a href="#ref620">620</a> </td><td> 
</td></tr>
<tr>
<td>bymA</td><td>E</td><td>Maltose utilization in <i>malT</i></td><td><a href="#ref327">327</a> </td><td> 
</td></tr>
<tr>
<td>cbt</td><td>E</td><td>Chloroacetate(β) resistance</td><td><a href="#ref68">68</a> </td><td>L
</td></tr>
<tr>
<td>cbt</td><td>E</td><td>Colicin B tolerance</td><td><a href="#ref623">623</a> </td><td>L
</td></tr>
<tr>
<td>cbt</td><td>E</td><td>Colicin D tolerance</td><td><a href="#ref623">623</a> </td><td>L
</td></tr>
<tr>
<td>cdd</td><td>E</td><td>Aza(5)-2′-deoxycytidine resistance</td><td><a href="#ref196">196</a> </td><td>L
</td></tr>
<tr>
<td>cdd</td><td>E, S</td><td>Fluoro(5)deoxycytidine resistance</td><td><a href="#ref53">53</a> <a href="#ref579">579</a> </td><td>L
</td></tr>
<tr>
<td>celABCDF</td><td>E</td><td>Cellobiose as C source</td><td><a href="#ref426">426</a> </td><td> 
</td></tr>
<tr>
<td>cet</td><td>E</td><td>Colicin E2 tolerance</td><td><a href="#ref201">201</a> </td><td> 
</td></tr>
<tr>
<td>che</td><td>S</td><td>Migration in a chemotaxis gradient</td><td><a href="#ref34">34</a> </td><td> 
</td></tr>
<tr>
<td>chlABDEG</td><td>E, S</td><td>Chlorate resistance</td><td><a href="#ref2">2</a> <a href="#ref121">121</a> <a href="#ref122">122</a> <a href="#ref228">228</a> <a href="#ref266">266</a> <a href="#ref523">523</a> <a href="#ref743">743</a> <a href="#ref744">744</a> </td><td>L
</td></tr>
<tr>
<td>chlABDE</td><td>E</td><td>Lactate + fumarate as anaerobic energy source</td><td><a href="#ref436">436</a> </td><td>L
</td></tr>
<tr>
<td>chlC</td><td>S</td><td>Chlorate resistance</td><td><a href="#ref45">45</a> </td><td>L
</td></tr>
<tr>
<td>chlE</td><td>E</td><td>Lactate + nitrate as anaerobic energy source</td><td><a href="#ref799">799</a> </td><td> 
</td></tr>
<tr>
<td>chlE</td><td>E</td><td>Biotin sulfoxide utilization in <i>chl bio</i></td><td><a href="#ref180">180</a> </td><td> 
</td></tr>
<tr>
<td>cirA</td><td>E</td><td>Colicin I resistance</td><td><a href="#ref77">77</a> <a href="#ref93">93</a> <a href="#ref620">620</a> </td><td>L
</td></tr>
<tr>
<td>cirA</td><td>E</td><td>Siderophore-beta-lactam conjugate resistance</td><td><a href="#ref93">93</a> </td><td>L
</td></tr>
<tr>
<td>cit</td><td>E</td><td>Citrate as C source</td><td><a href="#ref290">290</a> </td><td> 
</td></tr>
<tr>
<td>cls</td><td>E</td><td>Dihydroxy(3,4)butyl-1-phosphonate resistance</td><td><a href="#ref345">345</a> </td><td>L
</td></tr>
<tr>
<td>cmlA</td><td>E</td><td>Chloramphenicol resistance</td><td><a href="#ref48">48</a> </td><td> 
</td></tr>
<tr>
<td>cmlA</td><td>E</td><td>Tetracycline resistance</td><td><a href="#ref523">523</a> <a href="#ref638">638</a> </td><td> 
</td></tr>
<tr>
<td>coaA</td><td>E</td><td>Coenzyme A feedback resistance</td><td><a href="#ref789">789</a> <a href="#ref790">790</a> </td><td>FBI, S
</td></tr>
<tr>
<td>codA(<i>cod</i>)</td><td>E, S</td><td>Fluoro(5)cytosine resistance</td><td><a href="#ref4">4</a> <a href="#ref54">54</a> </td><td>L
</td></tr>
<tr>
<td>corABCD</td><td>E, S</td><td>Cobalt resistance</td><td><a href="#ref262">262</a> <a href="#ref600">600</a> <a href="#ref817">817</a> </td><td> 
</td></tr>
<tr>
<td>corABCD</td><td>E</td><td>Manganese resistance</td><td><a href="#ref600">600</a> </td><td> 
</td></tr>
<tr>
<td>corABCD</td><td>E</td><td>Nickel resistance</td><td><a href="#ref817">817</a> </td><td> 
</td></tr>
<tr>
<td>cpd</td><td>S</td><td>Succinate as improved C source</td><td><a href="#ref8">8</a> <a href="#ref530">530</a> </td><td>L
</td></tr>
<tr>
<td>cpd</td><td>S</td><td>Citrate as C source in presence of low cAMP</td><td><a href="#ref8">8</a> </td><td>L
</td></tr>
<tr>
<td>cpdB</td><td>E</td><td>Fluorouracil(5) and 3′-AMP resistance</td><td><a href="#ref51">51</a> </td><td>L
</td></tr>
<tr>
<td>cpxA</td><td>E</td><td>Amikacin resistance</td><td><a href="#ref629">629</a> </td><td> 
</td></tr>
<tr>
<td>cpxA</td><td>E</td><td>Colicin A tolerance</td><td><a href="#ref629">629</a> </td><td> 
</td></tr>
<tr>
<td>cpxA</td><td>E</td><td>Kanamycin resistance</td><td><a href="#ref768">768</a> </td><td> 
</td></tr>
<tr>
<td>cpxA</td><td>E</td><td>Q phage resistance</td><td><a href="#ref508">508</a> <a href="#ref711">711</a> </td><td> 
</td></tr>
<tr>
<td>cpxA</td><td>E</td><td>Serine (L) as C source</td><td><a href="#ref567">567</a> </td><td> 
</td></tr>
<tr>
<td>crg</td><td>E</td><td>Cold-resistant growth</td><td><a href="#ref398">398</a> </td><td> 
</td></tr>
<tr>
<td>crp</td><td>S</td><td>Fosfomycin resistance</td><td><a href="#ref10">10</a> </td><td>L
</td></tr>
<tr>
<td>crp</td><td>E</td><td>Serine (L) resistance in <i>relA</i></td><td><a href="#ref167">167</a> </td><td>L
</td></tr>
<tr>
<td>crp</td><td>E</td><td>Lambda phage + nalidixic acid resistance</td><td><a href="#ref429">429</a> </td><td>L
</td></tr>
<tr>
<td>crp</td><td>E</td><td>Streptomycin + cAMP resistance</td><td><a href="#ref31">31</a> </td><td>L
</td></tr>
<tr>
<td>crp</td><td>E</td><td>Lambda phage resistance with maltose and arabinose as C sources</td><td><a href="#ref92">92</a> <a href="#ref766">766</a> </td><td>L
</td></tr>
<tr>
<td>crp</td><td>E</td><td>cAMP + glucose 6-phosphate resistance</td><td><a href="#ref1a">1a</a> </td><td>L
</td></tr>
<tr>
<td>crp</td><td>E</td><td>cAMP + glucose 6-phosphate + D-xylose + L-arabinose resistance</td><td><a href="#ref1a">1a</a> </td><td>L
</td></tr>
<tr>
<td>crp</td><td>E</td><td>Fluorouracil(5) + 5′-AMP + 3′-AMP resistance</td><td><a href="#ref52">52</a> </td><td>L
</td></tr>
<tr>
<td>crp</td><td>E</td><td>Osmotolerance</td><td><a href="#ref282">282</a> </td><td>L
</td></tr>
<tr>
<td>crp</td><td>E</td><td>Beta-lactam resistance</td><td><a href="#ref362">362</a> </td><td>L
</td></tr>
<tr>
<td>crp</td><td>E</td><td>Mecillinam resistance</td><td><a href="#ref25">25</a> <a href="#ref168">168</a> <a href="#ref857">857</a> </td><td>L
</td></tr>
<tr>
<td>crp</td><td>E</td><td>UV light resistance</td><td><a href="#ref627">627</a> </td><td>L
</td></tr>
<tr>
<td>crp</td><td>E</td><td>Rifampin resistance</td><td><a href="#ref413">413</a> </td><td>L
</td></tr>
<tr>
<td>crp</td><td>E</td><td>T6 phage resistance</td><td><a href="#ref6">6</a> </td><td>L
</td></tr>
<tr>
<td>crr</td><td>E</td><td>Thio(5)glucose resistance</td><td><a href="#ref419">419</a> </td><td>L
</td></tr>
<tr>
<td>crr</td><td>S</td><td>Methylglucoside(α)-resistant utilization of glycerol in <i>ptsHI</i></td><td><a href="#ref563">563</a> </td><td>L
</td></tr>
<tr>
<td>cxm</td><td>E</td><td>Xylose (D) + cAMP resistance</td><td><a href="#ref1a">1a</a> </td><td>L
</td></tr>
<tr>
<td>cyaA</td><td>E</td><td>Lambda phage resistance with maltose and arabinose as C sources</td><td><a href="#ref92">92</a> </td><td>L
</td></tr>
<tr>
<td>cyaA</td><td>E</td><td>Lambda phage + nalidixic acid resistance</td><td><a href="#ref429">429</a> </td><td>L
</td></tr>
<tr>
<td>cyaA</td><td>E</td><td>Serine (L) resistance in <i>relA</i></td><td><a href="#ref167">167</a> </td><td>L
</td></tr>
<tr>
<td>cyaA</td><td>S</td><td>Fosfomycin resistance</td><td><a href="#ref10">10</a> </td><td>L
</td></tr>
<tr>
<td>cyaA</td><td>E</td><td>Osmotolerance</td><td><a href="#ref282">282</a> </td><td>L
</td></tr>
<tr>
<td>cyaA</td><td>E</td><td>Beta-lactam resistance</td><td><a href="#ref362">362</a> </td><td>L
</td></tr>
<tr>
<td>cyaA</td><td>E</td><td>Mecillinam resistance</td><td><a href="#ref25">25</a> <a href="#ref168">168</a> <a href="#ref857">857</a> </td><td>L
</td></tr>
<tr>
<td>cyaA</td><td>E</td><td>UV light resistance</td><td><a href="#ref627">627</a> </td><td>L
</td></tr>
<tr>
<td>cyaA</td><td>E</td><td>Rifampin resistance</td><td><a href="#ref413">413</a> </td><td>L
</td></tr>
<tr>
<td>cyaA</td><td>E</td><td>T6 phage resistance</td><td><a href="#ref6">6</a> </td><td>L
</td></tr>
<tr>
<td>cycA</td><td>E</td><td>Cycloserine (D) resistance</td><td><a href="#ref150">150</a> <a href="#ref646">646</a> <a href="#ref813">813</a> </td><td>L
</td></tr>
<tr>
<td>cycA</td><td>E</td><td>Serine (D) resistance</td><td><a href="#ref689">689</a> </td><td>L
</td></tr>
<tr>
<td>cysA</td><td>S</td><td>Chromate resistance</td><td><a href="#ref581">581</a> </td><td>L
</td></tr>
<tr>
<td>cysA</td><td>E</td><td>Selenate resistance</td><td><a href="#ref728">728</a> </td><td>L
</td></tr>
<tr>
<td>cysA</td><td>S</td><td>Azaserine resistance</td><td><a href="#ref340">340</a> <a href="#ref689">689</a> </td><td>L
</td></tr>
<tr>
<td>cysB</td><td>S</td><td>Chromate resistance</td><td><a href="#ref340">340</a> </td><td>L
</td></tr>
<tr>
<td>cysB</td><td>S</td><td>Azide resistance</td><td><a href="#ref227">227</a> </td><td>C
</td></tr>
<tr>
<td>cysB</td><td>S</td><td>Selenate resistance</td><td><a href="#ref339">339</a> </td><td>L
</td></tr>
<tr>
<td>cysB</td><td>E</td><td>Novobiocin resistance</td><td><a href="#ref631">631</a> </td><td>L
</td></tr>
<tr>
<td>cysB</td><td>S</td><td>Triazole resistance</td><td><a href="#ref227">227</a> </td><td>C
</td></tr>
<tr>
<td>cysB</td><td>S</td><td>Azaserine resistance</td><td><a href="#ref340">340</a> </td><td>L
</td></tr>
<tr>
<td>cysC</td><td>S</td><td>Azaserine resistance</td><td><a href="#ref340">340</a> </td><td>L
</td></tr>
<tr>
<td>cysC</td><td>S</td><td>Chromate resistance</td><td><a href="#ref581">581</a> </td><td>L
</td></tr>
<tr>
<td>cysD</td><td>S</td><td>Azaserine resistance</td><td><a href="#ref340">340</a> </td><td>L
</td></tr>
<tr>
<td>cysD</td><td>S</td><td>Chromate resistance</td><td><a href="#ref581">581</a> </td><td>L
</td></tr>
<tr>
<td>cysE</td><td>S</td><td>Triazole resistance in <i>cysM</i></td><td><a href="#ref717">717</a> <a href="#ref825">825</a> </td><td>C
</td></tr>
<tr>
<td>cysE</td><td>S</td><td>Triazole resistance</td><td><a href="#ref341">341</a> </td><td>C
</td></tr>
<tr>
<td>cysE</td><td>E</td><td>Novobiocin resistance</td><td><a href="#ref631">631</a> </td><td>L
</td></tr>
<tr>
<td>cysG</td><td>S</td><td>Azaserine resistance</td><td><a href="#ref340">340</a> </td><td>L
</td></tr>
<tr>
<td>cysH</td><td>S</td><td>Azaserine resistance</td><td><a href="#ref340">340</a> </td><td>L
</td></tr>
<tr>
<td>cysH</td><td>S</td><td>Chromate resistance</td><td><a href="#ref581">581</a> </td><td>L
</td></tr>
<tr>
<td>cysI</td><td>S</td><td>Azaserine resistance</td><td><a href="#ref340">340</a> </td><td>L
</td></tr>
<tr>
<td>cysI</td><td>S</td><td>Chromate resistance</td><td><a href="#ref581">581</a> </td><td>L
</td></tr>
<tr>
<td>cysJ</td><td>S</td><td>Azaserine resistance</td><td><a href="#ref340">340</a> </td><td>L
</td></tr>
<tr>
<td>cysJ</td><td>S</td><td>Chromate resistance</td><td><a href="#ref581">581</a> </td><td>L
</td></tr>
<tr>
<td>cysK</td><td>S, E</td><td>Azaserine resistance</td><td><a href="#ref340">340</a> <a href="#ref827">827</a> </td><td>L
</td></tr>
<tr>
<td>cysK</td><td>S</td><td>Azide resistance</td><td><a href="#ref147">147</a> <a href="#ref227">227</a> </td><td>L
</td></tr>
<tr>
<td>cysK</td><td>E</td><td>Selenate resistance</td><td><a href="#ref230">230</a> </td><td>L
</td></tr>
<tr>
<td>cysK</td><td>E, S</td><td>Triazole resistance</td><td><a href="#ref147">147</a> <a href="#ref227">227</a> <a href="#ref826">826</a> <a href="#ref827">827</a> </td><td>L, C
</td></tr>
<tr>
<td>cysK</td><td>E</td><td>Selenite resistance</td><td><a href="#ref230">230</a> </td><td>L
</td></tr>
<tr>
<td>cysL</td><td>S</td><td>Selenate resistance</td><td><a href="#ref673">673</a> <a href="#ref674">674</a> </td><td> 
</td></tr>
<tr>
<td>cysL</td><td>S</td><td>Chromate + selenate resistance</td><td><a href="#ref338">338</a> </td><td> 
</td></tr>
<tr>
<td>cysM</td><td>S</td><td>Azaserine resistance</td><td><a href="#ref340">340</a> </td><td>L
</td></tr>
<tr>
<td>cytR</td><td>E</td><td>Uridine as C source, improved utilization</td><td><a href="#ref546">546</a> </td><td> 
</td></tr>
<tr>
<td>dadA</td><td>E, S</td><td>Chloro(β)-D-alanine resistance</td><td><a href="#ref832">832</a> </td><td>L
</td></tr>
<tr>
<td>dadA</td><td>E, S</td><td>D-Amino acid satisfaction of amino acid auxotrophy</td><td><a href="#ref428">428</a> <a href="#ref831">831</a> </td><td>C
</td></tr>
<tr>
<td>dcm</td><td>E</td><td>Aza(5)cytidine resistance</td><td><a href="#ref247">247</a> </td><td>L
</td></tr>
<tr>
<td>dct</td><td>E</td><td>Fluoro(3)malate resistance</td><td><a href="#ref401">401</a> </td><td>L
</td></tr>
<tr>
<td>dct</td><td>E</td><td>Tartrate resistance</td><td><a href="#ref687">687</a> </td><td>L
</td></tr>
<tr>
<td>dct</td><td>S</td><td>Malate (L) as improved C source</td><td><a href="#ref745">745</a> </td><td>D
</td></tr>
<tr>
<td>deoA</td><td>E</td><td>Deoxyadenosine + fluorouracil resistance in <i>upp</i></td><td><a href="#ref3">3</a> </td><td>L
</td></tr>
<tr>
<td>deoA</td><td>E</td><td>Thymidine resistance in <i>deoC</i></td><td><a href="#ref3">3</a> </td><td> 
</td></tr>
<tr>
<td>deoB</td><td>E</td><td>Thymidine resistance in <i>deoC</i></td><td><a href="#ref3">3</a> <a href="#ref651">651</a> </td><td>L
</td></tr>
<tr>
<td>deoB</td><td>E, S</td><td>Thymineless death prevention by low thymine levels in <i>thyA</i></td><td><a href="#ref71">71</a> <a href="#ref471">471</a> </td><td> 
</td></tr>
<tr>
<td>deoC</td><td>E</td><td>Thymineless death prevention by low thymine levels in <i>thyA</i></td><td><a href="#ref325">325</a> <a href="#ref471">471</a> <a href="#ref651">651</a> </td><td>L
</td></tr>
<tr>
<td>deoD</td><td>E</td><td>Deoxy(2)adenosine resistance in <i>deoC</i></td><td><a href="#ref651">651</a> </td><td>L
</td></tr>
<tr>
<td>deoD</td><td>E</td><td>Deoxyadenosine + fluorouracil resistance in <i>upp</i></td><td><a href="#ref3">3</a> </td><td>L
</td></tr>
<tr>
<td>deoR</td><td>E</td><td>Inosine as improved C source</td><td><a href="#ref546">546</a> </td><td>C
</td></tr>
<tr>
<td>deoR</td><td>S</td><td>Diamino(2,6)purine satisfaction of purine auxotrophy</td><td><a href="#ref251">251</a> </td><td> 
</td></tr>
<tr>
<td>deoR</td><td>S</td><td>Uridine resistance in <i>thyA deoB</i></td><td><a href="#ref71">71</a> </td><td> 
</td></tr>
<tr>
<td>dgd</td><td>E</td><td>Arabitol (D) as C source in <i>fuc</i></td><td><a href="#ref849">849</a> </td><td> 
</td></tr>
<tr>
<td>dgoR</td><td>E</td><td>Keto(2)-3-deoxygalactonate as C source</td><td><a href="#ref144">144</a> </td><td>C
</td></tr>
<tr>
<td>dgsA</td><td>E</td><td>Glucosamine as anaerobic C source in <i>ptsG</i></td><td><a href="#ref652">652</a> </td><td>L
</td></tr>
<tr>
<td>dhuA</td><td>E, S</td><td>Histidine (D) satisfaction of <i>his</i> auxotrophs</td><td><a href="#ref422">422</a> <a href="#ref428">428</a> <a href="#ref546">546</a> </td><td> 
</td></tr>
<tr>
<td>dhuA</td><td>S</td><td>Azaserine + tryptophan resistance</td><td><a href="#ref432">432</a> </td><td> 
</td></tr>
<tr>
<td>dld</td><td>E</td><td>Vinylglycolate resistance</td><td><a href="#ref696">696</a> </td><td>L
</td></tr>
<tr>
<td>dml</td><td>S</td><td>Malate (D) as C source</td><td><a href="#ref737">737</a> </td><td> 
</td></tr>
<tr>
<td>dnaB</td><td>E</td><td>Lambdoid phage mixture coinfection resistance</td><td><a href="#ref259">259</a> </td><td> 
</td></tr>
<tr>
<td>dnaB</td><td>E</td><td>Phenethyl alcohol resistance</td><td><a href="#ref494">494</a> </td><td> 
</td></tr>
<tr>
<td>dnaE</td><td>E</td><td>Acriflavin (acridine) resistance</td><td><a href="#ref553">553</a> <a href="#ref554">554</a> <a href="#ref555">555</a> </td><td> 
</td></tr>
<tr>
<td>dnaE</td><td>E</td><td>Globomycin resistance</td><td><a href="#ref668">668</a> </td><td>S
</td></tr>
<tr>
<td>dnaJ</td><td>E</td><td>Lambda phage induction, resistance to</td><td><a href="#ref667">667</a> </td><td>S
</td></tr>
<tr>
<td>dnaJ</td><td>E</td><td>Lambdoid phage mixture coinfection resistance</td><td><a href="#ref257">257</a> <a href="#ref259">259</a> </td><td>S
</td></tr>
<tr>
<td>dnaJ</td><td>E</td><td>P2 phage + lambda phage coinfection resistance</td><td><a href="#ref753">753</a> </td><td>S
</td></tr>
<tr>
<td>dnaK</td><td>E</td><td>Lambda phage induction, resistance to</td><td><a href="#ref667">667</a> </td><td>S
</td></tr>
<tr>
<td>dnaK</td><td>E</td><td>Lambdoid phage mixture coinfection resistance</td><td><a href="#ref257">257</a> <a href="#ref259">259</a> </td><td>S
</td></tr>
<tr>
<td>dnaK</td><td>E</td><td>P2 phage + lambda phage coninfection resistance</td><td><a href="#ref753">753</a> </td><td>S
</td></tr>
<tr>
<td>dnaP</td><td>E</td><td>Phenethyl alcohol resistance</td><td><a href="#ref809">809</a> </td><td> 
</td></tr>
<tr>
<td>dor</td><td>S</td><td>Drug-resistant gene maintenance</td><td><a href="#ref814">814</a> </td><td> 
</td></tr>
<tr>
<td>dppA</td><td>E, S</td><td>Bacilysin resistance</td><td><a href="#ref1">1</a> </td><td>L
</td></tr>
<tr>
<td>dppA</td><td>E, S</td><td>Bialaphos resistance</td><td><a href="#ref1">1</a> </td><td>L
</td></tr>
<tr>
<td>dppA</td><td>E, S</td><td>Valine-containing dipeptide resistance</td><td><a href="#ref1">1</a> </td><td>L
</td></tr>
<tr>
<td>dppA</td><td>E</td><td>Lysyl-2-aminoxypropionate resistance</td><td><a href="#ref603">603</a> </td><td>L
</td></tr>
<tr>
<td>dppA</td><td>E</td><td>Glycylleucine resistance</td><td><a href="#ref269">269</a> </td><td>L
</td></tr>
<tr>
<td>dppA</td><td>E</td><td>Phenylalanylleucine resistance</td><td><a href="#ref269">269</a> </td><td>L
</td></tr>
<tr>
<td>dppA</td><td>E</td><td>Glycylvaline resistance in <i>opp</i></td><td><a href="#ref178">178</a> </td><td>L
</td></tr>
<tr>
<td>dsdA</td><td>E</td><td>Sucrose as C source</td><td><a href="#ref5">5</a> </td><td>L
</td></tr>
<tr>
<td>dsdC</td><td>E</td><td>Sucrose as C source</td><td><a href="#ref5">5</a> </td><td> 
</td></tr>
<tr>
<td>ebgA</td><td>E</td><td>Lactose utilization in ∆<i>lacZ</i></td><td><a href="#ref118">118</a> </td><td>Q
</td></tr>
<tr>
<td>ecfB</td><td>E</td><td>Serine (L) as C source</td><td><a href="#ref567">567</a> <a href="#ref768">768</a> </td><td> 
</td></tr>
<tr>
<td>ecfB</td><td>E</td><td>Kanamycin resistance</td><td><a href="#ref567">567</a> <a href="#ref768">768</a> </td><td> 
</td></tr>
<tr>
<td>ecfB</td><td>E</td><td>Neomycin resistance</td><td><a href="#ref608">608</a> </td><td> 
</td></tr>
<tr>
<td>ecfB</td><td>E</td><td>Colicin K resistance</td><td><a href="#ref608">608</a> </td><td> 
</td></tr>
<tr>
<td>edd</td><td>E</td><td>Gluconate as C source in <i>eda</i></td><td><a href="#ref239">239</a> <a href="#ref240">240</a> </td><td>L
</td></tr>
<tr>
<td>emrB</td><td>E</td><td>CCCP resistance due to gene dosage</td><td><a href="#ref472">472</a> </td><td>D
</td></tr>
<tr>
<td>emrB</td><td>E</td><td>Nalidixic acid resistance due to gene dosage</td><td><a href="#ref472">472</a> </td><td>D
</td></tr>
<tr>
<td>emrB</td><td>E</td><td>Thiolactomycin resistance due to gene dosage</td><td><a href="#ref250">250</a> </td><td>D
</td></tr>
<tr>
<td>emrB</td><td>E</td><td>Thiolactomycin resistance</td><td><a href="#ref250">250</a> </td><td> 
</td></tr>
<tr>
<td>envB</td><td>E, S</td><td>Mecillinam resistance</td><td><a href="#ref359">359</a> <a href="#ref587">587</a> </td><td> 
</td></tr>
<tr>
<td>envB</td><td>E</td><td>UV light resistance</td><td><a href="#ref24">24</a> </td><td> 
</td></tr>
<tr>
<td>envM</td><td>E, S</td><td>Diazoborine resistance</td><td><a href="#ref63">63</a> </td><td>S
</td></tr>
<tr>
<td>envZ</td><td>E</td><td>Ampicillin resistance</td><td><a href="#ref362">362</a> <a href="#ref363">363</a> </td><td> 
</td></tr>
<tr>
<td>envZ</td><td>E</td><td>Colicin L tolerance</td><td><a href="#ref812">812</a> </td><td> 
</td></tr>
<tr>
<td>envZ</td><td>E</td><td>Lambda phage resistance</td><td><a href="#ref812">812</a> </td><td> 
</td></tr>
<tr>
<td>envZ</td><td>E</td><td>Me1 phage resistance</td><td><a href="#ref801">801</a> </td><td> 
</td></tr>
<tr>
<td>envZ</td><td>E</td><td>TPI phage resistance</td><td><a href="#ref812">812</a> </td><td> 
</td></tr>
<tr>
<td>envZ</td><td>E</td><td>TuIa phage resistance</td><td><a href="#ref801">801</a> </td><td> 
</td></tr>
<tr>
<td>envZ</td><td>E</td><td>Freeze-thaw resistance</td><td><a href="#ref112">112</a> </td><td> 
</td></tr>
<tr>
<td>envZ</td><td>E</td><td>Colicin (multiple) resistance</td><td><a href="#ref626">626</a> </td><td> 
</td></tr>
<tr>
<td>eryC</td><td>E</td><td>Erythromycin resistance</td><td><a href="#ref599">599</a> </td><td> 
</td></tr>
<tr>
<td>eryD</td><td>E</td><td>Erythromycin resistance</td><td><a href="#ref830">830</a> </td><td> 
</td></tr>
<tr>
<td>exbB</td><td>E</td><td>Albomycin resistance</td><td><a href="#ref205">205</a> </td><td> 
</td></tr>
<tr>
<td>exbB</td><td>E</td><td>Colicin I resistance</td><td><a href="#ref283">283</a> <a href="#ref624">624</a> </td><td> 
</td></tr>
<tr>
<td>exbB</td><td>E</td><td>Colicin (other) resistance</td><td><a href="#ref283">283</a> <a href="#ref624">624</a> </td><td> 
</td></tr>
<tr>
<td>exbC</td><td>E</td><td>Colicin B resistance</td><td><a href="#ref624">624</a> </td><td> 
</td></tr>
<tr>
<td>exbC</td><td>E</td><td>Colicin I resistance</td><td><a href="#ref283">283</a> <a href="#ref624">624</a> </td><td> 
</td></tr>
<tr>
<td>exbC</td><td>E</td><td>Colicin (other) resistance</td><td><a href="#ref283">283</a> <a href="#ref624">624</a> </td><td> 
</td></tr>
<tr>
<td>exuR</td><td>E</td><td>Aldohexuronate resistance in <i>eda</i></td><td><a href="#ref612">612</a> </td><td> 
</td></tr>
<tr>
<td>exuR</td><td>E</td><td>Hexuronate as C course in noninducible <i>exuR eda</i></td><td><a href="#ref612">612</a> </td><td>C
</td></tr>
<tr>
<td>exuT</td><td>E</td><td>Aldohexuronate resistance in <i>eda</i></td><td><a href="#ref612">612</a> </td><td>L
</td></tr>
<tr>
<td>fadB</td><td>E</td><td>Thiolactomycin resistance (due to gene dosage)</td><td><a href="#ref778">778</a> </td><td>D
</td></tr>
<tr>
<td>fadL</td><td>E</td><td>T2 phage resistance in <i>ompF</i>+ or <i>ompF</i></td><td><a href="#ref70">70</a> <a href="#ref534">534</a> </td><td>L
</td></tr>
<tr>
<td>fadR</td><td>E</td><td>Decanoate as C source</td><td><a href="#ref713">713</a> </td><td>L
</td></tr>
<tr>
<td>fba</td><td>E</td><td>Streptozotocin resistance</td><td><a href="#ref456">456</a> </td><td>L
</td></tr>
<tr>
<td>fepABCDG</td><td>E</td><td>Colicin B resistance</td><td><a href="#ref623">623</a> <a href="#ref624">624</a> </td><td>L
</td></tr>
<tr>
<td>fepABDCG</td><td>E</td><td>Colicin D resistance</td><td><a href="#ref623">623</a> </td><td>L
</td></tr>
<tr>
<td>fhuA</td><td>E, S</td><td>Albomycin resistance</td><td><a href="#ref87">87</a> <a href="#ref387">387</a> <a href="#ref388">388</a> <a href="#ref478">478</a> <a href="#ref479">479</a> </td><td>S
</td></tr>
<tr>
<td>fhuA</td><td>S</td><td>ES18 phage resistance</td><td><a href="#ref87">87</a> <a href="#ref478">478</a> <a href="#ref479">479</a> </td><td>S
</td></tr>
<tr>
<td>fhuA</td><td>E</td><td>T1 phage resistance</td><td><a href="#ref93">93</a> <a href="#ref409">409</a> <a href="#ref870">870</a> </td><td>S, L
</td></tr>
<tr>
<td>fhuA</td><td>E</td><td>T5 phage resistance</td><td><a href="#ref93">93</a> <a href="#ref151">151</a> <a href="#ref387">387</a> <a href="#ref870">870</a> </td><td>L
</td></tr>
<tr>
<td>fhuA</td><td>E</td><td>Φ80 phage resistance</td><td><a href="#ref151">151</a> <a href="#ref387">387</a> <a href="#ref409">409</a> <a href="#ref870">870</a> </td><td>L, S
</td></tr>
<tr>
<td>fhuA</td><td>E</td><td>Colicin M resistance</td><td><a href="#ref151">151</a> <a href="#ref387">387</a> <a href="#ref409">409</a> </td><td>L, S
</td></tr>
<tr>
<td>fhuA</td><td>E</td><td>Beta-lactam conjugate resistance</td><td><a href="#ref93">93</a> </td><td>L
</td></tr>
<tr>
<td>fhuB</td><td>E</td><td>Albomycin resistance</td><td><a href="#ref618">618</a> </td><td>L
</td></tr>
<tr>
<td>fliC</td><td>E, S</td><td>Chi phage resistance</td><td><a href="#ref347">347</a> <a href="#ref349">349</a> <a href="#ref416">416</a> <a href="#ref417">417</a> <a href="#ref513">513</a> <a href="#ref710">710</a> </td><td> 
</td></tr>
<tr>
<td>fliC</td><td>S</td><td>Prolonged static cultivation</td><td><a href="#ref249">249</a> </td><td> 
</td></tr>
<tr>
<td>flrB</td><td>S</td><td>Trifluoroleucine resistance</td><td><a href="#ref115">115</a> <a href="#ref117">117</a> </td><td> 
</td></tr>
<tr>
<td>folA</td><td>E</td><td>Sulfonamide resistance</td><td><a href="#ref607">607</a> <a href="#ref697">697</a> </td><td>S, D
</td></tr>
<tr>
<td>folA</td><td>S</td><td>Trimethoprim resistance</td><td><a href="#ref405">405</a> <a href="#ref654">654</a> <a href="#ref718">718</a> <a href="#ref719">719</a> </td><td>S, D
</td></tr>
<tr>
<td>folP</td><td>E</td><td>Sulfonamide resistance</td><td><a href="#ref756">756</a> </td><td>S
</td></tr>
<tr>
<td>fruA</td><td>E</td><td>Deoxy(2)glucose-independent fructose utilization</td><td><a href="#ref11">11</a> </td><td> 
</td></tr>
<tr>
<td>fruA</td><td>E</td><td>Xylitol resistance</td><td><a href="#ref614">614</a> <a href="#ref639">639</a> </td><td>L
</td></tr>
<tr>
<td>fruA</td><td>E</td><td>Sorbose (L) resistance</td><td><a href="#ref715">715</a> </td><td>L
</td></tr>
<tr>
<td>fts (generic)</td><td>E</td><td>Filter retention</td><td><a href="#ref55">55</a> </td><td> 
</td></tr>
<tr>
<td>ftsZ</td><td>E</td><td>UV irradiation resistance in <i>lon</i></td><td><a href="#ref373">373</a> </td><td> 
</td></tr>
<tr>
<td>ftsZ</td><td>E</td><td>Methyl methanesulfonate resistance in <i>lon</i></td><td><a href="#ref373">373</a> </td><td> 
</td></tr>
<tr>
<td>ftsZ</td><td>E</td><td>Nitrofurantoin resistance in <i>lon</i></td><td><a href="#ref252">252</a> </td><td> 
</td></tr>
<tr>
<td>fucA</td><td>E</td><td>Propanediol as C source</td><td><a href="#ref287">287</a> </td><td> 
</td></tr>
<tr>
<td>fucA</td><td>E</td><td>Ethyleneglycol as C source in propanediol utilizer</td><td><a href="#ref79">79</a> </td><td> 
</td></tr>
<tr>
<td>fucA</td><td>E</td><td>Fucose (L) as C source in propanediol utilizer</td><td><a href="#ref287">287</a> </td><td> 
</td></tr>
<tr>
<td>fucA</td><td>E</td><td>Xylitol as C source in propanediol utilizer</td><td><a href="#ref848">848</a> </td><td> 
</td></tr>
<tr>
<td>fucA</td><td>E</td><td>Arabinose (D) as C source</td><td><a href="#ref450">450</a> </td><td> 
</td></tr>
<tr>
<td>fur</td><td>E</td><td>Manganese resistance</td><td><a href="#ref300">300</a> </td><td>C
</td></tr>
<tr>
<td>fusA</td><td>E</td><td>Fusidic acid resistance</td><td><a href="#ref65">65</a> <a href="#ref434">434</a> <a href="#ref695">695</a> <a href="#ref761">761</a> </td><td>S
</td></tr>
<tr>
<td>gabC</td><td>E</td><td>Aminobutyrate as N source</td><td><a href="#ref197">197</a> </td><td> 
</td></tr>
<tr>
<td>gadS</td><td>E</td><td>Glutamate as C source</td><td><a href="#ref292">292</a> </td><td>L
</td></tr>
<tr>
<td>gal operon</td><td>E</td><td>Galactose utilization in the presence of thiomethylglucoside</td><td><a href="#ref636">636</a> </td><td>D
</td></tr>
<tr>
<td>gal operon</td><td>S</td><td>Deoxy(2)galactose resistance</td><td><a href="#ref9">9</a> <a href="#ref552">552</a> </td><td>L
</td></tr>
<tr>
<td>gal operon</td><td>E</td><td>Galactose resistance in <i>galU</i></td><td><a href="#ref664">664</a> </td><td>L
</td></tr>
<tr>
<td>galC</td><td>S</td><td>Glucose as C source in ∆<i>ptsHI</i></td><td><a href="#ref613">613</a> </td><td> 
</td></tr>
<tr>
<td>galE</td><td>E, S</td><td>P1 phage infection of <i>Salmonella</i> sp. monitored by drug resistance</td><td><a href="#ref527">527</a> </td><td>L
</td></tr>
<tr>
<td>galE</td><td>E</td><td>U3 phage resistance</td><td><a href="#ref816">816</a> </td><td>L
</td></tr>
<tr>
<td>galE</td><td>S</td><td>Deoxy(2)galactose resistance</td><td><a href="#ref408">408</a> </td><td>L
</td></tr>
<tr>
<td>galE</td><td>S</td><td>Felix O phage resistance</td><td><a href="#ref334">334</a> <a href="#ref527">527</a> <a href="#ref591">591</a> <a href="#ref833">833</a> </td><td>L
</td></tr>
<tr>
<td>galE</td><td>S</td><td>P22 phage resistance</td><td><a href="#ref334">334</a> <a href="#ref527">527</a> <a href="#ref591">591</a> <a href="#ref833">833</a> </td><td>L
</td></tr>
<tr>
<td>galK</td><td>S</td><td>Deoxy(2)galactose resistance</td><td><a href="#ref408">408</a> </td><td>L
</td></tr>
<tr>
<td>galK</td><td>E, S</td><td>Galactose resistance in <i>galE</i></td><td><a href="#ref569">569</a> <a href="#ref752">752</a> </td><td>L
</td></tr>
<tr>
<td>galK</td><td>E</td><td>Galactose resistance in <i>galT</i></td><td><a href="#ref859">859</a> </td><td>L
</td></tr>
<tr>
<td>galK</td><td>E</td><td>Galactose resistance in <i>galU</i></td><td><a href="#ref664">664</a> </td><td>L
</td></tr>
<tr>
<td>galP</td><td>S</td><td>Deoxy(2)galactose resistance</td><td><a href="#ref552">552</a> </td><td>L
</td></tr>
<tr>
<td>galR</td><td>E</td><td>Thiomethylgalactoside-independent utilization of galactose in <i>galR</i>(Con)</td><td><a href="#ref108">108</a> </td><td> 
</td></tr>
<tr>
<td>galR</td><td>S</td><td>Glucose utilization in ∆<i>ptsHI</i></td><td><a href="#ref613">613</a> </td><td> 
</td></tr>
<tr>
<td>galR</td><td>E</td><td>Galactose resistance in <i>galT</i>+<i>K</i>+<i>E</i>+/<i>galT</i>+<i>K</i>+<i>E</i>+ <i>galU</i></td><td><a href="#ref664">664</a> </td><td> 
</td></tr>
<tr>
<td>galT</td><td>E</td><td>Streptozotocin resistance</td><td><a href="#ref456">456</a> </td><td>L
</td></tr>
<tr>
<td>galU</td><td>E</td><td>Ampicillin resistance</td><td><a href="#ref212">212</a> </td><td>L
</td></tr>
<tr>
<td>galU</td><td>E</td><td>P1 phage resistance</td><td><a href="#ref242">242</a> </td><td>L
</td></tr>
<tr>
<td>galU</td><td>E</td><td>U3 phage resistance</td><td><a href="#ref816">816</a> </td><td>L
</td></tr>
<tr>
<td>galU</td><td>S</td><td>Felix O phage resistance</td><td><a href="#ref334">334</a> </td><td>L
</td></tr>
<tr>
<td>garA</td><td>S</td><td>Gamma irradiation resistance</td><td><a href="#ref346">346</a> </td><td> 
</td></tr>
<tr>
<td>garA</td><td>E</td><td>Glucarate (D) resistance in <i>ppc</i></td><td><a href="#ref647">647</a> </td><td>L
</td></tr>
<tr>
<td>garB</td><td>S</td><td>Gamma irradiation resistance</td><td><a href="#ref346">346</a> </td><td> 
</td></tr>
<tr>
<td>gatACD</td><td>E</td><td>Fucitol resistance in galactitol-utilizing strains</td><td><a href="#ref184">184</a> </td><td>L
</td></tr>
<tr>
<td>gatACD</td><td>E</td><td>Deoxy(2)galactitol resistance in galactitol-utilizing strains</td><td><a href="#ref184">184</a> </td><td>L
</td></tr>
<tr>
<td>gatACD</td><td>E</td><td>Arabitol utilization</td><td><a href="#ref466">466</a> <a href="#ref845">845</a> </td><td> 
</td></tr>
<tr>
<td>gatACD</td><td>E</td><td>Ribitol utilization</td><td><a href="#ref466">466</a> <a href="#ref845">845</a> </td><td> 
</td></tr>
<tr>
<td>gatACD</td><td>E</td><td>Arbitol (D) resistance</td><td><a href="#ref639">639</a> </td><td>L
</td></tr>
<tr>
<td>gleR</td><td>S</td><td>Glycylleucine resistance in <i>ilv</i></td><td><a href="#ref580">580</a> </td><td> 
</td></tr>
<tr>
<td>glnA</td><td>S</td><td>Histidine(D) + glycylglutamine satisfaction of <i>his</i> auxotrophs</td><td><a href="#ref819">819</a> </td><td>L
</td></tr>
<tr>
<td>glnA</td><td>E</td><td>Methylammonium resistance</td><td><a href="#ref694">694</a> </td><td>S
</td></tr>
<tr>
<td>glnA</td><td>E</td><td>Methionine sulfoxamine resistance in <i>glnG</i></td><td><a href="#ref592">592</a> </td><td>C
</td></tr>
<tr>
<td>glnA</td><td>S</td><td>Glutamyl(γ)hydrazide resistance</td><td><a href="#ref518">518</a> <a href="#ref871">871</a> </td><td>S
</td></tr>
<tr>
<td>glnF</td><td>S</td><td>Glutamyl(γ)hydrazide resistance</td><td><a href="#ref433">433</a> </td><td>L
</td></tr>
<tr>
<td>glnH</td><td>S</td><td>433</td><td><a href="#refL">L</a> </td><td>Glutamyl(γ)hydrazide resistance
</td></tr>
<tr>
<td>glnP</td><td>E</td><td>Glutamine as C source</td><td><a href="#ref433">433</a> <a href="#ref501">501</a> <a href="#ref820">820</a> </td><td> 
</td></tr>
<tr>
<td>glnP</td><td>E, S</td><td>Methionine sulfoxamine resistance</td><td><a href="#ref37">37</a> <a href="#ref501">501</a> </td><td>L
</td></tr>
<tr>
<td>glnP</td><td>E</td><td>Glutamyl(γ)hydrazide resistance</td><td><a href="#ref501">501</a> <a href="#ref821">821</a> </td><td>L
</td></tr>
<tr>
<td>glnP</td><td>E</td><td>Glutamine (D) resistance</td><td><a href="#ref501">501</a> </td><td>L
</td></tr>
<tr>
<td>glnV</td><td>E</td><td>Caffeine resistance</td><td><a href="#ref185">185</a> </td><td>S
</td></tr>
<tr>
<td>glpD</td><td>E</td><td>Streptozotocin resistance</td><td><a href="#ref456">456</a> </td><td>L
</td></tr>
<tr>
<td>glpF</td><td>E</td><td>Glyceraldehyde (L) resistance</td><td><a href="#ref762">762</a> </td><td>L
</td></tr>
<tr>
<td>glpK</td><td>E</td><td>Glyceraldehyde (L) resistance</td><td><a href="#ref762">762</a> </td><td>L
</td></tr>
<tr>
<td>glpK</td><td>E, S</td><td>Methyl(α)glucoside-resistant glycerol utilization in <i>ptsI</i></td><td><a href="#ref64">64</a> <a href="#ref665">665</a> </td><td> 
</td></tr>
<tr>
<td>glpT</td><td>E, S</td><td>Fosfomycin resistance</td><td><a href="#ref10">10</a> <a href="#ref29">29</a> <a href="#ref313">313</a> <a href="#ref707">707</a> <a href="#ref800">800</a> </td><td>L
</td></tr>
<tr>
<td>glpT</td><td>E</td><td>Glyceraldehyde (DL) 3-phosphate resistance</td><td><a href="#ref762">762</a> </td><td>L
</td></tr>
<tr>
<td>glpT</td><td>E</td><td>Arsenate resistance</td><td><a href="#ref838">838</a> </td><td>L
</td></tr>
<tr>
<td>glpT</td><td>E</td><td>Dihydroxybutyl phosphonate resistance</td><td><a href="#ref285">285</a> <a href="#ref453">453</a> </td><td>L
</td></tr>
<tr>
<td>glpT</td><td>E</td><td>Fluorohydroxyacetone phosphate resistance</td><td><a href="#ref515">515</a> </td><td>L
</td></tr>
<tr>
<td>glpT</td><td>E</td><td>Chlorohydroxyacetone phosphate resistance</td><td><a href="#ref515">515</a> </td><td>L
</td></tr>
<tr>
<td>glpT</td><td>E</td><td>Glycerol 3-phosphorothioate resistance</td><td><a href="#ref295">295</a> </td><td>L
</td></tr>
<tr>
<td>gltA</td><td>E</td><td>Improved growth in <i>icd</i></td><td><a href="#ref435">435</a> </td><td>L
</td></tr>
<tr>
<td>gltH</td><td>E</td><td>Glutamate as C source</td><td><a href="#ref496">496</a> <a href="#ref497">497</a> <a href="#ref687">687</a> </td><td> 
</td></tr>
<tr>
<td>gltR</td><td>E</td><td>Glutamate as C source at 42°C</td><td><a href="#ref497">497</a> </td><td> 
</td></tr>
<tr>
<td>gltS</td><td>E</td><td>Glutamate as C source</td><td><a href="#ref496">496</a> <a href="#ref687">687</a> </td><td>Q
</td></tr>
<tr>
<td>gltS</td><td>E</td><td>Glutamate as C source</td><td><a href="#ref496">496</a> <a href="#ref497">497</a> <a href="#ref687">687</a> </td><td> 
</td></tr>
<tr>
<td>gltS</td><td>E</td><td>Glutamate (D) resistance in <i>gltS</i> (increased)</td><td><a href="#ref525">525</a> <a href="#ref716">716</a> </td><td>L
</td></tr>
<tr>
<td>gltS</td><td>E</td><td>Methyl(α)glutamate resistance in <i>gltS</i> (increased)</td><td><a href="#ref391">391</a> <a href="#ref525">525</a> <a href="#ref716">716</a> </td><td>L
</td></tr>
<tr>
<td>gltS</td><td>E</td><td>Homocysteic acid resistance</td><td><a href="#ref213">213</a> </td><td>L
</td></tr>
<tr>
<td>gltX</td><td>E</td><td>Glutamyl-γ-methyl ester resistance</td><td><a href="#ref425">425</a> </td><td> 
</td></tr>
<tr>
<td>glyA</td><td>E</td><td>Serine + methionine + glycine resistance in <i>relA</i></td><td><a href="#ref783">783</a> </td><td> 
</td></tr>
<tr>
<td>gntM</td><td>E</td><td>Glycerol + gluconate as C source in <i>eda</i></td><td><a href="#ref217">217</a> </td><td>L
</td></tr>
<tr>
<td>gntR</td><td>E</td><td>Acetate + gluconate as C source in <i>ppc</i></td><td><a href="#ref39">39</a> </td><td> 
</td></tr>
<tr>
<td>gntS</td><td>E</td><td>Acetate + gluconate as C source in <i>ppc</i></td><td><a href="#ref39">39</a> </td><td>L
</td></tr>
<tr>
<td>gor</td><td>E</td><td>Methyl viologen tolerance</td><td><a href="#ref430">430</a> </td><td>L
</td></tr>
<tr>
<td>gprAB</td><td>E</td><td>Lambda phage resistance</td><td><a href="#ref578">578</a> <a href="#ref670">670</a> </td><td> 
</td></tr>
<tr>
<td>gpt</td><td>S</td><td>Aza(8)guanine resistance</td><td><a href="#ref272">272</a> <a href="#ref764">764</a> </td><td>L
</td></tr>
<tr>
<td>gpt</td><td>E</td><td>Sulfanilamide resistance in presence of guanine</td><td><a href="#ref97">97</a> </td><td>L
</td></tr>
<tr>
<td>groM</td><td>E</td><td>T7 phage resistance</td><td><a href="#ref427">427</a> </td><td> 
</td></tr>
<tr>
<td>grpD</td><td>E</td><td>Lambda mutant prophage induction, survival of</td><td><a href="#ref667">667</a> </td><td> 
</td></tr>
<tr>
<td>grpE</td><td>E</td><td>Lambda mutant prophage induction, survival of</td><td><a href="#ref667">667</a> </td><td> 
</td></tr>
<tr>
<td>gsh (generic)</td><td>E</td><td>Nitrosoguanidine resistance</td><td><a href="#ref693">693</a> </td><td>L
</td></tr>
<tr>
<td>gsh (generic)</td><td>E</td><td>Methylglyoxal resistance</td><td><a href="#ref548">548</a> </td><td> 
</td></tr>
<tr>
<td>gshAB</td><td>E, S</td><td>Selenite resistance</td><td><a href="#ref423">423</a> </td><td>L
</td></tr>
<tr>
<td>guaAB</td><td>E</td><td>Psicofuranine resistance</td><td><a href="#ref779">779</a> </td><td> 
</td></tr>
<tr>
<td>guaB</td><td>S</td><td>Fluoro(5)uracil + fluoro(5)uridine resistance</td><td><a href="#ref367">367</a> </td><td>L
</td></tr>
<tr>
<td>gurBCD</td><td>E</td><td>Glycerol + methylglucuronide a C source in <i>eda</i></td><td><a href="#ref572">572</a> </td><td>L
</td></tr>
<tr>
<td>gyrA</td><td>E, S</td><td>Triazolealanine + aminotriazole resistance</td><td><a href="#ref660">660</a> </td><td>S
</td></tr>
<tr>
<td>gyrA</td><td>E, S</td><td>Nalidixic acid resistance</td><td><a href="#ref200">200</a> <a href="#ref298">298</a> </td><td>S
</td></tr>
<tr>
<td>gyrA</td><td>E</td><td>Quinolone resistance</td><td><a href="#ref860">860</a> </td><td>S
</td></tr>
<tr>
<td>gyrA</td><td>E</td><td>Oxolinic acid resistance</td><td><a href="#ref734">734</a> </td><td>S
</td></tr>
<tr>
<td>gyrA</td><td>E</td><td>Glucosides (β) as C source</td><td><a href="#ref194">194</a> </td><td> 
</td></tr>
<tr>
<td>gyrA</td><td>S</td><td>Thermotolerance</td><td><a href="#ref200">200</a> </td><td> 
</td></tr>
<tr>
<td>gyrB</td><td>S</td><td>Triaolealanine + aminotriazole resistance</td><td><a href="#ref660">660</a> </td><td>S
</td></tr>
<tr>
<td>gyrB</td><td>E</td><td>Aminotriazole resistance in <i>relA</i></td><td><a href="#ref772">772</a> </td><td>S
</td></tr>
<tr>
<td>gyrB</td><td>E</td><td>Novobiocin resistance</td><td><a href="#ref255">255</a> </td><td>S
</td></tr>
<tr>
<td>gyrB</td><td>E</td><td>Coumermycin resistance</td><td><a href="#ref181">181</a> <a href="#ref255">255</a> <a href="#ref593">593</a> </td><td>D, S
</td></tr>
<tr>
<td>gyrB</td><td>E</td><td>Chlorobiocin resistance</td><td><a href="#ref589">589</a> </td><td> 
</td></tr>
<tr>
<td>gyrB</td><td>E</td><td>Glucosides (β) as C source</td><td><a href="#ref194">194</a> </td><td> 
</td></tr>
<tr>
<td>gyrB</td><td>E</td><td>Lambda mutant infection resistance</td><td><a href="#ref521">521</a> </td><td> 
</td></tr>
<tr>
<td>gyrB</td><td>E</td><td>Nalidixic acid resistance</td><td><a href="#ref333">333</a> <a href="#ref855">855</a> </td><td>S
</td></tr>
<tr>
<td>hem</td><td>E</td><td>Streptomycin resistance</td><td><a href="#ref57">57</a> </td><td>L</td></tr>
<tr>
<td>hemA</td><td>E</td><td>Kanamycin resistance</td><td><a href="#ref585">585</a> </td><td>L
</td></tr>
<tr>
<td>hemA</td><td>E, S</td><td>Neomycin resistance</td><td><a href="#ref683">683</a> </td><td>L
</td></tr>
<tr>
<td>hemB</td><td>E</td><td>Kanamycin resistance</td><td><a href="#ref585">585</a> </td><td>L
</td></tr>
<tr>
<td>hemB</td><td>E</td><td>Kanamycin-resistant, hemin-supported growth of <i>hemA</i></td><td><a href="#ref507">507</a> </td><td>L
</td></tr>
<tr>
<td>hemB</td><td>E</td><td>Colicin K-type tolerance</td><td><a href="#ref83">83</a> </td><td> 
</td></tr>
<tr>
<td>hemC</td><td>S</td><td>Neomycin resistance</td><td><a href="#ref682">682</a> </td><td>L
</td></tr>
<tr>
<td>hemD</td><td>S</td><td>Neomycin resistance</td><td><a href="#ref681">681</a> </td><td>L
</td></tr>
<tr>
<td>hemE</td><td>S</td><td>Neomycin resistance</td><td><a href="#ref189">189</a> <a href="#ref680">680</a> </td><td>L
</td></tr>
<tr>
<td>hemBCF</td><td>E</td><td>Plate method</td><td><a href="#ref506">506</a> </td><td>L
</td></tr>
<tr>
<td>hemG</td><td>E</td><td>Neomycin resistance</td><td><a href="#ref679">679</a> </td><td>L
</td></tr>
<tr>
<td>hemL</td><td>E</td><td>Kanamycin resistance</td><td><a href="#ref350">350</a> </td><td>L
</td></tr>
<tr>
<td>hflC</td><td>E</td><td>Lambda mutant phage resistance</td><td><a href="#ref56">56</a> </td><td> 
</td></tr>
<tr>
<td>hflK</td><td>E</td><td>Lambda mutant phage resistance</td><td><a href="#ref56">56</a> </td><td> 
</td></tr>
<tr>
<td>hflX</td><td>E</td><td>Lambda mutant phage resistance</td><td><a href="#ref56">56</a> </td><td> 
</td></tr>
<tr>
<td>himA</td><td>E</td><td>Mu prophage induction, survival of</td><td><a href="#ref520">520</a> <a href="#ref522">522</a> </td><td> 
</td></tr>
<tr>
<td>himA</td><td>E</td><td>Mu phage lytic growth resistance</td><td><a href="#ref81">81</a> </td><td> 
</td></tr>
<tr>
<td>himA</td><td>E</td><td>Lambda mutant phage infection, survival of</td><td><a href="#ref519">519</a> <a href="#ref521">521</a> </td><td> 
</td></tr>
<tr>
<td><i>himA (himC)</i></td><td>E</td><td>Lambda mutant phage resistance</td><td><a href="#ref519">519</a> <a href="#ref521">521</a> </td><td> 
</td></tr>
<tr>
<td>himD</td><td>E</td><td>Mu prophage induction, survival of</td><td><a href="#ref520">520</a> <a href="#ref522">522</a> </td><td> 
</td></tr>
<tr>
<td>himD</td><td>E</td><td>Mu phage lytic growth resistance</td><td><a href="#ref81">81</a> </td><td> 
</td></tr>
<tr>
<td>hipA</td><td>E</td><td>Cycloserine resistance</td><td><a href="#ref541">541</a> </td><td> 
</td></tr>
<tr>
<td>hipA</td><td>E</td><td>Ampicillin resistance</td><td><a href="#ref541">541</a> </td><td> 
</td></tr>
<tr>
<td>hipA</td><td>E</td><td>Fosfomycin resistance</td><td><a href="#ref541">541</a> </td><td> 
</td></tr>
<tr>
<td>hipQ</td><td>E</td><td>Norfloxacin resistance</td><td><a href="#ref842">842</a> </td><td> 
</td></tr>
<tr>
<td>hipQ</td><td>E</td><td>Ampicillin resistance</td><td><a href="#ref842">842</a> </td><td> 
</td></tr>
<tr>
<td>his structural genes</td><td>E, S</td><td>Triazolealanine + aminotriazole resistance</td><td><a href="#ref98">98</a> <a href="#ref126">126</a> <a href="#ref658">658</a> </td><td>C, Q
</td></tr>
<tr>
<td>his structural genes</td><td>S</td><td>Aminotriazole resistance</td><td><a href="#ref20">20</a> <a href="#ref21">21</a> </td><td>D
</td></tr>
<tr>
<td>his structural genes</td><td>S</td><td>Growth at 42°C in <i>his</i>-overexpressing strain</td><td><a href="#ref804">804</a> </td><td>Q
</td></tr>
<tr>
<td>his structural genes</td><td>S</td><td>Histidinal satisfaction of <i>his</i> auxotroph requirement at 30°C</td><td><a href="#ref374">374</a> <a href="#ref375">375</a> </td><td>Q
</td></tr>
<tr>
<td>his structural genes</td><td>S</td><td>Histidine (D) satisfaction of <i>his</i> auxotroph requirement</td><td><a href="#ref374">374</a> <a href="#ref375">375</a> </td><td>Q
</td></tr>
<tr>
<td>hisG</td><td>E, S</td><td>Thiazolealanine resistance</td><td><a href="#ref540">540</a> <a href="#ref698">698</a> <a href="#ref824">824</a> </td><td>S, FBI
</td></tr>
<tr>
<td>hisG</td><td>S</td><td>Hydrazino (α) imidazole propionic acid resistance</td><td><a href="#ref699">699</a> </td><td>S, FBI
</td></tr>
<tr>
<td>hisG</td><td>S</td><td>Histidinal satisfaction of <i>his</i> auxotroph requirement at 30°C</td><td><a href="#ref374">374</a> <a href="#ref375">375</a> </td><td>L
</td></tr>
<tr>
<td>hisG</td><td>S</td><td>Histidine (D) satisfaction of <i>his</i> auxotroph requirement</td><td><a href="#ref374">374</a> <a href="#ref375">375</a> </td><td>L
</td></tr>
<tr>
<td>hisG</td><td>E</td><td>Histidine-resistant adenine → guanine nucleotide</td><td><a href="#ref33">33</a> </td><td>S,FBI</td></tr>
<tr>
<td>hisJ</td><td>S</td><td>Hydrazino (α) imidazole propionic acid resistance</td><td><a href="#ref13">13</a> <a href="#ref15">15</a> </td><td>L
</td></tr>
<tr>
<td>hisJ</td><td>S</td><td>Azaserine + tryptophan resistance in <i>dhuA</i></td><td><a href="#ref15">15</a> <a href="#ref432">432</a> </td><td> 
</td></tr>
<tr>
<td>hisP</td><td>S</td><td>Hydrazino (α) imidazole propionic acid resistance in <i>dhuA</i></td><td><a href="#ref14">14</a> <a href="#ref15">15</a> </td><td>L
</td></tr>
<tr>
<td>hisP</td><td>S</td><td>Arginine as N source</td><td><a href="#ref13">13</a> <a href="#ref15">15</a> <a href="#ref432">432</a> </td><td>D
</td></tr>
<tr>
<td>hisQ</td><td>S</td><td>Hydrazino (α) imidazole propionic acid resistance</td><td><a href="#ref13">13</a> <a href="#ref15">15</a> </td><td>L
</td></tr>
<tr>
<td>hisR</td><td>S</td><td>Triazolealanine + aminotriazole resistance</td><td><a href="#ref658">658</a> </td><td>Q
</td></tr>
<tr>
<td>hisR</td><td>S</td><td>Thiazolealanine resistance</td><td><a href="#ref706">706</a> </td><td>Q
</td></tr>
<tr>
<td>hisR</td><td>S</td><td>Methyl(2)histidine resistance + aminotriazole resistance</td><td><a href="#ref658">658</a> </td><td>Q
</td></tr>
<tr>
<td>hisS</td><td>S</td><td>Triazolealanine + aminotriazole resistance</td><td><a href="#ref658">658</a> </td><td>S
</td></tr>
<tr>
<td>hisS</td><td>S</td><td>Thiazolealanine resistance</td><td><a href="#ref657">657</a> </td><td>S
</td></tr>
<tr>
<td>hisT</td><td>E, S</td><td>Triazolealanine + aminotriazole resistance</td><td><a href="#ref98">98</a> <a href="#ref148">148</a> <a href="#ref658">658</a> <a href="#ref705">705</a> <a href="#ref784">784</a> </td><td>L
</td></tr>
<tr>
<td>hisT</td><td>E</td><td>Azaleucine resistance</td><td><a href="#ref98">98</a> <a href="#ref784">784</a> </td><td>L
</td></tr>
<tr>
<td>hisT</td><td>E</td><td>Serine resistance</td><td><a href="#ref98">98</a> <a href="#ref784">784</a> </td><td>L
</td></tr>
<tr>
<td>hisT</td><td>S</td><td>Amino(3)tyrosine resistance</td><td><a href="#ref148">148</a> <a href="#ref658">658</a> <a href="#ref705">705</a> </td><td>L
</td></tr>
<tr>
<td>hisT</td><td>S</td><td>Thialysine resistance</td><td><a href="#ref148">148</a> <a href="#ref658">658</a> <a href="#ref705">705</a> </td><td>L
</td></tr>
<tr>
<td>hisT</td><td>S</td><td>Trifluoroleucine resistance</td><td><a href="#ref148">148</a> <a href="#ref658">658</a> <a href="#ref705">705</a> </td><td>L
</td></tr>
<tr>
<td>hisT</td><td>S</td><td>Norleucine resistance</td><td><a href="#ref148">148</a> <a href="#ref658">658</a> <a href="#ref705">705</a> </td><td>L
</td></tr>
<tr>
<td>hisT</td><td>S</td><td>Hydroxy (β) leucine resistance</td><td><a href="#ref148">148</a> <a href="#ref658">658</a> <a href="#ref705">705</a> </td><td>L
</td></tr>
<tr>
<td>hisT</td><td>E</td><td>Phenyl galactoside resistance of <i>lacP lacI</i> (nonsense) <i>supE</i></td><td><a href="#ref747">747</a> </td><td> 
</td></tr>
<tr>
<td>hisU</td><td>S, E</td><td>Triazolealanine + aminotriazole resistance</td><td><a href="#ref23">23</a> <a href="#ref660">660</a> </td><td> 
</td></tr>
<tr>
<td>hisU</td><td>S</td><td>Nalidixic acid resistance</td><td><a href="#ref593">593</a> <a href="#ref660">660</a> </td><td> 
</td></tr>
<tr>
<td>hisW</td><td>S, E</td><td>Triazolealanine + aminotriazole resistance</td><td><a href="#ref23">23</a> </td><td> 
</td></tr>
<tr>
<td>hisW</td><td>S</td><td>Coumermycin resistance</td><td><a href="#ref593">593</a> <a href="#ref660">660</a> </td><td> 
</td></tr>
<tr>
<td>hpt</td><td>E</td><td>Mercapto(6)purine resistance in <i>gpt</i></td><td><a href="#ref371">371</a> </td><td> 
</td></tr>
<tr>
<td>hpt</td><td>E</td><td>Sulfanilamide + hypoxanthine resistance</td><td><a href="#ref97">97</a> </td><td>L
</td></tr>
<tr>
<td>hsd</td><td>E</td><td>Lambda <i>vir</i> resistance after infection with</td><td><a href="#ref183">183</a> </td><td>L</td></tr>
<tr>
<td>hsd</td><td>S</td><td>Lactose utilization after conjugation of <i>F</i>′<i>lac</i> into <i>Salmonella</i> sp.</td><td><a href="#ref103">103</a> </td><td>L
</td></tr>
<tr>
<td>hut</td><td>S</td><td>Histidine as sole N or C source</td><td><a href="#ref509">509</a> </td><td> 
</td></tr>
<tr>
<td>icdE</td><td>E, S</td><td>Nalidixic acid resistance</td><td><a href="#ref91">91</a> <a href="#ref311">311</a> </td><td>L
</td></tr>
<tr>
<td>iclR</td><td>E</td><td>Butyrate or valerate as better C source</td><td><a href="#ref602">602</a> </td><td>C
</td></tr>
<tr>
<td>iclR</td><td>E</td><td>Glucose or glycerol as C source in <i>ppc</i></td><td><a href="#ref488">488</a> </td><td> 
</td></tr>
<tr>
<td>ileR</td><td>E</td><td>Trifluoroleucine resistance</td><td><a href="#ref412">412</a> </td><td> 
</td></tr>
<tr>
<td>ileS</td><td>E</td><td>Pseudomonic acid resistance</td><td><a href="#ref858">858</a> </td><td>S
</td></tr>
<tr>
<td>ileS</td><td>E</td><td>Thiaisoleucine resistance</td><td><a href="#ref757">757</a> </td><td>S
</td></tr>
<tr>
<td>ilv structural genes</td><td>S</td><td>Sulfometuron methyl + valine resistance</td><td><a href="#ref872">872</a> </td><td>C
</td></tr>
<tr>
<td>ilv structural genes</td><td>E</td><td>Aminobutyrate resistance</td><td><a href="#ref633">633</a> <a href="#ref780">780</a> </td><td>C
</td></tr>
<tr>
<td>ilv structural genes</td><td>E</td><td>Valine resistance</td><td><a href="#ref268">268</a> </td><td>C
</td></tr>
<tr>
<td>ilv structural genes</td><td>S</td><td>Cyclopentane glycine resistance</td><td><a href="#ref586">586</a> </td><td>C
</td></tr>
<tr>
<td>ilvA</td><td>S</td><td>Cyclopentane glycine resistance</td><td><a href="#ref586">586</a> </td><td>S, FBI
</td></tr>
<tr>
<td>ilvA</td><td>E</td><td>Glyclleucine resistance</td><td><a href="#ref806">806</a> </td><td>S, FBI
</td></tr>
<tr>
<td>ilvA</td><td>S</td><td>Threonine as sole N source</td><td><a href="#ref106">106</a> </td><td>S, FBI
</td></tr>
<tr>
<td>ilvA</td><td>E</td><td>Growth on minimal medium of <i>ilvA+</i> (multicopy) <i>ilvDC</i> <i>hisT</i> overcoming 2-ketobutyrate accumulation</td><td><a href="#ref231">231</a> </td><td>L
</td></tr>
<tr>
<td>ilvB</td><td>E</td><td>Aminobutyrate resistance</td><td><a href="#ref633">633</a> <a href="#ref780">780</a> </td><td>S, FBI
</td></tr>
<tr>
<td>ilvB</td><td>E</td><td>Valine resistance</td><td><a href="#ref633">633</a> <a href="#ref755">755</a> </td><td>S, FBI
</td></tr>
<tr>
<td>ilvB</td><td>E</td><td>Glyclleucine resistance</td><td><a href="#ref269">269</a> </td><td>S
</td></tr>
<tr>
<td>ilvB</td><td>E</td><td>Phenylalanylleucine resistance</td><td><a href="#ref269">269</a> </td><td>S
</td></tr>
<tr>
<td>ilvF</td><td>E</td><td>Valine resistance</td><td><a href="#ref174">174</a> <a href="#ref610">610</a> </td><td>G
</td></tr>
<tr>
<td>ilvG</td><td>E, S</td><td>Sulfometuron methyl + valine resistance</td><td><a href="#ref441">441</a> <a href="#ref851">851</a> <a href="#ref872">872</a> </td><td>S, Q
</td></tr>
<tr>
<td>ilvG</td><td>E</td><td>Valine resistance</td><td><a href="#ref221">221</a> <a href="#ref448">448</a> <a href="#ref720">720</a> </td><td>G
</td></tr>
<tr>
<td>ilvH</td><td>E</td><td>Aminobutyrate resistance</td><td><a href="#ref633">633</a> <a href="#ref780">780</a> </td><td>S, FBI
</td></tr>
<tr>
<td>ilvH</td><td>E</td><td>Valine resistance</td><td><a href="#ref633">633</a> <a href="#ref755">755</a> </td><td>S, FBI
</td></tr>
<tr>
<td>ilvH</td><td>E</td><td>Glyclleucine resistance</td><td><a href="#ref269">269</a> </td><td>S
</td></tr>
<tr>
<td>ilvH</td><td>E</td><td>Phenylalanylleucine resistance</td><td><a href="#ref269">269</a> </td><td>S
</td></tr>
<tr>
<td>ilvI</td><td>E</td><td>Aminobutyrate resistance</td><td><a href="#ref633">633</a> <a href="#ref780">780</a> </td><td>S
</td></tr>
<tr>
<td>ilvI</td><td>E</td><td>Valine resistance</td><td><a href="#ref633">633</a> <a href="#ref755">755</a> </td><td> 
</td></tr>
<tr>
<td>ilvI</td><td>E</td><td>Glyclleucine resistance</td><td><a href="#ref269">269</a> </td><td> 
</td></tr>
<tr>
<td>ilvI</td><td>E</td><td>Phenylalanylleucine resistance</td><td><a href="#ref269">269</a> </td><td> 
</td></tr>
<tr>
<td>ilvJ</td><td>E</td><td>Valine resistance</td><td><a href="#ref174">174</a> <a href="#ref361">361</a> <a href="#ref649">649</a> </td><td>G
</td></tr>
<tr>
<td>ilvU</td><td>E</td><td>Thiaisoleucine resistance</td><td><a href="#ref222">222</a> </td><td> 
</td></tr>
<tr>
<td>kdgR</td><td>E</td><td>Keto(2)-3-deoxygluconate as C source</td><td><a href="#ref377">377</a> <a href="#ref615">615</a> </td><td>C
</td></tr>
<tr>
<td>kdsA</td><td>S</td><td>9NA phage resistance in <i>galE</i></td><td><a href="#ref452">452</a> <a href="#ref641">641</a> </td><td> 
</td></tr>
<tr>
<td>ksgA</td><td>E</td><td>Kasugamycin resistance</td><td><a href="#ref312">312</a> <a href="#ref726">726</a> <a href="#ref793">793</a> </td><td>L
</td></tr>
<tr>
<td>ksgB</td><td>E</td><td>Kasugamycin resistance</td><td><a href="#ref238">238</a> <a href="#ref726">726</a> </td><td> 
</td></tr>
<tr>
<td>ksgC</td><td>E</td><td>Kasugamycin resistance</td><td><a href="#ref861">861</a> </td><td> 
</td></tr>
<tr>
<td>ksgD</td><td>E</td><td>Kasugamycin resistance</td><td><a href="#ref238">238</a> </td><td> 
</td></tr>
<tr>
<td>lac</td><td>E</td><td>Growth in lactose-limited chemostats</td><td><a href="#ref330">330</a> </td><td>D
</td></tr>
<tr>
<td>lacI,operator</td><td>E</td><td>Methyl(α)glucoside-resistant utilization of lactose in <i>ptsH</i></td><td><a href="#ref665">665</a> </td><td>C</td></tr>
<tr>
<td>lacI,operator</td><td>E</td><td>Raffinose as C source</td><td><a href="#ref459">459</a> </td><td>C
</td></tr>
<tr>
<td>lacI,operator</td><td>E</td><td>Neolactose as C source</td><td><a href="#ref459">459</a> </td><td>C
</td></tr>
<tr>
<td>lacI,operator</td><td>E</td><td>Phenyl galactoside as C source</td><td><a href="#ref523">523</a> </td><td>C
</td></tr>
<tr>
<td>lacI,operator</td><td>E</td><td>Lactobionic acid as C source</td><td><a href="#ref440">440</a> </td><td>C
</td></tr>
<tr>
<td>lacI,operator</td><td>E</td><td>Lactitol as C source</td><td><a href="#ref457">457</a> </td><td>C
</td></tr>
<tr>
<td>lacI,operator</td><td>E</td><td>Melibiose as C source</td><td><a href="#ref475">475</a> </td><td>C
</td></tr>
<tr>
<td>lacIoperator</td><td>E</td><td>Melibiose as C source and acetyl(<i>N</i>)lactonate as N source</td><td><a href="#ref475">475</a> </td><td>C
</td></tr>
<tr>
<td>lacI,operator</td><td>E</td><td>Lactobionic acid as C source</td><td><a href="#ref474">474</a> </td><td> 
</td></tr>
<tr>
<td>lacI,operator</td><td>E</td><td>Acetyl(<i>N</i>)lactonate as N source</td><td><a href="#ref474">474</a> </td><td>Q
</td></tr>
<tr>
<td>lacI,operator</td><td>E</td><td>Nitro(<i>o</i>)-phenylthiogalactoside resistance in <i>lacI</i>(Con) or <i>lacO</i>(Con)</td><td><a href="#ref543">543</a> <a href="#ref571">571</a> </td><td>Q
</td></tr>
<tr>
<td>lacI,operator</td><td>E</td><td>Methyl(α)glucoside-resistant lactose utilization</td><td><a href="#ref406">406</a> </td><td>C
</td></tr>
<tr>
<td>lacY</td><td>E</td><td>Nitro(<i>o</i>)-phenylthiogalactoside resistance</td><td><a href="#ref523">523</a> <a href="#ref543">543</a> <a href="#ref721">721</a> </td><td>L
</td></tr>
<tr>
<td>lacY</td><td>E</td><td>Phenyl galactoside resistance</td><td><a href="#ref376">376</a> </td><td>L
</td></tr>
<tr>
<td>lacY</td><td>E</td><td>Phenethylyl galactoside resistance</td><td><a href="#ref376">376</a> </td><td>L
</td></tr>
<tr>
<td>lacY</td><td>E</td><td>Arabinose (L) growth in the presence of <i>lac</i> operon inducer</td><td><a href="#ref812">812</a> </td><td>L
</td></tr>
<tr>
<td>lacY</td><td>E</td><td>Thiodigalactoside resistance</td><td><a href="#ref241">241</a> </td><td> 
</td></tr>
<tr>
<td>lacZ</td><td>E</td><td>Lactobionic acid as C source</td><td><a href="#ref440">440</a> </td><td>D
</td></tr>
<tr>
<td>lacZ</td><td>E</td><td>Phenyl galactoside resistance</td><td><a href="#ref376">376</a> <a href="#ref500">500</a> </td><td>L
</td></tr>
<tr>
<td>lacZ</td><td>E</td><td>Phenethylyl galactoside resistance</td><td><a href="#ref376">376</a> <a href="#ref500">500</a> </td><td>L
</td></tr>
<tr>
<td>lacZ</td><td>E</td><td>Nitro(<i>o</i>)-phenyl galactoside resistance</td><td><a href="#ref376">376</a> <a href="#ref500">500</a> </td><td>L
</td></tr>
<tr>
<td>lamB</td><td>E</td><td>Lambda phage resistance</td><td><a href="#ref88">88</a> <a href="#ref134">134</a> <a href="#ref135">135</a> <a href="#ref253">253</a> <a href="#ref644">644</a> <a href="#ref766">766</a> </td><td>L, S
</td></tr>
<tr>
<td>lamB</td><td>E</td><td>K10 phage resistance</td><td><a href="#ref644">644</a> </td><td> 
</td></tr>
<tr>
<td>lamB</td><td>E</td><td>Thio(5)maltose resistance</td><td><a href="#ref223">223</a> </td><td>L
</td></tr>
<tr>
<td>lct</td><td>E</td><td>Vinylglycolate resistance</td><td><a href="#ref696">696</a> </td><td> 
</td></tr>
<tr>
<td>leuA</td><td>S</td><td>Trifluoroleucine resistance</td><td><a href="#ref115">115</a> </td><td>FBI, S
</td></tr>
<tr>
<td>leuJ</td><td>E</td><td>Trifluoroleucine resistance</td><td><a href="#ref575">575</a> </td><td> 
</td></tr>
<tr>
<td>leu(structural genes)</td><td>S</td><td>Trifluoroleucine resistance</td><td><a href="#ref115">115</a> <a href="#ref116">116</a> </td><td>C
</td></tr>
<tr>
<td>leuS</td><td>S, E</td><td>Azaleucine resistance</td><td><a href="#ref477">477</a> <a href="#ref514">514</a> </td><td>S
</td></tr>
<tr>
<td>leuS</td><td>S</td><td>Trifluoroleucine resistance</td><td><a href="#ref7">7</a> </td><td>S
</td></tr>
<tr>
<td>leuW</td><td>E</td><td>Calmodulin inhibitor resistance</td><td><a href="#ref129">129</a> </td><td>S
</td></tr>
<tr>
<td>lev</td><td>E</td><td>Levallorphan resistance</td><td><a href="#ref164">164</a> </td><td> 
</td></tr>
<tr>
<td>lexA</td><td>E</td><td>Mitomycin C resistance of <i>recA</i> (SOS-induced) <i>sfi</i> <i>lexA3</i>(Con)</td><td><a href="#ref538">538</a> </td><td>L
</td></tr>
<tr>
<td>lig</td><td>E</td><td>T4 phage <i>lig</i> mutant nibbling of colonies yielding <i>lig</i> overproducer</td><td><a href="#ref254">254</a> </td><td>Q
</td></tr>
<tr>
<td>lig</td><td>E</td><td>T4 phage <i>lig</i> mutant resistance from <i>lig</i> overproducer</td><td><a href="#ref254">254</a> </td><td>L
</td></tr>
<tr>
<td>linB</td><td>E</td><td>Lincomycin resistance</td><td><a href="#ref26">26</a> <a href="#ref343">343</a> </td><td> 
</td></tr>
<tr>
<td>lit</td><td>E</td><td>T4 mutant phage resistance</td><td><a href="#ref143">143</a> </td><td> 
</td></tr>
<tr>
<td>livG</td><td>E</td><td>Azaleucine resistance</td><td><a href="#ref303">303</a> <a href="#ref561">561</a> </td><td>L
</td></tr>
<tr>
<td>livH</td><td>E</td><td>Valine resistance in <i>leu</i></td><td><a href="#ref17">17</a> <a href="#ref595">595</a> </td><td>L
</td></tr>
<tr>
<td>livH</td><td>E</td><td>Azaleucine resistance</td><td><a href="#ref561">561</a> </td><td>L
</td></tr>
<tr>
<td>livJ</td><td>E</td><td>Valine resistance in <i>leu</i></td><td><a href="#ref17">17</a> <a href="#ref595">595</a> </td><td>L
</td></tr>
<tr>
<td>livK</td><td>E</td><td>Valine resistance in <i>leu</i></td><td><a href="#ref17">17</a> <a href="#ref595">595</a> </td><td>L
</td></tr>
<tr>
<td>lon</td><td>E, S</td><td>Chlorpromazine resistance</td><td><a href="#ref198">198</a> <a href="#ref529">529</a> </td><td>L
</td></tr>
<tr>
<td>lon</td><td>E</td><td>Promethazine resistance</td><td><a href="#ref529">529</a> </td><td>L
</td></tr>
<tr>
<td>lpp</td><td>E</td><td>Globomycin resistance</td><td><a href="#ref140">140</a> <a href="#ref344">344</a> <a href="#ref854">854</a> <a href="#ref868">868</a> </td><td>L
</td></tr>
<tr>
<td>lpcA</td><td>E</td><td>T3 phage resistance</td><td><a href="#ref792">792</a> </td><td>L
</td></tr>
<tr>
<td>lpcA</td><td>E</td><td>T4 phage resistance</td><td><a href="#ref792">792</a> </td><td>L
</td></tr>
<tr>
<td>lpcA</td><td>E</td><td>T7 phage resistance</td><td><a href="#ref792">792</a> </td><td>L
</td></tr>
<tr>
<td>lpcA</td><td>E</td><td>P1 phage resistance</td><td><a href="#ref40">40</a> <a href="#ref41">41</a> </td><td> 
</td></tr>
<tr>
<td>lpcB</td><td>E</td><td>T4 phage resistance</td><td><a href="#ref760">760</a> </td><td> 
</td></tr>
<tr>
<td>lpcB</td><td>E</td><td>T7 phage resistance</td><td><a href="#ref760">760</a> </td><td> 
</td></tr>
<tr>
<td>lrp</td><td>E</td><td>Leucine (D) satisfaction of <i>leu</i> auxotrophs</td><td><a href="#ref18">18</a> <a href="#ref428">428</a> </td><td> 
</td></tr>
<tr>
<td>lysC</td><td>E</td><td>Aminoethylcysteine (thialysine) resistance</td><td><a href="#ref82">82</a> <a href="#ref765">765</a> </td><td>S, FBI
</td></tr>
<tr>
<td>lysC</td><td>E</td><td>Lysine hydroxamate resistance</td><td><a href="#ref82">82</a> <a href="#ref765">765</a> </td><td>S, FBI
</td></tr>
<tr>
<td>lysP</td><td>E</td><td>Aminoethylcysteine (thialysine) resistance</td><td><a href="#ref611">611</a> <a href="#ref736">736</a> </td><td>L
</td></tr>
<tr>
<td>lysS</td><td>E</td><td>Aminoethylcysteine (thialysine) resistance</td><td><a href="#ref323">323</a> </td><td>S
</td></tr>
<tr>
<td>lysS</td><td>E</td><td>Phenyl galactoside resistance of <i>lacI </i>q<i> lacI</i>(nonsense) lacP supL</td><td><a href="#ref748">748</a> </td><td> 
</td></tr>
<tr>
<td>lyt</td><td>E</td><td>Beta-lactam tolerance</td><td><a href="#ref700">700</a> </td><td> 
</td></tr>
<tr>
<td>mac</td><td>E</td><td>Erythromycin growth dependence</td><td><a href="#ref725">725</a> </td><td> 
</td></tr>
<tr>
<td>malE</td><td>E, S</td><td>Methylglucoside(α)-resistant utilization of maltose</td><td><a href="#ref665">665</a> </td><td> 
</td></tr>
<tr>
<td>malE</td><td>E</td><td>Thio(5)maltose resistance</td><td><a href="#ref223">223</a> </td><td>L
</td></tr>
<tr>
<td>malF</td><td>E, S</td><td>Methylglucoside(α)-resistant utilization of maltose</td><td><a href="#ref665">665</a> </td><td> 
</td></tr>
<tr>
<td>malF</td><td>E</td><td>Thio(5)maltose resistance</td><td><a href="#ref223">223</a> </td><td>L
</td></tr>
<tr>
<td>malG</td><td>E, S</td><td>Methylglucoside(α)-resistant utilization of maltose</td><td><a href="#ref665">665</a> </td><td> 
</td></tr>
<tr>
<td>malG</td><td>E</td><td>Thio(5)maltose resistance</td><td><a href="#ref223">223</a> </td><td>L
</td></tr>
<tr>
<td>malK</td><td>E</td><td>Lambda phage resistance</td><td><a href="#ref766">766</a> </td><td>L
</td></tr>
<tr>
<td>malK</td><td>E, S</td><td>Methylglucoside(α)-resistant utilization of maltose</td><td><a href="#ref177">177</a> <a href="#ref665">665</a> </td><td> 
</td></tr>
<tr>
<td>malK</td><td>E</td><td>Thio(5)maltose resistance</td><td><a href="#ref223">223</a> </td><td>L
</td></tr>
<tr>
<td>malP</td><td>E</td><td>Maltose as C source in <i>malT</i></td><td><a href="#ref327">327</a> </td><td> 
</td></tr>
<tr>
<td>malQ</td><td>E</td><td>Maltose as C source in <i>malT</i></td><td><a href="#ref327">327</a> </td><td> 
</td></tr>
<tr>
<td>malT</td><td>E</td><td>Lambda phage resistance</td><td><a href="#ref766">766</a> </td><td>L
</td></tr>
<tr>
<td>malT</td><td>E</td><td>Thiomaltose resistance</td><td><a href="#ref223">223</a> </td><td>L
</td></tr>
<tr>
<td>manA</td><td>E</td><td>Streptozotocin resistance</td><td><a href="#ref456">456</a> </td><td> 
</td></tr>
<tr>
<td>manC</td><td>E</td><td>Lyxose (D) as C source</td><td><a href="#ref738">738</a> </td><td> 
</td></tr>
<tr>
<td>manXYZ</td><td>E</td><td>Phage N4 resistance</td><td><a href="#ref410">410</a> </td><td> 
</td></tr>
<tr>
<td>manXYZ</td><td>E</td><td>Deoxyglucose resistance</td><td><a href="#ref158">158</a> <a href="#ref379">379</a> </td><td>L
</td></tr>
<tr>
<td>marA</td><td>E</td><td>Menadionine resistance</td><td><a href="#ref274">274</a> </td><td> 
</td></tr>
<tr>
<td>marA</td><td>E</td><td>Chloramphenicol resistance</td><td><a href="#ref286">286</a> </td><td> 
</td></tr>
<tr>
<td>marA</td><td>E</td><td>Tetracycline resistance</td><td><a href="#ref286">286</a> <a href="#ref329">329</a> </td><td> 
</td></tr>
<tr>
<td>mbrABCD</td><td>E</td><td>Camphor resistance</td><td><a href="#ref776">776</a> <a href="#ref777">777</a> </td><td> 
</td></tr>
<tr>
<td>mdoB</td><td>E</td><td>Arbutin resistance in <i>dgk</i></td><td><a href="#ref360">360</a> </td><td>L
</td></tr>
<tr>
<td>mel (generic)</td><td>E</td><td>Raffinose as C source in <i>lacI</i></td><td><a href="#ref459">459</a> </td><td> 
</td></tr>
<tr>
<td>mel (generic)</td><td>S</td><td>Methylglucoside(α)-resistant utilization of melibiose</td><td><a href="#ref665">665</a> </td><td> 
</td></tr>
<tr>
<td>melB</td><td>E</td><td>Deoxy(2)glucose-resistant utilization of melibiose</td><td><a href="#ref431">431</a> </td><td>S
</td></tr>
<tr>
<td>melB</td><td>E</td><td>Lithium-resistant utilization of melibiose</td><td><a href="#ref397">397</a> </td><td>S
</td></tr>
<tr>
<td>metA</td><td>S</td><td>Methyl(α)methionine resistance</td><td><a href="#ref127">127</a> <a href="#ref447">447</a> </td><td>FBI, S
</td></tr>
<tr>
<td>metC (<i>ecfA</i>?)</td><td>E</td><td>Neomycin resistance</td><td><a href="#ref464">464</a> </td><td> 
</td></tr>
<tr>
<td>metC (<i>ecfA</i>?)</td><td>E</td><td>Colicin K resistance</td><td><a href="#ref465">465</a> </td><td> 
</td></tr>
<tr>
<td>metC (<i>ecfA</i>?)</td><td>E</td><td>Chloro(β)alanine resistance</td><td><a href="#ref832">832</a> </td><td> 
</td></tr>
<tr>
<td>metG</td><td>E</td><td>Ethionine resistance</td><td><a href="#ref28">28</a> </td><td>S
</td></tr>
<tr>
<td>metJ</td><td>E, S</td><td>Ethionine resistance</td><td><a href="#ref447">447</a> </td><td> 
</td></tr>
<tr>
<td>metJ</td><td>E</td><td>Glutamyl-γ-methyl ester resistance</td><td><a href="#ref425">425</a> </td><td>S
</td></tr>
<tr>
<td>metJ</td><td>E, S</td><td>Norleucine resistance</td><td><a href="#ref128">128</a> </td><td> 
</td></tr>
<tr>
<td>metJ</td><td>E</td><td>Methionine sulfoximine + methyl(α)methionine resistance</td><td><a href="#ref386">386</a> </td><td>S
</td></tr>
<tr>
<td>metK</td><td>E, S</td><td>Ethionine resistance</td><td><a href="#ref275">275</a> <a href="#ref289">289</a> <a href="#ref447">447</a> <a href="#ref735">735</a> </td><td>S
</td></tr>
<tr>
<td>metK</td><td>E</td><td>Glutamyl-γ-methyl ester resistance</td><td><a href="#ref425">425</a> </td><td>S
</td></tr>
<tr>
<td>metK</td><td>E, S</td><td>Norleucine resistance</td><td><a href="#ref128">128</a> <a href="#ref447">447</a> <a href="#ref735">735</a> </td><td>S
</td></tr>
<tr>
<td>metK</td><td>S</td><td>Methyl(α)methionine resistance</td><td><a href="#ref447">447</a> <a href="#ref735">735</a> </td><td>S
</td></tr>
<tr>
<td>metL</td><td>E</td><td>Hydroxy (β) norvaline resistance</td><td><a href="#ref138">138</a> </td><td>S, FBI
</td></tr>
<tr>
<td>metPD</td><td>E, S</td><td>Methyl(α)methionine resistance</td><td><a href="#ref37">37</a> <a href="#ref38">38</a> <a href="#ref386">386</a> </td><td>L
</td></tr>
<tr>
<td>metPD</td><td>E, S</td><td>Methionine sulfoximine resistance</td><td><a href="#ref37">37</a> <a href="#ref38">38</a> <a href="#ref386">386</a> </td><td>L
</td></tr>
<tr>
<td>mglA</td><td>E</td><td>Fucose + arabinose-supported growth in <i>galP</i></td><td><a href="#ref209">209</a> <a href="#ref613">613</a> </td><td> 
</td></tr>
<tr>
<td>mglB</td><td>E</td><td>Fucose + arabinose-supported growth in <i>galP</i></td><td><a href="#ref209">209</a> <a href="#ref613">613</a> </td><td> 
</td></tr>
<tr>
<td>mglC</td><td>E</td><td>Fucose + arabinose-supported growth in <i>galP</i></td><td><a href="#ref209">209</a> <a href="#ref613">613</a> </td><td> 
</td></tr>
<tr>
<td>mglD</td><td>E</td><td>Methyl-β-galactoside-supported growth</td><td><a href="#ref645">645</a> </td><td>C
</td></tr>
<tr>
<td>mng</td><td>E</td><td>Manganese resistance</td><td><a href="#ref708">708</a> </td><td> 
</td></tr>
<tr>
<td>mopA</td><td>E</td><td>T4 phage resistance</td><td><a href="#ref258">258</a> </td><td>S</td></tr>
<tr>
<td>mopB</td><td>E</td><td>T4 phage resistance</td><td><a href="#ref258">258</a> </td><td>S</td></tr>
<tr>
<td>motAB</td><td>E</td><td>Chi phage resistance</td><td><a href="#ref349">349</a> <a href="#ref709">709</a> <a href="#ref710">710</a> </td><td> 
</td></tr>
<tr>
<td>mrb</td><td>E</td><td>Fosfomycin resistance</td><td><a href="#ref800">800</a> </td><td> 
</td></tr>
<tr>
<td>mrdA</td><td>E</td><td>Mecillinam resistance</td><td><a href="#ref759">759</a> </td><td> 
</td></tr>
<tr>
<td>mrdB</td><td>E</td><td>Mecillinam resistance</td><td><a href="#ref358">358</a> <a href="#ref502">502</a> <a href="#ref759">759</a> </td><td> 
</td></tr>
<tr>
<td>mreBCD</td><td>E</td><td>Mecillinam resistance</td><td><a href="#ref582">582</a> <a href="#ref807">807</a> </td><td> 
</td></tr>
<tr>
<td>mtlA</td><td>E</td><td>Arabinitol (D) resistance</td><td><a href="#ref614">614</a> </td><td>L
</td></tr>
<tr>
<td>mtlC</td><td>E</td><td>Mannitol (limiting) as C source</td><td><a href="#ref722">722</a> </td><td>C
</td></tr>
<tr>
<td>mtr</td><td>E</td><td>Methyl(5)tryptophan resistance</td><td><a href="#ref305">305</a> <a href="#ref320">320</a> <a href="#ref523">523</a> </td><td>L
</td></tr>
<tr>
<td>murZ</td><td>E</td><td>Fosfomycin resistance due to increased gene dosage</td><td><a href="#ref499">499</a> </td><td>D
</td></tr>
<tr>
<td>mut (generic)</td><td>E</td><td>Streptomycin resistance</td><td><a href="#ref309">309</a> </td><td> 
</td></tr>
<tr>
<td>mut (generic)</td><td>E</td><td>Azaserine resistance</td><td><a href="#ref703">703</a> </td><td> 
</td></tr>
<tr>
<td>mut (generic)</td><td>E</td><td>Chemostat growth</td><td><a href="#ref152">152</a> </td><td> 
</td></tr>
<tr>
<td>mutH</td><td>E</td><td>Amino(2)purine resistance in <i>dam</i></td><td><a href="#ref267">267</a> </td><td>L
</td></tr>
<tr>
<td>mutL</td><td>E</td><td>Amino(2)purine resistance in <i>dam</i></td><td><a href="#ref267">267</a> </td><td>L
</td></tr>
<tr>
<td>mutS</td><td>E</td><td>Amino(2)purine resistance in <i>dam</i></td><td><a href="#ref267">267</a> </td><td>L
</td></tr>
<tr>
<td>mvrC</td><td>E</td><td>Methyl viologen resistance</td><td><a href="#ref532">532</a> </td><td>D
</td></tr>
<tr>
<td>nadB</td><td>S</td><td>Amino(6)nicotinamide resistance</td><td><a href="#ref142">142</a> <a href="#ref336">336</a> </td><td>S
</td></tr>
<tr>
<td>nadD</td><td>S</td><td>Amino(6)nicotinamide resistance</td><td><a href="#ref336">336</a> <a href="#ref337">337</a> </td><td>L
</td></tr>
<tr>
<td>nagA</td><td>E</td><td>Streptozotocin resistance</td><td><a href="#ref456">456</a> </td><td>L
</td></tr>
<tr>
<td>nagA</td><td>E</td><td>Iodoacetylglucosamine resistance</td><td><a href="#ref823">823</a> </td><td>L
</td></tr>
<tr>
<td>nagB</td><td>E</td><td>Iodoacetylglucosamine resistance</td><td><a href="#ref823">823</a> </td><td>L
</td></tr>
<tr>
<td>nagE</td><td>E</td><td>Streptozotocin resistance</td><td><a href="#ref456">456</a> </td><td>L
</td></tr>
<tr>
<td>nagE</td><td>E</td><td>Iodoacetylglucosamine resistance</td><td><a href="#ref823">823</a> </td><td>L
</td></tr>
<tr>
<td>nagE</td><td>E</td><td>Deoxy(2)-2-iodoacetamideoglucose resistance</td><td><a href="#ref379">379</a> </td><td>L
</td></tr>
<tr>
<td>nalB</td><td>E</td><td>Nalidixic acid resistance</td><td><a href="#ref298">298</a> </td><td> 
</td></tr>
<tr>
<td>nalD</td><td>E</td><td>Nalidixic acid resistance</td><td><a href="#ref333">333</a> </td><td> 
</td></tr>
<tr>
<td>nar (general)</td><td>E</td><td>Metronidazole resistance</td><td><a href="#ref677">677</a> </td><td>L
</td></tr>
<tr>
<td>narC</td><td>E</td><td>Chlorate resistance</td><td><a href="#ref266">266</a> <a href="#ref280">280</a> </td><td>L
</td></tr>
<tr>
<td>narG</td><td>E</td><td>Chlorate resistance</td><td><a href="#ref739">739</a> </td><td>L
</td></tr>
<tr>
<td>narH</td><td>E</td><td>Chlorate resistance</td><td><a href="#ref739">739</a> </td><td>L
</td></tr>
<tr>
<td>ndk</td><td>S</td><td>Aza(8)guanine resistance</td><td><a href="#ref264">264</a> <a href="#ref650">650</a> </td><td> 
</td></tr>
<tr>
<td>neaB</td><td>E, S</td><td>Neamine resistance</td><td><a href="#ref119">119</a> <a href="#ref182">182</a> <a href="#ref495">495</a> </td><td> 
</td></tr>
<tr>
<td>nek</td><td>E</td><td>Aminoglycoside resistance</td><td><a href="#ref27">27</a> <a href="#ref342">342</a> </td><td> 
</td></tr>
<tr>
<td>nfnA</td><td>E</td><td>Nitrofurantoin resistance</td><td><a href="#ref684">684</a> </td><td> 
</td></tr>
<tr>
<td>nfnB</td><td>E</td><td>Nitrofurantoin resistance</td><td><a href="#ref684">684</a> </td><td> 
</td></tr>
<tr>
<td>nfsA</td><td>E</td><td>Nitrofuranzone resistance</td><td><a href="#ref90">90</a> <a href="#ref505">505</a> </td><td> 
</td></tr>
<tr>
<td>nfsB</td><td>E</td><td>Nitrofuranzone resistance</td><td><a href="#ref90">90</a> <a href="#ref505">505</a> </td><td> 
</td></tr>
<tr>
<td>nhaA</td><td>E</td><td>Lithium resistance (multicopy)</td><td><a href="#ref628">628</a> </td><td>D
</td></tr>
<tr>
<td>nol</td><td>S</td><td>Norleucine resistance</td><td><a href="#ref324">324</a> </td><td> 
</td></tr>
<tr>
<td>nov</td><td>E</td><td>Novobiocin resistance</td><td><a href="#ref630">630</a> </td><td> 
</td></tr>
<tr>
<td>nrdA</td><td>E</td><td>Hydroxyurea resistance</td><td><a href="#ref609">609</a> </td><td>D
</td></tr>
<tr>
<td>nrdB</td><td>E</td><td>Hydroxyurea resistance</td><td><a href="#ref609">609</a> </td><td>D
</td></tr>
<tr>
<td>nupC</td><td>E</td><td>Showdomycin resistance</td><td><a href="#ref415">415</a> <a href="#ref545">545</a> </td><td>L
</td></tr>
<tr>
<td>nupC</td><td>E</td><td>Fluorouracil resistance</td><td><a href="#ref544">544</a> </td><td>L
</td></tr>
<tr>
<td>nupC</td><td>E</td><td>Fluorodeoxyuridine resistance</td><td><a href="#ref545">545</a> </td><td>L
</td></tr>
<tr>
<td>nupC</td><td>E</td><td>Fluoruridine resistance</td><td><a href="#ref544">544</a> </td><td>L
</td></tr>
<tr>
<td>nupC</td><td>E</td><td>Fluorodeoxycytidine resistance</td><td><a href="#ref544">544</a> </td><td>L
</td></tr>
<tr>
<td>nupG</td><td>E</td><td>Fluorouracil resistance</td><td><a href="#ref544">544</a> </td><td>L
</td></tr>
<tr>
<td>nupG</td><td>E</td><td>Fluorodeoxyuridine resistance</td><td><a href="#ref545">545</a> </td><td>L
</td></tr>
<tr>
<td>nupG</td><td>E</td><td>Fluorouridine resistance</td><td><a href="#ref544">544</a> </td><td>L
</td></tr>
<tr>
<td>nupG</td><td>E</td><td>Fluorodeoxycytidine resistance</td><td><a href="#ref544">544</a> </td><td>L
</td></tr>
<tr>
<td>nusA</td><td>E</td><td>Lambda prophage induction, survival of</td><td><a href="#ref244">244</a> </td><td> 
</td></tr>
<tr>
<td>nusB</td><td>E</td><td>Lambda prophage induction, survival of</td><td><a href="#ref245">245</a> <a href="#ref260">260</a> </td><td> 
</td></tr>
<tr>
<td>nusB</td><td>E</td><td>Lambdoid phage resistance</td><td><a href="#ref260">260</a> </td><td> 
</td></tr>
<tr>
<td>nuvA</td><td>E, S</td><td>Near-UV irradiation resistance</td><td><a href="#ref424">424</a> <a href="#ref468">468</a> <a href="#ref767">767</a> </td><td>L
</td></tr>
<tr>
<td>nuvC</td><td>E</td><td>Near-UV irradiation resistance</td><td><a href="#ref663">663</a> </td><td> 
</td></tr>
<tr>
<td>ompA</td><td>E</td><td>K3 phage resistance</td><td><a href="#ref306">306</a> <a href="#ref489">489</a> <a href="#ref491">491</a> <a href="#ref493">493</a> </td><td>L
</td></tr>
<tr>
<td>ompA</td><td>E</td><td>T-even-like phage resistance</td><td><a href="#ref199">199</a> <a href="#ref533">533</a> <a href="#ref535">535</a> </td><td>S
</td></tr>
<tr>
<td>ompA</td><td>E</td><td>TuII* phage resistance</td><td><a href="#ref171">171</a> <a href="#ref314">314</a> <a href="#ref493">493</a> </td><td>L
</td></tr>
<tr>
<td>ompA</td><td>E</td><td>Ox2 phage resistance</td><td><a href="#ref199">199</a> <a href="#ref493">493</a> <a href="#ref642">642</a> </td><td> 
</td></tr>
<tr>
<td>ompA</td><td>E</td><td>Bacteriocin JF246 resistance</td><td><a href="#ref124">124</a> <a href="#ref491">491</a> </td><td> 
</td></tr>
<tr>
<td>ompA</td><td>S</td><td>Bacteriocin 4-59 resistance</td><td><a href="#ref741">741</a> </td><td>L
</td></tr>
<tr>
<td>ompA</td><td>E</td><td>Colicin K tolerance</td><td><a href="#ref491">491</a> </td><td> 
</td></tr>
<tr>
<td>ompA</td><td>E</td><td>Colicin L tolerance</td><td><a href="#ref491">491</a> </td><td> 
</td></tr>
<tr>
<td>ompA</td><td>E</td><td>Chelator resistance</td><td><a href="#ref493">493</a> </td><td> 
</td></tr>
<tr>
<td>ompA</td><td>E</td><td>Novobiocin resistance</td><td><a href="#ref493">493</a> </td><td> 
</td></tr>
<tr>
<td>ompB</td><td>S</td><td>PH51 phage resistance</td><td><a href="#ref372">372</a> </td><td> 
</td></tr>
<tr>
<td>ompB</td><td>S</td><td>PH105 phage resistance</td><td><a href="#ref372">372</a> </td><td> 
</td></tr>
<tr>
<td>ompC</td><td>E</td><td>Me1 phage resistance</td><td><a href="#ref791">791</a> <a href="#ref801">801</a> </td><td>L
</td></tr>
<tr>
<td>ompC</td><td>E</td><td>TuIa phage resistance</td><td><a href="#ref801">801</a> </td><td>L
</td></tr>
<tr>
<td>ompC</td><td>S</td><td>PH51 phage resistance</td><td><a href="#ref372">372</a> </td><td>L
</td></tr>
<tr>
<td>ompC</td><td>S</td><td>PH105 phage resistance</td><td><a href="#ref372">372</a> </td><td>L
</td></tr>
<tr>
<td>ompC</td><td>E</td><td>Host range <i>Serratia marcescens</i> phage resistance</td><td><a href="#ref702">702</a> </td><td> 
</td></tr>
<tr>
<td>ompD</td><td>S</td><td>PH51 phage resistance</td><td><a href="#ref741">741</a> </td><td>L
</td></tr>
<tr>
<td>ompD</td><td>S</td><td>Bacteriocin 4-59 resistance</td><td><a href="#ref741">741</a> </td><td>L
</td></tr>
<tr>
<td>ompF</td><td>E</td><td>Beta-lactam (ampicillin, cefoxitin) resistance</td><td><a href="#ref362">362</a> <a href="#ref363">363</a> </td><td>L
</td></tr>
<tr>
<td>ompF</td><td>E</td><td>Chloramphenicol resistance</td><td><a href="#ref235">235</a> <a href="#ref523">523</a> </td><td>L
</td></tr>
<tr>
<td>ompF</td><td>E</td><td>Tetracycline resistance</td><td><a href="#ref235">235</a> <a href="#ref523">523</a> </td><td>L
</td></tr>
<tr>
<td>ompF</td><td>E</td><td>Fluoro(5)uracil + nucleotide resistance</td><td><a href="#ref52">52</a> </td><td>L
</td></tr>
<tr>
<td>ompF</td><td>E</td><td>Colicin A tolerance</td><td><a href="#ref125">125</a> <a href="#ref235">235</a> </td><td> 
</td></tr>
<tr>
<td>ompF</td><td>E</td><td>Cloacin DF13 resistance</td><td><a href="#ref846">846</a> </td><td>L
</td></tr>
<tr>
<td>ompF</td><td>E</td><td>Colicin L tolerance</td><td><a href="#ref235">235</a> <a href="#ref291">291</a> </td><td>L
</td></tr>
<tr>
<td>ompF</td><td>E</td><td>Colicin K tolerance</td><td><a href="#ref235">235</a> </td><td>L
</td></tr>
<tr>
<td>ompF</td><td>E</td><td>Colicin E2 tolerance</td><td><a href="#ref235">235</a> </td><td> 
</td></tr>
<tr>
<td>ompF</td><td>E</td><td>Colicin E3 tolerance</td><td><a href="#ref235">235</a> </td><td> 
</td></tr>
<tr>
<td>ompF</td><td>E</td><td>Me1 phage resistance</td><td><a href="#ref801">801</a> </td><td>L
</td></tr>
<tr>
<td>ompF</td><td>E</td><td>TuIa phage resistance</td><td><a href="#ref801">801</a> </td><td>L
</td></tr>
<tr>
<td>ompF</td><td>E</td><td>TuIb phage resistance</td><td><a href="#ref237">237</a> </td><td> 
</td></tr>
<tr>
<td>ompF</td><td>S</td><td>PH51 phage resistance</td><td><a href="#ref372">372</a> </td><td>L
</td></tr>
<tr>
<td>ompF</td><td>S</td><td>PH105 phage resistance</td><td><a href="#ref372">372</a> </td><td>L
</td></tr>
<tr>
<td>ompF</td><td>E</td><td>Copper resistance in <i>ompC</i></td><td><a href="#ref480">480</a> </td><td> 
</td></tr>
<tr>
<td>ompR</td><td>E</td><td>Me1 phage resistance</td><td><a href="#ref801">801</a> </td><td> 
</td></tr>
<tr>
<td>ompR</td><td>E</td><td>TuIa phage resistance</td><td><a href="#ref801">801</a> </td><td> 
</td></tr>
<tr>
<td>ompR</td><td>E</td><td>Copper resistance</td><td><a href="#ref52">52</a> <a href="#ref625">625</a> </td><td> 
</td></tr>
<tr>
<td>ompR</td><td>E</td><td>Beta-lactam (ampicillin, cefoxitin) resistance</td><td><a href="#ref362">362</a> <a href="#ref363">363</a> </td><td>L
</td></tr>
<tr>
<td>ompR</td><td>E</td><td>Chloramphenicol resistance</td><td><a href="#ref625">625</a> </td><td> 
</td></tr>
<tr>
<td>ompR</td><td>E</td><td>Fluoro(5)uracil + nucleotide resistance</td><td><a href="#ref52">52</a> </td><td> 
</td></tr>
<tr>
<td>oppA</td><td>E, S</td><td>Triornithine resistance</td><td><a href="#ref43">43</a> <a href="#ref317">317</a> </td><td>L
</td></tr>
<tr>
<td>oppA</td><td>E</td><td>Glycylleucine resistance</td><td><a href="#ref806">806</a> </td><td>L
</td></tr>
<tr>
<td>oppA</td><td>E</td><td>Alanyl-2-aminopropionate resistance</td><td><a href="#ref603">603</a> </td><td>L
</td></tr>
<tr>
<td>oppA</td><td>E, S</td><td>Trilysine resistance</td><td><a href="#ref317">317</a> </td><td>L
</td></tr>
<tr>
<td>oppA</td><td>E, S</td><td>Norleucylglycyl glycine resistance</td><td><a href="#ref317">317</a> </td><td>L
</td></tr>
<tr>
<td>oppA</td><td>S</td><td>Glycylglycyl histidinol phosphate ester resistance</td><td><a href="#ref317">317</a> </td><td>L
</td></tr>
<tr>
<td>oppA</td><td>E, S</td><td>Phaseolotoxin resistance</td><td><a href="#ref733">733</a> </td><td>L
</td></tr>
<tr>
<td>oppA</td><td>E</td><td>Tripeptide (toxic amino acid containing) resistance</td><td><a href="#ref604">604</a> </td><td>L
</td></tr>
<tr>
<td>oppA</td><td>E</td><td>Glycylglycyl-<i>N</i>-<i>S</i>-(phosphonoacetyl)-L-ornithine resistance</td><td><a href="#ref604">604</a> </td><td>L
</td></tr>
<tr>
<td>oppE</td><td>E</td><td>Tripeptide (toxic amino acid containing) resistance</td><td><a href="#ref22">22</a> </td><td>L
</td></tr>
<tr>
<td>optA</td><td>E</td><td>T7 phage gene 1.2 mutant resistance</td><td><a href="#ref666">666</a> </td><td> 
</td></tr>
<tr>
<td>osmB</td><td>E</td><td>Osmotolerance</td><td><a href="#ref384">384</a> </td><td>L
</td></tr>
<tr>
<td>osmZ</td><td>E</td><td>Arbutin as C source</td><td><a href="#ref179">179</a> </td><td> 
</td></tr>
<tr>
<td>osmZ</td><td>E</td><td>Salicin as C source</td><td><a href="#ref179">179</a> </td><td> 
</td></tr>
<tr>
<td>oxyR</td><td>E, S</td><td>Peroxide resistance</td><td><a href="#ref131">131</a> </td><td>C
</td></tr>
<tr>
<td>pab</td><td>E</td><td>Sulfonamide resistance</td><td><a href="#ref97">97</a> </td><td>C
</td></tr>
<tr>
<td>pan</td><td>E</td><td>Salicylate resistance</td><td><a href="#ref481">481</a> </td><td>C
</td></tr>
<tr>
<td>pan</td><td>E</td><td>Hydroxyaspartate resistance</td><td><a href="#ref701">701</a> </td><td>C
</td></tr>
<tr>
<td>pan</td><td>E</td><td>Pentylpantothenamide resistance</td><td><a href="#ref136">136</a> </td><td>C
</td></tr>
<tr>
<td>pdx</td><td>E</td><td>Isoniazid resistance</td><td><a href="#ref186">186</a> </td><td>C
</td></tr>
<tr>
<td>pepA</td><td>S</td><td>Alafosfalin resistance</td><td><a href="#ref263">263</a> </td><td>L
</td></tr>
<tr>
<td>pepA</td><td>E, S</td><td>Peptide (toxic, valine containing) resistance in <i>pepN</i></td><td><a href="#ref517">517</a> </td><td>L
</td></tr>
<tr>
<td>pepD</td><td>E, S</td><td>Peptide (toxic, valine containing) resistance in <i>pepN</i> <i>pepA</i></td><td><a href="#ref517">517</a> </td><td>L
</td></tr>
<tr>
<td>pepN</td><td>E, S</td><td>Albomycin resistance</td><td><a href="#ref86">86</a> </td><td>L
</td></tr>
<tr>
<td>pepN</td><td>E, S</td><td>Peptide (toxic, valine containing) resistance</td><td><a href="#ref517">517</a> </td><td>L
</td></tr>
<tr>
<td>pepN</td><td>E, S</td><td>Indicator plate</td><td><a href="#ref445">445</a> </td><td> 
</td></tr>
<tr>
<td>pepQ</td><td>E, S</td><td>Peptide (toxic, valine containing) resistance</td><td><a href="#ref517">517</a> </td><td>L
</td></tr>
<tr>
<td>pfkA</td><td>E</td><td>Streptozotocin resistance</td><td><a href="#ref456">456</a> </td><td>L
</td></tr>
<tr>
<td>pgi</td><td>E</td><td>U3 phage resistance</td><td><a href="#ref816">816</a> </td><td>L
</td></tr>
<tr>
<td>pgi</td><td>E</td><td>Fosfomycin + fructose-6-phosphate resistance</td><td><a href="#ref243">243</a> </td><td>L
</td></tr>
<tr>
<td>pgm</td><td>E</td><td>U3 phage resistance</td><td><a href="#ref816">816</a> </td><td>L
</td></tr>
<tr>
<td>pheA</td><td>E</td><td>Fluoro(4)phenylalanine resistance</td><td><a href="#ref351">351</a> <a href="#ref564">564</a> </td><td>C, FBI
</td></tr>
<tr>
<td>pheR</td><td>S</td><td>Fluoro(4)phenylalanine resistance</td><td><a href="#ref271">271</a> </td><td>C
</td></tr>
<tr>
<td>pheS</td><td>E</td><td>Fluoro(4)phenylalanine resistance</td><td><a href="#ref78">78</a> <a href="#ref218">218</a> <a href="#ref395">395</a> </td><td>S
</td></tr>
<tr>
<td>pheS</td><td>E</td><td>Thymineless death resistance in <i>thyA</i></td><td><a href="#ref394">394</a> </td><td>S
</td></tr>
<tr>
<td>pheU</td><td>S</td><td>Fluoro(4)phenylalanine resistance</td><td><a href="#ref729">729</a> </td><td> 
</td></tr>
<tr>
<td>pho</td><td>S</td><td>Glycerol(β)-phosphate as C source in the presence of high phosphate</td><td><a href="#ref407">407</a> </td><td>C
</td></tr>
<tr>
<td>phoA</td><td>E</td><td>Fluoro(5)uracil + adenosine resistance in <i>upp, phoS,</i> or <i>phoT</i></td><td><a href="#ref315">315</a> </td><td> 
</td></tr>
<tr>
<td>phoA</td><td>E</td><td>Glycerol(β)-phosphate as C source in the presence of high phosphate</td><td><a href="#ref773">773</a> </td><td>C
</td></tr>
<tr>
<td>phoA</td><td>E</td><td>Glycerol(β)-phosphate resistance in <i>glpD</i></td><td><a href="#ref678">678</a> </td><td> 
</td></tr>
<tr>
<td>phoB</td><td>E</td><td>Tellurite resistance</td><td><a href="#ref770">770</a> </td><td> 
</td></tr>
<tr>
<td>phoB</td><td>E</td><td>TC45 phage resistance</td><td><a href="#ref771">771</a> </td><td>L
</td></tr>
<tr>
<td>phoB</td><td>E</td><td>Fluoro(5)uracil + adenosine resistance in <i>upp, phoS,</i> or <i>phoT</i></td><td><a href="#ref316">316</a> </td><td> 
</td></tr>
<tr>
<td>phoE</td><td>E</td><td>Acid resistance</td><td><a href="#ref659">659</a> </td><td>D
</td></tr>
<tr>
<td>phoE</td><td>E</td><td>TC45 phage resistance</td><td><a href="#ref420">420</a> <a href="#ref771">771</a> </td><td>S, L
</td></tr>
<tr>
<td>phoR</td><td>E</td><td>Arsenate resistance</td><td><a href="#ref852">852</a> </td><td> 
</td></tr>
<tr>
<td>phoR</td><td>E</td><td>Glycerol(β)-phosphate as C source in the presence of high phosphate</td><td><a href="#ref204">204</a> <a href="#ref838">838</a> </td><td> 
</td></tr>
<tr>
<td>phoR</td><td>E</td><td>TC45 phage resistance</td><td><a href="#ref621">621</a> </td><td>S
</td></tr>
<tr>
<td>phoS</td><td>E</td><td>Arsenate resistance</td><td><a href="#ref852">852</a> </td><td> 
</td></tr>
<tr>
<td>phoS</td><td>E</td><td>Glycerol(β)-phosphate as C source in the presence of high phosphate</td><td><a href="#ref204">204</a> <a href="#ref838">838</a> </td><td> 
</td></tr>
<tr>
<td>phoS</td><td>E</td><td>TC45 phage resistance</td><td><a href="#ref621">621</a> </td><td>S
</td></tr>
<tr>
<td>phoT</td><td>E</td><td>Arsenate resistance</td><td><a href="#ref852">852</a> </td><td> 
</td></tr>
<tr>
<td>phoT</td><td>E</td><td>Glycerol(β)-phosphate as C source in the presence of high phosphate</td><td><a href="#ref204">204</a> <a href="#ref838">838</a> </td><td> 
</td></tr>
<tr>
<td>phoT</td><td>E</td><td>TC45 phage resistance</td><td><a href="#ref621">621</a> </td><td>S
</td></tr>
<tr>
<td>phr</td><td>E</td><td>T1 phage (UV irradiated) resistance</td><td><a href="#ref332">332</a> </td><td>L
</td></tr>
<tr>
<td>phxB</td><td>E</td><td>φX174 phage resistance</td><td><a href="#ref547">547</a> </td><td>L
</td></tr>
<tr>
<td>pit</td><td>E</td><td>Arsenate resistance</td><td><a href="#ref59">59</a> <a href="#ref727">727</a> </td><td>L
</td></tr>
<tr>
<td>pldA</td><td>E</td><td>Actinomycin D resistance in the presence of EDTA</td><td><a href="#ref301">301</a> <a href="#ref805">805</a> </td><td>L
</td></tr>
<tr>
<td>pmi</td><td>S</td><td>P22 phage resistance</td><td><a href="#ref656">656</a> <a href="#ref833">833</a> </td><td>L
</td></tr>
<tr>
<td>pmi</td><td>S</td><td>9NA phage resistance</td><td><a href="#ref656">656</a> <a href="#ref833">833</a> </td><td>L
</td></tr>
<tr>
<td>pmrA</td><td>E, S</td><td>Polymyxin resistance</td><td><a href="#ref219">219</a> <a href="#ref297">297</a> <a href="#ref485">485</a> <a href="#ref785">785</a> <a href="#ref786">786</a> <a href="#ref787">787</a> <a href="#ref788">788</a> </td><td> 
</td></tr>
<tr>
<td>pmrA</td><td>S</td><td>Neutrophil granule protein resistance</td><td><a href="#ref219">219</a> </td><td> 
</td></tr>
<tr>
<td>pncA</td><td>E, S</td><td>Amino(6)nicotinamide resistance</td><td><a href="#ref234">234</a> <a href="#ref455">455</a> <a href="#ref822">822</a> </td><td>L
</td></tr>
<tr>
<td>pncA</td><td>E</td><td>Nicotinamide as sole N source</td><td><a href="#ref598">598</a> </td><td>Q
</td></tr>
<tr>
<td>pncB</td><td>E, S</td><td>Amino(6)nicotinamide resistance</td><td><a href="#ref234">234</a> <a href="#ref455">455</a> </td><td>L
</td></tr>
<tr>
<td>pncB</td><td>E, S</td><td>Amino(6)nicotinate resistance</td><td><a href="#ref234">234</a> <a href="#ref455">455</a> </td><td>L
</td></tr>
<tr>
<td>pncX</td><td>S</td><td>Amino(6)nicotinamide resistance</td><td><a href="#ref336">336</a> </td><td> 
</td></tr>
<tr>
<td>pnp</td><td>E</td><td>Showdomycin resistance</td><td><a href="#ref58">58</a> </td><td> 
</td></tr>
<tr>
<td>polA</td><td>E</td><td>Lambda γ mutant phage resistance</td><td><a href="#ref867">867</a> </td><td>L
</td></tr>
<tr>
<td><i>praA,B</i></td><td>S</td><td>P221 phage resistance</td><td><a href="#ref576">576</a> <a href="#ref674">674</a> </td><td>L
</td></tr>
<tr>
<td><i>praA,B</i></td><td>S</td><td>PH51 phage resistance</td><td><a href="#ref576">576</a> </td><td>L
</td></tr>
<tr>
<td><i>praA,B</i></td><td>S</td><td>PH105 phage resistance</td><td><a href="#ref576">576</a> </td><td>L
</td></tr>
<tr>
<td><i>prbA,B</i></td><td>S</td><td>ES18 phage resistance</td><td><a href="#ref674">674</a> </td><td> 
</td></tr>
<tr>
<td>prdB</td><td>S</td><td>PH51 phage resistance</td><td><a href="#ref674">674</a> </td><td> 
</td></tr>
<tr>
<td>prfA</td><td>E</td><td>Phenylgalactoside resistance of <i>lacP lacI</i>(nonsense) <i>supE</i></td><td><a href="#ref747">747</a> </td><td> 
</td></tr>
<tr>
<td>prfB</td><td>E</td><td>Negamycin resistance</td><td><a href="#ref145">145</a> </td><td> 
</td></tr>
<tr>
<td>prh</td><td>S</td><td>HK009 phage resistance</td><td><a href="#ref674">674</a> </td><td> 
</td></tr>
<tr>
<td>prk</td><td>S</td><td>HK068 phage resistance</td><td><a href="#ref674">674</a> </td><td> 
</td></tr>
<tr>
<td>proAB</td><td>S</td><td>Osmotolerance</td><td><a href="#ref155">155</a> </td><td> 
</td></tr>
<tr>
<td>proA</td><td>E</td><td>Nitro(4)pyridine <i>N</i>-oxide resistance</td><td><a href="#ref62">62</a> <a href="#ref222">222</a> <a href="#ref321">321</a> </td><td>L
</td></tr>
<tr>
<td>proB</td><td>E, S</td><td>Dehydroproline resistance</td><td><a href="#ref62">62</a> <a href="#ref155">155</a> <a href="#ref635">635</a> </td><td>FBI, S
</td></tr>
<tr>
<td>proB</td><td>E</td><td>Nitro(4)pyridine <i>N</i>-oxide resistance</td><td><a href="#ref62">62</a> <a href="#ref222">222</a> <a href="#ref321">321</a> </td><td> 
</td></tr>
<tr>
<td>proB</td><td>E, S</td><td>Azetidine carboxylate resistance</td><td><a href="#ref62">62</a> <a href="#ref155">155</a> <a href="#ref635">635</a> </td><td>FBI, S
</td></tr>
<tr>
<td>proP</td><td>E, S</td><td>Dehydroproline resistance in <i>putA putP</i></td><td><a href="#ref156">156</a> <a href="#ref732">732</a> </td><td>L
</td></tr>
<tr>
<td>proU</td><td>S</td><td>Dehydroproline + azetidine carboxylate resistance at high osmolarity in <i>putA putP</i></td><td><a href="#ref156">156</a> </td><td>L
</td></tr>
<tr>
<td>prp</td><td>E</td><td>Propionate as C source</td><td><a href="#ref400">400</a> </td><td>Q
</td></tr>
<tr>
<td>pspABCDE</td><td>E</td><td>Sulfanilamide + hypoxanthine resistance</td><td><a href="#ref97">97</a> </td><td> 
</td></tr>
<tr>
<td>pss</td><td>E</td><td>Ethanol resistance</td><td><a href="#ref133">133</a> </td><td> 
</td></tr>
<tr>
<td>pss</td><td>E</td><td>Dimethyl sulfoxide resistance</td><td><a href="#ref133">133</a> </td><td> 
</td></tr>
<tr>
<td>pstABCS</td><td>E</td><td>Arsenate resistance</td><td><a href="#ref727">727</a> <a href="#ref838">838</a> </td><td>L
</td></tr>
<tr>
<td>pstABCS</td><td>E</td><td>TC45 phage resistance</td><td><a href="#ref621">621</a> </td><td>S
</td></tr>
<tr>
<td>pta</td><td>S</td><td>Fluoroacetate resistance</td><td><a href="#ref96">96</a> <a href="#ref281">281</a> <a href="#ref463">463</a> <a href="#ref797">797</a> </td><td> 
</td></tr>
<tr>
<td>pts (general)</td><td>E</td><td>Glucose + gluconate-independent motility</td><td><a href="#ref30">30</a> </td><td> 
</td></tr>
<tr>
<td>ptsG</td><td>E</td><td>Methyl(α)-glucoside resistance</td><td><a href="#ref102">102</a> <a href="#ref104">104</a> </td><td> 
</td></tr>
<tr>
<td>ptsG</td><td>E</td><td>Sorbose resistance</td><td><a href="#ref715">715</a> </td><td>L
</td></tr>
<tr>
<td>ptsG</td><td>E, S</td><td>Deoxy(2)glucose-independent utilization of fructose</td><td><a href="#ref418">418</a> <a href="#ref510">510</a> </td><td>L
</td></tr>
<tr>
<td>ptsG</td><td>E, S</td><td>Deoxy(3)-3-fluoroglucose-independent utilization of fructose</td><td><a href="#ref418">418</a> <a href="#ref510">510</a> </td><td>L
</td></tr>
<tr>
<td>ptsG</td><td>E</td><td>Methyl(α)-glucoside-independent lactose + mannitol utilization</td><td><a href="#ref80">80</a> </td><td>L
</td></tr>
<tr>
<td>ptsG</td><td>E</td><td>Glucosamine as C source</td><td><a href="#ref379">379</a> </td><td> 
</td></tr>
<tr>
<td>ptsG</td><td>E</td><td>Mannose as C source</td><td><a href="#ref379">379</a> </td><td> 
</td></tr>
<tr>
<td>ptsHI</td><td>E, S</td><td>Deoxy(3)-3-fluoroglucose-independent utilization of lactate</td><td><a href="#ref510">510</a> </td><td>L
</td></tr>
<tr>
<td>ptsHI</td><td>E, S</td><td>Fosfomycin resistance</td><td><a href="#ref146">146</a> <a href="#ref800">800</a> </td><td>L
</td></tr>
<tr>
<td>ptsHI</td><td>E</td><td>Streptozotocin resistance</td><td><a href="#ref16">16</a> <a href="#ref873">873</a> </td><td>L
</td></tr>
<tr>
<td>ptsI</td><td>E</td><td>Ampicillin resistance</td><td><a href="#ref212">212</a> </td><td> 
</td></tr>
<tr>
<td>ptsI</td><td>S</td><td>Deoxy(3)-3-fluoroglucose resistance</td><td><a href="#ref510">510</a> </td><td>L
</td></tr>
<tr>
<td>purA</td><td>E</td><td>Methyl(6)purine + hypoxanthine resistance</td><td><a href="#ref60">60</a> </td><td> 
</td></tr>
<tr>
<td>purB</td><td>E</td><td>Nalidixic acid resistance</td><td><a href="#ref310">310</a> </td><td>L
</td></tr>
<tr>
<td>purR</td><td>E</td><td>Adenine resistance in <i>hpt gpt</i></td><td><a href="#ref461">461</a> <a href="#ref462">462</a> </td><td> 
</td></tr>
<tr>
<td>putA</td><td>E, S</td><td>Azetidine carboxylic acid resistance</td><td><a href="#ref511">511</a> <a href="#ref635">635</a> <a href="#ref775">775</a> </td><td>L
</td></tr>
<tr>
<td>putAP</td><td>S</td><td>Baikiain resistance in constitutive background</td><td><a href="#ref187">187</a> </td><td>L
</td></tr>
<tr>
<td>putA</td><td>S</td><td>Proline as N source with glucose as C source</td><td><a href="#ref566">566</a> </td><td> 
</td></tr>
<tr>
<td>putP</td><td>E, S</td><td>Dehydroproline resistance</td><td><a href="#ref193">193</a> <a href="#ref511">511</a> <a href="#ref635">635</a> <a href="#ref732">732</a> <a href="#ref775">775</a> <a href="#ref843">843</a> </td><td>L
</td></tr>
<tr>
<td>putP</td><td>E, S</td><td>Azetidine carboxylic acid resistance</td><td><a href="#ref193">193</a> <a href="#ref511">511</a> <a href="#ref635">635</a> <a href="#ref775">775</a> <a href="#ref843">843</a> </td><td>L
</td></tr>
<tr>
<td>putP</td><td>S</td><td>Lithium-resistant proline utilization as C source</td><td><a href="#ref551">551</a> </td><td>S
</td></tr>
<tr>
<td>puvA</td><td>E</td><td>Psoralen + UV irradiation resistance</td><td><a href="#ref328">328</a> </td><td>C
</td></tr>
<tr>
<td>pyrB</td><td>S</td><td>Arginine-independent growth of <i>pyrH</i> in the presence of uracil</td><td><a href="#ref366">366</a> </td><td>L
</td></tr>
<tr>
<td>pyrC</td><td>S</td><td>Arginine-independent growth of <i>pyrH</i> in the presence of uracil</td><td><a href="#ref366">366</a> </td><td>L
</td></tr>
<tr>
<td>pyrD</td><td>S</td><td>Arginine-independent growth of <i>pyrH</i> in the presence of uracil</td><td><a href="#ref366">366</a> </td><td>L
</td></tr>
<tr>
<td>pyrF</td><td>E</td><td>Fluoro(5)orotic acid resistance</td><td><a href="#ref74">74</a> </td><td>L
</td></tr>
<tr>
<td>pyrH</td><td>S</td><td>Fluoro(5)orotic acid resistance</td><td><a href="#ref865">865</a> </td><td>C
</td></tr>
<tr>
<td>pyrH</td><td>S</td><td>Fluorouracil resistance</td><td><a href="#ref385">385</a> </td><td>C
</td></tr>
<tr>
<td>pyrH</td><td>S</td><td>Fluorouridine resistance</td><td><a href="#ref385">385</a> </td><td>C
</td></tr>
<tr>
<td>pyrH</td><td>E</td><td>Fluoro(5)uracil + 5-fluorouridine resistance in <i>udp</i></td><td><a href="#ref606">606</a> </td><td> 
</td></tr>
<tr>
<td>qmeACDE</td><td>E</td><td>Glycine tolerance</td><td><a href="#ref829">829</a> </td><td> 
</td></tr>
<tr>
<td>rap</td><td>E</td><td>Lambda phage resistance</td><td><a href="#ref279">279</a> </td><td> 
</td></tr>
<tr>
<td>recA</td><td>E</td><td>Lambda prophage induction deficiency during thymine deprivation</td><td><a href="#ref190">190</a> </td><td> 
</td></tr>
<tr>
<td>recF</td><td>E</td><td>Thymineless death resistance in <i>thyA</i></td><td><a href="#ref558">558</a> </td><td> 
</td></tr>
<tr>
<td>recJ</td><td>E</td><td>Thymineless death resistance in <i>thyA</i></td><td><a href="#ref558">558</a> </td><td> 
</td></tr>
<tr>
<td>recO</td><td>E</td><td>Thymineless death resistance in <i>thyA</i></td><td><a href="#ref558">558</a> </td><td> 
</td></tr>
<tr>
<td>recQ</td><td>E</td><td>Thymineless death resistance in <i>thyA</i></td><td><a href="#ref557">557</a> </td><td> 
</td></tr>
<tr>
<td>relA</td><td>E</td><td>UV (near) irradiation resistance</td><td><a href="#ref632">632</a> </td><td>L
</td></tr>
<tr>
<td>relA</td><td>E</td><td>Serine resistance in <i>relA</i></td><td><a href="#ref167">167</a> </td><td> 
</td></tr>
<tr>
<td>rep</td><td>E</td><td>P2 phage resistance</td><td><a href="#ref113">113</a> </td><td> 
</td></tr>
<tr>
<td>rfa (general)</td><td>E, S</td><td>U3 and FO phage resistance</td><td><a href="#ref35">35</a> <a href="#ref484">484</a> <a href="#ref486">486</a> <a href="#ref588">588</a> <a href="#ref833">833</a> </td><td> 
</td></tr>
<tr>
<td>rfa (general)</td><td>E, S</td><td>Ampicillin resistance</td><td><a href="#ref212">212</a> <a href="#ref550">550</a> <a href="#ref562">562</a> </td><td> 
</td></tr>
<tr>
<td>rfa (general)</td><td>S</td><td>Hemin satisfaction of <i>hemA</i> auxotrophy</td><td><a href="#ref364">364</a> </td><td> 
</td></tr>
<tr>
<td>rfa (general)</td><td>S</td><td>Cephalothin resistance</td><td><a href="#ref562">562</a> </td><td> 
</td></tr>
<tr>
<td>rfa (general)</td><td>S</td><td>Bacitracin resistance</td><td><a href="#ref562">562</a> </td><td> 
</td></tr>
<tr>
<td>rfaCDEF</td><td>S</td><td>FO phage resistance</td><td><a href="#ref676">676</a> </td><td> 
</td></tr>
<tr>
<td>rfaD</td><td>E</td><td>C21 phage resistance</td><td><a href="#ref141">141</a> </td><td> 
</td></tr>
<tr>
<td>rfaD</td><td>E</td><td>P1 phage resistance</td><td><a href="#ref141">141</a> </td><td> 
</td></tr>
<tr>
<td>rfaH</td><td>S</td><td>FO phage resistance</td><td><a href="#ref458">458</a> <a href="#ref675">675</a> </td><td> 
</td></tr>
<tr>
<td>rfaP</td><td>E</td><td>U3 phage + K3 phage coresistance</td><td><a href="#ref601">601</a> </td><td> 
</td></tr>
<tr>
<td>rfb</td><td>S</td><td>DCCD resistance in <i>rfa</i></td><td><a href="#ref833">833</a> </td><td> 
</td></tr>
<tr>
<td>rho</td><td>E</td><td>T4 phage resistance</td><td><a href="#ref740">740</a> </td><td> 
</td></tr>
<tr>
<td>rho</td><td>E</td><td>Lambda <i>sus N7 nin-5</i> resistance in P2 lysogen</td><td><a href="#ref352">352</a> <a href="#ref353">353</a> </td><td> 
</td></tr>
<tr>
<td>rho</td><td>E, S</td><td>Polarity suppression of <i>gal</i> leader insertion</td><td><a href="#ref169">169</a> <a href="#ref331">331</a> </td><td> 
</td></tr>
<tr>
<td>ridA</td><td>E</td><td>Rifampin + kasugamycin dependence</td><td><a href="#ref162">162</a> </td><td> 
</td></tr>
<tr>
<td>ridB</td><td>E</td><td>Rifampin dependence</td><td><a href="#ref160">160</a> </td><td> 
</td></tr>
<tr>
<td>rplC</td><td>E</td><td>Tiamulin resistance</td><td><a href="#ref73">73</a> </td><td> 
</td></tr>
<tr>
<td>rplD</td><td>E</td><td>Erythromycin resistance</td><td><a href="#ref725">725</a> <a href="#ref840">840</a> </td><td>S
</td></tr>
<tr>
<td>rplE</td><td>E</td><td>Thiopeptin resistance</td><td><a href="#ref467">467</a> </td><td> 
</td></tr>
<tr>
<td>rplF</td><td>E</td><td>Gentamicin resistance</td><td><a href="#ref101">101</a> </td><td>S
</td></tr>
<tr>
<td>rplK</td><td>E</td><td>Kasugamycin resistance</td><td><a href="#ref159">159</a> </td><td> 
</td></tr>
<tr>
<td>rplK</td><td>E</td><td>DAP starvation of ∆(<i>mal-asd</i>) in the presence of serine, methionine, and glycine</td><td><a href="#ref165">165</a> </td><td> 
</td></tr>
<tr>
<td>rplN</td><td>E</td><td>Kasugamycin resistance</td><td><a href="#ref159">159</a> </td><td> 
</td></tr>
<tr>
<td>rplN</td><td>E</td><td>Lincomycin resistance</td><td><a href="#ref343">343</a> </td><td> 
</td></tr>
<tr>
<td>rplO</td><td>E</td><td>Lincomycin resistance</td><td><a href="#ref343">343</a> </td><td> 
</td></tr>
<tr>
<td>rplV</td><td>E</td><td>Erythromycin resistance</td><td><a href="#ref840">840</a> </td><td> 
</td></tr>
<tr>
<td>rpoA</td><td>E</td><td>P2 <i>vir1</i> phage resistance</td><td><a href="#ref754">754</a> </td><td>S
</td></tr>
<tr>
<td>rpoB</td><td>E</td><td>Lambda <i>c</i>II + rifampin coresistance</td><td><a href="#ref296">296</a> </td><td>S
</td></tr>
<tr>
<td>rpoB</td><td>E</td><td>Streptovaricin resistance</td><td><a href="#ref863">863</a> </td><td>S
</td></tr>
<tr>
<td>rpoB</td><td>E</td><td>Rifampin resistance</td><td><a href="#ref216">216</a> <a href="#ref359">359</a> <a href="#ref369">369</a> </td><td>S
</td></tr>
<tr>
<td>rpoB</td><td>E</td><td>Streptolydigin resistance</td><td><a href="#ref357">357</a> <a href="#ref469">469</a> <a href="#ref688">688</a> </td><td>S
</td></tr>
<tr>
<td>rpoB</td><td>S</td><td>Fluorouracil resistance</td><td><a href="#ref368">368</a> </td><td>S
</td></tr>
<tr>
<td>rpoB</td><td>E</td><td>Lambda phage + 434 phage cross-resistance</td><td><a href="#ref256">256</a> </td><td>S
</td></tr>
<tr>
<td>rpoB</td><td>E</td><td>Lambda <i>Nmar </i>mutant phage resistance</td><td><a href="#ref261">261</a> </td><td>S
</td></tr>
<tr>
<td>rpoB</td><td>E</td><td>Serine resistance in <i>relA</i></td><td><a href="#ref784">784</a> </td><td>S
</td></tr>
<tr>
<td>rpoB</td><td>E</td><td>Rifampin resistance in <i>rpoB</i>+/<i>rpoB</i> (<i>rif</i>) merodiploids</td><td><a href="#ref36">36</a> </td><td>L
</td></tr>
<tr>
<td>rpoB</td><td>E</td><td>T7 phage resistance</td><td><a href="#ref690">690</a> </td><td>S
</td></tr>
<tr>
<td>rpoC</td><td>E</td><td>T7 phage gene 2 resistance</td><td><a href="#ref100">100</a> </td><td> 
</td></tr>
<tr>
<td>rpoD</td><td>E</td><td>Lambda <i>c</i>I71 resistance</td><td><a href="#ref556">556</a> </td><td> 
</td></tr>
<tr>
<td>rpoD</td><td>E</td><td>Arabinose (L) as C source in <i>cya</i> or <i>crp</i></td><td><a href="#ref712">712</a> </td><td>S
</td></tr>
<tr>
<td>rpoH</td><td>E</td><td>Temperature resistance in <i>rpoD</i>(Ts)</td><td><a href="#ref577">577</a> </td><td>S
</td></tr>
<tr>
<td>rpsB</td><td>E</td><td>Kasugamycin resistance</td><td><a href="#ref583">583</a> <a href="#ref861">861</a> </td><td>S
</td></tr>
<tr>
<td>rpsB</td><td>E</td><td>Cold resistance in <i>rpsE</i> cold-sensitive mutants</td><td><a href="#ref559">559</a> </td><td> 
</td></tr>
<tr>
<td>rpsC</td><td>E</td><td>Spectinomycin resistance and sucrose dependence</td><td><a href="#ref195">195</a> </td><td>S
</td></tr>
<tr>
<td>rpsD</td><td>E</td><td>Spectinomycin resistance and sucrose dependence</td><td><a href="#ref195">195</a> </td><td>S
</td></tr>
<tr>
<td>rpsD</td><td>E</td><td>Streptomycin independence of streptomycin-dependent <i>rpsL</i></td><td><a href="#ref304">304</a> </td><td> 
</td></tr>
<tr>
<td>rpsE</td><td>E</td><td>Spectinomycin resistance and sucrose dependence</td><td><a href="#ref195">195</a> <a href="#ref526">526</a> </td><td>S
</td></tr>
<tr>
<td>rpsE</td><td>E, S</td><td>Spectinomycin resistance</td><td><a href="#ref172">172</a> <a href="#ref495">495</a> <a href="#ref634">634</a> <a href="#ref672">672</a> <a href="#ref853">853</a> </td><td>S
</td></tr>
<tr>
<td>rpsE</td><td>E</td><td>Streptomycin independence of streptomycin-dependent <i>rpsL</i></td><td><a href="#ref304">304</a> </td><td> 
</td></tr>
<tr>
<td>rpsG</td><td>E</td><td>Lincomycin resistance</td><td><a href="#ref343">343</a> </td><td>S
</td></tr>
<tr>
<td>rpsI</td><td>E</td><td>Kasugamycin resistance and dependence</td><td><a href="#ref161">161</a> <a href="#ref163">163</a> </td><td>S
</td></tr>
<tr>
<td>rpsJ</td><td>E</td><td>Lambda prophage induction, resistance to</td><td><a href="#ref170">170</a> <a href="#ref246">246</a> </td><td>S
</td></tr>
<tr>
<td>rpsL</td><td>E, S</td><td>Streptomycin resistance</td><td><a href="#ref89">89</a> <a href="#ref495">495</a> <a href="#ref853">853</a> </td><td>S
</td></tr>
<tr>
<td>rpsL</td><td>E</td><td>Neamine dependence</td><td><a href="#ref798">798</a> </td><td> 
</td></tr>
<tr>
<td>rpsL</td><td>E</td><td>Paromomycin resistance or dependence</td><td><a href="#ref853">853</a> </td><td>S
</td></tr>
<tr>
<td>rpsL</td><td>E</td><td>Phenyl galactoside resistance of <i>lacP lacI</i> (nonsense) <i>supE</i></td><td><a href="#ref747">747</a> </td><td> 
</td></tr>
<tr>
<td>rpsM</td><td>E</td><td>Kasugamycin resistance</td><td><a href="#ref159">159</a> </td><td> 
</td></tr>
<tr>
<td>rpsQ</td><td>E</td><td>Neamine resistance</td><td><a href="#ref76">76</a> <a href="#ref119">119</a> </td><td>S
</td></tr>
<tr>
<td>rpsR</td><td>E</td><td>Kasugamycin resistance</td><td><a href="#ref159">159</a> </td><td> 
</td></tr>
<tr>
<td>rrn</td><td>E</td><td>Erythromycin resistance</td><td><a href="#ref704">704</a> </td><td>S
</td></tr>
<tr>
<td>rrn</td><td>E</td><td>Chloramphenicol resistance</td><td><a href="#ref704">704</a> </td><td>S
</td></tr>
<tr>
<td>rrn</td><td>E</td><td>Spectinomycin resistance</td><td><a href="#ref704">704</a> </td><td>S
</td></tr>
<tr>
<td>sbaA</td><td>E</td><td>Serine resistance</td><td><a href="#ref166">166</a> </td><td> 
</td></tr>
<tr>
<td>sbmA</td><td>E</td><td>Microcin B17 resistance</td><td><a href="#ref446">446</a> </td><td> 
</td></tr>
<tr>
<td>sdh</td><td>E</td><td>Succinate-independent growth in <i>lpd</i></td><td><a href="#ref154">154</a> </td><td> 
</td></tr>
<tr>
<td>secA</td><td>E, S</td><td>Azide resistance</td><td><a href="#ref227">227</a> <a href="#ref232">232</a> <a href="#ref584">584</a> <a href="#ref808">808</a> <a href="#ref864">864</a> </td><td> 
</td></tr>
<tr>
<td>secA</td><td>E</td><td>Phenethyl alcohol resistance</td><td><a href="#ref808">808</a> <a href="#ref864">864</a> </td><td> 
</td></tr>
<tr>
<td>semA</td><td>E</td><td>Microcin E492 resistance</td><td><a href="#ref622">622</a> </td><td> 
</td></tr>
<tr>
<td>serA</td><td>E</td><td>Serine hydroxamate resistance</td><td><a href="#ref774">774</a> </td><td>FBI, S
</td></tr>
<tr>
<td>serS</td><td>E</td><td>Serine hydroxamate resistance</td><td><a href="#ref774">774</a> </td><td>S
</td></tr>
<tr>
<td>sidCF</td><td>S</td><td>Albomycin resistance</td><td><a href="#ref87">87</a> <a href="#ref478">478</a> <a href="#ref479">479</a> <a href="#ref674">674</a> </td><td> 
</td></tr>
<tr>
<td>sidK</td><td>S</td><td>Albomycin resistance</td><td><a href="#ref87">87</a> <a href="#ref478">478</a> <a href="#ref479">479</a> <a href="#ref674">674</a> </td><td> 
</td></tr>
<tr>
<td>sidK</td><td>S</td><td>ES18 phage resistance</td><td><a href="#ref87">87</a> <a href="#ref478">478</a> <a href="#ref479">479</a> <a href="#ref674">674</a> </td><td> 
</td></tr>
<tr>
<td>sloB</td><td>E</td><td>Nalidixic acid tolerance</td><td><a href="#ref473">473</a> </td><td> 
</td></tr>
<tr>
<td>sloB</td><td>E</td><td>Amidinopenicillin tolerance</td><td><a href="#ref473">473</a> </td><td> 
</td></tr>
<tr>
<td>sorAT</td><td>E</td><td>Sorbose (L) as C source in crosses with wild strains</td><td><a href="#ref844">844</a> </td><td> 
</td></tr>
<tr>
<td>spcB</td><td>S</td><td>Spectinomycin resistance</td><td><a href="#ref853">853</a> </td><td> 
</td></tr>
<tr>
<td>srlA</td><td>E</td><td>Sorbitol + xylitol resistance</td><td><a href="#ref639">639</a> </td><td>L
</td></tr>
<tr>
<td>srlD</td><td>E</td><td>Fructose as C source in <i>ptsF</i> or <i>ptsM</i></td><td><a href="#ref378">378</a> </td><td> 
</td></tr>
<tr>
<td>sspA</td><td>E</td><td>P1 phage resistance</td><td><a href="#ref835">835</a> </td><td>L
</td></tr>
<tr>
<td>strB</td><td>S</td><td>Streptomycin resistance</td><td><a href="#ref273">273</a> <a href="#ref853">853</a> </td><td> 
</td></tr>
<tr>
<td>strC</td><td>S</td><td>Streptomycin resistance</td><td><a href="#ref648">648</a> </td><td> 
</td></tr>
<tr>
<td>strM</td><td>E</td><td>Streptomycin resistance</td><td><a href="#ref671">671</a> </td><td> 
</td></tr>
<tr>
<td>strM</td><td>E</td><td>Phenyl galactoside resistance of <i>lacI</i>q <i>lacI</i>(nonsense) <i>lacP supL</i></td><td><a href="#ref748">748</a> </td><td> 
</td></tr>
<tr>
<td>sulA</td><td>E</td><td>UV irradiation resistance in <i>lon</i></td><td><a href="#ref373">373</a> </td><td>L
</td></tr>
<tr>
<td>sulA</td><td>E</td><td>Methyl methanesulfonate resistance in <i>lon</i></td><td><a href="#ref373">373</a> </td><td>L
</td></tr>
<tr>
<td>sulA</td><td>E</td><td>Nitrofurantoin resistance in <i>lon</i></td><td><a href="#ref252">252</a> </td><td>L
</td></tr>
<tr>
<td>tabC</td><td>E</td><td>T4 phage resistance</td><td><a href="#ref258">258</a> </td><td> 
</td></tr>
<tr>
<td>tct</td><td>S</td><td>Fluorocitrate resistance</td><td><a href="#ref724">724</a> </td><td>L
</td></tr>
<tr>
<td>tct</td><td>S</td><td>Trifluorocitrate resistance</td><td><a href="#ref723">723</a> </td><td>L
</td></tr>
<tr>
<td>tct</td><td>S</td><td>2-Fluoro-L-erythrocitrate resistance</td><td><a href="#ref32">32</a> </td><td>L
</td></tr>
<tr>
<td>tdk</td><td>E, S</td><td>Azidothymidine resistance</td><td><a href="#ref207">207</a> </td><td>L
</td></tr>
<tr>
<td>tdk</td><td>S</td><td>Fluorodeoxyuridine + uracil resistance in <i>deoA</i></td><td><a href="#ref54">54</a> </td><td> 
</td></tr>
<tr>
<td>tdk</td><td>E</td><td>Fluoro(5)deoxyuridine resistance</td><td><a href="#ref749">749</a> </td><td>L
</td></tr>
<tr>
<td>tdk</td><td>E</td><td>Bromo(5)deoxyuridine + UV irradiation resistance</td><td><a href="#ref319">319</a> <a href="#ref348">348</a> </td><td>L
</td></tr>
<tr>
<td>tet</td><td>E, S</td><td>Fusaric acid resistance</td><td><a href="#ref72">72</a> <a href="#ref487">487</a> </td><td>L
</td></tr>
<tr>
<td>tet</td><td>S</td><td>Quinaldic acid resistance</td><td><a href="#ref72">72</a> </td><td>L
</td></tr>
<tr>
<td>thdA</td><td>E</td><td>Dapsone resistance</td><td><a href="#ref383">383</a> </td><td> 
</td></tr>
<tr>
<td>thi</td><td>E</td><td>Pyrithiamine resistance</td><td><a href="#ref399">399</a> </td><td>C
</td></tr>
<tr>
<td>thrA</td><td>E, S</td><td>Thialysine resistance</td><td><a href="#ref365">365</a> </td><td>S, FBI
</td></tr>
<tr>
<td>thrA</td><td>E</td><td>Serine resistance</td><td><a href="#ref294">294</a> </td><td>S, FBI
</td></tr>
<tr>
<td>thrABC</td><td>E</td><td>Amino(2)-hydroxy(3)-pentoate resistance</td><td><a href="#ref847">847</a> </td><td>D
</td></tr>
<tr>
<td>thrS</td><td>E</td><td>Borrelidin resistance</td><td><a href="#ref248">248</a> <a href="#ref276">276</a> <a href="#ref560">560</a> <a href="#ref596">596</a> </td><td>S, Q
</td></tr>
<tr>
<td>thyA</td><td>E, S</td><td>Aminopterin resistance</td><td><a href="#ref66">66</a> <a href="#ref123">123</a> <a href="#ref325">325</a> <a href="#ref730">730</a> </td><td>L
</td></tr>
<tr>
<td>thyA</td><td>E</td><td>Trimethoprim resistance</td><td><a href="#ref66">66</a> <a href="#ref730">730</a> </td><td>L
</td></tr>
<tr>
<td>tinA (<i>tlr</i>)</td><td>E, S</td><td>Thiolutin resistance</td><td><a href="#ref381">381</a> <a href="#ref382">382</a> <a href="#ref714">714</a> </td><td> 
</td></tr>
<tr>
<td>tmk</td><td>E</td><td>Dideoxy(2′,3′)thymidine resistance</td><td><a href="#ref176">176</a> </td><td>S
</td></tr>
<tr>
<td>tnaA</td><td>E</td><td>Tryptophan-supported growth at 13°C</td><td><a href="#ref568">568</a> </td><td>C
</td></tr>
<tr>
<td>tna</td><td>E</td><td>Indole + methyl(5)tryptophan-supported, glucose-resistant growth in ∆<i>trp</i></td><td><a href="#ref862">862</a> </td><td> 
</td></tr>
<tr>
<td>tol</td><td>E</td><td>Bacteriocin tolerance</td><td><a href="#ref85">85</a> <a href="#ref120">120</a> <a href="#ref173">173</a> <a href="#ref236">236</a> </td><td> 
</td></tr>
<tr>
<td>tolQAB</td><td>E</td><td>Colicin E1 resistance</td><td><a href="#ref449">449</a> </td><td> 
</td></tr>
<tr>
<td>tolQAB</td><td>E</td><td>TuIa phage resistance</td><td><a href="#ref449">449</a> </td><td> 
</td></tr>
<tr>
<td>tolB</td><td>E</td><td>Colicin E resistance</td><td><a href="#ref19">19</a> </td><td>L
</td></tr>
<tr>
<td>tolB</td><td>E</td><td>Azaleucine resistance</td><td><a href="#ref19">19</a> </td><td>L
</td></tr>
<tr>
<td>tolC</td><td>E</td><td>Colicin E1 tolerance</td><td><a href="#ref536">536</a> <a href="#ref653">653</a> </td><td>L
</td></tr>
<tr>
<td>tolD</td><td>E</td><td>Colicin E2 and E3 tolerance</td><td><a href="#ref210">210</a> </td><td> 
</td></tr>
<tr>
<td>tolD</td><td>E</td><td>Ampicillin resistance</td><td><a href="#ref105">105</a> <a href="#ref210">210</a> </td><td> 
</td></tr>
<tr>
<td>tolE</td><td>E</td><td>Colicin E2 and E3 tolerance</td><td><a href="#ref210">210</a> <a href="#ref211">211</a> </td><td> 
</td></tr>
<tr>
<td>tolE</td><td>E</td><td>Ampicillin resistance</td><td><a href="#ref210">210</a> <a href="#ref211">211</a> </td><td> 
</td></tr>
<tr>
<td>tolI</td><td>E</td><td>Colicin Ia and Ib tolerance</td><td><a href="#ref120">120</a> </td><td> 
</td></tr>
<tr>
<td>tolJ</td><td>E</td><td>Colicin L, A, and S4 tolerance</td><td><a href="#ref173">173</a> </td><td> 
</td></tr>
<tr>
<td>tolM</td><td>E</td><td>Colicin M tolerance</td><td><a href="#ref302">302</a> </td><td> 
</td></tr>
<tr>
<td>tolQ</td><td>E</td><td>Colicin group A tolerance</td><td><a href="#ref84a">84a</a> <a href="#ref750">750</a> </td><td> 
</td></tr>
<tr>
<td>tolQ</td><td>E</td><td>Filamentous phage tolerance</td><td><a href="#ref84a">84a</a> <a href="#ref750">750</a> </td><td> 
</td></tr>
<tr>
<td>tolR</td><td>E</td><td>Colicin group A tolerance</td><td><a href="#ref84a">84a</a> <a href="#ref750">750</a> </td><td> 
</td></tr>
<tr>
<td>tolR</td><td>E</td><td>Filamentous phage tolerance</td><td><a href="#ref84a">84a</a> <a href="#ref750">750</a> <a href="#ref751">751</a> </td><td> 
</td></tr>
<tr>
<td>tolZ</td><td>E</td><td>Colicin E2, E3, D, 1a, and 1b tolerance</td><td><a href="#ref503">503</a> </td><td> 
</td></tr>
<tr>
<td>tonB</td><td>S</td><td>ES18 phage resistance</td><td><a href="#ref149">149</a> <a href="#ref741">741</a> </td><td> 
</td></tr>
<tr>
<td>tonB</td><td>E</td><td>T1 phage resistance</td><td><a href="#ref46">46</a> </td><td> 
</td></tr>
<tr>
<td>tonB</td><td>E</td><td>φ80 phage resistance</td><td><a href="#ref46">46</a> </td><td> 
</td></tr>
<tr>
<td>tonB</td><td>E</td><td>Colicin B resistance</td><td><a href="#ref46">46</a> <a href="#ref623">623</a> </td><td> 
</td></tr>
<tr>
<td>tonB</td><td>E, S</td><td>Albomycin resistance</td><td><a href="#ref85">85</a> <a href="#ref505">505</a> <a href="#ref741">741</a> </td><td> 
</td></tr>
<tr>
<td>tonB</td><td>E</td><td>Colicin M resistance</td><td><a href="#ref85">85</a> <a href="#ref523">523</a> </td><td> 
</td></tr>
<tr>
<td>tonB</td><td>S</td><td>Bacteriocin 4-59 resistance</td><td><a href="#ref741">741</a> </td><td> 
</td></tr>
<tr>
<td>tonB</td><td>E</td><td>Cephalosporin E-0702 resistance</td><td><a href="#ref815">815</a> </td><td> 
</td></tr>
<tr>
<td>topA</td><td>S</td><td>Kanamycin resistance</td><td><a href="#ref202">202</a> </td><td> 
</td></tr>
<tr>
<td>topA</td><td>S</td><td>Neomycin resistance</td><td><a href="#ref202">202</a> </td><td> 
</td></tr>
<tr>
<td>topA</td><td>S</td><td>Streptomycin resistance</td><td><a href="#ref202">202</a> </td><td> 
</td></tr>
<tr>
<td>tpp</td><td>E</td><td>Alafosfalin resistance</td><td><a href="#ref603">603</a> </td><td>L
</td></tr>
<tr>
<td>tpp</td><td>E</td><td>Triornithine resistance in <i>opp</i></td><td><a href="#ref44">44</a> </td><td>L
</td></tr>
<tr>
<td>tppA</td><td>S</td><td>Tripeptide (toxic) resistance</td><td><a href="#ref263">263</a> </td><td> 
</td></tr>
<tr>
<td>tppB</td><td>S</td><td>Tripeptide (toxic) resistance</td><td><a href="#ref263">263</a> </td><td> 
</td></tr>
<tr>
<td>trmE?</td><td>E</td><td>Phenyl galactoside resistance of <i>lacl</i>q <i>lacI</i>(nonsense) <i>lacP supL</i></td><td><a href="#ref748">748</a> </td><td> 
</td></tr>
<tr>
<td>trp (generic)</td><td>E, S</td><td>Fluoro(6)tryptophan resistance</td><td><a href="#ref42">42</a> <a href="#ref318">318</a> </td><td>C
</td></tr>
<tr>
<td>trp (generic)</td><td>E, S</td><td>Methyl(5)tryptophan resistance</td><td><a href="#ref42">42</a> <a href="#ref318">318</a> </td><td>C
</td></tr>
<tr>
<td>trpE</td><td>E, S</td><td>Methyl(5)tryptophan resistance</td><td><a href="#ref114">114</a> <a href="#ref539">539</a> </td><td>S, FBI
</td></tr>
<tr>
<td>trpE</td><td>E</td><td>Methyl(3)anthranilate resistance</td><td><a href="#ref308">308</a> </td><td>S, FBI
</td></tr>
<tr>
<td>trpR</td><td>E, S</td><td>Methyl(5)tryptophan resistance</td><td><a href="#ref42">42</a> <a href="#ref137">137</a> <a href="#ref523">523</a> <a href="#ref746">746</a> </td><td> 
</td></tr>
<tr>
<td>trpR</td><td>E</td><td>Chuangxinmycin resistance</td><td><a href="#ref850">850</a> </td><td> 
</td></tr>
<tr>
<td>trpS</td><td>E</td><td>Indolmycin resistance</td><td><a href="#ref75">75</a> </td><td>D
</td></tr>
<tr>
<td>trxA</td><td>E</td><td>T7 phage resistance</td><td><a href="#ref498">498</a> </td><td>L
</td></tr>
<tr>
<td>trzA</td><td>S</td><td>Triazole resistance</td><td><a href="#ref339">339</a> </td><td> 
</td></tr>
<tr>
<td>tsx</td><td>E</td><td>T6 phage resistance</td><td><a href="#ref99">99</a> <a href="#ref299">299</a> <a href="#ref490">490</a> <a href="#ref492">492</a> <a href="#ref620">620</a> </td><td>L, S
</td></tr>
<tr>
<td>tsx</td><td>E</td><td>Colicin K resistance</td><td><a href="#ref99">99</a> <a href="#ref299">299</a> <a href="#ref490">490</a> <a href="#ref492">492</a> <a href="#ref620">620</a> </td><td>L, S
</td></tr>
<tr>
<td>tsx</td><td>E</td><td>Albicidin resistance</td><td><a href="#ref69">69</a> </td><td> 
</td></tr>
<tr>
<td>tufAB</td><td>E</td><td>Kirromycin resistance</td><td><a href="#ref220">220</a> <a href="#ref356">356</a> <a href="#ref763">763</a> <a href="#ref795">795</a> </td><td> 
</td></tr>
<tr>
<td>tufAB</td><td>E, S</td><td>Mocimycin resistance</td><td><a href="#ref335">335</a> <a href="#ref794">794</a> </td><td> 
</td></tr>
<tr>
<td>tyn</td><td>S</td><td>Tyramine as N source</td><td><a href="#ref549">549</a> </td><td> 
</td></tr>
<tr>
<td>tyrA</td><td>E, S</td><td>Fluorophenylalanine resistance</td><td><a href="#ref270">270</a> <a href="#ref729">729</a> </td><td>C
</td></tr>
<tr>
<td>tyrA</td><td>E</td><td>Amino(4)phenylalanine resistance</td><td><a href="#ref504">504</a> </td><td>C
</td></tr>
<tr>
<td>tyrR</td><td>E, S</td><td>Amino(4)phenylalanine resistance</td><td><a href="#ref95">95</a> <a href="#ref504">504</a> <a href="#ref811">811</a> </td><td> 
</td></tr>
<tr>
<td>tyrR</td><td>E</td><td>Fluorotyrosine resistance</td><td><a href="#ref130">130</a> </td><td> 
</td></tr>
<tr>
<td>tyrR</td><td>S</td><td>Fluoro(4)phenylalanine resistance</td><td><a href="#ref729">729</a> </td><td> 
</td></tr>
<tr>
<td>tyrS</td><td>E</td><td>Fluorotyrosine resistance</td><td><a href="#ref662">662</a> </td><td>S
</td></tr>
<tr>
<td>ubi (generic)</td><td>E</td><td>Neomycin resistance</td><td><a href="#ref265">265</a> </td><td>L
</td></tr>
<tr>
<td>ubiF</td><td>S</td><td>Fluoro(5)uracil + carbamylaspartate resistance</td><td><a href="#ref404">404</a> <a href="#ref866">866</a> </td><td>L
</td></tr>
<tr>
<td>ubiF</td><td>E</td><td>Gentamicin resistance</td><td><a href="#ref542">542</a> </td><td> 
</td></tr>
<tr>
<td>ubiF</td><td>E</td><td>Streptomycin resistance</td><td><a href="#ref542">542</a> </td><td> 
</td></tr>
<tr>
<td>udhA</td><td>S</td><td>Fluoro(5)uridine resistance</td><td><a href="#ref834">834</a> </td><td>L
</td></tr>
<tr>
<td>udk</td><td>E</td><td>Fluoro(5)uridine + uracil resistance</td><td><a href="#ref380">380</a> </td><td>L
</td></tr>
<tr>
<td>udk</td><td>E</td><td>Fluoro(5)uridine resistance in <i>upp</i></td><td><a href="#ref565">565</a> </td><td>L
</td></tr>
<tr>
<td>udp</td><td>E</td><td>Fluoro(5)uracil + adenosine resistance</td><td><a href="#ref606">606</a> <a href="#ref617">617</a> </td><td>L
</td></tr>
<tr>
<td>ugpAB</td><td>E</td><td>Fluorohydroxyacetone phosphate resistance</td><td><a href="#ref515">515</a> </td><td>L
</td></tr>
<tr>
<td>ugpAB</td><td>E</td><td>Dihydroxybutylphosphonate resistance in <i>glpT</i></td><td><a href="#ref691">691</a> <a href="#ref692">692</a> </td><td>L
</td></tr>
<tr>
<td>ugpAB</td><td>E</td><td>Glycerol-3-phosphate as C source in <i>glpT</i></td><td><a href="#ref691">691</a> </td><td>C
</td></tr>
<tr>
<td>uhpR</td><td>E</td><td>Glucosamine 6-phosphate as C source</td><td><a href="#ref191">191</a> </td><td>C
</td></tr>
<tr>
<td>uhpR</td><td>S</td><td>Glucose 1-phosphate as C source</td><td><a href="#ref192">192</a> </td><td>C
</td></tr>
<tr>
<td>uhpR</td><td>E</td><td>Fructose 1-phosphate as C source</td><td><a href="#ref224">224</a> </td><td>C
</td></tr>
<tr>
<td>uhpT</td><td>E</td><td>Chloro-3-hydroxyacetone resistance in <i>uhp</i>(Con)</td><td><a href="#ref515">515</a> </td><td>L
</td></tr>
<tr>
<td>uhpT</td><td>E</td><td>Fluoro-3-hydroxyacetone resistance in <i>uhp</i>(Con)</td><td><a href="#ref515">515</a> </td><td>L
</td></tr>
<tr>
<td>uhpT</td><td>E</td><td>Dihydroxybutylphosphonate resistance in <i>uhp</i>(Con)</td><td><a href="#ref285">285</a> </td><td>L
</td></tr>
<tr>
<td>uhpT</td><td>E</td><td>Hydroxybutylphosphonate resistance in <i>ubp</i>(Con)</td><td><a href="#ref285">285</a> </td><td>L
</td></tr>
<tr>
<td>uhpT</td><td>E</td><td>Deoxydihydroxyphosphonyl methyl fructose resistance</td><td><a href="#ref285">285</a> </td><td>L
</td></tr>
<tr>
<td>uhpT</td><td>E</td><td>L-Glyceraldehyde-3-phosphate resistance in <i>uhp</i>(Con)</td><td><a href="#ref285">285</a> </td><td>L
</td></tr>
<tr>
<td>uhpT</td><td>E</td><td>Fosfomycin resistance</td><td><a href="#ref230">230</a> <a href="#ref389">389</a> <a href="#ref800">800</a> </td><td>L
</td></tr>
<tr>
<td>uhpT</td><td>E</td><td>Deoxy(2)glucose-6-phosphate resistance</td><td><a href="#ref214">214</a> </td><td>L
</td></tr>
<tr>
<td>uidA</td><td>E</td><td>Galacturonide as C source</td><td><a href="#ref742">742</a> </td><td>C
</td></tr>
<tr>
<td>uidA</td><td>E</td><td>Methylgalacturonide as C source</td><td><a href="#ref573">573</a> </td><td>C
</td></tr>
<tr>
<td>uidA</td><td>E</td><td>Methylglucuronide + glycerol-supported growth in <i>eda</i></td><td><a href="#ref572">572</a> </td><td>L
</td></tr>
<tr>
<td>uidR</td><td>E</td><td>Galacturonide as C source</td><td><a href="#ref742">742</a> </td><td>C
</td></tr>
<tr>
<td>uidR</td><td>E</td><td>Methylgalacturonide as C source</td><td><a href="#ref573">573</a> </td><td>C
</td></tr>
<tr>
<td>ung</td><td>E</td><td>Bromodeoxyuridine + UV light resistance</td><td><a href="#ref856">856</a> </td><td>L
</td></tr>
<tr>
<td>ung</td><td>E</td><td>T4 phage (uracil containing) resistance</td><td><a href="#ref203">203</a> </td><td>L
</td></tr>
<tr>
<td>upp</td><td>E, S</td><td>Fluoro(5)uracil resistance</td><td><a href="#ref53">53</a> <a href="#ref605">605</a> <a href="#ref617">617</a> </td><td>L
</td></tr>
<tr>
<td>upp</td><td>E</td><td>Aza(6)uracil resistance</td><td><a href="#ref605">605</a> <a href="#ref617">617</a> </td><td>L
</td></tr>
<tr>
<td>upp</td><td>E</td><td>Canavanine + aza(6)uracil resistance</td><td><a href="#ref605">605</a> <a href="#ref617">617</a> </td><td>L
</td></tr>
<tr>
<td>ushA</td><td>E</td><td>Fluoro(5)uracil + 5′-AMP resistance in <i>upp</i></td><td><a href="#ref52">52</a> </td><td>L
</td></tr>
<tr>
<td>uvrA</td><td>E</td><td>UV-irradiated lytic phage resistance</td><td><a href="#ref332">332</a> </td><td> 
</td></tr>
<tr>
<td>uvrB</td><td>E</td><td>UV-irradiated lytic phage resistance</td><td><a href="#ref332">332</a> </td><td> 
</td></tr>
<tr>
<td>uvrC</td><td>E</td><td>UV-irradiated lytic phage resistance</td><td><a href="#ref332">332</a> </td><td> 
</td></tr>
<tr>
<td>uvrD</td><td>E</td><td>UV-irradiated lytic phage resistance</td><td><a href="#ref332">332</a> </td><td> 
</td></tr>
<tr>
<td>uxaBC</td><td>E</td><td>Hexuronate resistance in <i>eda</i></td><td><a href="#ref742">742</a> </td><td>L
</td></tr>
<tr>
<td>uxuAB</td><td>E</td><td>Hexuronate resistance in <i>eda</i></td><td><a href="#ref742">742</a> </td><td>L
</td></tr>
<tr>
<td>uxuR</td><td>E</td><td>Methylgalacturonide as C source</td><td><a href="#ref574">574</a> </td><td>C
</td></tr>
<tr>
<td>valS</td><td>E</td><td>Thymineless death resistance in <i>thyA</i></td><td><a href="#ref394">394</a> </td><td>S
</td></tr>
<tr>
<td>xapA</td><td>E</td><td>Fluoro(5)uracil + adenosine resistance in <i>upp deoD xapR(Con)</i></td><td><a href="#ref111">111</a> </td><td>L
</td></tr>
<tr>
<td>xapR</td><td>E</td><td>Adenosine as C source in <i>upp deoD</i></td><td><a href="#ref111">111</a> </td><td>C
</td></tr>
<tr>
<td>xapR</td><td>E</td><td>Inosine as C source in <i>upp deoD</i></td><td><a href="#ref111">111</a> </td><td>C
</td></tr>
<tr>
<td>xylE</td><td>E</td><td>Xylose resistance in <i>fda</i></td><td><a href="#ref175">175</a> </td><td>L
</td></tr>
</table>
<h1><a name="ref"></a>References</h1>
<table>
<tr><td><a name="ref1"></a>1</td><td>Abouhamad, W. N., M. Manson, M. M. Gibson, and C. F. Higgins <i>1991</i>. Peptide transport and chemotaxis in <i>Escherichia coli</i> and <i>Salmonella</i> <i>typhimurium:</i> characterization of the dipeptide permease <i>dpp</i> and the dipeptide-binding protein. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Peptide transport and chemotaxis in Escherichia coli and Salmonella typhimurium: characterization of the dipeptide permease dpp and the dipeptide-binding protein">google scholar</a></td></tr>
<tr><td><a name="ref1a"></a>1a</td><td>Ackerman, R. S., N. R. Cozzarelli, and W. Epstein <i>1974</i>. Accumulation of toxic concentrations of methylglyoxal by <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Accumulation of toxic concentrations of methylglyoxal by Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref2"></a>2</td><td>Adhya, S., P. Cleary, and A. Campbell <i>1968</i>. A deletion analysis of prophage lambda and adjacent genetic regions. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A deletion analysis of prophage lambda and adjacent genetic regions">google scholar</a></td></tr>
<tr><td><a name="ref3"></a>3</td><td>Ahmad, S. I., and R. H. Pritchard <i>1969</i>. A map of four genes specifying enzymes involved in catabolism of nucleosides and deoxynucleosides in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A map of four genes specifying enzymes involved in catabolism of nucleosides and deoxynucleosides in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref4"></a>4</td><td>Ahmad, S. I., and R. H. Pritchard <i>1972</i>. Location of the gene specifying cytosine deaminase in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Location of the gene specifying cytosine deaminase in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref5"></a>5</td><td>Alaeddinoglu, N. C., and H. P. Charles <i>1979</i>. Transfer of a gene for sucrose utilization into <i>Escherichia coli</i> K12, and consequent failure of expression of genes for D-serine utilization. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Transfer of a gene for sucrose utilization into Escherichia coli K12, and consequent failure of expression of genes for D-serine utilization">google scholar</a></td></tr>
<tr><td><a name="ref6"></a>6</td><td>Alderman, E. M., S. S. Dills, T. Melton, and W. J. Dobrogosz <i>1979</i>. Cyclic AMP regulation of the bacteriophage T6 colicin K receptor in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Cyclic AMP regulation of the bacteriophage T6 colicin K receptor in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref7"></a>7</td><td>Alexander, R. R., J. M. Calvo, and M. Freundllch <i>1971</i>. Mutants of <i>Salmonella typhimurium</i> with an altered leucyl-transfer ribonucleic acid synthetase. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutants of Salmonella typhimurium with an altered leucyl-transfer ribonucleic acid synthetase">google scholar</a></td></tr>
<tr><td><a name="ref8"></a>8</td><td>Alper, M. D., and B. N. Ames <i>1975</i>. Cyclic 3′,5′-adenosine monophosphate phosphodiesterase mutants of <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Cyclic 3′,5′-adenosine monophosphate phosphodiesterase mutants of Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref9"></a>9</td><td>Alper, M. D., and B. N. Ames <i>1975</i>. Positive selection of mutants with deletions of the <i>gal-chl</i> region of the <i>Salmonella</i> chromosome as a screening procedure for mutagens that cause deletions. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Positive selection of mutants with deletions of the gal-chl region of the Salmonella chromosome as a screening procedure for mutagens that cause deletions">google scholar</a></td></tr>
<tr><td><a name="ref10"></a>10</td><td>Alper, M. D., and B. N. Ames <i>1978</i>. Transport of antibiotics and metabolite analogs by systems under cyclic AMP control: positive selection of <i>Salmonella typhimurium cya</i> and <i>crp</i> mutants. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Transport of antibiotics and metabolite analogs by systems under cyclic AMP control: positive selection of Salmonella typhimurium cya and crp mutants">google scholar</a></td></tr>
<tr><td><a name="ref11"></a>11</td><td>Amaral, D., and H. L. Kornberg <i>1975</i>. Regulation of fructose uptake by glucose in <i>Escherichia</i> <i>coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation of fructose uptake by glucose in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref12"></a>12</td><td>Ames, G. F.-L <i>1964</i>. Uptake of amino acids by <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Uptake of amino acids by Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref13"></a>13</td><td>Ames, G. F.-L., D. P. Biek, and E. N. Spudich <i>1978</i>. Duplications of histidine transport genes in <i>Salmonella typhimurium</i> and their use for the selection of deletion mutants. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Duplications of histidine transport genes in Salmonella typhimurium and their use for the selection of deletion mutants">google scholar</a></td></tr>
<tr><td><a name="ref14"></a>14</td><td>Ames, G. F.-L., and J. Lever <i>1970</i>. Components of histidine transport: histidine-binding proteins and <i>hisP</i> protein. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Components of histidine transport: histidine-binding proteins and hisP protein">google scholar</a></td></tr>
<tr><td><a name="ref15"></a>15</td><td>Ames, G. F.-L., K. D. Noel, H. Tabor, E. N. Spudich, K. Nikaido, J. Afong, and F. Ardeshir <i>1977</i>. Fine-structure map of the histidine transport genes in <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Fine-structure map of the histidine transport genes in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref16"></a>16</td><td>Ammer, J., M. Brennenstuhl, P. Schindler, J.-V. Holtje, and H. Zahner <i>1979</i>. Phosphorylation of streptozotocin during uptake via the phosphoenolpyruvate:sugar phosphotransferase system in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Phosphorylation of streptozotocin during uptake via the phosphoenolpyruvate:sugar phosphotransferase system in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref17"></a>17</td><td>Anderson, J. J., and D. L. Oxender <i>1977</i>. <i>Escherichia coli</i> transport mutants lacking binding protein and other components of the branched-chain amino acid transport systems. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Escherichia coli transport mutants lacking binding protein and other components of the branched-chain amino acid transport systems">google scholar</a></td></tr>
<tr><td><a name="ref18"></a>18</td><td>Anderson, J. J., S. C. Quay, and D. L. Oxender <i>1976</i>. Mapping of two loci affecting the regulation of branched-chain amino acid transport in <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mapping of two loci affecting the regulation of branched-chain amino acid transport in Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref19"></a>19</td><td>Anderson, J. J., J. M. Wilson, and D. L. Oxender <i>1979</i>. Defective transport and other phenotypes of a periplasmic “leaky”  mutant of <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Defective transport and other phenotypes of a periplasmic “leaky”  mutant of Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref20"></a>20</td><td>Anderson, R. P., and J. R. Roth <i>1977</i>. Tandem genetic duplications in phage and bacteria. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Tandem genetic duplications in phage and bacteria">google scholar</a></td></tr>
<tr><td><a name="ref21"></a>21</td><td>Anderson, R. P., and J. R. Roth <i>1978</i>. Tandem genetic duplications in <i>Salmonella typhimurium:</i> amplification of the histidine operon. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Tandem genetic duplications in Salmonella typhimurium: amplification of the histidine operon">google scholar</a></td></tr>
<tr><td><a name="ref22"></a>22</td><td>Andrews, J. C., and S. A. Short <i>1985</i>. Genetic analysis of <i>Escherichia coli</i> oligopeptide transport mutants. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic analysis of Escherichia coli oligopeptide transport mutants">google scholar</a></td></tr>
<tr><td><a name="ref23"></a>23</td><td>Anton, D <i>1968</i>. Histidine regulatory mutants in <i>Salmonella typhimurium.</i> V. Two new classes of histidine regulatory mutants. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Histidine regulatory mutants in Salmonella typhimurium. V. Two new classes of histidine regulatory mutants">google scholar</a></td></tr>
<tr><td><a name="ref24"></a>24</td><td>Anton, D. N <i>1981</i>. <i>envB</i> mutations confer UV sensitivity to <i>Salmonella typhimurium</i> and UV resistance to <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=envB mutations confer UV sensitivity to Salmonella typhimurium and UV resistance to Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref25"></a>25</td><td>Aono, R., M. Yamasaki, and G. Tamura <i>1979</i>. High and selective resistance to mecillinam in adenylate cyclase-deficient or cyclic adenosine 3′,5′-monophosphate receptor protein-deficient mutants of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=High and selective resistance to mecillinam in adenylate cyclase-deficient or cyclic adenosine 3′,5′-monophosphate receptor protein-deficient mutants of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref26"></a>26</td><td>Apirion, D <i>1967</i>. Three genes that affect <i>Escherichia coli</i> ribosomes. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Three genes that affect Escherichia coli ribosomes">google scholar</a></td></tr>
<tr><td><a name="ref27"></a>27</td><td>Apirion, D., and D. Schlessinger <i>1968</i>. Coresistance to neomycin and kanamycin by mutations in an <i>Escherichia coli</i> locus that affects ribosomes. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Coresistance to neomycin and kanamycin by mutations in an Escherichia coli locus that affects ribosomes">google scholar</a></td></tr>
<tr><td><a name="ref28"></a>28</td><td>Archibold, E. R., and L. S. Williams <i>1973</i>. Regulation of methionyl-transfer ribonucleic acid synthetase formation in <i>Escherichia coli</i> and <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation of methionyl-transfer ribonucleic acid synthetase formation in Escherichia coli and Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref29"></a>29</td><td>Argast, M., D. Ludtke, T. J. Silhavy, and W. Boos <i>1978</i>. A second transport system for <i>sn</i>- glycerol-3-phosphate in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A second transport system for sn- glycerol-3-phosphate in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref30"></a>30</td><td>Armstrong, G. D., and H. Yamazaki <i>1977</i>. Isolation and characterization of catabolite repression resistant mutants of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation and characterization of catabolite repression resistant mutants of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref31"></a>31</td><td>Artman, M., and S. Werthamer <i>1979</i>. Use of streptomycin and cyclic adenosine 5′- monophosphate in the isolation of mutants deficient in CAP protein. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Use of streptomycin and cyclic adenosine 5′- monophosphate in the isolation of mutants deficient in CAP protein">google scholar</a></td></tr>
<tr><td><a name="ref32"></a>32</td><td>Ashton, D. M., G. D. Sweet, J. M. Somers, and W. W. Kay <i>1980</i>. Citrate transport in <i>Salmonella typhimurium:</i> studies with 2-fluoro-L-<i>erythro</i>-citrate as a substrate. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Citrate transport in Salmonella typhimurium: studies with 2-fluoro-L-erythro-citrate as a substrate">google scholar</a></td></tr>
<tr><td><a name="ref33"></a>33</td><td>Astvatsaturyants, G. V., A. F. Lysenkov, Y. V. Smirnov, R. and R. S. Shakulov <i>1988</i>. Mutants of <i>Escherichia coli</i> with impaired feedback inhibition of histidine biosynthesis. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutants of Escherichia coli with impaired feedback inhibition of histidine biosynthesis">google scholar</a></td></tr>
<tr><td><a name="ref34"></a>34</td><td>Aswad, D., and D. E. Koshland, Jr <i>1975</i>. Isolation, characterization and complementation of <i>Salmonella typhimurium</i> chemotaxis mutants. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation, characterization and complementation of Salmonella typhimurium chemotaxis mutants">google scholar</a></td></tr>
<tr><td><a name="ref35"></a>35</td><td>Austin, E. A., J. F. Graves, L. A. Hite, C. T. Parker, and C. A. Schnaitman <i>1990</i>. Genetic analysis of lipopolysaccharide core biosynthesis by <i>Escherichia coli</i> K-12 insertion mutagenesis of the <i>rfa</i> locus. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic analysis of lipopolysaccharide core biosynthesis by Escherichia coli K-12 insertion mutagenesis of the rfa locus">google scholar</a></td></tr>
<tr><td><a name="ref36"></a>36</td><td>Austin, S., and J. Scaife <i>1970</i>. A new method for selecting RNA polymerase mutants. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A new method for selecting RNA polymerase mutants">google scholar</a></td></tr>
<tr><td><a name="ref37"></a>37</td><td>Ayling, P. D <i>1981</i>. Methionine sulfoxide is transported by high-affinity methionine and glutamine transport systems in <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Methionine sulfoxide is transported by high-affinity methionine and glutamine transport systems in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref38"></a>38</td><td>Ayling, P. D., and E. S. Bridgeland <i>1972</i>. Methionine transport in wild-type and transport- defective mutants of <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Methionine transport in wild-type and transport- defective mutants of Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref39"></a>39</td><td>Bachi, B., and H. L. Kornberg <i>1975</i>. Genes involved in the uptake and catabolism of gluconate by <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genes involved in the uptake and catabolism of gluconate by Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref40"></a>40</td><td>Bachmann, B. J <i>1983</i>. Linkage map of <i>Escherichia coli</i> K-12, edition 7. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Linkage map of Escherichia coli K-12, edition 7">google scholar</a></td></tr>
<tr><td><a name="ref41"></a>41</td><td>Bachmann, B. J <i>1990</i>. Linkage map of <i>Escherichia coli </i>K-12, edition 8. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Linkage map of Escherichia coli K-12, edition 8">google scholar</a></td></tr>
<tr><td><a name="ref42"></a>42</td><td>Balbinder, E., R. Callahan III, P. P. McCann, J. C. Cordero, A. R. Weber, A. M. Smith, and F. Angelosanto <i>1970</i>. Regulatory mutants of the tryptophan operon of <i>Salmonella typhimurium</i>. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulatory mutants of the tryptophan operon of Salmonella typhimurium">google scholar</a></td></tr>
<tr><td><a name="ref43"></a>43</td><td>Barak, Z., and C. Gilvarg <i>1974</i>. Triornithine-resistant strains of <i>Escherichia coli.</i> Isolation, definition, and genetic studies. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Triornithine-resistant strains of Escherichia coli. Isolation, definition, and genetic studies">google scholar</a></td></tr>
<tr><td><a name="ref44"></a>44</td><td>Barak, Z., and C. Gilvarg <i>1975</i>. Specialized peptide transport system in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Specialized peptide transport system in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref45"></a>45</td><td>Barrett, E. L., and D. L. Riggs <i>1982</i>. Evidence for a second nitrate reductase activity that is distinct from the respiratory enzyme in <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Evidence for a second nitrate reductase activity that is distinct from the respiratory enzyme in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref46"></a>46</td><td>Bassford, P. J., Jr., C. Bradbeer, R. J. Kadner, and C. A. Schnaitman <i>1976</i>. Transport of vitamin B12 in <i>tonB</i> mutants of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Transport of vitamin B12 in tonB mutants of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref47"></a>47</td><td>Bassford, P. J., Jr., R. J. Kadner, and C. A. Schnaitman <i>1977</i>. Biosynthesis of the outer membrane receptor for vitamin B12, E colicins, and bacteriophage BF23 by <i>Escherichia coli:</i> kinetics of phenotypic expression after the introduction of <i>bfe</i>+ and <i>bfe</i> alleles. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Biosynthesis of the outer membrane receptor for vitamin B12, E colicins, and bacteriophage BF23 by Escherichia coli: kinetics of phenotypic expression after the introduction of bfe+ and bfe alleles">google scholar</a></td></tr>
<tr><td><a name="ref48"></a>48</td><td>Baughman, G. A., and S. R. Fahnestock <i>1979</i>. Chloramphenicol resistance mutation in <i>Escherichia coli</i> which maps in the major ribosomal protein gene cluster. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Chloramphenicol resistance mutation in Escherichia coli which maps in the major ribosomal protein gene cluster">google scholar</a></td></tr>
<tr><td><a name="ref49"></a>49</td><td>Baumberg, S <i>1970</i>. Acetylhistidine as substrate for acetylornithinase: a new system for the selection of arginine regulation mutants in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Acetylhistidine as substrate for acetylornithinase: a new system for the selection of arginine regulation mutants in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref50"></a>50</td><td>Baumberg, S <i>1976</i>. Genetic control of arginine metabolism in prokaryotes.</td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic control of arginine metabolism in prokaryotes">google scholar</a></td></tr>
<tr><td><a name="ref51"></a>51</td><td>Beacham, I. R., and S. Garrett <i>1980</i>. Isolation of <i>Escherichia coli</i> mutants <i>(cpdB)</i> deficient in periplasmic 2′:3′-cyclic phosphodiesterase and genetic mapping of the <i>cpdB</i> locus. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation of Escherichia coli mutants (cpdB) deficient in periplasmic 2′:3′-cyclic phosphodiesterase and genetic mapping of the cpdB locus">google scholar</a></td></tr>
<tr><td><a name="ref52"></a>52</td><td>Beacham, I. R., R. Kahana, L. Levy, and E. Yagil <i>1973</i>. Mutants of <i>Escherichia coli</i> K-12 “cryptic” or deficient in 5′-nucleotidase (uridine diphosphate-sugar hydrolase) and 3′-nucleotidase (cyclic phosphodiesterase) activity. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutants of Escherichia coli K-12 “cryptic” or deficient in 5′-nucleotidase (uridine diphosphate-sugar hydrolase) and 3′-nucleotidase (cyclic phosphodiesterase) activity">google scholar</a></td></tr>
<tr><td><a name="ref53"></a>53</td><td>Beck, C. F., and J. L. Ingraham <i>1971</i>. Location on the chromosome of <i>Salmonella typhimurium</i> of genes governing pyrimidine metabolism. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Location on the chromosome of Salmonella typhimurium of genes governing pyrimidine metabolism">google scholar</a></td></tr>
<tr><td><a name="ref54"></a>54</td><td>Beck, C. F., J. L. Ingraham, and J. Neuhard <i>1972</i>. Location on the chromosome of <i>Salmonella</i> <i>typhimurium</i> of genes governing pyrimidine metabolism. II. Uridine kinase, cytosine deaminase, and thymidine kinase. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Location on the chromosome of Salmonella typhimurium of genes governing pyrimidine metabolism. II. Uridine kinase, cytosine deaminase, and thymidine kinase">google scholar</a></td></tr>
<tr><td><a name="ref55"></a>55</td><td>Begg, K. J., G. F. Hatfull, and W. D. Donachie <i>1980</i>. Identification of new genes in a cell envelope-cell division gene cluster of <i>Escherichia coli:</i> cell division gene <i>ftsQ.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Identification of new genes in a cell envelope-cell division gene cluster of Escherichia coli: cell division gene ftsQ.">google scholar</a></td></tr>
<tr><td><a name="ref56"></a>56</td><td>Belfort, M., and D. L. Wulff <i>1973</i>. Genetic and biochemical investigation of the <i>Escherichia coli</i> mutant <i>hfl-1</i> which is lysogenized at high frequency by bacteriophage lambda. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic and biochemical investigation of the Escherichia coli mutant hfl-1 which is lysogenized at high frequency by bacteriophage lambda">google scholar</a></td></tr>
<tr><td><a name="ref57"></a>57</td><td>Beljanski, M., and M. Beljanski <i>1957</i>. Sur la formation d’enzymes respiratoires chez un mutant d’<i>Escherichia coli</i> streptomycino-résistant et auxotrophe pour l’hemine. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Sur la formation d’enzymes respiratoires chez un mutant d’Escherichia coli streptomycino-résistant et auxotrophe pour l’hemine">google scholar</a></td></tr>
<tr><td><a name="ref58"></a>58</td><td>Beljanski, M., P. Pourgarel, and M. Beljanski <i>1970</i>. Showdomycine et biosynthèse d’ARN non complementaires de l’ADN. 1. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Showdomycine et biosynthèse d’ARN non complementaires de l’ADN. 1">google scholar</a></td></tr>
<tr><td><a name="ref59"></a>59</td><td>Bennett, R. L., and M. H. Malamy <i>1970</i>. Arsenate resistant mutants of <i>Escherichia coli</i> and phosphate transport. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Arsenate resistant mutants of Escherichia coli and phosphate transport">google scholar</a></td></tr>
<tr><td><a name="ref60"></a>60</td><td>Benson, C. E., S. H. Love, and C. N. Remy <i>1970</i>. Inhibition of de novo purine biosynthesis and interconversion by 6-methyl purine in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Inhibition of de novo purine biosynthesis and interconversion by 6-methyl purine in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref61"></a>61</td><td>Berg, C. M <i>1990</i>. The branched chain amino acid transaminase genes and their products in <i>Escherichia coli,</i>.</td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The branched chain amino acid transaminase genes and their products in Escherichia coli,">google scholar</a></td></tr>
<tr><td><a name="ref62"></a>62</td><td>Berg, C. M., and J. J. Rossi <i>1974</i>. Proline excretion and indirect suppression in <i>Escherichia coli</i> and <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Proline excretion and indirect suppression in Escherichia coli and Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref63"></a>63</td><td>Bergler, H., G. Hogenauer, and F. Thurnowsky <i>1992</i>. Sequences of the <i>envM</i> gene and of two mutated alleles in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Sequences of the envM gene and of two mutated alleles in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref64"></a>64</td><td>Berman-Kurtz, M., E. C. C. Lin, and D. P. Richey <i>1971</i>. Promoter-like mutant with increased expression of the glycerol kinase operon of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Promoter-like mutant with increased expression of the glycerol kinase operon of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref65"></a>65</td><td>Bernardi, A., and P. Leder <i>1970</i>. Protein biosynthesis in <i>Escherichia coli:</i> purification and characteristics of a mutant G factor. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Protein biosynthesis in Escherichia coli: purification and characteristics of a mutant G factor">google scholar</a></td></tr>
<tr><td><a name="ref66"></a>66</td><td>Bertino, J. B., and K. A. Stacey <i>1966</i>. A suggested mechanism for the selective procedure for isolating thymine-requiring mutants of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A suggested mechanism for the selective procedure for isolating thymine-requiring mutants of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref67"></a>67</td><td>Beverin, S., D. E. Sheppard, and S. S. Park <i>1971</i>. D-Fucose as a gratuitous inducer of the L- arabinose operon in strains of <i>Escherichia coli</i> B/r mutant in gene <i>araC.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=D-Fucose as a gratuitous inducer of the L- arabinose operon in strains of Escherichia coli B/r mutant in gene araC.">google scholar</a></td></tr>
<tr><td><a name="ref68"></a>68</td><td>Bewick, M. A., and T. C. Y. Lo <i>1980</i>. Localization of the dicarboxylate binding protein in the cell envelope of <i>Escherichia coli</i> K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Localization of the dicarboxylate binding protein in the cell envelope of Escherichia coli K12">google scholar</a></td></tr>
<tr><td><a name="ref69"></a>69</td><td>Birch, R. G., J. M. Pemberton, and W. V. S. Basnayake <i>1990</i>. Stable albicidin resistance in <i>Escherichia coli</i> involves an altered outer-membrane nucleoside uptake system. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Stable albicidin resistance in Escherichia coli involves an altered outer-membrane nucleoside uptake system">google scholar</a></td></tr>
<tr><td><a name="ref70"></a>70</td><td>Black, P. N <i>1988</i>. The <i>fadL</i> gene product of <i>Escherichia coli</i> is an outer membrane protein required for uptake of long-chain fatty acids and involved in sensitivity in bacteriophage T2. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The fadL gene product of Escherichia coli is an outer membrane protein required for uptake of long-chain fatty acids and involved in sensitivity in bacteriophage T2">google scholar</a></td></tr>
<tr><td><a name="ref71"></a>71</td><td>Blank, J., and P. Hoffee <i>1972</i>. Regulatory mutants of the <i>deo</i> regulon in <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulatory mutants of the deo regulon in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref72"></a>72</td><td>Bochner, B. R., H.-C. Huang, G. L. Schieven, and B. N. Ames <i>1980</i>. Positive selection for loss of tetracycline resistance. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Positive selection for loss of tetracycline resistance">google scholar</a></td></tr>
<tr><td><a name="ref73"></a>73</td><td>Böck, A., F. Turnowsky, and G. Hoegenauer <i>1982</i>. Tiamulin resistance mutations in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Tiamulin resistance mutations in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref74"></a>74</td><td>Boeke, J. D., F. LaCroute, and G. R. Fink <i>1984</i>. A positive selection for mutants lacking orotidine-5′-phosphate decarboxylase activity in yeast: 5-fluoro-orotic acid resistance. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A positive selection for mutants lacking orotidine-5′-phosphate decarboxylase activity in yeast: 5-fluoro-orotic acid resistance">google scholar</a></td></tr>
<tr><td><a name="ref75"></a>75</td><td>Bogosian, G., P. V. Haydock, and R. L. Somerville <i>1983</i>. Indolmycin-mediated inhibition and stimulation of transcription at the <i>trp</i> promoter of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Indolmycin-mediated inhibition and stimulation of transcription at the trp promoter of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref76"></a>76</td><td>Bollen, A., T. Cabezon, M. De Wilde, R. Villarroel, and A. Herzog <i>1975</i>. Alteration of ribosomal protein S17 by mutation linked to neamine resistance in <i>Escherichia coli.</i> I. General properties of <i>neaA</i> mutants. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Alteration of ribosomal protein S17 by mutation linked to neamine resistance in Escherichia coli. I. General properties of neaA mutants">google scholar</a></td></tr>
<tr><td><a name="ref77"></a>77</td><td>Boos, W., C. Bantlow, D. Benner, and E. Roller <i>1983</i>. <i>cir,</i> a gene conferring resistance to colicin 1, maps between <i>mgl</i> and <i>fpk</i> on the <i>Escherichia coli</i> chromosome. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=cir, a gene conferring resistance to colicin 1, maps between mgl and fpk on the Escherichia coli chromosome">google scholar</a></td></tr>
<tr><td><a name="ref78"></a>78</td><td>Borg-Oliver, S. A., D. Tarlington, and K. D. Brown <i>1987</i>. Defective regulation of the phenylalanine biosynthetic operon in mutants of the phenylalanyl transfer RNA synthetase operon. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Defective regulation of the phenylalanine biosynthetic operon in mutants of the phenylalanyl transfer RNA synthetase operon">google scholar</a></td></tr>
<tr><td><a name="ref79"></a>79</td><td>Boronat, A., E. Caballero, and J. Aguilar <i>1983</i>. Experimental evolution of a metabolic pathway for ethylene glycol utilization by <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Experimental evolution of a metabolic pathway for ethylene glycol utilization by Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref80"></a>80</td><td>Bourd, G. I., R. S. Erlagaeva, T. N. Bolshakova, and V. N. Gershanovitch <i>1975</i>. Glucose catabolite repression in <i>Escherichia coli</i> K12 mutants defective in methyl-α-D-glucose transport. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Glucose catabolite repression in Escherichia coli K12 mutants defective in methyl-α-D-glucose transport">google scholar</a></td></tr>
<tr><td><a name="ref81"></a>81</td><td>Bourret, R. B., and M. S. Fox <i>1988</i>. Lysogenization of <i>Escherichia coli him</i>+, <i>himA,</i> and <i>himD</i> hosts by bacteriophage Mu. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Lysogenization of Escherichia coli him+, himA, and himD hosts by bacteriophage Mu">google scholar</a></td></tr>
<tr><td><a name="ref82"></a>82</td><td>Boy, E., and U. C. Patte <i>1972</i>. Multivalent repression of aspartic semialdehyde dehydrogenase in <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Multivalent repression of aspartic semialdehyde dehydrogenase in Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref83"></a>83</td><td>Brana, H., J. Hubacek, D. Michaljanicova, I. Holubova, and K. Cejka <i>1977</i>. Membrane mutation affecting energy-linked functions in <i>Escherichia coli</i> K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Membrane mutation affecting energy-linked functions in Escherichia coli K12">google scholar</a></td></tr>
<tr><td><a name="ref84"></a>84</td><td>Branes, L. V., J. M. Somers, and W. W. Kay <i>1981</i>. Hydrophobic peptide auxotrophy in <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Hydrophobic peptide auxotrophy in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref84a"></a>84a</td><td>Braun, V <i>1989</i>. The structurally related <i>exbB</i> and <i>tolQ</i> genes are interchangeable in conferring <i>tonB</i>-dependent colicin, bacteriophage, and albomycin sensitivity. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The structurally related exbB and tolQ genes are interchangeable in conferring tonB-dependent colicin, bacteriophage, and albomycin sensitivity">google scholar</a></td></tr>
<tr><td><a name="ref85"></a>85</td><td>Braun, V., J. Frenz, K. Hantke, and K. Schaller <i>1980</i>. Penetration of colicin M into cells of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Penetration of colicin M into cells of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref86"></a>86</td><td>Braun, V., K. Guenthner, K. Hantke, and L. Zimmermann <i>1983</i>. Intracellular activation of albomycin in <i>Escherichia coli</i> and <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Intracellular activation of albomycin in Escherichia coli and Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref87"></a>87</td><td>Braun, V., K. Hantke, and W. Stauder <i>1977</i>. Identification of the <i>sid</i> outer membrane receptor protein in <i>Salmonella typhimurium</i> SL1024. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Identification of the sid outer membrane receptor protein in Salmonella typhimurium SL1024">google scholar</a></td></tr>
<tr><td><a name="ref88"></a>88</td><td>Braun-Breton, C., and M. Hofnung <i>1981</i>. In vivo and in vitro functional alterations of the bacteriophage lambda receptor in <i>lamB</i> missense mutants of <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=In vivo and in vitro functional alterations of the bacteriophage lambda receptor in lamB missense mutants of Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref89"></a>89</td><td>Breckenridge, L., and L. Gorini <i>1970</i>. Genetic analysis of streptomycin resistance in <i>Escherichia</i> <i>coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic analysis of streptomycin resistance in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref90"></a>90</td><td>Breeze, A. S., and E. E. Obaseiki-Ebor <i>1983</i>. Mutations to nitrofurantoin and nitrofurazone resistance in <i>Escherichia coli</i> K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutations to nitrofurantoin and nitrofurazone resistance in Escherichia coli K12">google scholar</a></td></tr>
<tr><td><a name="ref91"></a>91</td><td>Brenchley, J. E., C. A. Baker, and L. G. Patil <i>1975</i>. Regulation of ammonia assimilatory enzymes in <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation of ammonia assimilatory enzymes in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref92"></a>92</td><td>Brickman, E., L. Soll, and J. Beckwith <i>1973</i>. Genetic characterization of mutations which affect catabolite-sensitive operons in <i>Escherichia coli,</i> including deletions of the gene for adenyl cyclase. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic characterization of mutations which affect catabolite-sensitive operons in Escherichia coli, including deletions of the gene for adenyl cyclase">google scholar</a></td></tr>
<tr><td><a name="ref93"></a>93</td><td>Brochu, A., N. Brochu, T. I. Nicas, T. R. Parr, Jr., A. A. Minnick, Jr., E. K. Dolence, J. A. McKee, M. J. Miller, M. C. Lavoie, and F. Malouin <i>1992</i>. Modes of action and inhibitory activities of new siderophore-beta lactam conjugates that use specific iron uptake pathways for entry into bacteria. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Modes of action and inhibitory activities of new siderophore-beta lactam conjugates that use specific iron uptake pathways for entry into bacteria">google scholar</a></td></tr>
<tr><td><a name="ref94"></a>94</td><td>Brown, K. D <i>1970</i>. Formation of aromatic amino acid pools in <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Formation of aromatic amino acid pools in Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref95"></a>95</td><td>Brown, K. D., and R. L. Somerville <i>1971</i>. Repression of aromatic amino acid biosynthesis in <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Repression of aromatic amino acid biosynthesis in Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref96"></a>96</td><td>Brown, T. D. K., M. C. Jones-Mortimer, and H. L. Kornberg <i>1977</i>. The enzymic interconversion of acetate and acetyl-coenzyme A in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The enzymic interconversion of acetate and acetyl-coenzyme A in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref97"></a>97</td><td>Bruce, I., J. Hardy, and K. A. Stacey <i>1984</i>. Potentiation by purines of the growth-inhibitory effects of sulphonamides on <i>Escherichia coli</i> and the location of the gene which mediates this effect. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Potentiation by purines of the growth-inhibitory effects of sulphonamides on Escherichia coli and the location of the gene which mediates this effect">google scholar</a></td></tr>
<tr><td><a name="ref98"></a>98</td><td>Bruni, C. B., V. Colantuoni, L. Sbordone, R. Cortese, and F. Blasi <i>1977</i>. Biochemical and regulatory properties of <i>Escherichia coli</i> K-12 <i>hisT</i> mutants. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Biochemical and regulatory properties of Escherichia coli K-12 hisT mutants">google scholar</a></td></tr>
<tr><td><a name="ref99"></a>99</td><td>Brunner, D. P., G. S. Graham, and R. W. Treick <i>1982</i>. Exchange of colicin receptor capacity between strains of <i>Escherichia coli</i> sensitive or resistant to colicin K-K235. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Exchange of colicin receptor capacity between strains of Escherichia coli sensitive or resistant to colicin K-K235">google scholar</a></td></tr>
<tr><td><a name="ref100"></a>100</td><td>Buchstein, S. R., and D. C. Hinkle <i>1982</i>. Genetic analysis of two bacterial RNA polymerase mutants that inhibit the growth of bacteriophage T7. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic analysis of two bacterial RNA polymerase mutants that inhibit the growth of bacteriophage T7">google scholar</a></td></tr>
<tr><td><a name="ref101"></a>101</td><td>Buckel, P., A. Buchberger, A. Boeck, and H. G. Wittmann <i>1977</i>. Alteration of ribosomal protein L6 in mutants of <i>Escherichia coli</i> resistant to gentamicin. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Alteration of ribosomal protein L6 in mutants of Escherichia coli resistant to gentamicin">google scholar</a></td></tr>
<tr><td><a name="ref102"></a>102</td><td>Buhr, A., G. A. Daniels, and B. Erin <i>1992</i>. The glucose transporter of <i>Escherichia coli</i> mutants with impaired translocation activity that retain phosphorylation activity. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The glucose transporter of Escherichia coli mutants with impaired translocation activity that retain phosphorylation activity">google scholar</a></td></tr>
<tr><td><a name="ref103"></a>103</td><td>Bullas, L. R., and C. Colson <i>1975</i>. DNA restriction and modification systems in <i>Salmonella.</i> III. SP, a <i>Salmonella potsdam</i> system allelic to the SB system in <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=DNA restriction and modification systems in Salmonella. III. SP, a Salmonella potsdam system allelic to the SB system in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref104"></a>104</td><td>Burd, G. I., G. I. Erlagaeva, T. N. Bol’shakova, and V. N. Gershanovich <i>1974</i>. Separation of the α-methylglucoside transport and phosphorylation system in a mutant of <i>Escherichia coli</i> K12 resistant to catabolic repression by glucose. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Separation of the α-methylglucoside transport and phosphorylation system in a mutant of Escherichia coli K12 resistant to catabolic repression by glucose">google scholar</a></td></tr>
<tr><td><a name="ref105"></a>105</td><td>Burman, L. G., and K. Nordstrom <i>1971</i>. Colicin tolerance induced by ampicillin resistance in a strain of <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Colicin tolerance induced by ampicillin resistance in a strain of Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref106"></a>106</td><td>Burns, R. O., J. G. Hofler, and G. H. Luginbuhl <i>1979</i>. Threonine deaminase from <i>Salmonella</i> <i>typhimurium.</i> Substrate-specific patterns of inhibition in an activation site-deficient form of the enzyme. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Threonine deaminase from Salmonella typhimurium. Substrate-specific patterns of inhibition in an activation site-deficient form of the enzyme">google scholar</a></td></tr>
<tr><td><a name="ref107"></a>107</td><td>Burton, K <i>1977</i>. Transport of adenine, hypoxanthine and uracil into <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Transport of adenine, hypoxanthine and uracil into Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref108"></a>108</td><td>Buttin, G <i>1963</i>. Mécanismes regulateurs dans la biosynthèse des enzymes du metabolisme du galactose chez <i>Escherichia coli</i> K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mécanismes regulateurs dans la biosynthèse des enzymes du metabolisme du galactose chez Escherichia coli K12">google scholar</a></td></tr>
<tr><td><a name="ref109"></a>109</td><td>Buxton, R. S <i>1971</i>. Genetic analysis of <i>Escherichia coli</i> K-12 mutants resistant to bacteriophage BF23 and the E-group colicins. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic analysis of Escherichia coli K-12 mutants resistant to bacteriophage BF23 and the E-group colicins">google scholar</a></td></tr>
<tr><td><a name="ref110"></a>110</td><td>Buxton, R. S., K. Hammer-Jespersen, and T. D. Hansen <i>1978</i>. Insertion of bacteriophage lambda into the <i>deo</i> operon of <i>Escherichia coli</i> K-12 and isolation of plaque-forming λ <i>deo</i>+ transducing bacteriophages. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Insertion of bacteriophage lambda into the deo operon of Escherichia coli K-12 and isolation of plaque-forming λ deo+ transducing bacteriophages">google scholar</a></td></tr>
<tr><td><a name="ref111"></a>111</td><td>Buxton, R. S., K. Hammer-Jespersen, and P. Valentin-Hansen <i>1980</i>. A second purine nucleoside phosphorylase in <i>Escherichia coli</i> K-12. I. Xanthosine phosphorylase regulatory mutants isolated as secondary-site revertants of a <i>deoD</i> mutant. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A second purine nucleoside phosphorylase in Escherichia coli K-12. I. Xanthosine phosphorylase regulatory mutants isolated as secondary-site revertants of a deoD mutant">google scholar</a></td></tr>
<tr><td><a name="ref112"></a>112</td><td>Calcott, P. H., and K. N. Calcott <i>1984</i>. Involvement of outer membrane proteins in freeze thaw resistance of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Involvement of outer membrane proteins in freeze thaw resistance of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref113"></a>113</td><td>Calendar, R., B. Lindquist, G. Sironi, and A. J. Clark <i>1970</i>. Characterization of REP– mutants and their interaction with P2 phage. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Characterization of REP– mutants and their interaction with P2 phage">google scholar</a></td></tr>
<tr><td><a name="ref114"></a>114</td><td>Caligiuri, M. G., and R. Bauerle <i>1991</i>. Identification of amino acid residues involved in feedback regulation of the anthranilate synthase complex from <i>Salmonella typhimurium:</i> evidence for an amino-terminal regulatory site. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Identification of amino acid residues involved in feedback regulation of the anthranilate synthase complex from Salmonella typhimurium: evidence for an amino-terminal regulatory site">google scholar</a></td></tr>
<tr><td><a name="ref115"></a>115</td><td>Calvo, J. M., M. Freundlich, and H. E. Umbarger <i>1969</i>. Regulation of branched-chain amino acid biosynthesis in <i>Salmonella typhimurium:</i> isolation of regulatory mutants. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation of branched-chain amino acid biosynthesis in Salmonella typhimurium: isolation of regulatory mutants">google scholar</a></td></tr>
<tr><td><a name="ref116"></a>116</td><td>Calvo, J. M., P. Margolin, and H. E. Umbarger <i>1969</i>. Operator constitutive mutations in the leucine operon of <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Operator constitutive mutations in the leucine operon of Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref117"></a>117</td><td>Calvo, R. A., and J. M. Calvo <i>1967</i>. Lack of end-product inhibition and repression of leucine synthesis in a strain of <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Lack of end-product inhibition and repression of leucine synthesis in a strain of Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref118"></a>118</td><td>Campbell, J. H., J. Lengyel, and J. Langridge <i>1973</i>. Evolution of a second gene for β- galactosidase in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Evolution of a second gene for β- galactosidase in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref119"></a>119</td><td>Cannon, M., T. Cabezon, and A. Bollen <i>1979</i>. Mapping of neamine resistance: identification of two genetic loci, <i>neaA</i> and <i>neaB.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mapping of neamine resistance: identification of two genetic loci, neaA and neaB.">google scholar</a></td></tr>
<tr><td><a name="ref120"></a>120</td><td>Cardelli, J., and J. Konisky <i>1974</i>. Isolation and characterization of an <i>Escherichia coli</i> mutant tolerant to colicins Ia and Ib. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation and characterization of an Escherichia coli mutant tolerant to colicins Ia and Ib">google scholar</a></td></tr>
<tr><td><a name="ref121"></a>121</td><td>Casse, F <i>1970</i>. Mapping of the gene <i>chlB</i> controlling membrane bound nitrate reductase and formic hydrogen-lyase activities in <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mapping of the gene chlB controlling membrane bound nitrate reductase and formic hydrogen-lyase activities in Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref122"></a>122</td><td>Casse, F., M. C. Pascal, M. Chippaux, and J. Ratouchniak <i>1972</i>. Mapping of the <i>chlB</i> gene in <i>Salmonella typhimurium</i> LT2. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mapping of the chlB gene in Salmonella typhimurium LT2">google scholar</a></td></tr>
<tr><td><a name="ref123"></a>123</td><td>Caster, J. H <i>1967</i>. Selection of thymine-requiring strains from <i>Escherichia coli</i> on solid medium. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Selection of thymine-requiring strains from Escherichia coli on solid medium">google scholar</a></td></tr>
<tr><td><a name="ref124"></a>124</td><td>Chai, T., and J. Foulds <i>1974</i>. Missing outer membrane protein in <i>tolG</i> mutants of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Missing outer membrane protein in tolG mutants of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref125"></a>125</td><td>Chai, T.-J., V. Wu, and J. Foulds <i>1982</i>. Colicin A receptor role of two <i>Escherichia coli</i> outer membrane proteins (OmpF protein and <i>btuB</i> gene product) and lipopolysaccharide. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Colicin A receptor role of two Escherichia coli outer membrane proteins (OmpF protein and btuB gene product) and lipopolysaccharide">google scholar</a></td></tr>
<tr><td><a name="ref126"></a>126</td><td>Chang, G. W., D. Straus, and B. N. Ames <i>1971</i>. Enriched selection of dominant mutations: histidine operator mutations. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Enriched selection of dominant mutations: histidine operator mutations">google scholar</a></td></tr>
<tr><td><a name="ref127"></a>127</td><td>Chater, K. F., and R. J. Rowbury <i>1970</i>. A genetical study of the feedback-sensitive enzyme of methionine synthesis in <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A genetical study of the feedback-sensitive enzyme of methionine synthesis in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref128"></a>128</td><td>Chattopadhyay, M. K., A. K. Ghosh, and S. Sengupta <i>1991</i>. Control of methionine biosynthesis in <i>Escherichia coli</i> K12: a closer study with analog resistant mutants. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Control of methionine biosynthesis in Escherichia coli K12: a closer study with analog resistant mutants">google scholar</a></td></tr>
<tr><td><a name="ref129"></a>129</td><td>Chen, M. X., N. Bouquin, V. Norris, S. Casaregola, S. J. Seror, J. Simone, and I. B. Holland <i>1991</i>. A single base change in the acceptor stem of tRNA Leu 3 confers resistance upon <i>Escherichia</i> <i>coli</i> to the calmodulin inhibitor 48/80. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A single base change in the acceptor stem of tRNA Leu 3 confers resistance upon Escherichia coli to the calmodulin inhibitor 48/80">google scholar</a></td></tr>
<tr><td><a name="ref130"></a>130</td><td>Chippaux, M., D. Giudici, A. Abou-Jaoude, and M. C. Pascal <i>1978</i>. A mutation leading to the total lack of nitrate reductase activity in <i>Escherichia</i> coli K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A mutation leading to the total lack of nitrate reductase activity in Escherichia coli K12">google scholar</a></td></tr>
<tr><td><a name="ref131"></a>131</td><td>Christman, M. F., R. W. Morgan, F. S. Jacobson, and B. N. Ames <i>1985</i>. Positive control of a regulon for defenses against oxidative stress and some heat-shock proteins in <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Positive control of a regulon for defenses against oxidative stress and some heat-shock proteins in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref132"></a>132</td><td>Clark, D., and J. E. Cronan, Jr <i>1980</i>. <i>Escherichia coli</i> mutants with altered control of alcohol dehydrogenase and nitrate reductase. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Escherichia coli mutants with altered control of alcohol dehydrogenase and nitrate reductase">google scholar</a></td></tr>
<tr><td><a name="ref133"></a>133</td><td>Clark, D. P., and J. P. Beard <i>1979</i>. Altered phospholipid composition in mutants of <i>Escherichia</i> <i>coli</i> sensitive or resistant to organic solvents. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Altered phospholipid composition in mutants of Escherichia coli sensitive or resistant to organic solvents">google scholar</a></td></tr>
<tr><td><a name="ref134"></a>134</td><td>Clement, J. M., C. Braun-Breton, E. Lepouce, C. Marchal, D. Perrin, H. Villarroya, and M. Hofnung <i>1982</i>. A system for genetic analysis in gene <i>lamB:</i> first results with lambda resistant tight mutants. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A system for genetic analysis in gene lamB: first results with lambda resistant tight mutants">google scholar</a></td></tr>
<tr><td><a name="ref135"></a>135</td><td>Clement, J.-M., E. Lepouce, C. Marchal, and M. Hofnung <i>1983</i>. Genetic study of a membrane protein: DNA sequence alterations due to 17 <i>lamB</i> point mutations affecting adsorption of phage lambda. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic study of a membrane protein: DNA sequence alterations due to 17 lamB point mutations affecting adsorption of phage lambda">google scholar</a></td></tr>
<tr><td><a name="ref136"></a>136</td><td>Clifton, G., S. R. Bryant, and C. G. Skinner <i>1970</i>. N1-(substituted) pantothenamides, antimetabolites of pantothenic acid. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=N1-(substituted) pantothenamides, antimetabolites of pantothenic acid">google scholar</a></td></tr>
<tr><td><a name="ref137"></a>137</td><td>Cohen, G., and F. Jacob <i>1959</i>. Sur la répression de la synthese des enzymes intervenant dans la formation du tryptophane chez <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Sur la répression de la synthese des enzymes intervenant dans la formation du tryptophane chez Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref138"></a>138</td><td>Cohen, G. N., J.-C. Patte, P. Truffa-Bachi, C. Sawas, and M. Doudoroff <i>1965</i>. Repression and end-product inhibition in a branched biosynthetic pathway.</td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Repression and end-product inhibition in a branched biosynthetic pathway">google scholar</a></td></tr>
<tr><td><a name="ref139"></a>139</td><td>Cole, S. T., and J. R Guest <i>1979</i>. Production of a soluble form of fumarate reductase by multiple gene duplications in <i>Escherichia coli</i> K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Production of a soluble form of fumarate reductase by multiple gene duplications in Escherichia coli K12">google scholar</a></td></tr>
<tr><td><a name="ref140"></a>140</td><td>Coleman, J., S. Inouye, and M. Inouye <i>1983</i>. Isolation of mutants of the major outer membrane lipoprotein of <i>Escherichia coli</i> for the study of its assembly. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation of mutants of the major outer membrane lipoprotein of Escherichia coli for the study of its assembly">google scholar</a></td></tr>
<tr><td><a name="ref141"></a>141</td><td>Coleman, W. G., and L. Leive <i>1979</i>. Two mutations which affect the barrier function of the <i>Escherichia coli</i> K-12 outer membrane. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Two mutations which affect the barrier function of the Escherichia coli K-12 outer membrane">google scholar</a></td></tr>
<tr><td><a name="ref142"></a>142</td><td>Cookson, B. T., B. M. Olivera, and J. R. Roth <i>1987</i>. Genetic characterization and regulation of the <i>nadB</i> locus of <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic characterization and regulation of the nadB locus of Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref143"></a>143</td><td>Cooley, W., K. Sirotkin, R. Green, and L. Snyder <i>1979</i>. A new gene of <i>Escherichia coli</i> K-12 whose product participates in T4 bacteriophage late gene expression: interaction of <i>lit</i> with the T4- induced polynucleotide 5′-kinase 3′-phosphatase. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A new gene of Escherichia coli K-12 whose product participates in T4 bacteriophage late gene expression: interaction of lit with the T4- induced polynucleotide 5′-kinase 3′-phosphatase">google scholar</a></td></tr>
<tr><td><a name="ref144"></a>144</td><td>Cooper, R. A <i>1978</i>. The utilization of D-galactonate and D-2-oxo-3-deoxygalactonate by <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The utilization of D-galactonate and D-2-oxo-3-deoxygalactonate by Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref145"></a>145</td><td>Corchuelo, M. C., A. Herzog, L. Desmarez, R. Lavalle, and A. Bollen <i>1981</i>. Resistance to the peptide-like antibiotic negamycin in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Resistance to the peptide-like antibiotic negamycin in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref146"></a>146</td><td>Cordaro, J. C., T. Melton, J. P. Stratis, M. Atagun, C. Gladding, P. E. Hartman, and S. Roseman <i>1976</i>. Fosfomycin resistance: selection method for internal and extended deletions of the phosphoenolpyruvate:sugar phosphotransferase genes of <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Fosfomycin resistance: selection method for internal and extended deletions of the phosphoenolpyruvate:sugar phosphotransferase genes of Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref147"></a>147</td><td>Cordaro, J. C., and S. Roseman <i>1972</i>. Deletion mapping of the genes coding for HPr and enzyme I of the phosphoenolpyruvate:sugar phosphotransferase system in <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Deletion mapping of the genes coding for HPr and enzyme I of the phosphoenolpyruvate:sugar phosphotransferase system in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref148"></a>148</td><td>Cortese, R., R. Landsberg, R. A. VanderHaar, H. E. Umbarger, and B. N. Ames <i>1974</i>. Pleiotrophy of <i>hisT</i> mutants blocked in pseudouridine synthesis in tRNA: <i>leucine</i> and <i>isoleucine- valine</i> operons. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Pleiotrophy of hisT mutants blocked in pseudouridine synthesis in tRNA: leucine and isoleucine- valine operons">google scholar</a></td></tr>
<tr><td><a name="ref149"></a>149</td><td>Corwin, L. M., G. R. Fanning, F. Feldman, and P. Margolin <i>1966</i>. Mutation leading to increased sensitivity to chromium ion in <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutation leading to increased sensitivity to chromium ion in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref150"></a>150</td><td>Cosloy, S. D <i>1973</i>. D-Serine transport system in <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=D-Serine transport system in Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref151"></a>151</td><td>Coulton, J. W., P. Mason, D. R. Cameron, G. Carmel, R. Jean, and H. N. Rode <i>1986</i>. Protein fusions of β-galactosidase to the ferrichrome-iron receptor of <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Protein fusions of β-galactosidase to the ferrichrome-iron receptor of Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref152"></a>152</td><td>Cox, E. C., and T. C. Gibson <i>1974</i>. Selection for high mutation rates in chemostats. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Selection for high mutation rates in chemostats">google scholar</a></td></tr>
<tr><td><a name="ref153"></a>153</td><td>Crabeel, M., D. Charlier, R. Cunin, A. Bogen, N. Glansdorf, and A. Pierard <i>1975</i>. Accumulation of arginine precursors in <i>Escherichia</i> coli: effects on growth, enzyme repression, and application to the forward selection of arginine auxotrophs. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Accumulation of arginine precursors in Escherichia coli: effects on growth, enzyme repression, and application to the forward selection of arginine auxotrophs">google scholar</a></td></tr>
<tr><td><a name="ref154"></a>154</td><td>Creaghan, I. T., and J. R. Guest <i>1977</i>. Suppression of the succinate requirement of lipoamide dehydrogenase mutants of <i>Escherichia coli</i> by mutations affecting succinate dehydrogenase activity. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Suppression of the succinate requirement of lipoamide dehydrogenase mutants of Escherichia coli by mutations affecting succinate dehydrogenase activity">google scholar</a></td></tr>
<tr><td><a name="ref155"></a>155</td><td>Csonka, L. N <i>1981</i>. Proline over-production results in enhanced osmotolerance in <i>Salmonella</i> <i>typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Proline over-production results in enhanced osmotolerance in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref156"></a>156</td><td>Csonka, L. N <i>1982</i>. A third L-proline permease in <i>Salmonella typhimurium</i> which functions in media of elevated osmotic strength. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A third L-proline permease in Salmonella typhimurium which functions in media of elevated osmotic strength">google scholar</a></td></tr>
<tr><td><a name="ref157"></a>157</td><td>Cunningham, P. R., and D. P. Clark <i>1986</i>. The use of suicide substrates to select mutants of <i>Escherichia coli</i> lacking enzymes of alcohol fermentation. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The use of suicide substrates to select mutants of Escherichia coli lacking enzymes of alcohol fermentation">google scholar</a></td></tr>
<tr><td><a name="ref158"></a>158</td><td>Curtis, S. J., and W. Epstein <i>1975</i>. Phosphorylation of D-glucose in <i>Escherichia coli</i> mutants defective in glucosephosphotransferase, mannosephosphotransferase, and glucokinase. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Phosphorylation of D-glucose in Escherichia coli mutants defective in glucosephosphotransferase, mannosephosphotransferase, and glucokinase">google scholar</a></td></tr>
<tr><td><a name="ref159"></a>159</td><td>Dabbs, E. R <i>1978</i>. Kasugamycin-dependent mutants of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Kasugamycin-dependent mutants of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref160"></a>160</td><td>Dabbs, E. R <i>1982</i>. Three additional loci of rifampicin dependence in <i>Escherichia</i> <i>coli.</i> <i>Mol. Gen.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Three additional loci of rifampicin dependence in Escherichia coli. Mol. Gen.">google scholar</a></td></tr>
<tr><td><a name="ref161"></a>161</td><td>Dabbs, E. R <i>1983</i>. <i>Escherichia coli</i> kasugamycin dependence arising from mutation at the <i>rpsI</i> locus. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Escherichia coli kasugamycin dependence arising from mutation at the rpsI locus">google scholar</a></td></tr>
<tr><td><a name="ref162"></a>162</td><td>Dabbs, E. R., and K. Looman <i>1981</i>. An antibiotic dependent conditional lethal mutant with a lesion affecting transcription and translation. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=An antibiotic dependent conditional lethal mutant with a lesion affecting transcription and translation">google scholar</a></td></tr>
<tr><td><a name="ref163"></a>163</td><td>Dabbs, E. R., B. Poldermans, H. Bakker, and P. H. Van Knippenberg <i>1980</i>. Biochemical characterization of ribosomes of kasugamycin-dependent mutants of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Biochemical characterization of ribosomes of kasugamycin-dependent mutants of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref164"></a>164</td><td>Dame, J. B., and B. M. Shapiro <i>1976</i>. Use of polymyxin B, levallorphan, and tetracaine to isolate novel mutants of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Use of polymyxin B, levallorphan, and tetracaine to isolate novel mutants of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref165"></a>165</td><td>Danchin, A <i>1977</i>. A new technique for selection of sensitive and auxotrophic mutants of <i>E. coli:</i> isolation of a strain sensitive to an excess of one-carbon metabolites. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A new technique for selection of sensitive and auxotrophic mutants of E. coli: isolation of a strain sensitive to an excess of one-carbon metabolites">google scholar</a></td></tr>
<tr><td><a name="ref166"></a>166</td><td>Danchin, A., and L. Dondon <i>1980</i>. Serine sensitivity of <i>Escherichia coli</i> K-12: partial characterization of a serine resistant mutant that is extremely sensitive to 2-ketobutyrate. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Serine sensitivity of Escherichia coli K-12: partial characterization of a serine resistant mutant that is extremely sensitive to 2-ketobutyrate">google scholar</a></td></tr>
<tr><td><a name="ref167"></a>167</td><td>Daniel, J., and A. Danchin <i>1979</i>. Involvement of cyclic AMP and its receptor protein in the sensitivity of <i>Escherichia coli</i> K12 toward serine. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Involvement of cyclic AMP and its receptor protein in the sensitivity of Escherichia coli K12 toward serine">google scholar</a></td></tr>
<tr><td><a name="ref168"></a>168</td><td>D’Ari, R., A. Jaffé, P. Bouloc, and A. Robin <i>1988</i>. Cyclic AMP and cell division in <i>Escherichia</i> <i>coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Cyclic AMP and cell division in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref169"></a>169</td><td>Das, A., D. Court, and S. Adhya <i>1976</i>. Isolation and characteristics of conditional-lethal mutants of <i>Escherichia coli</i> defective in transcription termination factor rho. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation and characteristics of conditional-lethal mutants of Escherichia coli defective in transcription termination factor rho">google scholar</a></td></tr>
<tr><td><a name="ref170"></a>170</td><td>Das, A., B. Ghosh, S. Barik, and K. Wolska <i>1985</i>. Evidence that ribosomal protein S-10 itself is a cellular component necessary for transcription antitermination by phage lambda N protein. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Evidence that ribosomal protein S-10 itself is a cellular component necessary for transcription antitermination by phage lambda N protein">google scholar</a></td></tr>
<tr><td><a name="ref171"></a>171</td><td>Datta, D. B., B. Arden, and U. Henning <i>1977</i>. Major proteins of the <i>Escherichia coli</i> outer cell envelope membrane as bacteriophage receptors. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Major proteins of the Escherichia coli outer cell envelope membrane as bacteriophage receptors">google scholar</a></td></tr>
<tr><td><a name="ref172"></a>172</td><td>Davies, J., P. Anderson, and B. D. Davis <i>1965</i>. Inhibition of protein synthesis by spectinomycin. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Inhibition of protein synthesis by spectinomycin">google scholar</a></td></tr>
<tr><td><a name="ref173"></a>173</td><td>Davies, J. K., and P. Reeves <i>1975</i>. Genetics of resistance to colicins in <i>Escherichia coli</i> K-12: cross-resistance among colicins of group A. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetics of resistance to colicins in Escherichia coli K-12: cross-resistance among colicins of group A">google scholar</a></td></tr>
<tr><td><a name="ref174"></a>174</td><td>Davis, E. J., J. M. Blatt, E. K. Henderson, J. J. Whittaker, and J. H. Jackson <i>1977</i>. Valine- sensitive acetohydroxy acid synthetases in <i>Escherichia coli</i> K-12: unique regulation modulated by multiple genetic sites. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Valine- sensitive acetohydroxy acid synthetases in Escherichia coli K-12: unique regulation modulated by multiple genetic sites">google scholar</a></td></tr>
<tr><td><a name="ref175"></a>175</td><td>Davis, E. O., M. J. Jones-Mortimer, and P. J. F. Henderson <i>1984</i>. Location of a structural gene for xylose proton symport at 91 minutes on the linkage map of <i>Escherichia coli</i> K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Location of a structural gene for xylose proton symport at 91 minutes on the linkage map of Escherichia coli K12">google scholar</a></td></tr>
<tr><td><a name="ref176"></a>176</td><td>Daws, T. D., and J. A. Fuchs <i>1984</i>. Isolation and characterization of an <i>Escherichia</i> <i>coli</i> mutant deficient in dTMP kinase activity. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation and characterization of an Escherichia coli mutant deficient in dTMP kinase activity">google scholar</a></td></tr>
<tr><td><a name="ref177"></a>177</td><td>Dean, D. A., J. Reizer, H. Nikaido, and M. H. Saier, Jr <i>1990</i>. Regulation of the maltose transport system of <i>Escherichia</i> <i>coli</i> by the glucose-specific enzyme III of the phosphoenolpyruvate- sugar phosphotransferase system. Characterization of inducer exclusion resistant mutants and reconstitution of inducer exlusion in proteoliposomes. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation of the maltose transport system of Escherichia coli by the glucose-specific enzyme III of the phosphoenolpyruvate- sugar phosphotransferase system. Characterization of inducer exclusion resistant mutants and reconstitution of inducer exlusion in proteoliposomes">google scholar</a></td></tr>
<tr><td><a name="ref178"></a>178</td><td>DeFelice, M., J. Guardiola, A. Lamberti, and M. Iaccarino <i>1973</i>. <i>Escherichia coli</i> K-12 mutants altered in the transport systems for oligo- and dipeptides. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Escherichia coli K-12 mutants altered in the transport systems for oligo- and dipeptides">google scholar</a></td></tr>
<tr><td><a name="ref179"></a>179</td><td>Defez, R., and M. DeFelice <i>1981</i>. Cryptic operon for β-glucoside metabolism in <i>Escherichia coli</i> K12: genetic evidence for a regulatory protein. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Cryptic operon for β-glucoside metabolism in Escherichia coli K12: genetic evidence for a regulatory protein">google scholar</a></td></tr>
<tr><td><a name="ref180"></a>180</td><td>Del Campillo-Campbell, A., and A. Campbell <i>1982</i>. Molybdenum cofactor requirement for biotin sulfoxide reduction in <i>Escherichia coli</i>. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Molybdenum cofactor requirement for biotin sulfoxide reduction in Escherichia coli">google scholar</a></td></tr>
<tr><td><a name="ref181"></a>181</td><td>Del Castillo, I., J. L. Vizan, M. Del Carmen Rodriguez-Sainz, and F. Moreno <i>1991</i>. An unusual mechanism for resistance to the antibiotic coumermycin A-1. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=An unusual mechanism for resistance to the antibiotic coumermycin A-1">google scholar</a></td></tr>
<tr><td><a name="ref182"></a>182</td><td>Delcuve, G., T. Cabezon, A. Herzog, M. Cannon, and A. Bollen <i>1978</i>. Resistance to the aminoglycoside antibiotic neamine in <i>Escherichia</i> <i>coli.</i> A new mutant whose NeaR phenotype results from the cumulative effects of two distinct mutations. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Resistance to the aminoglycoside antibiotic neamine in Escherichia coli. A new mutant whose NeaR phenotype results from the cumulative effects of two distinct mutations">google scholar</a></td></tr>
<tr><td><a name="ref183"></a>183</td><td>DelGiudice, L <i>1979</i>. Method for isolating restriction- and modification-less mutants of <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Method for isolating restriction- and modification-less mutants of Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref184"></a>184</td><td>Delidakis, C. E., M. C. Jones-Mortimer, and H. L. Kornberg <i>1982</i>. A mutant inducible for galactitol utilization in <i>Escherichia coli</i> K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A mutant inducible for galactitol utilization in Escherichia coli K12">google scholar</a></td></tr>
<tr><td><a name="ref185"></a>185</td><td>Delvaux, A. M., and R. Devoret <i>1969</i>. Occurrence of suppressors in caffeine-resistant mutants from <i>Escherichia coli</i> K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Occurrence of suppressors in caffeine-resistant mutants from Escherichia coli K12">google scholar</a></td></tr>
<tr><td><a name="ref186"></a>186</td><td>Dempsey, W. B., and L. J. Arcement <i>1971</i>. Identification of the forms of vitamin B6 present in the culture media of “vitamin B6 control” mutants. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Identification of the forms of vitamin B6 present in the culture media of “vitamin B6 control” mutants">google scholar</a></td></tr>
<tr><td><a name="ref187"></a>187</td><td>Dendinger, S., and W. J. Brill <i>1972</i>. Effect of the proline analogue baikiain on proline metabolism in <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Effect of the proline analogue baikiain on proline metabolism in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref188"></a>188</td><td>Dendinger, S. M., L. G. Patil, and J. E. Brenchley <i>1980</i>. <i>Salmonella typhimurium</i> mutants with altered glutamate dehydrogenase and glutamate synthase activities. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Salmonella typhimurium mutants with altered glutamate dehydrogenase and glutamate synthase activities">google scholar</a></td></tr>
<tr><td><a name="ref189"></a>189</td><td>Desrochers, M., L. Peloquin, and A. Sasarman <i>1978</i>. Mapping of the <i>hemE</i> locus in <i>Salmonella</i> <i>typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mapping of the hemE locus in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref190"></a>190</td><td>Devoret, R., and M. Blanco <i>1970</i>. Mutants of <i>Escherichia coli</i> K12 λ+ noninducible by thymine deprivation. I. Method of isolation and classes of mutants obtained. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutants of Escherichia coli K12 λ+ noninducible by thymine deprivation. I. Method of isolation and classes of mutants obtained">google scholar</a></td></tr>
<tr><td><a name="ref191"></a>191</td><td>Dietz, G. W., Jr <i>1978</i>. Growth of <i>Escherichia coli</i> on glucosamine 6-phosphate selection of a constitutive hexose phosphate transport system mutant. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Growth of Escherichia coli on glucosamine 6-phosphate selection of a constitutive hexose phosphate transport system mutant">google scholar</a></td></tr>
<tr><td><a name="ref192"></a>192</td><td>Dietz, G. W., and L. A. Heppel <i>1971</i>. Studies on the uptake of hexose phosphates. III. Mechanism of uptake of glucose 1-phosphate in </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Studies on the uptake of hexose phosphates. III. Mechanism of uptake of glucose 1-phosphate in">google scholar</a></td></tr>
<tr><td><a name="ref193"></a>193</td><td>Dila, D. K., and S. R. Maloy <i>1986</i>. Proline transport in <i>Salmonella typhimurium</i> <i>putP</i> permease mutants with altered substrate specificity. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Proline transport in Salmonella typhimurium putP permease mutants with altered substrate specificity">google scholar</a></td></tr>
<tr><td><a name="ref194"></a>194</td><td>DiNardo, S., K. A. Voelkel, R. Sternglanz, A. E. Reynolds, and A. Wright <i>1982</i>. <i>Escherichia</i> <i>coli</i> DNA topoisomerase I mutants have compensatory mutations in DNA gyrase genes. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Escherichia coli DNA topoisomerase I mutants have compensatory mutations in DNA gyrase genes">google scholar</a></td></tr>
<tr><td><a name="ref195"></a>195</td><td>Dombou, M., T. Mizuno, and S. Mizushima <i>1977</i>. Interaction of the cytoplasmic membrane and ribosomes in <i>Escherichia coli</i> altered ribosomal proteins in sucrose-dependent spectinomycin- resistant mutants. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Interaction of the cytoplasmic membrane and ribosomes in Escherichia coli altered ribosomal proteins in sucrose-dependent spectinomycin- resistant mutants">google scholar</a></td></tr>
<tr><td><a name="ref196"></a>196</td><td>Doskocil, J., and F. Sorm <i>1970</i>. The mode of action of 5-aza-2′-deoxycytidine in <i>Escherichia</i> <i>coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The mode of action of 5-aza-2′-deoxycytidine in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref197"></a>197</td><td>Dover, S., and Y. S. Halpern <i>1974</i>. Genetic analysis of the γ-aminobutyrate utilization pathway in <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic analysis of the γ-aminobutyrate utilization pathway in Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref198"></a>198</td><td>Downs, D., L. Waxman, A. L. Goldberg, and J. Roth <i>1986</i>. Isolation and characterization of <i>lon</i> mutants in <i>Salmonella</i> <i>typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation and characterization of lon mutants in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref199"></a>199</td><td>Drexler, K., I. Riede, D. Montag, M.-L. Eschbach, and U. Henning <i>1989</i>. Receptor specificity of the <i>Escherichia coli</i> T-even type phage Ox2: mutational alterations in host range mutants. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Receptor specificity of the Escherichia coli T-even type phage Ox2: mutational alterations in host range mutants">google scholar</a></td></tr>
<tr><td><a name="ref200"></a>200</td><td>Droffner, M. L., and N. Yamamoto <i>1992</i>. Role of nalidixic acid in isolation of <i>Salmonella</i> <i>typhimurium</i> strains capable of growth at 48°C. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Role of nalidixic acid in isolation of Salmonella typhimurium strains capable of growth at 48°C">google scholar</a></td></tr>
<tr><td><a name="ref201"></a>201</td><td>Drury, L. S., and R. S. Buxton <i>1988</i>. Identification and sequencing of the <i>Escherichia</i> <i>coli cet</i> gene which codes for an inner membrane protein, mutation of which causes tolerance to colicin E2. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Identification and sequencing of the Escherichia coli cet gene which codes for an inner membrane protein, mutation of which causes tolerance to colicin E2">google scholar</a></td></tr>
<tr><td><a name="ref202"></a>202</td><td>Dubnau, E., and P. Margolin <i>1972</i>. Suppression of promoter mutations by the pleiotrophic <i>supX</i> mutations. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Suppression of promoter mutations by the pleiotrophic supX mutations">google scholar</a></td></tr>
<tr><td><a name="ref203"></a>203</td><td>Duncan, B. K., P. A. Rockstroh, and H. R. Warner <i>1978</i>. <i>Escherichia coli</i> K-12 mutants deficient in uracil-DNA glycosylase. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Escherichia coli K-12 mutants deficient in uracil-DNA glycosylase">google scholar</a></td></tr>
<tr><td><a name="ref204"></a>204</td><td>Echols, H., A. Garen, S. Garen, and A. Torriani <i>1961</i>. Genetic control of repression of alkaline phosphatase in <i>E. coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic control of repression of alkaline phosphatase in E. coli.">google scholar</a></td></tr>
<tr><td><a name="ref205"></a>205</td><td>Eick-Helmerich, K., and V. Braun <i>1989</i>. Import of biopolymers into <i>Escherichia coli</i> nucleotide sequences of the <i>exbB</i> and <i>exbD</i> genes are homologous to those of the <i>tolQ</i> and <i>tolR</i> genes, respectively. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Import of biopolymers into Escherichia coli nucleotide sequences of the exbB and exbD genes are homologous to those of the tolQ and tolR genes, respectively">google scholar</a></td></tr>
<tr><td><a name="ref206"></a>206</td><td>Eisenberg, M. A., B. Mee, O. Prakash, and M. R. Eisenberg <i>1975</i>. Properties of α- dehydrobiotin-resistant mutant of <i>Escherichia</i> <i>coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Properties of α- dehydrobiotin-resistant mutant of Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref207"></a>207</td><td>Elwell, L. P., R. Ferone, G. A. Freeman, J. A. Fyfe, J. A. Hill, P. H. Ray, C. A. Richards, S. C. Singer, V. B. Knick, J. L. Rideout, and T. P. Zimmerman <i>1987</i>. Antibacterial activity and mechanism of action of 3′ azido-3′-deoxythymidine BWA-509U. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Antibacterial activity and mechanism of action of 3′ azido-3′-deoxythymidine BWA-509U">google scholar</a></td></tr>
<tr><td><a name="ref208"></a>208</td><td>Ely, B., D. B. Fankhauser, and P. E. Hartman <i>1974</i>. A fine structure map of the <i>Salmonella</i> histidine operator-promoter. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A fine structure map of the Salmonella histidine operator-promoter">google scholar</a></td></tr>
<tr><td><a name="ref209"></a>209</td><td>Englesberg, E., J. Irr, J. Power, and N. Lee <i>1965</i>. Positive control of enzyme synthesis by gene C in the L-arabinose system. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Positive control of enzyme synthesis by gene C in the L-arabinose system">google scholar</a></td></tr>
<tr><td><a name="ref210"></a>210</td><td>Eriksson-Grennberg, K. G., H. G. Boman, J. A. T. Jansson, and S. Thoren <i>1965</i>. Resistance of <i>Escherichia coli</i> to penicillins. I. Genetic study of some ampicillin-resistant mutants. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Resistance of Escherichia coli to penicillins. I. Genetic study of some ampicillin-resistant mutants">google scholar</a></td></tr>
<tr><td><a name="ref211"></a>211</td><td>Eriksson-Grennberg, K. G., and K. Nordstrom <i>1973</i>. Genetics and physiology of a <i>tolE</i> mutant of <i>Escherichia coli</i> K-12 and phenotypic suppression of its phenotype by galactose. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetics and physiology of a tolE mutant of Escherichia coli K-12 and phenotypic suppression of its phenotype by galactose">google scholar</a></td></tr>
<tr><td><a name="ref212"></a>212</td><td>Eriksson-Grennberg, K. G., K. Nordstrom, and P. Englund <i>1971</i>. Resistance of <i>Escherichia</i> <i>coli</i> to penicillins. IX. Genetics and physiology of class II ampicillin-resistant mutants that are galactose negative or sensitive to bacteriophage C21, or both. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Resistance of Escherichia coli to penicillins. IX. Genetics and physiology of class II ampicillin-resistant mutants that are galactose negative or sensitive to bacteriophage C21, or both">google scholar</a></td></tr>
<tr><td><a name="ref213"></a>213</td><td>Essenberg, R. C <i>1984</i>. Use of homocysteic acid for selecting mutants at the <i>gltS</i> locus of <i>Escherichia coli</i> K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Use of homocysteic acid for selecting mutants at the gltS locus of Escherichia coli K12">google scholar</a></td></tr>
<tr><td><a name="ref214"></a>214</td><td>Essenberg, R. C., and H. L. Kornberg <i>1977</i>. Location of the gene for hexose phosphate transport <i>(uhp)</i> on the chromosome of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Location of the gene for hexose phosphate transport (uhp) on the chromosome of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref215"></a>215</td><td>Ezekiel, D. H <i>1965</i>. False feedback inhibition of aromatic amino acid biosynthesis by β-2-thienyl- alanine. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=False feedback inhibition of aromatic amino acid biosynthesis by β-2-thienyl- alanine">google scholar</a></td></tr>
<tr><td><a name="ref216"></a>216</td><td>Ezekiel, D., and J. E. Hutchins <i>1968</i>. Mutations affecting RNA polymerase associated with rifampicin resistance in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutations affecting RNA polymerase associated with rifampicin resistance in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref217"></a>217</td><td>Faik, P., and H. L. Kornberg <i>1973</i>. Isolation and properties of <i>E. coli</i> mutants affected in gluconate uptake. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation and properties of E. coli mutants affected in gluconate uptake">google scholar</a></td></tr>
<tr><td><a name="ref218"></a>218</td><td>Fangman, W. L., and F. C. Neidhardt <i>1964</i>. Demonstration of an altered aminoacyl ribonucleic acid synthetase in a mutant of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Demonstration of an altered aminoacyl ribonucleic acid synthetase in a mutant of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref219"></a>219</td><td>Farley, M. M., W. M. Shafer, and J. K. Spitznagel <i>1988</i>. Lipopolysaccharide structure determines ionic and hydrophobic binding of a cationic antimicrobial neutrophil granule protein. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Lipopolysaccharide structure determines ionic and hydrophobic binding of a cationic antimicrobial neutrophil granule protein">google scholar</a></td></tr>
<tr><td><a name="ref220"></a>220</td><td>Fasano, O., and A. Parmeggiani <i>1981</i>. Altered regulation of the GTPase activity in a kirromycin resistant elongation factor Tu. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Altered regulation of the GTPase activity in a kirromycin resistant elongation factor Tu">google scholar</a></td></tr>
<tr><td><a name="ref221"></a>221</td><td>Favre, R., A. Wiater, S. Puppo, and M. Iaccarino <i>1976</i>. Expression of a valine resistant acetolactate synthase activity mediated by the <i>ilvO</i> and <i>ilvG</i> genes of <i>Escherichia coli</i> K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Expression of a valine resistant acetolactate synthase activity mediated by the ilvO and ilvG genes of Escherichia coli K12">google scholar</a></td></tr>
<tr><td><a name="ref222"></a>222</td><td>Fayerman, J. T., M. C. Vann, L S. Williams, and H. E. Umbarger <i>1979</i>. <i>ilvU,</i> a locus in <i>Escherichia coli</i> affecting the derepression of isoleucyl-tRNA synthetase and the RPC-5 chromatographic profiles of tRNAile and tRNAval. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=ilvU, a locus in Escherichia coli affecting the derepression of isoleucyl-tRNA synthetase and the RPC-5 chromatographic profiles of tRNAile and tRNAval">google scholar</a></td></tr>
<tr><td><a name="ref223"></a>223</td><td>Ferenci, T <i>1980</i>. Methyl-α-maltoside and 5-thiomaltose: analogs transported by the <i>Escherichia</i> <i>coli</i> maltose transport system. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Methyl-α-maltoside and 5-thiomaltose: analogs transported by the Escherichia coli maltose transport system">google scholar</a></td></tr>
<tr><td><a name="ref224"></a>224</td><td>Ferenci, T., H. L. Kornberg, and J. Smith <i>1971</i>. Isolation and properties of a regulatory mutant in the hexose phosphate transport system of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation and properties of a regulatory mutant in the hexose phosphate transport system of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref225"></a>225</td><td>Fillingame, R. H <i>1975</i>. Identification of the dicyclohexylcarbodiimide-reactive protein component of the adenosine 5′-triphosphate transducing system of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Identification of the dicyclohexylcarbodiimide-reactive protein component of the adenosine 5′-triphosphate transducing system of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref226"></a>226</td><td>Fillingame, R. H., M. Oldenburg, and D. Fraga <i>1991</i>. Mutation of alanine-24 to serine in subunit C of the <i>Escherichia coli</i> F-1 F-O ATP synthase reduces reactivity of aspartyl 61 with dicyclohexylcarbodiimide. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutation of alanine-24 to serine in subunit C of the Escherichia coli F-1 F-O ATP synthase reduces reactivity of aspartyl 61 with dicyclohexylcarbodiimide">google scholar</a></td></tr>
<tr><td><a name="ref227"></a>227</td><td>Filutowicz, M., Z. Ciesla, and T. Klopotowski <i>1979</i>. Interference of azide with cysteine biosynthesis in <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Interference of azide with cysteine biosynthesis in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref228"></a>228</td><td>Fimmel, A., and B. Haddock <i>1979</i>. Use of <i>chlC-lac</i> fusions to determine regulation of gene <i>chlC</i> in <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Use of chlC-lac fusions to determine regulation of gene chlC in Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref229"></a>229</td><td>Fimmel, A. L., D. A. Jans, L. Hatch, L. B. James, F. Gibson, and G. B. Cox <i>1985</i>. The F-1 F-0 ATPase of <i>Escherichia coli</i> : the substitution of alanine by threonine at position 25 in the C-subunit affects function but not assembly. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The F-1 F-0 ATPase of Escherichia coli : the substitution of alanine by threonine at position 25 in the C-subunit affects function but not assembly">google scholar</a></td></tr>
<tr><td><a name="ref230"></a>230</td><td>Fimmel, A. L., and R. E. Loughlin <i>1977</i>. Isolation and characterization of <i>cysK</i> mutants of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation and characterization of cysK mutants of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref231"></a>231</td><td>Fisher, K. E., and E. Eisenstein <i>1993</i>. An efficient approach to identify <i>ilvA</i> mutations reveals an amino-terminal catalytic domain in biosynthetic threonine deaminase from <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=An efficient approach to identify ilvA mutations reveals an amino-terminal catalytic domain in biosynthetic threonine deaminase from Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref232"></a>232</td><td>Fortin, T., P. Phoenix, and G. R. Drapeau <i>1990</i>. Mutations conferring resistance to azide in <i>Escherichia coli</i> occur primarily in the <i>secA</i> gene. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutations conferring resistance to azide in Escherichia coli occur primarily in the secA gene">google scholar</a></td></tr>
<tr><td><a name="ref233"></a>233</td><td>Foster, J. W., and H. K. Hall <i>1990</i>. Adaptive acidification tolerance response of <i>Salmonella</i> <i>typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Adaptive acidification tolerance response of Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref234"></a>234</td><td>Foster, J. W., D. M. Kinney, and A. G. Moat <i>1979</i>. Pyridine nucleotide cycle of <i>Salmonella</i> <i>typhimurium:</i> isolation and characterization of <i>pncA, pncB,</i> and <i>pncC</i> mutants and utilization of exogenous nicotinamide adenine dinucleotide. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Pyridine nucleotide cycle of Salmonella typhimurium: isolation and characterization of pncA, pncB, and pncC mutants and utilization of exogenous nicotinamide adenine dinucleotide">google scholar</a></td></tr>
<tr><td><a name="ref235"></a>235</td><td>Foulds, J <i>1976</i>. <i>tolF</i> locus in <i>Escherichia coli:</i> chromosomal location and relationship to loci <i>cmlB</i> and <i>tolD.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=tolF locus in Escherichia coli: chromosomal location and relationship to loci cmlB and tolD.">google scholar</a></td></tr>
<tr><td><a name="ref236"></a>236</td><td>Foulds, J., and C. Barrett <i>1973</i>. Characterization of <i>Escherichia coli</i> mutants tolerant to bacteriocin JF246: two new classes of tolerant mutants. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Characterization of Escherichia coli mutants tolerant to bacteriocin JF246: two new classes of tolerant mutants">google scholar</a></td></tr>
<tr><td><a name="ref237"></a>237</td><td>Foulds, J., and T.-J. Chai <i>1978</i>. New major outer membrane protein found in an <i>Escherichia coli</i> <i>tolF</i> mutant resistant to bacteriophage TuIb. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=New major outer membrane protein found in an Escherichia coli tolF mutant resistant to bacteriophage TuIb">google scholar</a></td></tr>
<tr><td><a name="ref238"></a>238</td><td>Fouts, K. E., and S. D. Barbour <i>1981</i>. Transductional mapping of <i>ksgB</i> and a new Tn<i>5</i>-induced kasugamycin resistance gene, <i>ksgD,</i> in <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Transductional mapping of ksgB and a new Tn5-induced kasugamycin resistance gene, ksgD, in Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref239"></a>239</td><td>Fradkin, J. E., and D. G. Fraenkel <i>1971</i>. 2-Keto-3-deoxygluconate 6-phosphate aldolase mutants of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=2-Keto-3-deoxygluconate 6-phosphate aldolase mutants of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref240"></a>240</td><td>Fraenkel, D. G., and S. Banerjee <i>1972</i>. Deletion mapping of <i>zwf,</i> the gene for a constitutive enzyme, glucose 6-phosphate dehydrogenase in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Deletion mapping of zwf, the gene for a constitutive enzyme, glucose 6-phosphate dehydrogenase in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref241"></a>241</td><td>Franco, P. J., J. A. Eelkema, and R. J. Brooker <i>1989</i>. Isolation and characterization of thiodigalactoside resistant mutants of the lactose permease which possess an enhanced recognition for maltose. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation and characterization of thiodigalactoside resistant mutants of the lactose permease which possess an enhanced recognition for maltose">google scholar</a></td></tr>
<tr><td><a name="ref242"></a>242</td><td>Franklin, N. C <i>1969</i>. Mutation in <i>galU</i> gene of <i>E. coli</i> blocks phage Pl infection. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutation in galU gene of E. coli blocks phage Pl infection">google scholar</a></td></tr>
<tr><td><a name="ref243"></a>243</td><td>Friedberg, I <i>1972</i>. Localization of phosphoglucose isomerase in <i>Escherichia coli</i> and its relation to the induction of the hexose phosphate transport system. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Localization of phosphoglucose isomerase in Escherichia coli and its relation to the induction of the hexose phosphate transport system">google scholar</a></td></tr>
<tr><td><a name="ref244"></a>244</td><td>Friedman, D. I <i>1971</i>. A bacterial mutant affecting lambda development.</td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A bacterial mutant affecting lambda development">google scholar</a></td></tr>
<tr><td><a name="ref245"></a>245</td><td>Friedman, D. I., and M. Baumann <i>1976</i>. Cooperative effects of bacterial mutations affecting λ N gene expression. 1. Isolation and characterization of a <i>nusB</i> mutant. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Cooperative effects of bacterial mutations affecting λ N gene expression. 1. Isolation and characterization of a nusB mutant">google scholar</a></td></tr>
<tr><td><a name="ref246"></a>246</td><td>Friedman, D. I., and M. Gottesman <i>1983</i>. Lytic mode of λ development.</td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Lytic mode of λ development">google scholar</a></td></tr>
<tr><td><a name="ref247"></a>247</td><td>Friedman, S <i>1982</i>. Bactericidal effect of 5-azacytidine on <i>Escherichia coli</i> carrying <i>Eco</i>RII restriction-modification enzymes. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Bactericidal effect of 5-azacytidine on Escherichia coli carrying EcoRII restriction-modification enzymes">google scholar</a></td></tr>
<tr><td><a name="ref248"></a>248</td><td>Froehler, J., A. Rechenmacher, J. Thomale, G. Nass, and A. Boeck <i>1980</i>. Genetic analysis of mutations causing borrelidin resistance by overproduction of threonyl-transfer ribonucleic acid synthetase. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic analysis of mutations causing borrelidin resistance by overproduction of threonyl-transfer ribonucleic acid synthetase">google scholar</a></td></tr>
<tr><td><a name="ref249"></a>249</td><td>Fujita, H., S. Yamaguchi, T. Taira, and T. Iino <i>1981</i>. A simple method for the isolation of flagellar shape mutants in <i>Salmonella.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A simple method for the isolation of flagellar shape mutants in Salmonella.">google scholar</a></td></tr>
<tr><td><a name="ref250"></a>250</td><td>Furukawa, H., J.-T. Tsay, S. Jackowski, Y. Takamura, and C. O. Rock <i>1993</i>. Thiolactomycin resistance in <i>Escherichia coli</i> is associated with the multidrug resistance efflux pump encoded by <i>emrAB.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Thiolactomycin resistance in Escherichia coli is associated with the multidrug resistance efflux pump encoded by emrAB.">google scholar</a></td></tr>
<tr><td><a name="ref251"></a>251</td><td>Garber, B. B., and J. S. Gots <i>1980</i>. Utilization of 2,6-diaminopurine by <i>Salmonella</i> <i>typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Utilization of 2,6-diaminopurine by Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref252"></a>252</td><td>Gayda, R. C., L. T. Yamamoto, and A. Markovitz <i>1976</i>. Second-site mutations in <i>capR(lon)</i> strains of <i>Escherichia coli</i> K-12 that prevent radiation sensitivity and allow bacteriophage lambda to lysogenize. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Second-site mutations in capR(lon) strains of Escherichia coli K-12 that prevent radiation sensitivity and allow bacteriophage lambda to lysogenize">google scholar</a></td></tr>
<tr><td><a name="ref253"></a>253</td><td>Gehring, K., A. Charbit, E. Brissaud, and M. Hofnung <i>1987</i>. Bacteriophage lambda receptor site on the <i>Escherichia coli</i> K-12 <i>lamB</i> protein. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Bacteriophage lambda receptor site on the Escherichia coli K-12 lamB protein">google scholar</a></td></tr>
<tr><td><a name="ref254"></a>254</td><td>Gellert, M., and M. L. Bullock <i>1970</i>. DNA ligase mutants of <i>Escherichia coli.</i> <i>Proc. Natl. Acad.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=DNA ligase mutants of Escherichia coli. Proc. Natl. Acad.">google scholar</a></td></tr>
<tr><td><a name="ref255"></a>255</td><td>Gellert, M., M. H. O’Dea, T. Itoh, and J.-I. Tomizawa <i>1976</i>. Novobiocin and coumermycin inhibit DNA supercoiling catalyzed by DNA gyrase. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Novobiocin and coumermycin inhibit DNA supercoiling catalyzed by DNA gyrase">google scholar</a></td></tr>
<tr><td><a name="ref256"></a>256</td><td>Georgopoulos, C. P <i>1971</i>. Bacterial mutants in which the gene N function of bacteriophage lambda is blocked have an altered RNA polymerase. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Bacterial mutants in which the gene N function of bacteriophage lambda is blocked have an altered RNA polymerase">google scholar</a></td></tr>
<tr><td><a name="ref257"></a>257</td><td>Georgopoulos, C. P <i>1977</i>. A new bacterial gene <i>groPC</i> which affects phage lambda DNA replication. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A new bacterial gene groPC which affects phage lambda DNA replication">google scholar</a></td></tr>
<tr><td><a name="ref258"></a>258</td><td>Georgopoulos, C. P., R. W. Hendrix, and A. D. Kaiser <i>1972</i>. Role of the host cell in bacteriophage morphogenesis: effects of a bacterial mutation on T4 head assembly. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Role of the host cell in bacteriophage morphogenesis: effects of a bacterial mutation on T4 head assembly">google scholar</a></td></tr>
<tr><td><a name="ref259"></a>259</td><td>Georgopoulos, C. P., and I. Herskowitz <i>1971</i>. <i>Escherichia coli</i> mutants blocked in lambda DNA synthesis.</td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Escherichia coli mutants blocked in lambda DNA synthesis">google scholar</a></td></tr>
<tr><td><a name="ref260"></a>260</td><td>Georgopoulos, C. P., J. Swindle, F. Keppel, M. Ballivet, R. Bisig, and H. Eisen <i>1980</i>. Studies on the <i>E.</i> coli <i>groNB (nusB)</i> gene which affects bacteriophage λ N gene function. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Studies on the E. coli groNB (nusB) gene which affects bacteriophage λ N gene function">google scholar</a></td></tr>
<tr><td><a name="ref261"></a>261</td><td>Ghysen, A., and M. Pirionio <i>1972</i>. Relationship between the <i>N</i> function of bacteriophage λ and host RNA polymerase. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Relationship between the N function of bacteriophage λ and host RNA polymerase">google scholar</a></td></tr>
<tr><td><a name="ref262"></a>262</td><td>Gibson, M. M., D. A. Bagga, C. G. Miller, and M. E. Maguire <i>1991</i>. Magnesium transport in <i>Salmonella typhimurium:</i> the influence of new mutations conferring Co2+ resistance on the CorA Mg2+transport system. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Magnesium transport in Salmonella typhimurium: the influence of new mutations conferring Co2+ resistance on the CorA Mg2+transport system">google scholar</a></td></tr>
<tr><td><a name="ref263"></a>263</td><td>Gibson, M. M., M. Price, and C. F. Higgins <i>1984</i>. Genetic characterization and molecular cloning of the tripeptide permease (<i>tpp</i>) genes of <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic characterization and molecular cloning of the tripeptide permease (tpp) genes of Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref264"></a>264</td><td>Ginther, C. L., and J. L. Ingraham <i>1974</i>. Cold-sensitive mutant of <i>Salmonella typhimurium</i> defective in nucleoside diphosphokinase. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Cold-sensitive mutant of Salmonella typhimurium defective in nucleoside diphosphokinase">google scholar</a></td></tr>
<tr><td><a name="ref265"></a>265</td><td>Giordano, G., L. Grillet, R. Rosset, J. H. Dou, and E. Azoulay <i>1978</i>. Characterization of an <i>E.</i> <i>coli</i> mutant that is sensitive to chlorate when grown aerobically. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Characterization of an E. coli mutant that is sensitive to chlorate when grown aerobically">google scholar</a></td></tr>
<tr><td><a name="ref266"></a>266</td><td>Glaser, J. H., and J. A. DeMoss <i>1972</i>. Comparison of nitrate reductase mutants of <i>Escherichia</i> <i>coli</i> selected by alternative procedures. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Comparison of nitrate reductase mutants of Escherichia coli selected by alternative procedures">google scholar</a></td></tr>
<tr><td><a name="ref267"></a>267</td><td>Glickman, B. W., and M. Radman <i>1980</i>. <i>Escherichia coli</i> mutator mutants deficient in methylation instructed DNA mismatch correction. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Escherichia coli mutator mutants deficient in methylation instructed DNA mismatch correction">google scholar</a></td></tr>
<tr><td><a name="ref268"></a>268</td><td>Glover, S. W <i>1962</i>. Valine-resistant mutants of <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Valine-resistant mutants of Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref269"></a>269</td><td>Gollop, N., H. Tavori, and Z. Barak <i>1982</i>. Acetohydroxy acid synthase is a target for leucine- containing peptide toxicity in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Acetohydroxy acid synthase is a target for leucine- containing peptide toxicity in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref270"></a>270</td><td>Gollub, E., and D. B. Sprinson <i>1969</i>. A regulatory mutation in tyrosine biosynthesis. <i>Biochem.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A regulatory mutation in tyrosine biosynthesis. Biochem.">google scholar</a></td></tr>
<tr><td><a name="ref271"></a>271</td><td>Gollub, E. G., K. P. Liu, and D. B. Sprinson <i>1973</i>. A regulatory gene of phenylalanine biosynthesis <i>(pheR)</i> in <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A regulatory gene of phenylalanine biosynthesis (pheR) in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref272"></a>272</td><td>Gots, J. S., C. E. Benson, and S. R. Shumas <i>1972</i>. Genetic separation of hypoxanthine and guanine-xanthine phosphoribosyltransferase activities by deletion mutations in <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic separation of hypoxanthine and guanine-xanthine phosphoribosyltransferase activities by deletion mutations in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref273"></a>273</td><td>Green, L., and C. G. Mlller <i>1980</i>. Genetic mapping of the <i>Salmonella typhimurium pepB</i> locus. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic mapping of the Salmonella typhimurium pepB locus">google scholar</a></td></tr>
<tr><td><a name="ref274"></a>274</td><td>Greenberg, J. T., J. H. Chou, P. A. Monach, and B. Demple <i>1991</i>. Activation of oxidative stress genes by mutations at the <i>soxQ</i>/<i>cfxB</i>/<i>marA</i> locus of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Activation of oxidative stress genes by mutations at the soxQ/cfxB/marA locus of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref275"></a>275</td><td>Greene, R. C., J. V. Hunter, and E. H. Coch <i>1973</i>. Properties of <i>metK</i> mutants of <i>Escherichia</i> <i>coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Properties of metK mutants of Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref276"></a>276</td><td>Gruell, J. M., H. Hennecke, J. Froehler, J. Thomale, G. Nass, and A. Böck <i>1979</i>. <i>Escherichia</i> <i>coli</i> mutants overproducing phenylalanyl transfer RNA synthetase and threonyl transfer RNA synthetase. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Escherichia coli mutants overproducing phenylalanyl transfer RNA synthetase and threonyl transfer RNA synthetase">google scholar</a></td></tr>
<tr><td><a name="ref277"></a>277</td><td>Guardiola, J., M. DeFelice, T. Klopotowski, and M. Iaccarino <i>1974</i>. Mutations affecting the different transport systems for isoleucine, leucine, and valine in <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutations affecting the different transport systems for isoleucine, leucine, and valine in Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref278"></a>278</td><td>Guardiola, J., and M. Iaccarino <i>1971</i>. <i>Escherichia coli</i> K-12 mutants altered in the transport of branched-chain amino acids. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Escherichia coli K-12 mutants altered in the transport of branched-chain amino acids">google scholar</a></td></tr>
<tr><td><a name="ref279"></a>279</td><td>Guarneros, G., G. Machado, P. Guzman and E. Garay <i>1987</i>. Genetic and physical location of the <i>Escherichia coli</i> <i>rap</i> locus, which is essential for growth of bacteriophage lambda. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic and physical location of the Escherichia coli rap locus, which is essential for growth of bacteriophage lambda">google scholar</a></td></tr>
<tr><td><a name="ref280"></a>280</td><td>Guest, J. R <i>1969</i>. Biochemical and genetic studies with nitrate reductase C-gene mutants of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Biochemical and genetic studies with nitrate reductase C-gene mutants of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref281"></a>281</td><td>Guest, J. R <i>1979</i>. Anaerobic growth of <i>Escherichia coli</i> K12 with fumarate as terminal electron acceptor. Genetic studies with menaquinone and fluoroacetate-resistant mutants. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Anaerobic growth of Escherichia coli K12 with fumarate as terminal electron acceptor. Genetic studies with menaquinone and fluoroacetate-resistant mutants">google scholar</a></td></tr>
<tr><td><a name="ref282"></a>282</td><td>Gupta, A., and S. Kumar <i>1985</i>. Roles of internally and externally supplied cyclic AMP in the growth response of <i>Escherichia coli</i> to sodium chloride. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Roles of internally and externally supplied cyclic AMP in the growth response of Escherichia coli to sodium chloride">google scholar</a></td></tr>
<tr><td><a name="ref283"></a>283</td><td>Guterman, S. K., and L. Dann <i>1973</i>. Excretion of enterochelin by <i>exbA</i> and <i>exbB</i> mutants of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Excretion of enterochelin by exbA and exbB mutants of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref284"></a>284</td><td>Guterman, S. K., A. Wright, and D. H. Boyd <i>1975</i>. Genes affecting coliphage BF23 and E colicin sensitivity in <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genes affecting coliphage BF23 and E colicin sensitivity in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref285"></a>285</td><td>Guth, A., R. Engel, and B. E. Tropp <i>1980</i>. Uptake of glycerol 3-phosphate and some of its analogs by the hexose phosphate transport system of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Uptake of glycerol 3-phosphate and some of its analogs by the hexose phosphate transport system of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref286"></a>286</td><td>Hachler, H., S. P. Cohen, and S. B. Levy <i>1991</i>. <i>marA,</i> a regulated locus which controls expression of chromosomal multiple antibiotic resistance in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=marA, a regulated locus which controls expression of chromosomal multiple antibiotic resistance in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref287"></a>287</td><td>Hacking, A. J., and E. C. C. Lin <i>1976</i>. Disruption of the fucose pathway as a consequence of genetic adaptation to propanediol as a carbon source in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Disruption of the fucose pathway as a consequence of genetic adaptation to propanediol as a carbon source in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref288"></a>288</td><td>Hadar, R., and J. Kuhn <i>1977</i>. A mutant constitutive for aromatic permease. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A mutant constitutive for aromatic permease">google scholar</a></td></tr>
<tr><td><a name="ref289"></a>289</td><td>Hafner, E. W., C. W. Tabor, and H. Tabor <i>1977</i>. Isolation of a <i>metK</i> mutant with a temperature- sensitive S-adenosylmethionine synthetase. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation of a metK mutant with a temperature- sensitive S-adenosylmethionine synthetase">google scholar</a></td></tr>
<tr><td><a name="ref290"></a>290</td><td>Hall, B. G <i>1982</i>. Chromosomal mutation for citrate utilization by <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Chromosomal mutation for citrate utilization by Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref291"></a>291</td><td>Hall, M. N., and T. J. Silhavy <i>1981</i>. The <i>ompB</i> locus and the regulation of the major outer membrane porin proteins of <i>Escherichia coli</i> K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The ompB locus and the regulation of the major outer membrane porin proteins of Escherichia coli K12">google scholar</a></td></tr>
<tr><td><a name="ref292"></a>292</td><td>Halpern, Y. S., and H. E. Umbarger <i>1961</i>. Utilization of L-glutamic acid and 2-oxoglutaric acid as sole sources of carbon by <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Utilization of L-glutamic acid and 2-oxoglutaric acid as sole sources of carbon by Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref293"></a>293</td><td>Halsall, D. M <i>1975</i>. Strains of <i>E. coli</i> with defective lysine transport systems. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Strains of E. coli with defective lysine transport systems">google scholar</a></td></tr>
<tr><td><a name="ref294"></a>294</td><td>Hama, H., Y. Sumita, Y. Kakutani, M. Tsuda, and T. Tsuchiya <i>1990</i>. Target of serine inhibition in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Target of serine inhibition in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref295"></a>295</td><td>Hammelburger, J. W., and G. A. Orr <i>1983</i>. Interaction of <i>sn</i>-glycerol 3-phosphorothioate with <i>Escherichia coli:</i> effect on cell growth and metabolism. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Interaction of sn-glycerol 3-phosphorothioate with Escherichia coli: effect on cell growth and metabolism">google scholar</a></td></tr>
<tr><td><a name="ref296"></a>296</td><td>Hammer, K., K. F. Jensen, P. Poulsen, A. B. Oppenheim, and M. Gottesman <i>1987</i>. Isolation of <i>Escherichia coli</i> <i>rpoB</i> mutants resistant to killing by lambda cII protein and altered in <i>pyrE</i> gene attenuation. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation of Escherichia coli rpoB mutants resistant to killing by lambda cII protein and altered in pyrE gene attenuation">google scholar</a></td></tr>
<tr><td><a name="ref297"></a>297</td><td>Hancock, R. E. W., S. W. Farmer, Z. Li, and K. Poole <i>1991</i>. Interaction of aminoglycosides with the outer membranes and purified lipopolysaccharide and <i>ompF</i> porin of <i>Escherichia coli</i>. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Interaction of aminoglycosides with the outer membranes and purified lipopolysaccharide and ompF porin of Escherichia coli">google scholar</a></td></tr>
<tr><td><a name="ref298"></a>298</td><td>Hane, M. W., and T. H. Wood <i>1969</i>. <i>Escherichia coli</i> K-12 mutants resistant to nalidixic acid: genetic mapping and dominance studies. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Escherichia coli K-12 mutants resistant to nalidixic acid: genetic mapping and dominance studies">google scholar</a></td></tr>
<tr><td><a name="ref299"></a>299</td><td>Hantke, K <i>1976</i>. Phage T6-colicin K receptor and nucleoside transport in <i>Escherichia coli.</i> <i>FEBS</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Phage T6-colicin K receptor and nucleoside transport in Escherichia coli. FEBS">google scholar</a></td></tr>
<tr><td><a name="ref300"></a>300</td><td>Hantke, K <i>1987</i>. Selection procedure for deregulated iron transport mutants (<i>fur</i>) in <i>Escherichia</i> <i>coli</i> K12: <i>fur</i> not only affects iron metabolism. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Selection procedure for deregulated iron transport mutants (fur) in Escherichia coli K12: fur not only affects iron metabolism">google scholar</a></td></tr>
<tr><td><a name="ref301"></a>301</td><td>Hardaway, K. L., and C. S. Buller <i>1979</i>. Effects of ethylenediaminetetraacetate on phospholipids and outer membrane function in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Effects of ethylenediaminetetraacetate on phospholipids and outer membrane function in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref302"></a>302</td><td>Harkness, R. E., and V. Braun <i>1990</i>. In vitro peptidoglycan synthesis by envelopes from <i>Escherichia coli tolM</i> mutants is inhibited by colicin M. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=In vitro peptidoglycan synthesis by envelopes from Escherichia coli tolM mutants is inhibited by colicin M">google scholar</a></td></tr>
<tr><td><a name="ref303"></a>303</td><td>Harrison, L. I., H. N. Christiansen, M. E. Handlogten, D. L. Oxender, and S. C. Quay <i>1975</i>. Transport of L-4-azaleucine in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Transport of L-4-azaleucine in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref304"></a>304</td><td>Hasenbank, R., C. Guthrie, G. Stoffler, H. G. Wittmann, L. Rosen, and D. Apirion <i>1973</i>. Electrophoretic and immunological studies on ribosomal proteins of 100 <i>Escherichia coli</i> revertants from streptomycin dependence. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Electrophoretic and immunological studies on ribosomal proteins of 100 Escherichia coli revertants from streptomycin dependence">google scholar</a></td></tr>
<tr><td><a name="ref305"></a>305</td><td>Heatwole, V. M., and R. L. Somerville <i>1991</i>. Cloning, nucleotide sequence, and characterization of <i>mtr,</i> the structural gene for a tryptophan-specific permease of <i>Escherichia</i> <i>coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Cloning, nucleotide sequence, and characterization of mtr, the structural gene for a tryptophan-specific permease of Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref306"></a>306</td><td>Hedges, R. W., and K. P. Shannon <i>1984</i>. Resistance to apramycin in <i>Escherichia coli</i> isolated from animals: detection of a novel aminoglycoside-modifying enzyme. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Resistance to apramycin in Escherichia coli isolated from animals: detection of a novel aminoglycoside-modifying enzyme">google scholar</a></td></tr>
<tr><td><a name="ref307"></a>307</td><td>Held, W. A., and O. H. Smith <i>1970</i>. Regulation of the <i>Escherichia coli</i> tryptophan operon by early reactions in the aromatic pathway. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation of the Escherichia coli tryptophan operon by early reactions in the aromatic pathway">google scholar</a></td></tr>
<tr><td><a name="ref308"></a>308</td><td>Held, W. A., and O. H. Smith <i>1970</i>. Mechanism of 3-methylanthranilic acid derepression of the tryptophan operon in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mechanism of 3-methylanthranilic acid derepression of the tryptophan operon in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref309"></a>309</td><td>Helling, R. B <i>1968</i>. Selection of a mutant of <i>Escherichia coli</i> which has high mutation rates. <i>J.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Selection of a mutant of Escherichia coli which has high mutation rates. J.">google scholar</a></td></tr>
<tr><td><a name="ref310"></a>310</td><td>Helling, R. B., and B. S. Adams <i>1970</i>. Nalidixic acid-resistant auxotrophs of <i>Escherichia coli.</i> <i>J.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Nalidixic acid-resistant auxotrophs of Escherichia coli. J.">google scholar</a></td></tr>
<tr><td><a name="ref311"></a>311</td><td>Helling, R. B., and J. Kukora <i>1971</i>. Nalidixic acid-resistant mutants of <i>Escherichia coli</i> deficient in isocitrate dehydrogenase. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Nalidixic acid-resistant mutants of Escherichia coli deficient in isocitrate dehydrogenase">google scholar</a></td></tr>
<tr><td><a name="ref312"></a>312</td><td>Helser, T. L., J. E. Davies, and J. E. Dahlberg <i>1972</i>. Mechanism of kasugamycin resistance in <i>Escherichia coli.</i> <i>Nature</i>  </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mechanism of kasugamycin resistance in Escherichia coli. Nature">google scholar</a></td></tr>
<tr><td><a name="ref313"></a>313</td><td>Hengge, R., T. J. Larson, and W. Boos <i>1983</i>. <i>sn</i>-Glycerol-3-phosphate transport in <i>Salmonella</i> <i>typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=sn-Glycerol-3-phosphate transport in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref314"></a>314</td><td>Henning, U., I. Hindennach, and I. Haller <i>1976</i>. The major proteins of the <i>Escherichia coli</i> outer cell envelope membrane: evidence for the structural gene of protein II. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The major proteins of the Escherichia coli outer cell envelope membrane: evidence for the structural gene of protein II">google scholar</a></td></tr>
<tr><td><a name="ref315"></a>315</td><td>Heyde, M., and R. Portalier <i>1982</i>. New pleiotropic alkaline phosphatase-negative mutants of <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=New pleiotropic alkaline phosphatase-negative mutants of Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref316"></a>316</td><td>Heyde, M., and R. Portalier <i>1983</i>. Isolation and characterization of a new type of <i>Escherichia</i> <i>coli</i> K12 <i>phoB</i> mutants. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation and characterization of a new type of Escherichia coli K12 phoB mutants">google scholar</a></td></tr>
<tr><td><a name="ref317"></a>317</td><td>Higgins, C. F., M. M. Hardie, D. Jamieson, and L. M. Powell <i>1983</i>. Genetic map of the <i>opp</i> (oligopeptide permease) locus of <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic map of the opp (oligopeptide permease) locus of Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref318"></a>318</td><td>Hiraga, S <i>1969</i>. Operator mutants of the tryptophan operon in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Operator mutants of the tryptophan operon in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref319"></a>319</td><td>Hiraga, S., K. Igarashi, and T. Yura <i>1967</i>. A deoxythymidine kinase deficient mutant of <i>Escherichia coli.</i> I. Isolation and some properties. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A deoxythymidine kinase deficient mutant of Escherichia coli. I. Isolation and some properties">google scholar</a></td></tr>
<tr><td><a name="ref320"></a>320</td><td>Hiraga, S., K. Ito, T. Matsuyama, H. Ozaki, and T. Yura <i>1968</i>. 5-Methyltryptophan-resistant mutations linked with the arginine G marker in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=5-Methyltryptophan-resistant mutations linked with the arginine G marker in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref321"></a>321</td><td>Hirota, Y., M. Inuzuka, and M. Tomoeda <i>1966</i>. Elective selection of proline-requiring mutants. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Elective selection of proline-requiring mutants">google scholar</a></td></tr>
<tr><td><a name="ref322"></a>322</td><td>Hirshfield, I. N., R. Dedeken, P. C. Horn, D. A. Hopwood, and W. K. Maas <i>1968</i>. Studies on the mechanism of repression of arginine biosynthesis in <i>Escherichia coli.</i> III. Repression of enzymes of arginine biosyntheses in arginyl-tRNA synthetase mutants. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Studies on the mechanism of repression of arginine biosynthesis in Escherichia coli. III. Repression of enzymes of arginine biosyntheses in arginyl-tRNA synthetase mutants">google scholar</a></td></tr>
<tr><td><a name="ref323"></a>323</td><td>Hirshfield, I. N., and P. C. Zamecnik <i>1972</i>. Thiosine-resistant mutants of <i>Escherichia coli</i> K-12 with growth medium-dependent lysyl-tRNA synthetase activity. I. Isolation and physiological characterization. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Thiosine-resistant mutants of Escherichia coli K-12 with growth medium-dependent lysyl-tRNA synthetase activity. I. Isolation and physiological characterization">google scholar</a></td></tr>
<tr><td><a name="ref324"></a>324</td><td>Hobson, A. C <i>1974</i>. A norleucine-resistant mutant of <i>Salmonella typhimurium</i> with a possible defect in valine uptake or regulation. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A norleucine-resistant mutant of Salmonella typhimurium with a possible defect in valine uptake or regulation">google scholar</a></td></tr>
<tr><td><a name="ref325"></a>325</td><td>Hoffee, P. A <i>1968</i>. 2-Deoxyribose gene-enzyme complex in <i>Salmonella typhimurium.</i> <i>J.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=2-Deoxyribose gene-enzyme complex in Salmonella typhimurium. J.">google scholar</a></td></tr>
<tr><td><a name="ref326"></a>326</td><td>Hoffmeyer, J., and J. Neuhard <i>1971</i>. Metabolism of exogenous purine bases and nucleosides by <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Metabolism of exogenous purine bases and nucleosides by Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref327"></a>327</td><td>Hofnung, M., and M. Schwartz <i>1971</i>. Mutations allowing growth on maltose of <i>Escherichia coli</i> K-12 strains with a deleted <i>malT</i> gene. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutations allowing growth on maltose of Escherichia coli K-12 strains with a deleted malT gene">google scholar</a></td></tr>
<tr><td><a name="ref328"></a>328</td><td>Holland, J., I. B. Holland, and S. I. Ahmad <i>1991</i>. DNA damage by 8-methoxypsoralen plus near UV light, <i>puvA</i> and its repair in <i>Escherichia coli:</i> genetic analysis. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=DNA damage by 8-methoxypsoralen plus near UV light, puvA and its repair in Escherichia coli: genetic analysis">google scholar</a></td></tr>
<tr><td><a name="ref329"></a>329</td><td>Hooper, D. C., J. S. Wolfson, M. A. Bozza, and E. Y. Ng <i>1992</i>. Genetics and regulation of outer membrane protein expression by quinolone resistance loci <i>nfxB,</i> <i>nfxC,</i> and <i>cfxB.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetics and regulation of outer membrane protein expression by quinolone resistance loci nfxB, nfxC, and cfxB.">google scholar</a></td></tr>
<tr><td><a name="ref330"></a>330</td><td>Horiuchi, T., S. Horiuchi, and A. Novick <i>1963</i>. The genetic basis of hyper-synthesis of β- galactoside. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The genetic basis of hyper-synthesis of β- galactoside">google scholar</a></td></tr>
<tr><td><a name="ref331"></a>331</td><td>Housley, P. R., A. D. Leavitt, and H. J. Whitfield <i>1981</i>. Genetic analysis of a temperature- sensitive <i>Salmonella typhimurium</i> <i>rho</i> mutant with an altered Rho-associated polycytidylate- dependent adenosine triphosphatase activity. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic analysis of a temperature- sensitive Salmonella typhimurium rho mutant with an altered Rho-associated polycytidylate- dependent adenosine triphosphatase activity">google scholar</a></td></tr>
<tr><td><a name="ref332"></a>332</td><td>Howard-Flanders, P., and L. Theriot <i>1962</i>. A method for selecting radiation-sensitive mutants of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A method for selecting radiation-sensitive mutants of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref333"></a>333</td><td>Hrebenda, J., H. Heleszko, K. Brzostek, and J. Bielecki <i>1985</i>. Mutation affecting resistance of <i>Escherichia coli</i> K12 to nalidixic acid. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutation affecting resistance of Escherichia coli K12 to nalidixic acid">google scholar</a></td></tr>
<tr><td><a name="ref334"></a>334</td><td>Hudson, H. P., A. A. Lindberg, and B. A. D. Stocker <i>1978</i>. Lipopolysaccharide core defects in <i>Salmonella typhimurium</i> mutants which are resistant to Felix O phage but retain smooth character. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Lipopolysaccharide core defects in Salmonella typhimurium mutants which are resistant to Felix O phage but retain smooth character">google scholar</a></td></tr>
<tr><td><a name="ref335"></a>335</td><td>Hughes, D <i>1986</i>. The isolation and mapping of EF-Tu mutations in <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The isolation and mapping of EF-Tu mutations in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref336"></a>336</td><td>Hughes, K. T., B. T. Cookson, D. Ladika, B. M. Olivera, and J. R. Roth <i>1983</i>. 6- Aminonicotinamide-resistant mutants of <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=6- Aminonicotinamide-resistant mutants of Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref337"></a>337</td><td>Hughes, K. T., D. Ladika, J. R. Roth, and B. M. Olivera <i>1983</i>. An indispensable gene for NAD biosynthesis in <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=An indispensable gene for NAD biosynthesis in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref338"></a>338</td><td>Hulanicka, D <i>1972</i>. Resistance to sulfate analogs in <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Resistance to sulfate analogs in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref339"></a>339</td><td>Hulanicka, D., and T. Klopotowsi <i>1972</i>. Mutants of <i>Salmonella typhimurium</i> resistant to triazole. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutants of Salmonella typhimurium resistant to triazole">google scholar</a></td></tr>
<tr><td><a name="ref340"></a>340</td><td>Hulanicka, M. D., S. G. Hallquist, N. M. Kredich, and T. Mojica-A <i>1979</i>. Regulation of O- acetylserine sulfhydrylase B by L-cysteine in <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation of O- acetylserine sulfhydrylase B by L-cysteine in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref341"></a>341</td><td>Hulanicka, M. D., and N. M. Kredich <i>1976</i>. A mutation affecting expression of the gene coding for serine transacetylase in <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A mutation affecting expression of the gene coding for serine transacetylase in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref342"></a>342</td><td>Hull, R., J. D. Klinger, and E. E. M. Moody <i>1976</i>. Isolation and characterization of mutants of <i>Escherichia</i> <i>coli</i> K12 resistant to the new aminoglycoside antibiotic, amikacin. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation and characterization of mutants of Escherichia coli K12 resistant to the new aminoglycoside antibiotic, amikacin">google scholar</a></td></tr>
<tr><td><a name="ref343"></a>343</td><td>Hummel, H., W. Piepersberg, and A. Böck <i>1979</i>. Analysis of lincomycin resistance mutations in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Analysis of lincomycin resistance mutations in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref344"></a>344</td><td>Hussain, M., S. Ichihara, and S. Mizushima <i>1980</i>. Accumulation of glyceride containing precursor of the outer membrane lipoprotein in the cytoplasmic membrane of <i>Escherichia coli</i> treated with globomycin. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Accumulation of glyceride containing precursor of the outer membrane lipoprotein in the cytoplasmic membrane of Escherichia coli treated with globomycin">google scholar</a></td></tr>
<tr><td><a name="ref345"></a>345</td><td>Hwang, Y.-W., R. Engel, and B. E. Tropp <i>1984</i>. Correlation of 3,4-dihydroxybutyl 1- phosphonate resistance with a defect in cardiolipin synthesis in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Correlation of 3,4-dihydroxybutyl 1- phosphonate resistance with a defect in cardiolipin synthesis in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref346"></a>346</td><td>Ibe, S. N., A. J. Sinskey, and D. Botstein <i>1982</i>. Genetic mapping of mutations in a highly radiation-resistant mutant of <i>Salmonella typhimurium</i> LT2. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic mapping of mutations in a highly radiation-resistant mutant of Salmonella typhimurium LT2">google scholar</a></td></tr>
<tr><td><a name="ref347"></a>347</td><td>Icho, T., and T. Iino <i>1978</i>. Isolation and characterization of motile <i>Escherichia coli</i> mutants resistant to bacteriophage χ. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation and characterization of motile Escherichia coli mutants resistant to bacteriophage χ">google scholar</a></td></tr>
<tr><td><a name="ref348"></a>348</td><td>Igarishi, K., S. Hiraga, and T. Yura <i>1967</i>. A deoxythymidine kinase deficient mutant of <i>Escherichia coli.</i> II. Mapping and transduction studies with phage φ80. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A deoxythymidine kinase deficient mutant of Escherichia coli. II. Mapping and transduction studies with phage φ80">google scholar</a></td></tr>
<tr><td><a name="ref349"></a>349</td><td>Iino, T <i>1977</i>. Genetics of structure and function of bacterial flagella. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetics of structure and function of bacterial flagella">google scholar</a></td></tr>
<tr><td><a name="ref350"></a>350</td><td>Ilag, L. L., D. Jahn, G. Eggertsson, and D. Söll <i>1991</i>. The <i>Escherichia coli</i> <i>hemL</i> gene encodes glutamate 1-semialdehyde aminotransferase. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The Escherichia coli hemL gene encodes glutamate 1-semialdehyde aminotransferase">google scholar</a></td></tr>
<tr><td><a name="ref351"></a>351</td><td>Im, S. W. K., and J. Pittard <i>1971</i>. Phenylalanine biosynthesis in <i>Escherichia coli</i> K-12: mutants derepressed for chorismate mutase P-prephenate dehydratase. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Phenylalanine biosynthesis in Escherichia coli K-12: mutants derepressed for chorismate mutase P-prephenate dehydratase">google scholar</a></td></tr>
<tr><td><a name="ref352"></a>352</td><td>Inoko, H., and M. Imai <i>1976</i>. Isolation and genetic characterization of the <i>nitA</i> mutants of <i>Escherichia coli</i> affecting the termination factor rho. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation and genetic characterization of the nitA mutants of Escherichia coli affecting the termination factor rho">google scholar</a></td></tr>
<tr><td><a name="ref353"></a>353</td><td>Inoko, H., K. Shigesada, and M. Imai <i>1977</i>. Isolation and characterization of conditional-lethal <i>rho</i> mutants of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation and characterization of conditional-lethal rho mutants of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref354"></a>354</td><td>Ishidsu, J.-I <i>1973</i>. Device of a method for isolation of arginine sensitive mutants in <i>Salmonella</i> <i>typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Device of a method for isolation of arginine sensitive mutants in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref355"></a>355</td><td>Ito, M., and Y. Ohnishi <i>1981</i>. Isolation of <i>Escherichia coli</i> mutants which are resistant to an inhibitor of proton-ATPase, tributyltin and also to uncouplers of oxidative phosphorylation. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation of Escherichia coli mutants which are resistant to an inhibitor of proton-ATPase, tributyltin and also to uncouplers of oxidative phosphorylation">google scholar</a></td></tr>
<tr><td><a name="ref356"></a>356</td><td>Ivell, R., O. Fasano, J.-B. Crechet, and A. Parmeggiani <i>1981</i>. Characterization of a kirromycin resistant elongation factor Tu from <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Characterization of a kirromycin resistant elongation factor Tu from Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref357"></a>357</td><td>Iwakura, Y., A. Ishihama, and T. Yura <i>1973</i>. RNA polymerase mutants of <i>Escherichia coli.</i> II. Streptolydigin resistance and its relation to rifampicin resistance. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=RNA polymerase mutants of Escherichia coli. II. Streptolydigin resistance and its relation to rifampicin resistance">google scholar</a></td></tr>
<tr><td><a name="ref358"></a>358</td><td>Iwaya, M., R. Goldman, D. J. Tipper, B. Feingold, and J. L. Strominger <i>1978</i>. Morphology of an <i>Escherichia coli</i> mutant with a temperature-dependent round cell shape. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Morphology of an Escherichia coli mutant with a temperature-dependent round cell shape">google scholar</a></td></tr>
<tr><td><a name="ref359"></a>359</td><td>Iwaya, M., C. W. Jones, J. Khorana, and J. L. Strominger <i>1978</i>. Mapping of the mecillinam- resistant, round morphological mutants of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mapping of the mecillinam- resistant, round morphological mutants of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref360"></a>360</td><td>Jackson, B. J., J.-P. Bohin, and E. P. Kennedy <i>1984</i>. Biosynthesis of membrane-derived oligosaccharides: characterization of <i>mdoB</i> mutants defective in phosphoglycerol transferase I activity. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Biosynthesis of membrane-derived oligosaccharides: characterization of mdoB mutants defective in phosphoglycerol transferase I activity">google scholar</a></td></tr>
<tr><td><a name="ref361"></a>361</td><td>Jackson, J. H., E. J. Davis, A. C. Madu, and S. E. Braxter <i>1981</i>. Three-factor reciprocal cross mapping of a gene that causes expression of feedback resistant acetohydroxy acid synthase in <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Three-factor reciprocal cross mapping of a gene that causes expression of feedback resistant acetohydroxy acid synthase in Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref362"></a>362</td><td>Jaffe, A., Y. A. Chabbert, and E. Derlot <i>1983</i>. Selection and characterization of beta-lactam- resistant <i>Escherichia coli</i> K-12 mutants. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Selection and characterization of beta-lactam- resistant Escherichia coli K-12 mutants">google scholar</a></td></tr>
<tr><td><a name="ref363"></a>363</td><td>Jaffe, A., Y. A. Chabbert, and O. Semonin <i>1982</i>. Role of porin proteins <i>ompF</i> and <i>ompC</i> in the permeation of beta lactams. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Role of porin proteins ompF and ompC in the permeation of beta lactams">google scholar</a></td></tr>
<tr><td><a name="ref364"></a>364</td><td>Janzer, J. J., H. Stan-Lotter, and K. E. Sanderson <i>1981</i>. Isolation and characterization of hemin-permeable envelope-defective mutants of <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation and characterization of hemin-permeable envelope-defective mutants of Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref365"></a>365</td><td>Jegede, V. A., F. Spencer, and J. E. Brenchley <i>1976</i>. Thialysine-resistant mutant of <i>Salmonella</i> <i>typhimurium</i> with a lesion in the <i>thrA</i> gene. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Thialysine-resistant mutant of Salmonella typhimurium with a lesion in the thrA gene">google scholar</a></td></tr>
<tr><td><a name="ref366"></a>366</td><td>Jenness, D. D., and H. K. Schachman <i>1980</i>. <i>pyrB</i> mutations as suppressors of arginine auxotrophy in <i>Salmonella</i> <i>typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=pyrB mutations as suppressors of arginine auxotrophy in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref367"></a>367</td><td>Jensen, K. F <i>1979</i>. Apparent involvement of purines in the control of expression of <i>Salmonella</i> <i>typhimurium pyr</i> genes: analysis of a leaky <i>guaB</i> mutant resistant to pyrimidine analogs. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Apparent involvement of purines in the control of expression of Salmonella typhimurium pyr genes: analysis of a leaky guaB mutant resistant to pyrimidine analogs">google scholar</a></td></tr>
<tr><td><a name="ref368"></a>368</td><td>Jensen, K. F., J. Neuhard, and L. Shack <i>1982</i>. RNA polymerase involvement in the control of <i>Salmonella typhimurium pyr</i> gene expression. Isolation and characteristics of a fluorouracil resistant mutant with high, constitutive expression of <i>pyrB</i> and <i>pyrE</i> due to a mutation in <i>rpoBC.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=RNA polymerase involvement in the control of Salmonella typhimurium pyr gene expression. Isolation and characteristics of a fluorouracil resistant mutant with high, constitutive expression of pyrB and pyrE due to a mutation in rpoBC.">google scholar</a></td></tr>
<tr><td><a name="ref369"></a>369</td><td>Jin, D. J., and C. A. Gross <i>1988</i>. Mapping and sequencing of mutations in the <i>Escherichia coli</i> <i>rpoB</i> gene that lead to rifampicin resistance. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mapping and sequencing of mutations in the Escherichia coli rpoB gene that lead to rifampicin resistance">google scholar</a></td></tr>
<tr><td><a name="ref370"></a>370</td><td>Jochimsen, B., P. Nygaard, and T. Vestergaard <i>1975</i>. Location on the chromosome of <i>Escherichia</i> coli of genes governing purine metabolism: adenosine deaminase <i>(add),</i> guanosine kinase <i>(gsk)</i> and hypoxanthine phosphoribosyltransferase <i>(hpt).</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Location on the chromosome of Escherichia coli of genes governing purine metabolism: adenosine deaminase (add), guanosine kinase (gsk) and hypoxanthine phosphoribosyltransferase (hpt).">google scholar</a></td></tr>
<tr><td><a name="ref371"></a>371</td><td>Jochimsen, B. U., B. Hove-Jensen, B. B. Carber, and J. S. Gots <i>1985</i>. Characteristics of a <i>Salmonella typhimurium</i> mutant defective in phosphoribosylpyrophosphate synthetase. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Characteristics of a Salmonella typhimurium mutant defective in phosphoribosylpyrophosphate synthetase">google scholar</a></td></tr>
<tr><td><a name="ref372"></a>372</td><td>Johansson, V., A. Aarti, M. Nurminen, and P. H. Makela <i>1978</i>. Outer membrane protein- specific bacteriophages of <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Outer membrane protein- specific bacteriophages of Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref373"></a>373</td><td>Johnson, B. F <i>1977</i>. Fine structure mapping and properties of mutations suppressing the <i>lon</i> mutation in <i>Escherichia coli</i> K-12 and B strains. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Fine structure mapping and properties of mutations suppressing the lon mutation in Escherichia coli K-12 and B strains">google scholar</a></td></tr>
<tr><td><a name="ref374"></a>374</td><td>Johnston, H. M., and J. R. Roth <i>1979</i>. Histidine mutants requiring adenine: selection of mutants with reduced <i>hisG</i> expression in <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Histidine mutants requiring adenine: selection of mutants with reduced hisG expression in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref375"></a>375</td><td>Johnston, H. M., and J. R. Roth <i>1981</i>. Genetic analysis of the histidine operon control region of <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic analysis of the histidine operon control region of Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref376"></a>376</td><td>Johnston, M. A., and H. Pivnick <i>1970</i>. Use of autocytotoxic β-D-galactosides for selective growth of <i>Salmonella typhimurium</i> in the presence of coliforms. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Use of autocytotoxic β-D-galactosides for selective growth of Salmonella typhimurium in the presence of coliforms">google scholar</a></td></tr>
<tr><td><a name="ref377"></a>377</td><td>Jones-Mortimer, M., and H. L. Kornberg <i>1974</i>. Genetical analysis of fructose utilization by <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetical analysis of fructose utilization by Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref378"></a>378</td><td>Jones-Mortimer, M. C., and H. L. Kornberg <i>1976</i>. Uptake of fructose by the sorbitol phosphotransferase of <i>Escherichia coli</i> K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Uptake of fructose by the sorbitol phosphotransferase of Escherichia coli K12">google scholar</a></td></tr>
<tr><td><a name="ref379"></a>379</td><td>Jones-Mortimer, M. C., and H. L Kornberg <i>1980</i>. Amino-sugar transport systems of <i>Escherichia coli</i> K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Amino-sugar transport systems of Escherichia coli K12">google scholar</a></td></tr>
<tr><td><a name="ref380"></a>380</td><td>Josephsen, J., K. Hammer-Jespersen, and T. D. Hansen <i>1983</i>. Mapping of the gene for cytidine deaminase <i>(cdd)</i> in <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mapping of the gene for cytidine deaminase (cdd) in Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref381"></a>381</td><td>Joshi, A., J. Z. Siddiqui, M. Verma, and M. Chakravorty <i>1982</i>. Participation of host proteins in the morphogenesis of bacteriophage P22. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Participation of host proteins in the morphogenesis of bacteriophage P22">google scholar</a></td></tr>
<tr><td><a name="ref382"></a>382</td><td>Joshi, A., M. Verma, and M. Chakravorty <i>1982</i>. Thiolutin-resistant mutants of <i>Salmonella</i> <i>typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Thiolutin-resistant mutants of Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref383"></a>383</td><td>Juhl, M. J., and D. P. Clark <i>1990</i>. Thiophene-degrading <i>Escherichia coli</i> mutants possess sulfone oxidase activity and show altered resistance to sulfur-containing antibiotics. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Thiophene-degrading Escherichia coli mutants possess sulfone oxidase activity and show altered resistance to sulfur-containing antibiotics">google scholar</a></td></tr>
<tr><td><a name="ref384"></a>384</td><td>Jung, J. U., C. Gutierrez, and M. R. Villarejo <i>1989</i>. Sequence of an osmotically inducible lipoprotein gene. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Sequence of an osmotically inducible lipoprotein gene">google scholar</a></td></tr>
<tr><td><a name="ref385"></a>385</td><td>Justesen, J., and J. Neuhard <i>1975</i>. <i>pyrR</i> identical to <i>pyrH</i> in <i>Salmonella typhimurium:</i> control of expression of the <i>pyr</i> genes. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=pyrR identical to pyrH in Salmonella typhimurium: control of expression of the pyr genes">google scholar</a></td></tr>
<tr><td><a name="ref386"></a>386</td><td>Kadner, R. J <i>1977</i>. Transport and utilization of D-methionine and other methionine sources in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Transport and utilization of D-methionine and other methionine sources in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref387"></a>387</td><td>Kadner, R. J., K. Heller, J. W. Coulton, and V. Braun <i>1980</i>. Genetic control of hydroxamate- mediated iron uptake in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic control of hydroxamate- mediated iron uptake in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref388"></a>388</td><td>Kadner, R. J., and C. L Liggins <i>1973</i>. Transport of vitamin B12 in <i>Escherichia coli:</i> genetic studies. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Transport of vitamin B12 in Escherichia coli: genetic studies">google scholar</a></td></tr>
<tr><td><a name="ref389"></a>389</td><td>Kadner, R. J., and D. M. Shattuck-Eidens <i>1983</i>. Genetic control of the hexose phosphate transport system of <i>Escherichia coli:</i> mapping of deletion and insertion mutations in the <i>uhp</i> region. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic control of the hexose phosphate transport system of Escherichia coli: mapping of deletion and insertion mutations in the uhp region">google scholar</a></td></tr>
<tr><td><a name="ref390"></a>390</td><td>Kalle, C. P., and J. S. Gots <i>1961</i>. Alterations in purine nucleotide pyrophosphorylases and resistance to purine analogues. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Alterations in purine nucleotide pyrophosphorylases and resistance to purine analogues">google scholar</a></td></tr>
<tr><td><a name="ref391"></a>391</td><td>Kalman, M., D. R. Gentry, and M. Cashel <i>1991</i>. Characterization of the <i>Escherichia coli</i> K12 <i>gltS</i> glutamate permease gene. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Characterization of the Escherichia coli K12 gltS glutamate permease gene">google scholar</a></td></tr>
<tr><td><a name="ref392"></a>392</td><td>Kanner, B. I., and D. L. Gutnick <i>1972</i>. Use of neomycin in the isolation of mutants blocked in energy conservation in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Use of neomycin in the isolation of mutants blocked in energy conservation in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref393"></a>393</td><td>Kaplan, L., H. C. Reilly, and C. C. Stock <i>1959</i>. Action of azaserine on <i>Escherichia coli.</i> <i>J.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Action of azaserine on Escherichia coli. J.">google scholar</a></td></tr>
<tr><td><a name="ref394"></a>394</td><td>Kaplan, S., and D. Anderson <i>1968</i>. Selection of temperature-sensitive activating enzyme mutants in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Selection of temperature-sensitive activating enzyme mutants in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref395"></a>395</td><td>Kast, P., and H. Hennecke <i>1991</i>. Amino acid substrate specificity of <i>Escherichia coli</i> phenylalanyl-tRNA synthetase altered by distinct mutations. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Amino acid substrate specificity of Escherichia coli phenylalanyl-tRNA synthetase altered by distinct mutations">google scholar</a></td></tr>
<tr><td><a name="ref396"></a>396</td><td>Katz, L <i>1970</i>. Selection of <i>AraB</i> and <i>AraC</i> mutants of <i>Escherichia coli</i> B/r by resistance to ribitol. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Selection of AraB and AraC mutants of Escherichia coli B/r by resistance to ribitol">google scholar</a></td></tr>
<tr><td><a name="ref397"></a>397</td><td>Kawakami, T., Y. Akizawa, T. Ishikawa, T. Shimamoto, M. Tsuda, and T. Tsuchiya <i>1988</i>. Amino acid substitutions and alteration in cation specificity in the melibiose carrier of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Amino acid substitutions and alteration in cation specificity in the melibiose carrier of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref398"></a>398</td><td>Kawamoto, S., S. Tokuyama, S. Yashima, and Y. Eguchi <i>1984</i>. Genetic mapping of cold resistance gene of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic mapping of cold resistance gene of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref399"></a>399</td><td>Kawasaki, T., and Y. Nose <i>1969</i>. Thiamine regulatory mutants in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Thiamine regulatory mutants in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref400"></a>400</td><td>Kay, W. W <i>1972</i>. Genetic control of the metabolism of propionate by <i>Escherichia</i> <i>coli</i> K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic control of the metabolism of propionate by Escherichia coli K12">google scholar</a></td></tr>
<tr><td><a name="ref401"></a>401</td><td>Kay, W. W., and H. L. Kornberg <i>1969</i>. Genetic control of the uptake of Ci-dicarboxylic acids by <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic control of the uptake of Ci-dicarboxylic acids by Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref402"></a>402</td><td>Kelker, N. E., and W. K. Maas <i>1974</i>. Selection for genetically repressible (Arg R+) strains of <i>Escherichia coli</i> K12 from genetically derepressed (Arg R-) mutants using acetylnorvaline. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Selection for genetically repressible (Arg R+) strains of Escherichia coli K12 from genetically derepressed (Arg R-) mutants using acetylnorvaline">google scholar</a></td></tr>
<tr><td><a name="ref403"></a>403</td><td>Kelln, R. A., and G. A. O’Donovan <i>1976</i>. Isolation and partial characteristics of an <i>argR</i> mutant of <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation and partial characteristics of an argR mutant of Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref404"></a>404</td><td>Kelln, R. A., and V. L. Zak <i>1980</i>. A mutation in <i>Salmonella typhimurium</i> imparting conditional resistance to 5-fluorouracil and a bioenergetic defect mapping of <i>cad.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A mutation in Salmonella typhimurium imparting conditional resistance to 5-fluorouracil and a bioenergetic defect mapping of cad.">google scholar</a></td></tr>
<tr><td><a name="ref405"></a>405</td><td>Kemper, J <i>1974</i>. Gene order and co-transduction in the <i>leu-ara-fol-pyrA</i> region of the <i>Salmonella</i> <i>typhimurium</i> linkage map. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Gene order and co-transduction in the leu-ara-fol-pyrA region of the Salmonella typhimurium linkage map">google scholar</a></td></tr>
<tr><td><a name="ref406"></a>406</td><td>Kessler, D. P., and H. V. Rickenberg <i>1964</i>. A new method for the selection of mutants of <i>Escherichia coli</i> forming β-galactosidase constitutively. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A new method for the selection of mutants of Escherichia coli forming β-galactosidase constitutively">google scholar</a></td></tr>
<tr><td><a name="ref407"></a>407</td><td>Kier, L. D., R. M. Weppelman, and B. N. Ames <i>1977</i>. Regulation of two phosphatases and a cyclic phosphodiesterase of <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation of two phosphatases and a cyclic phosphodiesterase of Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref408"></a>408</td><td>Kier, L. D., R. M. Weppelman, and B. N. Ames <i>1979</i>. Regulation of nonspecific acid phosphatase in <i>Salmonella: phoN</i> and <i>phoP</i> genes. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation of nonspecific acid phosphatase in Salmonella: phoN and phoP genes">google scholar</a></td></tr>
<tr><td><a name="ref409"></a>409</td><td>Killmann, H., and V. Braun <i>1992</i>. An aspartate deletion mutation defines a binding site of the multifunctional <i>fhuA</i> outer membrane receptor of <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=An aspartate deletion mutation defines a binding site of the multifunctional fhuA outer membrane receptor of Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref410"></a>410</td><td>Kim, K. T., and O. J. Yoo <i>1987</i>. Inhibition of coliphage N4 infection of <i>Escherichia coli</i> mutant defective in mannose permease. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Inhibition of coliphage N4 infection of Escherichia coli mutant defective in mannose permease">google scholar</a></td></tr>
<tr><td><a name="ref411"></a>411</td><td>Kleckner, N., J. Roth, and D. Botstein <i>1977</i>. Genetic engineering <i>in vivo</i> using translocatable drug-resistance elements. New methods in bacterial genetics. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic engineering in vivo using translocatable drug-resistance elements. New methods in bacterial genetics">google scholar</a></td></tr>
<tr><td><a name="ref412"></a>412</td><td>Kline, E. L <i>1972</i>. New amino acid regulatory locus having unusual properties in heterozygous merodiploids. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=New amino acid regulatory locus having unusual properties in heterozygous merodiploids">google scholar</a></td></tr>
<tr><td><a name="ref413"></a>413</td><td>Kocharyan, S. M <i>1979</i>. Positive method for selection of adenylate cyclase and cyclic adenosine 3′, 5′-monophosphate receptor protein deficient mutants of <i>Escherichia coli</i> K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Positive method for selection of adenylate cyclase and cyclic adenosine 3′, 5′-monophosphate receptor protein deficient mutants of Escherichia coli K12">google scholar</a></td></tr>
<tr><td><a name="ref414"></a>414</td><td>Kohara, Y., K. Akiyama, and K. Isono <i>1987</i>. The physical map of the whole <i>E. coli</i> chromosome: application of a new strategy for the rapid analysis and sorting of a large genomic library. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The physical map of the whole E. coli chromosome: application of a new strategy for the rapid analysis and sorting of a large genomic library">google scholar</a></td></tr>
<tr><td><a name="ref415"></a>415</td><td>Komatsu, Y., and K. Tanaka <i>1972</i>. A showdomycin-resistant mutant of <i>Escherichia coli</i> K-12 with altered nucleoside transport character. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A showdomycin-resistant mutant of Escherichia coli K-12 with altered nucleoside transport character">google scholar</a></td></tr>
<tr><td><a name="ref416"></a>416</td><td>Komeda, Y <i>1982</i>. Fusions of flagellar operons to lactose genes on a Mu <i>lac</i> bacteriophage. <i>J.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Fusions of flagellar operons to lactose genes on a Mu lac bacteriophage. J.">google scholar</a></td></tr>
<tr><td><a name="ref417"></a>417</td><td>Komeda, Y., M. Silverman, and M. Simon <i>1977</i>. Genetic analysis of <i>Escherichia coli</i> K-12 region I flagellar mutants. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic analysis of Escherichia coli K-12 region I flagellar mutants">google scholar</a></td></tr>
<tr><td><a name="ref418"></a>418</td><td>Kornberg, H. L., and J. Smith <i>1972</i>. Genetic control of glucose uptake by <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic control of glucose uptake by Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref419"></a>419</td><td>Kornberg, H. L., and P. D. Watts <i>1978</i>. Roles of <i>crr</i>-gene products in regulating carbohydrate uptake by <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Roles of crr-gene products in regulating carbohydrate uptake by Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref420"></a>420</td><td>Korteland, J., N. Overbeeke, P. De Graaff, P. Overduin, and B. Lugtenberg <i>1985</i>. Role of the Arg158 residue of the outer membrane <i>phoE</i> pore protein of <i>Escherichia coli</i> K12 in bacteriophage TC45 recognition and in channel characteristics. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Role of the Arg158 residue of the outer membrane phoE pore protein of Escherichia coli K12 in bacteriophage TC45 recognition and in channel characteristics">google scholar</a></td></tr>
<tr><td><a name="ref421"></a>421</td><td>Kowit, J. D., W.-N. Choy, S. P. Champe, and A. L. Goldberg <i>1976</i>. Role and location of “protease 1” from <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Role and location of “protease 1” from Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref422"></a>422</td><td>Krajewska-Grynkiewicz, K., W. Walczak, and T. Klopotowski <i>1971</i>. Mutants of <i>Salmonella</i> <i>typhimurium</i> able to utilize D-histidine as a source of L-histidine. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutants of Salmonella typhimurium able to utilize D-histidine as a source of L-histidine">google scholar</a></td></tr>
<tr><td><a name="ref423"></a>423</td><td>Kramer, G. F., and B. N. Ames <i>1988</i>. Mechanisms of mutagenicity and toxicity of sodium selenite in <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mechanisms of mutagenicity and toxicity of sodium selenite in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref424"></a>424</td><td>Kramer, G. F., J. C. Baker, and B. N. Ames <i>1988</i>. Near-UV stress in <i>Salmonella typhimurium:</i> 4-thiouridine in tRNA, ppGpp, and ApppGpp as components of an adaptive response. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Near-UV stress in Salmonella typhimurium: 4-thiouridine in tRNA, ppGpp, and ApppGpp as components of an adaptive response">google scholar</a></td></tr>
<tr><td><a name="ref425"></a>425</td><td>Kraus, J., D. Söll, and K. B. Low <i>1979</i>. Glutamyl-γ-methyl ester acts as a methionine analogue in <i>Escherichia coli:</i> analogue resistant mutants map at the <i>metJ</i> and <i>metK</i> loci. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Glutamyl-γ-methyl ester acts as a methionine analogue in Escherichia coli: analogue resistant mutants map at the metJ and metK loci">google scholar</a></td></tr>
<tr><td><a name="ref426"></a>426</td><td>Kricker, M., and B. G. Hall <i>1984</i>. Directed evolution of cellobiose utilization in <i>Escherichia coli</i> K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Directed evolution of cellobiose utilization in Escherichia coli K12">google scholar</a></td></tr>
<tr><td><a name="ref427"></a>427</td><td>Kuhn, A. H. U., M. L. J. Moncany, E. Kellenberger, and R. Hausmann <i>1982</i>. Involvement of the bacterial <i>groM</i> gene product in bacteriophage T7 reproduction. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Involvement of the bacterial groM gene product in bacteriophage T7 reproduction">google scholar</a></td></tr>
<tr><td><a name="ref428"></a>428</td><td>Kuhn, J., and R. L. Somerville <i>1971</i>. Mutant strains of <i>Escherichia coli</i> K12 that can use D- amino acids. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutant strains of Escherichia coli K12 that can use D- amino acids">google scholar</a></td></tr>
<tr><td><a name="ref429"></a>429</td><td>Kumar, S <i>1976</i>. Properties of adenyl cyclase and cyclic adenosine 3′,5′-monophosphate receptor protein-deficient mutants of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Properties of adenyl cyclase and cyclic adenosine 3′,5′-monophosphate receptor protein-deficient mutants of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref430"></a>430</td><td>Kunert, K. J., C. F. Cresswell, A. Schmidt, P. M. Mullineaux, and C. H. Foyer <i>1990</i>. Variations in the activity of glutathione reductase and the cellular glutathione content in relation to sensitivity to methyl viologen in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Variations in the activity of glutathione reductase and the cellular glutathione content in relation to sensitivity to methyl viologen in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref431"></a>431</td><td>Kuroda, M., S. De Waard, K. Mizushima, M. Tsuda, P. Postma, and T. Tsuchiya <i>1992</i>. Resistance of the melibiose carrier to inhibition by the phosphotransferase system due to substitutions of amino acid residues in the carrier of <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Resistance of the melibiose carrier to inhibition by the phosphotransferase system due to substitutions of amino acid residues in the carrier of Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref432"></a>432</td><td>Kustu, S. G., and G. F.-L. Ames <i>1973</i>. The <i>hisP</i> protein, a known histidine transport component in <i>Salmonella typhimurium,</i> is also an arginine transport component. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The hisP protein, a known histidine transport component in Salmonella typhimurium, is also an arginine transport component">google scholar</a></td></tr>
<tr><td><a name="ref433"></a>433</td><td>Kustu, S. G., N. C. McFarland, S. P. Hui, B. Esmon, and G. F.-L. Ames <i>1979</i>. Nitrogen control in <i>Salmonella typhimurium:</i> coregulation of synthesis of glutamine synthetase and amino acid transport systems. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Nitrogen control in Salmonella typhimurium: coregulation of synthesis of glutamine synthetase and amino acid transport systems">google scholar</a></td></tr>
<tr><td><a name="ref434"></a>434</td><td>Kuwano, M., D. Schlessinger, G. Rinaldi, L. Felicetti, and G. P. Tocchini-Valentini <i>1971</i>. G factor mutants of <i>Escherichia coli:</i> map location and properties. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=G factor mutants of Escherichia coli: map location and properties">google scholar</a></td></tr>
<tr><td><a name="ref435"></a>435</td><td>Lakshmi, T. M., and R. B. Helling <i>1976</i>. Selection for citrate synthease deficiency in <i>icd</i> mutants of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Selection for citrate synthease deficiency in icd mutants of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref436"></a>436</td><td>Lambden, P. R., and J. R. Guest <i>1976</i>. A novel method for isolating chlorate-resistant mutants of <i>Escherichia coli</i> K12 by anaerobic selection on a lactate plus fumarate medium. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A novel method for isolating chlorate-resistant mutants of Escherichia coli K12 by anaerobic selection on a lactate plus fumarate medium">google scholar</a></td></tr>
<tr><td><a name="ref437"></a>437</td><td>Langley, D., and J. R. Guest <i>1974</i>. Biochemical and genetic characteristics of deletion and other mutant strains of <i>Salmonella typhimurium</i> LT2 lacking α-ketoacid dehydrogenase complex activities. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Biochemical and genetic characteristics of deletion and other mutant strains of Salmonella typhimurium LT2 lacking α-ketoacid dehydrogenase complex activities">google scholar</a></td></tr>
<tr><td><a name="ref438"></a>438</td><td>Langley, D., and J. R. Guest <i>1977</i>. Biochemical studies of the α-ketoacid dehydrogenase complexes of <i>Escherichia coli</i> K12: isolation and biochemical properties of deletion mutants. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Biochemical studies of the α-ketoacid dehydrogenase complexes of Escherichia coli K12: isolation and biochemical properties of deletion mutants">google scholar</a></td></tr>
<tr><td><a name="ref439"></a>439</td><td>Langley, D., and J. R. Guest <i>1978</i>. Biochemical genetics of the α-keto acid dehydrogenase complexes of <i>Escherichia coli</i> K12: genetic characterization and regulatory properties of deletion mutants. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Biochemical genetics of the α-keto acid dehydrogenase complexes of Escherichia coli K12: genetic characterization and regulatory properties of deletion mutants">google scholar</a></td></tr>
<tr><td><a name="ref440"></a>440</td><td>Langridge, J <i>1969</i>. Mutations conferring quantitative and qualitative increases in β-galactosidase activity in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutations conferring quantitative and qualitative increases in β-galactosidase activity in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref441"></a>441</td><td>LaRossa, R. A., and J. V. Schloss <i>1984</i>. The sulfonylurea herbicide sulfometuron methyl is an extremely potent and selective inhibitor of acetolactate synthase in <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The sulfonylurea herbicide sulfometuron methyl is an extremely potent and selective inhibitor of acetolactate synthase in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref442"></a>442</td><td>LaRossa, R. A., and D. R. Smulski <i>1984</i>. <i>ilvB</i>-encloded acetolactate synthase is resistant to the herbicide sulfometuron methyl. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=ilvB-encloded acetolactate synthase is resistant to the herbicide sulfometuron methyl">google scholar</a></td></tr>
<tr><td><a name="ref443"></a>443</td><td>LaRossa, R. A., T. K. Van Dyk, and D. R. Smulski <i>1987</i>. Toxic accumulation of α-ketobutyrate caused by inhibition of the branched-chain amino acid biosynthetic enzyme acetolactate synthase in <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Toxic accumulation of α-ketobutyrate caused by inhibition of the branched-chain amino acid biosynthetic enzyme acetolactate synthase in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref444"></a>444</td><td>LaRossa, R. A., T. K. Van Dyk, and D. R. Smulski <i>1990</i>. A need for metabolic insulation: lessons from sulfonylurea genetics.</td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A need for metabolic insulation: lessons from sulfonylurea genetics">google scholar</a></td></tr>
<tr><td><a name="ref445"></a>445</td><td>Latil, M., M. Murgier, A. Lazdunski, and C. Lazdunski <i>1976</i>. Isolation and genetic mapping of <i>Escherichia coli</i> aminopeptidase mutants. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation and genetic mapping of Escherichia coli aminopeptidase mutants">google scholar</a></td></tr>
<tr><td><a name="ref446"></a>446</td><td>Lavina, M., A. P. Pugsley, and F. Moreno <i>1986</i>. Identification, mapping, cloning, and characterization of a gene (<i>sbmA</i>) required for microcin B17 action on <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Identification, mapping, cloning, and characterization of a gene (sbmA) required for microcin B17 action on Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref447"></a>447</td><td>Lawrence, D. A., D. A. Smith, and R. J. Rowbury <i>1968</i>. Regulation of methionine synthesis in <i>Salmonella typhimurium:</i> mutants resistant to inhibition by analogues of methionine. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation of methionine synthesis in Salmonella typhimurium: mutants resistant to inhibition by analogues of methionine">google scholar</a></td></tr>
<tr><td><a name="ref448"></a>448</td><td>Lawther, R. P., D. H. Calhoun, C. W. Adams, C. A. Hauser, J. Gray, and G. W. Hatfield <i>1981</i>. Molecular basis of valine resistance in <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Molecular basis of valine resistance in Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref449"></a>449</td><td>Lazzaroni, J. C., and R. C. Portalier <i>1981</i>. Genetic and biochemical characterization of periplasmic-leaky mutants of <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic and biochemical characterization of periplasmic-leaky mutants of Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref450"></a>450</td><td>LeBlanc, D. J., and R. P. Mortlock <i>1971</i>. Metabolism of D-arabinose: a new pathway in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Metabolism of D-arabinose: a new pathway in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref451"></a>451</td><td>Lee, R. S.-F., J. Pagan, S. Wilke-Mounts, and A. E. Senior <i>1991</i>. Characterization of <i>Escherichia coli</i> ATP synthase beta-subunit mutations using a chromosomal deletion strain. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Characterization of Escherichia coli ATP synthase beta-subunit mutations using a chromosomal deletion strain">google scholar</a></td></tr>
<tr><td><a name="ref452"></a>452</td><td>Lehmann, V., E. Rupprecht, and M. J. Osborn <i>1977</i>. Isolation of mutants conditionally blocked in the biosynthesis of the 3-deoxy-D-manno-octulosonic acid-lipid A part of lipopolysaccharides derived from <i>Salmonella</i> <i>typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation of mutants conditionally blocked in the biosynthesis of the 3-deoxy-D-manno-octulosonic acid-lipid A part of lipopolysaccharides derived from Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref453"></a>453</td><td>Leifer, Z., R. Engel, and B. E. Tropp <i>1977</i>. Transport of 3,4-dihydroxybutyl-l-phosphonate, an analogue of <i>sn</i>-glycerol 3-phosphate. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Transport of 3,4-dihydroxybutyl-l-phosphonate, an analogue of sn-glycerol 3-phosphate">google scholar</a></td></tr>
<tr><td><a name="ref454"></a>454</td><td>Leisinger, T., D. Haas, and M. P. Hegarty <i>1972</i>. Indospicine as an arginine antagonist in <i>Escherichia coli</i> and <i>Pseudomonas aeruginosa.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Indospicine as an arginine antagonist in Escherichia coli and Pseudomonas aeruginosa.">google scholar</a></td></tr>
<tr><td><a name="ref455"></a>455</td><td>Lemmon, R. D., J. J. Rowe, and G. J. Tritz <i>1980</i>. Isolation and characterization of mutants of <i>Escherichia coli</i> defective in pyridine nucleotide cycle enyzmes. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation and characterization of mutants of Escherichia coli defective in pyridine nucleotide cycle enyzmes">google scholar</a></td></tr>
<tr><td><a name="ref456"></a>456</td><td>Lengeler, J <i>1980</i>. Characterization of mutants of <i>Escherichia coli</i> K12, selected by resistance to streptozotocin. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Characterization of mutants of Escherichia coli K12, selected by resistance to streptozotocin">google scholar</a></td></tr>
<tr><td><a name="ref457"></a>457</td><td>Lengeler, J., and H. Steinberger <i>1978</i>. Analysis of the regulatory mechanisms controlling the synthesis of the hexitol transport systems in <i>Escherichia coli</i> K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Analysis of the regulatory mechanisms controlling the synthesis of the hexitol transport systems in Escherichia coli K12">google scholar</a></td></tr>
<tr><td><a name="ref458"></a>458</td><td>Lerman, R. D., and B. A. D. Stocker <i>1981</i>. Mutations in <i>Salmonella typhimurium</i> affecting synthesis of lipopolysaccharide core at high temperature. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutations in Salmonella typhimurium affecting synthesis of lipopolysaccharide core at high temperature">google scholar</a></td></tr>
<tr><td><a name="ref459"></a>459</td><td>Lester, B., and D. M. Bonner <i>1957</i>. Genetic control of raffinose utilization in <i>Escherichia coli.</i> <i>J.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic control of raffinose utilization in Escherichia coli. J.">google scholar</a></td></tr>
<tr><td><a name="ref460"></a>460</td><td>Levin, D. E., L. J. Marnett, and B. N. Ames <i>1984</i>. Spontaneous and mutagen-induced deletions: mechanistic studies in <i>Salmonella</i> tester strain TA102. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Spontaneous and mutagen-induced deletions: mechanistic studies in Salmonella tester strain TA102">google scholar</a></td></tr>
<tr><td><a name="ref461"></a>461</td><td>Levine, R. A., and M. W. Taylor <i>1981</i>. Selection for purine regulatory mutants in an <i>E. coli</i> hypoxanthine phosphoribosyl transferase-guanine phosphoribosyl transferase double mutant. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Selection for purine regulatory mutants in an E. coli hypoxanthine phosphoribosyl transferase-guanine phosphoribosyl transferase double mutant">google scholar</a></td></tr>
<tr><td><a name="ref462"></a>462</td><td>Levine, R. A., and M. W. Taylor <i>1982</i>. Mechanism of adenine toxicity in <i>Escherichia coli.</i> <i>J.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mechanism of adenine toxicity in Escherichia coli. J.">google scholar</a></td></tr>
<tr><td><a name="ref463"></a>463</td><td>LeVine, S. M., F. Ardeshir, and G. F.-L. Ames <i>1980</i>. Isolation and characterization of acetate kinase and phosphotransacetylase mutants of <i>Escherichia coli</i> and <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation and characterization of acetate kinase and phosphotransacetylase mutants of Escherichia coli and Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref464"></a>464</td><td>Lieberman, M. A., and J.-S. Hong <i>1974</i>. A mutant of <i>Escherichia coli</i> defective in the coupling of metabolic energy to active transport. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A mutant of Escherichia coli defective in the coupling of metabolic energy to active transport">google scholar</a></td></tr>
<tr><td><a name="ref465"></a>465</td><td>Lieberman, M. A., M. Simon, and J.-S. Hong <i>1977</i>. Characterization of <i>Escherichia coli</i> mutant incapable of maintaining a transmembrane potential. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Characterization of Escherichia coli mutant incapable of maintaining a transmembrane potential">google scholar</a></td></tr>
<tr><td><a name="ref466"></a>466</td><td>Link, C. D., and A. M. Reiner <i>1983</i>. Genotypic exclusion: a novel relationship between the ribitol-arabitol and galactitol genes of <i>E. coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genotypic exclusion: a novel relationship between the ribitol-arabitol and galactitol genes of E. coli.">google scholar</a></td></tr>
<tr><td><a name="ref467"></a>467</td><td>Liou, T.-F., M. Yoshikawa, and N. Tanaka <i>1975</i>. Alteration of ribosomal protein L5 in a thiopeptin resistant mutant of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Alteration of ribosomal protein L5 in a thiopeptin resistant mutant of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref468"></a>468</td><td>Lipsett, M. N <i>1978</i>. Enzymes producing 4-thiouridine in <i>Escherichia coli</i> tRNA: approximate chromosomal locations of the genes and enzyme activities in a 4-thiouridine-deficient mutant. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Enzymes producing 4-thiouridine in Escherichia coli tRNA: approximate chromosomal locations of the genes and enzyme activities in a 4-thiouridine-deficient mutant">google scholar</a></td></tr>
<tr><td><a name="ref469"></a>469</td><td>Lisitsyn, N. A., E. D. Sverdlov, E. P. Moiseeva, and V. G. Nikiforov <i>1985</i>. Localization of a mutation leading to antibiotic streptolydigin resistance of <i>E. coli</i> RNA polymerase in the <i>rpoB</i> gene coding for beta-subunit of the enzyme. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Localization of a mutation leading to antibiotic streptolydigin resistance of E. coli RNA polymerase in the rpoB gene coding for beta-subunit of the enzyme">google scholar</a></td></tr>
<tr><td><a name="ref470"></a>470</td><td>Livshits, V. A <i>1976</i>. Effect of 2,6-diaminopurine resistant mutations on the uptake of adenine and adenosine by adenine-requiring strains of <i>Escherichia coli</i> K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Effect of 2,6-diaminopurine resistant mutations on the uptake of adenine and adenosine by adenine-requiring strains of Escherichia coli K12">google scholar</a></td></tr>
<tr><td><a name="ref471"></a>471</td><td>Lomax, M. S., and G. R. Greenberg <i>1968</i>. Characteristics of the <i>deo</i> operon: role in thymine utilization and sensitivity to deoxyribonucleosides. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Characteristics of the deo operon: role in thymine utilization and sensitivity to deoxyribonucleosides">google scholar</a></td></tr>
<tr><td><a name="ref472"></a>472</td><td>Lomovskaya, O., and K. Lewis <i>1992</i>. <i>emr,</i> an <i>Escherichia coli</i> locus for multidrug resistance. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=emr, an Escherichia coli locus for multidrug resistance">google scholar</a></td></tr>
<tr><td><a name="ref473"></a>473</td><td>Long, W. S., C. L. Slayman, and K. B. Low <i>1978</i>. Production of giant cells of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Production of giant cells of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref474"></a>474</td><td>Loomis, W. F., Jr., and B. Magasanik <i>1965</i>. Genetic control of catabolite repression of the <i>lac</i> operon in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic control of catabolite repression of the lac operon in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref475"></a>475</td><td>Loomis, W. F., Jr., and B. Magasanik <i>1967</i>. The catabolite repression gene of the <i>lac</i> operon in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The catabolite repression gene of the lac operon in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref476"></a>476</td><td>Lorowitz, W., and D. Clark <i>1982</i>. <i>Escherichia coli</i> mutants with a temperature-sensitive alcohol dehydrogenase. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Escherichia coli mutants with a temperature-sensitive alcohol dehydrogenase">google scholar</a></td></tr>
<tr><td><a name="ref477"></a>477</td><td>Low, K. B., F. Gates, T. Goldstein, and D. Söll <i>1971</i>. Isolation and partial characterization of temperature-sensitive <i>Escherichia coli</i> mutants with altered leucyl- and seryl-transfer ribonucleic acid synthetases. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation and partial characterization of temperature-sensitive Escherichia coli mutants with altered leucyl- and seryl-transfer ribonucleic acid synthetases">google scholar</a></td></tr>
<tr><td><a name="ref478"></a>478</td><td>Luckey, M., and J. B. Neilands <i>1976</i>. Iron transport in <i>Salmonella typhimurium</i> LT-2: prevention, by ferrichrome, of adsorption of bacteriophages ES18 and ES18.hl to a common cell envelope receptor. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Iron transport in Salmonella typhimurium LT-2: prevention, by ferrichrome, of adsorption of bacteriophages ES18 and ES18.hl to a common cell envelope receptor">google scholar</a></td></tr>
<tr><td><a name="ref479"></a>479</td><td>Luckey, M., J. R. Pollack, R. Wayne, B. N. Ames, and J. B. Neilands <i>1972</i>. Iron uptake in <i>Salmonella typhimurium:</i> utilization of exogenous siderochromes as iron carriers. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Iron uptake in Salmonella typhimurium: utilization of exogenous siderochromes as iron carriers">google scholar</a></td></tr>
<tr><td><a name="ref480"></a>480</td><td>Lutkenhaus, J. F <i>1977</i>. Role of a major outer membrane protein in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Role of a major outer membrane protein in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref481"></a>481</td><td>Maas, W. K <i>1952</i>. Pantothenate studies. II. Evidence from mutants for interference by salicylate with pantoate synthesis. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Pantothenate studies. II. Evidence from mutants for interference by salicylate with pantoate synthesis">google scholar</a></td></tr>
<tr><td><a name="ref482"></a>482</td><td>Maas, W. K <i>1961</i>. Repression of arginine formation. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Repression of arginine formation">google scholar</a></td></tr>
<tr><td><a name="ref483"></a>483</td><td>Maas, W. K <i>1965</i>. Genetic defect affecting an arginine permease and repression of arginine synthesis in <i>E. coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic defect affecting an arginine permease and repression of arginine synthesis in E. coli.">google scholar</a></td></tr>
<tr><td><a name="ref484"></a>484</td><td>MacPhee, D. G., V. Krishnapillai, R. J. Roantree, and B. A. D. Stocker <i>1975</i>. Mutations in <i>Salmonella typhimurium</i> conferring resistance to Felix O phage without loss of smooth character. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutations in Salmonella typhimurium conferring resistance to Felix O phage without loss of smooth character">google scholar</a></td></tr>
<tr><td><a name="ref485"></a>485</td><td>Makela, P. H., M. Sarvas, S. Calcagno, and K. Lounatmaa <i>1978</i>. Isolation and genetic characterization of polymyxin resistant mutants of <i>Salmonella.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation and genetic characterization of polymyxin resistant mutants of Salmonella.">google scholar</a></td></tr>
<tr><td><a name="ref486"></a>486</td><td>Makela, P. H., and B. A. D. Stocker <i>1969</i>. Genetics of polysaccharide biosynthesis. <i>Annu. Rev.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetics of polysaccharide biosynthesis. Annu. Rev.">google scholar</a></td></tr>
<tr><td><a name="ref487"></a>487</td><td>Maloy, S. R., and W. D. Nunn <i>1981</i>. Selection for loss of tetracycline resistance by <i>Escherichia</i> <i>coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Selection for loss of tetracycline resistance by Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref488"></a>488</td><td>Maloy, S. R., and W. D. Nunn <i>1982</i>. Genetic regulation of the glyoxylate shunt in <i>Escherichia</i> <i>coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic regulation of the glyoxylate shunt in Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref489"></a>489</td><td>Manning, P. A., A. Puspurs, and P. Reeves <i>1976</i>. Outer membrane of <i>Escherichia coli</i> K12: isolation of mutants with altered protein 3A by using host range mutants of bacteriophage K3. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Outer membrane of Escherichia coli K12: isolation of mutants with altered protein 3A by using host range mutants of bacteriophage K3">google scholar</a></td></tr>
<tr><td><a name="ref490"></a>490</td><td>Manning, P. A., and P. Reeves <i>1976</i>. Outer membrane of <i>Escherichia coli</i> K12: <i>tsx</i> mutants (resistant to bacteriophage T6 and colicin K) lack an outer membrane protein. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Outer membrane of Escherichia coli K12: tsx mutants (resistant to bacteriophage T6 and colicin K) lack an outer membrane protein">google scholar</a></td></tr>
<tr><td><a name="ref491"></a>491</td><td>Manning, P. A., and P. Reeves <i>1976</i>. Outer membrane of <i>Escherichia coli</i> K12: differentiation of proteins 3A and 3B on acrylamide gels and further characterization of <i>con</i> <i>tolG</i> mutants. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Outer membrane of Escherichia coli K12: differentiation of proteins 3A and 3B on acrylamide gels and further characterization of con tolG mutants">google scholar</a></td></tr>
<tr><td><a name="ref492"></a>492</td><td>Manning, P. A., and P. Reeves <i>1978</i>. Outer membrane proteins of <i>Escherichia coli</i> K12. Isolation of a common receptor protein for bacteriophage T-6 and colicin K. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Outer membrane proteins of Escherichia coli K12. Isolation of a common receptor protein for bacteriophage T-6 and colicin K">google scholar</a></td></tr>
<tr><td><a name="ref493"></a>493</td><td>Manoil, C., and J. P. Rosenbusch <i>1982</i>. Conjugation deficient mutants of <i>Escherichia coli</i> distinguish classes of functions of the outer membrane <i>ompA</i> protein. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Conjugation deficient mutants of Escherichia coli distinguish classes of functions of the outer membrane ompA protein">google scholar</a></td></tr>
<tr><td><a name="ref494"></a>494</td><td>Mao, Y., and Z. Sheng <i>1986</i>. A phenylethyl alcohol resistant temperature sensitive <i>dnaB</i> mutant in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A phenylethyl alcohol resistant temperature sensitive dnaB mutant in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref495"></a>495</td><td>Marakusha, B. I., V. G. Petrovskaya, and K. I. Volkovoi <i>1986</i>. Obtaining <i>Salmonella</i> <i>typhimurium</i> and <i>Salmonella</i> <i>dublin</i> neamine resistant mutants suitable for the development of vaccines and the characterization of their biological and genetic properties. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Obtaining Salmonella typhimurium and Salmonella dublin neamine resistant mutants suitable for the development of vaccines and the characterization of their biological and genetic properties">google scholar</a></td></tr>
<tr><td><a name="ref496"></a>496</td><td>Marcus, M., and Y. S. Halpern <i>1967</i>. Genetic analysis of glutamate transport in <i>Escherichia</i> <i>coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic analysis of glutamate transport in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref497"></a>497</td><td>Marcus, M., and Y. S. Halpern <i>1969</i>. Genetic analysis of the glutamate permease in <i>Escherichia</i> <i>coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic analysis of the glutamate permease in Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref498"></a>498</td><td>Mark, D. F., J. W. Chang, and C. F. Richardson <i>1977</i>. Genetic mapping of <i>trxA,</i> a gene affecting thioredoxin in <i>Escherichia coli</i> K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic mapping of trxA, a gene affecting thioredoxin in Escherichia coli K12">google scholar</a></td></tr>
<tr><td><a name="ref499"></a>499</td><td>Marquardt, J. L., D. A. Siegele, R. Kolter, and C. T. Walsh <i>1992</i>. Cloning and sequencing of <i>Escherichia coli</i> <i>murZ</i> and purification of its product, a UDP-<i>N</i>-acetylglucosamine enolpyruvyl transferase. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Cloning and sequencing of Escherichia coli murZ and purification of its product, a UDP-N-acetylglucosamine enolpyruvyl transferase">google scholar</a></td></tr>
<tr><td><a name="ref500"></a>500</td><td>Martinez, D., and F. Whitehouse, Jr <i>1973</i>. Selective autocytotoxicity in a model system of <i>Escherichia coli</i> recombinants. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Selective autocytotoxicity in a model system of Escherichia coli recombinants">google scholar</a></td></tr>
<tr><td><a name="ref501"></a>501</td><td>Masters, P. S., and J.-S. Hong <i>1981</i>. Genetics of the glutamine transport system in <i>Escherichia</i> <i>coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetics of the glutamine transport system in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref502"></a>502</td><td>Matsuzawa, H., S. Asoh, T. Ohta, S. Tamaki, and M. Matsuhashi <i>1980</i>. Further studies on <i>rodA</i> mutant: a round morphological mutant of <i>Escherichia coli</i> K12 with wild-type penicillin-binding protein 2. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Further studies on rodA mutant: a round morphological mutant of Escherichia coli K12 with wild-type penicillin-binding protein 2">google scholar</a></td></tr>
<tr><td><a name="ref503"></a>503</td><td>Matsuzawa, H., S. Ushiyama, Y. Koyama, and T. Ohta <i>1984</i>. <i>Escherichia coli</i> K-12 <i>tolZ</i> mutants tolerant to colicins E2, E3, D, Ia and Ib: defect in generation of the electrochemical proton gradient. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Escherichia coli K-12 tolZ mutants tolerant to colicins E2, E3, D, Ia and Ib: defect in generation of the electrochemical proton gradient">google scholar</a></td></tr>
<tr><td><a name="ref504"></a>504</td><td>Mattern, I. E., and J. Pittard <i>1971</i>. Regulation of tyrosine biosynthesis in <i>Escherichia coli</i> K- 12: isolation and characterization of operator mutants. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation of tyrosine biosynthesis in Escherichia coli K- 12: isolation and characterization of operator mutants">google scholar</a></td></tr>
<tr><td><a name="ref505"></a>505</td><td>McCalla, D. R., C. Kaiser, and M. H. L. Green <i>1978</i>. Genetics of nitrofurazone resistance in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetics of nitrofurazone resistance in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref506"></a>506</td><td>McConville, M. L., and H. P. Charles <i>1979</i>. Mutants of <i>Escherichia coli</i> K12 accumulating porphobilinogen: a new locus, <i>hemC.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutants of Escherichia coli K12 accumulating porphobilinogen: a new locus, hemC.">google scholar</a></td></tr>
<tr><td><a name="ref507"></a>507</td><td>McConville, M. L., and H. P. Charles <i>1979</i>. Isolation of haemin-requiring mutants of <i>Escherichia coli</i> K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation of haemin-requiring mutants of Escherichia coli K12">google scholar</a></td></tr>
<tr><td><a name="ref508"></a>508</td><td>McEwen, J., and P. Silverman <i>1980</i>. Chromosomal mutations of <i>Escherichia coli</i> that alter expression of conjugative plasmid functions. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Chromosomal mutations of Escherichia coli that alter expression of conjugative plasmid functions">google scholar</a></td></tr>
<tr><td><a name="ref509"></a>509</td><td>Meiss, H. K., W. J. Brill, and B. Magasanik <i>1969</i>. Genetic control of histidine degradation in <i>Salmonella typhimurium</i> LT-2. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic control of histidine degradation in Salmonella typhimurium LT-2">google scholar</a></td></tr>
<tr><td><a name="ref510"></a>510</td><td>Melton, T., W. Kundig, P. E. Hartman, and N. Meadow <i>1976</i>. 3-Deoxy-3-fluoro-D-glucose- resistant <i>Salmonella typhimurium</i> mutants defective in the phosphoenolpyruvate:glycose phosphotransferase system. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=3-Deoxy-3-fluoro-D-glucose- resistant Salmonella typhimurium mutants defective in the phosphoenolpyruvate:glycose phosphotransferase system">google scholar</a></td></tr>
<tr><td><a name="ref511"></a>511</td><td>Menzel, R., and J. Roth <i>1981</i>. Regulation of the genes for proline utilization in <i>Salmonella</i> <i>typhimurium:</i> autogenous repression by the <i>putA</i> gene product. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation of the genes for proline utilization in Salmonella typhimurium: autogenous repression by the putA gene product">google scholar</a></td></tr>
<tr><td><a name="ref512"></a>512</td><td>Mett, V. L., F. I. Urmeeva, N. S. Kobets, T. V. Kolganova, K. A. Aliev, and E. S. Piruzyan <i>1991</i>. Cloning and expression of a mutant EPSP-synthase gene of <i>E. coli</i> in transgenic plants. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Cloning and expression of a mutant EPSP-synthase gene of E. coli in transgenic plants">google scholar</a></td></tr>
<tr><td><a name="ref513"></a>513</td><td>Meynell, E. W <i>1961</i>. A phage χ, which attacks motile bacteria. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A phage χ, which attacks motile bacteria">google scholar</a></td></tr>
<tr><td><a name="ref514"></a>514</td><td>Mikulka, T. W., B. I. Stieglitz, and J. M. Calvo <i>1972</i>. Leucyltransfer-ribonucleic acid synthetase from a wild-type and temperature-sensitive mutant of <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Leucyltransfer-ribonucleic acid synthetase from a wild-type and temperature-sensitive mutant of Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref515"></a>515</td><td>Mildener, B., T. P. Fondy, R. Engel, and B. E. Tropp <i>1981</i>. Effects of halo analogs of glycerol 3-phosphate and dihydroxyacetone phosphate upon <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Effects of halo analogs of glycerol 3-phosphate and dihydroxyacetone phosphate upon Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref516"></a>516</td><td>Miller, C. G., C. Heiman, and C. Yen <i>1976</i>. Mutants of <i>Salmonella typhimurium</i> deficient in an endopeptidase. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutants of Salmonella typhimurium deficient in an endopeptidase">google scholar</a></td></tr>
<tr><td><a name="ref517"></a>517</td><td>Miller, C. G., and G. Schwartz <i>1978</i>. Peptidase-deficient mutants of <i>Escherichia coli.</i> <i>J.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Peptidase-deficient mutants of Escherichia coli. J.">google scholar</a></td></tr>
<tr><td><a name="ref518"></a>518</td><td>Miller, E. S., and J. E. Brenchley <i>1981</i>. L-methionine SR-sulfoximine resistant glutamine synthetase from mutants of <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=L-methionine SR-sulfoximine resistant glutamine synthetase from mutants of Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref519"></a>519</td><td>Miller, H. I., and D. I. Friedman <i>1977</i>. Isolation of <i>Escherichia coli</i> mutants unable to support lambda integrative recombination.</td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation of Escherichia coli mutants unable to support lambda integrative recombination">google scholar</a></td></tr>
<tr><td><a name="ref520"></a>520</td><td>Miller, H. I., and D. I. Friedman <i>1980</i>. An <i>E. coli</i> gene product required for λ site specific recombination. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=An E. coli gene product required for λ site specific recombination">google scholar</a></td></tr>
<tr><td><a name="ref521"></a>521</td><td>Miller, H. I., A. Kikuchi, H. A. Nash, R. A. Weisberg, and D. I. Friedman <i>1978</i>. Site-specific recombination of bacteriophage λ: the role of host gene products. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Site-specific recombination of bacteriophage λ: the role of host gene products">google scholar</a></td></tr>
<tr><td><a name="ref522"></a>522</td><td>Miller, H. I., M. Kirk, and H. Echols <i>1981</i>. SOS induction and autoregulation of the <i>himA</i> gene for site-specific recombination in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=SOS induction and autoregulation of the himA gene for site-specific recombination in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref523"></a>523</td><td>Miller, J. H <i>1972</i>. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=">google scholar</a></td></tr>
<tr><td><a name="ref524"></a>524</td><td>Miller, J. H <i>1992</i>. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=">google scholar</a></td></tr>
<tr><td><a name="ref525"></a>525</td><td>Miner, K. M., and L. Frank <i>1974</i>. Sodium-stimulated glutamate transport in osmotically shocked cells and membrane vesicles of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Sodium-stimulated glutamate transport in osmotically shocked cells and membrane vesicles of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref526"></a>526</td><td>Miyoshi, Y., and H. Yamagata <i>1976</i>. Sucrose-dependent spectinomycin-resistant mutants of </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Sucrose-dependent spectinomycin-resistant mutants of">google scholar</a></td></tr>
<tr><td><a name="ref527"></a>527</td><td>Mojica, A. T <i>1975</i>. Transduction by phage PICM <i>clr-100</i> in <i>Salmonella typhimurium.</i> <i>Mol. Gen.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Transduction by phage PICM clr-100 in Salmonella typhimurium. Mol. Gen.">google scholar</a></td></tr>
<tr><td><a name="ref528"></a>528</td><td>Mojica, A. T., and E. Garcia <i>1976</i>. Growth of coliphage BF-23 on rough strains of <i>Salmonella</i> <i>typhimurium.</i> the <i>bfe</i> locus. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Growth of coliphage BF-23 on rough strains of Salmonella typhimurium. the bfe locus">google scholar</a></td></tr>
<tr><td><a name="ref529"></a>529</td><td>Molnar, J., I. B. Holland, and Y. Mandi <i>1977</i>. Selection of <i>lon</i> mutants in <i>Escherichia coli</i> by treatment with phenothiazines. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Selection of lon mutants in Escherichia coli by treatment with phenothiazines">google scholar</a></td></tr>
<tr><td><a name="ref530"></a>530</td><td>Monard, D., J. Janecek, and H. V. Rickenberg <i>1969</i>. The enzymatic degradation of 3′,5′ cyclic AMP in strains of <i>E. coli</i> sensitive and resistant to catabolite repression. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The enzymatic degradation of 3′,5′ cyclic AMP in strains of E. coli sensitive and resistant to catabolite repression">google scholar</a></td></tr>
<tr><td><a name="ref531"></a>531</td><td>Morimoto, R. I., A. Tissières, and C. Georgopoulos  <i>1994</i>. <i>The Biology of Heat Shock</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The Biology of Heat Shock">google scholar</a></td></tr>
<tr><td><a name="ref532"></a>532</td><td>Morimyo, M., E. Hongo, H. Hama-Inaba, and I. Machida <i>1992</i>. Cloning and characterization of the <i>mvrC</i> gene of <i>Escherichia coli</i> K12 which confers resistance against methyl viologen toxicity. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Cloning and characterization of the mvrC gene of Escherichia coli K12 which confers resistance against methyl viologen toxicity">google scholar</a></td></tr>
<tr><td><a name="ref533"></a>533</td><td>Morona, R., and U. Henning <i>1984</i>. Host range mutants of bacteriophage OX-2 can use two different outer membrane proteins of <i>Escherichia coli</i> K-12 as receptors. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Host range mutants of bacteriophage OX-2 can use two different outer membrane proteins of Escherichia coli K-12 as receptors">google scholar</a></td></tr>
<tr><td><a name="ref534"></a>534</td><td>Morona, R., and U. Henning <i>1986</i>. New locus (<i>ttr</i>) in <i>Escherichia coli</i> K-12 affecting sensitivity to bacteriophage T2 and growth on oleate as the sole carbon source. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=New locus (ttr) in Escherichia coli K-12 affecting sensitivity to bacteriophage T2 and growth on oleate as the sole carbon source">google scholar</a></td></tr>
<tr><td><a name="ref535"></a>535</td><td>Morona, R., M. Klose, and U. Henning <i>1984</i>. <i>Escherichia coli</i> K-12 outer membrane protein OmpA as a bacteriophage receptor: analysis of mutant genes expressing altered proteins. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Escherichia coli K-12 outer membrane protein OmpA as a bacteriophage receptor: analysis of mutant genes expressing altered proteins">google scholar</a></td></tr>
<tr><td><a name="ref536"></a>536</td><td>Morona, R., and P. Reeves <i>1982</i>. The <i>tolC</i> locus of <i>Escherichia coli</i> affects the expression of three major outer membrane proteins. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The tolC locus of Escherichia coli affects the expression of three major outer membrane proteins">google scholar</a></td></tr>
<tr><td><a name="ref537"></a>537</td><td>Mosher, M. E., L. K. White, J. Hermolin, and R. H. Fillingame <i>1985</i>. Proton-translocating ATPase of <i>Escherichia coli:</i> an <i>uncE</i> mutation impairing coupling between F-1 and F-0 but not F-0 mediated proton translocation. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Proton-translocating ATPase of Escherichia coli: an uncE mutation impairing coupling between F-1 and F-0 but not F-0 mediated proton translocation">google scholar</a></td></tr>
<tr><td><a name="ref538"></a>538</td><td>Mount, D. W <i>1977</i>. A mutant of <i>Escherichia coli</i> showing constitutive expression of the lysogenic induction and error-prone DNA repair pathways. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A mutant of Escherichia coli showing constitutive expression of the lysogenic induction and error-prone DNA repair pathways">google scholar</a></td></tr>
<tr><td><a name="ref539"></a>539</td><td>Moyed, H. S <i>1960</i>. False feedback inhibition: inhibition of tryptophan biosynthesis by 5- methyltryptophan. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=False feedback inhibition: inhibition of tryptophan biosynthesis by 5- methyltryptophan">google scholar</a></td></tr>
<tr><td><a name="ref540"></a>540</td><td>Moyed, H. S <i>1961</i>. Interference with the feed-back control of histidine biosynthesis. <i>J. Biol.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Interference with the feed-back control of histidine biosynthesis. J. Biol.">google scholar</a></td></tr>
<tr><td><a name="ref541"></a>541</td><td>Moyed, H. S., and K. P. Bertrand <i>1983</i>. <i>hipA,</i> a newly recognized gene of <i>Escherichia coli</i> K-12 that affects frequency of persistence after inhibition of murein synthesis. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=hipA, a newly recognized gene of Escherichia coli K-12 that affects frequency of persistence after inhibition of murein synthesis">google scholar</a></td></tr>
<tr><td><a name="ref542"></a>542</td><td>Muir, M. E., D. R. Hanwell, and B. J. Wallace <i>1981</i>. Characterization of a respiratory mutant of <i>Escherichia coli</i> with reduced uptake of aminoglycoside antibiotics. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Characterization of a respiratory mutant of Escherichia coli with reduced uptake of aminoglycoside antibiotics">google scholar</a></td></tr>
<tr><td><a name="ref543"></a>543</td><td>Muller-Hill, B., L. Crapo, and W. Gilbert <i>1968</i>. Mutants that make more lac repressor. <i>Proc.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutants that make more lac repressor. Proc.">google scholar</a></td></tr>
<tr><td><a name="ref544"></a>544</td><td>Munch-Petersen, A., and B. Mygind <i>1976</i>. Nucleoside transport systems in <i>Escherichia coli</i> K12: specificity and regulation. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Nucleoside transport systems in Escherichia coli K12: specificity and regulation">google scholar</a></td></tr>
<tr><td><a name="ref545"></a>545</td><td>Munch-Petersen, A., B. Mygind, A. Nicolaisen, and N. J. Pihl <i>1979</i>. Nucleoside transport in cells and membrane vesicles from <i>Escherichia coli</i> K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Nucleoside transport in cells and membrane vesicles from Escherichia coli K12">google scholar</a></td></tr>
<tr><td><a name="ref546"></a>546</td><td>Munch-Petersen, A., P. Nygaard, K. Hammer-Jespersen, and N. Fiil <i>1972</i>. Mutants constitutive for nucleoside-catabolizing enzymes in <i>Escherichia coli</i> K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutants constitutive for nucleoside-catabolizing enzymes in Escherichia coli K12">google scholar</a></td></tr>
<tr><td><a name="ref547"></a>547</td><td>Munekiyo, R., T. Tsuzuki, and M. Sekiguchi <i>1979</i>. A new locus of <i>Escherichia coli</i> that determines sensitivity to bacteriophage φX174. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A new locus of Escherichia coli that determines sensitivity to bacteriophage φX174">google scholar</a></td></tr>
<tr><td><a name="ref548"></a>548</td><td>Murata, K., K. Tani, J. Kato, and I. Chibata <i>1980</i>. Excretion of glutathione by methylglyoxal- resistant <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Excretion of glutathione by methylglyoxal- resistant Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref549"></a>549</td><td>Murooka, Y., and T. Harada <i>1981</i>. Regulation of derepressed synthesis of arylsulfatase by tyramine oxidase in <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation of derepressed synthesis of arylsulfatase by tyramine oxidase in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref550"></a>550</td><td>Myers, D. E., B. A. D. Stocker, and R. J. Roantree <i>1980</i>. Mapping of genes determining penicillin-resistance and serum-sensitivity in <i>Salmonella enteritidis.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mapping of genes determining penicillin-resistance and serum-sensitivity in Salmonella enteritidis.">google scholar</a></td></tr>
<tr><td><a name="ref551"></a>551</td><td>Myers, R. S., and S. R. Maloy <i>1988</i>. Mutations of <i>putP</i> that alter the lithium sensitivity of <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutations of putP that alter the lithium sensitivity of Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref552"></a>552</td><td>Nagelkerke, F., and P. W. Postma <i>1978</i>. 2-Deoxygalactose, a specific substrate of the <i>Salmonella typhimurium</i> galactose permease: its use for the isolation of <i>galP</i> mutants. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=2-Deoxygalactose, a specific substrate of the Salmonella typhimurium galactose permease: its use for the isolation of galP mutants">google scholar</a></td></tr>
<tr><td><a name="ref553"></a>553</td><td>Nakamura, H <i>1979</i>. Novel acriflavine resistance genes, <i>acrC</i> and <i>acrD,</i> in <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Novel acriflavine resistance genes, acrC and acrD, in Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref554"></a>554</td><td>Nakamura, H <i>1980</i>. Complementation analysis between genes determining cell division, <i>arcC,</i> and DNA polymerase III, <i>dnaE,</i> in <i>Escherichia coli</i> K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Complementation analysis between genes determining cell division, arcC, and DNA polymerase III, dnaE, in Escherichia coli K12">google scholar</a></td></tr>
<tr><td><a name="ref555"></a>555</td><td>Nakamura, H., A. Hase, and K. Hirayoshi <i>1987</i>. Temperature-sensitive phase in the cell cycle of ACR-C mutants of Escherichia coli K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Temperature-sensitive phase in the cell cycle of ACR-C mutants of Escherichia coli K12">google scholar</a></td></tr>
<tr><td><a name="ref556"></a>556</td><td>Nakamura, Y., T. Kurihara, H. Saito, and H. Uchida <i>1979</i>. σ-subunit of <i>Escherichia coli</i> RNA polymerase affects the function of λ N gene. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=σ-subunit of Escherichia coli RNA polymerase affects the function of λ N gene">google scholar</a></td></tr>
<tr><td><a name="ref557"></a>557</td><td>Nakayama, H., K. Nakayama, R. Nakayama, N. Irino, Y. Nakayama, and P. C. Hanawalt <i>1984</i>. Isolation and genetic characterization of a thymineless death-resistant mutant of <i>Escherichia coli</i> K12: identification of a new mutation (<i>recQ1</i>) that blocks the RecF recombination pathway. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation and genetic characterization of a thymineless death-resistant mutant of Escherichia coli K12: identification of a new mutation (recQ1) that blocks the RecF recombination pathway">google scholar</a></td></tr>
<tr><td><a name="ref558"></a>558</td><td>Nakayama, K., S. Shiota, and H. Nakayama <i>1988</i>. Thymineless death in <i>Escherichia coli</i> mutants deficient in the <i>recF</i> recombination pathway. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Thymineless death in Escherichia coli mutants deficient in the recF recombination pathway">google scholar</a></td></tr>
<tr><td><a name="ref559"></a>559</td><td>Nashimoto, H., and H. Uchida <i>1975</i>. Late steps in the assembly of 30s ribosomal proteins <i>in vivo</i> in a spectinomycin-resistant mutant of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Late steps in the assembly of 30s ribosomal proteins in vivo in a spectinomycin-resistant mutant of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref560"></a>560</td><td>Nass, G., and J. Thomale <i>1974</i>. Alteration of structure or level of threonyl-tRNA synthetase in borrelidin-resistant mutants of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Alteration of structure or level of threonyl-tRNA synthetase in borrelidin-resistant mutants of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref561"></a>561</td><td>Nazos, P. M., M. M. Mayo, T. Z. Su, J. J. Anderson, and D. L. Oxender <i>1985</i>. Identification of <i>livG,</i> a membrane-associated component of the branched-chain amino acid transport in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Identification of livG, a membrane-associated component of the branched-chain amino acid transport in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref562"></a>562</td><td>Nelson, B. W., and R. J. Roantree <i>1967</i>. Analysis of lipopolysaccharides extracted from penicillin-resistant, serum-sensitive <i>Salmonella</i> mutants. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Analysis of lipopolysaccharides extracted from penicillin-resistant, serum-sensitive Salmonella mutants">google scholar</a></td></tr>
<tr><td><a name="ref563"></a>563</td><td>Nelson, S. O., B. J. Scholte, and P. W. Postma <i>1982</i>. Phosphoenolpyruvate:sugar phosphotransferase system-mediated regulation of carbohydrate metabolism in <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Phosphoenolpyruvate:sugar phosphotransferase system-mediated regulation of carbohydrate metabolism in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref564"></a>564</td><td>Nelms, J., R. M. Edwards, J. Warwick, and I. Fotheringham <i>1992</i>. Novel mutations in the <i>pheA</i> gene of <i>Escherichia coli</i> K-12 which result in highly feedback inhibition resistant variants of chorismate mutase/prephenate dehydratase. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Novel mutations in the pheA gene of Escherichia coli K-12 which result in highly feedback inhibition resistant variants of chorismate mutase/prephenate dehydratase">google scholar</a></td></tr>
<tr><td><a name="ref565"></a>565</td><td>Neuhard, J., and K. Thomassen <i>1976</i>. Altered deoxyribonucleotide pools in P2 eductants of <i>Escherichia coli</i> K-12 due to deletion of the <i>dcd</i> gene. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Altered deoxyribonucleotide pools in P2 eductants of Escherichia coli K-12 due to deletion of the dcd gene">google scholar</a></td></tr>
<tr><td><a name="ref566"></a>566</td><td>Newell, S. L., and W. J. Brill <i>1972</i>. Mutants of <i>Salmonella typhimurium</i> that are insensitive to catabolite repression of proline degradation. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutants of Salmonella typhimurium that are insensitive to catabolite repression of proline degradation">google scholar</a></td></tr>
<tr><td><a name="ref567"></a>567</td><td>Newman, E. B., N. Malik, and C. Walter <i>1982</i>. L-Serine degradation in <i>Escherichia coli</i> K-12: directly isolated <i>ssd</i> mutants and their intragenic revertants. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=L-Serine degradation in Escherichia coli K-12: directly isolated ssd mutants and their intragenic revertants">google scholar</a></td></tr>
<tr><td><a name="ref568"></a>568</td><td>Ng, H., and T. K. Gartner <i>1963</i>. Selection of mutants of <i>Escherichia coli</i> constitutive for tryptophanase. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Selection of mutants of Escherichia coli constitutive for tryptophanase">google scholar</a></td></tr>
<tr><td><a name="ref569"></a>569</td><td>Nikaido, H <i>1961</i>. Galactose sensitive mutants of <i>Salmonella.</i> I. Metabolism of galactoside. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Galactose sensitive mutants of Salmonella. I. Metabolism of galactoside">google scholar</a></td></tr>
<tr><td><a name="ref570"></a>570</td><td>Normark, S., T. Edlund, T. Grundstrom, S. Bergstrom, and H. Wolf-Watz <i>1977</i>. <i>Escherichia</i> <i>coli</i> K-12 mutants hyperproducing chromosomal beta-lactamase by gene repetitions. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Escherichia coli K-12 mutants hyperproducing chromosomal beta-lactamase by gene repetitions">google scholar</a></td></tr>
<tr><td><a name="ref571"></a>571</td><td>Norwood, W. I., and J. R. Sadler <i>1977</i>. Pseudoreversion of lactose operator-constitutive mutants. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Pseudoreversion of lactose operator-constitutive mutants">google scholar</a></td></tr>
<tr><td><a name="ref572"></a>572</td><td>Novel, G., and M. Novel <i>1973</i>. Mutants d’<i>Escherichia coli</i> K12 affectés pour leur croissance sur méthyl-β-D-glucuronide: localisation du gène de structure de la β-D-glucuronidase (<i>uidA</i>). </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutants d’Escherichia coli K12 affectés pour leur croissance sur méthyl-β-D-glucuronide: localisation du gène de structure de la β-D-glucuronidase (uidA)">google scholar</a></td></tr>
<tr><td><a name="ref573"></a>573</td><td>Novel, M., and G. Novel <i>1976</i>. Regulation of β-glucuronidase synthesis in <i>Escherichia coli</i> K-12: constitutive mutants specifically derepressed for <i>uidA</i> expression. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation of β-glucuronidase synthesis in Escherichia coli K-12: constitutive mutants specifically derepressed for uidA expression">google scholar</a></td></tr>
<tr><td><a name="ref574"></a>574</td><td>Novel, M., and G. Novel <i>1976</i>. Regulation of β-glucuronidase synthesis in <i>Escherichia coli</i> K-12: pleiotropic constitutive mutations affecting <i>uxu</i> and <i>uidA</i> expression. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation of β-glucuronidase synthesis in Escherichia coli K-12: pleiotropic constitutive mutations affecting uxu and uidA expression">google scholar</a></td></tr>
<tr><td><a name="ref575"></a>575</td><td>Nummer, B. A., S. F. Barefoot, and E. L. Kline <i>1992</i>. Effects of the <i>flrA</i> regulatory locus on biosynthesis and excretion of amino acids in <i>Escherichia coli</i> B/r. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Effects of the flrA regulatory locus on biosynthesis and excretion of amino acids in Escherichia coli B/r">google scholar</a></td></tr>
<tr><td><a name="ref576"></a>576</td><td>Nurminen, M., K. Lounatmaa, M. Sarvas, P. H. Makela, and T. Nakae <i>1976</i>. Bacteriophage- resistant mutants of <i>Salmonella typhimurium</i> deficient in two major outer membrane proteins. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Bacteriophage- resistant mutants of Salmonella typhimurium deficient in two major outer membrane proteins">google scholar</a></td></tr>
<tr><td><a name="ref577"></a>577</td><td>Obukowicz, M. G., N. R. Staten, and G. G. Krivi <i>1992</i>. Enhanced heterologous gene expression in novel <i>rpoH</i> mutants of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Enhanced heterologous gene expression in novel rpoH mutants of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref578"></a>578</td><td>Odoevskaya, E. R., and S. P. Sineolsii <i>1987</i>. Isolation and genetic study of bacterial mutations <i>gpr</i> blocking the replication of certain lambdoid phages. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation and genetic study of bacterial mutations gpr blocking the replication of certain lambdoid phages">google scholar</a></td></tr>
<tr><td><a name="ref579"></a>579</td><td>O’Donovan, G. A., G. Edlin, J. A. Fuchs, J. Neuhard, and E. Thomassen <i>1971</i>. Deoxycytidine triphosphate deaminase: characterization of an <i>Escherichia coli</i> mutant deficient in the enzyme. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Deoxycytidine triphosphate deaminase: characterization of an Escherichia coli mutant deficient in the enzyme">google scholar</a></td></tr>
<tr><td><a name="ref580"></a>580</td><td>Ohnishi, K., and K. Kiritani <i>1978</i>. Glycyl-L-leucine-resistance mutation affecting transport of branched-chain amino acids in <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Glycyl-L-leucine-resistance mutation affecting transport of branched-chain amino acids in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref581"></a>581</td><td>Ohta, N., P. R. Galsworthy, and A. B. Pardee <i>1971</i>. Genetics of sulfate transport by <i>Salmonella</i> <i>typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetics of sulfate transport by Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref582"></a>582</td><td>Okada, Y., M. Wachi, K. Nagai, and M. Matsuhashi <i>1992</i>. Change of the quantity of penicillin-binding proteins and other cytoplasmic and membrane proteins by mutations of the cell shape-determination genes <i>mreB</i> <i>mrec</i> and <i>mreD</i> of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Change of the quantity of penicillin-binding proteins and other cytoplasmic and membrane proteins by mutations of the cell shape-determination genes mreB mrec and mreD of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref583"></a>583</td><td>Okuyama, A., M. Yoshikawa, and N. Tanaka <i>1974</i>. Alteration of ribosomal protein S2 in kasugamycin resistant mutant derived from <i>Escherichia coli</i> AB312. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Alteration of ribosomal protein S2 in kasugamycin resistant mutant derived from Escherichia coli AB312">google scholar</a></td></tr>
<tr><td><a name="ref584"></a>584</td><td>Oliver, D. B., R. J. Cabelli, K. M. Dolan, and G. P. Jarosik <i>1990</i>. Azide resistant mutants of <i>Escherichia coli</i> alter the SecA protein, an azide-sensitive component of the protein export machinery. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Azide resistant mutants of Escherichia coli alter the SecA protein, an azide-sensitive component of the protein export machinery">google scholar</a></td></tr>
<tr><td><a name="ref585"></a>585</td><td>O’Neill, G. P., S. Thorbjanardottir, U. Michelsen, S. Palsson, D. Söll, and G. Eggertsson <i>1991</i>. ∆-Aminolevulinic acid dehydratase deficiency can cause ∆-aminolevulinate auxotrophy in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=∆-Aminolevulinic acid dehydratase deficiency can cause ∆-aminolevulinate auxotrophy in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref586"></a>586</td><td>O’Neill, J. B., and M. Freundlich <i>1972</i>. Effects of cyclopentane glycine on metabolism in <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Effects of cyclopentane glycine on metabolism in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref587"></a>587</td><td>Oppezzo, O. J., B. Avanzati, and D. N. Anton <i>1991</i>. Increased susceptibility to beta lactam antibiotics and decreased porin content caused by <i>envB</i> mutations of </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Increased susceptibility to beta lactam antibiotics and decreased porin content caused by envB mutations of">google scholar</a></td></tr>
<tr><td><a name="ref588"></a>588</td><td>Ornellas, E. P., and B. A. D. Stocker <i>1974</i>. Relation of lipopolysaccharide character to Pl sensitivity in <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Relation of lipopolysaccharide character to Pl sensitivity in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref589"></a>589</td><td>Orr, E., N. F. Fairweather, I. B. Holland, and R. H. Pritchard <i>1979</i>. Isolation and characterization of a strain carrying a conditional lethal mutation in the <i>cou</i> gene of <i>Escherichia coli </i>K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation and characterization of a strain carrying a conditional lethal mutation in the cou gene of Escherichia coli K12">google scholar</a></td></tr>
<tr><td><a name="ref590"></a>590</td><td>Orr, J. C., D. W. Bryant, D. R. McCalla, and M. A. Quilliam <i>1985</i>. Dinitropyrene resistant <i>Salmonella typhimurium</i> are deficient in an acetyl-CoA acetyltransferase. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Dinitropyrene resistant Salmonella typhimurium are deficient in an acetyl-CoA acetyltransferase">google scholar</a></td></tr>
<tr><td><a name="ref591"></a>591</td><td>Osborn, M. J., S. M. Rosen, L. Rothfield, and B. L. Horecker <i>1962</i>. Biosynthesis of bacterial lipopolysaccharide. I. Enzymatic incorporation of galactose in a mutant strain of <i>Salmonella.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Biosynthesis of bacterial lipopolysaccharide. I. Enzymatic incorporation of galactose in a mutant strain of Salmonella.">google scholar</a></td></tr>
<tr><td><a name="ref592"></a>592</td><td>Osorio, A. V., L. Servin-Gonzalez, M. Rocha, A. A. Covarrubias, and F. Bastarrachea <i>1984</i>. <i>cis</i>-dominant, glutamine synthetase constitutive mutations of <i>Escherichia coli</i> independent of activation by the <i>glnG</i> and <i>glnF</i> products. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=cis-dominant, glutamine synthetase constitutive mutations of Escherichia coli independent of activation by the glnG and glnF products">google scholar</a></td></tr>
<tr><td><a name="ref593"></a>593</td><td>Ostrowski, J., and D. Hulanicka <i>1981</i>. Effect of DNA gyrase inhibitors on gene expression of the cysteine region. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Effect of DNA gyrase inhibitors on gene expression of the cysteine region">google scholar</a></td></tr>
<tr><td><a name="ref594"></a>594</td><td>Oxender, D. L <i>1972</i>. Membrane transport. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Membrane transport">google scholar</a></td></tr>
<tr><td><a name="ref595"></a>595</td><td>Oxender, D. L., J. J. Anderson, M. M. Mayo, and S. C. Quay <i>1977</i>. Leucine binding protein and regulation of transport in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Leucine binding protein and regulation of transport in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref596"></a>596</td><td>Paetz, W., and G. Nass <i>1973</i>. Biochemical and immunological characterization of threonyl- tRNA synthetase of two borrelidin resistant mutants of <i>Escherichia coli</i> K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Biochemical and immunological characterization of threonyl- tRNA synthetase of two borrelidin resistant mutants of Escherichia coli K12">google scholar</a></td></tr>
<tr><td><a name="ref597"></a>597</td><td>Pai, C. H <i>1974</i>. Biochemical and genetic characterization of dehydrobiotin resistant mutants of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Biochemical and genetic characterization of dehydrobiotin resistant mutants of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref598"></a>598</td><td>Pardee, A. B., E. J. Benz, Jr., D. A. St. Peter, J. N. Krieger, M. Meuth, and H. W. Trieshmann, Jr <i>1971</i>. Hyperproduction and purification of nicotinamide deamidase, a microconstitutive enzyme of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Hyperproduction and purification of nicotinamide deamidase, a microconstitutive enzyme of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref599"></a>599</td><td>Pardo, D., and R. Rosset <i>1977</i>. Properties of ribosomes from erythromycin resistant mutants of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Properties of ribosomes from erythromycin resistant mutants of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref600"></a>600</td><td>Park, M. H., B. B. Wong, and J. E. Lusk <i>1976</i>. Mutants in three genes affecting transport of magnesium in <i>Escherichia coli:</i> genetics and physiology. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutants in three genes affecting transport of magnesium in Escherichia coli: genetics and physiology">google scholar</a></td></tr>
<tr><td><a name="ref601"></a>601</td><td>Parker, C. T., A. W. Kloser, C. A. Schnaitman, M. A. Stein, S. Gottesman, and B. W. Gibson <i>1992</i>. Role of the <i>rfaG</i> and <i>rfaP</i> genes in determining the lipopolysaccharide core structure and cell surface properties of <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Role of the rfaG and rfaP genes in determining the lipopolysaccharide core structure and cell surface properties of Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref602"></a>602</td><td>Pauli, G., and P. Overath <i>1972</i>. <i>ato</i> operon: a highly inducible system for acetoacetate and butyrate degradation in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=ato operon: a highly inducible system for acetoacetate and butyrate degradation in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref603"></a>603</td><td>Payne, J. W., J. S. Morley, P. Armitage, and G. M. Payne <i>1984</i>. Transport and hydrolysis of antibacterial peptide analogues in <i>Escherichia coli:</i> backbone-modified aminoxy peptides. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Transport and hydrolysis of antibacterial peptide analogues in Escherichia coli: backbone-modified aminoxy peptides">google scholar</a></td></tr>
<tr><td><a name="ref604"></a>604</td><td>Penninckx, M., and D. Gigot <i>1979</i>. Synthesis of a peptide form of N-δ-(phosphonoacetyl)-L- ornithine. Its antibacterial effect through the specific inhibition of <i>Escherichia coli</i> L-ornithine carbamoyltransferase. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Synthesis of a peptide form of N-δ-(phosphonoacetyl)-L- ornithine. Its antibacterial effect through the specific inhibition of Escherichia coli L-ornithine carbamoyltransferase">google scholar</a></td></tr>
<tr><td><a name="ref605"></a>605</td><td>Pierard, A., and N. Glansdorf <i>1972</i>. Mutations affecting uridine monophosphate pyrophosphorylase or the <i>argR</i> gene in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutations affecting uridine monophosphate pyrophosphorylase or the argR gene in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref606"></a>606</td><td>Pierard, A., N. Glansdorf, D. Gigot, M. Crabeel, P. Halleux, and L. Thiry <i>1976</i>. Repression of <i>Escherichia coli</i> carbamoylphosphate synthase: relationships with enzyme synthesis in the arginine and pyrimidine pathways. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Repression of Escherichia coli carbamoylphosphate synthase: relationships with enzyme synthesis in the arginine and pyrimidine pathways">google scholar</a></td></tr>
<tr><td><a name="ref607"></a>607</td><td>Pinn, P. A., K. J. Towner, and F. W. O’Grady <i>1983</i>. Genetic analysis of chromosomal resistance to trimethoprim derived from clinical isolates of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic analysis of chromosomal resistance to trimethoprim derived from clinical isolates of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref608"></a>608</td><td>Plate, C. A., and J. L. Suit <i>1981</i>. The <i>eup</i> genetic locus of <i>Escherichia coli</i> and its role in proton solute symport. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The eup genetic locus of Escherichia coli and its role in proton solute symport">google scholar</a></td></tr>
<tr><td><a name="ref609"></a>609</td><td>Platz, A., and B.-M. Sjoberg <i>1980</i>. Construction and characterization of hybrid plasmids containing the <i>Escherichia coli nrd</i> region. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Construction and characterization of hybrid plasmids containing the Escherichia coli nrd region">google scholar</a></td></tr>
<tr><td><a name="ref610"></a>610</td><td>Pledger, W. J., and H. E. Umbarger <i>1973</i>. Isoleucine and valine metabolism in <i>Escherichia coli.</i> XXI. Mutations affecting derepression and valine resistance. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isoleucine and valine metabolism in Escherichia coli. XXI. Mutations affecting derepression and valine resistance">google scholar</a></td></tr>
<tr><td><a name="ref611"></a>611</td><td>Popkin, P. S., and W. K. Maas <i>1980</i>. <i>Escherichia coli</i> regulatory mutation affecting lysine transport and lysine decarboxylase. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Escherichia coli regulatory mutation affecting lysine transport and lysine decarboxylase">google scholar</a></td></tr>
<tr><td><a name="ref612"></a>612</td><td>Portalier, R., J. Robert-Baudouy, and F. Stoeber <i>1980</i>. Regulation of <i>Escherichia coli</i> K-12 hexuronate system genes: <i>exu</i> regulon. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation of Escherichia coli K-12 hexuronate system genes: exu regulon">google scholar</a></td></tr>
<tr><td><a name="ref613"></a>613</td><td>Postma, P. W <i>1977</i>. Galactose transport in <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Galactose transport in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref614"></a>614</td><td>Postma, P. W., and J. W. Lengeler <i>1985</i>. Phosphoenolpyruvate:carbohydrate phosphotransferase system of bacteria. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Phosphoenolpyruvate:carbohydrate phosphotransferase system of bacteria">google scholar</a></td></tr>
<tr><td><a name="ref615"></a>615</td><td>Pouyssegur, J. M., and F. R. Stoeber <i>1972</i>. Rameau degradatif commun des hexuronates chez <i>Escherichia coli</i> K12. Mécanisme d’induction dex enzymes assurant le métabolisme du 2-céto-3- desoxy-gluconate. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Rameau degradatif commun des hexuronates chez Escherichia coli K12. Mécanisme d’induction dex enzymes assurant le métabolisme du 2-céto-3- desoxy-gluconate">google scholar</a></td></tr>
<tr><td><a name="ref616"></a>616</td><td>Prasad, I., B. Young, and S. Schaefler <i>1973</i>. Genetic determination of the constitutive biosynthesis of phospho-β-glucosidase A in <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic determination of the constitutive biosynthesis of phospho-β-glucosidase A in Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref617"></a>617</td><td>Pritchard, R. H., and S. I. Ahmad <i>1971</i>. Fluorouracil and the isolation of mutants lacking uridine phosphorylase in <i>E. coli:</i> location of the gene. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Fluorouracil and the isolation of mutants lacking uridine phosphorylase in E. coli: location of the gene">google scholar</a></td></tr>
<tr><td><a name="ref618"></a>618</td><td>Prody, C. A., and J. B. Nellands <i>1984</i>. Genetic and biochemical characterization of the <i>Escherichia coli</i> K-12 <i>fhuB</i> mutation. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic and biochemical characterization of the Escherichia coli K-12 fhuB mutation">google scholar</a></td></tr>
<tr><td><a name="ref619"></a>619</td><td>Pueyo, C., and J. Lopez-Barea <i>1979</i>. The L-arabinose-resistance test with <i>Salmonella</i> <i>typhimurium</i> strain SV3 selects forward mutations at several <i>ara</i> genes. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The L-arabinose-resistance test with Salmonella typhimurium strain SV3 selects forward mutations at several ara genes">google scholar</a></td></tr>
<tr><td><a name="ref620"></a>620</td><td>Pugsley, A. P <i>1985</i>. <i>Escherichia coli</i> K12 strains for use in the identification and characterization of colicins. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Escherichia coli K12 strains for use in the identification and characterization of colicins">google scholar</a></td></tr>
<tr><td><a name="ref621"></a>621</td><td>Pugsley, A. P., D. R. Lee, and C. A. Schnaitman <i>1980</i>. Genes affecting the major outer membrane proteins of <i>Escherichia coli</i> K12: mutations at <i>nmpA</i> and <i>nmpB.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genes affecting the major outer membrane proteins of Escherichia coli K12: mutations at nmpA and nmpB.">google scholar</a></td></tr>
<tr><td><a name="ref622"></a>622</td><td>Pugsley, A. P., F. Moreno and V. de Lorenzo <i>1986</i>. Microcin-E492-insensitive mutants of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Microcin-E492-insensitive mutants of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref623"></a>623</td><td>Pugsley, A. P., and P. Reeves <i>1976</i>. Iron uptake in colicin B-resistant mutants of <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Iron uptake in colicin B-resistant mutants of Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref624"></a>624</td><td>Pugsley, A. P., and P. Reeves <i>1976</i>. Characterization of group B colicin-resistant mutants of <i>Escherichia coli</i> K-12: colicin resistance and the role of enterochelin. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Characterization of group B colicin-resistant mutants of Escherichia coli K-12: colicin resistance and the role of enterochelin">google scholar</a></td></tr>
<tr><td><a name="ref625"></a>625</td><td>Pugsley, A. P., and C. A. Schnaitman <i>1978</i>. Outer membrane proteins of <i>Escherichia coli.</i> VII. Evidence that bacteriophage-directed protein 2 functions as a pore. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Outer membrane proteins of Escherichia coli. VII. Evidence that bacteriophage-directed protein 2 functions as a pore">google scholar</a></td></tr>
<tr><td><a name="ref626"></a>626</td><td>Pugsley, A. P., and C. A. Schnaitman <i>1978</i>. Identification of three genes controlling production of new outer membrane pore proteins in <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Identification of three genes controlling production of new outer membrane pore proteins in Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref627"></a>627</td><td>Puyo, M. F., P. Calsou, and B. Salles <i>1992</i>. UV resistance of <i>E. coli</i> K12 deficient in cAMP/CRP regulation. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=UV resistance of E. coli K12 deficient in cAMP/CRP regulation">google scholar</a></td></tr>
<tr><td><a name="ref628"></a>628</td><td>Rahav-Manor, O., O. Carmel, R. Karpel, D. Taglicht, G. Glaser, S. Schuldiner, and E. Padan <i>1992</i>. NhaR, a protein homologous to a family of bacterial regulatory proteins (LysR), regulates <i>nhaA,</i> the sodium proton antiporter gene in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=NhaR, a protein homologous to a family of bacterial regulatory proteins (LysR), regulates nhaA, the sodium proton antiporter gene in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref629"></a>629</td><td>Rainwater, S., and P. M. Silverman <i>1990</i>. The Cpx proteins of <i>Escherichia coli</i> K-12: evidence that <i>cpxA,</i> <i>ecfB,</i> <i>ssd,</i> and <i>eup</i> mutations all identify the same gene. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The Cpx proteins of Escherichia coli K-12: evidence that cpxA, ecfB, ssd, and eup mutations all identify the same gene">google scholar</a></td></tr>
<tr><td><a name="ref630"></a>630</td><td>Rakonjac, J., M. Milic, D. Ajdic-Predic, D. Santos, R. Ivanisevic, and D. J. Savic <i>1992</i>. A new genetic locus that affects the response of <i>Escherichia coli</i> K12 to novobiocin. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A new genetic locus that affects the response of Escherichia coli K12 to novobiocin">google scholar</a></td></tr>
<tr><td><a name="ref631"></a>631</td><td>Rakonjac, J., M. Milic, and D. J. Savic <i>1991</i>. <i>cysB</i> and <i>cysE</i> mutants of <i>Escherichia coli</i> K12 show increased resistance to novobiocin. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=cysB and cysE mutants of Escherichia coli K12 show increased resistance to novobiocin">google scholar</a></td></tr>
<tr><td><a name="ref632"></a>632</td><td>Ramabhadran, T. V <i>1976</i>. Method for the isolation of <i>Escherichia coli</i> relaxed mutants, utilizing near-ultraviolet irradiation. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Method for the isolation of Escherichia coli relaxed mutants, utilizing near-ultraviolet irradiation">google scholar</a></td></tr>
<tr><td><a name="ref633"></a>633</td><td>Ramakrishnan, T., and E. A. Adelberg <i>1965</i>. Regulatory mechanisms in the biosynthesis of isoleucine and valine. II. Identification of two operator genes. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulatory mechanisms in the biosynthesis of isoleucine and valine. II. Identification of two operator genes">google scholar</a></td></tr>
<tr><td><a name="ref634"></a>634</td><td>Ramakrishnan, V., and S. W. White <i>1992</i>. The structure of ribosomal protein S5 reveals sites of interaction with 16s RRNA. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The structure of ribosomal protein S5 reveals sites of interaction with 16s RRNA">google scholar</a></td></tr>
<tr><td><a name="ref635"></a>635</td><td>Rancourt, D. E., J. T. Stephenson, G. A. Vickell, and J. M. Wood <i>1984</i>. Proline excretion by <i>Escherichia coli</i> K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Proline excretion by Escherichia coli K12">google scholar</a></td></tr>
<tr><td><a name="ref636"></a>636</td><td>Raney, M. E., and R. W. Elliott <i>1978</i>. The biochemical and genetic basis for high frequency thiomethylgalactoside resistance in lambda-lambda-dg lysogens of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The biochemical and genetic basis for high frequency thiomethylgalactoside resistance in lambda-lambda-dg lysogens of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref637"></a>637</td><td>Ray, J. M., C. Yanofsky, and R. Bauerle <i>1988</i>. Mutational analysis of the catalytic and feedback sites of the tryptophan-sensitive 3-deoxy-D-<i>arabino</i>-heptulosonate-7-phosphate synthase of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutational analysis of the catalytic and feedback sites of the tryptophan-sensitive 3-deoxy-D-arabino-heptulosonate-7-phosphate synthase of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref638"></a>638</td><td>Reeve, E. C. R <i>1968</i>. Genetic analysis of some mutations causing resistance to tetracycline in <i>Escherichia coli</i> K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic analysis of some mutations causing resistance to tetracycline in Escherichia coli K12">google scholar</a></td></tr>
<tr><td><a name="ref639"></a>639</td><td>Reiner, A. M <i>1977</i>. Xylitol and D-arabitol toxicities due to derepressed fructose, galactitol, and sorbitol phosphotransferases of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Xylitol and D-arabitol toxicities due to derepressed fructose, galactitol, and sorbitol phosphotransferases of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref640"></a>640</td><td>Reynolds, A. E., J. Felton, and A. Wright <i>1981</i>. Insertion of DNA activates the cryptic <i>bgl</i> operon in <i>E. coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Insertion of DNA activates the cryptic bgl operon in E. coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref641"></a>641</td><td>Rick, P. D., and M. J. Osborn <i>1972</i>. Isolation of a mutant of <i>Salmonella typhimurium</i> dependent on D-arabinose-5-P for growth and synthesis of 3-deoxy-D-mannooctulosonate (ketodeoxyoctonate). </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation of a mutant of Salmonella typhimurium dependent on D-arabinose-5-P for growth and synthesis of 3-deoxy-D-mannooctulosonate (ketodeoxyoctonate)">google scholar</a></td></tr>
<tr><td><a name="ref642"></a>642</td><td>Ried, G., I. Hindennach, and U. Henning <i>1990</i>. Role of lipopolysaccharide in assembly of <i>Escherichia coli</i> outer membrane proteins OmpA, OmpC, and OmpF. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Role of lipopolysaccharide in assembly of Escherichia coli outer membrane proteins OmpA, OmpC, and OmpF">google scholar</a></td></tr>
<tr><td><a name="ref643"></a>643</td><td>Riley, M <i>1993</i>. Functions of the gene products of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Functions of the gene products of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref644"></a>644</td><td>Roa, M <i>1979</i>. Interaction of bacteriophage K10 with its receptor, the <i>lamB</i> protein of <i>Escherichia</i> <i>coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Interaction of bacteriophage K10 with its receptor, the lamB protein of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref645"></a>645</td><td>Robbins, A. R <i>1975</i>. Regulation of the <i>Escherichia coli</i> methylgalactoside transport system by gene <i>mglD.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation of the Escherichia coli methylgalactoside transport system by gene mglD.">google scholar</a></td></tr>
<tr><td><a name="ref646"></a>646</td><td>Robbins, J. C., and D. L Oxender <i>1973</i>. Transport systems for alanine, serine, and glycine in <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Transport systems for alanine, serine, and glycine in Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref647"></a>647</td><td>Roberton, A. M., P. A. Sullivan, M. C. Jones-Mortimer, and H. L. Kornberg <i>1980</i>. Two genes affecting glucarate utilization in <i>Escherichia coli</i> K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Two genes affecting glucarate utilization in Escherichia coli K12">google scholar</a></td></tr>
<tr><td><a name="ref648"></a>648</td><td>Roberts, L. M., and E. C. R. Reeve <i>1970</i>. Two mutations giving low-level streptomycin resistance in <i>Escherichia coli</i> K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Two mutations giving low-level streptomycin resistance in Escherichia coli K12">google scholar</a></td></tr>
<tr><td><a name="ref649"></a>649</td><td>Robinson, C. L., and J. H. Jackson <i>1982</i>. New acetohydroxy acid synthetase activity from mutational activation of a cryptic gene in <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=New acetohydroxy acid synthetase activity from mutational activation of a cryptic gene in Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref650"></a>650</td><td>Rodriguez, S. B., and J. L. Ingraham <i>1983</i>. Location on the <i>Salmonella typhimurium</i> chromosome of the gene encoding nucleoside diphosphokinase <i>(ndk).</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Location on the Salmonella typhimurium chromosome of the gene encoding nucleoside diphosphokinase (ndk).">google scholar</a></td></tr>
<tr><td><a name="ref651"></a>651</td><td>Roeder, N., and R. L Somerville <i>1979</i>. Cloning the <i>trpR</i> gene. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Cloning the trpR gene">google scholar</a></td></tr>
<tr><td><a name="ref652"></a>652</td><td>Roehl, R. A., and R. T. Vinopal <i>1980</i>. Genetic locus, distant from <i>ptsM,</i> affecting enzyme IIA/IIB function in <i>Escherichia</i> coli K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic locus, distant from ptsM, affecting enzyme IIA/IIB function in Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref653"></a>653</td><td>Rolfe, B., and K. Onodera <i>1971</i>. Demonstration of missing membrane proteins in a colicin- tolerant mutant of <i>Escherichia.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Demonstration of missing membrane proteins in a colicin- tolerant mutant of Escherichia.">google scholar</a></td></tr>
<tr><td><a name="ref654"></a>654</td><td>Rood, J. I., A. J. Laird, and J. W. Williams <i>1980</i>. Cloning of the <i>Escherichia coli</i> K12 dihydrofolate reductase EC-1.5.1.3 gene following phage Mu mediated transposition. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Cloning of the Escherichia coli K12 dihydrofolate reductase EC-1.5.1.3 gene following phage Mu mediated transposition">google scholar</a></td></tr>
<tr><td><a name="ref655"></a>655</td><td>Rosen, B. P <i>1973</i>. Basic amino acid transport in <i>Escherichia coli:</i> properties of canavanine- resistant mutants. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Basic amino acid transport in Escherichia coli: properties of canavanine- resistant mutants">google scholar</a></td></tr>
<tr><td><a name="ref656"></a>656</td><td>Rosen, S. M., L. D. Zeleznick, D. Fraenkel, I. M. Wiener, M. J. Osborn, and B. L. Horecker <i>1965</i>. Characterization of the cell wall lipopolysaccharide of a mutant of <i>Salmonella typhimurium </i>lacking phosphomannose isomerase. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Characterization of the cell wall lipopolysaccharide of a mutant of Salmonella typhimurium lacking phosphomannose isomerase">google scholar</a></td></tr>
<tr><td><a name="ref657"></a>657</td><td>Roth, J. R., and B. N. Ames <i>1966</i>. Histidine regulatory mutants in <i>Salmonella typhimurium.</i> II. Histidine regulatory mutants having altered histidyl-tRNA synthetase. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Histidine regulatory mutants in Salmonella typhimurium. II. Histidine regulatory mutants having altered histidyl-tRNA synthetase">google scholar</a></td></tr>
<tr><td><a name="ref658"></a>658</td><td>Roth, J. R., D. N. Anton, and P. E. Hartman <i>1966</i>. Histidine regulatory mutants in <i>Salmonella</i> <i>typhimurium.</i> I. Isolation and general properties. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Histidine regulatory mutants in Salmonella typhimurium. I. Isolation and general properties">google scholar</a></td></tr>
<tr><td><a name="ref659"></a>659</td><td>Rowbury, R. J., M. Goodson, and A. D. Wallace <i>1992</i>. The <i>phoE</i> porin and transmission of the chemical stimulus for induction of acid resistance acid habituation in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The phoE porin and transmission of the chemical stimulus for induction of acid resistance acid habituation in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref660"></a>660</td><td>Rudd, K. E., and R. Menzel <i>1987</i>. <i>his</i> operons of <i>Escherichia coli</i> and <i>Salmonella typhimurium</i> are regulated by DNA supercoiling. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=his operons of Escherichia coli and Salmonella typhimurium are regulated by DNA supercoiling">google scholar</a></td></tr>
<tr><td><a name="ref661"></a>661</td><td>Rudd, K. E., W. Miller, C. Werner, J. Ostell, C. Tolstoshev, and S. G. Satterfield <i>1991</i>. Mapping sequenced <i>E. coli</i> genes by computer: software, strategies and examples. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mapping sequenced E. coli genes by computer: software, strategies and examples">google scholar</a></td></tr>
<tr><td><a name="ref662"></a>662</td><td>Russell, R. R. B., and A. J. Pittard <i>1971</i>. Mutants of <i>Escherichia</i> coli unable to make protein at 42°C. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutants of Escherichia coli unable to make protein at 42°C">google scholar</a></td></tr>
<tr><td><a name="ref663"></a>663</td><td>Ryals, J., R.-Y. Hsu, M. N. Lipsett, and H. Bremer <i>1982</i>. Isolation of single-site <i>Escherichia</i> <i>coli</i> mutants deficient in thiamine and 4-thiouridine syntheses: identification of a <i>nuvC</i> mutant. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation of single-site Escherichia coli mutants deficient in thiamine and 4-thiouridine syntheses: identification of a nuvC mutant">google scholar</a></td></tr>
<tr><td><a name="ref664"></a>664</td><td>Saedler, H., A. Gullon, L. Fiethen, and P. Starlinger <i>1968</i>. Negative control of the galactose operon in <i>E. coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Negative control of the galactose operon in E. coli.">google scholar</a></td></tr>
<tr><td><a name="ref665"></a>665</td><td>Saier, M. H., Jr., H. Straud, L. S. Massman, J. J. Judice, M. J. Newman, and B. U. Feucht <i>1978</i>. Permease-specific mutations in <i>Salmonella typhimurium</i> and <i>Escherichia coli</i> that release the glycerol, maltose, melibiose, and lactose transport systems from regulation by the phosphoenolpyruvate:sugar phosphotransferase system. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Permease-specific mutations in Salmonella typhimurium and Escherichia coli that release the glycerol, maltose, melibiose, and lactose transport systems from regulation by the phosphoenolpyruvate:sugar phosphotransferase system">google scholar</a></td></tr>
<tr><td><a name="ref666"></a>666</td><td>Saito, H., and C. C. Richardson <i>1981</i>. Genetic analysis of gene 1.2 of bacteriophage T7: isolation of a mutant of <i>Escherichia coli</i> unable to support the growth of T7 gene 1.2 mutants. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic analysis of gene 1.2 of bacteriophage T7: isolation of a mutant of Escherichia coli unable to support the growth of T7 gene 1.2 mutants">google scholar</a></td></tr>
<tr><td><a name="ref667"></a>667</td><td>Saito, H., and H. Uchida <i>1977</i>. Initiation of the DNA replication of bacteriophage lambda in <i>Escherichia coli</i> K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Initiation of the DNA replication of bacteriophage lambda in Escherichia coli K12">google scholar</a></td></tr>
<tr><td><a name="ref668"></a>668</td><td>Sakka, K., T. Watanabe, R. Beers, and H. C. Wu <i>1987</i>. Isolation and characterization of a new globomycin-resistant <i>dnaE</i> mutant of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation and characterization of a new globomycin-resistant dnaE mutant of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref669"></a>669</td><td>Salanitro, J. P., and W. S. Wegener <i>1971</i>. Growth of <i>Escherichia</i> coli on short-chain fatty acids: nature of the uptake system. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Growth of Escherichia coli on short-chain fatty acids: nature of the uptake system">google scholar</a></td></tr>
<tr><td><a name="ref670"></a>670</td><td>Samsonov, V. V., E. R. Odoevskaia, and S. P. Sineokii <i>1992</i>. Cloning and complementation analysis of the <i>Escherichia coli</i> <i>gpr</i> locus, influencing DNA replication of certain lambdoid phages. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Cloning and complementation analysis of the Escherichia coli gpr locus, influencing DNA replication of certain lambdoid phages">google scholar</a></td></tr>
<tr><td><a name="ref671"></a>671</td><td>Sanchez-Anzaldo, F. J., and F. Bastarrachea <i>1974</i>. Genetic characterization of streptomycin- resistant and dependent mutants of <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic characterization of streptomycin- resistant and dependent mutants of Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref672"></a>672</td><td>Sanchez-Anzaldo, F. J., R. Gomez, and F. Bastarrachea <i>1979</i>. Paromoycin resistant mutants of <i>Escherichia coli</i> K12. I. Cross-resistance to streptomycin and synergism of the mixture of both antibiotics. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Paromoycin resistant mutants of Escherichia coli K12. I. Cross-resistance to streptomycin and synergism of the mixture of both antibiotics">google scholar</a></td></tr>
<tr><td><a name="ref673"></a>673</td><td>Sanderson, K. E., and J. R. Roth <i>1983</i>. Linkage map of <i>Salmonella typhimurium,</i> edition VI. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Linkage map of Salmonella typhimurium, edition VI">google scholar</a></td></tr>
<tr><td><a name="ref674"></a>674</td><td>Sanderson, K. E., and J. R. Roth <i>1988</i>. Linkage map of <i>Salmonella typhimurium,</i> edition VII. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Linkage map of Salmonella typhimurium, edition VII">google scholar</a></td></tr>
<tr><td><a name="ref675"></a>675</td><td>Sanderson, K. E., and B. A. D. Stocker <i>1981</i>. Gene <i>rfaH,</i> which affects lipopolysaccharide core structure in <i>Salmonella typhimurium,</i> is required also for expression of F-factor functions. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Gene rfaH, which affects lipopolysaccharide core structure in Salmonella typhimurium, is required also for expression of F-factor functions">google scholar</a></td></tr>
<tr><td><a name="ref676"></a>676</td><td>Sanderson, K. E., J. Van Wyngaarten, O. Luderitz, and B. A. D. Stocker <i>1974</i>. Rough mutants of <i>Salmonella typhimurium</i> with defects in the heptose region of the lipopolysaccharide core. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Rough mutants of Salmonella typhimurium with defects in the heptose region of the lipopolysaccharide core">google scholar</a></td></tr>
<tr><td><a name="ref677"></a>677</td><td>Santangelo, J. D., D. T. Jones, and D. R. Woods <i>1991</i>. Metronidazole activation and isolation of <i>Clostridium</i> <i>acetobutylicum</i> electron transport genes. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Metronidazole activation and isolation of Clostridium acetobutylicum electron transport genes">google scholar</a></td></tr>
<tr><td><a name="ref678"></a>678</td><td>Sarrhy, A., S. Michaels, and J. Beckwith <i>1981</i>. Deletion map of the <i>Escherichia coli</i> structural gene for alkaline phosphatase, <i>phoA.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Deletion map of the Escherichia coli structural gene for alkaline phosphatase, phoA.">google scholar</a></td></tr>
<tr><td><a name="ref679"></a>679</td><td>Sasarman, A., P. Chartrand, M. Lavoie, D. Tardif, R. Proschek, and C. Lapointe <i>1979</i>. Mapping of a new <i>hem</i> gene in <i>Escherichia coli</i> K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mapping of a new hem gene in Escherichia coli K12">google scholar</a></td></tr>
<tr><td><a name="ref680"></a>680</td><td>Sasarman, A., P. Chartrand, R. Proschek, M. Desrochers, D. Tardif, and C. Lapointe <i>1975</i>. Uroporphyrin-accumulating mutant of <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Uroporphyrin-accumulating mutant of Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref681"></a>681</td><td>Sasarman, A., and M. Desrochers <i>1976</i>. Uroporphyrinogen III cosynthase-deficient mutant of <i>Salmonella typhimurium</i> LT2. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Uroporphyrinogen III cosynthase-deficient mutant of Salmonella typhimurium LT2">google scholar</a></td></tr>
<tr><td><a name="ref682"></a>682</td><td>Sasarman, A., M. Desrochers, S. Sonea, K. E. Sanderson, and M. Surdeanu <i>1976</i>. Porphobilogen-accumulating mutants of <i>Salmonella typhimurium</i> LT2. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Porphobilogen-accumulating mutants of Salmonella typhimurium LT2">google scholar</a></td></tr>
<tr><td><a name="ref683"></a>683</td><td>Sasarman, A., K. E. Sanderson, M. Surdeanu, and S. Sonea <i>1970</i>. Hemin-deficient mutants of <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Hemin-deficient mutants of Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref684"></a>684</td><td>Sastry, S. S., and R. Jayaraman <i>1984</i>. Nitrofurantoin-resistant mutants of <i>Escherichia coli:</i> isolation and mapping. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Nitrofurantoin-resistant mutants of Escherichia coli: isolation and mapping">google scholar</a></td></tr>
<tr><td><a name="ref685"></a>685</td><td>Satre, M., C. Klein, and P. V. Vignais <i>1978</i>. Isolation of <i>Escherichia coli</i> mutants with an adenosine triphosphatase insensitive to aurovertin. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation of Escherichia coli mutants with an adenosine triphosphatase insensitive to aurovertin">google scholar</a></td></tr>
<tr><td><a name="ref686"></a>686</td><td>Schaefler, S <i>1967</i>. Inducible system for the utilization of β-glucosides in <i>Escherichia coli.</i> I. Active transport and utilization of β-glucosides. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Inducible system for the utilization of β-glucosides in Escherichia coli. I. Active transport and utilization of β-glucosides">google scholar</a></td></tr>
<tr><td><a name="ref687"></a>687</td><td>Schellenberg, G. D., and C. E. Furlong <i>1977</i>. Resolution of the multiplicity of the glutamate and aspartate transport systems of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Resolution of the multiplicity of the glutamate and aspartate transport systems of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref688"></a>688</td><td>Schleif, R <i>1969</i>. Isolation and characterization of a streptolydigin resistant RNA polymerase. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation and characterization of a streptolydigin resistant RNA polymerase">google scholar</a></td></tr>
<tr><td><a name="ref689"></a>689</td><td>Schwartz, J. H., W. K. Mass, and E. J. Simon <i>1959</i>. An impaired concentrating mechanism for amino acids in mutants of <i>Escherichia coli</i> resistant to L-canavanine and D-serine. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=An impaired concentrating mechanism for amino acids in mutants of Escherichia coli resistant to L-canavanine and D-serine">google scholar</a></td></tr>
<tr><td><a name="ref690"></a>690</td><td>Schwarz, T. F. R., S. M. Yeats, and P. Connolly <i>1981</i>. Altered transcriptional termination in a rifampicin-resistant mutant of <i>Escherichia coli</i> which inhibits the growth of bacteriophage T7. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Altered transcriptional termination in a rifampicin-resistant mutant of Escherichia coli which inhibits the growth of bacteriophage T7">google scholar</a></td></tr>
<tr><td><a name="ref691"></a>691</td><td>Schweizer, H., M. Argast, and W. Boos <i>1982</i>. Characteristics of a binding protein-dependent transport system for <i>sn</i>-glycerol 3-phosphate in <i>Escherichia coli</i> that is part of the <i>pho</i> regulon. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Characteristics of a binding protein-dependent transport system for sn-glycerol 3-phosphate in Escherichia coli that is part of the pho regulon">google scholar</a></td></tr>
<tr><td><a name="ref692"></a>692</td><td>Schweizer, H., T. Grussenmeyer, and W. Boos <i>1982</i>. Mapping of two <i>ugp</i> genes coding for the <i>pho</i> regulon-dependent sn-glycerol 3-phosphate transport system of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mapping of two ugp genes coding for the pho regulon-dependent sn-glycerol 3-phosphate transport system of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref693"></a>693</td><td>Sedgwick, B., and P. Robins <i>1980</i>. Isolation of mutants of <i>Escherichia coli</i> with increased resistance to alkylating agents: mutants deficient in thiols and mutants constitutive for the adaptive response. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation of mutants of Escherichia coli with increased resistance to alkylating agents: mutants deficient in thiols and mutants constitutive for the adaptive response">google scholar</a></td></tr>
<tr><td><a name="ref694"></a>694</td><td>Servin-Gonzalez, L., M. Ortiz, A. Gonzalez, and F. Bastarrachea <i>1987</i>. <i>glnA</i> mutations conferring resistance to methylammonium in <i>Escherichia coli</i> K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=glnA mutations conferring resistance to methylammonium in Escherichia coli K12">google scholar</a></td></tr>
<tr><td><a name="ref695"></a>695</td><td>Shakulov, R. S., V. A. Livshits, G. G. Zaigraeva, L. F. Lideman, and T. T. D. Mukhamed <i>1978</i>. Partial suppression of the action of the gene <i>relA</i> in cells of <i>fus</i>r mutants of <i>Escherichia coli </i>K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Partial suppression of the action of the gene relA in cells of fusr mutants of Escherichia coli K12">google scholar</a></td></tr>
<tr><td><a name="ref696"></a>696</td><td>Shaw, L, F. Grau, H. R. Kaback, J. S. Hong, and C. Walsh <i>1975</i>. Vinylglycolate resistance in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Vinylglycolate resistance in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref697"></a>697</td><td>Sheldon, R., and S. Brenner <i>1976</i>. Regulatory mutants of dihydrofolate reductase in <i>Escherichia</i> <i>coli</i> K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulatory mutants of dihydrofolate reductase in Escherichia coli K12">google scholar</a></td></tr>
<tr><td><a name="ref698"></a>698</td><td>Sheppard, D. E <i>1964</i>. Mutants of <i>Salmonella typhimurium</i> resistant to feedback inhibition by L- histidine. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutants of Salmonella typhimurium resistant to feedback inhibition by L- histidine">google scholar</a></td></tr>
<tr><td><a name="ref699"></a>699</td><td>Shifrin, S., B. N. Ames, and G. F.-L. Ames <i>1966</i>. Effect of the α-hydrazino analogue of histidine on histidine transport and arginine biosynthesis. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Effect of the α-hydrazino analogue of histidine on histidine transport and arginine biosynthesis">google scholar</a></td></tr>
<tr><td><a name="ref700"></a>700</td><td>Shimmin, L. C., D. Vanderwel, R. E. Harkness, B. R. Currie, C. A. Galloway, and E. E. Ishiguro <i>1984</i>. Temperature-sensitive beta-lactam-tolerant mutants of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Temperature-sensitive beta-lactam-tolerant mutants of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref701"></a>701</td><td>Shive, W., and J. Macow <i>1946</i>. Biochemical transformations as determined by competitive analogue-metabolite growth inhibitions. I. Some transformations involving aspartic acid. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Biochemical transformations as determined by competitive analogue-metabolite growth inhibitions. I. Some transformations involving aspartic acid">google scholar</a></td></tr>
<tr><td><a name="ref702"></a>702</td><td>Siden, I., and H. G. Boman <i>1983</i>. <i>Escherichia coli</i> mutants with an altered sensitivity to cecropin D. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Escherichia coli mutants with an altered sensitivity to cecropin D">google scholar</a></td></tr>
<tr><td><a name="ref703"></a>703</td><td>Siegel, E. C., and V. Bryson <i>1967</i>. Mutator gene of <i>Escherichia coli</i> B. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutator gene of Escherichia coli B">google scholar</a></td></tr>
<tr><td><a name="ref704"></a>704</td><td>Sigmund, C. D., M. Ettayebi, and E. A. Morgan <i>1984</i>. Antibiotic resistance mutations in 16s and 23s ribosomal RNA genes of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Antibiotic resistance mutations in 16s and 23s ribosomal RNA genes of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref705"></a>705</td><td>Signer, C. E., G. R. Smith, R. Cortese, and B. N. Ames <i>1972</i>. Mutant tRNAhis ineffective in repression and lacking two pseudouridine modifications. <i>Nature</i>  </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutant tRNAhis ineffective in repression and lacking two pseudouridine modifications. Nature">google scholar</a></td></tr>
<tr><td><a name="ref706"></a>706</td><td>Silbert, D. F., G. R. Fink, and B. N. Ames <i>1966</i>. Histidine regulatory mutants in <i>Salmonella</i> <i>typhimurium.</i> III. A class of regulatory mutants deficient in tRNA for histidine. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Histidine regulatory mutants in Salmonella typhimurium. III. A class of regulatory mutants deficient in tRNA for histidine">google scholar</a></td></tr>
<tr><td><a name="ref707"></a>707</td><td>Silhavy, T. J., I. Hartig-Beecken, and W. Boos <i>1976</i>. Periplasmic protein related to the <i>sn</i>- glycerol 3-phosphate transport system of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Periplasmic protein related to the sn- glycerol 3-phosphate transport system of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref708"></a>708</td><td>Silver, S., P. Johnseine, E. Whitney, and D. Clark <i>1972</i>. Manganese-resistant mutants of <i>Escherichia coli:</i> physiological and genetic studies. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Manganese-resistant mutants of Escherichia coli: physiological and genetic studies">google scholar</a></td></tr>
<tr><td><a name="ref709"></a>709</td><td>Silverman, M., and M. Simon <i>1973</i>. Genetic analysis of flagellar mutants in <i>Escherichia coli.</i> <i>J.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic analysis of flagellar mutants in Escherichia coli. J.">google scholar</a></td></tr>
<tr><td><a name="ref710"></a>710</td><td>Silverman, M., and M. I. Simon <i>1977</i>. Bacterial flagella. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Bacterial flagella">google scholar</a></td></tr>
<tr><td><a name="ref711"></a>711</td><td>Silverman, P., K. Nat, J. McEwen, and R. Birchman <i>1980</i>. Selection of <i>Escherichia coli</i> K-12 mutants that prevent expression of F-plasmid functions. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Selection of Escherichia coli K-12 mutants that prevent expression of F-plasmid functions">google scholar</a></td></tr>
<tr><td><a name="ref712"></a>712</td><td>Silverstone, A. E., M. Goman, and J. G. Scaife <i>1972</i>. ALT: a new factor involved in the synthesis of RNA by <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=ALT: a new factor involved in the synthesis of RNA by Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref713"></a>713</td><td>Simons, R W., P. A. Egan, H. T. Chute, and W. D. Nunn <i>1980</i>. Regulation of fatty acid degradation in <i>Escherichia coli:</i> isolation and characterization of strains bearing insertion and temperature-sensitive mutations in <i>fadR.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation of fatty acid degradation in Escherichia coli: isolation and characterization of strains bearing insertion and temperature-sensitive mutations in fadR.">google scholar</a></td></tr>
<tr><td><a name="ref714"></a>714</td><td>Sivasubramanian, N., and R. Jayaraman <i>1980</i>. Mapping of two transcription mutations <i>(tlnI</i> and <i>tlnII</i>) conferring thiolutin resistance, adjacent to <i>dnaZ</i> and <i>rho</i> in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mapping of two transcription mutations (tlnI and tlnII) conferring thiolutin resistance, adjacent to dnaZ and rho in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref715"></a>715</td><td>Slater, A. C., M. C. Jones-Mortimer, and H. L Kornberg <i>1981</i>. L-sorbose phosphorylation in <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=L-sorbose phosphorylation in Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref716"></a>716</td><td>Slayman, C. W <i>1973</i>. The genetic control of membrane transport. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The genetic control of membrane transport">google scholar</a></td></tr>
<tr><td><a name="ref717"></a>717</td><td>Sledziewska, E., and D. Hulanicka <i>1978</i>. Method of isolation of cysteine constitutive mutants of the cysteine regulon in <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Method of isolation of cysteine constitutive mutants of the cysteine regulon in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref718"></a>718</td><td>Smith, D. R., and J. M. Calvo <i>1979</i>. Regulation of dihydrofolate reductase EC-1.5.1.3 synthesis in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation of dihydrofolate reductase EC-1.5.1.3 synthesis in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref719"></a>719</td><td>Smith, D. R., J. I. Rood, P. I. Bird, M. K, Sneddon, J. M. Calvo, and J. F. Morrison <i>1982</i>. Amplification and modification of dihydrofolate reductase EC-1.5.1.3 in <i>Escherichia coli</i> nucleotide sequence of <i>fol</i> genes from mutationally altered plasmids. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Amplification and modification of dihydrofolate reductase EC-1.5.1.3 in Escherichia coli nucleotide sequence of fol genes from mutationally altered plasmids">google scholar</a></td></tr>
<tr><td><a name="ref720"></a>720</td><td>Smith, J. M., F. J. Smith, and H. E. Umbarger <i>1979</i>. Mutations affecting the formation of acetohydroxyacid synthase II in <i>Escherichia coli</i> K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutations affecting the formation of acetohydroxyacid synthase II in Escherichia coli K12">google scholar</a></td></tr>
<tr><td><a name="ref721"></a>721</td><td>Smith, T. F., and J. R. Sadler <i>1971</i>. The nature of lactose operator constitutive mutations. <i>J.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The nature of lactose operator constitutive mutations. J.">google scholar</a></td></tr>
<tr><td><a name="ref722"></a>722</td><td>Solomon, E., and E. C. C. Lin <i>1972</i>. Mutations affecting the dissimilation of mannitol by <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutations affecting the dissimilation of mannitol by Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref723"></a>723</td><td>Somers, J. M., and W. W. Kay <i>1983</i>. Genetic fine structure of the tricarboxylate transport (<i>tct</i>) locus of <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic fine structure of the tricarboxylate transport (tct) locus of Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref724"></a>724</td><td>Somers, J. M., G. D. Sweet, and W. W. Kay <i>1981</i>. Fluorocitrate resistant tricarboxylate transport mutants of <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Fluorocitrate resistant tricarboxylate transport mutants of Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref725"></a>725</td><td>Sparling, P. F., and E. Blackman <i>1973</i>. Mutation to erythromycin dependence in <i>Escherichia</i> <i>coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutation to erythromycin dependence in Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref726"></a>726</td><td>Sparling, P. F., Y. Ikeya, and D. Elliot <i>1973</i>. Two genetic loci for resistance to kasugamycin in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Two genetic loci for resistance to kasugamycin in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref727"></a>727</td><td>Sprague, G. F., Jr., R. M. Bell, and J. E. Cronan, Jr <i>1975</i>. A mutant of <i>Escherichia coli</i> auxotrophic for organic phosphates: evidence for two defects in inorganic phosphate transport. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A mutant of Escherichia coli auxotrophic for organic phosphates: evidence for two defects in inorganic phosphate transport">google scholar</a></td></tr>
<tr><td><a name="ref728"></a>728</td><td>Springer, S. E., and R. E. Huber <i>1973</i>. Sulfate and selenate uptake and transport in wild and in two selenate-tolerant strains of <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Sulfate and selenate uptake and transport in wild and in two selenate-tolerant strains of Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref729"></a>729</td><td>Sprinson, D. B., E. G. Gollub, R. C. Hu, and K.-P. Liu <i>1976</i>. Regulation of tyrosine and phenylalanine biosynthesis in <i>Salmonella.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation of tyrosine and phenylalanine biosynthesis in Salmonella.">google scholar</a></td></tr>
<tr><td><a name="ref730"></a>730</td><td>Stacey, K. A., and E. Simson <i>1965</i>. Improved method for the isolation of thymine-requiring mutants of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Improved method for the isolation of thymine-requiring mutants of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref731"></a>731</td><td>Stalker, D. M., W. R. Hiatt, and L. Comai <i>1985</i>. A single amino acid substitution in the enzyme 5-enolpyruvylshikimate-3-phosphate synthase confers resistance to the herbicide glyphosate. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A single amino acid substitution in the enzyme 5-enolpyruvylshikimate-3-phosphate synthase confers resistance to the herbicide glyphosate">google scholar</a></td></tr>
<tr><td><a name="ref732"></a>732</td><td>Stalmach, M. E., S. Grothe, and J. M. Wood <i>1983</i>. Two proline porters in <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Two proline porters in Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref733"></a>733</td><td>Staskawicz, B. J., and N. J. Panopoulos <i>1980</i>. Phaseolotoxin transport in <i>Escherichia coli</i> and <i>Salmonella typhimurium</i> via the oligopeptide permease. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Phaseolotoxin transport in Escherichia coli and Salmonella typhimurium via the oligopeptide permease">google scholar</a></td></tr>
<tr><td><a name="ref734"></a>734</td><td>Staudenbauer, W. L <i>1976</i>. Replication of <i>Escherichia coli</i> DNA <i>in vitro:</i> inhibition by oxolinic acid. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Replication of Escherichia coli DNA in vitro: inhibition by oxolinic acid">google scholar</a></td></tr>
<tr><td><a name="ref735"></a>735</td><td>Stauffer, G. V., and J. E. Brenchley <i>1977</i>. Influence of methionine biosynthesis on serine transhydroxymethylase (EC 2.1.2.1) regulation in <i>Salmonella typhimurium</i> LT2. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Influence of methionine biosynthesis on serine transhydroxymethylase (EC 2.1.2.1) regulation in Salmonella typhimurium LT2">google scholar</a></td></tr>
<tr><td><a name="ref736"></a>736</td><td>Steffes, C., J. Ellis, J. Wu, and B. P. Rosen <i>1992</i>. The <i>lysP</i> gene encodes the lysine-specific permease. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The lysP gene encodes the lysine-specific permease">google scholar</a></td></tr>
<tr><td><a name="ref737"></a>737</td><td>Stern, J. R., and R. W. O’Brien <i>1969</i>. Oxidation of D-malic and β-alkyl malic acids by wild- type and mutant strains of <i>Salmonella typhimurium</i> and by <i>Aerobacter aerogenes.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Oxidation of D-malic and β-alkyl malic acids by wild- type and mutant strains of Salmonella typhimurium and by Aerobacter aerogenes.">google scholar</a></td></tr>
<tr><td><a name="ref738"></a>738</td><td>Stevens, F. J., and T. T. Wu <i>1976</i>. Growth on D-lyxose of a mutant strain of <i>Escherichia coli</i> K12 using a novel isomerase and enzymes related to D-xylose metabolism. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Growth on D-lyxose of a mutant strain of Escherichia coli K12 using a novel isomerase and enzymes related to D-xylose metabolism">google scholar</a></td></tr>
<tr><td><a name="ref739"></a>739</td><td>Stewart, V., and C. H. MacGregor <i>1982</i>. Nitrate reductase in <i>Escherichia coli</i> K-12: involvement of <i>chlC, chlE,</i> and <i>chlG</i> loci. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Nitrate reductase in Escherichia coli K-12: involvement of chlC, chlE, and chlG loci">google scholar</a></td></tr>
<tr><td><a name="ref740"></a>740</td><td>Stitt, B. L., H. R. Revel, I. Lielausis, and W. B. Wood <i>1980</i>. Role of the host cell in bacteriophage T4 development. II. Characterization of host mutants that have pleiotropic effects on T4 growth. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Role of the host cell in bacteriophage T4 development. II. Characterization of host mutants that have pleiotropic effects on T4 growth">google scholar</a></td></tr>
<tr><td><a name="ref741"></a>741</td><td>Stocker, B. A. D., M. Nurminen, and P. H. Makela <i>1979</i>. Mutants defective in the 33K outer membrane protein of <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutants defective in the 33K outer membrane protein of Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref742"></a>742</td><td>Stoeber, F., A. Lagarde, G. Nemoz, G. Novel, M. Novel, R Portalier, J. Pouyssegur, and J. Robert-Baudouy <i>1974</i>. Le métabolisme des hexuronides et des hexuronates chez <i>Escherichia coli </i>K12: aspects physiologiques et génétiques de sa régulation. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Le métabolisme des hexuronides et des hexuronates chez Escherichia coli K12: aspects physiologiques et génétiques de sa régulation">google scholar</a></td></tr>
<tr><td><a name="ref743"></a>743</td><td>Stouthamer, A. H <i>1969</i>. A genetical and biochemical study of chlorate-resistant mutants of <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A genetical and biochemical study of chlorate-resistant mutants of Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref744"></a>744</td><td>Stouthamer, A. H., and C. W. Bettenhaussen <i>1970</i>. Mapping a gene causing resistance to chlorate in <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mapping a gene causing resistance to chlorate in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref745"></a>745</td><td>Straus, D. S., and G. R. Hoffman <i>1975</i>. Selection for a large genetic duplication in <i>Salmonella</i> <i>typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Selection for a large genetic duplication in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref746"></a>746</td><td>Stuttard, C <i>1972</i>. Location of <i>trpR</i> mutations in the <i>serB-thr</i> region of <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Location of trpR mutations in the serB-thr region of Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref747"></a>747</td><td>Sullivan, M. A., and R. M. Bock <i>1985</i>. Isolation and characterization of antisuppressor mutations in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation and characterization of antisuppressor mutations in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref748"></a>748</td><td>Sullivan, M. A., J. F. Cannon, F. H. Webb, and R. M. Bock <i>1985</i>. Antisuppressor mutation in <i>Escherichia coli</i> defective in biosynthesis of 5-methylaminomethyl-2-thiouridine. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Antisuppressor mutation in Escherichia coli defective in biosynthesis of 5-methylaminomethyl-2-thiouridine">google scholar</a></td></tr>
<tr><td><a name="ref749"></a>749</td><td>Summers, W. C., and P. Raskin <i>1993</i>. A method for selection of mutations at the <i>tdk</i> locus in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A method for selection of mutations at the tdk locus in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref750"></a>750</td><td>Sun, T.-P., and R. E. Webster <i>1986</i>. <i>fii,</i> a bacterial locus required for filamentous phage infection, and its relation to colicin-tolerant <i>tolA</i> and <i>tolB.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=fii, a bacterial locus required for filamentous phage infection, and its relation to colicin-tolerant tolA and tolB.">google scholar</a></td></tr>
<tr><td><a name="ref751"></a>751</td><td>Sun, T.-P., and R. E. Webster <i>1987</i>. Nucleotide sequence of a gene cluster involved in entry of E colicins and single-stranded DNA of infecting filamentous bacteriophages into <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Nucleotide sequence of a gene cluster involved in entry of E colicins and single-stranded DNA of infecting filamentous bacteriophages into Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref752"></a>752</td><td>Sundararajan, T. A <i>1963</i>. Interference with glycerokinase induction in mutants of <i>E. coli</i> accumulating gal-1-P. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Interference with glycerokinase induction in mutants of E. coli accumulating gal-1-P">google scholar</a></td></tr>
<tr><td><a name="ref753"></a>753</td><td>Sunshine, M., M. Feiss, J. Stuart, and J. Yochem <i>1977</i>. A new host gene <i>(groPC)</i> necessary for lambda DNA replication. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A new host gene (groPC) necessary for lambda DNA replication">google scholar</a></td></tr>
<tr><td><a name="ref754"></a>754</td><td>Sunshine, M. G., and B. Sauer <i>1975</i>. A bacterial mutation blocking P2 phage late gene expression. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A bacterial mutation blocking P2 phage late gene expression">google scholar</a></td></tr>
<tr><td><a name="ref755"></a>755</td><td>Sutton, A., T. Newman, M. Francis, and M. Freundlich <i>1981</i>. Valine-resistant <i>Escherichia coli</i> K-12 strains with mutations in the <i>ilvB</i> operon. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Valine-resistant Escherichia coli K-12 strains with mutations in the ilvB operon">google scholar</a></td></tr>
<tr><td><a name="ref756"></a>756</td><td>Swedberg, C., S. Castensson, and O. Skold <i>1979</i>. Characterization of mutationally altered dihydropteroate synthase and its ability to form a sulfonamide-containing dihydrofolate analog. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Characterization of mutationally altered dihydropteroate synthase and its ability to form a sulfonamide-containing dihydrofolate analog">google scholar</a></td></tr>
<tr><td><a name="ref757"></a>757</td><td>Szentirmai, A., M. Szentirmai, and H. E. Umbarger <i>1968</i>. Isoleucine and valine metabolism of <i>Escherichia coli.</i> XV. Biochemical properties of mutants resistant to thiaisoleucine. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isoleucine and valine metabolism of Escherichia coli. XV. Biochemical properties of mutants resistant to thiaisoleucine">google scholar</a></td></tr>
<tr><td><a name="ref758"></a>758</td><td>Takahashi, H <i>1978</i>. Genetic and physiological characterization of <i>Escherichia coli</i> K12 mutants <i>(tabC)</i> which induce the abortive infection of bacteriophage T4. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic and physiological characterization of Escherichia coli K12 mutants (tabC) which induce the abortive infection of bacteriophage T4">google scholar</a></td></tr>
<tr><td><a name="ref759"></a>759</td><td>Tamaki, S., H. Matsuzawa, and M. Matsuhashi <i>1980</i>. Cluster of <i>mrdA</i> and <i>mrdB</i> genes responsible for the rod shape and mecillinam sensitivity of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Cluster of mrdA and mrdB genes responsible for the rod shape and mecillinam sensitivity of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref760"></a>760</td><td>Tamaki, S., T. Sato, and M. Matsuhashi <i>1971</i>. Role of lipopolysaccharides in antibiotic resistance and bacteriophage adsorption of <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Role of lipopolysaccharides in antibiotic resistance and bacteriophage adsorption of Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref761"></a>761</td><td>Tanaka, N., G. Kawano, and T. Kinoshita <i>1971</i>. Chromosomal location of a fusidic acid resistant marker in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Chromosomal location of a fusidic acid resistant marker in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref762"></a>762</td><td>Tang, C.-T., R. Engel, and B. E. Tropp <i>1977</i>. L-Glyceraldehyde 3-phosphate, a bactericidal agent. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=L-Glyceraldehyde 3-phosphate, a bactericidal agent">google scholar</a></td></tr>
<tr><td><a name="ref763"></a>763</td><td>Tapio, S., and L. A. Isaksson <i>1990</i>. Antisuppression by mutations in elongation factor Tu. <i>Eur.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Antisuppression by mutations in elongation factor Tu. Eur.">google scholar</a></td></tr>
<tr><td><a name="ref764"></a>764</td><td>Thakar, J. H., and G. P. Kalle <i>1968</i>. Defective guanine uptake in an 8-azaguanine-resistant mutant of <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Defective guanine uptake in an 8-azaguanine-resistant mutant of Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref765"></a>765</td><td>Thèze, J., D. Margarita, G. N. Cohen, F. Borne, and J. C. Patte <i>1974</i>. Mapping of the structural genes of the three aspartokinases and the two homoserine dehydrogenases of <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mapping of the structural genes of the three aspartokinases and the two homoserine dehydrogenases of Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref766"></a>766</td><td>Thirion, J. P., and M. Hofnung <i>1982</i>. On some genetic aspects of phage λ resistance in <i>E. coli</i> K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=On some genetic aspects of phage λ resistance in E. coli K12">google scholar</a></td></tr>
<tr><td><a name="ref767"></a>767</td><td>Thomas, G., and A. Favre <i>1977</i>. Genetic mapping of a mutant exhibiting no near ultraviolet induced growth delay and lacking 4-thiouridine in its tRNA. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic mapping of a mutant exhibiting no near ultraviolet induced growth delay and lacking 4-thiouridine in its tRNA">google scholar</a></td></tr>
<tr><td><a name="ref768"></a>768</td><td>Thorbjarnardottir, S. H., R. A. Magnusdottir, and C. Eggertsson <i>1978</i>. Mutations determining generalized resistance to aminoglycoside antibiotics in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutations determining generalized resistance to aminoglycoside antibiotics in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref769"></a>769</td><td>Thorne, C. M., and L. M. Corwin <i>1975</i>. Mutations affecting aromatic amino acid transport in <i>Escherichia coli</i> and <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutations affecting aromatic amino acid transport in Escherichia coli and Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref770"></a>770</td><td>Tomas, J. M., and W. W. Kay <i>1986</i>. Tellurite susceptibility and non-plasmid-mediated resistance in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Tellurite susceptibility and non-plasmid-mediated resistance in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref771"></a>771</td><td>Tommassen, J., and B. Lugtenberg <i>1981</i>. Localization of <i>phoE,</i> the structural gene for outer membrane protein E in <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Localization of phoE, the structural gene for outer membrane protein E in Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref772"></a>772</td><td>Toone, W. M., K. E. Rudd, and J. D. Friesen <i>1992</i>. Mutations causing aminotriazole resistance and temperature sensitivity reside in <i>gyrB,</i> which encodes the β subunit of DNA gyrase. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutations causing aminotriazole resistance and temperature sensitivity reside in gyrB, which encodes the β subunit of DNA gyrase">google scholar</a></td></tr>
<tr><td><a name="ref773"></a>773</td><td>Torriani, A., and F. Rothman <i>1961</i>. Mutants of <i>Escherichia coli</i> constitutive for alkaline phosphatase. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutants of Escherichia coli constitutive for alkaline phosphatase">google scholar</a></td></tr>
<tr><td><a name="ref774"></a>774</td><td>Tosa, T., and L. I. Pizer <i>1971</i>. Biochemical bases for the antimetabolite action of L-serine hydroxamate. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Biochemical bases for the antimetabolite action of L-serine hydroxamate">google scholar</a></td></tr>
<tr><td><a name="ref775"></a>775</td><td>Tristram, H., and S. Neale <i>1968</i>. The activity and specificity of the proline permease in wild- type and analogue-resistant strains of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The activity and specificity of the proline permease in wild- type and analogue-resistant strains of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref776"></a>776</td><td>Trun, N. J., and S. Gottesman <i>1990</i>. On the bacterial cell cycle: <i>Escherichia coli</i> mutants with altered ploidy. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=On the bacterial cell cycle: Escherichia coli mutants with altered ploidy">google scholar</a></td></tr>
<tr><td><a name="ref777"></a>777</td><td>Trun, N. J., and S. Gottesman <i>1991</i>. Characterization of <i>Escherichia coli</i> mutants with altered ploidy. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Characterization of Escherichia coli mutants with altered ploidy">google scholar</a></td></tr>
<tr><td><a name="ref778"></a>778</td><td>Tsay, J.-T., C. O. Rock, and S. Jackowski <i>1992</i>. Overproduction of beta ketoacyl-acyl carrier protein synthase I imparts thiolactomycin resistance to <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Overproduction of beta ketoacyl-acyl carrier protein synthase I imparts thiolactomycin resistance to Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref779"></a>779</td><td>Udaka, S., and H. S. Moyed <i>1963</i>. Inhibition of parental and mutant xanthosine 5′-phosphate aminases by psicofuranine. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Inhibition of parental and mutant xanthosine 5′-phosphate aminases by psicofuranine">google scholar</a></td></tr>
<tr><td><a name="ref780"></a>780</td><td>Umbarger, H. E <i>1969</i>. Regulation of the biosynthesis of the branched-chain amino acids. <i>Curr.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulation of the biosynthesis of the branched-chain amino acids. Curr.">google scholar</a></td></tr>
<tr><td><a name="ref781"></a>781</td><td>Umbarger, H. E <i>1971</i>. Metabolite analogs as genetic and biochemical probes. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Metabolite analogs as genetic and biochemical probes">google scholar</a></td></tr>
<tr><td><a name="ref782"></a>782</td><td>Umbarger, H. E <i>1987</i>. Biosynthesis of the branched-chain amino acids.</td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Biosynthesis of the branched-chain amino acids">google scholar</a></td></tr>
<tr><td><a name="ref783"></a>783</td><td>Uzan, M., and A. Danchin <i>1976</i>. A rapid test for the <i>relA</i> mutation in <i>E. coli.</i> <i>Biochem. Biophys.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A rapid test for the relA mutation in E. coli. Biochem. Biophys.">google scholar</a></td></tr>
<tr><td><a name="ref784"></a>784</td><td>Uzan, M., and A. Danchin <i>1978</i>. Correlation between the serine sensitivity and the derepressibility of the <i>ilv</i> genes in <i>Escherichia coli relA</i> mutants. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Correlation between the serine sensitivity and the derepressibility of the ilv genes in Escherichia coli relA mutants">google scholar</a></td></tr>
<tr><td><a name="ref785"></a>785</td><td>Vaara, M <i>1981</i>. Increased outer membrane resistance to ethylenediamine-tetraacetate and cations in novel lipid A mutants. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Increased outer membrane resistance to ethylenediamine-tetraacetate and cations in novel lipid A mutants">google scholar</a></td></tr>
<tr><td><a name="ref786"></a>786</td><td>Vaara, M., and T. Vaara <i>1981</i>. Outer membrane permeability barrier disruption by polymyxin in polymyxin-susceptible and resistant <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Outer membrane permeability barrier disruption by polymyxin in polymyxin-susceptible and resistant Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref787"></a>787</td><td>Vaara, M., T. Vaara, M. Jensen, I. Helander, M. Nurminen, E. T. Rietschel, and P. H. Makela <i>1981</i>. Characterization of the lipopolysaccharide from the polymyxin-resistant <i>pmrA</i> mutants of <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Characterization of the lipopolysaccharide from the polymyxin-resistant pmrA mutants of Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref788"></a>788</td><td>Vaara, M., T. Vaara, and M. Sarvas <i>1979</i>. Decreased binding of polymyxin by polymyxin- resistant mutants of <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Decreased binding of polymyxin by polymyxin- resistant mutants of Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref789"></a>789</td><td>Vallari, D. S., and S. Jackowski <i>1988</i>. Biosynthesis and degradation both contribute to the regulation of coenzyme A content in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Biosynthesis and degradation both contribute to the regulation of coenzyme A content in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref790"></a>790</td><td>Vallari, D. S., and C. O. Rock <i>1987</i>. Isolation and characterization of temperature-sensitive pantothenate kinase (<i>coaA</i>) mutants of <i>Escherichia</i> <i>coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation and characterization of temperature-sensitive pantothenate kinase (coaA) mutants of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref791"></a>791</td><td>Van Alphen, L., B. Lugtenberg, R. Van Boxtel, A.-M. Hack, C. Verhoef, and L. Havekes <i>1979</i>. <i>meoA</i> is the structural gene for outer membrane protein C of <i>Escherichia coli</i> K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=meoA is the structural gene for outer membrane protein C of Escherichia coli K12">google scholar</a></td></tr>
<tr><td><a name="ref792"></a>792</td><td>Van Alphen, W., B. Lugtenberg, and W. Berendsen <i>1976</i>. Heptose-deficient mutants of <i>Escherichia coli</i> K12 deficient in up to three major outer membrane proteins. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Heptose-deficient mutants of Escherichia coli K12 deficient in up to three major outer membrane proteins">google scholar</a></td></tr>
<tr><td><a name="ref793"></a>793</td><td>Van Buul, C. P. J. J., and P. H. Van Knippenberg <i>1985</i>. Nucleotide sequence of the <i>ksgA</i> gene of <i>Escherichia coli</i>—comparison of methyltransferases effecting dimethylation of adenosine in ribosomal RNA. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Nucleotide sequence of the ksgA gene of Escherichia coli—comparison of methyltransferases effecting dimethylation of adenosine in ribosomal RNA">google scholar</a></td></tr>
<tr><td><a name="ref794"></a>794</td><td>Van de Klundert, J. A. M., E. Den Turk, A. H. Borman, P. H. Van der Meide, and L. Bosch <i>1977</i>. Isolation and characterization of a mocimycin resistant mutant of <i>Escherichia coli</i> with an altered elongation factor EF-Tu. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation and characterization of a mocimycin resistant mutant of Escherichia coli with an altered elongation factor EF-Tu">google scholar</a></td></tr>
<tr><td><a name="ref795"></a>795</td><td>Van de Klundert, J. A. M., P. H. Van der Meide, P. Van de Putte, and L. Bosch <i>1978</i>. Mutants of <i>Escherichia coli</i> altered in both genes coding for the elongation factor Tu. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutants of Escherichia coli altered in both genes coding for the elongation factor Tu">google scholar</a></td></tr>
<tr><td><a name="ref796"></a>796</td><td>Van Dyk, T. K., and R. A. LaRossa <i>1986</i>. Sensitivity of a <i>Salmonella</i> <i>typhimurium</i> <i>aspC</i> mutant to sulfometuron methyl, a potent inhibitor of acetolactate synthase II. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Sensitivity of a Salmonella typhimurium aspC mutant to sulfometuron methyl, a potent inhibitor of acetolactate synthase II">google scholar</a></td></tr>
<tr><td><a name="ref797"></a>797</td><td>Van Dyk, T. K., and R. A. LaRossa <i>1987</i>. Involvement of <i>ack-pta</i> operon products in α- ketobutyrate metabolism by <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Involvement of ack-pta operon products in α- ketobutyrate metabolism by Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref798"></a>798</td><td>Vasiljevic, B., and L. Topisirovic <i>1987</i>. Properties of ribosomes from neamine-dependent (NeaD) mutants of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Properties of ribosomes from neamine-dependent (NeaD) mutants of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref799"></a>799</td><td>Venables, W. A., and J. R. Guest <i>1968</i>. Transduction of nitrate reductase loci of <i>Escherichia coli</i> by phages Pl and λ. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Transduction of nitrate reductase loci of Escherichia coli by phages Pl and λ">google scholar</a></td></tr>
<tr><td><a name="ref800"></a>800</td><td>Venkateswaran, P. S., and H. C. Wu <i>1972</i>. Isolation and characterization of a phosphonomycin- resistant mutant of <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation and characterization of a phosphonomycin- resistant mutant of Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref801"></a>801</td><td>Verhof, C., B. Lugtenberg, R. van Boxtel, P. de Craaff, and H. Verheij <i>1979</i>. Genetics and biochemistry of the peptidoglycan-associated proteins b and c of <i>Escherichia coli</i> K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetics and biochemistry of the peptidoglycan-associated proteins b and c of Escherichia coli K12">google scholar</a></td></tr>
<tr><td><a name="ref802"></a>802</td><td>Vinella, D., R. D’Ari, and P. Bouloc <i>1992</i>. Penicillin-binding protein 2 is dispensable in <i>Escherichia coli</i> when ppGpp synthesis is induced. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Penicillin-binding protein 2 is dispensable in Escherichia coli when ppGpp synthesis is induced">google scholar</a></td></tr>
<tr><td><a name="ref803"></a>803</td><td>Vinopal, R. T <i>1987</i>. Selectable phenotypes.</td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Selectable phenotypes">google scholar</a></td></tr>
<tr><td><a name="ref804"></a>804</td><td>Voll, M. J <i>1967</i>. Polarity in the histidine operon. III. The isolation of prototrophic polar mutations. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Polarity in the histidine operon. III. The isolation of prototrophic polar mutations">google scholar</a></td></tr>
<tr><td><a name="ref805"></a>805</td><td>Voll, M. J., and L. Leive <i>1970</i>. Release of lipopolysaccharide in <i>Escherichia coli</i> resistant to the permeability increase induced by ethylenediaminetetraacetate. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Release of lipopolysaccharide in Escherichia coli resistant to the permeability increase induced by ethylenediaminetetraacetate">google scholar</a></td></tr>
<tr><td><a name="ref806"></a>806</td><td>Vonder Haar, R. A., and H. E. Umbarger <i>1972</i>. Isoleucine and valine metabolism in <i>Escherichia coli.</i> XIX. Inhibition of isoleucine biosynthesis by glycyl-leucine. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isoleucine and valine metabolism in Escherichia coli. XIX. Inhibition of isoleucine biosynthesis by glycyl-leucine">google scholar</a></td></tr>
<tr><td><a name="ref807"></a>807</td><td>Wachi, M., M. Doi, S. Tamaki, W. Park, S. Nakajima-Iijima, and M. Matsuhashi <i>1987</i>. Mutant isolation and molecular cloning of <i>mre</i> genes which determine cell shape sensitivity to mecillinam and amount of penicillin-binding proteins in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutant isolation and molecular cloning of mre genes which determine cell shape sensitivity to mecillinam and amount of penicillin-binding proteins in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref808"></a>808</td><td>Wada, C., and T. Yura <i>1971</i>. Phenethyl alcohol resistance in <i>Escherichia coli.</i> II. Replication of F (fertility) factor in the resistant strain C600. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Phenethyl alcohol resistance in Escherichia coli. II. Replication of F (fertility) factor in the resistant strain C600">google scholar</a></td></tr>
<tr><td><a name="ref809"></a>809</td><td>Wada, C., and T. Yura <i>1974</i>. Phenethyl alcohol resistance in <i>Escherichia coli.</i> III. A temperature-sensitive mutation <i>(dnaP)</i> affecting DNA replication. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Phenethyl alcohol resistance in Escherichia coli. III. A temperature-sensitive mutation (dnaP) affecting DNA replication">google scholar</a></td></tr>
<tr><td><a name="ref810"></a>810</td><td>Walker, G. C <i>1987</i>. The SOS response of <i>Escherichia coli,</i>.</td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The SOS response of Escherichia coli,">google scholar</a></td></tr>
<tr><td><a name="ref811"></a>811</td><td>Wallace, B. J., and J. Pittard <i>1969</i>. Regulator gene controlling enzymes concerned in tyrosine biosynthesis in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Regulator gene controlling enzymes concerned in tyrosine biosynthesis in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref812"></a>812</td><td>Wandersman, C., F. Moreno, and M. Schwartz <i>1980</i>. Pleiotropic mutations rendering <i>Escherichia coli</i> K-12 resistant to bacteriophage TP1. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Pleiotropic mutations rendering Escherichia coli K-12 resistant to bacteriophage TP1">google scholar</a></td></tr>
<tr><td><a name="ref813"></a>813</td><td>Wargel, R. J., C. A. Shadur, and F. C. Neuhaus <i>1971</i>. Mechanism of D-cycloserine action: transport mutants for D-alanine, D-cycloserine, and glycine. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mechanism of D-cycloserine action: transport mutants for D-alanine, D-cycloserine, and glycine">google scholar</a></td></tr>
<tr><td><a name="ref814"></a>814</td><td>Watanabe, H., H. Hashimoto, and S. Mitsuhashi <i>1980</i>. <i>Salmonella typhimurium</i> LT2 mutation affecting the deletion of resistance determinants on R plasmids. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Salmonella typhimurium LT2 mutation affecting the deletion of resistance determinants on R plasmids">google scholar</a></td></tr>
<tr><td><a name="ref815"></a>815</td><td>Watanabe, N.-A., T. Nagasu, K. Katsu, and K. Kitoh <i>1987</i>. E-0702, a new cephalosporin, is incorporated into <i>Escherichia coli</i> cells via the <i>tonB</i>-dependent iron transport system. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=E-0702, a new cephalosporin, is incorporated into Escherichia coli cells via the tonB-dependent iron transport system">google scholar</a></td></tr>
<tr><td><a name="ref816"></a>816</td><td>Watson, C., and K. Paigen <i>1971</i>. Isolation and characterization of an <i>Escherichia coli</i> bacteriophage requiring cell wall galactose. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Isolation and characterization of an Escherichia coli bacteriophage requiring cell wall galactose">google scholar</a></td></tr>
<tr><td><a name="ref817"></a>817</td><td>Webb, M <i>1970</i>. The mechanism of acquired resistance to Co2+ and Ni2+ in gram-positive and gram-negative bacteria. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The mechanism of acquired resistance to Co2+ and Ni2+ in gram-positive and gram-negative bacteria">google scholar</a></td></tr>
<tr><td><a name="ref818"></a>818</td><td>Weber, J., R. S. F. Lee, E. Grell, and A. E. Senior <i>1992</i>. Investigation of the aurovertin binding site of <i>Escherichia coli</i> F-1 ATPase by fluorescence spectroscopy and site-directed mutagenesis. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Investigation of the aurovertin binding site of Escherichia coli F-1 ATPase by fluorescence spectroscopy and site-directed mutagenesis">google scholar</a></td></tr>
<tr><td><a name="ref819"></a>819</td><td>Wei, C. R., and S. Kustu <i>1981</i>. Glutamine auxotrophs with mutations in a nitrogen regulatory gene, <i>ntrC,</i> that is near <i>glnA.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Glutamine auxotrophs with mutations in a nitrogen regulatory gene, ntrC, that is near glnA.">google scholar</a></td></tr>
<tr><td><a name="ref820"></a>820</td><td>Weiner, J. H., C. E. Furlong, and L. A. Heppel <i>1971</i>. A binding protein for L-glutamine and its relation to active transport in <i>E.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A binding protein for L-glutamine and its relation to active transport in E.">google scholar</a></td></tr>
<tr><td><a name="ref821"></a>821</td><td>Weiner, J. H., and L. A. Heppel <i>1971</i>. A binding protein for glutamine and its relation to active transport in <i>E. coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A binding protein for glutamine and its relation to active transport in E. coli.">google scholar</a></td></tr>
<tr><td><a name="ref822"></a>822</td><td>White, B. J., S. J. Hochhauser, N. M. Cintron, and B. Weiss <i>1976</i>. Genetic mapping of <i>xthA,</i> the structural gene for exonuclease III in <i>Escherichia coli</i> strain K12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic mapping of xthA, the structural gene for exonuclease III in Escherichia coli strain K12">google scholar</a></td></tr>
<tr><td><a name="ref823"></a>823</td><td>White, R. J., and P. W. West <i>1970</i>. An examination of the inhibitory effects of N- iodoacetylglucosamine on <i>Escherichia coli</i> and isolation of resistant mutants. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=An examination of the inhibitory effects of N- iodoacetylglucosamine on Escherichia coli and isolation of resistant mutants">google scholar</a></td></tr>
<tr><td><a name="ref824"></a>824</td><td>Whitfield, H. J., Jr <i>1971</i>. Purification and properties of the wild type and a feedback resistant phosphoribosyladenosine triphosphate pyrophosphate phosphoribosyltransferase, the first enzyme of histidine biosynthesis in <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Purification and properties of the wild type and a feedback resistant phosphoribosyladenosine triphosphate pyrophosphate phosphoribosyltransferase, the first enzyme of histidine biosynthesis in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref825"></a>825</td><td>Wiater, A., M. Filutowicz, and D. Hulanicka <i>1982</i>. A new class of mutants of the <i>cysB</i> regulatory gene for cysteine biosynthesis in <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A new class of mutants of the cysB regulatory gene for cysteine biosynthesis in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref826"></a>826</td><td>Wiater, A., and D. Hulanicka <i>1978</i>. The regulatory <i>cysK</i> mutant of <i>S. typhimurium.</i> <i>Acta</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The regulatory cysK mutant of S. typhimurium. Acta">google scholar</a></td></tr>
<tr><td><a name="ref827"></a>827</td><td>Wiater, A., and D. Hulanicka <i>1979</i>. Properties of <i>cysK</i> mutants of <i>Escherichia coli</i> K12. <i>Acta</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Properties of cysK mutants of Escherichia coli K12. Acta">google scholar</a></td></tr>
<tr><td><a name="ref828"></a>828</td><td>Wiater, A., and T. Klopotowski <i>1972</i>. Mutations rendering <i>Salmonella typhimurium</i> resistant to 3-aminotriazole in the presence of histidine. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutations rendering Salmonella typhimurium resistant to 3-aminotriazole in the presence of histidine">google scholar</a></td></tr>
<tr><td><a name="ref829"></a>829</td><td>Wijsman, H. J. W., and H. C. Pafort <i>1974</i>. Pleiotropic mutations in <i>Escherichia coli</i> conferring tolerance to glycine and sensitivity to penicillin. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Pleiotropic mutations in Escherichia coli conferring tolerance to glycine and sensitivity to penicillin">google scholar</a></td></tr>
<tr><td><a name="ref830"></a>830</td><td>Wild, D. G <i>1988</i>. Reversion from erythromycin dependence in <i>Escherichia coli:</i> strains altered in ribosomal sub-unit association and ribosome assembly. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Reversion from erythromycin dependence in Escherichia coli: strains altered in ribosomal sub-unit association and ribosome assembly">google scholar</a></td></tr>
<tr><td><a name="ref831"></a>831</td><td>Wild, J., and T. Klopotowski <i>1975</i>. Insensitivity of D-amino acid dehydrogenase synthesis to catabolite repression in <i>dadR</i> mutants of <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Insensitivity of D-amino acid dehydrogenase synthesis to catabolite repression in dadR mutants of Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref832"></a>832</td><td>Wild, J., and T. Klopotowski <i>1981</i>. D-amino acid dehydrogenase of <i>Escherichia coli</i> K12: positive selection of mutants defective in the enzyme activity and localization of the structural gene. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=D-amino acid dehydrogenase of Escherichia coli K12: positive selection of mutants defective in the enzyme activity and localization of the structural gene">google scholar</a></td></tr>
<tr><td><a name="ref833"></a>833</td><td>Wilkinson, R. C., P. Gemski, Jr., and B. A. D. Stocker <i>1972</i>. Non-smooth mutants of <i>Salmonella typhimurium:</i> differentiation by phage sensitivity and genetic mapping. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Non-smooth mutants of Salmonella typhimurium: differentiation by phage sensitivity and genetic mapping">google scholar</a></td></tr>
<tr><td><a name="ref834"></a>834</td><td>Williams, J. C., C. E. Lee, and J. R. Wild <i>1980</i>. Genetic and biochemical characterization of distinct transport systems for uracil, uridine, and cytidine in <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetic and biochemical characterization of distinct transport systems for uracil, uridine, and cytidine in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref835"></a>835</td><td>Williams, M. D., J. A. Fuchs, and M. C. Flickinger <i>1991</i>. Null mutations in the stringent starvation protein of <i>Escherichia coli</i> disrupt lytic development of bacteriophage P1. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Null mutations in the stringent starvation protein of Escherichia coli disrupt lytic development of bacteriophage P1">google scholar</a></td></tr>
<tr><td><a name="ref836"></a>836</td><td>Williams, M. V., T. J. Kerr, R. D. Lemmon, and C. J. Tritz <i>1980</i>. Azaserine resistance in <i>Escherichia coli:</i> chromosomal location of multiple genes. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Azaserine resistance in Escherichia coli: chromosomal location of multiple genes">google scholar</a></td></tr>
<tr><td><a name="ref837"></a>837</td><td>Williams, M. V., J. J. Rowe, T. J. Kerr, and G. J. Tritz <i>1977</i>. Studies on the modes of action of azaserine in <i>Escherichia coli</i> mechanism of resistance to azaserine. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Studies on the modes of action of azaserine in Escherichia coli mechanism of resistance to azaserine">google scholar</a></td></tr>
<tr><td><a name="ref838"></a>838</td><td>Willsky, G. R., R. L. Bennett, and M. H. Malamy <i>1973</i>. Inorganic phosphate transport in <i>Escherichia coli:</i> involvement of two genes which play a role in alkaline phosphatase regulation. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Inorganic phosphate transport in Escherichia coli: involvement of two genes which play a role in alkaline phosphatase regulation">google scholar</a></td></tr>
<tr><td><a name="ref839"></a>839</td><td>Wittenbach, V. A., D. Rayner, and J. V. Schloss <i>1992</i>. Pressure points in the biosynthetic pathway for branched chain amino acids.</td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Pressure points in the biosynthetic pathway for branched chain amino acids">google scholar</a></td></tr>
<tr><td><a name="ref840"></a>840</td><td>Wittman, H. G., C. Stoffler, D. Apirion, L. Rosen, K. Tanaka, M. Tamaki, R. Takata, S. Dekio, E. Otaka, and S. Osawa <i>1973</i>. Biochemical and genetic studies on two different types of erythromycin resistant mutants of <i>Escherichia coli</i> with altered ribosomal proteins. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Biochemical and genetic studies on two different types of erythromycin resistant mutants of Escherichia coli with altered ribosomal proteins">google scholar</a></td></tr>
<tr><td><a name="ref841"></a>841</td><td>Wolfner, M. D. Yep, F. Messenguy, and G. Fink <i>1975</i>. Integration of amino acid biosynthesis into the cell cycle of <i>Saccharomyces</i> <i>cerevisiae.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Integration of amino acid biosynthesis into the cell cycle of Saccharomyces cerevisiae.">google scholar</a></td></tr>
<tr><td><a name="ref842"></a>842</td><td>Wolfson, J. S., D. C. Hooper, G. L. McHugh, M. A. Bozza, and M. N. Swartz <i>1990</i>. Mutants of <i>Escherichia coli</i> K12 exhibiting reduced killing by both quinolone and beta-lactam antimicrobial agents. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Mutants of Escherichia coli K12 exhibiting reduced killing by both quinolone and beta-lactam antimicrobial agents">google scholar</a></td></tr>
<tr><td><a name="ref843"></a>843</td><td>Wood, J. M <i>1981</i>. Genetics of L-proline utilization in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genetics of L-proline utilization in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref844"></a>844</td><td>Woodward, M. J., and H. P. Charles <i>1982</i>. Genes for L-sorbose utilization in <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Genes for L-sorbose utilization in Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref845"></a>845</td><td>Woodward, M. J., and H. P. Charles <i>1983</i>. Polymorphism in <i>Escherichia coli: rtl, atl,</i> and <i>gat</i> regions behave as chromosomal alterations. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Polymorphism in Escherichia coli: rtl, atl, and gat regions behave as chromosomal alterations">google scholar</a></td></tr>
<tr><td><a name="ref846"></a>846</td><td>Wooldridge, K. G., and P. H. Williams <i>1991</i>. Sensitivity of <i>Escherichia coli</i> to cloacin DF13 involves the major outer membrane protein OmpF. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Sensitivity of Escherichia coli to cloacin DF13 involves the major outer membrane protein OmpF">google scholar</a></td></tr>
<tr><td><a name="ref847"></a>847</td><td>Wu, R., S. Yang, K. Jin, X. Chu, and M. Li <i>1987</i>. Cloning and mutagenesis of threonine operon. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Cloning and mutagenesis of threonine operon">google scholar</a></td></tr>
<tr><td><a name="ref848"></a>848</td><td>Wu, T. T <i>1976</i>. Growth of a mutant of <i>Escherichia coli</i> K12 on xylitol by recruiting enzymes for D-xylose and L-1,2-propanediol. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Growth of a mutant of Escherichia coli K12 on xylitol by recruiting enzymes for D-xylose and L-1,2-propanediol">google scholar</a></td></tr>
<tr><td><a name="ref849"></a>849</td><td>Wu, T. T <i>1976</i>. Growth on D-arabitol of a mutant strain of <i>Escherichia coli</i> K12 using a novel dehydrogenase and enzymes related to L-1,2-propanediol and D-xylose metabolism. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Growth on D-arabitol of a mutant strain of Escherichia coli K12 using a novel dehydrogenase and enzymes related to L-1,2-propanediol and D-xylose metabolism">google scholar</a></td></tr>
<tr><td><a name="ref850"></a>850</td><td>Xu, S., and T. Qi <i>1990</i>. Enhancing effect of chuangxinmycin on synthesis of enzymes in tryptophan synthesis pathway of chuangxinmycin resistant mutant strain of <i>E. coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Enhancing effect of chuangxinmycin on synthesis of enzymes in tryptophan synthesis pathway of chuangxinmycin resistant mutant strain of E. coli.">google scholar</a></td></tr>
<tr><td><a name="ref851"></a>851</td><td>Yadav, N., R. E. McDevitt, S. Benard, and S. C. Falco <i>1986</i>. Single amino acid substitutions in the enzyme acetolactate synthase confer resistance to the herbicide sulfometuron methyl. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Single amino acid substitutions in the enzyme acetolactate synthase confer resistance to the herbicide sulfometuron methyl">google scholar</a></td></tr>
<tr><td><a name="ref852"></a>852</td><td>Yagil, E., and H. Be’eri <i>1977</i>. Arsenate-resistant alkaline phosphatase-constitutive mutants of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Arsenate-resistant alkaline phosphatase-constitutive mutants of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref853"></a>853</td><td>Yamada, T., and J. Davies <i>1971</i>. A genetic and biochemical study of streptomycin- and spectinomycin-resistance in <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A genetic and biochemical study of streptomycin- and spectinomycin-resistance in Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref854"></a>854</td><td>Yamagata, H., C. Ippolito, M. Inukai, and M. Inouye <i>1982</i>. Temperature-sensitive processing of outer membrane lipoprotein in an <i>Escherichia coli</i> mutant. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Temperature-sensitive processing of outer membrane lipoprotein in an Escherichia coli mutant">google scholar</a></td></tr>
<tr><td><a name="ref855"></a>855</td><td>Yamagishi, J., H. Yoshida, M. Yamayoshi, and S. Nakamura <i>1986</i>. Nalidixic acid resistant mutations of the <i>gyrB</i> gene of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Nalidixic acid resistant mutations of the gyrB gene of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref856"></a>856</td><td>Yamamoto, T., and Y. Fujiwara <i>1990</i>. Uracil DNA glycosylase causes 5-bromodeoxyuridine photosensitization in <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Uracil DNA glycosylase causes 5-bromodeoxyuridine photosensitization in Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref857"></a>857</td><td>Yamasaki, M., R. Aono, and G. Tamura <i>1976</i>. FL-1060 binding protein of <i>Escherichia coli</i> is probably under the control of cyclic AMP. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=FL-1060 binding protein of Escherichia coli is probably under the control of cyclic AMP">google scholar</a></td></tr>
<tr><td><a name="ref858"></a>858</td><td>Yanagisawa, T., J. T. Lee, H. C. Wu, and M. Kawakam <i>1994</i>. Relationship of protein structure of isoleucyl-tRNA synthetase with pseudomonic acid resistance of <i>Escherichia coli:</i> a proposed mode of action of pseudomonic acid as an inhibitor of isoleucyl-tRNA synthetase. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Relationship of protein structure of isoleucyl-tRNA synthetase with pseudomonic acid resistance of Escherichia coli: a proposed mode of action of pseudomonic acid as an inhibitor of isoleucyl-tRNA synthetase">google scholar</a></td></tr>
<tr><td><a name="ref859"></a>859</td><td>Yarmolinsky, M. B., H. Wiesmeyer, H. M. Kalckar, and E. Jordan <i>1959</i>. Hereditary defects in galactose metabolism in <i>Escherichia coli</i> mutants. II. Galactose-induced sensitivity. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Hereditary defects in galactose metabolism in Escherichia coli mutants. II. Galactose-induced sensitivity">google scholar</a></td></tr>
<tr><td><a name="ref860"></a>860</td><td>Yoshida, H., M. Bogaki, M. Nakamura, and S. Nakamura <i>1990</i>. Quinolone resistance- determining region in the DNA gyrase <i>gyrA</i> gene of <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Quinolone resistance- determining region in the DNA gyrase gyrA gene of Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref861"></a>861</td><td>Yoshikawa, M., A. Okuyuma, and N. Tanaka <i>1975</i>. A third kasugamycin resistance locus, <i>ksgC,</i> affecting ribosomal protein S2 in <i>Escherichia coli</i> K-12. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A third kasugamycin resistance locus, ksgC, affecting ribosomal protein S2 in Escherichia coli K-12">google scholar</a></td></tr>
<tr><td><a name="ref862"></a>862</td><td>Yudkin, M. D <i>1977</i>. Unstable mutations that relieve catabolite repression of tryptophanase synthesis by <i>Escherichia coli.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Unstable mutations that relieve catabolite repression of tryptophanase synthesis by Escherichia coli.">google scholar</a></td></tr>
<tr><td><a name="ref863"></a>863</td><td>Yura, T., and K. Igarishi <i>1968</i>. RNA polymerase mutants of <i>Escherichia coli.</i> I. Mutants resistant to streptovaricin. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=RNA polymerase mutants of Escherichia coli. I. Mutants resistant to streptovaricin">google scholar</a></td></tr>
<tr><td><a name="ref864"></a>864</td><td>Yura, T., and C. Wada <i>1968</i>. Phenethyl alcohol resistance in <i>Escherichia coli.</i> I. Resistance of strain C600 and its relation to azide resistance. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Phenethyl alcohol resistance in Escherichia coli. I. Resistance of strain C600 and its relation to azide resistance">google scholar</a></td></tr>
<tr><td><a name="ref865"></a>865</td><td>Zak, V. L., and R. A. Kelln <i>1978</i>. 5-Fluoroorotate-resistant mutants of <i>Salmonella typhimurium.</i> </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=5-Fluoroorotate-resistant mutants of Salmonella typhimurium.">google scholar</a></td></tr>
<tr><td><a name="ref866"></a>866</td><td>Zak, V. L., and R. A. Kelln <i>1981</i>. A <i>Salmonella typhimurium</i> mutant dependent upon carbamyl aspartate for resistance to 5-fluorouracil is specifically altered in ubiquinone biosynthesis. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=A Salmonella typhimurium mutant dependent upon carbamyl aspartate for resistance to 5-fluorouracil is specifically altered in ubiquinone biosynthesis">google scholar</a></td></tr>
<tr><td><a name="ref867"></a>867</td><td>Zissler, J., E. Signer, and F. Schaefer <i>1971</i>. The role of recombination in growth of bacteriophage lambda. I. The gamma gene.</td><td><a href="https://scholar.google.ch/scholar?hl=en&q=The role of recombination in growth of bacteriophage lambda. I. The gamma gene">google scholar</a></td></tr>
<tr><td><a name="ref868"></a>868</td><td>Zwiebel, L. J., M. Inukai, K. Nakamura, and M. Inouye <i>1981</i>. Preferential selection of deletion mutations of the outer membrane lipoprotein gene of <i>Escherichia coli</i> by globomycin. </td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Preferential selection of deletion mutations of the outer membrane lipoprotein gene of Escherichia coli by globomycin">google scholar</a></td></tr>
<tr><td><a name="ref869"></a>869</td><td>Comai, L. <i>1985</i>. Inhibition resistant 5-enolpyruvyl-3-phosphoshikimate synthetase, production and use.</td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Inhibition resistant 5-enolpyruvyl-3-phosphoshikimate synthetase, production and use">google scholar</a></td></tr>
<tr><td><a name="ref870"></a>870</td><td>Lin, S. C. L., D. T. Palmer, and H. I. Miller <i>1986</i>. Phage resistant E. Coli bacterium and method for making the same.</td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Phage resistant E. Coli bacterium and method for making the same">google scholar</a></td></tr>
<tr><td><a name="ref871"></a>871</td><td>Miller, E. S. <i>1984</i>. Thesis.</td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Thesis">google scholar</a></td></tr>
<tr><td><a name="ref872"></a>872</td><td>Epelbaum, S., Z. Barak, R. Larossa, and D. Chipman <i>0000</i>. Unpublished data.</td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Unpublished data">google scholar</a></td></tr>
<tr><td><a name="ref873"></a>873</td><td>Freitag, C. S. <i>1982</i>. Thesis.</td><td><a href="https://scholar.google.ch/scholar?hl=en&q=Thesis">google scholar</a></td></tr>
</table>
</div></body>
</html>
